{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# ATAC-seq Analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "### header ###\n",
    "__author__ = \"Jenhan Tao\"\n",
    "__license__ = \"BSD\"\n",
    "__email__ = \"jenhantao@gmail.com\"\n",
    "%load_ext autoreload\n",
    "%autoreload 2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/jenhan/.local/lib/python3.4/site-packages/sklearn/externals/joblib/_multiprocessing_helpers.py:29: UserWarning: [Errno 13] Permission denied.  joblib will operate in serial mode\n",
      "  warnings.warn('%s.  joblib will operate in serial mode' % (e,))\n"
     ]
    }
   ],
   "source": [
    "### imports ###\n",
    "import sys\n",
    "import os\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "import argparse\n",
    "import matplotlib\n",
    "import itertools\n",
    "import scipy\n",
    "import matplotlib.pyplot as plt \n",
    "import seaborn as sns\n",
    "from IPython.display import Image, display\n",
    "from scipy.spatial.distance import pdist, squareform\n",
    "from scipy.cluster.hierarchy import linkage, dendrogram\n",
    "import scipy\n",
    "import scipy.stats\n",
    "import sklearn\n",
    "from sklearn import preprocessing\n",
    "from sklearn import decomposition\n",
    "from mpl_toolkits.mplot3d import Axes3D\n",
    "from matplotlib_venn import venn3, venn2\n",
    "import pickle\n",
    "from matplotlib import patches as mpatches\n",
    "from sklearn.cluster import KMeans\n",
    "from sklearn import svm, datasets\n",
    "from sklearn.utils import shuffle\n",
    "from sklearn.metrics import roc_curve, auc\n",
    "from sklearn import ensemble\n",
    "from sklearn import neighbors\n",
    "import networkx as nx\n",
    "import matplotlib_venn\n",
    "from sklearn.cross_validation import train_test_split\n",
    "from random import shuffle\n",
    "%matplotlib inline\n",
    "matplotlib.pylab.rcParams['savefig.dpi'] = 200\n",
    "sys.setrecursionlimit(3000)\n",
    "sns.set_context('notebook')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Functions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "\n",
    "def get_matched_split(features, labels, test_size):\n",
    "    '''\n",
    "    feature: 2D array (samples x features)\n",
    "    labels: 1D boolean array (samples x)\n",
    "    test_size: fraction of data to test on\n",
    "    '''\n",
    "    \n",
    "    ### match GC content of samples labelled True with those labelled False by thowing out False samples\n",
    "    # retrieve sequences using index of labels\n",
    "    index_label_tuples = tuple(zip(labels.index.values, labels.values))\n",
    "    \n",
    "    true_ids = [x[0] for x in index_label_tuples if x[1]]\n",
    "    \n",
    "    false_ids = [x[0] for x in index_label_tuples if not x[1]]\n",
    "    \n",
    "        \n",
    "    filtered_ids = true_ids + false_ids\n",
    "    filtered_features = features[features.index.isin(filtered_ids)]\n",
    "    filtered_labels = labels[labels.index.isin(filtered_ids)]\n",
    "\n",
    "\n",
    "    if test_size <= 0.5:\n",
    "        training_indices, test_indices = next(iter(\n",
    "                sklearn.cross_validation.StratifiedKFold(filtered_labels, int(1/test_size), shuffle=True)))\n",
    "    else:\n",
    "        test_indices, training_indices = next(\n",
    "            iter(sklearn.cross_validation.StratifiedKFold(filtered_labels, int(1/(1-test_size)), shuffle=True)))\n",
    "    training_ids = [filtered_ids[i] for i in training_indices]\n",
    "    test_ids = [filtered_ids[i] for i in test_indices]\n",
    "    \n",
    "    training_features = filtered_features[filtered_features.index.isin(training_ids)]\n",
    "    test_features = filtered_features[filtered_features.index.isin(test_ids)]\n",
    "    training_labels = filtered_labels[filtered_labels.index.isin(training_ids)]\n",
    "    test_labels = filtered_labels[filtered_labels.index.isin(test_ids)]\n",
    "    \n",
    "    return training_features, test_features, training_labels, test_labels\n",
    "    "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Preprocessing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "# file directories\n",
    "workingDirectory = '/home/jenhan/analysis/atac_analysis/'\n",
    "\n",
    "genomes = ['hg19','mm10', 'hg38']\n",
    "scratchDirectory = workingDirectory + '/scratch/'\n",
    "os.chdir(workingDirectory)\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### ATAC-seq"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "\n",
    "tagDirPath = workingDirectory + '/tag_directories_atac/'\n",
    "\n",
    "strains = []\n",
    "treatments = []\n",
    "owners = []\n",
    "dates = []\n",
    "originalNames = []\n",
    "cellTypes = []\n",
    "for f in os.listdir(tagDirPath):\n",
    "    originalName = f\n",
    "    metaDataTokens = f.split('_')\n",
    "    \n",
    "    originalNames.append(originalName)\n",
    "    # grab metadata from string tokens\n",
    "    strain = metaDataTokens[0].strip()\n",
    "    cellType = metaDataTokens[1].strip()\n",
    "    treatment = metaDataTokens[3].strip().split('-')[0]\n",
    "    owner = metaDataTokens[5].strip()\n",
    "    date = metaDataTokens[4].strip()\n",
    "    \n",
    "    # append to overall lists\n",
    "    strains.append(strain)\n",
    "    treatments.append(treatment)\n",
    "    owners.append(owner)\n",
    "    dates.append(date)\n",
    "    cellTypes.append(cellType)\n",
    "metadata_frame = pd.DataFrame([strains, \n",
    "                                  cellTypes,\n",
    "                                  treatments, \n",
    "                                  owners, \n",
    "                                  originalNames, \n",
    "                                  ]).transpose()\n",
    "metadata_frame.columns = ['strain', \n",
    "                          'Cell Type',\n",
    "                         'treatment',\n",
    "                         'owner', \n",
    "                         'sampleName']\n",
    "\n",
    "# ### read in log files ###\n",
    "\n",
    "# create data frame summarizing mapping quality logs\n",
    "\n",
    "# from tagInfo.txt\n",
    "_originalNames =[]\n",
    "_totalReads = []\n",
    "_fragmentLengthEstimate =[]\n",
    "_peakSizeEstimate = []\n",
    "_tagsPerBP = []\n",
    "_averageTagsPerPosition = []\n",
    "_averageTagLength = []\n",
    "_averageFragmentGCcontent = []\n",
    "\n",
    "\n",
    "for tagdir in os.listdir(tagDirPath):\n",
    "    with open(tagDirPath + '/' + tagdir + '/tagInfo.txt') as f:\n",
    "        data = f.readlines()\n",
    "        _originalNames.append(tagdir)\n",
    "        _fragmentLengthEstimate.append(float(data[2].strip().split('=')[1]))\n",
    "        _peakSizeEstimate.append(float(data[3].strip().split('=')[1]))\n",
    "        _tagsPerBP.append(float(data[4].strip().split('=')[1]))\n",
    "        _averageTagsPerPosition.append(float(data[5].strip().split('=')[1]))\n",
    "        _averageTagLength.append(float(data[6].strip().split('=')[1]))\n",
    "        _averageFragmentGCcontent.append(float(data[8].strip().split('=')[1]))\n",
    "        _totalReads.append(float(data[1].split()[2]))\n",
    "mappingStats_frame = pd.DataFrame([_originalNames,\n",
    "                                   _totalReads, \n",
    "\n",
    "                                   _fragmentLengthEstimate,\n",
    "                                   _peakSizeEstimate,\n",
    "                                   _tagsPerBP,\n",
    "                                   _averageTagsPerPosition,\n",
    "                                   _averageTagLength,\n",
    "                                   _averageFragmentGCcontent,\n",
    "                                  ]).transpose()\n",
    "mappingStats_frame.columns = ['sampleName',\n",
    "                              'totalReads', \n",
    "                              'fragmentLengthEstimate',\n",
    "                              'peakSizeEstimate',\n",
    "                              'tagsPerBP',\n",
    "                              'averageTagsPerPosition',\n",
    "                              'averageTagLength',\n",
    "                              'averageFragmentGCcontent']\n",
    "\n",
    "summary_frame = metadata_frame.merge(mappingStats_frame, on='sampleName')\n",
    "\n",
    "\n",
    "summary_frame.index  = pd.MultiIndex.from_arrays([list(summary_frame['strain'].values), \n",
    "                                                  list(summary_frame['Cell Type'].values), \n",
    "                                                  list(summary_frame['treatment'].values)])\n",
    "summary_frame = summary_frame.sort()\n",
    "summary_frame.to_csv('qc_metrics_atac.tsv', sep='\\t', index=False)\n",
    "summary_frame_atac = summary_frame.copy()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Filter Data on QC Metrics"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": false,
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAABWEAAAP7CAYAAAA6RExZAAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAAewgAAHsIBbtB1PgAAIABJREFUeJzs3XeYbFlZL+BfnzMzTCA5gCRJgn6gIIIO8QKCkhzCCIoC\nEkWGoAh64QoGgglBkKDkMEhSkCGDRAEVkCBBQRY5DkEYGMIMMM6p+8faNV2n6equ7tO7+/Q57/s8\n/XR1165dq/ZelX7722stTSaTAAAAAAAwjj073QAAAAAAgEOZEBYAAAAAYERCWAAAAACAEQlhAQAA\nAABGJIQFAAAAABiREBYAAAAAYERCWAAAAACAEQlhAQAAAABGJIQFAAAAABiREBYAAAAAYERCWAAA\nAACAEQlhAQAAAABGJIQFAAAAABiREBYAAAAAYERCWAAAAACAEQlhAQAAAABGJIQFAAAAABiREBYA\nAAAAYERCWAAAAACAER2x0w0AgN2qqp6d5C7Dnz/XWnvbTrZnPVV1mSSfGv58S2vtRiuuf0uS6w9/\nHvSPZ6qq7pLk2cOfD2utPWIn23Owqqrjkzw8yYlJLpFkX5L3ttaut6MNY1uteL6c0lq7+06251B0\nMG/jqrpzklOGP/+4tfanO9icw0JV3TrJqUmWkjymtfbAHW4SADtECAvAIauqbpDkn9dY5Jwk30jy\n9SQtybuSvKG19s4N3tVkcy3cMfPaO1nn+oPdbm336KrqiCRvSnLV7L+dLrHB9cwG+VMntNbeu4n2\nnJbkwjP/3jXB/yFi08+XOf1grfv5dpKvJvlAktcleWFr7Zubvf9d5KB6Taqq6yd5enq7XrRoAFtV\nz0tyh5l/ndhae+0Ct/tSkh/eTFtX+F5r7ZgF7u9iST6XZO/wr4+21q54IHdcVUcmuXmSGye5dpKL\npr9uTZKckeSTSd6b5FXpnx9+YJ+31l5eVX+U5E+T/G5VfbS19vQDaRcAu5PhCAA4HEzm/OxJcnyS\nyyf5xfQqwbdX1Qer6g5z1jXri0k+MvycOUK7U1U3qap9VfXmLVjd2eltbUk+swXr2xZVdfKwDZ41\nZ5Ezsvy4vrp9LdtVfiHLAew3kvxmkhOS/Nom1zf7PNpMld8tshxkTH/Yfea9tq7cr+dNctkkJyV5\ncpJPVtVm+x6bMFTCvyC9COczSe654O0ulOS22X+fLnTbLNY/FvnZt+D9/WZ6ADvtdz82HIzdlKq6\nR5KPJnlZkvsmuVr6gaujkpwnPWC+VpLfSvJPSVpV3XLO6v4iydvSq2EfX1U/udl2AbB7qYQF4HDx\n8fQAYGnmf0ck+aEkl0pyzSS3TnLJJFdO8ryq+pUkd22tnbHaCltrD0nykDEbneQaW7Wi1tppSX5i\nq9a3jU7IGiFda+1l6V+Sme/HZi4/ubU2L9DeiOmBjF+rqge01r6/gdvedcU6hLC701L6AagTsv9r\n66y96a+zV01yx2HZ45O8oKqObq2dsg3tJHlqeoA4SfIbrbVvLXi7u6cHjpP08PaySU6sqou31r64\nzm2vn+TIOdcdleQ/hsuT9H7x3TnLrhvCVtWeJPcY1vXZJJcZrjo5yVvXu/2KdR2T5O/Sw+dp+z6Q\nPqTAe5N8Ob1fXzLJDZLcPslFklwhycur6i9baw+eXWdrbVJVd03yoSRHJ3l+VV29tbZowAzAIUAI\nC8Dh4nuttf9e4/rnVdXvJPnVJE9IDwluneSVVXXj1tr3tqORq9iyEHYXsw0O3A/NXP7EFq3z3ekH\nLy6YHla8cJEbVdVF0ivPJ0neE/t3t9u3zmvr1NuSPLGqHpzkz9L3/+Or6jWtta+M2sLDXFX9fJar\nWV/RWltrmJ6VplWvX03y0CTPSQ8g75HkT9a6YWvtY2u06Twr/vWR1tqBnFFyi/QDqpMkz0x///6Z\nJL9UVRdqrX1tkZVU1VKSlya5ybCuryW5V2vt1FUWf3eSl1XVH6T36fsN/39QVZ3eWnv07MKttU9X\n1V+nH7y9SnoF7RM29jAB2M0MRwAAg9bavtbaC5P8dJZP179u+umzO2XNKtBDXVUdl+RKO92OQ8Bs\nleI5W7TO/04PdJeysSEJ7pReCPDdJG/ZorawS7TWZk/LPm96FSHjmg0DFz57o6punD5czyTJP6af\ncTA9IPkbQ2B5sDh55vKLhp+kV9zebQPreUiWA9ivJrn+nAD2XK21M1trD0gPpZeGnz+tqsuvsvgj\n04eEWUry0OE9DoDDhEpYAFihtfaFqjox/VTJ8yS5c1U9sbX2vtnlqurZSe4y/LnqpELDl7DfSPJz\n6aeEnz89BPtq+mmJr06fPfvbM7dZbUKxn6uqc09bbK3tWWXZv2qtPaiqbpfkj5L8ePqpoBdsrX1z\nxWQ6b2mt3Wi9bVFVN0mveDohycXSTz3+WPppmU9orf3A6aMr2rTm/axo06dbaz86/H92dvFJ+hfW\nuw6ncybJZ1prl1tl2Ye11h4x5772Jvn1JLdKr5C68LDeryX5zySvTfKsedVYVfWwJH88/Hnz1trr\nqupS6WMFnpR++uskyZfSQ6bHtNb+c95j34ihku326QcFLpbk2CSnp08K86YkT22tfWGV26081XUp\nySlVdcrw97nbcZNeluT3ktywqi7dWvvsAre5c/p2elOSsxa5k6o633C7X0yvILtQ+ufYbyT5cPq+\ne3pr7etzbn+jJG8c/vz71todhrEufzu9Yu7S6dt0uu/+trX2rjnr+rv0fpQkN2utvb6qrp3kPun7\n56JJvp9+IOdlSR7XWvvGOo9vKb0P3S69Mviiw1X/k+SDSV6e5LmttbPXWc95k/zu8Jgun76Nvph+\nOvaTWmv/scbNt9Op6aeqJ72aeq6qumSSeyX5+Sy/hp6R5Avp+/SU1tqH1rvDqrpsehh3wySV5ALp\np7l/Ncn700O7F7bW1jxIsVXbeKgEvUuSW6b36Yukv15/K/15/bYkz1iwynit+7lx+oHFSZI3bnB9\n95q5/MLW2req6pVJfjm96vTmSV5zIO3bCsP7yE3TH+P7W2sfraoXpo/DupQ+VuxfLbCeH0ry+1k+\n8Hnv1tpHNtCUR6Rvm8sneX36e8x+Zx601r5dVc9Mf928YPrnA9WwAIcJlbAAsIrhi+rThj+Xkjx4\njcVXrVStqt9OD4h+Pz1oOD79C/t50seSu2n6l69WVStPyV45sc16k5RMhvv85SR/nz726xFrLbue\nqnp0+mQjt00PqY5K/9J4jfRqnvcPs1HPs5EK3tWWXTlh00LbYDXDJCgfSQ9rfyk9QDgmfWy+S6aH\nCU9I8omqut46bZwk2VtVN0wPbx+YHg5NJ2u5THpg+K6qutm8Ni2iqs5fVa9N8ob0atMfTw+hjsjy\npDB/mOTjVXW/VVax2sRXm5nwZp6XDL+XskC1WVVdLclPDX++eJE7GGZ0/1iSJ6bvp0um77cj0kOO\n66f3xw9X1bXmrGYarE2SHFdVV0of2/GPh/ZcMH3/XTq9Uvffqup3F1zX/dIDszuk7/uj0/fRTw3r\n/8+qmjsW8xAgvSt9W/5q+pibxww/l04P6Z4xrOen5qwmVXWF9Nebh6VPIHS+YR0/mr5v3llV95l3\n+202O/zA8fMWqqr7pweSf5D9X0MvlL59fy/9degv17qzYaiZj6QfnLpuer85MsuvxbdIHwP0HVV1\n4TXWsyXbeOh//53kKUlOTH89Ojr9NP8LJrl6kgek7/M/Wm9965gNUp+66I2q6uLpfW+SfoDsX4er\nTplZ7OSVt9sh98ry99rnJElr7fNJ3pz+2nSF4fV6PfdNMq1Mfdd6FbArDeO7npjkh1trt26t/fuc\nRWf3w703ch8A7G5CWACY72+H30tJbjJUUi5kqLx7fHpg8J0kf54eul4tPQS4a3qwNkly8fSxZy8w\n3Pxd6ZVRNx3uezp25lWGn3lBzJFJHpv+5f4O6dWeNxruf6PumB5wvDe9EvZaSa6d5HfSq74m6cHj\nS+at4AC9NP2x/kaWt8HLs7wNbrroioaQ623pQckkyb8P671WeoXv7dMrKSfpFYivq6qfXme1lx3a\neEaS+6dvm59N324fGNZ1VJJnVdWmzjwabvfGLFd4fT498L1+ekhzi/Qv82cP9/XXVXWvFauZbq+n\nDH9P0kPbDW/H1bTW3pnk0+n76C5rL51kOag9K337rWmognx5epXgJD1UuXP6c+jqSW6TfqBguu9e\nOVSzreXo9ErMC6dXx900/blyy/TK1elkYY9aIES/UZK/Tg+J75PeD66ZHk59YljXJYd2rRwDczrz\n/NuG+58k+bf0MTivN6znzulV5ZP0AP6NVfWjq6znyPSq+ksOy34svfrvmkObTk4PIR+f5P+s85i2\nwyVmLq86VmdVPSj99eyIJN9OP6X+pPRtdeP0KsdvpO+rB1bVY+es5xbp++jI9G3z7GE91xh+7pH+\nmjkZ1v2iOevZkm08rOflWa6cf0X6a9A1hvu/RfoBoTOHx/awqrrDvPWtpaqOTTLtw2elv84t6p5Z\nPmvy2TP//6ckp6U/528+PEd3zPA6OX1dOTvJ82eunp2AcJHA+MSZy0/fTHtaa59Zb9Kz1trHk/xX\n+jb88aq6ymbuC4Ddx3AEADDHcErj59KrlM6XHtr924I3n62GunNrbWXg9M4kzx2Cg/unB0L3SD+F\n/az0qr7Z8PQ7rbUPr3Oft02v0rtOa+2MBds5z2+mhwO3WTF787uq6hXpQzUcn+RaVXXL1torD/D+\n9tNa+2b6NrjIzL+/scA2WM3T0iemmiR5QWvtTiuu/48kL6qqx6VPrHKe9C/gJ6yxzr9ID2FuNLR1\n6n1D5epH04PDi6aPL7iZU3YfmB7sTtKHrrj+itPaP5DktVX1hvQwfJIeHL60tfblJJlur6qarTz8\nwia34zynpFcGXqaqfr619qbVFhrCp9sP7Tx1OC13vXX/dvpp45P08OcWrbXZqt4PpM9G/qr0oQqO\nT3/u/dka67xxelhzo9ba7PP5/UleU1VPSJ8wJ+kVtv+0xrrumz45z41aa7PP1/dU1UuSvC+9mvWy\n6RVvj1tx+7/N8mRCj2mtPWjF9e9Jn0X9kUkelF4B+oT0oG7WPdMPikzSK0dPWNEv3zUMo/C69AMQ\nO+02M5ffufLKoXJ4OnnXl5LcYAiuZr25qp6R5B3pz7P7VdXfrzKMxB9muRL8D1prj1xx/Xur6tT0\n59glktygqq67om8kW7eNb5rkCsN6XtlaO2nF9e9Pf14/b3hse9PPxHjBGuuc5+fSK3UnSd42vLes\nq6r2ZPkx7MtQXZr0as+qes7QpukEXQ/fRNu2ym3TzwqYTjo2G+q/NMnX01//T6qqi7TW/me1lQyB\n9c/M/Gsjk5dtxmuSXHm4fGL6WRUAHOJUwgLA2v5r5vKlNnC72cmkXrfGcg9PDxtOzvKYlZt1ySQP\n34IAdil90qR7rQhgkyTDuJ9PnPnXrx3g/Y2mqq6YHrpN0qvm1jpV+PfTT5NeSnL1qpoXwk4nFLrr\nihAmSTJs/9nQ/WdWLrNAu/emB3xT9543rugQ8L9yaNdx6dWT2+mULIdca03Qdcv0EHF6m0V8dlj2\n1UkeuyKAnfWkmcvrjXU8SfK0VUK2qT9Mr7xcSnKVoQ+tZlqhfZ8VAWySZBifdnZ84v2eJ8Op7b88\nrOMDqwSwsx6S5ONZrj5cWQ17x5nLD5vTL7+ffnBl5fAU26qqHpDkOsOf30nywlUWmwZ8SXLfVQLY\nJH22+fRwOunb5rdmrx/CxH9LDzBfnznVjcNzdjbkXK0PbdU2nn1veP2cZdJae2/6a8DvpffJzbj2\nzOW3b+B2t0ryIxnGbm6tfW7F9bOVsTs9Qdds9f9s5Wtaa9/L8n49MmsPmXK5LBcofau19qk1lt0K\ns0MVzBtGBYBDjErYg9xQNfKw9A+Ye7PGhCM7aZjgY0Nf+qaTygAc5Garai4yd6kfNDth1c+mn3L8\nA4Yv/wc65l/SA4h96adTH6hJkte31r60xjKvSvLQ4fK8MVQPBrNVZqeudZpoa+27VfXSLJ+2+ovp\nVY4rTZK8u7X2X6tcNzU7+c0PL9rYGSekV+VNknxijcBw6gXpIWfS2/3oNZbdUq21z1bVm9MnTjqp\nqi4w50DAXYffn2utvXnBdT9p/aWS9KrkqUvMXWo5OH3OvAWGSezelr4dk96/V5ucZ5Lkv4ewbJ5X\nzNzvz1TV0TOT2d0qvSBikj4e6VxD9eHzs/ycu2X6ae/TCr7p5FbnpAfy89bz8ap6e/pwDtsSxA4B\n3QXTD0bcO31M5gz3f//W2ldXWX5a6fuNLG/DeV6cPizH0VlRITwcRPq9BZs6tw9t8TZe+d4wV2tt\nU6fEz5gduub9G7jd7Kn7z1x55fAY35Y+NMol0ys5X7WpFh6A6qX0N0jfzl/I6gc8n5nlA1r3SPKo\nOaubHiCapE/WNrbZ/bHe8DcAHCKEsAexqrpq+ofyq2SHqxYW8Jks9uHuwulH1r8/bnMAtsyZM5eP\n3cDt3pw+/utSkpcNk6s8d7XqqS30lfVmYd+AeROKTE1nI19KcsmqOm61asCDwGzIsd5jSvoYuFNX\nXWO596yzntn9vJF+MzXb7pWnV69m2u6lrN3usTw7PYQ9On084ifPXllVF00fm3LdwHE9VfXD6UMU\nHDXz79lxKX9g7NUVvp8+TMBaPpjlEPbH11huzT7VWvtqVX0pycXSA9fLZ/m5c92ZRT+x8rarmO1z\ns/v4SlkOlz+7wGvMu1fc94GYJDlvVW1kgrdJ+vik92+tPWuV66+c5SEoPr1aNf6s4eDJh9JD3gtU\n1aWHav1VDVXmF0+vZp8d53v2INvKPrSV23h6AGIpyV2H0PlRw2SQW+2yM5c/vcgNqupy6UOoJMnp\nmX9g71npIWzSQ9ttD2Gz/6RWp6xWKd9ae39VvT896Lz8GkOmnHfm8pmrXL/VPpse6O9Ncomq2rNe\nXwdg9xPCHoSGU6f+X3q1w5HpEwTcLgdxCNtae2iWqzNWNXzIfEf6F6V5R6EBDjazM3evOoHMHH+a\nPpP7ldKrwJ6Y5LFV9e4kb0kfeuDfWmtnb1E7J0lWHetukz691pVD8DEday/p1Z5jn765GbNDSCzS\nvk/PXF5rwpn1+sLsl+nNnKp7IO2+QFUds+j4j1vk1PRJys6fPiTBk1dcf6f0z50bDmGHs4Lukj6e\n7DUzP9SeZP1tPUmvxD1nneVOm7l80TWWWyQ8PS09hJ2uaxrCzlZbvnyB8XGnlrJ//5hdzxcWuP1n\nFr2jBW3k8+n70ielevoalfazj+dqGwx4k75t9gthq+r49PGeb5Ue8m70O9CWbePW2oeq6s/Th1xI\net++S1V9Mv294c1J3jBv7NINuvjM5dPmLrW/e2U5cH7+Gu9RL04fn/j8SW5WVT/SWvv8plu6QVV1\ndPY/C+/Z85ZNr4adDqFzcpLVQtjZ6v0LrHL9lmqtnVNVX07vW3vTXxu+OPb9ArCznA5+cDoxfXzA\n7yW5W2vtoB1rb4Punz7z6yey9oQZAAeT2S+xC5+iOJyOfZ30U4a/k/6F9ojhfw9J/6L9lap6ZlWt\nVWm3EV/fovUkyZqzOw9mq4WO28L73kqzX6a/vcDys8ucf43l1gvxDtSG2t1a+9/sf5bJWm3fcsMp\n9i/M8ni6V16xyJ3TnwPvmDe+52qGCtq3p0+udsMsTzK02s+itrJvH8i6ppPFJfMf07yf2f07W8G3\nSDX6Is+DRS2lV7VeOf3MrdV+XpLlcPxdrbU/WWeokx+aubzR7ZKs6PtVdZ0kH07yx+nVkHvXuf1q\ntnQbt9b+MP3AxEdn7vtH0w9gPC/JF6vqrVX1Kwvc11o21O7hgMddZ/41N9gcDvL8/fDnnvSxcLfT\n7dMPcE4nHVvrYNUL0oeBWEpyqxUTPk5N39+XklxkKIoZ2254DwVgC6mEPTjtSZ8p9g7DhAMbUlU/\nluR30icVuPTw7y+nH13/29baf2xNMzfUpsukT04xSZ9c5Hvb3QaAjaqqo7L/pEof2Mjth1NWf7eq\n/iR9Buebp782T0OC86dPFHLnqnpga23lzOkbtZWnMi4SMs5WHR6sp1HOBiuLVKTOfvHeyce00Xav\nXG4n2v7sLE+S8xtJHpAkVfUz6SHdJGtXq63mxenPwUl6sPWo9Em6Pp/k9Onpu8PnjE9lsTD2fxdY\nZrYfrPVcOJB17ctyxeGvZ2OvL/OqnBd5/Fv9+X/fWqfSV9XvJLlpkvMluWdVPa+1ttYkUbN99+3Z\nf3zSRZxbhTqE+C9NHw5rkuSTSR6ZPkb3/8wO31JVd8li/XNLtnFr7QVJXlBVN04f4/dm6cNVJL1f\nXC/J9arqnkluu8mhbM4dVmHBz96/nD4sw7Sq/H0LVmgvJbl7VT18G0+pn/aLpSQ32EDF9JHpYfdf\nrvj/J9JD0WPTt9vVsv/wNGOYHR94vWFUADgECGEPTv+a5JWb+RBTVXdKrxY5T/qH2C+kfzi5bPoX\n/btU1YNba9s2Ycfgqekfal44ZxwmgIPRjdLHuJwk+eRmZ0seZkl/RpJnDNU110wPZO+UfrBsb5LH\nVNXHWmuv3pKWH7jzrr/IfqeFb7a6buzPIrNj5J5vgeVnH/dqk0ttlw21ezhgcOTMv7a97a21dw9j\nc/5kkjsOBxb+N8szkp+VPsTSQqrqBkn+T/rz77tJrt9amxdU7p3z/9Us0rdnq9LWqiA8kHXN7qOv\nttY+vMC6VjO7zkXGH17kebBlWmtfrKqHJXlM+mfSp1fVT69xmvvsdtlzANsl6QcFpqHix5L8bGtt\n3mvVWn1otG3cWntDkjck5x5MuEn6MGQ3Gha5Ufrp9Jupiv1e+ntYquo8CwSx95q5vJHq8qX00+pv\nkfUnUjtgVXW19LPrNjNU21J61e5+IWxr7X+r6p1Z3u6/mPFD2KNnLitQATgMCGEPQq21jYw5eK6q\num76h7S9Sf4hyQOmp3tV1YXSP/zeOckjq+q/W2vbMoB+Vd0q/QPlmUkeuB33CbBFfnfm8j9uxQqH\nA2zvSPKOqnpE+nAF08lF/ji9yu9gcKm1rhxmC7/g8Oe+9DMupma/GK93SueF1rn+QH0my7OaXz5D\n2LGGH11x250ye9+Xn7vUsmm7J+kTtO3UBJjPTvJX6fv1xlX1+iS/OrTrpa21RU7fn7rhzOVT1whg\nk/3321qW0icIXc+iY4Cu+TxZZ12fyHLf/PGs3zfn+crM5YvNXWrZIv1pqz0h/TT3qyS5YpI/SPKw\nOctOx9ldSvJjB3i/s33o0WsEsMnafWhbtnFr7TNJnp4eVP9C+vi5xyS5TVVdaROTd30ny0HfcVkj\n6KuqK6VX304Pejwki4WcJyS543D55GxDCJv9J+R6fZLXLni7B6aP9X25qrrxEIDPenGWQ9h7VtWf\nLzB+9H6q6kfSx8j+6yR/v9pkYTNmA/2DcWJLALaYEPbQ8sj0APZfWmu3n71iCHbvWlUXT3Lj9Alj\nRg9hh4qvR6Z/iHtsa82A88CuUFUnJfmF4c8z0w9kbamh8uYB6QfIjksPKA4WP7vO9T81/J4k+dSK\nCqvZU6XXqwg7Yfi9mcmrFvHv6VVlSXLtJE9ZZ/lrzFx+9ygtWsy/z1y+1gLLHyztfm6Sv0j/jHnL\n9HFqL5TeT56zwXXNhl3rhU+3HX4v0o/OW1U/3lr76BrLXHXm8kfWWG7N50lVXSy9EjPp22K2mv5d\nSe4wXP6FJH+71rrWMN02S+nh0tHDGL3zLNKfttQwCdG9k/xLejv/X1X9w2qhYmvt4zOT/l2oqq66\nTgC/lo30oZPWuG7bt3Fr7Y1V9YL0oT2S/v6w0RD2i1k+0HWJJKevsexssPmi1trjF7mDqrpwepXu\nUUluUlWXbq19dp2bbVpVnS99PNip31+0f1TVBbMc/p+cHzzw8Xfp81Ycn769/ijzDxbM8+T014Xn\nD79/b05bppNxJf1A5ldWWw6AQ4uJuQ4RVXWpJNcd/vzrNRb9m+H3Vapq0aqRA3GH9IqHb6VXxgAc\n9Krq6ulnFiQ9PPqT1trCk3JV1XWq6ilV9c6qWjOkGSoWp1/mz1xj0e08cLqU5MThC+s8s4HFymFm\nprN6LyW5wjoTnNx9A+3azDZ4aZbH3jxprcdUVefP8uOaJHnZJu5vq7w/ffzKpSSXqqqfX2f5u85c\nPnWsRq1nmNH91entvkmSWw9XfaG19sYNrm42zJ9bMV1VP5E+w/y04uzIecvOLHOnNdb3Q+kVgVP/\nPGfRpSTXGMbin+eXZu73X1dU1Z2aPkbsUpJfXO9zWVU9oqreXVUPHYKvJOdOAvhfw59Hpg91Mm8d\nV00PmDdzGvcBGcaBPWX486j0is95Zs88uN9a662q46vqE1X1/Kr6xRVXL9qH7p7kJzKnD23VNq6q\n81XVn1fVa6rqL+atY8bssCRrvT/M8+mZy5edt1BVHZP9nxNPW/QOhvfG6WvldkzQNT1oOUnyng0G\n9M/M8nPulsOYwecaJht70PDnUpIHV9UvZUFV9Zj0CZaTPlHmWt/JLpXlSeJO22jFLQC7kxD20HGd\nmcsfWmO5d81cvvr0QlV9uqr2bfBnkQ9oD0r/cPGUTU4oALBtqmpPVf1mkrdmeebyU1trj9rgqo5N\ncs/0Ks/HDeHevPv8xSxXCq6crGb2C/h2HDibdVzmVOZV1RWy/9iBz5+9fphUctr286afjr7aeh6e\n5EpZe3Kj2W2w4VOTh7ZMQ8nzJnlKVf1AteTwvydlebbtl7XWPrnR+9sqwymsj5351xOHoYV+QFWd\nnOTnhj+/kOUZy3fKdHKjy6VXIU/SK8w2ajZcufUw7u1+quqK6Wf2vC3L/egiVbXWJDeTJPcfxpVc\nzaPSTwFPkre21uYNRzAN2Z46p20/nH7a/dRzZ69vrX0+y/vqiPRJmlatHK+qW6SfSv0z6RV8K0+r\nn93nj6iqHxirdhhC5GlZPiixEx6UXo25lOTaVXWfOcs9LsnZw3J3qarbr7bQEB6+ML2v/VqSlbNI\nzfahea9DJ6UXKbwqy9vlMqssuhXb+Nvpk7DdLMn9qup6qywzXdf5shzin5P9P8Mv6oMzl686d6le\nNHGB9D66R4FQAAAgAElEQVT9odbaOzZ4P7PfCe62zoG3A3XPmcvrndmwn+G5PB264IischCwtfas\n9Pe0SXrg/qKq+sPVnuNTQ7j+rAyTEab33TsOz/F5fnrm8mYrvQHYZQxHcOiYPd3qo7XYTKazY6Lt\ny8arItZcvqpumD4b8r70L7YAO+k8VfWTq/z/uPTX0OsmuU2Wx/KbzuQ+LySYaziN9I3ppxhfJ8kH\nquqU9FPMv5L+xe7i6ZOY/Ppws3OS/MmK9ZxRVaennxp58eHU1Nelf1l+5WYnClvAJP3shQcNE8U8\nJf2U7L3pp/T/fpLzD8u9srX2r6us43lJfivLE/FcLslb0qu5Lp9eLXXj9C+tDx8e02o+nf4+sie9\n6vCJ6WHEDyd55uzs5mv4rfTKxoumh4KXqqonzTymn0gPtqbjc34pm9jvI3hyep+8YfpZJf8xVFq9\nK/3U9sukP57pcAtnJ7nLOqdJb4fXpI8R/MPDz2aGIkh6FfPj0vvajyb556p6dJLPpj9nb5E+6de3\n08OUt6b3rSPTJ7p7dvr4uJ9bsd53pAfyb6uqv0k/Jfn09Mq0k9Mn5En69nzIOm18Wnoo9N6qekJ6\nmDJJPw35/2V5PNgPpj8nVrpf+uRjl04fUuKDVfX49CElzs7yPv6l9OfAJMn9V9nHT0jv5xdLnxjt\nXcO2+uCwPX42yf2H9T16aNtW2NBnx9ba16rqwekTtibJn1fVy1prp61Y7r+H5R6d/rinVa4vSvL5\n9ANXV0ty3/RtN0mvHl954Og56ZWTS0nuUFVnpwds30jvK3dM70dvH9Z1i2HZaw0HN96VPinjGdmC\nbdxam1TVQ9IPShyT5A1V9Q/pAfAX0l8fj08fzuDu6f1+WsywmdPVZ8PUa6+x3MkzlzcUbCZJa+3N\nVfXxJFdIf2+7VUY4k6CqrpM+LMMkyTezuQNOT03fz0l/H1qtIvnO6WfxnZze/x6R5OSqenH6AZ8v\nph/0+ZH01+c7Zvlg6jeT/Hpr7Z/Wacc1Zy5vNPQGYJcSwh46Zmfe/c/0L/PrObeKorU2RoXVPYbf\n/zLm2FAAC5hO8PKf6yw3GX4+nuShrbUDqSi8XfoptTdMDwkeusZ9np7kbq211cbyfEqWg6BfG36m\ngcNsCLuRyrZFlv2H9C+ZD87+Z1tM2zxJ8p70L6ur+eP0CU6ulB42/Okq63hca+0JQ0XsqoYg+u/T\nxwBcSg9K7jvc/sXZv1J23jq+Un3yylcM7blWfjCQmD6mDyT5pU0GHltqCGxumR7enZT+hf9xqyw6\nSQ/379Ram3fq/LYZxv98bpL/O/zrna21j21iPWdU1T3SQ7Mj0vfZ7FALk/RA7uattdOq6oVJ/jC9\nn9xn+HlYeoAy6+z0/vTq9MrMlWHZZFjmXq21d67TzHemn4H02CwHi7PrmST5WJKThkn5Vj7Grw/B\n0qnpIeyls38F9Oy6vpnkt1prL1plPd8e+so/pYd4V8zykCpT56SPT/nxbF0Iu+GK2tba04fT/6+Z\nHrA/OcvDVswu99iq+l56ZfLR6UHXHVcsNt3Gb0xyh5UT0rXW/nkItadDGtxl+Jm9/VuT3Kq19q2q\n+tf0A3JHDO1KepX527ZqG7fWnjdUSf9Fenh7p6w+PMb0sT0tyxWWG/WW9OFujk5yg1plLNthyJzp\nsDlnZfWDBYt4epK/TO8TJ2ec4VxmD449bxg+YKNem+Rz6QddLlNVN22tvW52geFMhPsMB1P/Ij1c\nvkR6yH7/VdY53VenJvm/wwRr65kd0uI1G34UAOxKhiM4dMyelnbr1trVF/h5xliNGU4POynLX5IB\ndspknZ+z06ta/iO9iupWSa64gQB2up79tNa+0Vr7hfSKm+emBzVnpAebZ6WHR69L8rtJrtBae+Wc\n9T8sPcD8ZPrM1v+T5F/TqzVXe4yLtne1ZffMXPed1tofpo/r+eIkn0n/Mv/19Kqd305ynXlDzQwV\nqtdKD5/flx4g/e/Q/pcluUlrbTphyTnDfc4LdE5OD0Q+m74NvpTkzemVSrOPa65haIGfSg9gXp7+\nJfys9BmpPz08xl8d3h/X+gK9kcq/yYrfG9ZaO6u1dtv0QPuUJB9Nf9zfS3JaehXnA5JcfpWZvldr\nz1aMBbrI43rWcP2+LA9PsOF2tdb+MT0Ue3H68/Ts9Mf/nvTnzk+21j48LP7n6dWK03374cw5zbe1\n9pH0Sso/SK86/Vp6//7U0N4TWmunrNPu6br+Nr2vPyM9fDsz/eDA+4f1r9mnWmtfbK1dO73q+QVJ\nPpH+ue776eH6W9IPhlyhtTY3IGutvTc9GPzL9ANO3xra8sn0itDrttaeMPx/ur0P5LvARl53Vrp3\n+uvBJMktquqXV1to2LaXTw/S356+PaZ9oKVXlN6stXbTYRLa1dbxgPSDV/+c/vr1v+n7+/XpE0rd\nqLU2fS25fZJXDtd/a7jP02bWtSXbuLX22GE9fzHcx5fTn9Nnpx+Ue3eSxyf5mdbavTc7Xmhr7Tvp\noXHSh8m52SqLnZzl/fj3BzB82LPTH8MkyS9U1WUXuM3Cfaiqjk+fgG82nN6w4WDIM2fu8+Q1lj01\n/UyJE5M8Mf1153PprxVnplcvvz79fa5aa7+ySAA7jP/8U0MbPn4AE88BsMssTSbbPi4/m1BV0+EC\nHt5aW1nRkaq6TXrF1STJDeacGrptqurE9A+xk/QPJR/fyfYAAIevqrpBegg3SR/n9UYHsK5nZ3ki\nsLu11jYz3i1si6q6SXoQO0nyT621E9e5CSOrqkdmed6M322tPX6HmwTANlEJe+iYHaz/WmstWFV7\nR25L0iunkuRLAlgAANh+rbXXp49bu5TkprXgxBGMY5i8bTpk2xn5wSEtADiEGRN2xhBO3jF99tSr\np4/39N300x//OcmTW2v/vXMtnK+19vlhHKv/k+S+VfWk1tqZK5erqjsleXxVvTLJXYcxj8Zw9eH3\nh0ZaPwAAsL4HpVfDLqUPb/MrO9ucw9qD0r9jTpI8orX27XWWB+AQIoQdDOMMvTbJCelvimekj7t2\nTPokHj+ZPivm77TWNjxr6AbbcsH02ZqnZsfIO7aqLrTiJqcPYervp8/YeZkk/1RVd2utfWJY51Hp\np849Ln1w/jNGDGCT5ZlL24j3AQAArKG19vqqenn6BGi3qarrtdb+ZafbdbipqkunT9yW9DGr/2YH\nmwPADjAcwbLnpAew30vym62141trV2mtXSF9NuJXp89g+jdVdcLIbXl/+sQl05/pDM1L6UdP/2fF\nz6WSpLX29iR3S5/I4bpJPlZVn6qqlj4RwVPTA9g3DOsZRVUdnT7bbdInNgAAAHbOPdMnAFtK8uyq\nOm6H23M4enaS49K/b96xtfa/O9weALaZEDbJMHvniemVm49vrT1r9vrW2pfSZ0s9M/2Dy71HbtJ0\nEq5FfvataOtz0yt3/yb9COuF0itjv5HkVUlu11q7WWvtuyO2/wIz7XOKDQBwsFhafxE49LTW/if9\n+8zZSS6X5Ok726LDS1U9OMkN078f3b+19sEdbhIAO8BwBN3sAPWrnprTWvv2UFF6tRXLb7nW2o8e\n4O0/leR3tqg5m7n/L2f/4RQAAHbaZMXvA13XmMM6wZZrrb2lqu6Z5FlJfrWqPtRa+7Odbtehrqpu\nlT4W7yTJY1trT93hJgGwQ5YmE58fq+qqSd6X/sZ4u9baS+Ys98H0sWFf2lr75W1sIgAAAACwSxmO\noPtQkk+ln6J2h9UWqKpLJbni8Ocrt6ldAAAAAMAupxJ2UFU/n+TlSY5J8pQkj0sPZo9On+TqMelj\nrb4hyc1bazYcAAAAALAuIeyMqvrpJA9Octv8YJXwp5I8LcljzGQJAAAAACzKcAT7OyG92nUpyTeT\nfDjJJ5PsS3KJ9IrYn9ix1gEAAAAAu44QdlBVT07y1CQXT/IrSX6otXbl1toVklwsyTOS3CLJO6rq\n+jvXUgAAAABgNzEcQc4dD/YNSSZJbtdae8mc5U5NclKSD7fWrjxCU+wMAAAAANi8pZ1uwGqO2OkG\nHCTuNPz+1rwAdjANYa9UVVdtrX1gqxsymUyyb58slvHs2bOUpaUlfY3R6WtsF32N7aKvsV30NbaL\nvsZ20dfYLtO+djASwnYXH35/cZ3lvrriNlsewu7bN8npp39nq1cL5zr++OOyd++Svsbo9DW2i77G\ndtHX2C76GttFX2O76Gtsl2lfOxgZE7b7+vD70ussN3v96SO1BQAAAAA4hAhhu7cOv4+uqlutsdwv\nDb+/neR94zYJAAAAADgUCGG75yT5fPrAvc+qqttU1bnbpqqOr6onJLlp+uRZj26tnb0zTQUAAAAA\ndhNjwiZprZ1ZVTdP8ookl03yj0m+VVWfT3JUksulB9aTJE9vrf3pTrUVAAAAANhdVMIOWmsfSnLl\nJPdL8qYkZya5fJKLJvlEklOSXL+1dq+daiMAAAAAsPuohJ3RWjsryd8OPwAAAAAAB0wlLAAAAADA\niISwAAAAAAAjEsICAAAAAIxICAsAAAAAMCIhLAAAAADAiISwAAAAAAAjEsICAAAAAIxICAsAAAAA\nMCIhLAAAAADAiISwAAAAAAAjEsICAAAAAIxICAsAAAAAMCIhLAAAAADAiISwAAAAAAAjEsICAAAA\nAIxICAsAAAAAMCIhLAAAAADAiISwAAAAAAAjEsICAAAAAIxICAsAAAAAMCIhLAAAAADAiISwAAAA\nAAAjEsICAAAAAIxICAsAAAAAMCIhLAAAAADAiISwAAAAAAAjEsICAAAAAIxICAsAAAAAMCIhLAAA\nAADAiISwAAAAAAAjEsICAAAAAIxICAsAAAAAMCIhLAAAAADAiISwAAAAAAAjEsICAAAAAIxICAsA\nAAAAMCIhLAAAAADAiISwAAAAAAAjEsICAAAAAIxICAsAAAAAMCIhLAAAAADAiISwAAAAAAAjEsIC\nAAAAAIxICAsAAAAAMCIhLAAAAADAiISwAAAAAAAjEsICAAAAAIxICAsAAAAAMCIhLAAAAADAiISw\nAAAAAAAjEsICAAAAAIxICAsAAAAAMCIhLAAAAADAiISwAAAAAAAjEsICAAAAAIxICAsAAAAAMCIh\nLAAAAADAiISwAAAAAAAjEsICAAAAAIxICAsAAAAAMCIhLAAAAADAiISwAAAAAAAjEsICAAAAAIxI\nCAsAAAAAMCIhLAAAAADAiISwAAAAAAAjEsICAAAAAIxICAsAAAAAMCIhLAAAAADAiISwAAAAAAAj\nEsICAAAAAIxICAsAAAAAMCIhLAAAAADAiISwAAAAAAAjEsICAAAAAIxICAsAAAAAMCIhLAAAAADA\niISwAAAAAAAjEsICAAAAAIxICAsAAAAAMKIjdroBcCg67bQv5Lvf/e5ON2NVp59+bPbu3ZNzztmX\nb3zjzA3d9uijj84lLnHJkVoGAAAAcGgSwsIITn3V63L+C11mp5uxqmOOPTJ7lpaybzLJWWeevaHb\nfvNrn8lv3fPuI7VsZ73kZa/I6Wd8a6ebseWOv8D5ctuTbrXTzQCADfPeDMBu5T2M1QhhYQRHHnlU\nLn35K+50M1Z13LHnyZ49S9m3b5LvnPm9Dd32I2ecNlKrdt7pZ3wrP3bVn9vpZmy5j33gLTvdBADY\nFO/NAOxW3sNYjTFhAQAAAABGJIQFAAAAABiREBYAAAAAYETGhE1SVackufNGbtNaE2ADAAAAAOsS\nwnafSfL+BZa7cJIfSfL9cZsDAAAAABwqhLBJWmsPTfLQtZapqqUk70hyySSP2o52AQAAAAC7n1Pq\nF3f/JNdI8okkf7bDbQEAAAAAdgkh7AKq6jJJHpFkkuTerbXv7XCTAAAAAIBdQgi7mKcmOTbJC1tr\nb9rpxgAAAAAAu4cQdh1VdaskN0lyVpIH7nBzAAAAAIBdRgi7hqrak+SR6cMQPLa19sUdbhIAAAAA\nsMsIYdd2hyRXTPKtJH+1w20BAAAAAHYhIezaHpReBfuU1to3d7oxAAAAAMDuI4Sdo6pumOTK6SHs\nk3a4OQAAAADALnXETjfgIHaP4fe/tNY+u113umfPUo4//rjtujtGcswxR+W4Y8+z081Y1dKe5d8b\nbeMxxxx1yPbPYw/ifXYgjt3BfbZnz9K5vw/VfsPBQV9ju+hr2+twfm/W19gu+hrb5XDra4fze9hO\nm/a1g5EQdhVVdUySk9KrYF+8nfe9tLSUvXsP3g7DYvbsWTqon/hJspSlcwPZRe3Zs5S9ew/NAvo9\nSwf/PtuMPUs7v8+8rrFd9DW2i762Pbw362tsH32N7XK49DXvYaxGCLu6GyU5Jj2Eff123vFkMsm+\nfZPtvEtGsG/fwbsfl/b0AHaSSSb7NnbbffsmOeecDd5ol9h3iD739k12bp/t2bOUpaUlr2uMTl9j\nu+hr2+twfm/W19gu+hrb5XDra4fze9hOm/a1g5EQdnU3GX5/qbX28e284337Jjn99O9s510ygrPO\n+n6+c+b3droZqzru2PNkaU8y2ZcNt/Gss75/yPbPMw/ifXYgztzBfXb88cdl794lr2uMTl9ju+hr\n2+twfm/W19gu+hrb5XDra4fze9hOm/a1g5Ea4tVdffj9oR1tBQAAAACw6wlhV3eV9KEI2k43BAAA\nAADY3YSwK1TV0UnOP/z5tZ1sCwAAAACw+wlhf9AF0qtgJ0m+vcNtAQAAAAB2ORNzrdBa+3KSvTvd\nDgAAAADg0KASFgAAAABgREJYAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABgREJYAAAAAIAR\nCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABgREJYAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABg\nREJYAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABgREJYAAAAAIARCWEBAAAAAEYkhAUAAAAA\nGJEQFgAAAABgREJYAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABgREJYAAAAAIARCWEBAAAA\nAEYkhAUAAAAAGJEQFgAAAABgREJYAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABgREJYAAAA\nAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABgREJYAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAA\nAABgREJYAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABgREJYAAAAAIARCWEBAAAAAEYkhAUA\nAAAAGJEQFgAAAABgREJYAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABgREJYAAAAAIARCWEB\nAAAAAEYkhAUAAAAAGJEQFgAAAABgREJYAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABgREJY\nAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABgREJYAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQ\nFgAAAABgREJYAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABgREJYAAAAAIARCWEBAAAAAEYk\nhAUAAAAAGJEQFgAAAABgREJYAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABgREJYAAAAAIAR\nCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABgREJYAAAAAIARCWEBAAAAAEZ0xE434GBTVXuT/GaSX09y\nxSTHJDktyduSPKW19u4dbB4AAAAAsMuohJ1RVT+U5B1JnpTk2km+k+QLSS6T5G5J3llV99m5FgIA\nAAAAu40Qdn8vS/KzSf4jyU+31i7TWvvxJJcarltK8tdVdaUdbCMAAAAAsIsIYQdVdcck10sfeuDG\nrbX/nF7XWvtykjskeX2SFya56I40EgAAAADYdYwJu+y3k0ySPKq19vWVV7bWvpvkZtveKgAAAABg\nV1MJm6SqLpHkhOHPF+9kWwAAAACAQ4tK2O5n08d7/XJr7YtV9SNJ7p7kWkmOT/LlJG9K8qzW2rd3\nrpkAAAAAwG4jhO1+cvj9+aq6VZLnJjnvimVumeT3q+rWrbV3b2vrAAAAAIBdy3AE3YWG3xdJ8vwk\nb0mfpOsCSS6c5DeSfDXJxZK8sqpMzAUAAAAALEQlbHe+4felkryqtXbrFdefUlX/leQd6UHtA5P8\n321sHwAAAACwSwlhu8nM5T9bbYHW2nuq6rVJbpHkthkphN2zZynHH3/cGKtmGx1zzFE57tjz7HQz\nVrW0Z/n3Rtt4zDFHHbL989iDeJ8diGN3cJ/t2bN07u9Dtd9wcNDX2C762vY6nN+b9TW2i77Gdjnc\n+trh/B6206Z97WAkhO2+NXP5A2ss9y/pIeylq+q8Y0zStbS0lL17D94Ow2L27Fk6qJ/4SbKUpXMD\n2UXt2bOUvXsPzVFM9iwd/PtsM/Ys7fw+87rGdtHX2C762vbw3qyvsX30NbbL4dLXvIexGiFs97kF\nlzt95vJ5k2x5CDuZTLJv32T9BTmo7dt38O7HpT09gJ1kksm+jd12375JzjlngzfaJfYdos+9fZOd\n22d79ixlaWnJ6xqj09fYLvra9jqc35v1NbaLvsZ2Odz62uH8HrbTpn3tYCSE7f5z5vLlknxkznIX\nnLn8jTEasm/fJKef/p0xVs02Ouus7+c7Z35vp5uxquOOPU+W9iSTfdlwG8866/uHbP888yDeZwfi\nzB3cZ8cff1z27l3yusbo9DW2i762vQ7n92Z9je2ir7FdDre+dji/h+20aV87GKkh7v4tyTeHy7db\nY7lrDr8/2lr77rhNAgAAAAAOBULYJK217yd5QZKlJPetqvOvXKaqLpfklumTeL1ke1sIAAAAAOxW\nQthlj0jy9SQXTvK6qrr89Iqq+ukkr05yniRfSfL4HWkhAAAAALDrGBN20Fr7UlWdmOQVSa6R5KNV\n9YkkRyW5dHoF7NeSnNRa+5+daykAAAAAsJuohJ3RWntnkp9I8pdJPpzkYkkulORDw/+u1Fr7951r\nIQAAAACw26iEXaG19tUkDxl+AAAAAAAOiEpYAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABg\nREJYAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABgREJYAAAAAIARCWEBAAAAAEYkhAUAAAAA\nGJEQFgAAAABgREJYAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABgREJYAAAAAIARCWEBAAAA\nAEYkhAUAAAAAGJEQFgAAAABgREJYAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABgREJYAAAA\nAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABgREJYAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAA\nAABgREJYAAAAAIARCWEBAAAAAEYkhAUAAAAAGJEQFgAAAABgREJYAAD+P3t3Hm3ZVdcJ/HvuqyFV\nxaAVhkgDBlywMYwidgtq44RGbOgmgiC0IKIiS0HaAVFGBRHBtp0QtBEIioKCkogyKDKIDTiCjJtB\nwCACIQUJSVVqePf2H+e+qpekUnkvdfc5b/h81rpr3+Gce3555/feWeubXfsAAAANCWEBAAAAABoS\nwgIAAAAANCSEBQAAAABoSAgLAAAAANCQEBYAAAAAoCEhLAAAAABAQ0JYAAAAAICGhLAAAAAAAA0J\nYQEAAAAAGhLCAgAAAAA0JIQFAAAAAGhICAsAAAAA0JAQFgAAAACgISEsAAAAAEBDQlgAAAAAgIaE\nsAAAAAAADQlhAQAAAAAaEsICAAAAADQkhAUAAAAAaEgICwAAAADQkBAWAAAAAKAhISwAAAAAQENC\nWAAAAACAhoSwAAAAAAANCWEBAAAAABoSwgIAAAAANCSEBQAAAABoSAgLAAAAANCQEBYAAAAAoCEh\nLAAAAABAQ0JYAAAAAICGhLAAAAAAAA0JYQEAAAAAGhLCAgAAAAA0JIQFAAAAAGhICAsAAAAA0JAQ\nFgAAAACgISEsAAAAAEBDQlgAAAAAgIaEsAAAAAAADQlhAQAAAAAaEsICAAAAADQkhAUAAAAAaEgI\nCwAAAADQkBAWAAAAAKAhISwAAAAAQENCWAAAAACAhoSwAAAAAAANCWEBAAAAABoSwgIAAAAANLRj\n7AI2ilLKdB2b363W+i/NigEAAAAAtgwh7FXNklyU5MB1bHNomHIAAAAAgM1OCHtNT621vnTsIgAA\nAACArcGasAAAAAAADQlhAQAAAAAaEsICAAAAADRkTdhr6kop35HkvCS3T7I7yaeTvDnJi2qtl41Y\nGwAAAACwyQhhr+mXk5yZZHa19++f5MmllAfXWt84fFkAAAAAwGZkOYJr2pPkaUnukOSMJLdI8qNJ\nDiTZn+TVpZRzxisPAAAAANhMzIQ94cnz8c9qre9Z9f5nkjy/lPL2JG9PsjfJs9PPjAUAAAAAOCUh\n7Fyt9VnX8fm7SinnJ/mhJOeWUm5ca7100XVMJl3279+36K9lYHv27Mq+vbvHLuOkusmJcb017tmz\na8v2594NfM5Ox94Rz9lk0h0ft2rfsDHoNYai14a1na/Neo2h6DWGst16bTtfw8a20msbkRB2ff4q\nfQi7lOROSf520Qfoui5LSxu3YVibyaTb0L/4SdKlOx7IrtVk0mVpaWuuYjLpNv45uz4m3fjnzN81\nhqLXGIpeG4Zrs15jOHqNoWyXXnMN42SEsOvz+VXPb9DiALPZLNPp1e8JxmYznW7c89hN+gB2lllm\n0/XtO53Osry8zp02iekW/d2bzsY7Z5NJl67r/F2jOb3GUPTasLbztVmvMRS9xlC2W69t52vY2FZ6\nbSMSwq7PzVY9v6TFAabTWQ4cuKLFVzOgQ4eO5IqDh8cu46T27d2dbpLMpll3jYcOHdmy/XlwA5+z\n03FwxHO2f/++LC11/q7RnF5jKHptWNv52qzXGIpeYyjbrde28zVsbCu9thEJYZOUUp6a5BuTXFxr\nfc23ZD4AACAASURBVPApNr33fDyc5D2n2A4AAAAAIEliIYfeGelD2O8qpdzzZBuUUs5O8rAksySv\nqrVuvf+lAQAAAAAsnBC296tJDqT/ebyylHK/1R+WUr4xyRuT7EtyaZKnDF0gAAAAALA5WY4gSa31\ns/Pg9U+TnJXkglLKJUk+nX4d2JumnwH7H0nOq7V+fKxaAQAAAIDNxUzYuVrr25N8ZZKnJXln+p/N\n7efj3yb5mSSl1vrO0YoEAAAAADYdM2FXqbV+Pskz5w8AAAAAgNNmJiwAAAAAQENCWAAAAACAhoSw\nAAAAAAANCWEBAAAAABoSwgIAAAAANCSEBQAAAABoSAgLAAAAANCQEBYAAAAAoCEhLAAAAABAQ0JY\nAAAAAICGhLAAAAAAAA0JYQEAAAAAGhLCAgAAAAA0JIQFAAAAAGhICAsAAAAA0JAQFgAAAACgISEs\nAAAAAEBDQlgAAAAAgIaEsAAAAAAADQlhAQAAAAAaEsICAAAAADQkhAUAAAAAaEgICwAAAADQkBAW\nAAAAAKAhISwAAAAAQENCWAAAAACAhoSwAAAAAAANCWEBAAAAABoSwgIAAAAANCSEBQAAAABoaMeQ\nByul/GuSlyV5Wa31g0MeGwAAAABgDEPPhD07yc8meV8p5R9KKT9WSrn5wDUAAAAAAAxm6BD24iTd\n/PFVSX4lyUWllNeVUh5WStk7cD0AAAAAAE0NuhxBkrOSfEOS85I8IMmt5jXcZ/44WEp5dfolC95Q\na50OXB8AAAAAwEINGsLWWmdJ3jp/PL6Uco/0YewDktwhyb4kD50/Li6lvDz9+rF/P2SdAAAAAACL\nMvRM2Kuotf5Dkn9I8qRSylfmxAzZuye5WZLHJnlsKeXDSX4/fSD7sbHqBQAAAABYr6HXhL1WtdYP\n1Fp/odZ6jyS3TfLEJP+Ufv3Y2yf5uSQfKaX8TSnlEaWU3SOWCwAAAACwJhsmhF2t1vrxWutzkjws\nyW8nWVkbtkvydUlelP6GXj8ljAUAAAAANrJRlyM4mVLKHZM8KP3SBHecv93Nx88k+dckX5vkJkme\nneQRpZTzaq0fGrpWAAAAAIDrsiFmwpZSbllKeUIp5d1J/iXJU5LcKX34eiTJHyf5b0luWWv9uvQ3\n8fr9+efnJHlLKeUWoxQPAAAAAHAKo82ELaV8afoZrw9N8vU5Mdt1ZXxHkvOTvKLW+oXV+9ZaP5zk\n4aWUC5K8PP1NvJ6R5FEDlA4AAAAAsGaDhrDz9Vvvn36t13OT7Jx/tBK8XpTk95KcPw9aT6nW+qpS\nyq8l+fEk9118xQAAAAAAp2fombCfTXKD+fOV4PWKJH+Sftbrm2qts3V+5x+nD2FvtpAKAQAAAAAW\naOgQ9obzcZbkzemD11fWWq84je9cWarg8tP4DgAAAACAJoYOYT+S5KVJfq/W+okFfee/JrlNkqML\n+j4AAAAAgIUZNISttd6+wXceTbKoQBcAAAAAYKGGngmbJCml3CjJTyS5V631Pteyzb2SvCzJHyZ5\nbq318wOWCAAAAACwEJOhD1hKuXOS9yd5cpJ7nmLTLsmXJ/npJO8rpdx1gPIAAAAAABZq0BB2PgP2\nNUm+LH3IerSUcm2zcQ8k+eR8u7OSXDjfHwAAAABg0xh6JuxjktwqyZXz5zettR472Ya11g/UWm+d\n5CFJDia5ZZIfGapQAAAAAIBFGDqEPS/JLMkza62/fW0B7Gq11j9K8qz0M2LPa1wfAAAAAMBCDR3C\n3mE+/sk693vVfLzdAmsBAAAAAGhu6BB2Zf3Xi9e53yXzcecCawEAAAAAaG7oEPbf5+NXrHO/u8zH\nzyywFgAAAACA5oYOYd+Sfm3XH1/rDqWUnUmemX4t2bc3qgsAAAAAoImhQ9gXz8fvLqW8spRy12vb\nsJSys5Ry3yTvSPK187d/t3WBAAAAAACLtOO6N1mcWuv/K6X8bpJHJXlAkgeUUg4k+UiSA0mOJdmf\n5MwkX57kjFW7/26t9a+HrBcAAAAA4HQNGsLOPTrJZUl+LP1M3DPTB69X183H5SS/muQJg1QHAAAA\nALBAg4ewtdZpkp8opbwwyQ8muU+S2yfZuWqzw0k+kOQNSV5ca61D1wkAAAAAsAhjzIRNktRaP5D5\nDbpKKZP0s2F3JzmY5NJ5WAsAAAAAsKmNFsKuNg9cPzd2HQAAAAAAizYZuwAAAAAAgK1slJmwpZT9\nSe6b5I5JvjTJrjXuOqu1PqpZYQAAAAAACzZ4CFtK+dEkz0m//uv1IYQFAAAAADaNQUPYUsp9k/x6\nklmS7np8xWyxFQEAAAAAtDX0TNjHzsdpkhckuTDJRUkODVwHAAAAAMAghg5h75F+NuvP1lqfO/Cx\nAQAAAAAGNxn4eDeej3808HEBAAAAAEYxdAh72Xz8wsDHBQAAAAAYxdAh7Nvn49kDHxcAAAAAYBRD\nh7C/kaRL8kMDHxcAAAAAYBSDhrC11jckeUaSx5RSnlhK6YY8PgAAAADA0HYMebBSyl2TXJBkZ5Jn\nJvmhUsoFST6c5PK1fEet9aXtKgQAAAAAWKxBQ9gk/5xktur1lyd53Dr2nyURwgIAAAAAm8bQIWzS\nrwkLAAAAALAtDB3Cfn+uOhMWAAAAAGBLGzSErbW+ZMjjAQAAAACMbTJ2AQAAAAAAW9kYa8JeQynl\njCQ3S3LDWuv7xq4HAAAAAGBRRgthSyn3SL9G7LclOTv9Dbtmq2sqpexM8rQkv1JrPTBCmQAAAAAA\np2Xw5QhKKUullN9J8o4kj05y23kd3fyx2r2T/GyS95RS7jRooQAAAAAACzDGmrDnJ3lUTgSv703y\nx9ey7dnpZ8d+WZJXz5ctAAAAAADYNAYNYUsp90ny0PnL1yS5fa31LumXJbiGWusLk9w3yZEkt0ny\nfQOUCQAAAACwMEPPhH3EfHxjrfX+tdaPXNcOtdbXJ/nN9LNmz2tZHAAAAADAog0dwt4z/fICz17n\nfn84H++y2HIAAAAAANoaOoS9+Xx89zr3+8R8/JIF1gIAAAAA0NyOgY83m483SHLJOvZbuSHX4cWW\nszallBsneX/6G4R9vNZ62zHqAAAAAAA2n6Fnwl40H+++zv3uPR8/ucBa1uM30gews+vaEAAAAABg\ntaFD2Denv8HWU0spe9ayQynlJkmekT4AfVO70q71+PdP8j+THEtfOwAAAADAmg0dwv5W+jD1Lkne\nWkq517VtWErZUUp5YJJ3Jjk7yTTJC4YoclUNX5rkt9PX/JIhjw0AAAAAbA2DhrC11vcm+YX0M0rv\nnuRvSimfyqoZrqWU15RS3pnkQJJXJLnN/KNnz/cf0vOS3CzJy5P87cDHBgAAAAC2gKFnwqbW+tQk\nP5sT/7z/rCRfnRPrrX5Hknukv3lXl+RokifXWp8yZJ2llPOSPCTJZ5I8dshjAwAAAABbx+AhbJLU\nWp+d5CuSPCfJP6VfaqBb9TiS5O/Tz5ottdZnDVlfKeXMJM9PHww/ptZ6YMjjAwAAAABbx46xDlxr\n/WSSJyZJKaVLcuMk+5JcnuSyWuvsFLu39vwkN03yslrrBSPWAQAAAABscqOFsKvNA9cvzB+jKqV8\nd5IHJvlULEMAAAAAAJymUZYj2KhKKTdNfzOuWZJH11pHD4UBAAAAgM1t0JmwpZSnnuZXTGqtT19E\nLdfi+UnOTHJ+rfXPGx7nWk0mXfbv3zfGoVmgPXt2Zd/e3WOXcVLd5MS43hr37Nm1Zftz7wY+Z6dj\n74jnbDLpjo9btW/YGPQaQ9Frw9rO12a9xlD0GkPZbr22na9hY1vptY1o6OUInp5+lunpfsfClVK+\nJ8l5ST6Z5HEtjrEWXddlaWnjNgxrM5l0G/oXP0m6dMcD2bWaTLosLW3NCfSTbuOfs+tj0o1/zvxd\nYyh6jaHotWG4Nus1hqPXGMp26TXXME5mjDVh19uFs1X7NLlZVynlZkl+PclykkfWWr94ks0G+e2Z\nzWaZTse8JxmLMJ1u3PPYTfoAdpZZZtP17TudzrK8vM6dNonpFv3dm87GO2eTSZeu6/xdozm9xlD0\n2rC287VZrzEUvcZQtluvbedr2NhWem0jGjqE/aY1bneDJCXJ/0hyryS/mOTXkhxtVNe56ZchSJK/\nLKWcatuzSykrHfeSWuv3L7KQ6XSWAweuWORXMoJDh47kioOHxy7jpPbt3Z1uksymWXeNhw4d2bL9\neXADn7PTcXDEc7Z//74sLXX+rtGcXmMoem1Y2/narNcYil5jKNut17bzNWxsK722EQ0awtZa37KO\nzf88ya+UUh6S5Pwkt6y1fl+Twvpw97puwrUryd4k0ySXzd/b2J0HAAAAAIxujOUI1qXW+vJSyjlJ\nnlRKeW2t9RUNjvGHSf7wVNuUUh6R5MVJ/q3WettF1wAAAAAAbE2bZTXd30+/Juujxy4EAAAAAGA9\nNksI+5n5ePdRqwAAAAAAWKfNEsLeYj7uGbWKZDZ/AAAAAACsyYZfE3buJ+fjxWMVUGs9P/0NwgAA\nAAAA1mzQELaU8vB1bH5Gklsn+c4kd0k/A/WtLeoCAAAAAGhl6JmwL8n1++f8XZJDSX5xodUAAAAA\nADQ2xpqw3fV4vDfJubXW94xQLwAAAADA9Tb0TNhHrmPbo0kuTfLBWutHG9UDAAAAANDUoCHs/OZW\nAAAAAADbxhjLEQAAAAAAbBtCWAAAAACAhgZdjqCU8rIkhxt9/azW+qhG3w0AAAAAcL0MfWOu70ky\na/j9QlgAAAAAYEMZOoT9bJK96ZdB2JOkW8M+szVuBwAAAACw4QwawtZazyql3CLJC5N8a5I/SHJh\nkvckuSTJlUn2Jbl5krsmeVCS+yV5TZJH11r/Y8h6AQAAAABO19Brwt44yV8luUmS/1xrfddJNjuY\n5OIk703yslLKvZNckOSNpZR71Vq/MFjBAAAAAACnaTLw8R6X5A5JnnYtAew11FrfkuTp8/0e3640\nAAAAAIDFGzqEfWD6NV5fu879LpyP5y22HAAAAACAtoYOYW89Hy9d536fv9r+AAAAAACbwtAh7Moa\ntGWd+93uavsDAAAAAGwKQ4ewH5uPTymlrOfYT5qPn1hwPQAAAAAATQ09s/RPktwpyblJ3lFKeU6S\nv6q1fuHqG5ZS9iX5piRPSPL16deSvfDq2wEAAAAAbGRDh7DPSfI96ZcX+Ookr0iSUsolST6X5EiS\nXUm+NMlNk3Sr9v1Ukl8aslgAAAAAgNM16HIEtdaD6We1vjZ9wLryuEn6dWLvPB9vPq9t5fN3JvmG\nk82YBQAAAADYyAa/0VWt9eIk31lKuXuS70ryX5LcPv3s1z1JDie5NMlHk/xjkgtqrW8auk4AAAAA\ngEUYPIRdUWv9pyT/NNbxAQAAAACGMOhyBAAAAAAA281oM2FXlFJumeScJGcluWGt9XkjlwQAAAAA\nsDCjhLCllJ1JHpfkUelvxLXa81Zt9yVJXpPkibXWtw1XIQAAAADAYgy+HEEp5Rbpb7j1nPQBbLfq\ncXXfnuReSd5USnnwYEUCAAAAACzIoCFsKWUp/czWO6UPXS9K8pvpZ8WezBeS/EeSpST/t5TyZUPU\nCQAAAACwKEPPhH14krslmSV5QpLb1lofl+TFJ9u41vr69DNhP51kX5IfHqhOAAAAAICFGDqEfXD6\nAPa3aq2/XGudXtcOtdZPpF+6oEu/PAEAAAAAwKYxdAh7x/n4onXu9/r5eLsF1gIAAAAA0NzQIexN\n5+PH17nfp+fjDRdXCgAAAABAe0OHsAfn41nr3G8lvL10gbUAAAAAADQ3dAj7ofl473Xu94D5+NEF\n1gIAAAAA0NzQIezr099g6+dLKWta37WU8rVJnpz+hl6va1gbAAAAAMDC7Rj4eL+Z5PFJzkzyd6WU\n5yZ5VZLPrd6olLInyd2SPCzJDyTZleSL8/0BAAAAADaNQWfC1lovTvLwJMtJbpzkGUnen+Sz6We6\nppRyeZLLk7wtyWPSB7DLSb631vq5k3wtAAAAAMCGNfRyBKm1XpDkm5N8MP3SBKsfSbL3au99KMk3\n11ovHLpWAAAAAIDTNfRyBEmSWuvbkpxTSjk3ybcm+ar0SxTsSz8L9rNJ3pXkDbXWN45RIwAAAADA\nIowSwq6otb4ubrYFAAAAAGxhg4awpZT7z5/+da318iGPDQAAAAAwhqFnwr46yTTJWemXHQAAAAAA\n2NKGvjHXpelvtrU08HEBAAAAAEYxdAj7J/PxewY+LgAAAADAKIZejuAnk9w6yS+VUpaSPK/WeuXA\nNQAAAAAADGboEPabkjw/yQeTPC3JU0spb0/yySSfS3Lwur6g1vrzTSsEAAAAAFigoUPYVyaZrXrd\nJbnPOr9DCAsAAAAAbBpDh7BJH7ye6vWpzK57EwAAAACAjWOM5QgAAAAAALaNQUPYWutbhjweAAAA\nAMDYmoWwpZTHzZ8+r9a6fB3b3ihJaq2XtaoHAAAAAGAMLWfC/mqSaZIXJjl4bRuVUvYl+cJ82zHW\nqAUAAAAAaGbS+PvXc9Ot9WwLAAAAALAptA5hAQAAAAC2NSEsAAAAAEBDQlgAAAAAgIaEsAAAAAAA\nDQlhAQAAAAAaEsICAAAAADQkhAUAAAAAaEgICwAAAADQkBAWAAAAAKChHQMc4ztLKYdP8fkZK09K\nKfdL0p3qy2qtFy6qMAAAAACA1oYIYV++hm1m8/HVa9huiJoBAAAAABaidaB5ylmtAAAAAABbXcsQ\n9ucafjcAAAAAwKbQLISttQphAQAAAIBtbzJ2AQAAAAAAW5kQFgAAAACgISEsAAAAAEBDQlgAAAAA\ngIaEsAAAAAAADQlhAQAAAAAaEsICAAAAADQkhAUAAAAAaEgICwAAAADQkBAWAAAAAKAhISwAAAAA\nQENCWAAAAACAhoSwAAAAAAANCWEBAAAAABoSwgIAAAAANCSEBQAAAABoSAgLAAAAANCQEBYAAAAA\noCEhLAAAAABAQ0JYAAAAAICGhLAAAAAAAA3tGLuAjaSUsiPJw5M8KMndkpyZ5MokFyV5a5IX1Frf\nPV6FAAAAAMBmYybsXCnlPyX5xyQvTPJtSW6Y5KNJDie5Q5JHJ/nHUspPjVYkAAAAALDpCGFP+NMk\nd05yWZLvTXLDWutX1lpvmn5W7D+k/3k9u5TyX8crEwAAAADYTISwSUop5ya5R5JZkkfVWv+g1jpb\n+bzW+p4kD0xydP7WDwxfJQAAAACwGQlhezuTvDLJ65K8+mQb1Fr/LUmdv/yKgeoCAAAAADY5N+ZK\nUmv9syR/toZNu/n4qYblAAAAAABbiJmwazRfsuCO85d/MWYtAAAAAMDmYSbsKZRS9iQ5O8mDk/x0\n+jVjX1prffGYdQEAAAAAm4cQ9iRKKf89yZ+ueutI+rViX1xrff04VQEAAAAAm5HlCE7u0iTvSvKh\nJJenv3HX/ZL8QCnlnDELAwAAAAA2FzNhT6LW+uYkd195XUr5miTPSvJdSb6jlHJurfVtLY49mXTZ\nv39fi69mQHv27Mq+vbvHLuOkusmJcb017tmza8v2594NfM5Ox94Rz9lk0h0ft2rfsDHoNYai14a1\nna/Neo2h6DWGst16bTtfw8a20msbkRB2DWqtfz+/Mddbk9wzyUtKKbertc4Wfayu67K0tHEbhrWZ\nTLoN/YufJF2644HsWk0mXZaWtuYE+km38c/Z9THpxj9n/q4xFL3GUPTaMFyb9RrD0WsMZbv0mmsY\nJyOEXaNa63Ip5fnpQ9jbJLlbkn9e9HFms1mm04VnuwxsOt2457Gb9AHsLLPMpuvbdzqdZXl5nTtt\nEtMt+rs3nY13ziaTLl3X+btGc3qNoei1YW3na7NeYyh6jaFst17bztewsa302kYkhE1SSrl7klsn\n+WKt9Y2n2PQzq57fvEUt0+ksBw5c0eKrGdChQ0dyxcHDY5dxUvv27k43SWbTrLvGQ4eObNn+PLiB\nz9npODjiOdu/f1+Wljp/12hOrzEUvTas7Xxt1msMRa8xlO3Wa9v5Gja2lV7biISwvf+d5N5JPpbk\nK06x3a1WPf9s04oAAAAAgC3BQg69C+fj2aWUB51sg1JKl+SR85efT/KuIQoDAAAAADY3IWzvd5Jc\nlKRL8sJSyg+XUpZWPiyl3CbJnyb5uiSzJL9Ua93Yi2AAAAAAABuCEDZJrfWKJOcm+UiSGyT5rSSf\nL6W8r5TymSQfTXL/9AHs79RanztasQAAAADApiKEnau1fiDJXZL8SJK/THJF+vVh96UPZ38/yTfX\nWh8zWpEAAAAAwKbjxlyr1FqvTPKC+QMAAAAA4LSZCQsAAAAA0JAQFgAAAACgISEsAAAAAEBDQlgA\nAAAAgIaEsAAAAAAADQlhAQAAAAAaEsICAAAAADQkhAUAAAAAaEgICwAAAADQkBAWAAAAAKAhISwA\nAAAAQENCWAAAAACAhoSwAAAAAAANCWEBAAAAABoSwgIAAAAANCSEBQAAAABoSAgLAAAAANCQEBYA\nAAAAoCEhLAAAAABAQ0JYAAAAAICGhLAAAAAAAA0JYQEAAAAAGhLCAgAAAAA0JIQFAAAAAGhICAsA\nAAAA0JAQFgAAAACgISEsAAAAAEBDQlgAAAAAgIaEsAAAAAAADQlhAQAAAAAaEsICAAAAADQkhAUA\nAAAAaEgICwAAAADQkBAWAAAAAKAhISwAAAAAQENCWAAAAACAhoSwAAAAAAANCWEBAAAAABoSwgIA\nAAAANCSEBQAAAABoSAgLAAAAANCQEBYAAAAAoCEhLAAAAABAQ0JYAAAAAICGhLAAAAAAAA0JYQEA\nAAAAGhLCAgAAAAA0JIQFAAAAAGhICAsAAAAA0JAQFgAAAACgISEsAAAAAEBDQlgAAAAAgIaEsAAA\nAAAADQlhAQAAAAAaEsICAAAAADQkhAUAAAAAaEgICwAAAADQkBAWAAAAAKAhISwAAAAAQENCWAAA\nAACAhoSwAAAAAAANCWEBAAAAABoSwgIAAAAANCSEBQAAAABoSAgLAAAAANCQEBYAAAAAoCEhLAAA\nAABAQ0JYAAAAAICGhLAAAAAAAA0JYQEAAAAAGhLCAgAAAAA0JIQFAAAAAGhICAsAAAAA0JAQFgAA\nAACgISEsAAAAAEBDQlgAAAAAgIaEsAAAAAAADQlhAQAAAAAaEsICAAAAADQkhAUAAAAAaEgICwAA\nAADQkBAWAAAAAKAhISwAAAAAQENCWAAAAACAhoSwAAAAAAANCWEBAAAAABraMXYBG00p5bwkD0/y\nNUlukuRwko8leWOS59VaPzpieQAAAADAJmMm7FwpZU8p5S+SvDLJ/ZKcmeTjSY4muVOSxyd5Tynl\ngaMVCQAAAABsOkLYE16U5Nwk0yRPSnLjWmuptZ6Z5L8m+WiSM5KcX0q51XhlAgAAAACbiRA2SSnl\nnCQPTjJL8qxa67NrrYdXPq+1/m2Sh81fnpHkkcNXCQAAAABsRkLY3lcnuTjJcpIXnmyDWuvfJfnE\n/OXdBqoLAAAAANjkhLBJaq2/V2u9eZLdtdZ/O8WmR+fj7gHKAgAAAAC2ACHsKrXW2bV9Vko5M8nZ\n85fvH6QgAAAAAGDTE8Ku3ROT7EhyLMnvjFwLAAAAALBJCGHXoJTygCT/K/2Nu36t1vrhkUsCAAAA\nADYJIex1KKV8b5KXJ+mSvDb9jFgAAAAAgDURwp5CKeUpSc5PvwzB65M8qNY6HbcqAAAAAGAz2TF2\nARtRKWVXkhcleWj6JQhenOTRtdbl1seeTLrs37+v9WFobM+eXdm3d/fYZZxUNzkxrrfGPXt2bdn+\n3LuBz9np2DviOZtMuuPjVu0bNga9xlD02rC287VZrzEUvcZQtluvbedr2NhWem0jEsJeTSnlRumX\nHbhnkuUkT6i1/p+hjt91XZaWNm7DsDaTSbehf/GTpEt3PJBdq8mky9LS1pxAP+k2/jm7Pibd+OfM\n3zWGotcYil4bhmuzXmM4eo2hbJdecw3jZISwq5RS9iT5i/QB7BeTPKTW+toha5jNZplOZ0Mekgam\n0417HrtJH8DOMstsnYtrTKezLC9vzRU5plv0d286G++cTSZduq7zd43m9BpD0WvD2s7XZr3GUPQa\nQ9luvbadr2FjW+m1jUgIO1dKWUpyYZJ7JTmQ5Ftqre8euo7pdJYDB64Y+rAs2KFDR3LFwcNjl3FS\n+/buTjdJZtOsu8ZDh45s2f48uIHP2ek4OOI5279/X5aWOn/XaE6vMRS9NqztfG3WawxFrzGU7dZr\n2/kaNraVXtuIzCE+4elJviXJFUm+fYwAFgAAAADYeoSwSUopt0ny0+lvwvXTtdZ/HLkkAAAAAGCL\nsBxB73E58bN4TCnlB69rh1rrV7UtCQAAAADYCoSwvS9JPws2Sc5Zw/Zbb3VlAAAAAKAJIWySWusj\nkzxy7DoAAAAAgK3HmrAAAAAAAA0JYQEAAAAAGhLCAgAAAAA0JIQFAAAAAGhICAsAAAAA0JAQFgAA\nAACgISEsAAAAAEBDQlgAAAAAgIaEsAAAAAAADQlhAQAAAAAaEsICAAAAADQkhAUAAAAAaEgICwAA\nAADQkBAWAAAAAKAhISwAAAAAQENCWAAAAACAhoSwAAAAAAANCWEBAAAAABoSwgIAAAAANCSEBQAA\nAABoSAgLAAAAANCQEBYAAAAAoCEhLAAAAABAQ0JYAAAAAICGhLAAAAAAAA0JYQEAAAAAGhLCAgAA\nAAA0JIQFAAAAAGhICAsAAAAA0JAQFgAAAACgISEsAAAAAEBDQlgAAAAAgIaEsAAAAAAADQlhAQAA\nAAAaEsICAAAAADQkhAUAAAAAaEgICwAAAADQkBAWAAAAAKAhISwAAAAAQENCWAAAAACAhoSwAAAA\nAAANCWEBAAAAABoSwgIAAAAANCSEBQAAAABoSAgLAAAAANCQEBYAAAAAoCEhLAAAAABAQ0JYAAAA\nAICGhLAAAAAAAA0JYQEAAAAAGhLCAgAAAAA0JIQFAAAAAGhICAsAAAAA0JAQFgAAAACgISEsfcrf\nwQAAIABJREFUAAAAAEBDQlgAAAAAgIaEsAAAAAAADQlhAQAAAAAaEsICAAAAADQkhAUAAAAAaEgI\nCwAAAADQkBAWAAAAAKAhISwAAAAAQENCWAAAAACAhoSwAAAAAAANCWEBAAAAABoSwgIAAAAANCSE\nBQAAAABoSAgLAAAAANCQEBYAAAAAoCEhLAAAAABAQ0JYAAAAAICGhLAAAAAAAA0JYQEAAAAAGhLC\nAgAAAAA0JIQFAAAAAGhICAsAAAAA0JAQFgAAAACgISEsAAAAAEBDQlgAAAAAgIaEsAAAAAAADQlh\nAQAAAAAaEsICAAAAADQkhAUAAAAAaEgICwAAAADQkBAWAAAAAKAhISwAAAAAQEM7xi5gIyql7Ezy\n9CRPSLKU5Om11p8ftSgAAAAAYFMSwl5NKeWuSV6a5M5JZvMHAAAAAMD1YjmCuVLKpJTyM0nemeRO\nSf4oSTduVQAAAADAZieEPeE7k/xCksNJHllrfcjI9QAAAAAAW4DlCE6YJHlHkofWWj8+ci0AAAAA\nwBYhhD3hbUn+rNY6HbsQAAAAAGDrEMLO1VovGbsGgI1kNptleTrL0WPTE+PyNEeXp1lenvXjdJbl\n5WmOTWdZXp5ledp/dmw6zZ49uzKdJUeOLueyL16Z6XSW6bT/zpVH/3q66nn/mE1nmc6PP53laq/7\nz5ens8xmfZ3Tq42zWfrtrvJ8/lmSzJJZ+vdW/ltnyVVeH/85XOMHcy0/sO7E0HUnlhTvutUfd+m6\nzB/d8W2Pv+5yjfcmXTLpuuPPVz6bTLoTz7suk67L0qRLN5nvM+nfu+rYv7806U6M3erXk1XvJ0tL\nk/7zpf69pcmkH4+/nr+38nppkh3z5zuWJtcYJxNLrQMAANuTEBZgA1sJQo+tDjiXp1d53X++Eoj2\n7x+bv3/JpTfKS177wRw9Ns2x5elJx6NXf38lWF2+trQRrp8uOR7U7ljqw9sdk6u9Xppkx/FA98Rn\nO5Ym2bmjf3/n1T670Q3PyO5dS1madDly+Gi/7dIkO3acbOyyc8dSdix12bljkp07JlmaWCIfAABo\nSwgLcJpmsz4kvUq4uTzNsWOz+TidB6fT49utft7vcyI8PbY8vUqYenr25dPv/tRC/jvhdM2See8n\nyfLI1Zww6U4Esjt3TOYB7VJ2Lk2u8v7OHZPsWnm+tJSdO/twd9d87LdZ6l/vmGTXjqV+3Dl///j+\n/TZL89nMAADA1ieEBbal6WyWKw8v59DhYzl0+FguO7Iz/37x5VeZGXqVxzxMPenMUTNGYVObzmY5\nfHQ5h48OGwx3XbJr54lwdtfOpauEt7t3nghxd82D22uMK/uvHncuZffx5/22loIAAIBxCWE3mMmk\ny/79+8Yug9O0Z8+u7Nu7e+wyTqqbnBjXW+OePbs2RH/OZrNceWQ5Vxw6moNXHssV/7+9+46S5Crv\n///u7sm7qxUSAiEESIC5ZBFtkUTGwoCIIoMJBiOCDfwwYAwmHEwyyf6SwYCIJkeDSTIGTDJBJOGH\nKEAIlMPuTp7u3x+3erqmtyftdE18v87pU91dVberZ+7pqf7MredOznBgorhNznBgcnb+8fjkLOOT\nM4xPFcvJWcYnc/C60BX5yUW/35D3o421sD4rzNdtXbBR1z5dT7TahWJbnZKxrXbx2eJ+q7hTrj2r\nna3VgqnpOaamqw9/BwfqjAwVAe1gg+Gh0nKowfDgACMHPdeY32dkaGD++ZHux0MNBhr1HTGqtx1m\ne762PsY28fnUWoyt4HzKvqb1Yl/TetlpfW0n/w3baJt58IEh7CZTKyZA0dZWLya52cxq1OYD2ZWq\nF3Ua+6EdpO4bn2bfgWn2T8zk2/gMByam5+/nZdf6yRmaTVOsflt4Ofbyy3adzUapZud8jc+BhXU9\n8/P5cXtip3q9M4lTeXKn+QmiajXqdeYni1o4sVS9mJiK+Uu6610TVs1PaFXsB8WyFLhuZGjUnkiM\n9pKFk5ABeTKz9qRj7UnJWszfbxbPt59rT3DWfn7+cbPFXKt0v1muIdyi2X5cem5urvxcs1N3uKhB\n3Gx2Sme0180WI8Y7E7rlshyzzaIsR7NVrC9Gls912lK12qP6YaaS9uv12nyI2w5oR4by/ZHh8nNF\n2Fu6PzI80HPb0eL5zRjwer62PtqTCm43ecLDlZ1P2de0XuxrWi87pa/5N0y9GMJuMu0v2tra2kHH\nZlSr5wC2RYtWc3X7Noswptv0TBGmjs9w+YHp+WC1/Vw7SJ1/PD7D/olpL+NfRA1yiNk9MdHAwRMV\nDTSKoLNrIqNGEXCe95sfc79T7jZ/ifNAuablYGO+nmUVIUc7DN10n2sLRqNujuNaEBJvw5O15ZQD\n3HLN5JnZuWJZmkhudm6+DMjM7Fwn3J1rMj3TZHpmrrNde5+ZcpmROaZnmsX2c/Ph5HTRbnudVqfZ\nbM1faQBTfW27HfDOh7bzwe0iIe5g7/Wjw50geHT40MLdTfu5tk01t+nPudnqfT5VZl/TerGvab3s\ntL62k/+GbbR2X9uMDGE3mWazxcUXH9jow9AaTUxMc2C8v19C+2XX2DC1OrSa9DzGZjPXRpwsLpGd\nnJljcnqWqek5/nD+KC95x7c4MDHDvuJy//0Ts+teR3GzaNRrpZGhtUVGjJYD067HA7X5WdzLo0nr\nfRyhue93B7jakWMHr2i1aE7PMgVMTfTlpQ5yxBG7aDRqfq5p1WrAIHl0NgPAcGPJ7XNfqzM31+xL\nX2u2WsVI3R71oUuBbblm9MJAd2GoOzM7x3TX/uXnp2fz/pap6G1hwNs/9VptfsRuuwzDyGDj4OdK\nI3yPuMIYu0YGGRqsMz050xnVW2w7NNigvklP+rei8U18PrUW4xPTy35W+TdU68W+pvWy0/raTv4b\nttHafW0zMoSVtrlOqDrLxNQc1MaZnJ5jfHKWfQemmJzO69qh6/TsUv/V2s35Pz1/3Y69CvVajdHh\nPCJq/jbU4Pe/P4cjjrpK55L8HrOil58baNS35eUlkvLnRHuCq/XSauVSD9MzRUg722RmZm4+oJ0u\nAt0c4M6V7peWM3NMtbcv7zvTZKq0zdw2HJVxKJqt1vzkjP1UDm9HSjV2R4YGFoS8C4PegVLYW6wr\nth8a3HzlGCRJkrR6hrDSFjQ712Ryao6JIjydmFq4nGw/np5lemZzXyqwWsODDcZGBvKtCFHb9/Ny\nkLGR4vn5oLUTug4N9P4y+9bTf8yf3OAmG/COJCmPfm+PlB+r+PRsvnRDEehOtQPcdmA7M9d5PNvM\nk4eV7rfXlbcv77PTS820fx6X92mQSA0OCnI7dXXbYW6pVEN5srV2eYauOrwD1nKTJElad4awhZTS\n4UB5yEs5pRlLKR3ZtcvFEbGzv2Wor5rN1vxo1YmpWSbK96fy/bx+dst/wR1o1Nk9OsCu0UF2jQyy\na6R9f4Cx4nEOVdvPDbBrJIerfnGUpLVplz6pKuydazaZml4Y5s7fpps9nis/bi54PF2Ux9nJAW8L\niqtW5riM6b60OdCo5QnQBhtdQe3AgiC3HPqODA8sGKlbrtM7uMg/OCVJktRhCNtxJnD1Hs/XgGcW\nt7LjgN9WfEzaBmZmm/NB6ngpUO2Eq/nxVqyrWoMcno4Osnt0gD2jQ+waHWD36CC728+PdO63w9bh\ndbzEV5K0vhr1OmMjdcZG+n+a2Z5QbXL64BC3XMt8wTbTPbYr9p0s/v7utBINs3OtPGnmxExf2ivX\n1+0OaMsjcMuB7sLyCwML9rMEgyRJ2o4MYTuarHya7J11pq4Vm5lt8sXv/o6fXnwkP/3Kr5iYyrNz\nbxWDjXpnEpLuCUqGGlz4u7N4wL3uyu7RQfaMDTE2PGBdVEnSupkfxTsy2Nd2Z+eaC0Lcqek5pqZn\n5++3l/VGg6mZOcanZrjs8qki3C1tVwp6Z5assb699Lu+bq0GI0MNZmeuxE+++uv5muwDA3UGuybC\nXHB/wXaddQONzTtLsiRJ2jkMYQsRcc2NPgZtfe//4s/48pnnAsMw05/RJWvRqNcYHV54aeFhu4YZ\nGxlkZKgBrVa+DLEIXBvLXOo/d/4kf3Ls4et09JIkrY9cJqfO7tGlw908226dubnmsjMDl8sylCfA\n7AS2s/PBbTnEnZyenR+5O7kg2N365YhWqtUiTyZKg5kDay/BUIODgtnuULcd4g6Ulz0m5RwcqNOo\nG+pKkqTVM4SV+ug7cUHlrzE4UGe0uJSvezky1GB0qJiIY2iAwYGDQ9VdY8PU6zWazRYHxqcqP15J\nknaihWUZhvvS5uxcc76MwuTMwiC3HexOdgW4C8Le6YX7TU3P7YjLu1rkq5VmZpvQh1OfWo0FgW33\n6Nvu0bkLA90aB2YGuODSifnyC73O1yRJ0vZjCCv10VGHjx5SfbXyiNXR4QFGh9vL4lYKW5cbrSpJ\nkrandjmGXX0qx9BstZiZac6Ht5NTnZG4k4sGubML1s2P9J3Kj5ut7R/rtlowPdtk+pBLThzFD9/0\njflHjXqtUxu3uEJpz65hxkYGGBkaoNZqHVRrt7ztyFBxrlhc9dSoe64oSdJmZAgr9dEj/zzxuo/+\niIsunwTyKInR4QajIzlMHSsHq0XQOjY84KzCkiRp3bUn1BoearC3D+21Wi1m55pMtEPaqe5AtxPg\nTpUeTx0U7ub745MztNj+50dzzRYHJmc5MNmfmrpDA/WDJzwbHugd5HY/N1xcVeUkaZIk9Z0hrNRH\n1zh6D6847Va8/u3vJ51wOy8vkyRJO0atVmNwoMHgQIPDxtbe3ltPfy/XvNFJ86UEZudapft5OTPX\nZHa2c3/Buq7tZmab7ICBuvOjdC8fX/v8BO1J0noGtgtG5Haemw9xewS/A17RJUnawQxhpT6r1WoM\n1psGsJIkSWvUqNdpDNUZGVp7W61Wi2az1SOsLT23SJjbM+ydbW77mrrtSdLyRGlrNzg/SrdURmGx\nUboLRuUW25cCX68kkyRtNYawkiRJkra9Wq1Go1Gj0ehfqDvXbPUMbMuBbjnYnZltcunFF3DlKx99\nUPmFnTBRWvtnsK8Po3Q7tXQXjsgdXVBeoVMvt3sS2/KI3SEDXUnSOjCElSRJkqRVqtVqDDRqq77E\n/uc/+BmPe+hfHPR8s9ViuqihOzw6xPRMk/0T01xw4YGeNXMXPDd18PpDnzhsa1hYS3dqTW3Va7WD\na+IOlwLd4YUjcUe7R+YW+40O55ILBrqSpF4MYSVJkiRpg+UgcICRoQGOOGIXjUadubkmF19h9JDa\nm2s2i0nP5orJ0oqgdmq5ILdX4DvHXHP7jtNttlqMT80yPrX2QLc9Qnd0uBzmtgPbdni7sBTDaLHN\nfAhcbNuoW95MkrYTQ1hJkiRJ2mYa9TpjI3XGRgbX3Far1WJ2rlmEuYsEtqXRuBPzge/CQLcdBk/P\nbN9RugtH6K7N0GC9CGVzaDvaK7QtBbqj5e1K950QTZI2B0NYSZIkSdKiarUagwMNBgcaHDa29vaa\nzdaCEHdiqVG6U6X107NMdG8ztX1r6U7PNJmemeayA9NramdwoN4JaotRuTnQHeAKe0fYNTrI6NAA\nrbm5g7cZHmCsCH8NcyVpbQxhJUmSJEnrpl6vMTYywNjI2r+OtlotpmbmOqNwpxYGtAuC3qneI3Xb\n6yamZ2ltw0S3PSHa5WucEG1ooF4KZnNphbHh8ojcPBp3bGThqNzOrcHwYMOauZJ2LENYSZIkSdKW\nVCvV0l2rVqvF9GyzZzjbDnQnyiNyp7oetwPfYv/tZnq2yfTs2kbm1mu1BSNxx9oBbhHcjnWFtuXH\n7fsjQwa5krYmQ1hJkiRJ0o5Xq9UYHsyjNfeusa1mq9WZGG1qdsHo24mphaNxJ+afXxjiThTh7uzc\n9qmh22ytvWZurcb8qNvRYlTu2MjgfLg7PxK3CG7b4e1YKegdHKgb5Epad4awkiRJkiT1UR7xmUO/\nK+wZXlNbM7PNzijcIqw9KLjtGp070b3dNpoQrdWC8alZxqcOPchtFCUx5oPaJe6PDQ8Wy846R+NK\nOhSGsJIkSZIkbVKDA3UGB4bYs8ZJ0eaazRzYTnaC2oGhASan5zgwMc2FF4/Ph7nzIW4p1G0HvDOz\nWz/MnWu22Dc+w75DrJNbq9EzpB0dKYe3XetLz1kbV9qZDGElSZIkSdrmGvU6u0bq7BoZnH/uiCN2\n0WjUmZtrcvHFB1bUzvzI3KmuwLYIcMfbgW0xWrVdfqEc7E5OzbGV50BrtSiVVZhc9f6Nem2+RMKu\n+XC2E9buGhnshLjtx6UQt1Gv9/9NSaqcIawkSZIkSVqRzsjcoUNuo9lqzU92Nl4KZ8e7gt35dZML\nH48XpRe2apA712yxf2KG/ROHNhJ3ZKiRw9uRQXYVJRLaQe3886Pl5zrrDHCljWMIK0mSJEmS1k29\nVpsf5XnEIbbRnvxs/KCAthTqltaN93huq9bJnSwmfbvo8qlV79sd4O4qAtvy47GRAXaNltaPDDAy\nPEDdEgrSmhjCSpIkSZKkLaU8+dmhmp1rLgxwJ7uC2677nedm5kftbjWHGuDWaswHsjmgXRjiLlyW\nt3P0rdRmCCtJkiRJknacgUadPWOHXlqh2WwxMd0Jb8cnZw4KbPP9mc5zpeenZrZOiNtq0SmhcMnE\nqvYdHR7gsF1D7BnLAe3wQH0+rN3dDmtHB9ldhLa7i5C3XnfkrbYXQ1hJkiRJkqRVqtdrxYjQweU3\n7mF2rjkfzh6YnGGimOwrB7U5uD1QhLsHugPdya1TE7ddHuK8i1e339hwEciODsyHtLtHSoHtaF5f\nfn5kqEHNsgnapAxhJUmSJEmS1tlAo85hu4Y4bNfqR+Lmyc1mO+FsO6idmuXAxExXeNsJcQ8Uo3Vb\nWyDBbY8c5tKV79Oo1zrB7HxYO3jQc7vHOvfHRqx3q/VhCCtJkiRJkrSF5MnNBhk7hFG45QD3QBHQ\nHpiYmQ9py4/3twPdqRkOTGz+EgpzzRaXHZjmsgPTK96nXe+2Hc7uKYLbPe2wdqT9/NB8eGtwq0Nh\nCCtJkiRJkrRDlAPcoxhd1b4zs03GJ2fY3x3cTsywvxTgdpY5vB2fmq3o3azdgnq3KyyZUKsxP5I2\nh7VD+X4R0l4wPsrYBfsZHmowPNhgZGiAgUbNUgk7nCGsJEmSJEmSljU4UGfv7mH27h5e1X57944y\nMT3HZfunOPePlxcjbHNA277fHnW7f2JmPsCdmNqcI29bLdg3PsO+8Rn+0HOLw/nFd3+/4Jl6vcbI\nfCjbYHiowchgsRzKQW35ueGhhqNttxlDWEmSJEmSJFWm0aizd3eeSGu0sfJgcXauOT+qth3O7i9G\n3XYezy5cNzHDXHPzFb1tNlvzk6qt1PCCkLYT4I4MDXSC3FKA26gb2m5mhrCSJEmSJEnadAYadfbu\nGmLvKiYva7VaTE7PcWBihn2lcHZfV1DbfZueaVb4Tg7N1MwcUzNzXH5gZdsPDdQXBLMj3feHS6Ht\nYIO6oe26MoSVJEmSJEnStlCr1RgdHmB0eIArHr7ymrfTM3MHBbP7xmfmw9z9EzPsH5+ev79vfIaZ\n2c0V3E7PNpmebbJvfGZF2w8N1hcNa0fb5RHmQ9t6xUe//RnCSpIkSZIkaUcbGmxwxGCDIw4bWfE+\nUzNz7B9vj7SdZn9RJ/Yr3/oeu65wLJPTc0xN59Gs7fubqVDC9EyT6ZnpFY20rdVgsH4Ue//rF9z/\n9tekUTeUXS1DWEmSJEmSJGmVhgcbDO9tcOTehcHt2Wd9hT+5wdEHbd9qtZieaRah7Ox8MDtZ3Dph\n7ez8c5ulvm2rBdNzA/znt37LyGCDU257/EYf0pZjCCtJkiRJkiRVrFarMVxMqHXYCuvczs41OyFt\nKZzNt9mFIe70HDNz1ZdIiN9dWvlrbEeGsJIkSZIkSdImNNCos3u0zu7RwRVtP1cKbdtB7WKh7eT0\nLLNzqx9pe62r7l31PjKElSRJkiRJkraFRqPOrtE6u1YY2nZG2s4yOdU7uJ2cnmNyao7ZmSlufeOr\nca9bX6Pid7E9GcJKkiRJkiRJO9BqRtr+/Adf5pEn33Udjmp7ciozSZIkSZIkSaqQIawkSZIkSZIk\nVcgQVpIkSZIkSZIqZAgrSZIkSZIkSRUyhJUkSZIkSZKkChnCSpIkSZIkSVKFDGElSZIkSZIkqUKG\nsJIkSZIkSZJUIUNYSZIkSZIkSaqQIawkSZIkSZIkVcgQVpIkSZIkSZIqZAgrSZIkSZIkSRUyhJUk\nSZIkSZKkChnCSpIkSZIkSVKFDGElSZIkSZIkqUKGsJIkSZIkSZJUIUNYSZIkSZIkSaqQIawkSZIk\nSZIkVcgQVpIkSZIkSZIqZAgrSZIkSZIkSRUyhJUkSZIkSZKkChnCSpIkSZIkSVKFDGElSZIkSZIk\nqUKGsJIkSZIkSZJUIUNYSZIkSZIkSaqQIawkSZIkSZIkVcgQVpIkSZIkSZIqZAgrSZIkSZIkSRUy\nhJUkSZIkSZKkChnCSpIkSZIkSVKFDGElSZIkSZIkqUKGsJIkSZIkSZJUIUNYSZIkSZIkSaqQIawk\nSZIkSZIkVcgQVpIkSZIkSZIqZAgrSZIkSZIkSRUyhJUkSZIkSZKkChnCSpIkSZIkSVKFDGElSZIk\nSZIkqUKGsJIkSZIkSZJUIUNYSZIkSZIkSaqQIawkSZIkSZIkVcgQVpIkSZIkSZIqZAgrSZIkSZIk\nSRUyhJUkSZIkSZKkChnCSpIkSZIkSVKFDGElSZIkSZIkqUKGsJIkSZIkSZJUIUNYSZIkSZIkSaqQ\nIawkSZIkSZIkVcgQVpIkSZIkSZIqZAgrSZIkSZIkSRUa2OgD2GxSSicCfwPcGrgyMAkE8DHg/0XE\n+AYeniRJkiRJkqQtxpGwJSml5wBfBx4EXAn4NbAfuCXwUuB7KaVjNu4IJUmSJEmSJG01hrCFlNIp\nwIuBFvAa4KiIuH5EXA04EfglcB3gQxt3lJIkSZIkSZK2GkPYjleSA9hPRsQzIuJAe0VE/C9warH+\nxJTSqRt0jJIkSZIkSZK2GENYIKV0G+DaxcNX9tomIs4EzigePmodDkuSJEmSJEnSNmAIm92xWO4H\nvrHEdl8AasBJlR+RJEmSJEmSpG3BEDa7cbH8eUQ0l9ju/4rlWEopVXxMkiRJkiRJkrYBQ9jsWHK9\n13OW2a68/tjqDkeSJEmSJEnSdmEIm+0pluPLbFdev2fRrSRJkiRJkiSpYAibjRbL6WW2myrdH6vo\nWCRJkiRJkiRtI4awWXuE69Ay24302EeSJEmSJEmSFjWw0QewSewrlruW2W536f7lVRxIvV7jiCOW\nOwxtdrvHBvj9L/53ow9jUbUatFqr32/PrsFt2z/HRofYNTa80YfRd2OjQxv2O6vXa/PL7dpvtDnY\n17Re7Gvrayf/bbavab3Y17Redlpf28l/wzZau69tRrXWoSQx20xK6X3Ag4HvR8TNl9juVOAD5Em8\nrhURZ6/PEUqSJEmSJEnaqixHkP2gWF4npbTU6OAbF8tLDWAlSZIkSZIkrYQhbPa5YjkGnLTEdieT\nR8F+pvIjkiRJkiRJkrQtGMICEXEm8H2gBjyz1zYppbsA7VIF/7ZOhyZJkiRJkiRpizOE7XgaeZTr\nXVNKr08p7WmvSCndGXh3sf6jEfHljTlESZIkSZIkSVuNE3OVpJQeD7yeHE5PA2cDe4GjyQHs14B7\nRMT+jTpGSZIkSZIkSVuLIWyXlNINgacDdwCuAhwAfgi8B3hHRPgDkyRJkiRJkrRihrCSJEmSJEmS\nVCFrwkqSJEmSJElShQxhJUmSJEmSJKlChrCSJEmSJEmSVCFDWEmSJEmSJEmqkCGsJEmSJEmSJFXI\nEFaSJEmSJEmSKmQIK0mSJEmSJEkVMoSVJEmSJEmSpAoZwkqSJEmSJElShQxhJUmSJEmSJKlChrCS\nJEmSJEmSVCFDWEmSJEmSJEmqkCGsJEmSJEmSJFVoYKMPYLtJKZ0I/A1wa+DKwCQQwMeA/xcR46ts\nr7mKzW8SET9cTfvauvrd10rtNoDHAQ8HrguMAucCXwHeFBH/u/aj11bSz76WUnon8MjVvH5E+A/D\nHaKKz7XiM+1hwIOAmwFHFO3+Fvgv4I0R8dO+vAFtGRX1tUHgUcCpwAnA4cDFwHeBt0TEJ/ty8Npy\nir7xAuCZQAN4QUS8aI1tVnIeqK2tir5WZbvauir6XLsf+XvCLYErAlPAr4EvAa+PiF+upX1tTf3u\naymlAXI/OxW4CXAk+W/o7+hkHj9Y42Evyi+2fZRSeg7wdfIXvSuRPzD2kz9EXgp8L6V0zCE03SJ/\nWTxzidv3gYk1vgVtEVX1tZTSFYBvAG8AbgUcAH4PXAN4NPDNlNIT+/EetDVU0Nd+w9KfZe3bOcX2\n02t/F9oKqvhcSykdUbT5TuBkYBj4GXABcD3gycCZKaUn9OddaCuoqK9dBfg28GbgzsAg8HNyn7s7\n8PGU0lv69R60daSUTgC+A/w9+btXqw9tVvWdQ1tYFX2tyna1dfW7T6SURlNKnwE+DNyLHIqdDcwA\nNwSeCvwopfSAtbyOtp4K+tpVyf8cfxtwN2AP8Ety4H9d4K+B76aU/m4tr7MUQ9g+SSmdAryY3Cle\nAxwVEdePiKsBJ5J/sdcBPnSIL/GPEXGzJW43j4if9+XNaFOruK99HLgF8D3yyOprRMR1gKsV62rA\na1JK11v7O9FmV0Vfi4jnL/NZdjPg5uTR1y3gFf1+X9p8KvxcO50cSkwBj4uIIyLiRhFxbeBY4D/I\nYdnrUkq37M+70WZWYV/7GHn06wHy6IojI+KGwFHAaeQvko+t8qRem0tKqZ5S+nvgW+QQ4YPk86i1\ntlv1dw5tMRX2tUra1dZVYZ94O/mf5U3gH4C9EZEi4kjgJPLn2ghwekrpan14PW1yFfa1jwE3Ai4H\nHgHsiYjrRcRR5FGx3yHnpC9LKZ3Uh9c7iCFs/7ySfDL0yYh4RkQcaK8oLt8+tVh/YkqXH6Q6AAAf\n7ElEQVTp1A06Rm0PlfS1lNLDgNuRw6+7RsSPSu2eBzwU+DzwfvJlb9r+Nupz7anAn5JPuP6pj+1q\n8+p7X0spHQfco9jvXyLi7eX1EfFH4CHAOPmk7rQ+vA9tflX0tXuQP7NawFMi4r0R0SranIuItwDP\nIPezf0gp7e3rO9JmdQ/y37Ap4NER8eA+tet3DnWrqq9V1a62rr73iZTS9cmj+lvASyLiZREx1V4f\nEf9DLisFOYh99FpfU1tCFX3tZPKAsxbw2Ih4X/t8DaDIPx5A/sc5wF+t9TV7MYTtg5TSbYBrFw9f\n2WubiDgTOKN4+Kh1OCxtQxX3tadQjDyMiEt6tDsZESdHxKMi4suraFdb0EZ9rqWUrgG8iNwXTyuf\nhGl7qrCvpdL9ry7S7n5yDcVa1/bahirsa39RLC8H3rXINm8i14fdA9xnhe1qa6sD3wROiIjF+sWq\n+J1Di+h7X6u4XW1dVfSJm5PLRM2RLxE/SER8m1zSDPJoRW1/VfS1QXLJi/8kX+V7kIj4Lfm7AcC1\n+vS6CxjC9scdi+V+cj3NxXyB/EWvkmHN2hEq6WtF3bD2pbhevibYuM+1NwNjwPsj4kt9alObW1V9\n7Y+l+yNLbDdIDv3/sMJ2tXVV1deOK5Y/jYieE6pGxAz5ywTAXVbYrra2rwG3jYiz+9im3znUSxV9\nrcp2tXX1vU9ExLsj4srAcBGALaY9OnG4X6+tTa2KvvapiHhgRNwjIuaW2LRd9uDcfr122UAVje5A\nNy6WP1/s5Lvwf8VyLKWUIiKW2LZbLaV0d+B+5DpPw+QvmF8G3h4Rl6/ymLU1VdXXbkH+sDkvIv6Q\nUjoWeAy5ttgRwHnkWSnfXowc0/a3Hp9rCxR17u5Gvjzcuok7R1V97SfkyWqOJ5dT+Uj3BkVdsesW\nDz+18kPWFlX159quZdZfSP5be4MVtqctLCIuqqDZdf/brM2vor5WWbvauqrsE+XLwrullI6k8w/P\ns6o6Bm0eG/X5U5QsuAF5gMZnqngNQ9j+OJb8Szpnme3K64+lM8x5JV5JniWw+8PpFOC5KaUHOWps\nR6iqr7W/EJ5TBGHvBnZ3bXMv4NkppXsXNce0va3H59q8lFIdeFnxmq+OCEcl7hyV9LWImE0pPR74\nBHCflNLrgdeSg9kR4DbAq4AGedSYl1puf1V9rrUvkbxWSmmwGPXaS3sykasu0560mHX92yxJm8Sz\nydnVDPCWDT4WbTMppVFyyP8g4Fnkv7Pvioh3VPF6liPojz3FcnyZ7crr9yy6VW+jwPPJI3ZGgGOA\nJ5Prix0BfLwoaq3traq+dmSxPAp4L3mE9e2AvcAVgceSR/AcDXwqpeTEXNvfenyulT2U/Pm2j0Xq\n3GnbqqyvFf+cvC25/tNfk4OIaXLtzs+S/7Y+B7jnUiMwtG1U1dc+VyxHgSf02qCo5dm+NLz7n5zS\nSq3332ZJ2lAppfsCT6Mz0erPN/iQtE2klO6dUmoCB8hX0D2bPHjjLyKisgngHAnbH6PFcnqZ7coT\nzIytsO3nFstPlWerJ18e/saU0jfINaHGyKPITllhu9qaqupr7RP0qwGfjoh7d61/Z0rpx+S+dhT5\nUvFnrKBdbV1Vfq718kzyydWbLK+y41Td124JXI98Gfjl5BFiI+T/eB9DHhH7WeCHq2hTW1NVfe2T\nwA+AE4CXp5QmgbdFRCulVCOPrHgtcD5wFfLoa+lQrPffZknaMCmlR5An66qRz9WevbFHpG3mMuBM\n8t/JY8hlpe4FNFJKv4uISkpfOBK2P9r/bR5aZrvyxCDL/QcbgIh4SXH70SLrzwROJ38wnZxS2ruS\ndrVlVdXXyiPA/qnXBhHxHfIfvxpw/xW0qa2tss+1bimlOwI3JPfDNxxKG9rSKutrKaU3kid7uwpw\nKnCFiLhhRFybPLL/bcA9gW+klJzAZvurpK8Vo6gfQC51MUzucxemlM4ij+5/L3kU9muKXfat4pil\nsnX72yxJGyml9DxyzjFAvuLk1GVqYUurEhFfjoibRcR1I+Iw8nw4XydnHd9OKd22itc1hO2P9sn0\nchMylC8/6+dIry8WywY5yND2VVVfK38h/MES2321WF49peTllNvben6u/VWx/Ooys6Jqe6qkr6WU\n7kwuQdACnhARHy2XHIiICyPiKcDHyaPL/AfA9lfZ51pE/BK4KfBi8iVtw8CVgO+SSxTcgc4oRie7\n0aHa6O8cklSplNJQSuk9wAvJ53DvAO4VERMbe2Ta7op5b06mc6X5O4srmvrKcgT98RvgVuTC90s5\nrnT/V318/UtK9w3Gtreq+trvVvj6F5fu7wb2r3A/bT3r8rlWFEK/D/kk60Or3V/bQlV97RHFcl9E\nfGSJ7T5K7oPXSymdEBFL/SNKW1uln2tFKZXnF7eDpJRScfcnK21T6rLR3zkkqTIppcPIV17eCpgD\nnhkRr1l6L6l/ImKuuJLuVsDxwE2A7/fzNRwJ2x/tL2zXSSktFWzfuFheGhFn9/H1r1S67+iK7a2q\nvlYud3H8EtsdXrp/6Qra1da1Xp9rd6IzOuzzh7C/tr6q+tpViuUfltnuwh77aHva6PO125H/4fSd\nPrapnWWj+7AkVaIYmPEZcvi1DzjFAFb9lFK6WUrpPsXVcks5r3S/7xOSG8L2R3tW3DE6M9/2cjL5\n5PszK2k0pfSPKaUzUkofWGbT2xfLKRaGadp+KulrwP/QuVztgUts92fF8mcRMbnCtrU1VdXXut2t\nWP4xIn5xiG1oa6uqr7WvErn6MtuV11+86FbaDir7XEspXSOldPeU0uAi629DnvwS8uhr6VCs199m\nSVo3KaUGeZLLW5PPxU6KiM9u7FFpG3oV+RzsLctsd7XS/fP7fRCGsH1QTI71ffKERc/stU1K6S7A\nzYuH/7bCpkfINcTun1K61SLtHgc8jHyi9ZGImOq1nbaHqvpaREwD7yvafVJxKUh3u8eTZwtsAUtd\n2qttoMLPtW43K5ZenrtDVdjX/rtYjqSUTlliu/sWy/30+XIjbS5V9bWU0k3Ik3J9GnhIj/V14KXF\nwzMiIlZ35FK2jn+bJWk9vQC4M3AA+HNLQ6kinyyWx6WUTu21QVED9tHFw0uAM/t9ELVWq7X8VlpW\nMavyfxUP3wQ8OyL2FevuDLyHXDbgoxFxamm/q5NrNbWAF0bEi0vrrgScBRxBvpzyCRHxqdL6O5BP\nro4nXxp+My852v6q6GvF+qPJQdjhwLeBhxcTjbS/YL4PuC55eP6NI+KCyt6kNoWq+lrXa1wK7AHe\nUEySpB2oor+hY8D/kWsnXgw8Hvh4e2bdlNIR5JP+Jxf7P3+pvqrtocK/oV8mj0y8BHhgRHypeP7K\nwOuB+5Fnqf+ziPCfTjtASulw8qS5bTXyiJoW8M/FreziiGitoK8dUh/W9lVhXzukdtf4drSJVdHX\nioE+UbT7lIhwolRV1dd2kfO1q5FLXjwLeGtEzBXrjwdeA5xS7P/siOh+nTVzYq4+iYivpJROI59o\nPwF4TErpbGAvcDT5l/g1Oql6W408IrlF18jkiDg/pXQv4GNFG59IKV0E/JF8cnVUsd8fgPsZwO4M\nVfS1ot0/ppTuQf4P0Z8CP0sp/RIYIl+u2yLXHL6PAezOUFVfa0spjQCH0elb2qEq+hs6nlK6O/kz\n7Tjgw8C+lNI55M+140v7vtUAdmeo8HPtMcBXyHWFv5BSOo9c5ud48peIy4AHG8DuKGfSuxxKexRr\n90jW44Dfsvz52qH2YW1flfS1NbSr7auKvvY3dHKp01JKj1vuICLipis/ZG1Rfe9rEXEgpXQy8Ang\nWsAbgFeklH4HXJGcr1Hs+5YqAli6D0prExFvAW4KnE4ORo8jf9H7MvA44A4R0Ws2+UX/YxgR3wCu\nR55p91vk39l1iuX/AH8PpIj4Vr/ehza/Kvpa0e43gesDLyf/l+ho4EjyCNmXA9ezr+0sVfW1wt5i\nuxb5UnDtYBX9Df0JcEPyCf6XyCMRr0Uusv9L4J3kumNP6NPb0BZQUV/7FXkG3VeRJ08aBK4K/AJ4\nNXCDiPjcYvtrW2rS+Ru33K3Zte9y52uH2oe1PVXV19bSrranKvra4aV9rk+eWHCp24369m60mVXy\nuRYRPyX3oycBXyCXwLgWsIt8zvYe4E4RcVof38sCliOQJEmSJEmSpAo5ElaSJEmSJEmSKmQIK0mS\nJEmSJEkVMoSVJEmSJEmSpAoZwkqSJEmSJElShQxhJUmSJEmSJKlChrCSJEmSJEmSVCFDWEmSJEmS\nJEmqkCGsJEmSJEmSJFXIEFaSJEmSJEmSKmQIK0mSJEmSJEkVMoSVJEmSJEmSpAoZwkqSJEmSJElS\nhQxhJUmSJEmSJKlChrCSJEmSJEmSVKGBjT4ASZIkSZIkSZtfSmkQeAHwTKABvCAiXlTB65wNXH0V\nu3w5Iu7U7+PoJ0NYSZIkSZIkSUtKKZ0AvAu4EdAqblX5MXDxCrY7DtgLTFd4LH1hCCtJkiRJkiSp\np5RSHXgW8HxgEPgg8EAqDGEj4p4rOK5jgLOKh6+o6lj6xZqwkiRJkiRJkhZzD+CfgCng0RHx4A0+\nnrY3AIcB74uIMzb6YJbjSFhJkiRJkiRJi6kD3wQeGhFnr3bnlNKfAH8L3IlOndfzgC8Dr4+I7x1C\nm6cCpwCXAE9f7f4bwZGwkiRJ0iqklN6RUmoWt9VMGKFCSukvSz/DR2708UiSpCV9DbjtIQawjwB+\nCDwRuA5wEbnW63HAo4Fvp5T+bpVt7gX+lVwO4VkRccFqj2sjOBJWkiRpE0opPZ9cd6vt6RHx2lXs\nPwD8ETiieOrsiLhmHw9Rh1gHbRWz/U6Sv6icCXwaeHdEjB/Ka25iVU7oIUmS+iAiLjqU/VJKtwH+\nDWgAHwCeFhF/LNYdCbwKeCTwspTSTyPi0yts+vnAlYHvRcTbDuXYNoIjYSVJkja39syzj1jlfncn\nB7BVz1yr1Wut8DYMHAP8BfBGIFJKt9uIA5YkSToELyMHsF+NiIe0A1jIwW5EPAr4AlADXrySBlNK\n1yCPqm0Bz+z7EVfIEFaSJGnzmwNuklK6/ir2eXixnCWf2GpzqQGXAbuBPYvcjgHuCryP/EXjqsBn\nUkonbMQBS5IkrVRK6WrAbYqHr1li09cVyxullFZy1dbzgCHgK1thMq4yyxFIkiRtfl8Hbke+XOvZ\ny22cUtoD3Isc3H0LuG2lR6dDFhETS6weJ09acUZK6SvAm4Ex8heZO63D4UmSJB2qW5fu/2SJ7b5d\nun8z4FeLbZhSugp5oEELePmajm4DGMJKkiRtfp8FTgIeygpCWOBUYIR8EvtLDGG3vIh4a0rpscCf\nArdPKR0TEedu9HFJkiQt4ujS/Z+llFayz7HLrD+NPAr2VxHxn4d6YBvFEFaSJGnz+xTwUuCqKaU7\nR8SXltn+YeQRAp8G9i7XeDHD7BPIdWSvX+wzDfwO+G/gdRHRcwRDSul0cr3aFnmUJsDTgAeRZ70d\nAs4p3sPLe81e2482utq7O/CXwInkSRsmgXOBzwNvjoj/W2b/OwJ/W+y/Fzgf+B/gtRHx7aX2rdgX\nyCEswAnk93SQlNINybXS7gBcjVz64DzgG8DpEfGFpV4kpTQMPBa4N3Bj4Arkkhh/II/KfmNEfGOZ\nNo4H/gG4C/lL2CXAj4G3RcQHlnmfpJQawKOA+xfHcGTxPi4EfgB8CHhPRMwu15YkSdoQu0r3f0Q+\nl1jO/sVWpJRq5PO7FvCutR3axjCElSRJ2uQi4qyU0i+Ba5LDykVD2JTSMcDti4cfAR6zVNsppZsB\nnwGuxMIJvAaABFwXeExK6a8j4p09mmiW7h9NDsdu0dXWtYGnAw9PKd2hRwjajzbaZRjeT57IitL+\nQ8BhwPWA01JKz4qIf+nxXkgpPYN8eVuttP+x5ED4/imlx/Xab51cVrrfM1xPKb2IHH6Wjx9ymH08\n8NCU0vuAv4yIg74MFZNdfIH88y7vP1jsf03gYSml50dEzwk0UkonAf9B/vLVbuNK5BIKd0op3Ykc\n5vaUUjoC+Bxwcw6eVO4qxe3u5N/l3SLiMiRJ0mZTDlTvHRFnr7G9k8j/XG4BH15jWxvCibkkSZK2\nhveTg7X7pZRGl9juYeRzvN9HxFeXajClNAZ8HDiKfEL7SvKowyuSR8Q+EdhHDmTfnFK63jLH+Arg\nBsCzyOHtkeQg7QNF+0cBHylGMlTRxgfIAWwL+Bh5FOZVyCfsDwN+QQ5kX51SemiPn8et6NQXuwR4\nfLHvlYA7AmcAbyQHlBvhqqX7vUYUPxt4bvHwB8ADyKHplcgTfH2R/LN5CPl99PJhOgHs6cAti/2v\nQ67B9kdyP3xhSumuPY5hNzlEHyOH6y+l83u8KfCv5FG2D1vifb6WTgD7ZuDPyL/HK5NLa7yrWHcL\n4A1LtCNJkjbOOaX7y5UZWIl7FstzI+KsPrS37hwJK0mStDW8mxyw7QLuB7x3ke3apQjev4I2H0g+\nKW4Bb4iIZ5XWXQJESmkSeAf5vPE04G8WaatGDv3uGxGfLD1/Jnn05V7y6MXrFsf/kX62kVJ6MHBy\n8V7eHBFP7Gr731NKXwC+V7znV6WUPh4R46VtnktnBOlDIuLzpXVfSSl9jTxq+G6L/Ayqdo9iOQt8\nt7wipXQs8ELysX8POKlr0q8zyBN8fZL8JeYxKaW3RMR3Sm3cgU74+R8RUR5FfRHwy5TSucB/Fc89\nhTxqtuw0OqH+KyLiuaV1lwJPSyldACw2inaA3AdawGd6/B4vBL6RUpoG/go4NaX05Ii4pFd7kiRp\nw5RLOJ0IfG2xDVNKjV5X6HS5G/n84Ct9OLYN4UhYSZKkLSAifk7nZPYRvbZJKd2APJIV4D0raPYA\n8O/kuq+nL7LNB+jU8PrTRbaBfFL87a7wtOylpfunVNDGU4vlpcDf9do5Ii4CXkQOWq9EZ0RFewTn\nnYtj+GFXANvevwk8Z5Fjq1RK6Xl0Rqh+LCIu7drkyeSSAQB/2xXAlj29WNY4uFRFnRzunwG8tdfO\nEfHf5NqwNXr3h/sWyznyyOpe/pkc6vZyRfKkcrD0TMr/ANwGuLoBrCRJm09EnEMOXmvAk4orsA6S\nUnoEcEFK6fTFrpYq6tVfv3i41PnBpuZIWEmSpK3j3eTg684ppaMj4o9d69vh7FkR8aPlGouID5Ev\nHV9qm8li5OLR5EvKl/LxJdZ9E5gilwO4eT/bSCkdRqeG7Ocj4sAy7bcDxrsBHyzu37Rot0W+bL+n\niPheSukP5Mvj1yyltGuJ1buBG5FHl7bDzfPpHTLfpVieFxGL1luNiF+klH4M3JCuEb0RcQY5gF3O\nb8jvf0F/KCbTuhn5Z/iDiLh4kWOYSSl9nlwWodsl5JG+DeABKaXXFV/iutu4kDwqVpIkVSyldDj5\nb3NbOSwdSyl1nyNeHBEt4NnkkavXAP4zpfToiPhl0eYQeaKt15L/AXtZsU8v1ytevwXEWt/PRjGE\nlSRJ2jr+HXg1+RzuYcCrutY/hHxyupJRsKsxVSwbS24FP1xsRUTMppR+C/wJeZKofrZxAnkUZws4\nd5lgc4pc1/TKdEZUULTZ9ssl9gc4i7WHsC3gcHLN3ZVs2/7S8YCI+F15ZTFq5IRim3OWef8APyWH\nsMellEYiYnKVx97uD91X1V2DTpC9kp/hQSJiKqX0QeCh5Hq2Z6aU3kWuVfutFVyqKEmS+u9M4Oo9\nnq8BzyxuZccBv42Ir6eUHk3+B/htgJ+nlH4DTJPLQ42Szxu+0KONsqNL9xe7mmbTM4SVJEnaIiLi\nopTS58iX0T+CUgibUro9eRKpJvC+1bSbUro3uT7sjciX6R/J8oFrL+cvs749MnI0pTQQEbN9aqM8\n+uJpxW0lyif0V+jxGovp18n/YqM9uv0X8Bbgw0VJhG576YwOuQUrC3ahU5bht+Uni9qwDyePar0y\nucbrSr439Otn+CTyF73bFG0+tbjtSymdAXwK+EhEXLaCY5IkSWvXZOXnLQu2i4h3F3X1n0ou/XR1\n8j9tLwK+BJweEb3mCijbS+ef0vtXcdybiiGsJEnS1vJucgh7o5TSjSOiPXK0XYrgf7pHSi6muIz/\nY8Adi6fKJ83dJ9o9a3R1WaoMAORRD21D5MvO+9FGeeTnSr8gQL7cv220dH+qe8Muy61fiRpwaUQc\n0WtlSukW5PILNWA0Ij7Ya7vCob7/FrCn9JoD5P71oB5traQ/9OVnWISrJ6WU7kcO1E8kj7rdDdy7\nuL0mpfSyiHjpYu1IkqT+iIhrrnH/XwN/u4b9P0Cep2BLM4SVJEnaWj4JXAYcRg5e/66oqXV/Vl+K\n4N/IAWwL+DXwBvLlYOezcDTlWfS+BK3b8DLrh0r3F7sE/lDaKI+IeFZELDYh1FLKoeDQoltlPSeW\n6KeI+E5K6S3AE4ATU0pPiojXL7J5+f1/NiLuuch2y3kpOYBtARcArwc+A5wLXE4niP1P4LY99u/r\nzzAiPgp8tKgzd3Jx+3PyyOc9wD+llI6JiKcs15YkSdJGM4SVJEnaQoqamR8GHgs8mDxJ0z3Il2lN\nk2tnLiuldHXgfuRg7ffALSLi0kW2HVzh4S03cddRxXLfIpfVH2ob5/dYv1qXl+5fYdGtsqOXWd8v\nzyGH60eRA8ePRcS5Pba7nM6EZYf0/lNKo8Bfk/vDAeDEiDh7kW0XK1VRyc8wIi4C3gu8t6h/+0Dg\ndeS+clpK6U0RsWVnSpYkSTtDdzF9SZIkbX7vLpbHpJRuCtynePyZiLhkhW2cQOeS8vcvEcAeRQ7M\nVnKZ+/UXW1GM1j22aOdXfW7jh+RaZQA3WcFx9lKeSGrRS+6KEHDRY+yn4nfSnqRiD3mkcq/tWuSf\nQQ24XlFWYLWuTac8w2eXCWCvt0gbv6bze1jussUbrfYAIb/X4pLE04qnauT6cpIkSZuaIawkSdIW\nExFfAX5TPDwZuDs5mFzNhFzlOqJLBbdPJgdd7dtSlroM/rZ0LlH/Zj/biIgDwDeK47t9SukqizWQ\nUhpMKT2zCK/Lvk8nQLzTEsdwB5Yf5dk3EXE68DXye7tXSun+i2z6+WK5i04o31NK6a9SSnfpGuG8\n0v7wSODwRY51GvhJcaw3TSntXeT1d7PIzzildN2U0mNSSqf1Wl/yo9L95UofSJIkbThDWEmSpK3p\nveSw6/HAFcmXgn9qFfufU7r/Z702SCndk1zuoH25/1KlAmrAXVNKt19k/bNL9z9WQRtvLJaDwBuL\nEau9vAB4GfCdlNJd2k8Wl7x/vTiGm6eU7ti9YxFavoTVTX7VD0+kM4nZvxYTqnX7N2CGfPwvTyld\nsVdDKaU/B95MDm2fX1pV7g+3XGTfPwVeTa4R236u+3U+USwHgKf3agd4IQsnRSt7PPA24F9SSics\nsg0sHP1qKQJJkrTpWRNWkiRpa3o3uWbo1cmh4EcjYrkZ6cu+BZwHXBk4JaX0IuCd5DA3kWvOPhJ4\nOXA8uf7srpTSE4H3AxMRUZ5cqwV8FvhESun55ED4YuBaxXG2A89vRcQXFjmmQ24jIt6XUvpL4K7A\nKcAZKaWXAGeSR0om4EnAfYvX+WJEfLHr9V8K/Ac5yPxQSun/I4eVM+TyDf9Ivoz+U8VrrIuI+HFK\n6V/JoebRwD+T67eWtzk7pfRC4MXk39e3U0rPA/6bPIHZNYCH0BnZ/AdyoNre/5yU0pnkcg43KSYF\nezU5gD8eeDi5BMCHyaOw/77Y9akppdeQ+8M4uVbrU8kh6z+klADeRf49Hle08VjyDMcP6vF2/5U8\nGdkw8OWU0j+Tfyft4Peq5FrG7UD+p3RGAUuSJG1ajoSVJEnagiIigO/QKRHw3lXuP0MO5OaKp54L\n/IIcun0V+EvgHcDzgE+Xdn0dcBFwtx7N/j3wM+A1RVsXA/9LJ/j8NTnMXcpa2rgv8Jliu5OAz5GD\n5t8BXyy18SXy5E4LRMRnyQFni1xy4B3kScvOB74A3Ir8M/vZMu+hCi8gB5E14LEppZO6N4iIlwD/\nRP6dXoMc1P+WfPz/Sw5xB8n1dP88Ii7uauJJwHhx/6+As4ALi33/hvwzfDw5FG17DnBBsT0RcT7w\nCPIkcTVy//k5uc98lxzAvh14T683WdSiPRXYDxxGDpW/T/49ngd8j9xXG+Tfw70jYq5XW5IkSZuJ\nIawkSdLm1WLpS9/fXaw/NyLOWG07EfER4I7kUO1C8iXvFxWP7xERj4uIJnnk60vJIeAk8GNysNnt\nMuA2wDOAb5Nri44D/0cuAXCziPjtEse5pjYiYjwi7gncizzS8jfARHHMvyGP4rxPRNwtIi5bpI1n\nkWuqfr74mUwV7/XfgdtFxDuBA3R+po1l3s9ilvvddh/XfuBpxT414M3FRGXd2z0PuCm5PMNPgX3k\nQPR8coj6JOCGEfHjHvt+AzgR+CDwR3J/uAz4MvBQ4F7Fz/jr5NGuvyH/fH4ORKmdTwA3J/9j4Jxi\nm/PJ4ffDI+Jx5JC1588wIv6DPHL5ReQSEReQRyNPkUPxzwCPA24UEeUJ1SRJkjatWqu13iWtJEmS\ntF2klN5BHjXbAo5fQchaSRuSJEnSZuZIWEmSJEmSJEmqkCGsJEmSJEmSJFXIEFaSJEmSJEmSKmQI\nK0mSJEmSJEkVMoSVJEmSJEmSpAoZwkqSJGmtWsVto9uQJEmSNqVaq+W5riRJkiRJkiRVxZGwkiRJ\nkiRJklQhQ1hJkiRJkiRJqpAhrCRJkiRJkiRVyBBWkiRJkiRJkipkCCtJkiRJkiRJFTKElSRJkiRJ\nkqQKGcJKkiRJkiRJUoUMYSVJkiRJkiSpQoawkiRJkiRJklQhQ1hJkiRJkiRJqpAhrCRJkiRJkiRV\nyBBWkiRJkiRJkipkCCtJkiRJkiRJFTKElSRJkiRJkqQKGcJKkiRJkiRJUoUMYSVJkiRJkiSpQoaw\nkiRJkiRJklQhQ1hJkiRJkiRJqtD/D4SYnUYibkbwAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<matplotlib.figure.Figure at 0x7fe65d5260b8>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAABTkAAAP7CAYAAACQh4rlAAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAAewgAAHsIBbtB1PgAAIABJREFUeJzs3XmYZFddP/53z4TsEAi7CSQQ5EMQAZEdWUQFIiFEIoiA\nJICKCLigoCyCIPhDZBP8KipgECQIKAiIsgmCsskeWQ4KBAhLAmQhJjEJmf79cW7RNZ1eqme6enIn\nr9fz9FO36p5776nl9ky/63PPWVhcXAwAAAAAwFht29MdAAAAAADYHUJOAAAAAGDUhJwAAAAAwKgJ\nOQEAAACAURNyAgAAAACjJuQEAAAAAEZNyAkAAAAAjJqQEwAAAAAYNSEnAAAAADBqQk4AAAAAYNSE\nnAAAAADAqAk5AQAAAIBRE3ICAAAAAKMm5AQAAAAARk3ICQAAAACMmpATAAAAABg1IScAAAAAMGpC\nTgAAAABg1IScAAAAAMCo7bOnOwAAdFX110lOHO7erbX23j3Zn/VU1RFJvjTcfU9r7e7L1r8nyV2G\nu5f75zNRVScm+evh7u+31p6xJ/tzeVVVhyZ5epJ7J/mBJDuSfLS1duc92rEpVfW0JE8b7p7UWvub\nPdmfrbTW75OqumuSdw93T26tPXyr+7c7qupmSf4jyZWTvD3JMa21xT3bq71bVd0gyYeTXD39tb97\na+2SPdsrANiZkBMAZrQsGFjJpUnOSXJ2kpb+B+E7Wmsf3OChxvbH+mr9XVxn/eXdWPs9d1W1T5J3\nJblFdn6dfmCT9n/XJPdMD8kPS3LN9P+3npfky0k+leRfkvxja+2iGXZ5RX4v13ruo3tdqupaSf4p\nycHpv2fvP0vAWVW/mOQvpx7649ba78yw3WuSPGAXu7vc7VtrH57hmKcm+aHh7o4kR7TWvrY7B66q\nOyS5V5IfT3J4kmsk2S/9nPpqklOTvC3JG1prFyzfvrX2pao6If28v2OSk5M8eHf6BACbzeXqALBx\ni6v8bEtyaJKjkvx0epXb+6vqU1X1oBn2+40knxt+LvNH5maoqntU1Y6q+tdN2N0l6X1t6cHTKFTV\nI4fX4OWrNDk3S8/r21vXs1H5ySwFnOck+aUkt0nywN3ZaVXdq6o+mv5lwu8muUOS6yc5IMmV0s+v\nH0lyUpLXJPlKVT1md455BXVBlj7j31ipQVWdMpwnD93Snq3vb5JcL8n3kjy4tXbejNs9Mjv/vj5x\nCOvXs9rv+43+7Jilk1X1Y+kB52S7hSS/OMu2q+zvrlX1/vTqy99LcqckRyQ5KP2Lg6sluXmShyR5\nZZKvVtUTquoyfycO1cDPHvr0wCE4BoDLDZWcALBr/ifJ8el/7E1M/mC8XpLbJblvehXazZK8qqru\nn37J7Lkr7bC19qQkT5pnp5PcdrN21Fr7epKbbtb+ttBtskYFW2vtjUneuHXdGaUfnFr+89baaoHx\nzKrqmVn6/C8mOS3JPyR5X5Izklyc5NpJbp/kQelfJlwzyYuq6ieTPKC1dvHu9uOKoLX2n1n/3F3z\nPNkTqurRSe6R3q9nt9Y+NuN2t07yo8N2X05yZPpn535JXrvO5r+V5A/WWP/q9JBwMcmjk6w1LMcX\nZujuo4bbS5N8M73q8hFV9YyNXpJfVb+T5FnpX8AtJvla+jn1nmHf/5fkWun/Lvx8kpuk/xv27CT3\nqKrjVqjqfHqS45L8cJIXVNU7W2unbaRfADAvQk4A2DUXtdY+u8b6V1XVryf5uSQvSq9Au2+SN1fV\nT814ie08bFrIOWJeg913tanlWYKbNVXVc5L8dnoQc0l6FeeLW2uXrtD8X6rq6Ukek+SP0ys875Pk\n5enVaOymYbzVG+ZyFHJW1dXTw8bF9OrTZ29g81+dWn5MkrcMy4/MOiHn8GXO19fo14VTd7/SWvvM\nBvq1fF9XT3JC+nN8b5KPpZ8XhyU5NsmbN7CvJyV55rCvS9PHpn3uKl8EvD3JM6vqEen/Xu2ffln7\n36WfW9/XWvteVf3WsM2BSZ6fHhYDwB7ncnUAmJPW2o7W2ilJbpmly7nvlOTP91yvLn/VWVupqg5K\ncvSe7sdeYLqCeaUgcmZVdWyWAs7vJTm+tfbCVQLOJElrbbG19uL06rOF4efnq+q+u9MXvu92e7oD\nK3hKkqsOy89srV24VuOJqjokfUzNxSSfa629NckH0j8zd6uqG82js7voEUn2HZZfm+R1U+seOetO\nquou6RWXk0veH9Ra+8P1Kp1bay9Lr9Lckf76/PQwEdvydu9M8q9Dm/tW1R1n7RsAzJNKTgCYs9ba\n16rq3ulVOfsleWhVvbi19vHpdrPMrl5VR6X/IXy39EuGr5IeMn07yafTJ+Q4ubX2v1PbrDRh0t2q\n6vtjxLXWtq3Q9rmttSdU1QPSx3K7cXrV3FVba99db3b1lVTVPdLHl7tNkuukjw343+mXUL6otfZ/\nK2wz3ac1j7OsT6e11m44PD49Y/pknLuTquqk4bEvt9ZusELbVWdXr6rt6ZWDx6VfCnuNYb/fSZ/E\n45+TvHylSTyG7X8/yVOHu8e01t5WVddLv+T1+PRx8xbTLyt9b5LntdZOXe25b0RV/UR6QHin9Pfh\nwCRnJfli+sQif7HSRCfTn5nBQpKTq+rk4f73X8cNmK7I+6PW2r/MumFr7Q1V9dokP5s+5uD2DR47\nSVJVB6SfV8ekX3p89SydVx9Lr6B7VWvte6tsf3KSydiVR7fWWlXdJMmvpU/28gPpAe7XkrwzfdKb\n09bp0w+ljz36Y+nn+pXTq1zPTPKRJH/bWvvHXXy+K86uvux5JJd9f1+R5LtJHjvcf0Fr7bdmON6T\ns3TJ95+31h69wf5eNf39SfoYsK/YwOYnpn++F5OcMjz2t+njvSbJLyd5wkb6M0e/NNxekuT1rbWz\nquq/09//e1bV4a2102fYzx+lnwuLSf6stfa6ddp/X2vtXVX1sqEvH8rqX2K8IMnkd/FvRzUnAJcD\nKjkBYAsMl7ZPZvZdSPLENZqvWGlZVY9N8pn0S3lvl34J/D7pwelh6bNRvyhJq6rll2RPT4Cx/P5K\nE2IsDsf82fQJXm46HGvVtuupqj9OnxH7hPTJZPZNr8y6bXrQ9Ymqus4au9hIBepKbReXPT7Ta7CS\nIYD6XHoY+jPp47AekH6Z52HpYdmLknyhqu68Th8Xk2yvqh9PD0cfnx5q7Jv+3h6RHjx9uKrutVqf\nZlFVV6mqf07yjiQPTw+ur5L+3l4rfbzLpyT5n6r6tVX6vNHXca3+3DtLY0Oek+QPN7L94DFJrtda\nu2tr7R82uvFQ9fbF9PfrmPRAcr/0YOz66e/vy5N8tqpWG8dy+Xv5wCSfSK++OyL9y4ED0t/XRyX5\neFX9yBp9+qMkn0wfD/K26cMD7DPs44j0UPcNVfXWqtp/o895hX5P31/v/X3p1OMPWmmCmhXcf2qb\nXRm/9RfSZ1NfTA93Z6riHEwqIBeTvGpYPiXJRVn6suNKu9CnTTV8AXRUej//ubV21rBqEuhuy1II\nutZ+7pSlStz/y9IXKRvxO0mObK3dsbX2qpUatNb+Kcnp6a/hfarqB3bhOACwqYScALB1/t9wu5A+\nqcPMVWdVdfckf5IedJyfHgbdM32m6TulV3y9I/0P5Oumj/15yLD5h9MnibjncOzF9EqwHx5+br7K\nYa+UPt7aZ9MnevnR9Mqd82ft95QHpwc2H02v5Lx9eiXVr6ePr7eYHgD9/S7sexZvSH+uj8jSa/CP\nWXoN7jnrjoZq0fdmaczCDw37vX16herPp1dxLqZPlPO2qrrlOrs9cujjuUl+I/21uXX66/bJYV/7\nJnn5jDNCr9TvfdKrCO857O/09ED1LklulT7m31+kV5Htmz6pyK8s283k9XrJcH8xPRTd8Os4OHZq\nP3+7UiXvelpr326trThD+HqGLwPelh7wLqaH8A9ODxZvl+RhST44rDsqyb/NEObcNn0G8P9OD6Vu\nO/z8YnqYupgeLL90pY2H2eIfn/45vTjJC5P8dPrn4Q7p1aGnD/u5Z5I/2/ATX92Tsvb7+5Shmvgj\nQ/+ulR4Mr6qqbpyliXk+01r76C7062enlmf+HTEE2EcPx/63SfVsa+3s9PM/6VW7J+xCnzbb9Lk2\nHQS/IktfHjxihlB5+pz6+9baORvtSGvt3NbaV2doOpmgbVsuH68hAFdwLlcHgC3SWvt8VX01verv\nyumh2H/MuPn0xBkPba29Ydn6DyZ5ZVU9Pz0ku0Z6qPK8oerpM1U1HU6eP8MEGSekX6p4x9VmhN+A\nX0rypiT3a61NV/t9uKrelH5J8KFJbl9V92mtzTzBxixaa99Nfw2uOfXwObs4SchfplfWLSZ5dWvt\nF5at/1iS11bVC9MDqf2S/FV6ALqa/y89FLv70NeJjw+Vl59Pnw362umzS791F/r9+PSgbDF9aIO7\nLAtAPpnkn6vqHelB0mKS51TVG1prZyTJ5PWqqjOntvvabky2ctep5eVDKsxVVS2kB0iTMRCf3Vp7\n8rJmH6mqVyZ5fXpF56HpXzbcP6v7k/Qw+b6ttUumHv9oVb0n/UuDfZPcsqpuOv3aDX16UpYqKU9q\nrb1m2f4/XFVvSa/qPiB9+IunzRhKrWkIi78xw/v70vTPUtIvB/+nNXb7gKnlv1611Sqq6srpX+Qk\n/QuWtWYvX+5RU8svW7buZVN9e2R6xfoeMQTnx6a/72dk6vUchjt5W3qYfN30iYDWGqZgK8+pt6ZX\nUifJvZO8eM7HA4A1qeQEgK31X1PL19vAdtOT5bxtjXZPT/Ks9D/a37mB/a/ksCRP34SAcyH9sslf\nWRZwJklaa1/Jzn8cP3A3jzc3wziLP5UeRpyTncPn5X43ffzEhSS3qqrVQs6F9EtxT1oWcCbpVVXp\nVZ4TP7oL/d6ePtbnxKNWq/AaAvQ3D/06KDuP0bjZpid9+fiqrebjnklqWP7v9HFnL6O1tpj+Pl+S\npYlWVhtWYSG9iODEZQHnZF9fSvKeqYeWv5dXTZ9s5jVJ3rpCwDnZz5ezFIQtpI/Ru5VOSQ8cJ5cq\nX3WNtpMg8dIsXS6+EbdN/5tlMcmHh/djXcMXGj8z3D03l60AfWeSr6Q/h7sMFad7yi9nqfjkFSv8\nnpyu7FxvAqIfnFqe9zn14eF2If19AoA9SsgJAFvrO1PL11y11WVNX8Z769UaDZcZ/l5r7aWttU9u\nuHdLJpd0v3G9hjNYTPL21to312jzlqnl1cawvDw4fmr5H1pr563WcLj0ejqc/OlVmi4m+c/W2n+t\nsj7p1X8T11q3l5d1m/SxJheTfKG1tl4F8aunllfr926pqslYoBPfnsdx1jD9Xr5ypQB+Yqhk/dfh\n7vasfln+YpI3t9a+tcZxV30vW2tnt9Z+vbX24NbafdbYR9KD2YktHQ9x+NxPJrPZN32IhssYvhS4\nWZbGmTxzpXbrmB5O4xMb2G4yU/liklNaaxdNrxzC0pOnHvrlXejbbhsuP3/E1EMrVbu+Kf38WEjy\nU1V1/VX2tZBeZT4x13NqGDd0UkF8SFUdOc/jAcB6hJwAsLWmZ9o+cAPbTQKWhSRvrKpHDyHRPJ25\nK+O5reJD66z/9HC7kOSwqjpok4672aYD5vWeU9LHIJ24xRrtPrLOfqYrPDfyuZmY7veHV221ZNLv\nhazd791x8LL7K85CP0ejeC+r6kpVdf2qOrqqfmjyk171ObHfevuZg+kxRU9cpc30Zf0n7+Jxjpxa\nPm2WDYawb3qSntUmO/rr9BB0IcmJVbXvKu3m6bj0qvnFJO9vrX1+eYOhKnhSBbvWBEQHZOe/77bi\nnPrS1PIRW3A8AFiVkBMAttahU8vfWbXVZT0zfQy+xfRw48VJvlVV/15Vz6yqu23yDMGLSdaqRtuo\n09ZaOVQ9nj310K5UK26F6SEGvrRqqyWnTS0ftka79T4L01WGCzMcd7nd6fchVXXALhxzPcuHQThk\nxVbzc7l9L6vq8Kp6blV9JsmFw7E/neTUqZ9HrbTtVmmtvT+9KnUhyW2qqlZoNrlU/TvpQyDsiutO\nLX99xm3uleQG6b/HTl1tsqPhsv/JF0iHZucJjrbK9IRDy8cNzSrrHr7SBESttQuSfG/qoa04p06f\nWjbDOgB7lJATALbW9B/sM19KOIzLeMf0SU3OT//jfZ/hsSel/6F+ZlW9bBPHljt7/SYzW/Wy7inT\nVUeX10rO6dDgf2doP91mrcrbS3etOzPbUL9ba99Ln9l7YtOrhltr5yeZvoR4qwOSy+V7WVX3TQ8P\nH5fkJlkaOmKlnz1tOng7aXpFVR2d5IfS+/mq4TO1K6Yrfs9ftdXOpsetXG+yo+nnsN54l5uqqm6Q\nPsZv0p/ba1dr21r7dHoV9kKS6yS57ypNp0P2rTinxvB7G4ArCLOrA8AWGS6FnJ5oZENjZg6T0jyu\nqv4gfebzY5LcPUuBy1WSPCx9tuXHt9ZeuJtdXnWMwl0wS/AzXdW2mcfeTNPB0iwVldNfKO/J57TR\nfi9vN6++fypLs87fPkvDFmyFy917WVU3TZ/UZ7/0/n0iyR+nX07/7ekxYKvqaUmeNo9+bMDfJPnD\nJFdK8pCqetLUxEDTs6q/YjeOMX0p/kWrthpU1WHpM31PLkN/QVW9YIbjLCT5saqq1lrbpZ5u3K9k\nKcQ+OMn/rlwQu6Jfzs5j/k58KkvB6e2TrDf+7u6aHi96TwybAADfp5ITALbO3ZPsn/4H7ReHmZY3\nbJic5KWttROSXD3Jj6XPqP6VYd/bkzyvqu69Od3eFMvHX1zJ9PiEs1TWrWTeX+BOj1F65RnaTz/v\n3Z2lfndsqN9DID89/MG8+v6+qeW5THC0hsvje/mE9N8RSX9tbtdae01r7UsrTHK1fU59mFlr7dvp\nk+IspFcN3n1q9STk/ORuToI2HWzOEqI9MkuvzWoVsGtVxW5JNecwvMhJU8feSF8nExAducKut/qc\n2n9qed0QGgDmSSUnAGydx00tv34zdjjMCP2BJB+oqmekX84+GavvqUn+aTOOswmut9bKqjowSxOp\n7EhyxtTq6QBivS9or77xrm3Il5Pcblg+Ksk71ml/w2Xb7inTxz5qhvaTfi+mT0B18VqNd8Pr0s+L\nhST3qarrtda+us42OxkC2bcn+cckfz6M7zqLLye59rB8VHaerXwlW/Fe3m1q+Q/WucT7hmus20ov\nzdJYlg9M8q6q+pEkR6d/flab9GdW05eor3k5dFVtT/LwqYeel53HjFzN1dKrYhfSK+F/d46f+Yn7\nJ7lmhi+90sdZnsXx6Z+TyeRKT162/vVJnj6sv1tV3bS19pmNdGwY7/OtSf4tyYtba2t96TT95dSs\nwwkAwFwIOQFgC1TV8Ul+crh7Qfof35uqtfa9qvrNJA9NDwN+eLOPsRtuvc76mw+3i0m+1Fqbrgi6\ncGp5vYq7yaXPuzI5zyw+lKUKtTskeck67W87tfyfc+nRbKZnD7/9DO23pN+ttQ9V1QfT+7Q9yYuS\n/MwGd/N7Se4y/Byb5Cdm3O5DWXqed0jyL+u034rX5DpTy59drdEwEdS95tSHjXpHeuh7RJLjq+qR\nSR40rLskyat3c//fmFpeb4zJ+w5tFpN8ubX2hFkPMoyFesv0wPMBWZrNfF6mK0b/tLX2olk2qqpP\nJHnPcPdhVfXU1tr3hwNprX2uqt6Wpc/Hn2bnCttZ/GaSeww/J2Tt39+HTy1/Y9VWALAFXK4OAHNW\nVbfK0uQWi+kVWjNPOlRVd6yql1TVB6tqzbBwqD6aVLJdsEbTrfyicyHJvavqqmu0OX5q+V3L1k1m\neV9IcqOVZhWe8vA11i23K6/BG9IrTRfSA51Vn1NVXSVLz2sxyRt34Xib5RPp1WILSa5XVesFgSdN\nLf/DvDo1+I30GaEXkhxXVU+ZdcOqOinJE4e7lyb5/Q0c9++nlh9SVat+HoYJYu4y3D0/vXJ0HqYD\n/bWqkp+YPhv4xJVWa7gJ1jxPhjE4J5P7HJoejD0g/TP/5tbaWbt5/NOmlo9cp+30TOV/ucHjTLef\n6yXrw9irdx7uXpQ+tulMWmvvTdLSz5drZ+ffnROPT/93YCHJXavqjzfQtxPSx1lN+nu43rivR04t\n78lqdQAQcgLAvFTVtqr6pfRL/q6W/gfjP7TWnrPBXR2YPsnEbZK8cAjPVjvmT6eHI4tJ3r9s9fQY\nhFt9qetBSf7fSiuq6kbZOZz42+n1rbXTstT3g5P83Cr7eXr6JbJrXeI7/Rr84Jo9XsHQl0nod3CS\nl1TVZapGh8f+LP0S/MUkb2ytfXGjx9ssQxD1/KmHXlxVK4ZoQyXe3Ya7X0vymjn37cPpod1krMFn\nVNXLqurQ1bapqu1V9cT0Lw+2Ddv+bmvtfatts8Jx35vkI8Mxj8xSsLP8WPstO85frnP57u6YHrty\ntc/5ryZ5UpK3TD18xCb3Y6PnycuzNBnTH2VpeIqTN6Evn5pavsVqjarqqCxV8X4v68+qvtzfpgfY\nC0nuOMwOPy+TIUUWk7yutXb2Brf/q6nlX16+cpiJ/deydE79VlWdUlXXXt52oqoWqurX0s/3Kw3b\n/mFrbdUhT6rqakmuP9z97q6OMw0Am8Xl6gCwa/arqh9a4fGD0i85vVOS+2Vp/MNJtdOvbvRArbV3\nVtU70y93v2OST1bVyemX256Z/gfpddMv1X3IsNmlSf5g2X7Oraqz0qutrltVr07ytiSHpFdczesP\n1MUkz03yhKo6Iv0S78+lX558hyS/mz4z/KTy699X2Merkjwm/Q/2vxoq696TXq16VPrYdD+Vfpnl\n04fntJLT0sOYbUluW1UvTvLhJNdK8rLW2jmrbDftMelVWNdOr1i7XlX92dRzuml6Jdhk7M5vZhfe\n9zn48/TP5I8nuUmSj1XV89Kf/8XpQdkDsnQ5/iVJTtzAGJe7rLX2vKq6KMkL0t+bhyW5X1X9Q3rV\n5Onp7/W10j8zv5DkBumfme8leUJr7YW7cOiT0s+jg5L89nBOvzy96nX/9MuXH53+ni4m+UySmStN\nd8Er0itGF5L8znBZ+pvSq/Jukl6pfOckr02/tP/YYbv7VtXPpY8r+l+bMJ7kF6aWH1VV301/Dw5p\nrf3p8sattdOr6u3pl0jfbHj4jCT/vJv9SPr7M33OLkzN4D5teqbyf2ytnbFCm1W11s6rqtckecTw\n0CPTq4w31fCePmTqoY1WnCb9c/Ks9ImYfqKqbrD893dr7aVVdcmw/33SQ/N7V9Ub04dm+GqS85Jc\nI/131S8kuXH667cjyTNaa89Ypx+TIRwW03+PAMAeJeQEgI1bSK9uOnWddpOZcP8nydNaa7tTEfeA\n9Aklfjy9cma1SwgXk5yV5GGttZXGDXxJehVY0icJeeCwzSeSTP+RvJExLWdp+3fpYdQT04Pa5X1e\nTK+qe+gq2z81fVy5o5MckOSZK+zjha21Fw0VnSsagt7XJPn5od+PHn4W0yfBWTfkbK2dWVV3Sg+f\njk4fT/IOqzynTyb5mdbamevtd95aa4tVdZ/0wPj49LH0VgoGF9PD819orb17C/v3p1X1kSR/nP4Z\nuUp62PmwVfo4qVZ+3Cqf9VmO+Zmq+vH06tzDkhwz/Kx0rHcnecA8Q9/W2l9X1bHp78+29ND+N5f1\n5bXpgdSO9HP2yPSxak8Z1t8gyVd2sytvS/9C4Ij09+FZw+OnpY/xuJKXZmkcyMUkrxwmRtstQ/j4\ngfQvjg5O8mPZeQbxyeRTJ009tCvB4WS7R6T/bviFqvqdZeMDb4YHZ+lLmM+21v5joztorX1n+AJg\n8nvskelfFi1v94qq+mT6OXX39DD/Idk5ZJ2YfM4/ln5OzVIVPX2uvHVDTwIA5sDl6gCwMYvr/FyS\nPvnCx9Ivzz4uyU02EHBO9rOT1to5rbWfTK/cemWSTyc5Nz04vDC9yupt6TNV36i19uZV9v/76QHh\nF9PHgvtWkn9PrzZc6TnO2t+V2m6bWnd+a+0p6eP1vS597Lb/S3J2+uzwj01yx9bad1c6yFBhefv0\ncPfjSb6b/ty/lT7W5T1aa781NL80S5dpruSR6VWNX0l/Db6Z5F/Tq5qmn9eqhkvPb57kxPRZvb+a\n/j6cnx4EvS7Jz7XWbtVaW2uculle4+VtN7LNTlprF7bWTkgPPE5O8vn0531Rkq+nTyLzm0mOaq2t\nN3P8rJ+RjfTvg621O6cHWX+Y5D/Sg7zzhz6ekf55fV6S27fW7jxDwLlmP1trH0n/0uKx6RVuXx+O\ndV56ZeQrk/x0a+0n1xlfciOvx1rtfjb9cuYPpp/jl6Q/7zcmuWdr7YGttUuGyWbuk17RfG56QP/u\n9HNjlj6t+nlqrV2Sfun3PyX5dvpn+7Qk71yj329KD8cn593Ja7TdqNdNLZ+wwvr7p1eoLyb5Qmtt\nrX6uavgsfXzYzyFZZciAFWzkvX/kVPv1Ji5by19MHfPE1caUba19orX2U+lVl09P8t703//npVdw\nn5n+O/hPktyltXabDQz7MD3m8OvWaggAW2FhcXFT/28KAABcwVTVweljuB6c5IOttTtt4r6vlv6l\nxEFJvpPk8DlUWLIBVXWv9OrNxSRvaq39zB7uEgCo5AQAAHbbw9Mvm0/6pFubZpiY52XD3UOz+rAW\nbJ3p8Uqfu8d6AQBT5lLJOUyGsKH/fLTWdgpchxk/H5c+1stRSfZN/3b4PUme11r77Gb0FQAA2HVV\nddUkn02fjOv0JDcYLqffzGNcM334gKukDw9xY9Wce0ZV3S19iI/JZHHHr70FAGyNeVVyfjl9AoP1\nfk4f2u80A2RV3SL9P0pPTHKL9PG6TksfIP/h6bPK/vyc+g4AAMxgmC381ekB52KSp252wJkkrbVv\nZWnCtcOTPGGzj8H6qmp7kucPdy9M8ltrNAeALbXHxuSsqoX0Qa5vk+RZrbWnDo8flB5wHpYehD5k\nUrU5fEv8wvQq0UuS3Kq19uk90H0AALhCqqo7pF+afnSSX0uf0X0xybtaa/eY87HfkT4p0sVJbtda\n++Q8j8fOqupp6WHzYpJfba39xR7uEgB8354ck/M30mf5+0KSZy17/PD0bwaPnb4sfZhZ9eFJPppk\nnyTP2bJ/o1lJAAAgAElEQVTeAgAASZ/5/l+SvCBLAefHkzxwC479kPSrwfZN8pqqusoWHJMkVXWX\nJE9Of7//TsAJwOXNHgk5q+qIJM9I/wfyUcvG0zlxePyU1to3lm/bWtuRXs25kOQeVXWdLegyAADQ\nnZ3+//WL0q/AekqSO7bWzpr3gVtrZyS5d5Jzk9w4PehcmPdxr+iq6sgkr0+yPcn7k5y0J/sDACvZ\nZw8d9y+SHJgeZL5r8mBVHZ7kRun/aXrHGttP1m1LctckfzenfgIAAFNaa/fbw8c/NcnV9mQfrmha\na6cludae7gcArGXLKzmr6rgk90i/HP3xy1bffGr5M6vto7V2ZpJzhru33NQOAgAAAACjsqUhZ1Vt\nS/Ls9ErN569wOfrhU8unZ22T9Yev2QoAAAAA2KttdSXng5LcJMl5SZ67wvorTy1fsM6+JuuvvGYr\nAAAAAGCvttUh5xPSqzhf0lr77grrD5gstNYuXmdfF6VPPnTg5nUPAAAAABibLQs5q+rHk9wsPeT8\ns1WaXTDVft91drn/sK/1Kj4BAAAAgL3YVs6u/ovD7ftaa19Zpc15U8sHJVmrmvPg4XalitDNsDin\n/QIAAADAFcHCVh1oS0LOqjogyfHpweHr1mj65anlw5OcvUbbI4bbL+5e71a3uLiYHTtknbA32bZt\nIQsLC85v2As5v2Hv5fyGvZfzG/Zek/N7q2xVJefd08fbXEzy9jXafXJq+WZJTl2pUVUdlT4W52KS\nj21SHy9jx47FnHXW+fPaPbAHHHroQdm+fcH5DXsh5zfsvZzfsPdyfsPea3J+b5WtGpPzHsPtN1tr\n/7Nao9baGUk+Ndw9Zo39TdZdmOTdu989AAAAAGCstirkvNVw++kZ2r4s/Xr9+1fVkctXVtX+SR6b\n4dL31tp5y9sAAAAAAFccWxVy/nB6KNlmaPuSJJ9Lsl+St1TVzScrquq6SV6f5AfTJxx68uZ3FQAA\nAAAYk7mPyTlUXl4lPeT8znrtW2uXVNWxSd6Z5Ogkn6iqryS5KMkN04PZc5Mc11r7+tw6DgAAAACM\nwlZUch6SHnAuJvnfWTZorX0xvfrzqUk+muSqSQ5L8vkkz09ydGvtfXPpLQAAAAAwKnOv5BwmE9q+\nC9udn+RZww8AAAAAwIq2akxOAAAAAIC5EHICAAAAAKMm5AQAAAAARk3ICQAAAACMmpATAAAAABg1\nIScAAAAAMGpCTgAAAABg1IScAAAAAMCoCTkBAAAAgFETcgIAAAAAoybkBAAAAABGTcgJAAAAAIya\nkBMAAAAAGDUhJwAAAAAwakJOAAAAAGDUhJwAAAAAwKgJOQEAAACAURNyAgAAAACjJuQEAAAAAEZN\nyAkAAAAAjJqQEwAAAAAYNSEnAAAAADBqQk4AAAAAYNSEnAAAAADAqAk5AQAAAIBRE3ICAAAAAKMm\n5AQAAAAARk3ICQAAAACMmpATAAAAABg1IScAAAAAMGpCTgAAAABg1IScAAAAAMCoCTkBAAAAgFET\ncgIAAAAAoybkBAAAAABGTcgJAAAAAIyakBMAAAAAGDUhJwAAAAAwakJOAAAAAGDUhJwAAAAAwKgJ\nOQEAAACAURNyAgAAAACjJuQEAAAAAEZNyAkAAAAAjJqQEwAAAAAYNSEnAAAAADBqQk4AAAAAYNSE\nnAAAAADAqAk5AQAAAIBRE3ICAAAAAKMm5AQAAAAARk3ICQAAAACMmpATAAAAABg1IScAAAAAMGpC\nTgAAAABg1IScAAAAAMCoCTkBAAAAgFETcgIAAAAAoybkBAAAAABGTcgJAAAAAIyakBMAAAAAGDUh\nJwAAAAAwakJOAAAAAGDUhJwAAAAAwKgJOQEAAACAURNyAgAAAACjJuQEAAAAAEZNyAkAAAAAjJqQ\nEwAAAAAYNSEnAAAAADBqQk4AAAAAYNSEnAAAAADAqAk5AQBgDaf+13/ls5/73J7uBgAAa9hnT3cA\nAAAuz97/4Y9n+/aFHH2Tm8zU/u/f+KZ85GMfz61v9SM54fjjLrMuyWUeBwBg9wg5AQBgDdu3b8+2\nhdnbn3XuefnOOeflrHPPW3EdAACbz+XqAAAAAMCoCTkBAAAAgFETcgIAAAAAoybkBAAAAABGTcgJ\nAAAAAIyakBMAAAAAGDUhJwAAAAAwakJOAAAAAGDUhJwAAAAAwKgJOQEAAACAURNyAgAAAACjJuQE\nAAAAAEZNyAkAAAAAjJqQEwAAAAAYNSEnAAAAADBqQk4AAAAAYNSEnAAAAADAqAk5AQAAAIBRE3IC\nAAAAAKMm5AQAAAAARk3ICQAAAACMmpATAAAAABg1IScAAAAAMGpCTgAAAABg1IScAAAAAMCoCTkB\nAAAAgFETcgIAAAAAo7bPvA9QVduT/FKShyS5SZIDknw9yXuTvKS19p+rbHf1JI9LckySo5Lsm+Rr\nSd6T5Hmttc/Ou+8AAAAAwOXfXCs5q+pqST6Q5M+S3CHJ+elB5RFJHpbkg1X1qytsd4skn03yxCS3\nSHJ2ktOSHJ7k4Uk+WVU/P8++AwAAAADjMO/L1d+Y5NZJPpbklq21I1prN05yvWHdQpIXVNXRkw2q\n6qAkb05y9SQfT3Kz1tqRrbWjk1wnyd+kV6CeXFU/NOf+AwAAAACXc3MLOavqwUnunH5p+k+11k6d\nrGutnZHkQUnenuSUJNee2vQ30is2L0xy7PRl6a21c9IrOT+aHnQ+Z179BwAAAADGYZ5jcj42yWKS\n57TWzl6+srX2f0nutcJ2Jw7bndJa+8YK2+2oqhcmeWWSe1TVdVpr39zcrgMAAAAAYzGXSs6q+oEk\ntxnuvm4D2x2e5EbD3Xes0XSybluSu264gwAAAADAXmNelZy3Th9v84zW2jeG8PLhSW6f5NAkZyR5\nV5KXt9b+d2q7m08tf2a1nbfWzqyqc5IckuSWSf5uk/sPAAAAAIzEvELOyYRAp1fVcemXlh+8rM19\nkvxuVd23tfafw2OHT60/fZ1jnJ4ech6+TjsAAAAAYC82r4mHrj7cXjPJ3yZ5T/okRIckuUaSRyT5\ndvps6W+uqsnEQ1ee2scF6xxjsv7Ka7YCAAAAAPZq86rknASP10vyltbafZetP7mq/ivJB9KD0Mcn\n+e0kB0watNYuXucYF6VfEn/gpvQYAAAAABileYWci1PLz1qpQWvtI1X1z0mOTXJCesj5/erNqtp3\nnaBz/+E461V87rJt2xZy6KEHzWv3wB6wbdvC92+d37B3cX4zLwccsG+2b8vMn6sDD9g3V7rSPjnw\ngH0vs82BB+ybZPZ90Tm/Ye/l/Ia91+T83irzCjnPm1r+5Brt3pcecl6/qg5ett1BSdYKOSdjfH53\nl3o4g4WFhWzfvrVvCLA1nN+w93J+s9kWtiXbtm3L9u2zjfS0bWEhCwv9dvk22xb6Z3PWfbEz5zfs\nvZzfwO6aV8j51RnbnTW1fHCSL0/dPzzJ2Wtse8Rw+8UN9GtDFhcXs2PH4voNgdHYtm0hCwsLzm/Y\nCzm/mZfFHcmO7Mill+6Yqf2OxcUsLvbb5dvsWOyfzVn3Ref8hr2X8xv2XpPze6vMK+Q8dWr5Bkk+\nt0q7q04tn5Odqz5vtmw/31dVR6WPxbmY5GO73s217dixmLPOOn9euwf2gEMPPSjbty84v2Ev5Pxm\nXi688OJsW8jMn6sLLrw4l1zyvVxw4cWX2eaCC/uFSj6jG+P8hr2X8xv2XpPze6vM6zqZ/8jSZeQP\nWKPd7Ybbz7fW/q+1dkaSTw2PHbPGdpN1FyZ59y73EgAAAAAYvbmEnMOEQa9On/380VV1leVtquoG\nSe6TXo3591OrXjZsd/+qOnKF7fZP8thhu9e11s5b3gYAAAAAuOKY54jnz0gfU/MaSd42XGKeJKmq\nWyb5pyT7JTkzyZ9MbfeS9Mvb90vylqq6+dR2103y+iQ/mF4p+uQ59h8AAAAAGIF5jcmZ1to3q+re\nSd6U5LZJPl9VX0iyb5Lrp1dififJ8a21b01td0lVHZvknUmOTvKJqvpKkouS3DA9mD03yXGtta/P\nq/8AAAAAwDjMs5IzrbUPJrlpkj9K8pkk10ly9SSfHh47urX2oRW2+2KSH07y1CQfTZ+g6LAkn0/y\n/GG7982z7wAAAADAOMytknOitfbtJE8afjay3flJnjX8AAAAAACsaK6VnAAAAAAA8ybkBAAAAABG\nTcgJAAAAAIyakBMAAAAAGDUhJwAAAAAwakJOAAAAAGDUhJwAAAAAwKgJOQEAAACAURNyAgAAAACj\nJuQEAAAAAEZNyAkAAAAAjJqQEwAAAAAYNSEnAAAAADBqQk4AAAAAYNSEnAAAAADAqAk5AQAAAIBR\nE3ICAAAAAKMm5AQAAAAARk3ICQAAAACMmpATAAAAABg1IScAAAAAMGpCTgAAAABg1IScAAAAAMCo\nCTkBAAAAgFETcgIAAAAAoybkBAAAAABGTcgJAAAAAIyakBMAAAAAGDUhJwAAAAAwakJOAAAAAGDU\nhJwAAAAAwKgJOQEAAACAURNyAgAAAACjJuQEAAAAAEZNyAkAAAAAjJqQEwAAAAAYNSEnAAAAADBq\nQk4AAAAAYNSEnAAAAADAqAk5AQAAAIBRE3ICAAAAAKMm5AQAAAAARk3ICQAAAACMmpATAAAAABg1\nIScAAAAAMGpCTgAAAABg1IScAAAAAMCoCTkBAAAAgFETcgIAAAAAoybkBAAAAABGTcgJAAAAAIya\nkBMAAAAAGDUhJwAAAAAwakJOAAAAAGDUhJwAAAAAwKgJOQEAAACAURNyAgAAAACjJuQEAAAAAEZN\nyAkAAAAAjJqQEwAAAAAYNSEnAAAAADBqQk4AAAAAYNSEnAAAAADAqAk5AQAAAIBRE3ICAAAAAKMm\n5AQAAAAARk3ICQAAAACMmpATAAAAABg1IScAAAAAMGpCTgAAAABg1IScAAAAAMCoCTkBAAAAgFET\ncgIAAAAAoybkBAAAAABGTcgJAAAAAIyakBMAAAAAGDUhJwAAAAAwakJOAAAAAGDUhJwAAAAAwKgJ\nOQEAAACAURNyAgAAAACjJuQEAAAAAEZNyAkAAAAAjJqQEwAAAAAYNSEnAAAAADBqQk4AAAAAYNSE\nnAAAAADAqAk5AQAAAIBRE3ICAAAAAKMm5AQAAAAARk3ICQAAAACMmpATAAAAABg1IScAAAAAMGpC\nTgAAAABg1IScAAAAAMCoCTkBAAAAgFETcgIAAAAAoybkBAAAAABGTcgJAAAAAIyakBMAAAAAGDUh\nJwAAAAAwavvMa8dVtWMDzW/ZWvvUsu2vnuRxSY5JclSSfZN8Lcl7kjyvtfbZTeoqAAAAADBicws5\nB4tJvprkrHXaXDj9QFXdIsk7klxjah8XJrlBkocneWhVndhaO2UenQYAAAAAxmPeIWeSPLW19jez\nNq6qg5K8OcnVk3w8yUMmVZtVddUkL0zy0CQnV9WnWmufnkOfAQAAAICRuDyOyfkbSQ5Pr9w8dvqy\n9NbaOemVnB9ND2ifs0d6CAAAAABcblweQ84T0y9RP6W19o3lK1trO9KrOReS3KOqrrPF/QMAAAAA\nLkcuVyFnVR2e5EbD3Xes0XSybluSu861UwAAAADA5dpWjMm5UFXHJLlfkhsn2S/JN9NnSX95a+27\nU21vPrX8mdV22Fo7s6rOSXJIklsm+bvN7jQAAAAAMA5bEXI+N30SocVljx+X5ClV9XOttXcNjx0+\ntf70dfZ7enrIefg67QAAAACAvdhWXK5+QJKnJblJkv2T/ECSxyQ5K8mhSd5YVTcd2l55arsL1tnv\nZP2V12wFAAAAAOzV5lnJ+ZTh9s2ttVOnHj8jyZ9X1QeSfCDJgUmenV7ZecCkUWvt4nX2f1H65EMH\nblqPAQAAAIDRmVvI2Vr7w3XWf6KqXpHkl5Pcq6oOyVT1ZlXtu07QuX/6JfDrVXwCAAAAAHuxrRiT\ncy3vTA85tye5WZLzptYdlGStkPPg4fa7a7TZLdu2LeTQQw+a1+6BPWDbtoXv3zq/Ye/i/GZeDjhg\n32zflpk/VwcesG+udKV9cuAB+15mmwMP2DfJ7Puic37D3sv5DXuvyfm9VfZ0yHn21PLBSb48df/w\nZeuXO2K4/eJmd2piYWEh27dv7RsCbA3nN+y9nN9stoVtybZt27J9+2zD2W9bWMjCQr9dvs22hf7Z\nnHVf7Mz5DXsv5zewu/Z0yHmtqeXvJPnq1P2bJTk1K6iqo9LH4lxM8rF5dW5xcTE7diyfFB4Ys23b\nFrKwsOD8hr2Q85t5WdyR7MiOXHrpjpna71hczOJiv12+zY7F/tmcdV90zm/Yezm/Ye81Ob+3ylxC\nzqp6apK7JflWa+3n1mh61+H2oiSnttYuqqpPJfnhJMckOWWV7Y4Zbi9M8u7d7/HKduxYzFlnnT+v\n3QN7wKGHHpTt2xec37AXcn4zLxdeeHG2LWTmz9UFF16cSy75Xi648OLLbHPBhX00Jp/RjXF+w97L\n+Q17r8n5vVXmdZ3M/ukh5wlVdYeVGlTVkUkenF6N+fettYuGVS9LnzX9/kOb5dvtn+Sxw3ava62d\nt7wNAAAAAHDFMa+Q84VJzhr2//qqus/0yqq6W5J3pU8udG6S35ta/ZIkn0uyX5K3VNXNp7a7bpLX\nJ/nB9AmHnjyn/gMAAAAAIzGXy9Vba2cOweYbklwnyT9W1XeSfDN9HM5rpldifiPJ/Vprp01te0lV\nHZs+8/rRST5RVV9Jv6T9hunB6blJjmutfX0e/QcAAAAAxmNu0zq21j6QHlI+LcmHhmPdeLj9jyRP\nTFKttQ+tsO0X08flfGqSjya5apLDknw+yfOTHN1ae9+8+g4AAAAAjMdcZ1dvrZ2d5JnDz0a3PT/J\ns4YfAAAAAIAVza2SEwAAAABgKwg5AQAAAIBRE3ICAAAAAKMm5AQAAAAARk3ICQAAAACMmpATAAAA\nABg1IScAAAAAMGpCTgAAAABg1IScAAAAAMCoCTkBAAAAgFETcgIAAAAAoybkBAAAAABGTcgJAAAA\nAIyakBMAAAAAGDUhJwAAAAAwakJOAAAAAGDUhJwAAAAAwKgJOQEAAACAURNyAgAAAACjJuQEAAAA\nAEZNyAkAAAAAjJqQEwAAAAAYNSEnAAAAADBqQk4AAAAAYNSEnAAAAADAqAk5AQAAAIBRE3ICAAAA\nAKMm5AQAAAAARk3ICQAAAACMmpATAAAAABg1IScAAAAAMGpCTgAAAABg1IScAAAAAMCoCTkBAAAA\ngFETcgIAAAAAoybkBAAAAABGTcgJAAAAAIyakBMAAAAAGDUhJwAAAAAwakJOAAAAAGDUhJwAAAAA\nwKgJOQEAAACAURNyAgAAAACjJuQEAAAAAEZNyAkAAAAAjJqQEwAAAAAYNSEnAAAAADBqQk4AAAAA\nYNSEnAAAAADAqAk5AQAAAIBRE3ICAAAAAKMm5AQAAAAARk3ICQAAAACMmpATAAAAABg1IScAAAAA\nMGpCTgAAAABg1IScAAAAAMCoCTkB/n/27j3G0vOw6/jvnNmd3dmL197NxZusE1+SPnXr2OYmrmoq\nkEINaYDSQAOUKAVUKhQUKtESRZSCFARBKQUEDaVtLkBTaERpU3pzqya9iF7ixk5o3MdJfL/G9tq7\n693Zmdk5hz/OjDOXM/c5M/vMfD7S6px5z/u+80Q+rzT65nnfBwAAAGiayAkAAAAANE3kBAAAAACa\nJnICAAAAAE0TOQEAAACApomcAAAAAEDTRE4AAAAAoGkiJwAAAADQNJETAAAAAGiayAkAAAAANE3k\nBAAAAACaJnICAAAAAE0TOQEAAACApomcAAAAAEDTRE4AAAAAoGkiJwAAAADQNJETAAAAAGiayAkA\nAAAANE3kBAAAAACaJnICAAAAAE0TOQEAAACApomcAAAAAEDTRE4AAAAAoGkiJwAAAADQNJETAAAA\nAGiayAkAAAAANE3kBAAAAACaJnICAAAAAE0TOQEAAACApomcAAAAAEDTRE4AAAAAoGkiJwAAAADQ\nNJETAAAAAGiayAkAAAAANE3kBAAAAACaJnICAAAAAE0TOQEAAACApomcAAAAAEDTRE4AAAAAoGki\nJwAAAADQNJETAAAAAGiayAkAAAAANE3kBAAAAACaJnICAAAAAE0TOQEAAACAph3YjV9aSjmR5AtJ\nTid5uNZ68wr7nUry3UnuSnJLkvEkTyT5VJIP1lrv35EBAwAAAABXrd2ayfkfMgic/ZV2KKXckeT+\nJO9NckeSF5I8nORMku9Icl8p5R0jHykAAAAAcFXb8chZSnlbkr+V5EqSzgr7HE3yySSnknw2yW21\n1htrrbcmuT7JxzKYhfqRUsrX78jAAQAAAICr0o5GzlLKdUn+cwYzOD+yyq7vyWDG5mSSty68Lb3W\n+mIGMznvySB0fmBU4wUAAAAArn47PZPzPyZ5VZKfSPKbq+z3zgxC6MdrrU8t/bDW2kvygxnMBH1L\nKeX6EYwVAAAAAGjAjkXOUsq3JPm2JM8kefcq+51J8oa5H+9e5ZTzn3WTvHk7xggAAAAAtGdHIufc\nKuk/lMHszO+qtZ5dZffbF7z/wko71Vq/kuTFuR/v3PIgAQAAAIAm7dRMzh9K8sokP15r/ek19j2z\n4P3ja+w7//mZVfcCAAAAAPaskUfOUspfS/KtSZ7MKrepL3B8wftLa+w7//nxVfcCAAAAAPaskUbO\nUsorM1hsqJ/kO+dWRl/LxPybWuv0GvtOZbD40JFNDxIAAAAAaNqoZ3L+UJJTST5Wa/0/6zzm5dmb\npZTxNfY9nEFAXWvGJwAAAACwRx0Y1YlLKe9I8i0ZPDfzH27g0AsL3h9NstpszmNzr+c3Nrr16XY7\nOXny6ChODeySbrfz8qvrG/YW1zejMjExnrFu1v29OjIxnoMHD+TIxPiyY45MDP4/fN/RjXF9w97l\n+oa9a/763ikjiZyllFcl+fdJZpO8q9Z6YchuK/0vfWTB+zNJXljlV71+7vXBDQ9yHTqdTsbGdvY/\nCLAzXN+wd7m+2W6dbtLtdjM2tr6boLqdTjqdwevSY7qdwXdzvediMdc37F2ub2CrRjWT85syuE09\nSe4upay2742llN7c+48kee+Cz25L8vlhB5VSbsngWZz9JL+3lcGupN/vp9frj+LUwC7pdjvpdDqu\nb9iDXN+MSr+X9NLL7Gxv7Z2T9Pr99PuD16XH9PqD7+Z6z8WA6xv2Ltc37F3z1/dOGVXknEmy1iJD\n4xlEyl6+erv5xVrrM6WUzyV5U5K7knx8hePvmnudTPKrWxvucL1eP2fPXhzFqYFdcvLk0YyNdVzf\nsAe5vhmVycnpdDtZ9/fq0uR0Zmau5NLk9LJjLk0OnsTkO7oxrm/Yu1zfsHfNX987ZSSRs9b68awc\nJ5MkpZR3JvlwkkdrrTcv+fhHk/y7JG8vpXxfrfXhJcceTvLuDGZx/uQKt8MDAAAAAPvA1fowoA8l\n+YMkh5L8bCnl9vkPSimnk3wiyRszmAH6vl0ZIQAAAABwVRjZ6upbUWudKaW8NckvJ7k1yb2llEeT\nTCW5OYM4ey7J22qtT+7eSAEAAACA3bbbMzn7c/+WqbU+mMFzOb8vyT1Jrk3y2iQPJPmBJLfWWn99\nh8YJAAAAAFyldm0mZ631o0k+usY+F5O8f+4fAAAAAMAyuz2TEwAAAABgS0ROAAAAAKBpIicAAAAA\n0DSREwAAAABomsgJAAAAADRN5AQAAAAAmiZyAgAAAABNEzkBAAAAgKaJnAAAAABA00ROAAAAAKBp\nIicAAAAA0DSREwAAAABomsgJAAAAADRN5AQAAAAAmiZyAgAAAABNEzkBAAAAgKaJnAAAAABA00RO\nAAAAAKBpIicAAAAA0DSREwAAAABomsgJAAAAADRN5AQAAAAAmiZyAgAAAABNEzkBAAAAgKaJnAAA\nAABA00ROAAAAAKBpIicAAAAA0DSREwAAAABomsgJAAAAADRN5AQAAAAAmiZyAgAAAABNEzkBAAAA\ngKaJnAAAAABA00ROAAAAAKBpIicAAAAA0DSREwAAAABomsgJAAAAADRN5AQAAAAAmiZyAgAAAABN\nEzkBAAAAgKaJnAAAAABA00ROAAAAAKBpIicAAAAA0DSREwAAAABomsgJAAAAADRN5AQAAAAAmiZy\nAgAAAABNEzkBAAAAgKaJnAAAAABA00ROAAAAAKBpIicAAAAA0DSREwAAAABomsgJAAAAADRN5AQA\nAAAAmiZyAgAAAABNEzkBAAAAgKaJnAAAAABA00ROAAAAAKBpIicAAAAA0DSREwAAAABomsgJAAAA\nADRN5AQAAAAAmiZyAgAAAABNEzkBAAAAgKaJnAAAAABA00ROAAAAAKBpIicAAAAA0DSREwAAAABo\nmsgJAAAAADRN5AQAAAAAmiZyAgAAAABNEzkBAAAAgKaJnAAAAABA00ROAAAAAKBpIicAAAAA0DSR\nEwAAAABomsgJAAAAADRN5AQAAAAAmiZyAgAAAABNEzkBAAAAgKaJnAAAAABA00ROAAAAAKBpIicA\nAAAA0DSREwAAAABomsgJAAAAADRN5AQAAAAAmiZyAgAAAABNEzkBAAAAgKaJnAAAAABA00ROAAAA\nAKBpIicAAAAA0DSREwAAAABomsgJAAAAADRN5AQAAAAAmiZyAgAAAABNEzkBAAAAgKaJnAAAAABA\n00ROAAAAAKBpIicAAAAA0DSREwAAAABomsgJAAAAADRN5AQAAAAAmnZglCcvpRxI8reTvD3JnUlO\nJbmc5LEkv5bkQ7XW+1Y49lSS705yV5JbkowneSLJp5J8sNZ6/yjHDgAAAAC0YWQzOUspr01yT5If\nSfKWJMeTfDnJVJKvTfKdSe4ppfzjIcfekeT+JO9NckeSF5I8nORMku9Icl8p5R2jGjsAAAAA0I5R\n3q7+U0nelOR8km9PcrzWemut9ZUZzOr8zNzv/1ellG+YP6iUcjTJJzOY9fnZJLfVWm+std6a5Pok\nHwZ7nrAAACAASURBVMtgBupHSilfP8LxAwAAAAANGEnkLKV8U5I/mqSf5O/UWn+81tqf/7zW+vkk\n35pkZm7T311w+HsymLE5meStC29Lr7W+mMFMznsyCJ0fGMX4AQAAAIB2jGom58Ekn0jyC0n+97Ad\naq2PJqlzP96y4KN3ZhBHP15rfWrIcb0kP5ikk+QtpZTrt3HcAAAAAEBjRrLwUK31kxnccr6Wztzr\nk0lSSjmT5A0ZRM67Vzlu/rNukjcn+R+bGykAAAAA0LpRPpNzVXO3tM8/U/Pn5l5vX7DLF1Y6ttb6\nlSQvzv145/aPDgAAAABoxUhmcq6klDKR5MYkfz3J92YwY/NjtdYPz+1yZsHuj69xuseTnFhyDAAA\nAACwz+xI5Cyl/KUMVlufN53Bszo/XGv9xQXbjy94f2mN085/fnzVvQAAAACAPW2nblc/l+TeJA8k\neSmDhYm+OcnfLaV83YL9Jubf1Fqn1zjnVAbP9DyyvUMFAAAAAFqyIzM5a62fSvKH538upfyxJP8y\nyV9Nclcp5Ztqrb+RBbM3Synja4TOwxnc7r7WjM9N63Y7OXny6KhOD+yCbrfz8qvrG/YW1zejMjEx\nnrFu1v29OjIxnoMHD+TIxPiyY45MjCdZ/7kYcH3D3uX6hr1r/vreKTv6TM55tdbfnVt46NeS/Mkk\nHymlvDHJhQW7Hc3gtvaVHJt7PT+aUSadTidjYzv7HwTYGa5v2Ltc32y3TjfpdrsZG1vfTVDdTied\nzuB16THdzuC7ud5zsZjrG/Yu1zewVbsSOZOk1jpbSvmhDCLnTRmskv7Igl3OJHlhlVO8fu71wdGM\nMOn3++n1+qM6PbALut1OOp2O6xv2INc3o9LvJb30MjvbW9f+vX4//f7gdekxvf7gu7neczHg+oa9\ny/UNe9f89b1TRhI5Syl/OMnrklyotf7KKrs+s+D9q5N8dsHPtyX5/ArnvyWDZ3H2k/ze1ka7sl6v\nn7NnL47q9MAuOHnyaMbGOq5v2INc34zK5OR0up2s+3t1aXI6MzNXcmlyetkxlyYHNyr5jm6M6xv2\nLtc37F3z1/dOGdV9Mh9M8r+S/PAa+92w4P1Xaq3PJPnc3M93rXLc/GeTSX51UyMEAAAAAPaEUUXO\nn5l7vbGU8vZhO5RSOkneNffjCxmsvp4kP5rBqulvL6XcOOS4w0nencEszp+stV5Yug8AAAAAsH+M\nKnL+cJLHMoiVP1JK+fullLH5D0spNyX5qSR/OoNY+a9rrfMPJvpQkj9IcijJz5ZSbl9w3Okkn0jy\nxgwWHHrfiMYPAAAAADRiJM/krLVenFs9/aeT3JLkPyX5QCnlsSSvSPLKuV37SX641vpvFhw7U0p5\na5JfTnJrkntLKY8mmUpycwZh9lySt9VanxzF+AEAAACAdoxqJmdqrfcnuT3JP0hyd5KLGQTPo0m+\nlOS/JfmztdbvGnLsg0nelOT7ktyT5Nokr03yQJIfSHJrrfXXRzV2AAAAAKAdI5nJOa/WejmD288/\ntIljLyZ5/9w/AAAAAIChRjaTEwAAAABgJ4icAAAAAEDTRE4AAAAAoGkiJwAAAADQNJETAAAAAGia\nyAkAAAAANE3kBAAAAACaJnICAAAAAE0TOQEAAACApomcAAAAAEDTRE4AAAAAoGkiJwAAAADQNJET\nAAAAAGiayAkAAAAANE3kBAAAAACaJnICAAAAAE0TOQEAAACApomcAAAAAEDTRE4AAAAAoGkiJwAA\nAADQNJETAAAAAGiayAkAAAAANE3kBAAAAACaJnICAAAAAE0TOQEAAACApomcAAAAAEDTRE4AAAAA\noGkiJwAAAADQNJETAAAAAGiayAkAAAAANE3kBAAAAACaJnICAAAAAE0TOQEAAACApomcAAAAAEDT\nRE4AAAAAoGkiJwAAAADQNJETAAAAAGiayAkAAAAANE3kBAAAAACaJnICAAAAAE0TOQEAAACApomc\nAAAAAEDTRE4AAAAAoGkiJwAAAADQNJETAAAAAGiayAkAAAAANE3kBAAAAACaJnICAAAAAE0TOQEA\nAACApomcAAAAAEDTRE4AAAAAoGkiJwAAAADQNJETAAAAAGiayAkAAAAANE3kBAAAAACaJnICAAAA\nAE0TOQEAAACApomcAAAAAEDTRE4AAAAAoGkiJwAAAADQNJETAAAAAGiayAkAAAAANE3kBAAAAACa\nJnICAAAAAE0TOQEAAACApomcAAAAAEDTRE4AAAAAoGkiJwAAAADQNJETAAAAAGiayAkAAAAANE3k\nBAAAAACaJnICAAAAAE0TOQEAAACApomcAAAAAEDTRE4AAAAAoGkiJwAAAADQNJETAAAAAGiayAkA\nAAAANE3kBAAAAACaJnICAAAAAE0TOQEAAACApomcAAAAAEDTRE4AAAAAoGkiJwAAAADQNJETAAAA\nAGiayAkAAAAANE3kBAAAAACaJnICAAAAAE0TOQEAAACApomcAAAAAEDTRE4AAAAAoGkiJwAAAADQ\nNJETAAAAAGiayAkAAAAANE3kBAAAAACaJnICAAAAAE0TOQEAAACApomcAAAAAEDTRE4AAAAAoGki\nJwAAAADQNJETAAAAAGiayAkAAAAANO3AqH9BKeVbkvztJH8sySuSTCV5KMmvJPmPtdYvr3DcqSTf\nneSuJLckGU/yRJJPJflgrfX+UY8dAAAAALj6jWwmZyllopTyc0k+keSbk5xK8nCSmSS3JXlPks+X\nUr51yLF3JLk/yXuT3JHkhbljzyT5jiT3lVLeMaqxAwAAAADtGOXt6j+W5JuS9JK8L8mJWmuptZ5K\n8g1JvpzkcJKPllJumD+olHI0yScziKKfTXJbrfXGWuutSa5P8rEMZqB+pJTy9SMcPwAAAADQgJFE\nzlLK1yX560n6Sf5lrfVf1Vqn5j+vtf5mkr859+PhJO9acPh7MpixOZnkrQtvS6+1vpjBTM57Mgid\nHxjF+AEAAACAdoxqJucfSfJsktkkPzJsh1rr7yR5ZO7HOxd89M4M4ujHa61PDTmul+QHk3SSvKWU\ncv02jhsAAAAAaMxIImet9b/WWl+d5FCt9dFVdp2Zez2UJKWUM0neMLft7lWOm/+sm+TNWxkrAAAA\nANC2UT6TM7XW/kqfza2efuPcj1+Ye719wS5fyApqrV9J8uLcj3eutB8AAAAAsPeNNHKu4Z9k8FzN\nK0l+eG7bmQWfP77G8fOfn1l1LwAAAABgT9uVyFlK+StJ/lEGz978d7XWL859dHzBbpfWOM3858dX\n3QsAAAAA2NN2PHKWUr49yU9ksHDQz2cwo3PexPybWuv0GqeamjvHke0eIwAAAADQjgM7+ctKKf80\nyT/PYAbnLyZ5+9xq6fMuLdh3fI3QeXjuPGvN+Ny0breTkyePjur0wC7odjsvv7q+YW9xfTMqExPj\nGetm3d+rIxPjOXjwQI5MjC875sjEeJL1n4sB1zfsXa5v2Lvmr++dsiORs5QynuTHkvyNDMLkh5N8\nZ611dsmuFxa8P5pktch5bO71/HaNc6lOp5OxsZ39DwLsDNc37F2ub7Zbp5t0u92Mja3vJqhup5NO\nZ/C69JhuZ/DdXO+5WMz1DXuX6xvYqpFHzlLKNRnclv4nk8wm+Z5a679dYfdHFrw/k+SFVU79+rnX\nB7c8yBX0+/30eisuEA80qNvtpNPpuL5hD3J9Myr9XtJLL7OzvbV3TtLr99PvD16XHtPrD76b6z0X\nA65v2Ltc37B3zV/fO2WkkbOUMpHk5zIInBeSfFut9edXOeS+Be9vS/L5Fc57SwbP4uwn+b3tGe1y\nvV4/Z89eHNXpgV1w8uTRjI11XN+wB7m+GZXJyel0O1n39+rS5HRmZq7k0uT0smMuTQ5uVPId3RjX\nN+xdrm/Yu+av750ysvtkSiljSX4myZ9KcjbJN6wROFNrfSbJ5+Z+vGuVXec/m0zyq1scKgAAAADQ\nsFE+DOj7k/y5JBeT/Pla632r7/6yH81g1fS3l1JuXPphKeVwkndnMIvzJ2utF5buAwAAAADsHyOJ\nnKWUm5J8bwYh8ntrrfds4PAPJfmDJIeS/Gwp5fYF5z2d5BNJ3pjBgkPv27ZBAwAAAABNGtUzOf/h\ngnN/Vynl7611QK31D829zpRS3prkl5PcmuTeUsqjSaaS3JxBmD2X5G211idHMXgAAAAAoB2jipzX\nZjCLM0m+bh37L1pCrdb6YCnlTUnek+QvZzBz80CSBzJYyOiDtdant2+4AAAAAECrRhI5a63vSvKu\nLZ7jYpL3z/0DAAAAABhqlAsPAQAAAACMnMgJAAAAADRN5AQAAAAAmiZyAgAAAABNEzkBAAAAgKaJ\nnAAAAABA00ROAAAAAKBpIicAAAAA0DSREwAAAABomsgJAAAAADRN5AQAAAAAmiZyAgAAAABNEzkB\nAAAAgKaJnAAAAABA00ROAAAAAKBpIicAAAAA0DSREwAAAABomsgJAAAAADRN5AQAAAAAmiZyAgAA\nAABNEzkBAAAAgKaJnAAAAABA00ROAAAAAKBpIicAAAAA0DSREwAAAABomsgJAAAAADRN5AQAAAAA\nmiZyAgAAAABNEzkBAAAAgKaJnAAAAABA00ROAAAAAKBpIicAAAAA0DSREwAAAABomsgJAAAAADRN\n5AQAAAAAmiZyAgAAAABNEzkBAAAAgKaJnAAAAABA00ROAAAAAKBpIicAAAAA0DSREwAAAABomsgJ\nAAAAADRN5AQAAAAAmiZyAgAAAABNEzkBAAAAgKaJnAAAAABA00ROAAAAAKBpIicAAAAA0DSREwAA\nAABomsgJAAAAADRN5AQAAAAAmiZyAgAAAABNEzkBAAAAgKaJnAAAAABA00ROAAAAAKBpIicAAAAA\n0DSREwAAAABomsgJAAAAADRN5AQAAAAAmiZyAgAAAABNEzkBAAAAgKaJnAAAAABA00ROAAAAAKBp\nIicAAAAA0DSREwAAAABomsgJAAAAADRN5AQAAAAAmiZyAgAAAABNEzkBAAAAgKaJnAAAAABA00RO\nAAAAAKBpIicAAAAA0DSREwAAAABomsgJAAAAADRN5AQAAAAAmiZyAgAAAABNEzkBAAAAgKaJnAAA\nAABA00ROAAAAAKBpIicAAAAA0DSREwAAAABomsgJAAAAADRN5AQAAAAAmiZyAgAAAABNEzkBAAAA\ngKaJnAAAAABA00ROAAAAAKBpIicAAAAA0DSREwAAAABomsgJAAAAADRN5AQAAAAAmiZyAgAAAABN\nEzkBAAAAgKaJnAAAAABA00ROAAAAAKBpIicAAAAA0DSREwAAAABomsgJAAAAADRN5AQAAAAAmiZy\nAgAAAABNEzkBAAAAgKaJnAAAAABA0w7sxC8ppRxM8v1JvifJWJLvr7X+izWOOZXku5PcleSWJONJ\nnkjyqSQfrLXeP8IhAwAAAACNGPlMzlLKHUk+k+S9c7+vv85j7p875o4kLyR5OMmZJN+R5L5SyjtG\nNGQAAAAAoCEji5yllG4p5b1JfjvJbUn+Z5LOOo47muSTSU4l+WyS22qtN9Zab01yfZKPZTAD9SOl\nlK8f1fgBAGArev3k0/c+kV+55/G8NDmz28MBANjTRjmT8y8meX+SqSTvqrV+2zqPe08GMzYnk7x1\n4W3ptdYXM5jJeU8GofMD2zpiAADYBv0k/+/5U/noL9T897sfyD/7sd/JxctCJwDAqIwycnaT/FaS\nO2qtH9vAce/M4O/Cj9dan1r6Ya21l+QHM5gV+pZSyvXbMVgAANgu/cOvzsWZ8Zd/fuHCVH7r95/Z\nxREBAOxto4ycv5Hkz9RaH17vAaWUM0neMPfj3avsOv9ZN8mbNzU6AAAYkf74qWXbnnju4i6MBABg\nfxhZ5Ky1Pj8363Ijbl/w/gurnPsrSV6c+/HOjY4NAABGqT9+3bJtz5+7vAsjAQDYH0a+uvoGnVnw\n/vE19p3//MyqewEAwA660uskB69Ztv3seZETAGBUrrbIeXzB+0tr7Dv/+fFV9wIAgB300szBpNNZ\ntv2585fT7+/CgAAA9oGrLXJOzL+ptU6vse9UBosPHRnpiAAAYAPOT48P3T41PZvZ/vL4CQDA1h3Y\n7QEs8fLszVLK+Bqh83AGq7CvNeNz07rdTk6ePDqq0wO7oNvtvPzq+oa9xfXNqExMjGesm3V/ry5e\nObziZ50Dh3Pk0Kzv6Aa5vmHvcn3D3jV/fe+Uqy1yXljw/miS1SLnsbnX86MaTKfTydiY/7cd9iLX\nN+xdrm+2W6ebdLvdjI2tfRPUzJVezk+t/Cf29OxYup3eus7Fcq5v2Ltc38BWXW2R85EF788keWGV\nfV8/9/rgqAbT7/fT63lwEuwl3W4nnU7H9Q17kOubUen3kl56mZ3trbnvlx57Ib1Vbkm/NNNJr99f\n17n4Ktc37F2ub9i75q/vnXK1Rc77Fry/Lcnnh+1USrklg2dx9pP83qgG0+v1c/bsxVGdHtgFJ08e\nzdhYx/UNe5Drm1GZnJxOt5N1fa8+8/tPr/r5S5eTS+PTvqMb5PqGvcv1DXvX/PW9U66q+2Rqrc8k\n+dzcj3etsuv8Z5NJfnWkgwIAgHX64uMvrvr51OzYDo0EAGB/uaoi55wfzWDV9LeXUm5c+mEp5XCS\nd2cwi/Mna60Xlu4DAAA7rd/v54uPn1t1n2mREwBgJEZ2u3op5dokC/+KWzg/9Ugp5dSSQ87WWvtJ\nPpTku5J8bZKfLaX8jVrr5+bOeTrJf0nyxiTnkrxvVOMHAICNePrspbw0ObPqPmZyAgCMxiifyXlv\nktcN2d5J8j1z/xa6McmjtdaZUspbk/xykluT3FtKeTTJVJKbM5h9ei7J22qtT45o7AAAsCFrzeJM\nkuleN9bVAADYfqO8Xb2XwS3l6/m3aHnJWuuDSd6U5PuS3JPk2iSvTfJAkh9Icmut9ddHOHYAANiQ\nBx5b/XmcAx23rAMAjMDIZnLWWm/e4vEXk7x/7h8AAFzVhi06dNPpa/LQU+cXbRM5AQC239W48BAA\nADTlhQtTefbFy4u2HezOprzu2mX7ei4nAMD2EzkBAGCLvvTE8udxXjM+nVPXHF62farnT3AAgO3m\nLywAANiiYc/jPL5C5HS7OgDA9hM5AQBgi4Y9j/Oa8emcvObQsu1uVwcA2H4iJwAAbMHk1JU89pWX\nFm/szeTIgSt5xYkht6uLnAAA207kBACALfjyk+fS7y/e1pl+MZ1OMnHoQA6PL46aU7Nj6S89AACA\nLRE5AQBgCx54bPmiQ53ps4PXTmfZczl7/W4uTV3ZkbEBAOwXIicAAGzBl4Y8jzPTL7z89uSQxYee\nP3d5lEMCANh3RE4AANikK7O9PPjk+UXbOumnM/PV8HlqyOJDZ89PjXxsAAD7icgJAACb9MgzFzJ9\npbdo29GDM+n0v7pt6EzO82ZyAgBsJ5ETAAA26YtDnsd5fHx60c9Ln8mZiJwAANtN5AQAgE364pDn\ncV6zNHKeWB45z4qcAADbSuQEAIBN6Pf7+eLja8/kPDnkmZxmcgIAbC+REwAANuHps5fy0uTMom2n\nTx3JwW5/0bZrjx1Kp7P4WAsPAQBsL5ETAAA2YdgszjeeuXbZtgNj3Vx7bPFszhcvTOXKbG/ZvgAA\nbI7ICQAAm/DAY8ufx/nGMyeG7rt08aF+khcumM0JALBdRE4AANiEYYsOvfGG5TM5k+HP5bT4EADA\n9hE5AQBgg164MJVnX1wcKa89Np5XDllJPRm+wrrFhwAAto/ICQAAG/SlJ4Y/j7OzdIWhOUtvV0+S\n5y0+BACwbUROAADYoI08jzNJTg6JnG5XBwDYPiInAABs0NDncQ5ZWX3e0Jmc50ROAIDtInICAMAG\nTE5dyWNfeWnRtsPjY7nhVcdWPObUkIWHPJMTAGD7iJwAALABX37yXPr9xdve8NoT6XaHP48zSSYO\nHcjh8bFF286en0p/6YkAANgUkRMAADbgi48NW3Ro5edxJkmn01m2wvrUzGwuXr6yrWMDANivRE4A\nANiAjT6Pc96w53JafAgAYHuInAAAsE5XZnt58Mnzi7aNdTu56TXXrHnssBXWPZcTAGB7iJwAALBO\njzxzIdNXeou23Xj98Rw6OLbCEV81dPEhK6wDAGwLkRMAANZp+PM4175VPRk+k/Ps+aktjwkAAJET\nAADWbfjzOFdfdGjesGdyul0dAGB7iJwAALAO/X4/X3x8+UzON2whclp4CABge4icAACwDk+fvZSX\nJmcWbTt96kiOHxlf1/HXHh9P0l+0zUxOAIDtIXICAMA6DJvFud7ncSbJWLeb8e7iRYvOvTSdK7O9\nFY4AAGC9RE4AAFiHBx7b/PM45x0am130cz/J2QsWHwIA2CqREwAA1mHYokNfc8P6Z3ImyfiSyJkk\nZ8+5ZR0AYKtETgAAWMPMbCfPvrg4Rp44Np5XnFi+mNBqls7kTDyXEwBgO4icAACwhgszB5dtu+U1\nJ9LpdDZ0nmGR0wrrAABbJ3ICAMAaXhoSOW86fXzD5xk+k9MzOQEAtkrkBACANVyYXh45bz59zYbP\nM/SZnGZyAgBsmcgJAACr6PeTCzPji7Z1krz++o1HTs/kBAAYDZETAABWMTU7liu9xX82X3/qSI4c\nPrDhcx3o9jPW6S3a9vz5y+n3+1saIwDAfidyAgDAKoY9j3Mzt6rPW3rL+vRMLxcvX9n0+QAAEDkB\nAGBVw1ZWv+k1m4+cQ29ZP+eWdQCArRA5AQBgFS8NWXTopi3M5BwWOS0+BACwNSInAACsYLbXy8Ul\nMzkPjHVyw6uObfqcwyLncyInAMCWiJwAALCCJ569mF46i7bd8KrjOTC2+T+jlz6TMzGTEwBgq0RO\nAABYwUNPnV+2bSuLDiUrPJPz/NSWzgkAsN+JnAAAsIKHnrqwbNtNrzm+pXN6JicAwPYTOQEAYAXD\nZnJuZdGhJBnv9tLtLL4F/nmREwBgS0ROAAAYYmp6Nk88e3HRtolDB/Lqk0e2dN5OJ7nu+Piibede\nms7Mld6WzgsAsJ+JnAAAMMQjz1xIr99ftO2m08eXzcLcjJPXHF627YULZnMCAGyWyAkAAEOM4lb1\neaeGRE6LDwEAbJ7ICQAAQ4xiZfV5w2ZyWnwIAGDzRE4AABjiwSeHzOR8zXbN5Dy0bJvFhwAANk/k\nBACAJc5fms5z5xZHx+uOH8q1x5bHyc04dcJMTgCA7SRyAgDAEg+P8Fb1ZPjt6p7JCQCweSInAAAs\nMexW9RtPH9+28w9deOicmZwAAJslcgIAwBIPPXVh2bbtnMk5cehAJg4dWLTt7PnL6ff72/Y7AAD2\nE5ETAAAW6Pf7Q1ZW7+f1129f5EyWLz40faWXlyZntvV3AADsFyInAAAs8Oy5y8ti48SB2Rw5fGCF\nIzZn2HM5z3ouJwDApoicAACwwLBFh44f3P4ZlsNWWH/eCusAAJsicgIAwALDFh06Pj697b9n6OJD\nIicAwKaInAAAsMDy53Emx0Ywk/PkkmdyJlZYBwDYLJETAADmzPZ6eeTpxSurd9LP0YNXtv13DZvJ\nedZMTgCATRE5AQBgzhPPXsz0ld6ibUcPzqTb2f7fNfx2dQsPAQBshsgJAABzdupW9SQ5cWw83c7i\nemomJwDA5oicAAAwZ1jkHMXK6kky1u3muuOLn8t57uJ0ZpbMJAUAYG0iJwAAzHnwyQvLth0bH03k\nTJJTQxYfOnvBbE4AgI0SOQEAIMnU9GyeeO6lRdsmDh3I4bHZkf3OkyeGLD5khXUAgA0TOQEAIMkj\nz1xIv794202nj6czgkWH5ll8CABge4icAACQ5MEnlz+P86bT14z0d54cEjktPgQAsHEiJwAAZPii\nQzePOHIOeybn8yInAMCGiZwAAJDhkfOm14w6cprJCQCwHUROAAD2vfOXpvPckgV/rjt+KNceWz7T\ncjsNu139Oc/kBADYMJETAIB97+FduFU9GazefuTQgUXbzp6/nP7SFZAAAFiVyAkAwL43dNGhEd+q\nPm/pbM6ZK71cmJzZkd8NALBXiJwAAOx7Dz11Ydm2Ua+sPm/Y4kOeywkAsDEiJwAA+1q/31+26FAn\nyY3XH9+R33/yxPLncj5/znM5AQA2QuQEAGBfe/bc5by05Pbw0684moklz8oclVdYYR0AYMtETgAA\n9rWHhj2P8/TOzOJMhq+w/rzICQCwISInAAD72tJb1ZOdex5nkpwSOQEAtkzkBABgX3twlyPnSQsP\nAQBsmcgJAMC+dWW2l0efXryy+oGxTm541bEdG8O1xw6l2+ks2vb8eQsPAQBshMgJAMC+9eRzFzN9\npbdo2+tefTwHxnbuz+Rut5Prji+ezXn+4nQmp67s2BgAAFoncgIAsG/t9q3q81513cSybb9z/zM7\nPg4AgFaJnAAA7FvDVla/eRci5+23nFq27dP3Prnj4wAAaJXICQDAvjV0ZfXX7Hzk/FO3XZ8DY4uf\ny/nw0xfyyJLnhQIAMJzICQDAvjQ1PZsnnru4aNuRQweG3jo+asePjOePlFct2/7p+8zmBABYD5ET\nAIB96ZFnLqTfX7ztptPHl610vlO+8c7XLNv2W7//dC5PW4AIAGAtIicAAPvS/3vo7LJtu3Gr+ryv\nueHavPrkkUXbLk/P5nfu/8oujQgAoB0iJwAA+86lyzP5lXseX7b95tMndmE0A51OJ2++Y/lszk/f\n+8QujAYAoC0iJwAA+87dn3k8k1OLbwO/5uh4vu7G63ZpRAN/+k3LFyB66CkLEAEArEXkBABgX7l0\neSa/9LuPLdv+F/746zJ+cGwXRvRVKy1A9GsWIAIAWJXICQDAvvJLv/vYslmcJ46O5xv/0Gt3aUSL\nDbtl/f/+/tOZmp7dhdEAALRB5AQAYN+4eHkmd39m+SzOu/7E63d9Fue88rpr8+rrJhZtuzw9m9++\n/5ldGhEAwNVP5AQAYN+4+3cfy+TU4hmRJ46O5xvvXD57crd0Op28+c7ls0o/fe//b+/Oo+QuUCEo\nPQAAGOxJREFU63yPvzsbJJCwJsgeEPkqyj5oWAQFQUbAGRhRREBQBq9njl70Ao7O9crggvcelHuH\nyTDiIJuIDiojHkFQIXAQnAACOi5f1pBgRESQbJCt+/7x/JoUneot6e6qX+X9OienflW/p+r3reY0\nXf3p7/M8TlmXJEnqjyGnJEmSNgj9dXG+o426OHsd1HQDokXM/4MbEEmSJDVjyClJkqQNwq1zm3Rx\nbjqJw9qoi7PXtCmT2G/36Ws9focbEEmSJDVlyClJkqSOt+TFlfz4/np0cfZqNmX9Z25AJEmS1JQh\npyRJkjrerc3W4ty0vdbi7Ou1O23OjD4bEL24fDVz3YBIkiRpLYackiRJ6mhLXlzJj5usxXnMrJ2Z\nOKE9uzihdwOitUNYp6xLkiStzZBTkiRJHe3We+fzUp8p3pu36VqcfR2857aMH/fKDYgeX+gGRJIk\nSX1NaHUBg4mIWcBHgYOAbYCXgARuAC7JzGUtLE+SJEltrHRxPrXW48ccOLOtuzh7TZsyif1jOnN/\n88wrHr/zoYWcclS0qCpJkqT209adnBHxKeBu4D3ADOAJYAlwAHAh8POIaP8/wUuSJKklbpm7dhfn\nFlM34tC9t21RRcN32N5rf9y951dPs3ylGxBJkiT1atuQMyLeCXwO6AEuBqZn5h6ZuSMwC3gM2B24\nvnVVSpIkqV2VHdXX7uJ8R5uvxdlX7LyFGxBJkiQNom1DTuAiSsB5Y2aek5lLe09k5r3AidX5WRFx\nYotqlCRJUpu6Ze58lte8ixNgXFdX027OOx90AyJJkqRebRlyRsTBwG7V3YuajcnMB4Hbqrunj0FZ\nkiRJqonFy1Y07eI85sB6dXH2arYB0WMLF7HgmSUtqkiSJKm9tGXICby1ul0C3DPAuB8BXcCho16R\nJEmSauOWuQuadnG+ea96Luc+bZNJ7Lf79LUet5tTkiSpaNeQc6/q9pHM7B5g3G+r2ykR4faSkiRJ\nYvGyFfykSRfnsQfuzMQJ7frxd3CH7bN2QHu3GxBJkiQB7Rty7kBZb3PtT6ev1Hh+h9ErR5IkSXXx\nw7nz1wr+tpy2EYfUtIuz12t33oIZm/fdgGgV9/32mRZVJEmS1D7aNeScWt0uG2Rc4/mp/Y6SJElS\nx+ru6WH5itUsXraChc8u5bb7f7fWmGMOnFnrLk6oNiBq0s15+wO/47lFL7HkxZUsX7ma7p6eFlQn\nSZLUWhNaXUA/ev9EvWKQccsbjqeMUi1aBz09Pdz50EJ+/vCzvLhiVavLkV5WfsHtAnpYuWqg1TAk\n1Y3f3x2uB1au7mblqm5WrFxdbld1s3LValatHjjU23LaRhyyZ712VO/PwXtuy3fvfJzV3Wve8+ML\nF3HOv9z9inETJ4xj0oRxTJo4/hXH4/psXlQXfn9Lncvvb6meXrXlFI5+405st/UmrS7lZe0acvZ2\naE4aZNzGTZ6jNjDngd9xza0Pt7oMSZIkju2ALs5e0zaZxL67Tx90ivrKVSUQXvqSf2yWJEkj79Gn\nXuAXjz7LF846kCkbt0e82B5VrG1xdTtYHLxpw/GikS5i3LguttyyfRLpOvnlvOdbXYIkSRJbb74x\nxx2223qFnNOmTmKjSZOG/LlwyuRJTJw4gSmT137OlMnlb/jr8xnzuDfv6jqckiSp5RYtW8kfFy9n\n3+02a3p+rGeQdPW04Zo9EfEN4CTggczcf4BxJwLfomxS9OrMnDc2FUqSJEmSJElqF+06b+eh6nb3\niBio23Sv6vbPBpySJEmSJEnShqldQ85bqtspwKEDjDua0sV506hXJEmSJEmSJKkttWXImZkPAg9Q\ntlg7r9mYiHgb0DuV/fIxKk2SJEmSJElSm2nLkLPyMUqX5pERMTsipvaeiIgjgGuq89/NzDmtKVGS\nJEmSJElSq7XlxkO9IuIsYDYljF0BzAM2A15FCTjvAo7JzCWtqlGSJEmSJElSa7V1yAkQEW8APg68\nBdgWWAr8Avg6cEVmtvcbkCRJkiRJkjSq2j7klCRJkiRJkqSBtPOanJIkSZIkSZI0KENOSZIkSZIk\nSbVmyClJkiRJkiSp1gw5JUmSJEmSJNWaIackSZIkSZKkWjPklCRJkiRJklRrhpySJEmSJEmSas2Q\nU5IkSZIkSVKtGXJKkiRJkiRJqjVDTkmSJEmSJEm1ZsgpSZIkSZIkqdYMOSVJkiRJkiTVmiGnJEmS\nJEmSpFqb0OoC6iYi3gvMBjYH5mTm4S0uSdIQRMQs4KPAQcA2wEtAAjcAl2TmshaWJ2k9RcRE4Hzg\nPGA8cH5mXtDSoiStt4g4ATgNOADYGlgOPAH8BJidmY+1sDxJ6ygiJlC+t08E9gG2onw+XwDcCfxr\nZj7UugoljaSI2Az4NbAtMC8zdx2N69jJOUQRsVVEXA9cC2wG9LS4JElDFBGfAu4G3gPMoPxytITy\nC9OFwM8jYrvWVShpfUTE3sB9wCcpn238GS3VXERMjoibgG8Dx1ECkHnASuANwNnALyPiXS0rUtI6\niYjtgfuBfwOOAqYCj1H+iPFa4EPA/RFxbsuKlDTSLqEEnKP6Od2Qcwgi4p3Ar4ATgIeAh4GulhYl\naUiq79/PUf5nejEwPTP3yMwdgVmUD1S7A9e3rkpJ6yIixkXEJ4H/pIQe/44/n6VO8TXgaKAb+Adg\ns8yMzNwKOJTy83tj4KqI2LF1ZUpaBzcAewKLgFOBqZn5usycTunqvI+SVXwxIg5tXZmSRkL1O/kp\nwCpG+bO6IecgImIa8B+U6TFfBt4IPN3SoiQNx0WUgPPGzDwnM5f2nsjMeylTZHqAWRFxYotqlLRu\njgE+T+n8OCMzT2pxPZJGQETsQZl90QN8ITO/mJnLe89n5k+B91V3NwbOGPsqJa2LiDga+AvK9/cH\nM/MbmflyZ1dm/hJ4F6VrG+DMsa9S0kiJiC2Ar1C+568c7esZcg6uC1gIHJWZ52bmysGeIKk9RMTB\nwG7V3YuajcnMB4Hbqrunj0FZkkbOOOBnwN6ZeXWri5E0YvYH/gispkxnXUtmzgWerO7uM0Z1SVp/\nEynLUPyQ0ky0lsycT1k7H+DVY1SXpNExm7Jk3DeBn472xdx4aHBLgT0z8/lWFyJp2N5a3S4B7hlg\n3I+AIyjT3yTVx13A9zOzu9WFSBo5mXkNcE1EdDV2eDXR23yw0RiUJWkEZOb3ge8PYWjvlNaFo1iO\npFFUbR54EmU29Ecoa2yPKjs5B5GZqww4pdraq7p9ZJAQ5LfV7ZSIiFGuSdIIycw/GXBKnWuggDMi\ntgJmVnd/PSYFSRoT1ZT211d3b2plLZLWTfVz+lLKNPUPZ+ZzY3FdOzkldbIdKP9TfWqQcY3nd2DN\n9BhJktSe/p7yu8xK4LIW1yJpPUXEZMofLt4DfILyGf7qzLyilXVJWmeXAtOBazPze2N1UUNOSZ1s\nanW7bJBxjeen9jtKkiS1XEQcD3yMEoL8v8x8pMUlSVpHEfFXlN3We62grNV5RWbe0pqqJK2PiHg3\nZQOxhZRp6mOm40POiLgSOG2YT3ssM18zCuVIGluTq9sVg4xb3nA8ZZRqkSRJ6ykiTqVsRtQF3Ezp\n6JRUXy8AD1I+g28HbEJZt298RCzITJejkGokIqZTNhvqAT6UmX8ey+tvCGty9qzDP9f3kjpDb4fm\npEHGbdzkOZIkqY1ExKeBqyiNGrcAJ7our1RvmTknM/fLzNdm5jRgFnA38DfA3Ig4pLUVShqmS4Gt\nKMtN/GCsL97xnZyZeQZwRqvrkNQSi6vbTQYZt2nD8aJRqkWSJK2DiJgEfA04mdKQcAWlO2R1SwuT\nNOIy895q46E7gQOBKyPiNQNtRCapPUTEe4ETKHtefLQVNWwInZySNlxPUqaz7TDIuJkNx4+PWjWS\nJGlYImIacDsl4FwNnJOZZxpwSp2r+v6+tLq7C7BPC8uRNAQRMQP4J8rP6jMyc3GTYV2jXUfHd3JK\n2qA9BJwE7B4REzJzVT/j9qpu/5yZ88akMkmSNKBqt+WbKN1ci4GTMvPm1lYlaX1ExH7ATsDizPzJ\nAEP/0HC8zehWJWkEHE2Zpg7wo4gYaOzMiOhdbubKzPzASBVhyCmpk90CXEhZyPxQ4LZ+xh1Nmf52\n0xjVJUmSBhAR44EbgYOA54AjMvOh1lYlaQR8CTgMeAJ49QDjdmw4fmZUK5I0ElYCg20yNInyu3k3\na5aJWzqSRRhySupYmflgRDwA7AucR5OQMyLeBuxPCTkvH9sKJUlSP84HjqD88vN2A06pY9xICTln\nRsSJmXl93wER0cWafTWep+y+LqmNZeZ1wHUDjYmI91PW1Z6fmbuORh2GnIOIiE145c7LXcBESiAy\nMSK26vOUxZm5YqzqkzSoj1HW8joyImYDf9+7PkhEHAFcQ/l+/m5mzmlZlZKGLSI2B8Y3PNS4zs+U\nJj+jn3PjAqn9RcQuwCcoP58/kZn3t7gkSSPnMuBsSqfmv1U/q7/au85u9f1/MXAw5f8B/zszu/t7\nMUlq1NXT42f9gUTElcBpw3jK6Zl59SiVI2kdRMRZwGzKZmsrgHnAZsCrKB+e7gKOycwlrapR0vBF\nxDzKul5DNTMz549ONZJGSkRcDPz36u6vgP7W1H5ZZu47qkVJGjER8Trge5Tp6l3AEmABsDUwvRrW\nA1yWmR9uSZGSRlxDJ+c8Ozlbp6f6N9SxktpMZl4WEXcDHwfeQtlNfSkwB/g6cIXdXVItdePPaKkT\nbc6a79k9hjDe72+pRjLzNxGxF3A6cDxlE9BXU/6g8SjwM+DyzLyjZUVKGi3DydiGzU5OSZIkSZIk\nSbU2rtUFSJIkSZIkSdL6MOSUJEmSJEmSVGuGnJIkSZIkSZJqzZBTkiRJkiRJUq0ZckqSJEmSJEmq\nNUNOSZIkSZIkSbVmyClJkiRJkiSp1gw5JUmSJEmSJNWaIackSZIkSZKkWjPklCRJkiRJklRrhpyS\nJEmSJEmSas2QU5IkSZIkSVKtGXJKkiRJkiRJqjVDTkmSJEmSJEm1ZsgpSZIkSZIkqdYMOSVJkiRJ\nkiTVmiGnJEmSJEmSpFoz5JQkSZIkSZJUa4ackiRJkiRJkmptQqsLkCRJUj1ExM7AE9Xd8zPzglbW\nM1oi4jPAZ6q7MzNzfsO5ecBOwJzMPHzsq5MkSVIzhpySJEkbkIg4HDgWOBTYFtgaWAE8C/wa+DFw\nXWY+PcDL9Ix2nW2i2fvsrh7fUL4GkiRJtWDIKUmStAGIiIOAi4EDqocaQ7oJlO7EnYC/BL4QEf8E\nfDozV4xpoe1vD8qST6v7noiIScAngXmZedVYFyZJkrQhc01OSZKkDhcR7wdupwScPcAPgdMogd1W\nwPbA24B/BpYBk4BzgVsjYkoram5XmflSZi7LzOVNTu9Lmeb+/jEuS5IkaYNnJ6ckSVIHi4ijgK9V\ndxcB783Mm5sMfRq4PSIuBr4H7Am8Gfgq8L6xqLUDzGp1AZIkSRsqOzklSZI6VERsCvROm14N/GU/\nAefLMnMecBTwR6ALOCkiDhnNOjuIIackSVKL2MkpSZLUuf4W2IYyRf3izLxnKE/KzD9ExAXAB4Br\ngUeGc9GIOJbS/TkLmFFd/w/A3cA1mXlrP887H/hf1fgtM/OFiHg38N8oU8E3Af5Uvc5FA72fiNgI\n+CDwV8BewBaUoPf31fMvHerXo8/rzqPP7uoRMYeykVOvt0REd3V8B2Uzp89W90/NzGsHucaXgI9V\ndw/PzDnDrVOSJGlDY8gpSZLUuc6qbldTNh0assycDcweznOq9Tuvp2xeBK/c3GgmsAvwvoi4gTJt\nvu+mRo3jJ0bE14DT+zw+AzgeeGdEnJiZ/9Gkjp2BHwG79X3NqoZdqzo+k5mfG857pPnO6o2PdfU5\n3w1cDVxQ3T+ZEhwP5K+r11hgwClJkjQ0TleXJEnqQBGxDRCUsOzOzHx6DC77TUrA2QPcSpn2vg0l\nmDwS+El17q+Bywd5rY9TukE/D7ye0okZwJeq8+OAf4mIZp9nv82agPMqyoZLM4DdgVMo6492Af8Y\nEUeuw/vs62hgKrCguuZdwKbVY+/IzAXAnOqab4uIrft7oYjYhxLEAlw3ArVJkiRtEOzklCRJ6kxv\najieO9oXq6aoH0sJ+W4GjsvMxo7G2yLiTsou7wcDJ0fE7Mz8WZOX6wLOA07JzG82PL4IOC8iXgcc\nQwlQD6KEir11vAXYv6rjB5n5gYbn/wl4LCIWVnUAfITS9bnOendaj4je97s6M1/sM+wq4K2Uz9/v\nof8u2RMajgfr+JQkSVLFTk5JkqTO9KqG48fH4HpnNhyf2yfgBCAzV1HW3Ox1Wj+v1QPc1SfgbHRj\nw/Hr+5wbRwkHb6PsDL+WzLyDsjZnF/DGfq4x0r4DLK2OB9qtvjfk/GVm/tfoliRJktQ57OSUJEnq\nTFs0HP95DK53GCWcfDQzfzPAuDuAZcBk4M0DjLthgHMLGo4b3yeZeRsl4BzMk8C2wFZDGLveMnNp\nRHyHEuy+KSJ2ycwnGsdExGuAPShfx2vGoi5JkqROYSenJElSZ+puOB4/mheKiJ2Azaq7A3YfZmY3\nkJQuyuhnTU0YeEf3xqngk4ZaZx/Lq9ux/Dx8VcPxyU3O93ZxduN6nJIkScNiJ6ckSVJneq7heMtR\nvlbjRjp/HML4P1W344FpNO80XbI+BVVrc54C7EdZu3M6Lf7sm5m3R8R8YCfWbKrU6ARKF+cdmblw\nrOuTJEmqMzs5JUmSOtP8huO+61aOtMkNxy8NYXzjmCkjWUhETIiI6yhT1j8A7E1Zn3Q8JUBs/NcK\nvdPQIyL2Zc2d7YG/6DNGkiRJQ2TIKUmS1Jn+kzVB3kBrX46EpQ3HQwktG0PR9erYbOJCyu7lPcAz\nwPmUzYV2oHSNTq3+/XSErztUVzccN05ZP54yhf8lyiZFkiRJGgZDTkmSpA6UmYuAeynB2RsiYp/h\nvkZE/F1E7DiEoY1T1GcMYXzv9PaVVZ0jIiImAx+iBJxLgVmZ+dnMvD8zf5+ZSzNzWWYuY5TXKe1P\nZj4C3EP57/KuhlPvotR9Y2aOdPArSZLU8Qw5JUmSOte/NhyfP5wnRsQ7gEuAhyPiwwONzczfsWad\nzb0Hed0JQFACvQE3KVoHuwGbVsc3Z+a8fmoYD7xuhK89HL0bEO0UEftFxDbAIdVj17aoJkmSpFoz\n5JQkSepc1wEPU7oGj4uIDw7lSRGxK/CV6u4ESkfoYH5cXWenQbpG3w5sXB3fNpR6hmGThuPnBxh3\nGrD5CF+7Udcg57/Fmt3d/4bSxTmOslnUzaNYlyRJUscy5JQkSepQmbkcOBVYRQnevhIRZ0dEvyFc\nROxBCR+3p3Rbnp+Z9w3hco1doxc2u0ZEbARcUN1dDXx1SG9k6J5qOD6g2YCIeCPwZWBhw2NbNxu7\nDpZRvs7bDTQoM18AvleNPZ4109a/lZmrR6gWSZKkDcqEVhcgSZKk0ZOZ90bEO4HrKZsCfRk4NSIu\nB+4Efk/5TPh64N3A6cAkSsD55cz8/BCvc0dEfIOymc7bgZsj4guUKekTKTuHfxrYt3rt/1OtTzli\nMvOpiHgQ2AfYJyIuo7zfZ4BdgFOADwPfBp4EPlk99eyIuBh4sVqvc109SZkGv1tE/AMlLJ4B/CAz\nV/UZexXl6x3A7pSviVPVJUmS1pGdnJIkSR0uM38IHAjMoYRp+wD/DPyCsmnQ7ynTzc+iBJILgHdn\n5rnDvNQHKTuD9wBHVtd7tnr971O6K3uA/5uZ/3N93tMA/o7SUQlwJvDrqoZ7gY+y5n3+oOE5n6J8\nHc4c4jX664S9puH4s5Qd3L9L8x3nbwGero7HAU9k5j1DvL4kSZL6MOSUJEnaAGTmf2XmEcChwJeA\n+yhTtlcAi4FHKGt4vhfYLTO/089L9QxwjeWZeSJwDPDvlM7Gl4AllLVBvwockJn/Y4BSewa6xmB1\nVEHhrOr6T1Om6r9ACVxPBo6rdli/Gzi7qnE55f3nEOtoei4zrwPOAR6jfF2fBe6ojvuO7aZ0bnZV\nr/X1gd6sJEmSBtbV0zOUz5CSJEmSRlJE/C1lg6ceIDLz0RaXJEmSVFt2ckqSJEmtcXp1e6cBpyRJ\n0vox5JQkSZLGWEQcTlkntQe4pMXlSJIk1Z4hpyRJkjSGImI6cFl191fADS0sR5IkqSNMaHUBkiRJ\nUqeLiJnAZOBNlJ3Xtwe6gY9kpovkS5IkrSdDTkmSJGn0XUnZ2b5XD3BuZt7RmnIkSZI6iyGnJEmS\nNPqepwSby4CfA1/MzJtbW5IkSVLn6OrpcXaMJEmSJEmSpPpy4yFJkiRJkiRJtWbIKUmSJEmSJKnW\nDDklSZIkSZIk1ZohpyRJkiRJkqRaM+SUJEmSJEmSVGuGnJIkSZIkSZJqzZBTkiRJkiRJUq0ZckqS\nJEmSJEmqNUNOSZIkSZIkSbVmyClJkiRJkiSp1gw5JUmSJEmSJNWaIackSZIkSZKkWjPklCRJkiRJ\nklRrhpySJEmSJEmSas2QU5IkSZIkSVKtGXJKkiRJkiRJqjVDTkmSJEmSJEm1ZsgpSZIkSZIkqdb+\nP0BW74iavVz4AAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<matplotlib.figure.Figure at 0x7fe65b00f4a8>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "sns.distplot(summary_frame_atac['totalReads'], bins=20, kde=True)\n",
    "plt.title('Distribution of Mapped Reads (ATAC)')\n",
    "plt.xlabel('Mapped Reads')\n",
    "plt.ylabel('Frequency')\n",
    "plt.show()\n",
    "\n",
    "sns.distplot(summary_frame_atac['averageTagsPerPosition'], bins=20)\n",
    "plt.title('Distribution of Clonality (ATAC)')\n",
    "plt.xlabel('Clonality')\n",
    "plt.show()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "filtered_summary_frame_atac = summary_frame_atac.copy()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# filter samples according to threshold for the fraction of uniquely mapped reads\n",
    "readsThreshold = 1000000\n",
    "pbcThreshold=0.0\n",
    "# gcThreshold=0.10\n",
    "clonalityThreshold = 20\n",
    "# spotThreshold = 0.005\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## ATAC-seq Peak Calling"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "findPeaks /home/jenhan/analysis/atac_analysis//tag_directories_atac//C57Bl6_Thiomac_ATAC_KLA-1h-1_SD_16-03-22 -L 0 -C 0 -fdr 0.9 -style factor -size 200 -minDist 200  > /home/jenhan/analysis/atac_analysis/peak_files/C57Bl6_Thiomac_ATAC_KLA-1h-1_SD_16-03-22_peaks.tsv &\n",
      "findPeaks /home/jenhan/analysis/atac_analysis//tag_directories_atac//C57Bl6_Thiomac_ATAC_KLA-1h-2_SD_16-03-22 -L 0 -C 0 -fdr 0.9 -style factor -size 200 -minDist 200  > /home/jenhan/analysis/atac_analysis/peak_files/C57Bl6_Thiomac_ATAC_KLA-1h-2_SD_16-03-22_peaks.tsv &\n",
      "findPeaks /home/jenhan/analysis/atac_analysis//tag_directories_atac//C57Bl6_Thiomac_ATAC_Veh_SD_16-03-22 -L 0 -C 0 -fdr 0.9 -style factor -size 200 -minDist 200  > /home/jenhan/analysis/atac_analysis/peak_files/C57Bl6_Thiomac_ATAC_Veh_SD_16-03-22_peaks.tsv &\n",
      "findPeaks /home/jenhan/analysis/atac_analysis//tag_directories_atac//C57Bl6_Thiomac_ATAC_Veh_SD_16-03-22-and-16-04-11 -L 0 -C 0 -fdr 0.9 -style factor -size 200 -minDist 200  > /home/jenhan/analysis/atac_analysis/peak_files/C57Bl6_Thiomac_ATAC_Veh_SD_16-03-22-and-16-04-11_peaks.tsv &\n"
     ]
    }
   ],
   "source": [
    "### call peaks ###\n",
    "peakDirectory = '/home/jenhan/analysis/atac_analysis/peak_files'\n",
    "# iterate through each individual file\n",
    "if not os.path.exists(peakDirectory):\n",
    "    os.makedirs(peakDirectory)\n",
    "\n",
    "# make peak files with simplified names\n",
    "\n",
    "for sn in filtered_summary_frame_atac['sampleName'].values:\n",
    "    \n",
    "    peakFileName = sn + \"_peaks.tsv\"\n",
    "\n",
    "    print('findPeaks ' + tagDirPath + '/' + sn + \n",
    "                     ' -L 0 -C 0 -fdr 0.9' +\n",
    "                     ' -style factor -size 200 -minDist 200  > ' + \n",
    "                     peakDirectory +'/' + peakFileName + ' &')\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\tFragment Length = 67\n",
      "\tTotal Tags = 12733303.0\n",
      "\tTags per bp = 0.006367\n",
      "\tMax tags per bp set automatically to 1.0\n",
      "\tFinding peaks of size 200, no closer than 200\n",
      "\tFragment Length = 67\n",
      "\tTotal Tags = 6737504.0\n",
      "\tTags per bp = 0.003369\n",
      "\tMax tags per bp set automatically to 1.0\n",
      "\tFinding peaks of size 200, no closer than 200\n",
      "\tFragment Length = 77\n",
      "\tTotal Tags = 9726543.0\n",
      "\tTags per bp = 0.004863\n",
      "\tMax tags per bp set automatically to 1.0\n",
      "\tFinding peaks of size 200, no closer than 200\n",
      "\tFragment Length = 67\n",
      "\tTotal Tags = 5746574.0\n",
      "\tTags per bp = 0.002873\n",
      "\tMax tags per bp set automatically to 1.0\n",
      "\tFinding peaks of size 200, no closer than 200\n",
      "\t\tFinding peaks on chr1 (minCount=-0.3), total tags positions = 317898\n",
      "\t\tFinding peaks on chr1 (minCount=-0.4), total tags positions = 273690\n",
      "\t\tFinding peaks on chr2 (minCount=-0.3), total tags positions = 298232\n",
      "\t\tFinding peaks on chr2 (minCount=-0.4), total tags positions = 267441\n",
      "\t\tFinding peaks on chr1 (minCount=-0.0), total tags positions = 448119\n",
      "\t\tFinding peaks on chr1 (minCount=0.3), total tags positions = 571010\n",
      "\t\tFinding peaks on chr3 (minCount=-0.3), total tags positions = 264569\n",
      "\t\tFinding peaks on chr3 (minCount=-0.4), total tags positions = 232210\n",
      "\t\tFinding peaks on chr4 (minCount=-0.3), total tags positions = 246291\n",
      "\t\tFinding peaks on chr4 (minCount=-0.4), total tags positions = 213947\n",
      "\t\tFinding peaks on chr2 (minCount=-0.0), total tags positions = 467469\n",
      "\t\tFinding peaks on chr2 (minCount=0.3), total tags positions = 595160\n",
      "\t\tFinding peaks on chr5 (minCount=-0.3), total tags positions = 235812\n",
      "\t\tFinding peaks on chr5 (minCount=-0.4), total tags positions = 209463\n",
      "\t\tFinding peaks on chr3 (minCount=-0.0), total tags positions = 361506\n",
      "\t\tFinding peaks on chr6 (minCount=-0.3), total tags positions = 242671\n",
      "\t\tFinding peaks on chr6 (minCount=-0.4), total tags positions = 212432\n",
      "\t\tFinding peaks on chr3 (minCount=0.3), total tags positions = 463249\n",
      "\t\tFinding peaks on chr7 (minCount=-0.3), total tags positions = 228626\n",
      "\t\tFinding peaks on chr4 (minCount=-0.0), total tags positions = 341390\n",
      "\t\tFinding peaks on chr7 (minCount=-0.4), total tags positions = 198963\n",
      "\t\tFinding peaks on chr8 (minCount=-0.3), total tags positions = 214705\n",
      "\t\tFinding peaks on chr8 (minCount=-0.4), total tags positions = 182362\n",
      "\t\tFinding peaks on chr5 (minCount=-0.0), total tags positions = 341525\n",
      "\t\tFinding peaks on chr4 (minCount=0.3), total tags positions = 449620\n",
      "\t\tFinding peaks on chr9 (minCount=-0.3), total tags positions = 204779\n",
      "\t\tFinding peaks on chr9 (minCount=-0.4), total tags positions = 180869\n",
      "\t\tFinding peaks on chr10 (minCount=-0.4), total tags positions = 198523\n",
      "\t\tFinding peaks on chr10 (minCount=-0.3), total tags positions = 224245\n",
      "\t\tFinding peaks on chr6 (minCount=-0.0), total tags positions = 346462\n",
      "\t\tFinding peaks on chr11 (minCount=-0.4), total tags positions = 173868\n",
      "\t\tFinding peaks on chr5 (minCount=0.3), total tags positions = 440550\n",
      "\t\tFinding peaks on chr12 (minCount=-0.4), total tags positions = 163691\n",
      "\t\tFinding peaks on chr11 (minCount=-0.3), total tags positions = 190964\n",
      "\t\tFinding peaks on chr7 (minCount=-0.0), total tags positions = 328758\n",
      "\t\tFinding peaks on chr13 (minCount=-0.4), total tags positions = 178683\n",
      "\t\tFinding peaks on chr12 (minCount=-0.3), total tags positions = 188778\n",
      "\t\tFinding peaks on chr8 (minCount=-0.0), total tags positions = 290410\n",
      "\t\tFinding peaks on chr14 (minCount=-0.4), total tags positions = 154124\n",
      "\t\tFinding peaks on chr6 (minCount=0.3), total tags positions = 442135\n",
      "\t\tFinding peaks on chr13 (minCount=-0.3), total tags positions = 202353\n",
      "\t\tFinding peaks on chr15 (minCount=-0.4), total tags positions = 146670\n",
      "\t\tFinding peaks on chr9 (minCount=-0.0), total tags positions = 293906\n",
      "\t\tFinding peaks on chr14 (minCount=-0.3), total tags positions = 179282\n",
      "\t\tFinding peaks on chr16 (minCount=-0.4), total tags positions = 136713\n",
      "\t\tFinding peaks on chr7 (minCount=0.3), total tags positions = 427613\n",
      "\t\tFinding peaks on chr15 (minCount=-0.3), total tags positions = 165061\n",
      "\t\tFinding peaks on chr17 (minCount=-0.4), total tags positions = 135189\n",
      "\t\tFinding peaks on chr18 (minCount=-0.4), total tags positions = 130726\n",
      "\t\tFinding peaks on chr16 (minCount=-0.3), total tags positions = 154518\n",
      "\t\tFinding peaks on chr19 (minCount=-0.4), total tags positions = 84819\n",
      "\t\tFinding peaks on chr17 (minCount=-0.3), total tags positions = 149643\n",
      "\t\tFinding peaks on chr8 (minCount=0.3), total tags positions = 385037\n",
      "\t\tFinding peaks on chrX (minCount=-0.4), total tags positions = 95846\n",
      "\t\tFinding peaks on chrY (minCount=-0.4), total tags positions = 3126\n",
      "\t\tFinding peaks on chr1_GL456210_random (minCount=-0.4), total tags positions = 36\n",
      "\t\tFinding peaks on chr1_GL456211_random (minCount=-0.4), total tags positions = 55\n",
      "\t\tFinding peaks on chr1_GL456212_random (minCount=-0.4), total tags positions = 63\n",
      "\t\tFinding peaks on chr1_GL456221_random (minCount=-0.4), total tags positions = 41\n",
      "\t\tFinding peaks on chr4_GL456216_random (minCount=-0.4), total tags positions = 277\n",
      "\t\tFinding peaks on chr4_JH584292_random (minCount=-0.4), total tags positions = 7\n",
      "\t\tFinding peaks on chr10 (minCount=-0.0), total tags positions = 345585\n",
      "\t\tFinding peaks on chr4_JH584295_random (minCount=-0.4), total tags positions = 6\n",
      "\t\tFinding peaks on chr5_GL456354_random (minCount=-0.4), total tags positions = 2\n",
      "\t\tFinding peaks on chr5_JH584296_random (minCount=-0.4), total tags positions = 4\n",
      "\t\tFinding peaks on chr5_JH584297_random (minCount=-0.4), total tags positions = 1\n",
      "\t\tFinding peaks on chr5_JH584299_random (minCount=-0.4), total tags positions = 54\n",
      "\t\tFinding peaks on chrUn_GL456239 (minCount=-0.4), total tags positions = 137\n",
      "\t\tFinding peaks on chrUn_GL456359 (minCount=-0.4), total tags positions = 150\n",
      "\t\tFinding peaks on chrUn_GL456360 (minCount=-0.4), total tags positions = 23\n",
      "\t\tFinding peaks on chrUn_GL456366 (minCount=-0.4), total tags positions = 35\n",
      "\t\tFinding peaks on chrUn_GL456367 (minCount=-0.4), total tags positions = 20\n",
      "\t\tFinding peaks on chrUn_GL456368 (minCount=-0.4), total tags positions = 12\n",
      "\t\tFinding peaks on chrUn_GL456370 (minCount=-0.4), total tags positions = 7\n",
      "\t\tFinding peaks on chrUn_GL456372 (minCount=-0.4), total tags positions = 7\n",
      "\t\tFinding peaks on chrUn_GL456378 (minCount=-0.4), total tags positions = 18\n",
      "\t\tFinding peaks on chrUn_GL456379 (minCount=-0.4), total tags positions = 23\n",
      "\t\tFinding peaks on chrUn_GL456381 (minCount=-0.4), total tags positions = 16\n",
      "\t\tFinding peaks on chrUn_GL456382 (minCount=-0.4), total tags positions = 3\n",
      "\t\tFinding peaks on chrUn_GL456383 (minCount=-0.4), total tags positions = 36\n",
      "\t\tFinding peaks on chrUn_GL456385 (minCount=-0.4), total tags positions = 29\n",
      "\t\tFinding peaks on chrUn_GL456387 (minCount=-0.4), total tags positions = 60\n",
      "\t\tFinding peaks on chrUn_GL456389 (minCount=-0.4), total tags positions = 24\n",
      "\t\tFinding peaks on chrUn_GL456390 (minCount=-0.4), total tags positions = 22\n",
      "\t\tFinding peaks on chrUn_GL456392 (minCount=-0.4), total tags positions = 105\n",
      "\t\tFinding peaks on chrUn_GL456393 (minCount=-0.4), total tags positions = 89\n",
      "\t\tFinding peaks on chrUn_GL456394 (minCount=-0.4), total tags positions = 19\n",
      "\t\tFinding peaks on chrUn_GL456396 (minCount=-0.4), total tags positions = 14\n",
      "\t\tFinding peaks on chrUn_JH584304 (minCount=-0.4), total tags positions = 3824\n",
      "\t\tFinding peaks on chrX_GL456233_random (minCount=-0.4), total tags positions = 366\n",
      "\t\tFinding peaks on chr9 (minCount=0.3), total tags positions = 383535\n",
      "\t\tFinding peaks on chr11 (minCount=-0.0), total tags positions = 307992\n",
      "\t\tFinding peaks on chr18 (minCount=-0.3), total tags positions = 150231\n",
      "\t\tFinding peaks on chr19 (minCount=-0.3), total tags positions = 92826\n",
      "\t\tFinding peaks on chr12 (minCount=-0.0), total tags positions = 260044\n",
      "\t\tFinding peaks on chrX (minCount=-0.3), total tags positions = 111072\n",
      "\t\tFinding peaks on chr10 (minCount=0.3), total tags positions = 430963\n",
      "\t\tFinding peaks on chrY (minCount=-0.3), total tags positions = 4178\n",
      "\t\tFinding peaks on chr1_GL456210_random (minCount=-0.3), total tags positions = 63\n",
      "\t\tFinding peaks on chr1_GL456211_random (minCount=-0.3), total tags positions = 116\n",
      "\t\tFinding peaks on chr1_GL456212_random (minCount=-0.3), total tags positions = 77\n",
      "\t\tFinding peaks on chr1_GL456221_random (minCount=-0.3), total tags positions = 47\n",
      "\t\tFinding peaks on chr4_GL456216_random (minCount=-0.3), total tags positions = 260\n",
      "\t\tFinding peaks on chr4_JH584292_random (minCount=-0.3), total tags positions = 20\n",
      "\t\tFinding peaks on chr4_JH584295_random (minCount=-0.3), total tags positions = 7\n",
      "\t\tFinding peaks on chr5_GL456354_random (minCount=-0.3), total tags positions = 9\n",
      "\t\tFinding peaks on chr5_JH584296_random (minCount=-0.3), total tags positions = 2\n",
      "\t\tFinding peaks on chr5_JH584297_random (minCount=-0.3), total tags positions = 4\n",
      "\t\tFinding peaks on chr5_JH584299_random (minCount=-0.3), total tags positions = 51\n",
      "\t\tFinding peaks on chrUn_GL456239 (minCount=-0.3), total tags positions = 195\n",
      "\t\tFinding peaks on chrUn_GL456359 (minCount=-0.3), total tags positions = 172\n",
      "\t\tFinding peaks on chrUn_GL456360 (minCount=-0.3), total tags positions = 18\n",
      "\t\tFinding peaks on chrUn_GL456366 (minCount=-0.3), total tags positions = 29\n",
      "\t\tFinding peaks on chrUn_GL456367 (minCount=-0.3), total tags positions = 15\n",
      "\t\tFinding peaks on chrUn_GL456368 (minCount=-0.3), total tags positions = 13\n",
      "\t\tFinding peaks on chrUn_GL456370 (minCount=-0.3), total tags positions = 15\n",
      "\t\tFinding peaks on chrUn_GL456372 (minCount=-0.3), total tags positions = 12\n",
      "\t\tFinding peaks on chrUn_GL456378 (minCount=-0.3), total tags positions = 39\n",
      "\t\tFinding peaks on chrUn_GL456379 (minCount=-0.3), total tags positions = 35\n",
      "\t\tFinding peaks on chrUn_GL456381 (minCount=-0.3), total tags positions = 33\n",
      "\t\tFinding peaks on chrUn_GL456382 (minCount=-0.3), total tags positions = 6\n",
      "\t\tFinding peaks on chrUn_GL456383 (minCount=-0.3), total tags positions = 35\n",
      "\t\tFinding peaks on chrUn_GL456385 (minCount=-0.3), total tags positions = 35\n",
      "\t\tFinding peaks on chrUn_GL456387 (minCount=-0.3), total tags positions = 68\n",
      "\t\tFinding peaks on chrUn_GL456389 (minCount=-0.3), total tags positions = 46\n",
      "\t\tFinding peaks on chrUn_GL456390 (minCount=-0.3), total tags positions = 27\n",
      "\t\tFinding peaks on chrUn_GL456392 (minCount=-0.3), total tags positions = 142\n",
      "\t\tFinding peaks on chrUn_GL456393 (minCount=-0.3), total tags positions = 141\n",
      "\t\tFinding peaks on chrUn_GL456394 (minCount=-0.3), total tags positions = 23\n",
      "\t\tFinding peaks on chrUn_GL456396 (minCount=-0.3), total tags positions = 21\n",
      "\t\tFinding peaks on chrUn_JH584304 (minCount=-0.3), total tags positions = 5526\n",
      "\t\tFinding peaks on chrX_GL456233_random (minCount=-0.3), total tags positions = 409\n",
      "\t\tFinding peaks on chrY_JH584300_random (minCount=-0.3), total tags positions = 1\n",
      "\t\tFinding peaks on chr13 (minCount=-0.0), total tags positions = 297083\n",
      "\t\tTags Used for cluster (less clonal tags) = 3578940.0 / 5746574.0\n",
      "\tExpected tags per peak = 0.357894 (tbp = 0.001789)\n",
      "\t\tFinding peaks on chr14 (minCount=-0.0), total tags positions = 232400\n",
      "\t\tFinding peaks on chr11 (minCount=0.3), total tags positions = 403758\n",
      "\t\tFinding peaks on chr15 (minCount=-0.0), total tags positions = 239543\n",
      "\t\tThreshold\tPeak Count\tExpected Peak Count\tFDR\tPoisson\n",
      "\t\t6\t60279.000\t85.993\t1.43e-03\t2.15e-06\n",
      "\t\t5\t70658.000\t1454.413\t2.06e-02\t3.64e-05\n",
      "\t\t4\t89084.000\t20572.086\t2.31e-01\t5.14e-04\n",
      "\t\t3\t146390.000\t234240.637\t1.00e+00\t5.86e-03\n",
      "\t\t2\t425930.000\t2025289.555\t1.00e+00\t5.06e-02\n",
      "\t\t1\t1708871.000\t12034112.859\t1.00e+00\t3.01e-01\n",
      "\t\t0\t1708871.000\t39999999.991\t1.00e+00\t1.00e+00\n",
      "\t90.00% FDR Threshold set at 4.0 (poisson pvalue ~ 5.14e-04)\n",
      "\t\tFinding peaks on chr16 (minCount=-0.0), total tags positions = 223528\n",
      "\t\tFinding peaks on chr12 (minCount=0.3), total tags positions = 336436\n",
      "\t\tTags Used for cluster (less clonal tags) = 4074446.0 / 6737504.0\n",
      "\tExpected tags per peak = 0.407445 (tbp = 0.002037)\n",
      "\t89084 peaks passed threshold\n",
      "\t\tFinding peaks on chr17 (minCount=-0.0), total tags positions = 231242\n",
      "\t\tFinding peaks on chr13 (minCount=0.3), total tags positions = 376041\n",
      "\t\tFinding peaks on chr18 (minCount=-0.0), total tags positions = 205176\n",
      "\t\tThreshold\tPeak Count\tExpected Peak Count\tFDR\tPoisson\n",
      "\t\t7\t49374.000\t10.369\t2.10e-04\t2.59e-07\n",
      "\t\t6\t56728.000\t179.486\t3.16e-03\t4.49e-06\n",
      "\t\t5\t67801.000\t2669.894\t3.94e-02\t6.67e-05\n",
      "\t\t4\t91880.000\t33231.210\t3.62e-01\t8.31e-04\n",
      "\t\t3\t180378.000\t333260.379\t1.00e+00\t8.33e-03\n",
      "\t\t2\t579927.000\t2542364.469\t1.00e+00\t6.36e-02\n",
      "\t\t1\t2085044.000\t13386067.463\t1.00e+00\t3.35e-01\n",
      "\t\t0\t2085044.000\t39999999.988\t1.00e+00\t1.00e+00\n",
      "\t90.00% FDR Threshold set at 4.0 (poisson pvalue ~ 8.31e-04)\n",
      "\t\tFinding peaks on chr19 (minCount=-0.0), total tags positions = 145467\n",
      "\t\tFinding peaks on chr14 (minCount=0.3), total tags positions = 310129\n",
      "\t\tFinding peaks on chrX (minCount=-0.0), total tags positions = 140379\n",
      "\t\tFinding peaks on chrY (minCount=-0.0), total tags positions = 3519\n",
      "\t\tFinding peaks on chr1_GL456210_random (minCount=-0.0), total tags positions = 32\n",
      "\t\tFinding peaks on chr1_GL456211_random (minCount=-0.0), total tags positions = 67\n",
      "\t\tFinding peaks on chr1_GL456212_random (minCount=-0.0), total tags positions = 41\n",
      "\t\tFinding peaks on chr1_GL456221_random (minCount=-0.0), total tags positions = 34\n",
      "\t\tFinding peaks on chr4_GL456216_random (minCount=-0.0), total tags positions = 593\n",
      "\t\tFinding peaks on chr4_GL456350_random (minCount=-0.0), total tags positions = 1\n",
      "\t\tFinding peaks on chr4_JH584292_random (minCount=-0.0), total tags positions = 10\n",
      "\t\tFinding peaks on chr4_JH584295_random (minCount=-0.0), total tags positions = 22\n",
      "\t\tFinding peaks on chr5_GL456354_random (minCount=-0.0), total tags positions = 4\n",
      "\t\tFinding peaks on chr5_JH584296_random (minCount=-0.0), total tags positions = 3\n",
      "\t\tFinding peaks on chr5_JH584297_random (minCount=-0.0), total tags positions = 1\n",
      "\t\tFinding peaks on chr5_JH584299_random (minCount=-0.0), total tags positions = 36\n",
      "\t\tFinding peaks on chrUn_GL456239 (minCount=-0.0), total tags positions = 128\n",
      "\t\tFinding peaks on chrUn_GL456359 (minCount=-0.0), total tags positions = 242\n",
      "\t\tFinding peaks on chrUn_GL456360 (minCount=-0.0), total tags positions = 11\n",
      "\t\tFinding peaks on chrUn_GL456366 (minCount=-0.0), total tags positions = 19\n",
      "\t\tFinding peaks on chrUn_GL456367 (minCount=-0.0), total tags positions = 10\n",
      "\t\tFinding peaks on chrUn_GL456368 (minCount=-0.0), total tags positions = 4\n",
      "\t\tFinding peaks on chrUn_GL456370 (minCount=-0.0), total tags positions = 8\n",
      "\t\tFinding peaks on chrUn_GL456372 (minCount=-0.0), total tags positions = 16\n",
      "\t\tFinding peaks on chrUn_GL456378 (minCount=-0.0), total tags positions = 22\n",
      "\t\tFinding peaks on chrUn_GL456379 (minCount=-0.0), total tags positions = 31\n",
      "\t\tFinding peaks on chrUn_GL456381 (minCount=-0.0), total tags positions = 18\n",
      "\t\tFinding peaks on chrUn_GL456382 (minCount=-0.0), total tags positions = 7\n",
      "\t\tFinding peaks on chrUn_GL456383 (minCount=-0.0), total tags positions = 30\n",
      "\t\tFinding peaks on chrUn_GL456385 (minCount=-0.0), total tags positions = 14\n",
      "\t\tFinding peaks on chrUn_GL456387 (minCount=-0.0), total tags positions = 47\n",
      "\t\tFinding peaks on chrUn_GL456389 (minCount=-0.0), total tags positions = 24\n",
      "\t\tFinding peaks on chrUn_GL456390 (minCount=-0.0), total tags positions = 20\n",
      "\t\tFinding peaks on chrUn_GL456392 (minCount=-0.0), total tags positions = 105\n",
      "\t\tFinding peaks on chrUn_GL456393 (minCount=-0.0), total tags positions = 99\n",
      "\t\tFinding peaks on chrUn_GL456394 (minCount=-0.0), total tags positions = 11\n",
      "\t\tFinding peaks on chrUn_GL456396 (minCount=-0.0), total tags positions = 9\n",
      "\t91880 peaks passed threshold\n",
      "\t\tFinding peaks on chrUn_JH584304 (minCount=-0.0), total tags positions = 6445\n",
      "\t\tFinding peaks on chrX_GL456233_random (minCount=-0.0), total tags positions = 563\n",
      "\t\tFinding peaks on chr15 (minCount=0.3), total tags positions = 308908\n",
      "\t\tFinding peaks on chr16 (minCount=0.3), total tags positions = 284081\n",
      "\t\tFinding peaks on chr17 (minCount=0.3), total tags positions = 297788\n",
      "\t\tTags Used for cluster (less clonal tags) = 5860230.0 / 9726543.0\n",
      "\tExpected tags per peak = 0.586023 (tbp = 0.002930)\n",
      "\t\tFinding peaks on chr18 (minCount=0.3), total tags positions = 268009\n",
      "\t\tThreshold\tPeak Count\tExpected Peak Count\tFDR\tPoisson\n",
      "\t\t8\t88179.000\t8.211\t9.31e-05\t2.05e-07\n",
      "\t\t7\t96957.000\t113.050\t1.17e-03\t2.83e-06\n",
      "\t\t6\t108300.000\t1365.351\t1.26e-02\t3.41e-05\n",
      "\t\t5\t123699.000\t14187.045\t1.15e-01\t3.55e-04\n",
      "\t\t4\t149337.000\t123582.873\t8.28e-01\t3.09e-03\n",
      "\t\t3\t214208.000\t870282.806\t1.00e+00\t2.18e-02\n",
      "\t\t2\t488477.000\t4692828.928\t1.00e+00\t1.17e-01\n",
      "\t\t1\t1739524.000\t17738550.016\t1.00e+00\t4.43e-01\n",
      "\t\t0\t1739524.000\t39999999.973\t1.00e+00\t1.00e+00\n",
      "\t90.00% FDR Threshold set at 4.0 (poisson pvalue ~ 3.09e-03)\n",
      "\t\tFinding peaks on chr19 (minCount=0.3), total tags positions = 184777\n",
      "\t149337 peaks passed threshold\n",
      "\t\tFinding peaks on chrX (minCount=0.3), total tags positions = 183136\n",
      "\t\tFinding peaks on chrY (minCount=0.3), total tags positions = 5044\n",
      "\t\tFinding peaks on chr1_GL456210_random (minCount=0.3), total tags positions = 51\n",
      "\t\tFinding peaks on chr1_GL456211_random (minCount=0.3), total tags positions = 119\n",
      "\t\tFinding peaks on chr1_GL456212_random (minCount=0.3), total tags positions = 69\n",
      "\t\tFinding peaks on chr1_GL456221_random (minCount=0.3), total tags positions = 67\n",
      "\t\tFinding peaks on chr4_GL456216_random (minCount=0.3), total tags positions = 697\n",
      "\t\tFinding peaks on chr4_JH584292_random (minCount=0.3), total tags positions = 19\n",
      "\t\tFinding peaks on chr4_JH584295_random (minCount=0.3), total tags positions = 31\n",
      "\t\tFinding peaks on chr5_GL456354_random (minCount=0.3), total tags positions = 16\n",
      "\t\tFinding peaks on chr5_JH584297_random (minCount=0.3), total tags positions = 1\n",
      "\t\tFinding peaks on chr5_JH584299_random (minCount=0.3), total tags positions = 75\n",
      "\t\tFinding peaks on chrUn_GL456239 (minCount=0.3), total tags positions = 204\n",
      "\t\tFinding peaks on chrUn_GL456359 (minCount=0.3), total tags positions = 264\n",
      "\t\tFinding peaks on chrUn_GL456360 (minCount=0.3), total tags positions = 31\n",
      "\t\tFinding peaks on chrUn_GL456366 (minCount=0.3), total tags positions = 42\n",
      "\t\tFinding peaks on chrUn_GL456367 (minCount=0.3), total tags positions = 14\n",
      "\t\tFinding peaks on chrUn_GL456368 (minCount=0.3), total tags positions = 17\n",
      "\t\tFinding peaks on chrUn_GL456370 (minCount=0.3), total tags positions = 13\n",
      "\t\tFinding peaks on chrUn_GL456372 (minCount=0.3), total tags positions = 16\n",
      "\t\tFinding peaks on chrUn_GL456378 (minCount=0.3), total tags positions = 33\n",
      "\t\tFinding peaks on chrUn_GL456379 (minCount=0.3), total tags positions = 27\n",
      "\t\tFinding peaks on chrUn_GL456381 (minCount=0.3), total tags positions = 36\n",
      "\t\tFinding peaks on chrUn_GL456382 (minCount=0.3), total tags positions = 10\n",
      "\t\tFinding peaks on chrUn_GL456383 (minCount=0.3), total tags positions = 28\n",
      "\t\tFinding peaks on chrUn_GL456385 (minCount=0.3), total tags positions = 26\n",
      "\t\tFinding peaks on chrUn_GL456387 (minCount=0.3), total tags positions = 62\n",
      "\t\tFinding peaks on chrUn_GL456389 (minCount=0.3), total tags positions = 34\n",
      "\t\tFinding peaks on chrUn_GL456390 (minCount=0.3), total tags positions = 20\n",
      "\t\tFinding peaks on chrUn_GL456392 (minCount=0.3), total tags positions = 124\n",
      "\t\tFinding peaks on chrUn_GL456393 (minCount=0.3), total tags positions = 115\n",
      "\t\tFinding peaks on chrUn_GL456394 (minCount=0.3), total tags positions = 27\n",
      "\t\tFinding peaks on chrUn_GL456396 (minCount=0.3), total tags positions = 22\n",
      "\t\tFinding peaks on chrUn_JH584304 (minCount=0.3), total tags positions = 6575\n",
      "\t\tFinding peaks on chrX_GL456233_random (minCount=0.3), total tags positions = 624\n",
      "\tTotal Peaks identified = 89084\n",
      "\t\tTags Used for cluster (less clonal tags) = 7556488.0 / 12733303.0\n",
      "\tExpected tags per peak = 0.755649 (tbp = 0.003778)\n",
      "\tCentering peaks of size 200 using a fragment length of 67\n",
      "\t\tThreshold\tPeak Count\tExpected Peak Count\tFDR\tPoisson\n",
      "\t\t8\t89534.000\t54.033\t6.03e-04\t1.35e-06\n",
      "\t\t7\t98954.000\t578.472\t5.85e-03\t1.45e-05\n",
      "\t\t6\t112032.000\t5436.649\t4.85e-02\t1.36e-04\n",
      "\t\t5\t133173.000\t44011.529\t3.30e-01\t1.10e-03\n",
      "\t\t4\t181760.000\t299254.979\t1.00e+00\t7.48e-03\n",
      "\t\t3\t344716.000\t1650377.111\t1.00e+00\t4.13e-02\n",
      "\t\t2\t921703.000\t7014464.924\t1.00e+00\t1.75e-01\n",
      "\t\t1\t2608052.000\t21211768.967\t1.00e+00\t5.30e-01\n",
      "\t\t0\t2608052.000\t39999999.951\t1.00e+00\t1.00e+00\n",
      "\t90.00% FDR Threshold set at 5.0 (poisson pvalue ~ 1.10e-03)\n",
      "\t133173 peaks passed threshold\n",
      "\tTotal Peaks identified = 91880\n",
      "\tCentering peaks of size 200 using a fragment length of 67\n",
      "\tTotal Peaks identified = 149337\n",
      "\tCentering peaks of size 200 using a fragment length of 77\n",
      "\tTotal Peaks identified = 133173\n",
      "\tCentering peaks of size 200 using a fragment length of 67\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "findPeaks /home/jenhan/analysis/atac_analysis//tag_directories_atac//C57Bl6_Thiomac_ATAC_KLA-1h-2_SD_16-03-22 -L 0 -C 0 -fdr 0.9 -style factor -size 200 -minDist 200  > /home/jenhan/analysis/atac_analysis/peak_files/C57Bl6_Thiomac_ATAC_KLA-1h-2_SD_16-03-22_peaks.tsv &\n",
    "findPeaks /home/jenhan/analysis/atac_analysis//tag_directories_atac//C57Bl6_Thiomac_ATAC_KLA-1h-1_SD_16-03-22 -L 0 -C 0 -fdr 0.9 -style factor -size 200 -minDist 200  > /home/jenhan/analysis/atac_analysis/peak_files/C57Bl6_Thiomac_ATAC_KLA-1h-1_SD_16-03-22_peaks.tsv &\n",
    "findPeaks /home/jenhan/analysis/atac_analysis//tag_directories_atac//C57Bl6_Thiomac_ATAC_Veh_SD_16-03-22-and-16-04-11 -L 0 -C 0 -fdr 0.9 -style factor -size 200 -minDist 200  > /home/jenhan/analysis/atac_analysis/peak_files/C57Bl6_Thiomac_ATAC_Veh_SD_16-03-22-and-16-04-11_peaks.tsv &\n",
    "findPeaks /home/jenhan/analysis/atac_analysis//tag_directories_atac//C57Bl6_Thiomac_ATAC_Veh_SD_16-03-22 -L 0 -C 0 -fdr 0.9 -style factor -size 200 -minDist 200  > /home/jenhan/analysis/atac_analysis/peak_files/C57Bl6_Thiomac_ATAC_Veh_SD_16-03-22_peaks.tsv &"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Run IDR"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "python /home/jenhan/code/seq_merge_pipe/run_idr_homerPeaks.py /home/jenhan/analysis/atac_analysis/peak_files/C57Bl6_Thiomac_ATAC_Veh_SD_16-03-22-and-16-04-11_peaks.tsv /home/jenhan/analysis/atac_analysis/peak_files/C57Bl6_Thiomac_ATAC_Veh_SD_16-03-22_peaks.tsv /home/jenhan/analysis/atac_analysis/idr_atac/ &\n",
      "python /home/jenhan/code/seq_merge_pipe/run_idr_homerPeaks.py /home/jenhan/analysis/atac_analysis/peak_files/C57Bl6_Thiomac_ATAC_KLA-1h-2_SD_16-03-22_peaks.tsv /home/jenhan/analysis/atac_analysis/peak_files/C57Bl6_Thiomac_ATAC_KLA-1h-1_SD_16-03-22_peaks.tsv /home/jenhan/analysis/atac_analysis/idr_atac/ &\n"
     ]
    }
   ],
   "source": [
    "! rm ./idr_atac/*\n",
    "idrDirectory = '/home/jenhan/analysis/atac_analysis/idr_atac/'\n",
    "\n",
    "! if [ ! -d $idrDirectory ]; then mkdir $idrDirectory; fi\n",
    "\n",
    "replicateIndices = []\n",
    "for index in set(filtered_summary_frame_atac.index):\n",
    "    factor_treatment_rows = filtered_summary_frame_atac.loc[index]\n",
    "    if len(factor_treatment_rows) > 1:       \n",
    "        replicateIndices.append(index)\n",
    "\n",
    "# homer peaks\n",
    "for ri in replicateIndices:\n",
    "    samples = filtered_summary_frame_atac.ix[ri, 'sampleName'].values\n",
    "    for i in range(len(samples) - 1):\n",
    "        for j in range(i+1, len(samples)):\n",
    "            sample1 = samples[i]\n",
    "            sample2 = samples[j]\n",
    "            peakFile1 = peakDirectory + '/' + sample1 + '_peaks.tsv'\n",
    "            peakFile2 = peakDirectory + '/' + sample2 + '_peaks.tsv'\n",
    "            !echo \"python /home/jenhan/code/seq_merge_pipe/run_idr_homerPeaks.py $peakFile1 $peakFile2 $idrDirectory &\"\n",
    "            "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "python /home/jenhan/code/seq_merge_pipe/run_idr_homerPeaks.py /home/jenhan/analysis/atac_analysis/peak_files/C57Bl6_Thiomac_ATAC_KLA-1h-2_SD_16-03-22_peaks.tsv /home/jenhan/analysis/atac_analysis/peak_files/C57Bl6_Thiomac_ATAC_KLA-1h-1_SD_16-03-22_peaks.tsv /home/jenhan/analysis/atac_analysis/idr_atac/ &\n",
    "python /home/jenhan/code/seq_merge_pipe/run_idr_homerPeaks.py /home/jenhan/analysis/atac_analysis/peak_files/C57Bl6_Thiomac_ATAC_Veh_SD_16-03-22-and-16-04-11_peaks.tsv /home/jenhan/analysis/atac_analysis/peak_files/C57Bl6_Thiomac_ATAC_Veh_SD_16-03-22_peaks.tsv /home/jenhan/analysis/atac_analysis/idr_atac/ &"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "idrDirectory = '/home/jenhan/analysis/atac_analysis/idr_atac/'\n",
    "count_dict = {}\n",
    "for f in os.listdir(idrDirectory):\n",
    "    if 'idr.tsv' in f:\n",
    "        tokens = f.split('_')\n",
    "        strain = tokens[0]\n",
    "        cellType = tokens[1]\n",
    "        treatment = tokens[3].split('-')[0]\n",
    "        condition = '_'.join([strain, cellType, treatment])\n",
    "        with open(idrDirectory + f) as file:\n",
    "            data = file.readlines()\n",
    "        numPeaks = len(data) - 1\n",
    "        if not condition in count_dict:\n",
    "            count_dict[condition] = (f, numPeaks)\n",
    "        else:\n",
    "            if numPeaks > count_dict[condition][1]:\n",
    "                count_dict[condition] = (f, numPeaks)        \n",
    "for key in count_dict:\n",
    "    idrResult = count_dict[key][0]\n",
    "    peakFileName = key+'_peaks.tsv'\n",
    "\n",
    "    !echo cp $idrDirectory/$idrResult ./idr_peak_files/$peakFileName\n",
    "    !cp $idrDirectory/$idrResult ./idr_peak_files/$peakFileName"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Merge peak files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\tMax distance to merge: direct overlap required (-d given)\n",
      "\tMerging peaks... \n",
      "\tComparing ./idr_peak_files/C57Bl6_Thiomac_KLA_peaks.tsv (59388 total) and ./idr_peak_files/C57Bl6_Thiomac_KLA_peaks.tsv (59388 total)\n",
      "\tComparing ./idr_peak_files/C57Bl6_Thiomac_KLA_peaks.tsv (59388 total) and ./idr_peak_files/C57Bl6_Thiomac_Veh_peaks.tsv (34035 total)\n",
      "\tComparing ./idr_peak_files/C57Bl6_Thiomac_Veh_peaks.tsv (34035 total) and ./idr_peak_files/C57Bl6_Thiomac_KLA_peaks.tsv (59388 total)\n",
      "\tComparing ./idr_peak_files/C57Bl6_Thiomac_Veh_peaks.tsv (34035 total) and ./idr_peak_files/C57Bl6_Thiomac_Veh_peaks.tsv (34035 total)\n",
      "\n",
      "Finished reading merged peak file...\n",
      "Integrating scores for C57Bl6_Thiomac_KLA\n",
      "Integrating scores for C57Bl6_Thiomac_Veh\n"
     ]
    }
   ],
   "source": [
    "! mergePeaks -d given -venn venn.txt ./idr_peak_files/* > ./merged_peaks.tsv\n",
    "! makeSummaryFile.py ./merged_peaks.tsv ./group_summary.tsv ./idr_peak_files/*"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# read in peak data data\n",
    "peak_frame = pd.read_csv('./group_summary.tsv' , sep='\\t')\n",
    "peak_frame = peak_frame.fillna('0')\n",
    "for col in peak_frame.columns[5:]:\n",
    "    floatValues = []\n",
    "    for val in peak_frame[col].values.astype(str):\n",
    "        if ',' in val:\n",
    "            maxVal = np.max([float(x) for x in val.split(',')])\n",
    "            floatValues.append(maxVal)\n",
    "        else:\n",
    "            floatValues.append(float(val))\n",
    "    peak_frame[col] = floatValues\n",
    "peak_frame.index = peak_frame['ID'].values\n",
    "\n",
    "# remove peaks in unknown/random chromosomes\n",
    "peak_frame = peak_frame[~peak_frame['chr'].str.contains('random')]\n",
    "peak_frame = peak_frame[~peak_frame['chr'].str.contains('Un')]\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Create background peaks from genomic sequences from each chromosome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "! if [ ! -d ./group_by_chromosome ]; then mkdir ./group_by_chromosome; fi\n",
    "! rm -rf ./group_by_chromosome/*\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "chr19 (964, 7)\n",
      "chr6 (1989, 7)\n",
      "chr7 (1997, 7)\n",
      "chr18 (1183, 7)\n",
      "chr5 (1910, 7)\n",
      "chr4 (1940, 7)\n",
      "chr1 (2480, 7)\n",
      "chr3 (2144, 7)\n",
      "chr2 (2649, 7)\n",
      "chr14 (1300, 7)\n",
      "chrX (820, 7)\n",
      "chr15 (1385, 7)\n",
      "chr17 (1345, 7)\n",
      "chr8 (1692, 7)\n",
      "chr9 (1710, 7)\n",
      "chr16 (1266, 7)\n",
      "chr12 (1478, 7)\n",
      "chr13 (1848, 7)\n",
      "chr11 (1782, 7)\n",
      "chr10 (2084, 7)\n"
     ]
    }
   ],
   "source": [
    "peakSize = 200 # size of artificial background peaks\n",
    "N_threshold = 0.5 # throw away background peaks containing greater than this fraction of N\n",
    "\n",
    "tile_peak_file = open('./group/merged_tile_peaks.tsv','w')\n",
    "\n",
    "tile_peak_file.write('\\t'.join(['#ID', 'chr', 'start', 'end', 'strand\\n']))\n",
    "\n",
    "for f in os.listdir('./mm10_genome/'):\n",
    "    if '.fa' in f and not f in ['chrY.fa', 'chrM.fa'] :\n",
    "        current_chromosome = f.split('.')[0]\n",
    "        \n",
    "        # create seperate peak file containing just this chromosome\n",
    "        if not os.path.isdir('./group_by_chromosome/'+current_chromosome):\n",
    "            os.mkdir('./group_by_chromosome/'+current_chromosome)\n",
    "        current_tile_peak_file = open(\n",
    "            './group_by_chromosome/'+ current_chromosome + '/' + current_chromosome +'_tile_peaks.tsv','w')\n",
    "        current_tile_peak_file.write('\\t'.join(['#ID', 'chr', 'start', 'end', 'strand\\n']))\n",
    "        \n",
    "        # read in fastq file line by line\n",
    "        with open('./mm10_genome/' + f) as chromosome_file:\n",
    "            data = chromosome_file.readlines()\n",
    "        genome_seq = ''\n",
    "        for line in data[1:]:\n",
    "            genome_seq += line.strip().upper()\n",
    "        genome_seq_size = len(genome_seq)\n",
    "        \n",
    "        # create array indicating if position is N\n",
    "        is_N = [True if x == 'N' else False for x in genome_seq]\n",
    "\n",
    "        # create array indicating if position overlaps with a peak\n",
    "        # simultaneously write ATAC-seq peaks\n",
    "        is_peak = [False for x in genome_seq]\n",
    "        current_peak_frame = peak_frame[(peak_frame['chr'] == current_chromosome) &\n",
    "                                        (peak_frame['Factors'].str.contains('Veh'))] # only work with vehicle peaks for now\n",
    "        print(current_chromosome, current_peak_frame.shape)\n",
    "        positions = list(zip(current_peak_frame['ID'].values, current_peak_frame['start'].values, current_peak_frame['end'].values))\n",
    "        for pos in positions:\n",
    "            \n",
    "            ID = pos[0]\n",
    "            start = pos[1] - 1\n",
    "            end = pos[2]\n",
    "            for i in range(start, end):\n",
    "                is_peak[i] = True\n",
    "            tile_peak_file.write('\\t'.join([ID, current_chromosome, str(start), str(end), '+\\n']))  \n",
    "            current_tile_peak_file.write('\\t'.join([ID, current_chromosome, str(start), str(end), '+\\n']))  \n",
    "        \n",
    "        # write background peak files\n",
    "        for i in range(0,genome_seq_size, peakSize):\n",
    "            n_count = np.sum(is_N[i:i+peakSize+1])\n",
    "            if not n_count > peakSize * N_threshold:\n",
    "                if not np.sum(is_peak[i:i+peakSize+1]) > 0:\n",
    "                    tile_peak_file.write('\\t'.join(['tile_'+str(i), current_chromosome, str(i), str(i+peakSize), '+\\n']))        \n",
    "                    current_tile_peak_file.write('\\t'.join([current_chromosome + '_tile_'+str(i), current_chromosome, str(i), str(i+peakSize), '+\\n']))        \n",
    "\n",
    "        current_tile_peak_file.close()\n",
    "tile_peak_file.close()  "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Extract genomic sequences for each set of peak files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 757817 sequences from chr4\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 632682 sequences from chr10\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 816421 sequences from chrX\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 456834 sequences from chr17\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 603627 sequences from chr9\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 779297 sequences from chr3\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 706716 sequences from chr7\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 289762 sequences from chr19\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 583318 sequences from chr13\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 605593 sequences from chr14\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 888267 sequences from chr2\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 625867 sequences from chr8\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 473514 sequences from chr16\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 591421 sequences from chr11\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 737162 sequences from chr5\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 956420 sequences from chr1\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 501482 sequences from chr15\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 582712 sequences from chr12\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 435763 sequences from chr18\n",
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 729181 sequences from chr6\n"
     ]
    }
   ],
   "source": [
    "peakDirectory =  './group_by_chromosome/'\n",
    "for chrom in os.listdir(peakDirectory):\n",
    "    peakPath = peakDirectory + chrom + '/' + chrom + '_tile_peaks.tsv'\n",
    "    seqPath = peakPath.replace('_peaks.tsv','.fa')\n",
    "    !homerTools extract $peakPath /bioinformatics/homer/data/genomes/mm10 -fa > $seqPath\n",
    "\n",
    "    "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Scan Peaks for Motifs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "# create a script to scan for motifs using FIMO\n",
    "! if [ ! -d ./fimo_results/ ]; then mkdir ./fimo_results/; fi\n",
    "! if [ ! -d ./fimo_out/ ]; then mkdir ./fimo_out/; fi\n",
    "\n",
    "\n",
    "pthresh = 0.01\n",
    "motif_dir = '/home/jenhan/analysis/cobinding_motif_analysis/fimo_motifs/'\n",
    "fimo_results_dir = './fimo_results'\n",
    "\n",
    "\n",
    "peakDirectory =  './group_by_chromosome/'\n",
    "for chrom in os.listdir(peakDirectory):\n",
    "    scriptFile = open('scanMotifs_' + chrom + '.sh','w')\n",
    "    for m in os.listdir(motif_dir):\n",
    "        fimo_out_dir = './fimo_out/' + chrom + '_' + m.replace('.fimo','')\n",
    "\n",
    "        if 'fimo' in m:\n",
    "            outPath = fimo_results_dir + '/' +chrom + '_'+ m.replace('.fimo','') +'.txt'\n",
    "            scriptFile.write(\n",
    "                'fimo --text --max-stored-scores 2000000 --output-pthresh ' + \n",
    "                str(pthresh)  + ' ' +\n",
    "                motif_dir + '/' + m + ' ./group_by_chromosome/' + chrom + '/' + chrom + '_tile.fa ' +\n",
    "                '> ' + outPath + ' & \\n')\n",
    "    scriptFile.close()\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "sleeping...\n",
      "./scanMotifs_chr10.sh\n",
      "sleeping...\n",
      "./scanMotifs_chr11.sh\n",
      "sleeping...\n",
      "./scanMotifs_chr12.sh\n",
      "sleeping...\n",
      "./scanMotifs_chr13.sh\n",
      "sleeping...\n",
      "./scanMotifs_chr14.sh\n",
      "sleeping...\n",
      "./scanMotifs_chr15.sh\n",
      "sleeping...\n",
      "./scanMotifs_chr16.sh\n",
      "sleeping...\n",
      "./scanMotifs_chr17.sh\n",
      "sleeping...\n",
      "./scanMotifs_chr18.sh\n",
      "sleeping...\n",
      "./scanMotifs_chr19.sh\n",
      "sleeping...\n",
      "./scanMotifs_chr1.sh\n",
      "sleeping...\n",
      "./scanMotifs_chr2.sh\n",
      "sleeping...\n",
      "./scanMotifs_chr3.sh\n",
      "sleeping...\n",
      "./scanMotifs_chr4.sh\n",
      "sleeping...\n",
      "./scanMotifs_chr5.sh\n",
      "sleeping...\n",
      "./scanMotifs_chr6.sh\n",
      "sleeping...\n",
      "./scanMotifs_chr7.sh\n",
      "sleeping...\n",
      "./scanMotifs_chr8.sh\n",
      "sleeping...\n",
      "./scanMotifs_chr9.sh\n",
      "sleeping...\n",
      "./scanMotifs_chrX.sh\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dmbx1 of width 17.\n",
      "Using motif +spz1 of width 11.\n",
      "Using motif -spz1 of width 11.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux4 of width 11.\n",
      "Using motif -dux4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfcp2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +zfx of width 14.\n",
      "Using motif -zfx of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lef1 of width 15.\n",
      "Using motif -lef1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +znf740 of width 10.\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Using motif -znf740 of width 10.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srf of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -srf of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -sox2 of width 8.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp53 of width 18.\n",
      "Using motif -tp53 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfya of width 18.\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Using motif -nfya of width 18.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif -gfi1b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa11 of width 12.\n",
      "Using motif -hoxa11 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox17 of width 9.\n",
      "Using motif -sox17 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hes of width 12.\n",
      "Using motif -hes of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hsf of width 13.\n",
      "Using motif -hsf of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +insm1 of width 12.\n",
      "Using motif -insm1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srebpf of width 10.\n",
      "Using motif -srebpf of width 10.\n",
      "Using motif +bcl6b of width 17.\n",
      "Using motif -bcl6b of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tbp of width 15.\n",
      "Using motif -tbp of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Using motif +irf2 of width 18.\n",
      "Using motif -irf2 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +cux of width 10.\n",
      "Using motif -cux of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +nr2f1 of width 13.\n",
      "Read 1 motifs.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Using motif +esr2 of width 15.\n",
      "Using motif -esr2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfi of width 6.\n",
      "Using motif -nfi of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +six3 of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif -six3 of width 17.\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +irf1 of width 21.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f1 of width 12.\n",
      "Using motif -irf1 of width 21.\n",
      "Using motif -e2f1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +mef2c of width 15.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +mtf1 of width 14.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -mef2c of width 15.\n",
      "Using motif -mtf1 of width 14.\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +ap-1 of width 7.\n",
      "Read 1 motifs.\n",
      "Using motif -ap-1 of width 7.\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb33 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -zbtb33 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Using motif +rest of width 21.\n",
      "Using motif -rest of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +klf of width 11.\n",
      "Using motif -klf of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nrf1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nrf1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +t of width 16.\n",
      "Using motif -t of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +roravar of width 14.\n",
      "Using motif -roravar of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pbx1 of width 12.\n",
      "Using motif -pbx1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prdm1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -prdm1 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfyb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nfyb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +xbp1 of width 14.\n",
      "Using motif +e2f2 of width 16.\n",
      "Using motif -xbp1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -e2f2 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -bhlha15 of width 8.\n",
      "Using motif +scrt of width 13.\n",
      "Using motif -scrt of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elf of width 12.\n",
      "Using motif -elf of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +rora of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif -rora of width 10.\n",
      "Using motif +ctcf of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -ctcf of width 19.\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rxravdr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -rxravdr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +rreb1 of width 20.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +mafg of width 21.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl1 of width 12.\n",
      "Using motif -mafg of width 21.\n",
      "Using motif -mybl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -rreb1 of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ebox of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -ebox of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf1 of width 13.\n",
      "Using motif -hnf1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +znf143 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +znf263 of width 21.\n",
      "Using motif -znf143 of width 16.\n",
      "Read 1 motifs.\n",
      "Using motif +plag1 of width 14.\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -plag1 of width 14.\n",
      "Using motif -znf263 of width 21.\n",
      "Using motif -arid5a of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2e3 of width 7.\n",
      "Using motif -nr2e3 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mecom of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -mecom of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hinfp of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hinfp of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebf1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -ebf1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf410 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -znf410 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc13 of width 11.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esrr of width 10.\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif -esrr of width 10.\n",
      "Using motif -grhl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif +hnf4a of width 16.\n",
      "Using motif -hnf4a of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +fox of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -fox of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +onecut of width 14.\n",
      "Using motif -onecut of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +sp4 of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif +nfat of width 10.\n",
      "Using motif -sp4 of width 17.\n",
      "Using motif -nfat of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myb of width 10.\n",
      "Using motif -myb of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf21 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -tcf21 of width 14.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cenpb of width 15.\n",
      "Using motif -cenpb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif -bcl6 of width 14.\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Using motif -prox1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gli2 of width 12.\n",
      "Using motif -gli2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +thap1 of width 9.\n",
      "Using motif -thap1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +hoxa9 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gfi1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -gfi1 of width 10.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl2 of width 15.\n",
      "Using motif +pax6 of width 14.\n",
      "Using motif -mybl2 of width 15.\n",
      "Using motif -pax6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +arid3b of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif -arid3b of width 11.\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif -yy2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +myc of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif -myc of width 10.\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Using motif +nr2e1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nr2e1 of width 9.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rargvar of width 17.\n",
      "Using motif -rargvar of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Using motif +e2f8 of width 12.\n",
      "Using motif -e2f8 of width 12.\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou1f1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pou1f1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfkb of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -nfkb of width 13.\n",
      "Read 1 motifs.\n",
      "Using motif +duxa of width 13.\n",
      "Using motif -duxa of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr5a2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nr5a2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +usf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -usf of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zic of width 14.\n",
      "Using motif -zic of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +tp63 of width 18.\n",
      "Using motif +e2f7 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -tp63 of width 18.\n",
      "Using motif -e2f7 of width 14.\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif -sox10 of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +gata of width 9.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -gata of width 9.\n",
      "Using motif +zbed1 of width 13.\n",
      "Using motif -zbed1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +hltf of width 10.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +esr1 of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif -hltf of width 10.\n",
      "Using motif -esr1 of width 17.\n",
      "Using motif +tcf7 of width 12.\n",
      "Using motif +rhox11 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +sp1 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1var of width 10.\n",
      "Using motif -tcf7 of width 12.\n",
      "Using motif -sp1 of width 11.\n",
      "Using motif -mzf1var of width 10.\n",
      "Using motif -rhox11 of width 17.\n",
      "Using motif +znf354c of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -znf354c of width 6.\n",
      "Read 1 motifs.\n",
      "Using motif +cdx of width 11.\n",
      "Using motif +rar_rxr of width 17.\n",
      "Using motif -cdx of width 11.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -tfap1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox8 of width 16.\n",
      "Using motif +pou6f2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -sox8 of width 16.\n",
      "Using motif -pou6f2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +rel of width 10.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -rel of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax5 of width 19.\n",
      "Using motif -pax5 of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb18 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +cebp of width 10.\n",
      "Using motif -cebp of width 10.\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg of width 20.\n",
      "Using motif +tp73 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Using motif -tp73 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pparg of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +phox2_prop of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spi1-c of width 14.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Using motif -spi1-c of width 14.\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Using motif +glis of width 14.\n",
      "Using motif -glis of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Using motif +rfx of width 16.\n",
      "Using motif +nfil3 of width 11.\n",
      "Using motif -nfil3 of width 11.\n",
      "Using motif -rfx of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -znf423 of width 15.\n",
      "Using motif +rar of width 16.\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif -hnf4g of width 15.\n",
      "Using motif -rar of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf1 of width 13.\n",
      "Using motif -hnf1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl1 of width 12.\n",
      "Using motif -mybl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif -gfi1b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cenpb of width 15.\n",
      "Using motif -cenpb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif -prox1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hinfp of width 12.\n",
      "Using motif -hinfp of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Using motif +e2f1 of width 12.\n",
      "Using motif -e2f1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2f1 of width 13.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mecom of width 14.\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb33 of width 15.\n",
      "Using motif -zbtb33 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -mecom of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ctcf of width 19.\n",
      "Using motif -ctcf of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rest of width 21.\n",
      "Using motif -rest of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prdm1 of width 15.\n",
      "Using motif -prdm1 of width 15.\n",
      "Using motif +srebpf of width 10.\n",
      "Using motif -srebpf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +roravar of width 14.\n",
      "Using motif -roravar of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hes of width 12.\n",
      "Using motif -hes of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rxravdr of width 15.\n",
      "Using motif -rxravdr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf1 of width 21.\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Using motif -irf1 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4a of width 16.\n",
      "Using motif +insm1 of width 12.\n",
      "Using motif -hnf4a of width 16.\n",
      "Using motif -insm1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +scrt of width 13.\n",
      "Using motif -scrt of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rreb1 of width 20.\n",
      "Using motif -rreb1 of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -arid5a of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mafg of width 21.\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -mafg of width 21.\n",
      "Using motif +ap-1 of width 7.\n",
      "Using motif -ap-1 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax6 of width 14.\n",
      "Using motif -pax6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gli2 of width 12.\n",
      "Using motif -gli2 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +thap1 of width 9.\n",
      "Using motif -thap1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +znf263 of width 21.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +myb of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif -znf263 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -myb of width 10.\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif -znf410 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfat of width 10.\n",
      "Using motif -nfat of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pbx1 of width 12.\n",
      "Using motif -pbx1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +six3 of width 17.\n",
      "Using motif -six3 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfya of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -nfya of width 18.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbp of width 15.\n",
      "Using motif -tbp of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gfi1 of width 10.\n",
      "Using motif +myc of width 10.\n",
      "Using motif -gfi1 of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid3b of width 11.\n",
      "Using motif -arid3b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f8 of width 12.\n",
      "Using motif -e2f8 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pax2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfkb of width 13.\n",
      "Using motif +hoxa9 of width 11.\n",
      "Using motif -nfkb of width 13.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif -yy2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dux4 of width 11.\n",
      "Using motif -dux4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spz1 of width 11.\n",
      "Using motif -spz1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zfx of width 14.\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Using motif -zfx of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmbx1 of width 17.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lef1 of width 15.\n",
      "Using motif -lef1 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox17 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf740 of width 10.\n",
      "Using motif -znf740 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -sox17 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hsf of width 13.\n",
      "Using motif +hoxa11 of width 12.\n",
      "Using motif -hsf of width 13.\n",
      "Using motif -hoxa11 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +esr2 of width 15.\n",
      "Using motif -esr2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cux of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -cux of width 10.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +klf of width 11.\n",
      "Using motif -klf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Using motif -mtf1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +xbp1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -xbp1 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +nfyb of width 15.\n",
      "Using motif -nfyb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -bhlha15 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elf of width 12.\n",
      "Using motif -elf of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rora of width 10.\n",
      "Using motif -rora of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebox of width 11.\n",
      "Using motif -ebox of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +plag1 of width 14.\n",
      "Using motif -plag1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif -grhl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e3 of width 7.\n",
      "Using motif -nr2e3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc13 of width 11.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sp4 of width 17.\n",
      "Using motif -sp4 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +duxa of width 13.\n",
      "Using motif -duxa of width 13.\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +srf of width 16.\n",
      "Using motif -srf of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bcl6b of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -bcl6b of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +t of width 16.\n",
      "Using motif -t of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +esrr of width 10.\n",
      "Using motif -esrr of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +onecut of width 14.\n",
      "Using motif -onecut of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf143 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -znf143 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf21 of width 14.\n",
      "Using motif -tcf21 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl2 of width 15.\n",
      "Using motif -mybl2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp53 of width 18.\n",
      "Using motif -tp53 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf2 of width 18.\n",
      "Using motif -irf2 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfi of width 6.\n",
      "Using motif -nfi of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nrf1 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f2 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -e2f2 of width 16.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif -bcl6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rargvar of width 17.\n",
      "Using motif -rargvar of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou1f1 of width 14.\n",
      "Using motif -pou1f1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cdx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -cdx of width 11.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +nr5a2 of width 15.\n",
      "Read 1 motifs.\n",
      "Using motif -nr5a2 of width 15.\n",
      "Using motif +e2f7 of width 14.\n",
      "Using motif -e2f7 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rar_rxr of width 17.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ventx of width 9.\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +usf of width 11.\n",
      "Using motif -usf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou6f2 of width 10.\n",
      "Using motif -pou6f2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +zic of width 14.\n",
      "Using motif +rhox11 of width 17.\n",
      "Using motif -zic of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -rhox11 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcf7 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -tcf7 of width 12.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif -sox10 of width 6.\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mzf1var of width 10.\n",
      "Using motif -mzf1var of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rel of width 10.\n",
      "Using motif -rel of width 10.\n",
      "Using motif +esr1 of width 17.\n",
      "Using motif -esr1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbed1 of width 13.\n",
      "Using motif -zbed1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp63 of width 18.\n",
      "Using motif -tp63 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox8 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -sox8 of width 16.\n",
      "Read 1 motifs.\n",
      "Using motif +pax5 of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -pax5 of width 19.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap1 of width 12.\n",
      "Using motif -tfap1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb18 of width 13.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sp1 of width 11.\n",
      "Using motif -sp1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cebp of width 10.\n",
      "Using motif -cebp of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg of width 20.\n",
      "Using motif -pparg of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif +tp73 of width 18.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Using motif -tp73 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +glis of width 14.\n",
      "Using motif -glis of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif -hnf4g of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rfx of width 16.\n",
      "Using motif -rfx of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfil3 of width 11.\n",
      "Using motif -nfil3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +phox2_prop of width 11.\n",
      "Using motif +spi1-c of width 14.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Using motif -spi1-c of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -znf423 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar of width 16.\n",
      "Using motif -rar of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hes of width 12.\n",
      "Using motif -hes of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hinfp of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hinfp of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2f1 of width 13.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rreb1 of width 20.\n",
      "Using motif -rreb1 of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf1 of width 13.\n",
      "Using motif -hnf1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prdm1 of width 15.\n",
      "Using motif -prdm1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfya of width 18.\n",
      "Using motif -nfya of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mafg of width 21.\n",
      "Using motif -mafg of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb33 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -zbtb33 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4a of width 16.\n",
      "Using motif -hnf4a of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif +pbx1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -pbx1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +roravar of width 14.\n",
      "Using motif -roravar of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif -arid5a of width 14.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srebpf of width 10.\n",
      "Using motif -srebpf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +myb of width 10.\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -myb of width 10.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Using motif +mybl1 of width 12.\n",
      "Using motif -mybl1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ctcf of width 19.\n",
      "Using motif -ctcf of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf1 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -irf1 of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +insm1 of width 12.\n",
      "Using motif -insm1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbp of width 15.\n",
      "Using motif -tbp of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +scrt of width 13.\n",
      "Using motif -scrt of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif -znf410 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gli2 of width 12.\n",
      "Using motif -gli2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfat of width 10.\n",
      "Using motif -nfat of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rxravdr of width 15.\n",
      "Using motif -rxravdr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mecom of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -mecom of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +thap1 of width 9.\n",
      "Using motif -thap1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif -prox1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf263 of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -znf263 of width 21.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ap-1 of width 7.\n",
      "Using motif -ap-1 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f1 of width 12.\n",
      "Using motif -e2f1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cenpb of width 15.\n",
      "Using motif -cenpb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +six3 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -six3 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif -gfi1b of width 11.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rest of width 21.\n",
      "Using motif -rest of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax6 of width 14.\n",
      "Using motif -pax6 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myc of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f8 of width 12.\n",
      "Using motif -e2f8 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gfi1 of width 10.\n",
      "Using motif -gfi1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid3b of width 11.\n",
      "Using motif -arid3b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa9 of width 11.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif -yy2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfkb of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nfkb of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux4 of width 11.\n",
      "Using motif -dux4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +spz1 of width 11.\n",
      "Using motif -spz1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zfx of width 14.\n",
      "Using motif -zfx of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dmbx1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lef1 of width 15.\n",
      "Using motif -lef1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf740 of width 10.\n",
      "Using motif -znf740 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox17 of width 9.\n",
      "Using motif -sox17 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa11 of width 12.\n",
      "Using motif -hoxa11 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hsf of width 13.\n",
      "Using motif -hsf of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Using motif +cux of width 10.\n",
      "Using motif -cux of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +esr2 of width 15.\n",
      "Using motif -esr2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif -mtf1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +klf of width 11.\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Using motif -klf of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -bhlha15 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +xbp1 of width 14.\n",
      "Using motif -xbp1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfyb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nfyb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Using motif +elf of width 12.\n",
      "Using motif -elf of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rora of width 10.\n",
      "Using motif -rora of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebox of width 11.\n",
      "Using motif -ebox of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif -grhl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +plag1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -plag1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nr2e3 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxc13 of width 11.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sp4 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -sp4 of width 17.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +duxa of width 13.\n",
      "Using motif -duxa of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +srf of width 16.\n",
      "Using motif -srf of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bcl6b of width 17.\n",
      "Using motif -bcl6b of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +t of width 16.\n",
      "Using motif +znf143 of width 16.\n",
      "Using motif -t of width 16.\n",
      "Using motif -znf143 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +onecut of width 14.\n",
      "Using motif -onecut of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esrr of width 10.\n",
      "Using motif -esrr of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcf21 of width 14.\n",
      "Using motif -tcf21 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mybl2 of width 15.\n",
      "Using motif -mybl2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp53 of width 18.\n",
      "Using motif -tp53 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfi of width 6.\n",
      "Using motif -nfi of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf2 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -irf2 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nrf1 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f2 of width 16.\n",
      "Using motif -e2f2 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Using motif -bcl6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rargvar of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -rargvar of width 17.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Using motif +pou1f1 of width 14.\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Using motif -pou1f1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr5a2 of width 15.\n",
      "Using motif -nr5a2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cdx of width 11.\n",
      "Using motif -cdx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f7 of width 14.\n",
      "Using motif -e2f7 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rar_rxr of width 17.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rhox11 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -rhox11 of width 17.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +usf of width 11.\n",
      "Using motif -usf of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zic of width 14.\n",
      "Using motif -zic of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp63 of width 18.\n",
      "Using motif -tp63 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou6f2 of width 10.\n",
      "Using motif -pou6f2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcf7 of width 12.\n",
      "Using motif -tcf7 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rel of width 10.\n",
      "Using motif -rel of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Using motif +esr1 of width 17.\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Using motif -esr1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mzf1var of width 10.\n",
      "Using motif -mzf1var of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif +zbed1 of width 13.\n",
      "Using motif -sox10 of width 6.\n",
      "Using motif -zbed1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfap1 of width 12.\n",
      "Using motif -tfap1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox8 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sp1 of width 11.\n",
      "Using motif -sp1 of width 11.\n",
      "Using motif -sox8 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb18 of width 13.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Using motif +pax5 of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -pax5 of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pparg of width 20.\n",
      "Using motif -pparg of width 20.\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cebp of width 10.\n",
      "Using motif -cebp of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp73 of width 18.\n",
      "Using motif -tp73 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif -hnf4g of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rfx of width 16.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Using motif -rfx of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +glis of width 14.\n",
      "Using motif -glis of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +spi1-c of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -spi1-c of width 14.\n",
      "Using motif +phox2_prop of width 11.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +nfil3 of width 11.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nfil3 of width 11.\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -znf423 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar of width 16.\n",
      "Using motif -rar of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf1 of width 21.\n",
      "Using motif -irf1 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hes of width 12.\n",
      "Using motif -hes of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +six3 of width 17.\n",
      "Using motif -six3 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif -znf410 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rxravdr of width 15.\n",
      "Using motif -rxravdr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mafg of width 21.\n",
      "Using motif -mafg of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2f1 of width 13.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cenpb of width 15.\n",
      "Using motif -cenpb of width 15.\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +scrt of width 13.\n",
      "Using motif -scrt of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rreb1 of width 20.\n",
      "Using motif -rreb1 of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myb of width 10.\n",
      "Using motif -myb of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f1 of width 12.\n",
      "Using motif -e2f1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -mybl1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prdm1 of width 15.\n",
      "Using motif -prdm1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbp of width 15.\n",
      "Using motif -tbp of width 15.\n",
      "Using motif +zbtb33 of width 15.\n",
      "Using motif -zbtb33 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +srebpf of width 10.\n",
      "Using motif -srebpf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfya of width 18.\n",
      "Using motif -nfya of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rest of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -rest of width 21.\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ctcf of width 19.\n",
      "Using motif -ctcf of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hinfp of width 12.\n",
      "Using motif -hinfp of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pbx1 of width 12.\n",
      "Using motif -pbx1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif -prox1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +gli2 of width 12.\n",
      "Using motif -gli2 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf1 of width 13.\n",
      "Using motif -hnf1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -arid5a of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mecom of width 14.\n",
      "Using motif -mecom of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax6 of width 14.\n",
      "Using motif -pax6 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +insm1 of width 12.\n",
      "Using motif -insm1 of width 12.\n",
      "Using motif +znf263 of width 21.\n",
      "Using motif -znf263 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +hnf4a of width 16.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -hnf4a of width 16.\n",
      "Using motif +roravar of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -roravar of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ap-1 of width 7.\n",
      "Using motif -ap-1 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Using motif +nfat of width 10.\n",
      "Using motif -nfat of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +thap1 of width 9.\n",
      "Using motif -thap1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gfi1b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -gfi1b of width 11.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f8 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myc of width 10.\n",
      "Using motif -e2f8 of width 12.\n",
      "Using motif -myc of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1 of width 10.\n",
      "Using motif -gfi1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa9 of width 11.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid3b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -arid3b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfkb of width 13.\n",
      "Using motif -nfkb of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif -yy2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dux4 of width 11.\n",
      "Using motif -dux4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +spz1 of width 11.\n",
      "Using motif -spz1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +zfx of width 14.\n",
      "Using motif -zfx of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmbx1 of width 17.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox17 of width 9.\n",
      "Using motif -sox17 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf740 of width 10.\n",
      "Using motif -znf740 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lef1 of width 15.\n",
      "Using motif -lef1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa11 of width 12.\n",
      "Using motif -hoxa11 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hsf of width 13.\n",
      "Using motif -hsf of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cux of width 10.\n",
      "Using motif -cux of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +esr2 of width 15.\n",
      "Using motif -esr2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif -mtf1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +klf of width 11.\n",
      "Using motif -klf of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -bhlha15 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +xbp1 of width 14.\n",
      "Using motif -xbp1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfyb of width 15.\n",
      "Using motif -nfyb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elf of width 12.\n",
      "Using motif -elf of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rora of width 10.\n",
      "Using motif -rora of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +plag1 of width 14.\n",
      "Using motif -plag1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif -grhl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e3 of width 7.\n",
      "Using motif -nr2e3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ebox of width 11.\n",
      "Using motif -ebox of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +srf of width 16.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -srf of width 16.\n",
      "Using motif +sp4 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -sp4 of width 17.\n",
      "Using motif +bcl6b of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -bcl6b of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Using motif +t of width 16.\n",
      "Using motif -t of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf143 of width 16.\n",
      "Using motif -znf143 of width 16.\n",
      "Using motif +hoxc13 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Using motif +duxa of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -duxa of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esrr of width 10.\n",
      "Using motif -esrr of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +onecut of width 14.\n",
      "Using motif -onecut of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcf21 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -tcf21 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp53 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -tp53 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +mybl2 of width 15.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif -mybl2 of width 15.\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +irf2 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -irf2 of width 18.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfi of width 6.\n",
      "Using motif -nfi of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nrf1 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f2 of width 16.\n",
      "Using motif -e2f2 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Using motif -bcl6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rargvar of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -rargvar of width 17.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou1f1 of width 14.\n",
      "Using motif -pou1f1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr5a2 of width 15.\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif -nr5a2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cdx of width 11.\n",
      "Using motif -cdx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f7 of width 14.\n",
      "Using motif -e2f7 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rar_rxr of width 17.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rhox11 of width 17.\n",
      "Using motif -rhox11 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +usf of width 11.\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -usf of width 11.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp63 of width 18.\n",
      "Using motif -tp63 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf7 of width 12.\n",
      "Using motif -tcf7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zic of width 14.\n",
      "Using motif -zic of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +esr1 of width 17.\n",
      "Using motif -esr1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rel of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -rel of width 10.\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif -sox10 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1var of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -mzf1var of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Using motif +pou6f2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pou6f2 of width 10.\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Using motif +sp1 of width 11.\n",
      "Using motif -sp1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbed1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -zbed1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfap1 of width 12.\n",
      "Using motif -tfap1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox8 of width 16.\n",
      "Using motif -sox8 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax5 of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pax5 of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Using motif +pparg of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -pparg of width 20.\n",
      "Using motif +cebp of width 10.\n",
      "Using motif -cebp of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Using motif +zbtb18 of width 13.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp73 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -tp73 of width 18.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +glis of width 14.\n",
      "Using motif -glis of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif -hnf4g of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rfx of width 16.\n",
      "Using motif -rfx of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +spi1-c of width 14.\n",
      "Using motif -spi1-c of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +phox2_prop of width 11.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfil3 of width 11.\n",
      "Using motif -nfil3 of width 11.\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -znf423 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar of width 16.\n",
      "Using motif -rar of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Using motif +e2f1 of width 12.\n",
      "Using motif +mecom of width 14.\n",
      "Using motif -e2f1 of width 12.\n",
      "Using motif -mecom of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myb of width 10.\n",
      "Using motif -myb of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rreb1 of width 20.\n",
      "Using motif -rreb1 of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cenpb of width 15.\n",
      "Using motif -cenpb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf1 of width 13.\n",
      "Using motif -hnf1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rxravdr of width 15.\n",
      "Using motif -rxravdr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f8 of width 12.\n",
      "Using motif -e2f8 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myc of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfya of width 18.\n",
      "Using motif -nfya of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbp of width 15.\n",
      "Using motif -tbp of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1 of width 10.\n",
      "Using motif -gfi1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hes of width 12.\n",
      "Using motif -hes of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hinfp of width 12.\n",
      "Using motif -hinfp of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif -gfi1b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb33 of width 15.\n",
      "Using motif -zbtb33 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid3b of width 11.\n",
      "Using motif -arid3b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +srebpf of width 10.\n",
      "Using motif -srebpf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ctcf of width 19.\n",
      "Using motif -ctcf of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2f1 of width 13.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Using motif +six3 of width 17.\n",
      "Using motif -six3 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa9 of width 11.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ap-1 of width 7.\n",
      "Using motif -ap-1 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +prdm1 of width 15.\n",
      "Using motif -prdm1 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rest of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -rest of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gli2 of width 12.\n",
      "Using motif -gli2 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfkb of width 13.\n",
      "Using motif -nfkb of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif -prox1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mafg of width 21.\n",
      "Using motif -mafg of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +roravar of width 14.\n",
      "Using motif -roravar of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif -yy2 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +insm1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -insm1 of width 12.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -arid5a of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf263 of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -znf263 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +thap1 of width 9.\n",
      "Using motif -thap1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mybl1 of width 12.\n",
      "Using motif -mybl1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif -znf410 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pbx1 of width 12.\n",
      "Using motif -pbx1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +irf1 of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -irf1 of width 21.\n",
      "Using motif +scrt of width 13.\n",
      "Using motif -scrt of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax6 of width 14.\n",
      "Using motif -pax6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4a of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -hnf4a of width 16.\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif +nfat of width 10.\n",
      "Using motif -nfat of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux4 of width 11.\n",
      "Using motif -dux4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +spz1 of width 11.\n",
      "Using motif -spz1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zfx of width 14.\n",
      "Using motif -zfx of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dmbx1 of width 17.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lef1 of width 15.\n",
      "Using motif -lef1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf740 of width 10.\n",
      "Using motif -znf740 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox17 of width 9.\n",
      "Using motif -sox17 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa11 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hoxa11 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cux of width 10.\n",
      "Using motif -cux of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Using motif +hsf of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -hsf of width 13.\n",
      "Read 1 motifs.\n",
      "Using motif +esr2 of width 15.\n",
      "Using motif -esr2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif -mtf1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +klf of width 11.\n",
      "Using motif -klf of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -bhlha15 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +xbp1 of width 14.\n",
      "Using motif -xbp1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfyb of width 15.\n",
      "Using motif -nfyb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +elf of width 12.\n",
      "Using motif -elf of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rora of width 10.\n",
      "Using motif -rora of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebox of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -ebox of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif -grhl1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e3 of width 7.\n",
      "Using motif -nr2e3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +duxa of width 13.\n",
      "Using motif +plag1 of width 14.\n",
      "Using motif -duxa of width 13.\n",
      "Using motif -plag1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp4 of width 17.\n",
      "Using motif -sp4 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +srf of width 16.\n",
      "Using motif -srf of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bcl6b of width 17.\n",
      "Using motif -bcl6b of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxc13 of width 11.\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +t of width 16.\n",
      "Using motif -t of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf143 of width 16.\n",
      "Using motif -znf143 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +esrr of width 10.\n",
      "Using motif -esrr of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +onecut of width 14.\n",
      "Using motif -onecut of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf21 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -tcf21 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp53 of width 18.\n",
      "Using motif -tp53 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Using motif +mybl2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +spib of width 7.\n",
      "Using motif -mybl2 of width 15.\n",
      "Using motif -spib of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +irf2 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -irf2 of width 18.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfi of width 6.\n",
      "Using motif -nfi of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nrf1 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f2 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -e2f2 of width 16.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif -bcl6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rargvar of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -rargvar of width 17.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou1f1 of width 14.\n",
      "Using motif -pou1f1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr5a2 of width 15.\n",
      "Using motif -nr5a2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cdx of width 11.\n",
      "Using motif -cdx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Using motif +e2f7 of width 14.\n",
      "Using motif -e2f7 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +usf of width 11.\n",
      "Using motif -usf of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +rar_rxr of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zic of width 14.\n",
      "Using motif -zic of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp63 of width 18.\n",
      "Using motif -tp63 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rhox11 of width 17.\n",
      "Using motif -rhox11 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou6f2 of width 10.\n",
      "Using motif -pou6f2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rel of width 10.\n",
      "Using motif -rel of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -tcf7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esr1 of width 17.\n",
      "Using motif -esr1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1var of width 10.\n",
      "Using motif -mzf1var of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp1 of width 11.\n",
      "Using motif -sp1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif -sox10 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +zbed1 of width 13.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -zbed1 of width 13.\n",
      "Using motif +tfap1 of width 12.\n",
      "Using motif -tfap1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox8 of width 16.\n",
      "Using motif -sox8 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Using motif +pax5 of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -pax5 of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb18 of width 13.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pparg of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp73 of width 18.\n",
      "Using motif -tp73 of width 18.\n",
      "Using motif +mga_tbx of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +cebp of width 10.\n",
      "Using motif -cebp of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +glis of width 14.\n",
      "Using motif -glis of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif -hnf4g of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rfx of width 16.\n",
      "Using motif -rfx of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfil3 of width 11.\n",
      "Using motif -nfil3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +phox2_prop of width 11.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spi1-c of width 14.\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Using motif -spi1-c of width 14.\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar of width 16.\n",
      "Using motif -rar of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -znf423 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf1 of width 13.\n",
      "Using motif -hnf1 of width 13.\n",
      "Using motif +ap-1 of width 7.\n",
      "Using motif -ap-1 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hes of width 12.\n",
      "Using motif -hes of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -e2f1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prdm1 of width 15.\n",
      "Using motif -prdm1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pbx1 of width 12.\n",
      "Using motif -pbx1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rest of width 21.\n",
      "Using motif -rest of width 21.\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hinfp of width 12.\n",
      "Using motif -hinfp of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf263 of width 21.\n",
      "Using motif -znf263 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rreb1 of width 20.\n",
      "Using motif -rreb1 of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf1 of width 21.\n",
      "Using motif -irf1 of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb33 of width 15.\n",
      "Using motif -zbtb33 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfat of width 10.\n",
      "Using motif -nfat of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +scrt of width 13.\n",
      "Using motif -scrt of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mafg of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -mafg of width 21.\n",
      "Read 1 motifs.\n",
      "Using motif +ctcf of width 19.\n",
      "Using motif -ctcf of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif -gfi1b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +insm1 of width 12.\n",
      "Using motif -insm1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +thap1 of width 9.\n",
      "Using motif -thap1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +srebpf of width 10.\n",
      "Using motif -srebpf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Read 1 motifs.\n",
      "Using motif +rxravdr of width 15.\n",
      "Using motif -rxravdr of width 15.\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax6 of width 14.\n",
      "Using motif -pax6 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif -znf410 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f8 of width 12.\n",
      "Using motif -e2f8 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myc of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1 of width 10.\n",
      "Using motif -gfi1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid3b of width 11.\n",
      "Using motif -arid3b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +yy2 of width 11.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -yy2 of width 11.\n",
      "Using motif +nr2f1 of width 13.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +six3 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -six3 of width 17.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -arid5a of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa9 of width 11.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mecom of width 14.\n",
      "Using motif -mecom of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl1 of width 12.\n",
      "Using motif -mybl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Using motif +dux4 of width 11.\n",
      "Using motif -dux4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tbp of width 15.\n",
      "Using motif -tbp of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4a of width 16.\n",
      "Using motif -hnf4a of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfya of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -nfya of width 18.\n",
      "Read 1 motifs.\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myb of width 10.\n",
      "Using motif -myb of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cenpb of width 15.\n",
      "Using motif -cenpb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Using motif +roravar of width 14.\n",
      "Using motif -roravar of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif -prox1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfkb of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -nfkb of width 13.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Using motif +gli2 of width 12.\n",
      "Using motif -gli2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +spz1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -spz1 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zfx of width 14.\n",
      "Using motif -zfx of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esr2 of width 15.\n",
      "Using motif -esr2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif -mtf1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lef1 of width 15.\n",
      "Using motif -lef1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf740 of width 10.\n",
      "Using motif -znf740 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa11 of width 12.\n",
      "Using motif -hoxa11 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +plag1 of width 14.\n",
      "Using motif -plag1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hsf of width 13.\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif -grhl1 of width 12.\n",
      "Using motif -hsf of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +xbp1 of width 14.\n",
      "Using motif -xbp1 of width 14.\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfyb of width 15.\n",
      "Using motif -nfyb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elf of width 12.\n",
      "Using motif -elf of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cux of width 10.\n",
      "Using motif -cux of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +klf of width 11.\n",
      "Using motif -klf of width 11.\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Using motif +duxa of width 13.\n",
      "Using motif -duxa of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmbx1 of width 17.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -bhlha15 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srf of width 16.\n",
      "Using motif -srf of width 16.\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc13 of width 11.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Using motif +sox17 of width 9.\n",
      "Using motif -sox17 of width 9.\n",
      "Using motif +rora of width 10.\n",
      "Using motif -rora of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +irf2 of width 18.\n",
      "Using motif +sp4 of width 17.\n",
      "Using motif -irf2 of width 18.\n",
      "Using motif -sp4 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebox of width 11.\n",
      "Using motif -ebox of width 11.\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfi of width 6.\n",
      "Using motif -nfi of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +esrr of width 10.\n",
      "Using motif -esrr of width 10.\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp53 of width 18.\n",
      "Using motif -tp53 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif +znf143 of width 16.\n",
      "Using motif +nr2e3 of width 7.\n",
      "Using motif -bcl6 of width 14.\n",
      "Using motif -nr2e3 of width 7.\n",
      "Using motif -znf143 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Using motif +onecut of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -onecut of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f7 of width 14.\n",
      "Using motif -e2f7 of width 14.\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar_rxr of width 17.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +t of width 16.\n",
      "Using motif -t of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +rhox11 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -rhox11 of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6b of width 17.\n",
      "Using motif -bcl6b of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cdx of width 11.\n",
      "Using motif -cdx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +usf of width 11.\n",
      "Using motif -usf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf21 of width 14.\n",
      "Using motif -tcf21 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Using motif +mybl2 of width 15.\n",
      "Using motif -mybl2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nrf1 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rargvar of width 17.\n",
      "Using motif -rargvar of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f2 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -e2f2 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zic of width 14.\n",
      "Using motif -zic of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr5a2 of width 15.\n",
      "Using motif -nr5a2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou6f2 of width 10.\n",
      "Using motif -pou6f2 of width 10.\n",
      "Using motif +tp63 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -tp63 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf7 of width 12.\n",
      "Using motif -tcf7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou1f1 of width 14.\n",
      "Using motif -pou1f1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rel of width 10.\n",
      "Using motif -rel of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfil3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nfil3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rfx of width 16.\n",
      "Using motif -rfx of width 16.\n",
      "Using motif +spi1-c of width 14.\n",
      "Using motif -spi1-c of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg of width 20.\n",
      "Using motif -pparg of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4g of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hnf4g of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp73 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +phox2_prop of width 11.\n",
      "Using motif -tp73 of width 18.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +glis of width 14.\n",
      "Using motif -glis of width 14.\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cebp of width 10.\n",
      "Using motif -cebp of width 10.\n",
      "Using motif +zbed1 of width 13.\n",
      "Using motif -zbed1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap1 of width 12.\n",
      "Using motif -tfap1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb18 of width 13.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sp1 of width 11.\n",
      "Using motif -sp1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +sox8 of width 16.\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif -sox10 of width 6.\n",
      "Using motif -sox8 of width 16.\n",
      "Using motif +pax5 of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -pax5 of width 19.\n",
      "Using motif +mzf1var of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -mzf1var of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +esr1 of width 17.\n",
      "Using motif -esr1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -znf423 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar of width 16.\n",
      "Using motif -rar of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rxravdr of width 15.\n",
      "Using motif -rxravdr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srebpf of width 10.\n",
      "Using motif -srebpf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +scrt of width 13.\n",
      "Using motif -scrt of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hinfp of width 12.\n",
      "Using motif -hinfp of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf1 of width 13.\n",
      "Using motif -hnf1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hes of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hes of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pbx1 of width 12.\n",
      "Using motif -pbx1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prdm1 of width 15.\n",
      "Using motif -prdm1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ap-1 of width 7.\n",
      "Using motif -ap-1 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb33 of width 15.\n",
      "Using motif -zbtb33 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mybl1 of width 12.\n",
      "Using motif -mybl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rreb1 of width 20.\n",
      "Using motif -rreb1 of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lin54 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -lin54 of width 9.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mecom of width 14.\n",
      "Using motif -mecom of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ctcf of width 19.\n",
      "Using motif -ctcf of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfat of width 10.\n",
      "Using motif -nfat of width 10.\n",
      "Using motif +insm1 of width 12.\n",
      "Using motif -insm1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f1 of width 12.\n",
      "Using motif -e2f1 of width 12.\n",
      "Using motif +six3 of width 17.\n",
      "Using motif -six3 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mafg of width 21.\n",
      "Using motif -mafg of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf1 of width 21.\n",
      "Using motif -irf1 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +thap1 of width 9.\n",
      "Using motif -thap1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf263 of width 21.\n",
      "Using motif -znf263 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif -gfi1b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfya of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -nfya of width 18.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +roravar of width 14.\n",
      "Using motif -roravar of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf4a of width 16.\n",
      "Using motif -hnf4a of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myb of width 10.\n",
      "Using motif -myb of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbp of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -tbp of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2f1 of width 13.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rest of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -rest of width 21.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -arid5a of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif -znf410 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cenpb of width 15.\n",
      "Using motif -cenpb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gli2 of width 12.\n",
      "Using motif -gli2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif -prox1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax6 of width 14.\n",
      "Using motif -pax6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f8 of width 12.\n",
      "Using motif -e2f8 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myc of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid3b of width 11.\n",
      "Using motif -arid3b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +nfkb of width 13.\n",
      "Using motif -nfkb of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif -yy2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa9 of width 11.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1 of width 10.\n",
      "Using motif -gfi1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux4 of width 11.\n",
      "Using motif -dux4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +spz1 of width 11.\n",
      "Using motif -spz1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zfx of width 14.\n",
      "Using motif -zfx of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmbx1 of width 17.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lef1 of width 15.\n",
      "Using motif -lef1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox17 of width 9.\n",
      "Using motif -sox17 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf740 of width 10.\n",
      "Using motif -znf740 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa11 of width 12.\n",
      "Using motif -hoxa11 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hsf of width 13.\n",
      "Using motif -hsf of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cux of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -cux of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esr2 of width 15.\n",
      "Using motif -esr2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif -mtf1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -bhlha15 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +xbp1 of width 14.\n",
      "Using motif -xbp1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfyb of width 15.\n",
      "Using motif -nfyb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elf of width 12.\n",
      "Using motif -elf of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +klf of width 11.\n",
      "Using motif -klf of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rora of width 10.\n",
      "Using motif -rora of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ebox of width 11.\n",
      "Using motif -ebox of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +plag1 of width 14.\n",
      "Using motif -plag1 of width 14.\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif -grhl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2e3 of width 7.\n",
      "Using motif -nr2e3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc13 of width 11.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Using motif +sp4 of width 17.\n",
      "Using motif -sp4 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +duxa of width 13.\n",
      "Using motif -duxa of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srf of width 16.\n",
      "Using motif -srf of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6b of width 17.\n",
      "Using motif -bcl6b of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf143 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -znf143 of width 16.\n",
      "Using motif +esrr of width 10.\n",
      "Using motif -esrr of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf21 of width 14.\n",
      "Using motif -tcf21 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +onecut of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -onecut of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl2 of width 15.\n",
      "Using motif -mybl2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp53 of width 18.\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Using motif -tp53 of width 18.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +t of width 16.\n",
      "Using motif -t of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +irf2 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -irf2 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f2 of width 16.\n",
      "Using motif -e2f2 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif -bcl6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nrf1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rargvar of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -rargvar of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr5a2 of width 15.\n",
      "Using motif -nr5a2 of width 15.\n",
      "Using motif +pou1f1 of width 14.\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Using motif -pou1f1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cdx of width 11.\n",
      "Using motif -cdx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f7 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -e2f7 of width 14.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Using motif +nfi of width 6.\n",
      "Using motif -nfi of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar_rxr of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Using motif -ventx of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rhox11 of width 17.\n",
      "Using motif -rhox11 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +usf of width 11.\n",
      "Using motif -usf of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zic of width 14.\n",
      "Using motif -zic of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp63 of width 18.\n",
      "Using motif -tp63 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou6f2 of width 10.\n",
      "Using motif -pou6f2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf7 of width 12.\n",
      "Using motif -tcf7 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rel of width 10.\n",
      "Using motif -rel of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esr1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -esr1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif -sox10 of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1var of width 10.\n",
      "Using motif -mzf1var of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sp1 of width 11.\n",
      "Using motif -sp1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbed1 of width 13.\n",
      "Using motif -zbed1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap1 of width 12.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -tfap1 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +sox8 of width 16.\n",
      "Using motif -sox8 of width 16.\n",
      "Using motif +pax5 of width 19.\n",
      "Using motif +zbtb18 of width 13.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Using motif -pax5 of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg of width 20.\n",
      "Using motif -pparg of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cebp of width 10.\n",
      "Using motif -cebp of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp73 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -tp73 of width 18.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +glis of width 14.\n",
      "Using motif -glis of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif -hnf4g of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rfx of width 16.\n",
      "Using motif -rfx of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spi1-c of width 14.\n",
      "Using motif -spi1-c of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfil3 of width 11.\n",
      "Using motif -nfil3 of width 11.\n",
      "Using motif +phox2_prop of width 11.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -znf423 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar of width 16.\n",
      "Using motif -rar of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf1 of width 13.\n",
      "Using motif -hnf1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfat of width 10.\n",
      "Using motif -nfat of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f1 of width 12.\n",
      "Using motif -e2f1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +scrt of width 13.\n",
      "Using motif -scrt of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mecom of width 14.\n",
      "Using motif -mecom of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pbx1 of width 12.\n",
      "Using motif -pbx1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif -gfi1b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hes of width 12.\n",
      "Using motif -hes of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rreb1 of width 20.\n",
      "Using motif -rreb1 of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +srebpf of width 10.\n",
      "Using motif -srebpf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -mybl1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prdm1 of width 15.\n",
      "Using motif -prdm1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hinfp of width 12.\n",
      "Using motif -hinfp of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +irf1 of width 21.\n",
      "Using motif -irf1 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ctcf of width 19.\n",
      "Using motif -ctcf of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +thap1 of width 9.\n",
      "Using motif -thap1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rxravdr of width 15.\n",
      "Using motif -rxravdr of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2f1 of width 13.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mafg of width 21.\n",
      "Using motif -mafg of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rest of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -rest of width 21.\n",
      "Using motif +zeb1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -zeb1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfya of width 18.\n",
      "Using motif -nfya of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ap-1 of width 7.\n",
      "Using motif -ap-1 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf263 of width 21.\n",
      "Using motif -znf263 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +six3 of width 17.\n",
      "Using motif -six3 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +insm1 of width 12.\n",
      "Using motif -insm1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Using motif -arid5a of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif -znf410 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +cenpb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -cenpb of width 15.\n",
      "Read 1 motifs.\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gli2 of width 12.\n",
      "Using motif -gli2 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif -prox1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax6 of width 14.\n",
      "Using motif -pax6 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myc of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f8 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -e2f8 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gfi1 of width 10.\n",
      "Using motif -gfi1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tbp of width 15.\n",
      "Using motif -tbp of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid3b of width 11.\n",
      "Using motif -arid3b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myb of width 10.\n",
      "Using motif -myb of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa9 of width 11.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf4a of width 16.\n",
      "Using motif -hnf4a of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -dux4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif +spz1 of width 11.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Using motif -spz1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +nfkb of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -nfkb of width 13.\n",
      "Using motif +zfx of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -zfx of width 14.\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dmbx1 of width 17.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Using motif -yy2 of width 11.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Using motif +sox17 of width 9.\n",
      "Using motif -sox17 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf740 of width 10.\n",
      "Using motif -znf740 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa11 of width 12.\n",
      "Using motif -hoxa11 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +hsf of width 13.\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -hsf of width 13.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cux of width 10.\n",
      "Using motif -cux of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +lef1 of width 15.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -lef1 of width 15.\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif -mtf1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esr2 of width 15.\n",
      "Using motif -esr2 of width 15.\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlha15 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -bhlha15 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +xbp1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -xbp1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +klf of width 11.\n",
      "Using motif -klf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfyb of width 15.\n",
      "Using motif -nfyb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elf of width 12.\n",
      "Using motif -elf of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebox of width 11.\n",
      "Using motif -ebox of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rora of width 10.\n",
      "Using motif -rora of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +plag1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -plag1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif -grhl1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e3 of width 7.\n",
      "Using motif -nr2e3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxc13 of width 11.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp4 of width 17.\n",
      "Using motif -sp4 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +duxa of width 13.\n",
      "Using motif -duxa of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +srf of width 16.\n",
      "Using motif -srf of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6b of width 17.\n",
      "Using motif -bcl6b of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +t of width 16.\n",
      "Using motif -t of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esrr of width 10.\n",
      "Using motif -esrr of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf143 of width 16.\n",
      "Using motif -znf143 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +onecut of width 14.\n",
      "Using motif -onecut of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcf21 of width 14.\n",
      "Using motif -tcf21 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +roravar of width 14.\n",
      "Using motif -roravar of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl2 of width 15.\n",
      "Using motif -mybl2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Using motif +tp53 of width 18.\n",
      "Using motif -tp53 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf2 of width 18.\n",
      "Using motif -irf2 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfi of width 6.\n",
      "Using motif -nfi of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nrf1 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f2 of width 16.\n",
      "Using motif -e2f2 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Using motif -bcl6 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rargvar of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -rargvar of width 17.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +pou1f1 of width 14.\n",
      "Using motif -pou1f1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr5a2 of width 15.\n",
      "Using motif -nr5a2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cdx of width 11.\n",
      "Using motif -cdx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f7 of width 14.\n",
      "Using motif -e2f7 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rar_rxr of width 17.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rhox11 of width 17.\n",
      "Using motif -rhox11 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb33 of width 15.\n",
      "Using motif -zbtb33 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +usf of width 11.\n",
      "Using motif -usf of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp63 of width 18.\n",
      "Using motif -tp63 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zic of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -zic of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou6f2 of width 10.\n",
      "Using motif -pou6f2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcf7 of width 12.\n",
      "Using motif -tcf7 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rel of width 10.\n",
      "Using motif -rel of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +esr1 of width 17.\n",
      "Using motif -esr1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif -sox10 of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1var of width 10.\n",
      "Using motif -mzf1var of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +sp1 of width 11.\n",
      "Using motif -sp1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbed1 of width 13.\n",
      "Using motif -zbed1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfap1 of width 12.\n",
      "Using motif -tfap1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox8 of width 16.\n",
      "Using motif -sox8 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax5 of width 19.\n",
      "Using motif -pax5 of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb18 of width 13.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pparg of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cebp of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -cebp of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp73 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -tp73 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +glis of width 14.\n",
      "Using motif -glis of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif -hnf4g of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rfx of width 16.\n",
      "Using motif -rfx of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +spi1-c of width 14.\n",
      "Using motif -spi1-c of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfil3 of width 11.\n",
      "Using motif -nfil3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +phox2_prop of width 11.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -znf423 of width 15.\n",
      "Using motif +rar of width 16.\n",
      "Using motif -rar of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Using motif +scrt of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -scrt of width 13.\n",
      "Using motif +irf1 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -irf1 of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif -yy2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +thap1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -thap1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf263 of width 21.\n",
      "Using motif -znf263 of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pax6 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myb of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -myb of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pbx1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pbx1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cenpb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -cenpb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tbp of width 15.\n",
      "Using motif -tbp of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf1 of width 13.\n",
      "Using motif -hnf1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zeb1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -zeb1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dux4 of width 11.\n",
      "Using motif -dux4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfat of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nfat of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +spz1 of width 11.\n",
      "Using motif -spz1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hes of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hes of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ap-1 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -ap-1 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2f1 of width 13.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hinfp of width 12.\n",
      "Using motif -hinfp of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f8 of width 12.\n",
      "Using motif -e2f8 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfya of width 18.\n",
      "Using motif -nfya of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +six3 of width 17.\n",
      "Using motif -six3 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc9 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hoxc9 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rest of width 21.\n",
      "Using motif -rest of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rxravdr of width 15.\n",
      "Using motif -rxravdr of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zfx of width 14.\n",
      "Using motif -zfx of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myc of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mybl1 of width 12.\n",
      "Using motif -mybl1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +egr of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -egr of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f1 of width 12.\n",
      "Using motif -e2f1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +gli2 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif -gli2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb33 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -zbtb33 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +prdm1 of width 15.\n",
      "Using motif -prdm1 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +insm1 of width 12.\n",
      "Using motif -insm1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srebpf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -srebpf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +roravar of width 14.\n",
      "Using motif -roravar of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gfi1 of width 10.\n",
      "Using motif -gfi1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mecom of width 14.\n",
      "Using motif -mecom of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif -gfi1b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid3b of width 11.\n",
      "Using motif -arid3b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmrt3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4a of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -hnf4a of width 16.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dmbx1 of width 17.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif -znf410 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rreb1 of width 20.\n",
      "Using motif -rreb1 of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ctcf of width 19.\n",
      "Using motif -ctcf of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -arid5a of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfkb of width 13.\n",
      "Using motif -nfkb of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif -prox1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa9 of width 11.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox17 of width 9.\n",
      "Using motif -sox17 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hsf of width 13.\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Using motif -hsf of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf740 of width 10.\n",
      "Using motif -znf740 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mafg of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -mafg of width 21.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa11 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Using motif -hoxa11 of width 12.\n",
      "Using motif +cux of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -cux of width 10.\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lhx3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -lhx3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +esr2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -esr2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lef1 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -lef1 of width 15.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +elf of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -elf of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Using motif -mtf1 of width 14.\n",
      "Using motif +nfyb of width 15.\n",
      "Using motif -nfyb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +mzf1 of width 6.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -mzf1 of width 6.\n",
      "Using motif +klf of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif -bhlha15 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -klf of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rora of width 10.\n",
      "Using motif -rora of width 10.\n",
      "Using motif +plag1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +xbp1 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif -plag1 of width 14.\n",
      "Using motif -xbp1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc13 of width 11.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Using motif +grhl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +duxa of width 13.\n",
      "Using motif -duxa of width 13.\n",
      "Using motif +nr2e3 of width 7.\n",
      "Using motif -nr2e3 of width 7.\n",
      "Using motif +ebox of width 11.\n",
      "Using motif -ebox of width 11.\n",
      "Using motif +sp4 of width 17.\n",
      "Using motif -sp4 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -grhl1 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +srf of width 16.\n",
      "Using motif -srf of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +t of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -t of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf143 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -znf143 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +onecut of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -onecut of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6b of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -bcl6b of width 17.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Using motif +tcf21 of width 14.\n",
      "Using motif +esrr of width 10.\n",
      "Using motif -esrr of width 10.\n",
      "Using motif -tcf21 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlh of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -bhlh of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl2 of width 15.\n",
      "Using motif -mybl2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp53 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -tp53 of width 18.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +irf2 of width 18.\n",
      "Using motif -irf2 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nrf1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nrf1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f2 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -e2f2 of width 16.\n",
      "Read 1 motifs.\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif -bcl6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfi of width 6.\n",
      "Using motif -nfi of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +dux of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif -dux of width 8.\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rargvar of width 17.\n",
      "Using motif -rargvar of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou1f1 of width 14.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Using motif -pou1f1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr5a2 of width 15.\n",
      "Using motif -nr5a2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +cdx of width 11.\n",
      "Using motif -cdx of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f7 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -e2f7 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rar_rxr of width 17.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rhox11 of width 17.\n",
      "Using motif -rhox11 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +usf of width 11.\n",
      "Using motif -usf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zic of width 14.\n",
      "Using motif -zic of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp63 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -tp63 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcf7 of width 12.\n",
      "Using motif -tcf7 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +rel of width 10.\n",
      "Using motif -rel of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Using motif +pou6f2 of width 10.\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Using motif -pou6f2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif -sox10 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esr1 of width 17.\n",
      "Using motif -esr1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1var of width 10.\n",
      "Using motif -mzf1var of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbed1 of width 13.\n",
      "Using motif -zbed1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp1 of width 11.\n",
      "Using motif +tfap1 of width 12.\n",
      "Using motif -sp1 of width 11.\n",
      "Using motif -tfap1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb18 of width 13.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox8 of width 16.\n",
      "Using motif -sox8 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pparg of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -pparg of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cebp of width 10.\n",
      "Using motif -cebp of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax5 of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pax5 of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +tp73 of width 18.\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Using motif -tp73 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4g of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -hnf4g of width 15.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rfx of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -rfx of width 16.\n",
      "Using motif +glis of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -glis of width 14.\n",
      "Using motif +spi1-c of width 14.\n",
      "Using motif -spi1-c of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfil3 of width 11.\n",
      "Using motif -nfil3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +phox2_prop of width 11.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf423 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -znf423 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar of width 16.\n",
      "Using motif -rar of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mecom of width 14.\n",
      "Using motif -mecom of width 14.\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfat of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nfat of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rreb1 of width 20.\n",
      "Using motif -rreb1 of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4a of width 16.\n",
      "Using motif -hnf4a of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mafg of width 21.\n",
      "Using motif -mafg of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ctcf of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -ctcf of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf263 of width 21.\n",
      "Using motif -znf263 of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -mybl1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +roravar of width 14.\n",
      "Using motif -roravar of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +insm1 of width 12.\n",
      "Using motif -insm1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -arid5a of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif -znf410 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +thap1 of width 9.\n",
      "Using motif -thap1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cenpb of width 15.\n",
      "Using motif -cenpb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srebpf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -srebpf of width 10.\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f1 of width 12.\n",
      "Using motif -e2f1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myb of width 10.\n",
      "Using motif -myb of width 10.\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif -gfi1b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hinfp of width 12.\n",
      "Using motif -hinfp of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hes of width 12.\n",
      "Using motif -hes of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfya of width 18.\n",
      "Using motif -nfya of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gli2 of width 12.\n",
      "Using motif -gli2 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf1 of width 13.\n",
      "Using motif -hnf1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +prox1 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif -prox1 of width 12.\n",
      "Using motif +tbp of width 15.\n",
      "Using motif -tbp of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rxravdr of width 15.\n",
      "Using motif -rxravdr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +six3 of width 17.\n",
      "Using motif -six3 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +prdm1 of width 15.\n",
      "Using motif -prdm1 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2f1 of width 13.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gcm of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -gcm of width 10.\n",
      "Using motif +rest of width 21.\n",
      "Using motif -rest of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +irf1 of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -irf1 of width 21.\n",
      "Using motif +pbx1 of width 12.\n",
      "Using motif -pbx1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb33 of width 15.\n",
      "Using motif -zbtb33 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ap-1 of width 7.\n",
      "Using motif -ap-1 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +stat6 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -stat6 of width 15.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +scrt of width 13.\n",
      "Using motif -scrt of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax6 of width 14.\n",
      "Using motif -pax6 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f8 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -e2f8 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myc of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1 of width 10.\n",
      "Using motif -gfi1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid3b of width 11.\n",
      "Using motif -arid3b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa9 of width 11.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfkb of width 13.\n",
      "Using motif -nfkb of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif -yy2 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -dux4 of width 11.\n",
      "Using motif +spz1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -spz1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zfx of width 14.\n",
      "Using motif -zfx of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dmbx1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Using motif +lef1 of width 15.\n",
      "Using motif -lef1 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox17 of width 9.\n",
      "Using motif -sox17 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf740 of width 10.\n",
      "Using motif -znf740 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa11 of width 12.\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Using motif -hoxa11 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cux of width 10.\n",
      "Using motif -cux of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esr2 of width 15.\n",
      "Using motif -esr2 of width 15.\n",
      "Using motif +hsf of width 13.\n",
      "Using motif -hsf of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif -mtf1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +klf of width 11.\n",
      "Using motif -klf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -bhlha15 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +xbp1 of width 14.\n",
      "Using motif -xbp1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfyb of width 15.\n",
      "Using motif -nfyb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +elf of width 12.\n",
      "Using motif -elf of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rora of width 10.\n",
      "Using motif -rora of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +plag1 of width 14.\n",
      "Using motif -plag1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebox of width 11.\n",
      "Using motif -ebox of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif -grhl1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e3 of width 7.\n",
      "Using motif -nr2e3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc13 of width 11.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sp4 of width 17.\n",
      "Using motif -sp4 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +duxa of width 13.\n",
      "Using motif -duxa of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +srf of width 16.\n",
      "Using motif -srf of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6b of width 17.\n",
      "Using motif -bcl6b of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +t of width 16.\n",
      "Using motif -t of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf143 of width 16.\n",
      "Using motif -znf143 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +esrr of width 10.\n",
      "Using motif -esrr of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +onecut of width 14.\n",
      "Using motif -onecut of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcf21 of width 14.\n",
      "Using motif -tcf21 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl2 of width 15.\n",
      "Using motif -mybl2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp53 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -tp53 of width 18.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +irf2 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -irf2 of width 18.\n",
      "Read 1 motifs.\n",
      "Using motif +nfi of width 6.\n",
      "Using motif -nfi of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +nrf1 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +fox of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -fox of width 8.\n",
      "Using motif +bcl6 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -bcl6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Using motif +e2f2 of width 16.\n",
      "Using motif -e2f2 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +pou1f1 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +rargvar of width 17.\n",
      "Using motif -pou1f1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -rargvar of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr5a2 of width 15.\n",
      "Using motif -nr5a2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cdx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -cdx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rar_rxr of width 17.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Using motif +e2f7 of width 14.\n",
      "Using motif -e2f7 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rhox11 of width 17.\n",
      "Using motif -rhox11 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +usf of width 11.\n",
      "Using motif -usf of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zic of width 14.\n",
      "Using motif -zic of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp63 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -tp63 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou6f2 of width 10.\n",
      "Using motif -pou6f2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +tcf7 of width 12.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -tcf7 of width 12.\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rel of width 10.\n",
      "Using motif -rel of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +esr1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -esr1 of width 17.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif -sox10 of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mzf1var of width 10.\n",
      "Using motif -mzf1var of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp1 of width 11.\n",
      "Using motif -sp1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbed1 of width 13.\n",
      "Using motif -zbed1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfap1 of width 12.\n",
      "Using motif -tfap1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox8 of width 16.\n",
      "Using motif -sox8 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb18 of width 13.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax5 of width 19.\n",
      "Using motif -pax5 of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pparg of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -pparg of width 20.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cebp of width 10.\n",
      "Using motif -cebp of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid3a of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -arid3a of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Using motif +glis of width 14.\n",
      "Using motif -glis of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif -hnf4g of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rfx of width 16.\n",
      "Using motif -rfx of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp73 of width 18.\n",
      "Using motif -tp73 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfil3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nfil3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spi1-c of width 14.\n",
      "Using motif -spi1-c of width 14.\n",
      "Using motif +phox2_prop of width 11.\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Using motif -znf423 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar of width 16.\n",
      "Using motif -rar of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Using motif -znf410 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tbp of width 15.\n",
      "Using motif -tbp of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +scrt of width 13.\n",
      "Using motif -scrt of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f8 of width 12.\n",
      "Using motif -e2f8 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfat of width 10.\n",
      "Using motif -nfat of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hes of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hes of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfya of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nfya of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myc of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +roravar of width 14.\n",
      "Using motif -roravar of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -gfi1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rreb1 of width 20.\n",
      "Using motif -rreb1 of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f1 of width 12.\n",
      "Using motif -e2f1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif -gfi1b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rxravdr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -rxravdr of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ap-1 of width 7.\n",
      "Using motif -ap-1 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +six3 of width 17.\n",
      "Using motif -six3 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid3b of width 11.\n",
      "Using motif -arid3b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf4a of width 16.\n",
      "Using motif -hnf4a of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hinfp of width 12.\n",
      "Using motif -hinfp of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr4a2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nr4a2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rest of width 21.\n",
      "Using motif -rest of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cenpb of width 15.\n",
      "Using motif -cenpb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pbx1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pbx1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb33 of width 15.\n",
      "Using motif -zbtb33 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ctcf of width 19.\n",
      "Using motif -ctcf of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -arid5a of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mybl1 of width 12.\n",
      "Using motif -mybl1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa9 of width 11.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf263 of width 21.\n",
      "Using motif -znf263 of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Using motif +nfkb of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nfkb of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +insm1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -insm1 of width 12.\n",
      "Using motif +thap1 of width 9.\n",
      "Using motif -thap1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myb of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -myb of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +mafg of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -mafg of width 21.\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif -yy2 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +irf1 of width 21.\n",
      "Using motif -irf1 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srebpf of width 10.\n",
      "Using motif -srebpf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gli2 of width 12.\n",
      "Using motif -gli2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prdm1 of width 15.\n",
      "Using motif -prdm1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +gsc of width 10.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -gsc of width 10.\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +nr2f1 of width 13.\n",
      "Read 1 motifs.\n",
      "Using motif +pax6 of width 14.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Using motif -pax6 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +mecom of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Using motif -mecom of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +prox1 of width 12.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +crem of width 10.\n",
      "Using motif -prox1 of width 12.\n",
      "Using motif -crem of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux4 of width 11.\n",
      "Using motif -dux4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spz1 of width 11.\n",
      "Using motif -spz1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf1 of width 13.\n",
      "Using motif -hnf1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zfx of width 14.\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Using motif -zfx of width 14.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmbx1 of width 17.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf740 of width 10.\n",
      "Using motif -znf740 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lef1 of width 15.\n",
      "Using motif -lef1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa11 of width 12.\n",
      "Using motif -hoxa11 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +sox17 of width 9.\n",
      "Using motif -sox17 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hsf of width 13.\n",
      "Using motif -hsf of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif -mtf1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Using motif +cux of width 10.\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Using motif -cux of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -bhlha15 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +xbp1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -xbp1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfyb of width 15.\n",
      "Using motif -nfyb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elf of width 12.\n",
      "Using motif -elf of width 12.\n",
      "Using motif +esr2 of width 15.\n",
      "Using motif -esr2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Using motif +klf of width 11.\n",
      "Using motif -klf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rora of width 10.\n",
      "Using motif -rora of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2e3 of width 7.\n",
      "Using motif -nr2e3 of width 7.\n",
      "Using motif +ebox of width 11.\n",
      "Using motif -ebox of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif +sp4 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -grhl1 of width 12.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -sp4 of width 17.\n",
      "Using motif +hoxc13 of width 11.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +duxa of width 13.\n",
      "Using motif -duxa of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6b of width 17.\n",
      "Using motif -bcl6b of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif +plag1 of width 14.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -plag1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srf of width 16.\n",
      "Using motif -srf of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +t of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -t of width 16.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf143 of width 16.\n",
      "Using motif -znf143 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esrr of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -esrr of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcf21 of width 14.\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Using motif -tcf21 of width 14.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Using motif +onecut of width 14.\n",
      "Using motif -onecut of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mybl2 of width 15.\n",
      "Using motif -mybl2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp53 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -tp53 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfi of width 6.\n",
      "Using motif -nfi of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nrf1 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif -bcl6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Using motif +e2f2 of width 16.\n",
      "Using motif +irf2 of width 18.\n",
      "Using motif -e2f2 of width 16.\n",
      "Using motif -irf2 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Using motif +rargvar of width 17.\n",
      "Using motif -rargvar of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou1f1 of width 14.\n",
      "Using motif -pou1f1 of width 14.\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr5a2 of width 15.\n",
      "Using motif -nr5a2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cdx of width 11.\n",
      "Using motif -cdx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f7 of width 14.\n",
      "Using motif -e2f7 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rar_rxr of width 17.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +usf of width 11.\n",
      "Using motif -usf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +tp63 of width 18.\n",
      "Using motif +rhox11 of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif +zic of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -rhox11 of width 17.\n",
      "Using motif -zic of width 14.\n",
      "Using motif -tp63 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +pou6f2 of width 10.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pou6f2 of width 10.\n",
      "Using motif +tcf7 of width 12.\n",
      "Using motif -tcf7 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rel of width 10.\n",
      "Using motif -rel of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Read 1 motifs.\n",
      "Using motif +gmeb2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -gmeb2 of width 8.\n",
      "Using motif +esr1 of width 17.\n",
      "Using motif -esr1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif -sox10 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1var of width 10.\n",
      "Using motif -mzf1var of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbed1 of width 13.\n",
      "Using motif -zbed1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -tfap1 of width 12.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox8 of width 16.\n",
      "Using motif -sox8 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb18 of width 13.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp1 of width 11.\n",
      "Using motif -sp1 of width 11.\n",
      "Using motif +cebp of width 10.\n",
      "Using motif -cebp of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax5 of width 19.\n",
      "Using motif -pax5 of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -pparg of width 20.\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp73 of width 18.\n",
      "Using motif -tp73 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +glis of width 14.\n",
      "Using motif -glis of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Using motif -hnf4g of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rfx of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -rfx of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spi1-c of width 14.\n",
      "Using motif -spi1-c of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfil3 of width 11.\n",
      "Using motif -nfil3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -znf423 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +rar of width 16.\n",
      "Using motif -rar of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif +phox2_prop of width 11.\n",
      "Using motif -yy1 of width 12.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfkb of width 13.\n",
      "Using motif -nfkb of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ctcf of width 19.\n",
      "Using motif -ctcf of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa9 of width 11.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid3b of width 11.\n",
      "Using motif -arid3b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +insm1 of width 12.\n",
      "Using motif -insm1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hes of width 12.\n",
      "Using motif -hes of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srebpf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -srebpf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ap-1 of width 7.\n",
      "Using motif -ap-1 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif +e2f1 of width 12.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Using motif -e2f1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +six3 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -six3 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf1 of width 13.\n",
      "Using motif -hnf1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +irf1 of width 21.\n",
      "Using motif -irf1 of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +scrt of width 13.\n",
      "Using motif -scrt of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mafg of width 21.\n",
      "Using motif -mafg of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -gfi1b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf263 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -znf263 of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prdm1 of width 15.\n",
      "Using motif -prdm1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +roravar of width 14.\n",
      "Using motif -roravar of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mecom of width 14.\n",
      "Using motif -mecom of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rreb1 of width 20.\n",
      "Using motif -rreb1 of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rxravdr of width 15.\n",
      "Using motif -rxravdr of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pbx1 of width 12.\n",
      "Using motif -pbx1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbp of width 15.\n",
      "Using motif -tbp of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pax6 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hinfp of width 12.\n",
      "Using motif -hinfp of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl1 of width 12.\n",
      "Using motif -mybl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f8 of width 12.\n",
      "Using motif -e2f8 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myc of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gfi1 of width 10.\n",
      "Using motif -gfi1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb33 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -zbtb33 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfya of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -nfya of width 18.\n",
      "Read 1 motifs.\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif -yy2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -arid5a of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2f1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nr2f1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4a of width 16.\n",
      "Using motif -hnf4a of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmbx1 of width 17.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rest of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -rest of width 21.\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif -znf410 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif +nfat of width 10.\n",
      "Using motif -nfat of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux4 of width 11.\n",
      "Using motif -dux4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cenpb of width 15.\n",
      "Using motif -cenpb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lef1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -lef1 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat1-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -stat1-3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox17 of width 9.\n",
      "Using motif -sox17 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf740 of width 10.\n",
      "Using motif -znf740 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +thap1 of width 9.\n",
      "Using motif -thap1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hsf of width 13.\n",
      "Using motif -hsf of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spz1 of width 11.\n",
      "Using motif -spz1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myb of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -myb of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cux of width 10.\n",
      "Using motif -cux of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gli2 of width 12.\n",
      "Using motif -gli2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif -prox1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa11 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hoxa11 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zfx of width 14.\n",
      "Using motif -zfx of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +esr2 of width 15.\n",
      "Using motif -esr2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif -mtf1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mybl2 of width 15.\n",
      "Using motif -mybl2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp53 of width 18.\n",
      "Using motif -tp53 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +duxa of width 13.\n",
      "Using motif -duxa of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +srf of width 16.\n",
      "Using motif -srf of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6b of width 17.\n",
      "Using motif -bcl6b of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +xbp1 of width 14.\n",
      "Using motif -xbp1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rora of width 10.\n",
      "Using motif -rora of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Using motif +klf of width 11.\n",
      "Using motif -klf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf143 of width 16.\n",
      "Using motif -znf143 of width 16.\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +plag1 of width 14.\n",
      "Using motif -plag1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esrr of width 10.\n",
      "Using motif -esrr of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf2 of width 18.\n",
      "Using motif -irf2 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +onecut of width 14.\n",
      "Using motif -onecut of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcf21 of width 14.\n",
      "Using motif -tcf21 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif -grhl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +t of width 16.\n",
      "Using motif -t of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elf of width 12.\n",
      "Using motif -elf of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfi of width 6.\n",
      "Using motif -nfi of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -bhlha15 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ebox of width 11.\n",
      "Using motif -ebox of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f2 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -e2f2 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e3 of width 7.\n",
      "Using motif -nr2e3 of width 7.\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfyb of width 15.\n",
      "Using motif -nfyb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif -bcl6 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc13 of width 11.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nrf1 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp4 of width 17.\n",
      "Using motif -sp4 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f7 of width 14.\n",
      "Using motif -e2f7 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Using motif +pou1f1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pou1f1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou6f2 of width 10.\n",
      "Using motif -pou6f2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr5a2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nr5a2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Using motif +sp1 of width 11.\n",
      "Using motif -sp1 of width 11.\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif +rar_rxr of width 17.\n",
      "Using motif -sox10 of width 6.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rhox11 of width 17.\n",
      "Using motif -rhox11 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbed1 of width 13.\n",
      "Using motif -zbed1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rargvar of width 17.\n",
      "Using motif -rargvar of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp73 of width 18.\n",
      "Using motif -tp73 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg of width 20.\n",
      "Using motif -pparg of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax5 of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pax5 of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox8 of width 16.\n",
      "Using motif -sox8 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfil3 of width 11.\n",
      "Using motif -nfil3 of width 11.\n",
      "Using motif +mzf1var of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -mzf1var of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb18 of width 13.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap1 of width 12.\n",
      "Using motif -tfap1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cdx of width 11.\n",
      "Using motif -cdx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zic of width 14.\n",
      "Using motif -zic of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rel of width 10.\n",
      "Using motif -rel of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp63 of width 18.\n",
      "Using motif -tp63 of width 18.\n",
      "Using motif +usf of width 11.\n",
      "Using motif -usf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +esr1 of width 17.\n",
      "Using motif -esr1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cebp of width 10.\n",
      "Using motif -cebp of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf7 of width 12.\n",
      "Using motif -tcf7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +phox2_prop of width 11.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif -hnf4g of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +rar of width 16.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -rar of width 16.\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +glis of width 14.\n",
      "Using motif -glis of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rfx of width 16.\n",
      "Using motif -rfx of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spi1-c of width 14.\n",
      "Using motif -spi1-c of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -znf423 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pbx1 of width 12.\n",
      "Using motif -pbx1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif -gfi1b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox17 of width 9.\n",
      "Using motif -sox17 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hsf of width 13.\n",
      "Using motif -hsf of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hes of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hes of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ap-1 of width 7.\n",
      "Using motif -ap-1 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srebpf of width 10.\n",
      "Using motif -srebpf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +thap1 of width 9.\n",
      "Using motif -thap1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mecom of width 14.\n",
      "Using motif -mecom of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2f1 of width 13.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prdm1 of width 15.\n",
      "Using motif -prdm1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif -znf410 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lef1 of width 15.\n",
      "Using motif -lef1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +scrt of width 13.\n",
      "Using motif -scrt of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmrt3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Using motif +hnf1 of width 13.\n",
      "Using motif -hnf1 of width 13.\n",
      "Using motif +insm1 of width 12.\n",
      "Using motif -insm1 of width 12.\n",
      "Using motif +e2f1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -e2f1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl1 of width 12.\n",
      "Using motif -mybl1 of width 12.\n",
      "Using motif +rreb1 of width 20.\n",
      "Using motif -rreb1 of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myb of width 10.\n",
      "Using motif -myb of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elk_etv of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -elk_etv of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ctcf of width 19.\n",
      "Using motif -ctcf of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf4a of width 16.\n",
      "Using motif -hnf4a of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf263 of width 21.\n",
      "Using motif -znf263 of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfat of width 10.\n",
      "Using motif -nfat of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mafg of width 21.\n",
      "Using motif -mafg of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb33 of width 15.\n",
      "Using motif -zbtb33 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +roravar of width 14.\n",
      "Using motif -roravar of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa9 of width 11.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Using motif +six3 of width 17.\n",
      "Using motif -six3 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Using motif +gfi1 of width 10.\n",
      "Using motif -gfi1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rest of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -rest of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rxravdr of width 15.\n",
      "Using motif -rxravdr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf1 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -irf1 of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbp of width 15.\n",
      "Using motif -tbp of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gli2 of width 12.\n",
      "Using motif -gli2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -arid5a of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hinfp of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -hinfp of width 12.\n",
      "Using motif +nfya of width 18.\n",
      "Using motif -nfya of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfkb of width 13.\n",
      "Using motif -nfkb of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif -yy2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf740 of width 10.\n",
      "Using motif -znf740 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa11 of width 12.\n",
      "Using motif -hoxa11 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f8 of width 12.\n",
      "Using motif -e2f8 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid3b of width 11.\n",
      "Using motif -arid3b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif -prox1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zfx of width 14.\n",
      "Using motif -zfx of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux4 of width 11.\n",
      "Using motif -dux4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax6 of width 14.\n",
      "Using motif -pax6 of width 14.\n",
      "Using motif +myc of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spz1 of width 11.\n",
      "Using motif -spz1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cenpb of width 15.\n",
      "Using motif -cenpb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cux of width 10.\n",
      "Using motif -cux of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmbx1 of width 17.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e3 of width 7.\n",
      "Using motif -nr2e3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ebox of width 11.\n",
      "Using motif -ebox of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -bhlha15 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +plag1 of width 14.\n",
      "Using motif -plag1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc13 of width 11.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esr2 of width 15.\n",
      "Using motif -esr2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif -mtf1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfyb of width 15.\n",
      "Using motif -nfyb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif -grhl1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rora of width 10.\n",
      "Using motif -rora of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +elf of width 12.\n",
      "Using motif -elf of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +klf of width 11.\n",
      "Using motif -klf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -zbtb7 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +xbp1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -xbp1 of width 14.\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp4 of width 17.\n",
      "Using motif -sp4 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +duxa of width 13.\n",
      "Using motif -duxa of width 13.\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6b of width 17.\n",
      "Using motif -bcl6b of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +srf of width 16.\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Using motif -srf of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcf21 of width 14.\n",
      "Using motif -tcf21 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfi of width 6.\n",
      "Using motif -nfi of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mybl2 of width 15.\n",
      "Using motif -mybl2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf143 of width 16.\n",
      "Using motif -znf143 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +esrr of width 10.\n",
      "Using motif -esrr of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf2 of width 18.\n",
      "Using motif -irf2 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp53 of width 18.\n",
      "Using motif -tp53 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +onecut of width 14.\n",
      "Using motif -onecut of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +t of width 16.\n",
      "Using motif -t of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nrf1 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rargvar of width 17.\n",
      "Using motif -rargvar of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou1f1 of width 14.\n",
      "Using motif -pou1f1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar_rxr of width 17.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Using motif +nr5a2 of width 15.\n",
      "Using motif -nr5a2 of width 15.\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Using motif +cdx of width 11.\n",
      "Using motif -cdx of width 11.\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Using motif +e2f2 of width 16.\n",
      "Using motif -e2f2 of width 16.\n",
      "Using motif +e2f7 of width 14.\n",
      "Using motif -e2f7 of width 14.\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif -bcl6 of width 14.\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Using motif +pax5 of width 19.\n",
      "Using motif -pax5 of width 19.\n",
      "Using motif +rhox11 of width 17.\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Using motif -rhox11 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Using motif +usf of width 11.\n",
      "Using motif -usf of width 11.\n",
      "Using motif +sox8 of width 16.\n",
      "Using motif -sox8 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zic of width 14.\n",
      "Using motif -zic of width 14.\n",
      "Using motif +esr1 of width 17.\n",
      "Using motif -esr1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb18 of width 13.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou6f2 of width 10.\n",
      "Using motif -pou6f2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf7 of width 12.\n",
      "Using motif -tcf7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rel of width 10.\n",
      "Using motif -rel of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp63 of width 18.\n",
      "Using motif -tp63 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif -sox10 of width 6.\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1var of width 10.\n",
      "Using motif -mzf1var of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -sp1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbed1 of width 13.\n",
      "Using motif -zbed1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap1 of width 12.\n",
      "Using motif -tfap1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pparg of width 20.\n",
      "Using motif -pparg of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rfx of width 16.\n",
      "Using motif -rfx of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfil3 of width 11.\n",
      "Using motif -nfil3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +phox2_prop of width 11.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp73 of width 18.\n",
      "Using motif -tp73 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar of width 16.\n",
      "Using motif -rar of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +glis of width 14.\n",
      "Using motif -glis of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Using motif +cebp of width 10.\n",
      "Using motif -cebp of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spi1-c of width 14.\n",
      "Using motif -spi1-c of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif -hnf4g of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -znf423 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif -gfi1b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rxravdr of width 15.\n",
      "Using motif -rxravdr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbp of width 15.\n",
      "Using motif -tbp of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf263 of width 21.\n",
      "Using motif -znf263 of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srebpf of width 10.\n",
      "Using motif -srebpf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hinfp of width 12.\n",
      "Using motif -hinfp of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f8 of width 12.\n",
      "Using motif -e2f8 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esr2 of width 15.\n",
      "Using motif -esr2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Using motif +mecom of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -mecom of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +six3 of width 17.\n",
      "Using motif -six3 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +insm1 of width 12.\n",
      "Using motif -insm1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax6 of width 14.\n",
      "Using motif -pax6 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif -znf410 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1 of width 10.\n",
      "Using motif -gfi1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ap-1 of width 7.\n",
      "Using motif -ap-1 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid3b of width 11.\n",
      "Using motif -arid3b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +thap1 of width 9.\n",
      "Using motif -thap1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfat of width 10.\n",
      "Using motif -nfat of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myb of width 10.\n",
      "Using motif -myb of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4a of width 16.\n",
      "Using motif -hnf4a of width 16.\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfya of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nfya of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prdm1 of width 15.\n",
      "Using motif -prdm1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf2 of width 18.\n",
      "Using motif -irf2 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myc of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Using motif +hsf of width 13.\n",
      "Using motif -hsf of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa11 of width 12.\n",
      "Using motif -hoxa11 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gli2 of width 12.\n",
      "Using motif -gli2 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif -prox1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zfx of width 14.\n",
      "Using motif -zfx of width 14.\n",
      "Using motif +rreb1 of width 20.\n",
      "Using motif -rreb1 of width 20.\n",
      "Using motif +rest of width 21.\n",
      "Using motif -rest of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif -yy2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e3 of width 7.\n",
      "Using motif -nr2e3 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +t of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -t of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif -bcl6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf1 of width 13.\n",
      "Using motif -hnf1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl1 of width 12.\n",
      "Using motif -mybl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spz1 of width 11.\n",
      "Using motif -spz1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ctcf of width 19.\n",
      "Using motif -ctcf of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hes of width 12.\n",
      "Using motif -hes of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +hoxa5 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfkb of width 13.\n",
      "Using motif -nfkb of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Using motif +nr2f1 of width 13.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +roravar of width 14.\n",
      "Using motif -roravar of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -mybl2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f1 of width 12.\n",
      "Using motif -e2f1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb33 of width 15.\n",
      "Using motif -zbtb33 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pbx1 of width 12.\n",
      "Using motif -pbx1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmbx1 of width 17.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf143 of width 16.\n",
      "Using motif -znf143 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif -mtf1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +scrt of width 13.\n",
      "Using motif -scrt of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox17 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -sox17 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +srf of width 16.\n",
      "Read 1 motifs.\n",
      "Using motif -srf of width 16.\n",
      "Using motif +plag1 of width 14.\n",
      "Using motif -plag1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6b of width 17.\n",
      "Using motif -bcl6b of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dux4 of width 11.\n",
      "Using motif -dux4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +duxa of width 13.\n",
      "Using motif -duxa of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc13 of width 11.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp4 of width 17.\n",
      "Using motif -sp4 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +cdx of width 11.\n",
      "Using motif -cdx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Using motif +e2f7 of width 14.\n",
      "Using motif -e2f7 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rargvar of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -rargvar of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcf21 of width 14.\n",
      "Using motif -tcf21 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Using motif +onecut of width 14.\n",
      "Using motif -onecut of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr5a2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -nr5a2 of width 15.\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +meis of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -meis of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Using motif +ebox of width 11.\n",
      "Using motif +pou1f1 of width 14.\n",
      "Using motif -ebox of width 11.\n",
      "Using motif -pou1f1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mafg of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +rhox11 of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif -mafg of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -rhox11 of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lef1 of width 15.\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Using motif -lef1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif +e2f2 of width 16.\n",
      "Using motif -stat6 of width 15.\n",
      "Using motif -e2f2 of width 16.\n",
      "Using motif +rar_rxr of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf1 of width 21.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -irf1 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cux of width 10.\n",
      "Using motif -cux of width 10.\n",
      "Using motif +usf of width 11.\n",
      "Using motif -usf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +tead of width 8.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -tead of width 8.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cenpb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif -cenpb of width 15.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif +elf of width 12.\n",
      "Using motif -elf of width 12.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +nfyb of width 15.\n",
      "Read 1 motifs.\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -nfyb of width 15.\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -arid5a of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -bhlha15 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esrr of width 10.\n",
      "Using motif +klf of width 11.\n",
      "Using motif -esrr of width 10.\n",
      "Using motif -klf of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rora of width 10.\n",
      "Using motif +nfi of width 6.\n",
      "Using motif -rora of width 10.\n",
      "Using motif -nfi of width 6.\n",
      "Using motif +xbp1 of width 14.\n",
      "Using motif -xbp1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf740 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -znf740 of width 10.\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif -grhl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa9 of width 11.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp53 of width 18.\n",
      "Using motif -tp53 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox8 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -sox8 of width 16.\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nrf1 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zic of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -zic of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf7 of width 12.\n",
      "Using motif -tcf7 of width 12.\n",
      "Using motif +pou6f2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -pou6f2 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +tp63 of width 18.\n",
      "Using motif -tp63 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb18 of width 13.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pparg of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -pparg of width 20.\n",
      "Read 1 motifs.\n",
      "Using motif +pax5 of width 19.\n",
      "Using motif -pax5 of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rel of width 10.\n",
      "Using motif -rel of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esr1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -esr1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfap1 of width 12.\n",
      "Using motif -tfap1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Using motif +cebp of width 10.\n",
      "Using motif -cebp of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif -sox10 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1var of width 10.\n",
      "Using motif -mzf1var of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp1 of width 11.\n",
      "Using motif -sp1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbed1 of width 13.\n",
      "Using motif -zbed1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp73 of width 18.\n",
      "Using motif -tp73 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +glis of width 14.\n",
      "Using motif -glis of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif -hnf4g of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +spi1-c of width 14.\n",
      "Using motif -spi1-c of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif +nfil3 of width 11.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Using motif -nfil3 of width 11.\n",
      "Using motif +rfx of width 16.\n",
      "Using motif -rfx of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +phox2_prop of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -phox2_prop of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar of width 16.\n",
      "Using motif -rar of width 16.\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -znf423 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb33 of width 15.\n",
      "Using motif -zbtb33 of width 15.\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Using motif +insm1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rxravdr of width 15.\n",
      "Using motif -rxravdr of width 15.\n",
      "Using motif -insm1 of width 12.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ctcf of width 19.\n",
      "Using motif -ctcf of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf1 of width 13.\n",
      "Using motif -hnf1 of width 13.\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif -gfi1b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl1 of width 12.\n",
      "Using motif -mybl1 of width 12.\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf263 of width 21.\n",
      "Using motif -znf263 of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax6 of width 14.\n",
      "Using motif -pax6 of width 14.\n",
      "Using motif +rreb1 of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -rreb1 of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -e2f1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf1 of width 21.\n",
      "Using motif -irf1 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +scrt of width 13.\n",
      "Using motif -scrt of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srebpf of width 10.\n",
      "Using motif -srebpf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mafg of width 21.\n",
      "Using motif -mafg of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +thap1 of width 9.\n",
      "Using motif -thap1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfat of width 10.\n",
      "Using motif -nfat of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prdm1 of width 15.\n",
      "Using motif -prdm1 of width 15.\n",
      "Using motif +hes of width 12.\n",
      "Using motif -hes of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hinfp of width 12.\n",
      "Using motif +six3 of width 17.\n",
      "Using motif -hinfp of width 12.\n",
      "Using motif -six3 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ap-1 of width 7.\n",
      "Using motif -ap-1 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pbx1 of width 12.\n",
      "Using motif -pbx1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mecom of width 14.\n",
      "Using motif -mecom of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfya of width 18.\n",
      "Using motif -nfya of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbp of width 15.\n",
      "Using motif -tbp of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2f1 of width 13.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rest of width 21.\n",
      "Using motif -rest of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -arid5a of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myb of width 10.\n",
      "Using motif -myb of width 10.\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif -znf410 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf4a of width 16.\n",
      "Using motif +cenpb of width 15.\n",
      "Using motif -cenpb of width 15.\n",
      "Using motif -hnf4a of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Using motif +roravar of width 14.\n",
      "Using motif -roravar of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gli2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -gli2 of width 12.\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif -prox1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Using motif +myc of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +arid3b of width 11.\n",
      "Using motif -arid3b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gfi1 of width 10.\n",
      "Using motif -gfi1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Using motif +e2f8 of width 12.\n",
      "Using motif -e2f8 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa9 of width 11.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +onecut of width 14.\n",
      "Using motif -onecut of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux4 of width 11.\n",
      "Using motif -dux4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfkb of width 13.\n",
      "Using motif -nfkb of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif -yy2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hic of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hic of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zfx of width 14.\n",
      "Using motif -zfx of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spz1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -spz1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lef1 of width 15.\n",
      "Using motif -lef1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa11 of width 12.\n",
      "Using motif -hoxa11 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hsf of width 13.\n",
      "Using motif -hsf of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +xbp1 of width 14.\n",
      "Using motif -xbp1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -bhlha15 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cux of width 10.\n",
      "Using motif -cux of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf740 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -znf740 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmbx1 of width 17.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif -mtf1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esr2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -esr2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox17 of width 9.\n",
      "Using motif -sox17 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +klf of width 11.\n",
      "Using motif -klf of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +nfyb of width 15.\n",
      "Read 1 motifs.\n",
      "Using motif -nfyb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc13 of width 11.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elf of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -elf of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +plag1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif -plag1 of width 14.\n",
      "Using motif -grhl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +t of width 16.\n",
      "Using motif -t of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e3 of width 7.\n",
      "Using motif -nr2e3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +duxa of width 13.\n",
      "Using motif -duxa of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif +sp4 of width 17.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Using motif -sp4 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srf of width 16.\n",
      "Using motif -srf of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esrr of width 10.\n",
      "Using motif -esrr of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rora of width 10.\n",
      "Using motif -rora of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +tcf21 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +tp63 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -tcf21 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -tp63 of width 18.\n",
      "Using motif -bhlh of width 10.\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Using motif +bcl6b of width 17.\n",
      "Using motif -bcl6b of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf143 of width 16.\n",
      "Using motif -znf143 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebox of width 11.\n",
      "Using motif -ebox of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif -bcl6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf2 of width 18.\n",
      "Using motif -irf2 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Using motif +tp53 of width 18.\n",
      "Using motif -tp53 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rargvar of width 17.\n",
      "Using motif -rargvar of width 17.\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou1f1 of width 14.\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -pou1f1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f2 of width 16.\n",
      "Using motif -e2f2 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nrf1 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rhox11 of width 17.\n",
      "Using motif -rhox11 of width 17.\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +usf of width 11.\n",
      "Using motif -usf of width 11.\n",
      "Using motif +nfi of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -nfi of width 6.\n",
      "Read 1 motifs.\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zic of width 14.\n",
      "Using motif -zic of width 14.\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl2 of width 15.\n",
      "Using motif -mybl2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f7 of width 14.\n",
      "Using motif -e2f7 of width 14.\n",
      "Using motif +esr1 of width 17.\n",
      "Using motif -esr1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cdx of width 11.\n",
      "Using motif -cdx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar_rxr of width 17.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr5a2 of width 15.\n",
      "Using motif -nr5a2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif -sox10 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rel of width 10.\n",
      "Using motif -rel of width 10.\n",
      "Using motif +pou6f2 of width 10.\n",
      "Using motif -pou6f2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf7 of width 12.\n",
      "Using motif -tcf7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif -hnf4g of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rfx of width 16.\n",
      "Using motif -rfx of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb18 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -zbtb18 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfil3 of width 11.\n",
      "Using motif -nfil3 of width 11.\n",
      "Using motif +spi1-c of width 14.\n",
      "Using motif -spi1-c of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp73 of width 18.\n",
      "Using motif -tp73 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1var of width 10.\n",
      "Using motif -mzf1var of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp1 of width 11.\n",
      "Using motif -sp1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pparg of width 20.\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Using motif -pparg of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbed1 of width 13.\n",
      "Using motif -zbed1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +glis of width 14.\n",
      "Using motif -glis of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Using motif +tfap1 of width 12.\n",
      "Using motif -tfap1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +phox2_prop of width 11.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox8 of width 16.\n",
      "Using motif -sox8 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cebp of width 10.\n",
      "Using motif -cebp of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +pax5 of width 19.\n",
      "Using motif -pax5 of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -znf423 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar of width 16.\n",
      "Using motif -rar of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ap-1 of width 7.\n",
      "Using motif -ap-1 of width 7.\n",
      "Using motif +mafg of width 21.\n",
      "Using motif -mafg of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pbx1 of width 12.\n",
      "Using motif -pbx1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf263 of width 21.\n",
      "Using motif -znf263 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prdm1 of width 15.\n",
      "Using motif -prdm1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa9 of width 11.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfkb of width 13.\n",
      "Using motif -nfkb of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif -yy2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rxravdr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -rxravdr of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif -gfi1b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srebpf of width 10.\n",
      "Using motif -srebpf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f1 of width 12.\n",
      "Using motif -e2f1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +scrt of width 13.\n",
      "Using motif -scrt of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf1 of width 13.\n",
      "Using motif -hnf1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rreb1 of width 20.\n",
      "Using motif -rreb1 of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl1 of width 12.\n",
      "Using motif -mybl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf1 of width 21.\n",
      "Using motif -irf1 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mecom of width 14.\n",
      "Using motif -mecom of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zfx of width 14.\n",
      "Using motif -zfx of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dmbx1 of width 17.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox17 of width 9.\n",
      "Using motif -sox17 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lef1 of width 15.\n",
      "Using motif -lef1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf740 of width 10.\n",
      "Using motif -znf740 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +thap1 of width 9.\n",
      "Using motif -thap1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +six3 of width 17.\n",
      "Using motif -six3 of width 17.\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hes of width 12.\n",
      "Using motif -hes of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +duxa of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -duxa of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srf of width 16.\n",
      "Using motif -srf of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6b of width 17.\n",
      "Using motif -bcl6b of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +t of width 16.\n",
      "Using motif +esrr of width 10.\n",
      "Using motif -t of width 16.\n",
      "Using motif -esrr of width 10.\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using motif +znf143 of width 16.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Using motif -znf143 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gsc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -gsc of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +roravar of width 14.\n",
      "Using motif -roravar of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbp of width 15.\n",
      "Using motif -tbp of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +onecut of width 14.\n",
      "Using motif -onecut of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf21 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -tcf21 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gli2 of width 12.\n",
      "Using motif -gli2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif -prox1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rest of width 21.\n",
      "Using motif -rest of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2f1 of width 13.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -arid5a of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cux of width 10.\n",
      "Using motif -cux of width 10.\n",
      "Using motif +myc of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Using motif +ctcf of width 19.\n",
      "Using motif -ctcf of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf410 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -znf410 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cenpb of width 15.\n",
      "Using motif -cenpb of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Using motif +pax6 of width 14.\n",
      "Using motif -pax6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myb of width 10.\n",
      "Using motif -myb of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4a of width 16.\n",
      "Using motif -hnf4a of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f8 of width 12.\n",
      "Using motif -e2f8 of width 12.\n",
      "Using motif +arid3b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -arid3b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfya of width 18.\n",
      "Using motif -nfya of width 18.\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using motif +insm1 of width 12.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Using motif -insm1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux4 of width 11.\n",
      "Using motif -dux4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb33 of width 15.\n",
      "Using motif -zbtb33 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfat of width 10.\n",
      "Using motif -nfat of width 10.\n",
      "Using motif +hinfp of width 12.\n",
      "Using motif -hinfp of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spz1 of width 11.\n",
      "Using motif -spz1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f7 of width 14.\n",
      "Using motif -e2f7 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cdx of width 11.\n",
      "Using motif -cdx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -gfi1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +hsf of width 13.\n",
      "Using motif -hsf of width 13.\n",
      "Using motif +rar_rxr of width 17.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr5a2 of width 15.\n",
      "Using motif +rhox11 of width 17.\n",
      "Using motif -nr5a2 of width 15.\n",
      "Using motif -rhox11 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa11 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -hoxa11 of width 12.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou1f1 of width 14.\n",
      "Using motif -pou1f1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +usf of width 11.\n",
      "Using motif -usf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rargvar of width 17.\n",
      "Using motif -rargvar of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfil3 of width 11.\n",
      "Using motif -nfil3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +phox2_prop of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -phox2_prop of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spi1-c of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -spi1-c of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +glis of width 14.\n",
      "Using motif -glis of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Using motif -bcl6 of width 14.\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif -hnf4g of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -znf423 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rfx of width 16.\n",
      "Using motif -rfx of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rora of width 10.\n",
      "Using motif -rora of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfyb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nfyb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebox of width 11.\n",
      "Using motif -ebox of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp4 of width 17.\n",
      "Using motif -sp4 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif -mtf1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +xbp1 of width 14.\n",
      "Using motif -xbp1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elf of width 12.\n",
      "Using motif -elf of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbed1 of width 13.\n",
      "Using motif -zbed1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -bhlha15 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +klf of width 11.\n",
      "Using motif -klf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb18 of width 13.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif -grhl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc13 of width 11.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zic of width 14.\n",
      "Using motif -zic of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -zbtb7 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +meis of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif -meis of width 8.\n",
      "Using motif +esr2 of width 15.\n",
      "Using motif -esr2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf7 of width 12.\n",
      "Using motif -tcf7 of width 12.\n",
      "Using motif +nr2e3 of width 7.\n",
      "Using motif -nr2e3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +plag1 of width 14.\n",
      "Using motif -plag1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cebp of width 10.\n",
      "Using motif -cebp of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou6f2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pou6f2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp63 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -tp63 of width 18.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl2 of width 15.\n",
      "Using motif -mybl2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox8 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -sox8 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg of width 20.\n",
      "Using motif -pparg of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp73 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -tp73 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Using motif +rel of width 10.\n",
      "Using motif -rel of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sp1 of width 11.\n",
      "Using motif -sp1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif -sox10 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esr1 of width 17.\n",
      "Using motif -esr1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1var of width 10.\n",
      "Using motif -mzf1var of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax5 of width 19.\n",
      "Using motif -pax5 of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -tfap1 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif -rar of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf2 of width 18.\n",
      "Using motif -irf2 of width 18.\n",
      "Using motif +nfi of width 6.\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif +tp53 of width 18.\n",
      "Using motif -nfi of width 6.\n",
      "Using motif -smad3 of width 10.\n",
      "Using motif -tp53 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Using motif +nrf1 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f2 of width 16.\n",
      "Using motif -e2f2 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +six3 of width 17.\n",
      "Using motif -six3 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f1 of width 12.\n",
      "Using motif -e2f1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +thap1 of width 9.\n",
      "Using motif -thap1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cux of width 10.\n",
      "Using motif -cux of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hsf of width 13.\n",
      "Using motif -hsf of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf21 of width 14.\n",
      "Using motif -tcf21 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esrr of width 10.\n",
      "Using motif -esrr of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ap-1 of width 7.\n",
      "Using motif -ap-1 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf143 of width 16.\n",
      "Using motif -znf143 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -arid5a of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rxravdr of width 15.\n",
      "Using motif -rxravdr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myb of width 10.\n",
      "Using motif -myb of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf1 of width 13.\n",
      "Using motif -hnf1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif -znf410 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lef1 of width 15.\n",
      "Using motif -lef1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pbx1 of width 12.\n",
      "Using motif -pbx1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf1 of width 21.\n",
      "Using motif -irf1 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +insm1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -insm1 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfya of width 18.\n",
      "Using motif -nfya of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rreb1 of width 20.\n",
      "Using motif -rreb1 of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Using motif +prdm1 of width 15.\n",
      "Using motif -prdm1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl2 of width 15.\n",
      "Using motif -mybl2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Using motif +mecom of width 14.\n",
      "Using motif -mecom of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mafg of width 21.\n",
      "Using motif -mafg of width 21.\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +xbp1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -xbp1 of width 14.\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp53 of width 18.\n",
      "Using motif -tp53 of width 18.\n",
      "Using motif +nrf1 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Using motif +nr2f1 of width 13.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f2 of width 16.\n",
      "Using motif -e2f2 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rora of width 10.\n",
      "Using motif -rora of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif -gfi1b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spz1 of width 11.\n",
      "Using motif -spz1 of width 11.\n",
      "Using motif +nfkb of width 13.\n",
      "Using motif -nfkb of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Using motif +pax6 of width 14.\n",
      "Using motif -pax6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -dux4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif -yy2 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +tbp of width 15.\n",
      "Using motif +hnf4a of width 16.\n",
      "Using motif -tbp of width 15.\n",
      "Using motif -hnf4a of width 16.\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +roravar of width 14.\n",
      "Using motif -roravar of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cenpb of width 15.\n",
      "Using motif -cenpb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif -prox1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1 of width 10.\n",
      "Using motif -gfi1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +stat1-3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f8 of width 12.\n",
      "Using motif -e2f8 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Using motif +gcm of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -gcm of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +myc of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zfx of width 14.\n",
      "Using motif -zfx of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gli2 of width 12.\n",
      "Using motif -gli2 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid3b of width 11.\n",
      "Using motif -arid3b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rest of width 21.\n",
      "Using motif -rest of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa9 of width 11.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlha15 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -bhlha15 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +esr2 of width 15.\n",
      "Using motif -esr2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfyb of width 15.\n",
      "Using motif -nfyb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif -mtf1 of width 14.\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Using motif +srf of width 16.\n",
      "Using motif -srf of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +nfi of width 6.\n",
      "Using motif -nfi of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nr2e3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ebox of width 11.\n",
      "Using motif +irf2 of width 18.\n",
      "Using motif -ebox of width 11.\n",
      "Using motif -irf2 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +t of width 16.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +grhl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -t of width 16.\n",
      "Using motif -grhl1 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +nr5a2 of width 15.\n",
      "Using motif -nr5a2 of width 15.\n",
      "Using motif +cdx of width 11.\n",
      "Using motif -cdx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif -bcl6 of width 14.\n",
      "Using motif +plag1 of width 14.\n",
      "Using motif +hoxc13 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -plag1 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Using motif +ctcf of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +nfat of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +sp4 of width 17.\n",
      "Using motif -nfat of width 10.\n",
      "Using motif -ctcf of width 19.\n",
      "Using motif -sp4 of width 17.\n",
      "Using motif +duxa of width 13.\n",
      "Using motif +hinfp of width 12.\n",
      "Using motif -duxa of width 13.\n",
      "Using motif -hinfp of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Using motif +bcl6b of width 17.\n",
      "Using motif +e2f7 of width 14.\n",
      "Using motif -e2f7 of width 14.\n",
      "Using motif -bcl6b of width 17.\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif +znf263 of width 21.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Using motif -znf263 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Using motif +elf of width 12.\n",
      "Using motif -elf of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxa11 of width 12.\n",
      "Using motif -hoxa11 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl1 of width 12.\n",
      "Using motif -mybl1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox17 of width 9.\n",
      "Using motif -sox17 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hes of width 12.\n",
      "Using motif -hes of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif +znf740 of width 10.\n",
      "Using motif -sox2 of width 8.\n",
      "Using motif -znf740 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Using motif +scrt of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -scrt of width 13.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +srebpf of width 10.\n",
      "Using motif -srebpf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +onecut of width 14.\n",
      "Using motif -onecut of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmbx1 of width 17.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar_rxr of width 17.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif -sox10 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Using motif +klf of width 11.\n",
      "Using motif -klf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +glis of width 14.\n",
      "Using motif -glis of width 14.\n",
      "Using motif +rargvar of width 17.\n",
      "Using motif -rargvar of width 17.\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Using motif +rhox11 of width 17.\n",
      "Using motif -rhox11 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif -hnf4g of width 15.\n",
      "Using motif +zbtb33 of width 15.\n",
      "Using motif -zbtb33 of width 15.\n",
      "Using motif +pou1f1 of width 14.\n",
      "Using motif -pou1f1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +usf of width 11.\n",
      "Using motif -usf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rfx of width 16.\n",
      "Using motif -rfx of width 16.\n",
      "Using motif +zic of width 14.\n",
      "Using motif -zic of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spi1-c of width 14.\n",
      "Using motif -spi1-c of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfil3 of width 11.\n",
      "Using motif -nfil3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp63 of width 18.\n",
      "Using motif -tp63 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax5 of width 19.\n",
      "Using motif -pax5 of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +phox2_prop of width 11.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou6f2 of width 10.\n",
      "Using motif -pou6f2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +pparg of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -pparg of width 20.\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esr1 of width 17.\n",
      "Using motif -esr1 of width 17.\n",
      "Using motif +mzf1var of width 10.\n",
      "Using motif -mzf1var of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap1 of width 12.\n",
      "Using motif -tfap1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cebp of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -cebp of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbed1 of width 13.\n",
      "Using motif -zbed1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -sp1 of width 11.\n",
      "Using motif +rel of width 10.\n",
      "Using motif -rel of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Read 1 motifs.\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +tcf7 of width 12.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -tcf7 of width 12.\n",
      "Using motif -znf423 of width 15.\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif +zbtb18 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -arid3a of width 6.\n",
      "Read 1 motifs.\n",
      "Using motif +rar of width 16.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Using motif -rar of width 16.\n",
      "Using motif +tp73 of width 18.\n",
      "Using motif +sox8 of width 16.\n",
      "Using motif -sox8 of width 16.\n",
      "Using motif -tp73 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf1 of width 21.\n",
      "Using motif -irf1 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfya of width 18.\n",
      "Using motif -nfya of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pbx1 of width 12.\n",
      "Using motif -pbx1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rxravdr of width 15.\n",
      "Using motif -rxravdr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +six3 of width 17.\n",
      "Using motif -six3 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +thap1 of width 9.\n",
      "Using motif -thap1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mafg of width 21.\n",
      "Using motif -mafg of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf263 of width 21.\n",
      "Using motif -znf263 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mecom of width 14.\n",
      "Using motif -mecom of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prdm1 of width 15.\n",
      "Using motif -prdm1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb33 of width 15.\n",
      "Using motif -zbtb33 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfat of width 10.\n",
      "Using motif -nfat of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif -gfi1b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +srebpf of width 10.\n",
      "Using motif -srebpf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif -prox1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl1 of width 12.\n",
      "Using motif -mybl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf1 of width 13.\n",
      "Using motif -hnf1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +scrt of width 13.\n",
      "Using motif -scrt of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +insm1 of width 12.\n",
      "Using motif -insm1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ctcf of width 19.\n",
      "Using motif -ctcf of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f1 of width 12.\n",
      "Using motif -e2f1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ap-1 of width 7.\n",
      "Using motif -ap-1 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax6 of width 14.\n",
      "Using motif -pax6 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hes of width 12.\n",
      "Using motif -hes of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf4a of width 16.\n",
      "Using motif -hnf4a of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hinfp of width 12.\n",
      "Using motif -hinfp of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myb of width 10.\n",
      "Using motif -myb of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbp of width 15.\n",
      "Using motif -tbp of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2f1 of width 13.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -arid5a of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif -znf410 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Using motif +rreb1 of width 20.\n",
      "Using motif -rreb1 of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rest of width 21.\n",
      "Using motif -rest of width 21.\n",
      "Using motif +cenpb of width 15.\n",
      "Using motif -cenpb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gli2 of width 12.\n",
      "Using motif -gli2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f8 of width 12.\n",
      "Using motif -e2f8 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myc of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1 of width 10.\n",
      "Using motif -gfi1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid3b of width 11.\n",
      "Using motif -arid3b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux4 of width 11.\n",
      "Using motif -dux4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +roravar of width 14.\n",
      "Using motif -roravar of width 14.\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmbx1 of width 17.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox17 of width 9.\n",
      "Using motif -sox17 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +spz1 of width 11.\n",
      "Using motif -spz1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Using motif +hsf of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hsf of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lef1 of width 15.\n",
      "Using motif -lef1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zfx of width 14.\n",
      "Using motif -zfx of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf740 of width 10.\n",
      "Using motif -znf740 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa11 of width 12.\n",
      "Using motif -hoxa11 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfkb of width 13.\n",
      "Using motif -nfkb of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif -yy2 of width 11.\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif -grhl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa9 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hoxa9 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e3 of width 7.\n",
      "Using motif -nr2e3 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc13 of width 11.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Using motif +klf of width 11.\n",
      "Using motif -klf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp4 of width 17.\n",
      "Using motif -sp4 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cux of width 10.\n",
      "Using motif -cux of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -bhlha15 of width 8.\n",
      "Using motif +xbp1 of width 14.\n",
      "Using motif -xbp1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +plag1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -plag1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Using motif +rora of width 10.\n",
      "Using motif -rora of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif +ebox of width 11.\n",
      "Using motif -mtf1 of width 14.\n",
      "Using motif -ebox of width 11.\n",
      "Using motif +nfyb of width 15.\n",
      "Using motif +elf of width 12.\n",
      "Using motif -elf of width 12.\n",
      "Using motif -nfyb of width 15.\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +duxa of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -duxa of width 13.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srf of width 16.\n",
      "Using motif -srf of width 16.\n",
      "Using motif +esr2 of width 15.\n",
      "Using motif -esr2 of width 15.\n",
      "Using motif +bcl6b of width 17.\n",
      "Using motif -bcl6b of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +t of width 16.\n",
      "Using motif -t of width 16.\n",
      "Using motif +znf143 of width 16.\n",
      "Using motif -znf143 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +onecut of width 14.\n",
      "Using motif -onecut of width 14.\n",
      "Using motif +esrr of width 10.\n",
      "Using motif -esrr of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spib of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -spib of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp53 of width 18.\n",
      "Using motif -tp53 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif +tcf21 of width 14.\n",
      "Using motif -tcf21 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl2 of width 15.\n",
      "Using motif -mybl2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +irf2 of width 18.\n",
      "Using motif -irf2 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f2 of width 16.\n",
      "Using motif -e2f2 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nrf1 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou1f1 of width 14.\n",
      "Using motif -pou1f1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfi of width 6.\n",
      "Using motif -nfi of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rargvar of width 17.\n",
      "Using motif -rargvar of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif -bcl6 of width 14.\n",
      "Using motif +e2f7 of width 14.\n",
      "Using motif -e2f7 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rhox11 of width 17.\n",
      "Using motif -rhox11 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr5a2 of width 15.\n",
      "Using motif -nr5a2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar_rxr of width 17.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cdx of width 11.\n",
      "Using motif -cdx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zic of width 14.\n",
      "Using motif -zic of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +usf of width 11.\n",
      "Using motif -usf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou6f2 of width 10.\n",
      "Using motif -pou6f2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp63 of width 18.\n",
      "Using motif -tp63 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf7 of width 12.\n",
      "Using motif -tcf7 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox8 of width 16.\n",
      "Using motif -sox8 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb18 of width 13.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax5 of width 19.\n",
      "Using motif -pax5 of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp73 of width 18.\n",
      "Using motif -tp73 of width 18.\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rel of width 10.\n",
      "Using motif -rel of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbed1 of width 13.\n",
      "Using motif -zbed1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp1 of width 11.\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Using motif -sp1 of width 11.\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif -sox10 of width 6.\n",
      "Using motif +mzf1var of width 10.\n",
      "Using motif -mzf1var of width 10.\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cebp of width 10.\n",
      "Using motif -cebp of width 10.\n",
      "Using motif +esr1 of width 17.\n",
      "Using motif -esr1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pparg of width 20.\n",
      "Using motif -pparg of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap1 of width 12.\n",
      "Using motif -tfap1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar of width 16.\n",
      "Using motif -rar of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +phox2_prop of width 11.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif -hnf4g of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rfx of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -rfx of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +glis of width 14.\n",
      "Using motif -glis of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfil3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nfil3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spi1-c of width 14.\n",
      "Using motif -spi1-c of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -znf423 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif -gfi1b of width 11.\n",
      "Using motif +scrt of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -scrt of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl1 of width 12.\n",
      "Using motif -mybl1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +insm1 of width 12.\n",
      "Using motif -insm1 of width 12.\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prdm1 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -prdm1 of width 15.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf1 of width 21.\n",
      "Using motif -irf1 of width 21.\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rreb1 of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -rreb1 of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +thap1 of width 9.\n",
      "Using motif -thap1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfat of width 10.\n",
      "Using motif -nfat of width 10.\n",
      "Using motif +zbtb33 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -zbtb33 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +six3 of width 17.\n",
      "Using motif -six3 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mafg of width 21.\n",
      "Using motif -mafg of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +hnf1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -hnf1 of width 13.\n",
      "Using motif +ap-1 of width 7.\n",
      "Using motif -ap-1 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hes of width 12.\n",
      "Using motif -hes of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfya of width 18.\n",
      "Using motif -nfya of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2f1 of width 13.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4a of width 16.\n",
      "Using motif -hnf4a of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif -znf410 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +roravar of width 14.\n",
      "Using motif -roravar of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -arid5a of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gli2 of width 12.\n",
      "Using motif +rxravdr of width 15.\n",
      "Using motif -rxravdr of width 15.\n",
      "Using motif -gli2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +tbp of width 15.\n",
      "Using motif -tbp of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srebpf of width 10.\n",
      "Using motif -srebpf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f1 of width 12.\n",
      "Using motif -e2f1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ctcf of width 19.\n",
      "Using motif -ctcf of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pbx1 of width 12.\n",
      "Using motif -pbx1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mecom of width 14.\n",
      "Using motif -mecom of width 14.\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hinfp of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf263 of width 21.\n",
      "Using motif -hinfp of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -znf263 of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif -prox1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cenpb of width 15.\n",
      "Using motif -cenpb of width 15.\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Using motif +myb of width 10.\n",
      "Using motif -myb of width 10.\n",
      "Using motif +rest of width 21.\n",
      "Using motif -rest of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif -yy2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid3b of width 11.\n",
      "Using motif -arid3b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfkb of width 13.\n",
      "Using motif -nfkb of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gfi1 of width 10.\n",
      "Using motif -gfi1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa9 of width 11.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux4 of width 11.\n",
      "Using motif -dux4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +spz1 of width 11.\n",
      "Using motif -spz1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dmbx1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zfx of width 14.\n",
      "Using motif -zfx of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f8 of width 12.\n",
      "Using motif -e2f8 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax6 of width 14.\n",
      "Using motif -pax6 of width 14.\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myc of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +klf of width 11.\n",
      "Using motif -klf of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lef1 of width 15.\n",
      "Using motif -lef1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox17 of width 9.\n",
      "Using motif -sox17 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf740 of width 10.\n",
      "Using motif -znf740 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cux of width 10.\n",
      "Using motif -cux of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -bhlha15 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif +esr2 of width 15.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Using motif -esr2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif -mtf1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Using motif +hoxa11 of width 12.\n",
      "Using motif -hoxa11 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hsf of width 13.\n",
      "Using motif -hsf of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +xbp1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -xbp1 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxd8 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Using motif +srf of width 16.\n",
      "Using motif -srf of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif -grhl1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebox of width 11.\n",
      "Using motif -ebox of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rora of width 10.\n",
      "Using motif -rora of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +plag1 of width 14.\n",
      "Using motif -plag1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elf of width 12.\n",
      "Using motif -elf of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +onecut of width 14.\n",
      "Using motif -onecut of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp4 of width 17.\n",
      "Using motif -sp4 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc13 of width 11.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e3 of width 7.\n",
      "Using motif -nr2e3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfyb of width 15.\n",
      "Using motif -nfyb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +t of width 16.\n",
      "Using motif -t of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf143 of width 16.\n",
      "Using motif -znf143 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +duxa of width 13.\n",
      "Using motif -duxa of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esrr of width 10.\n",
      "Using motif -esrr of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6b of width 17.\n",
      "Using motif -bcl6b of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mybl2 of width 15.\n",
      "Using motif -mybl2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp53 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -tp53 of width 18.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf2 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -irf2 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfi of width 6.\n",
      "Using motif -nfi of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nrf1 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f2 of width 16.\n",
      "Using motif -e2f2 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou1f1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rargvar of width 17.\n",
      "Using motif -pou1f1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -rargvar of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr5a2 of width 15.\n",
      "Using motif -nr5a2 of width 15.\n",
      "Using motif +tcf21 of width 14.\n",
      "Using motif -tcf21 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cdx of width 11.\n",
      "Using motif -cdx of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rhox11 of width 17.\n",
      "Using motif -rhox11 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rar_rxr of width 17.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -sp1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf7 of width 12.\n",
      "Using motif -tcf7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f7 of width 14.\n",
      "Using motif -e2f7 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbed1 of width 13.\n",
      "Using motif -zbed1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +mzf1var of width 10.\n",
      "Using motif -mzf1var of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif -sox10 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +usf of width 11.\n",
      "Using motif -usf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp63 of width 18.\n",
      "Using motif -tp63 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rel of width 10.\n",
      "Using motif -rel of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zic of width 14.\n",
      "Using motif -zic of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou6f2 of width 10.\n",
      "Using motif -pou6f2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esr1 of width 17.\n",
      "Using motif -esr1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox8 of width 16.\n",
      "Using motif -sox8 of width 16.\n",
      "Using motif +nfil3 of width 11.\n",
      "Using motif -nfil3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +phox2_prop of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cebp of width 10.\n",
      "Using motif -cebp of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +glis of width 14.\n",
      "Using motif -glis of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rfx of width 16.\n",
      "Using motif -rfx of width 16.\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb18 of width 13.\n",
      "Using motif +pax5 of width 19.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Using motif -pax5 of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spi1-c of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -spi1-c of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap1 of width 12.\n",
      "Using motif -tfap1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp73 of width 18.\n",
      "Using motif -tp73 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pparg of width 20.\n",
      "Using motif -pparg of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif -hnf4g of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rar of width 16.\n",
      "Using motif -rar of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -znf423 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif -bcl6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hes of width 12.\n",
      "Using motif -hes of width 12.\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +pbx1 of width 12.\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif +rxravdr of width 15.\n",
      "Using motif -pbx1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -gfi1b of width 11.\n",
      "Using motif -rxravdr of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ap-1 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -ap-1 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif +six3 of width 17.\n",
      "Using motif -six3 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -dmrt3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mecom of width 14.\n",
      "Using motif -mecom of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prdm1 of width 15.\n",
      "Using motif -prdm1 of width 15.\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Using motif +rest of width 21.\n",
      "Using motif -rest of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +insm1 of width 12.\n",
      "Using motif -insm1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfya of width 18.\n",
      "Using motif -nfya of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +runx of width 11.\n",
      "Using motif -runx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Using motif +e2f1 of width 12.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Using motif -e2f1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +crem of width 10.\n",
      "Using motif -crem of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl1 of width 12.\n",
      "Using motif -mybl1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +scrt of width 13.\n",
      "Using motif -scrt of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srebpf of width 10.\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Using motif -srebpf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rreb1 of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -rreb1 of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hnf1 of width 13.\n",
      "Using motif +egr of width 11.\n",
      "Using motif -egr of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf4a of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hnf4a of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myb of width 10.\n",
      "Using motif -myb of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +thap1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -thap1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbp of width 15.\n",
      "Using motif -tbp of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ctcf of width 19.\n",
      "Using motif -ctcf of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf1 of width 21.\n",
      "Using motif -irf1 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +roravar of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -roravar of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf263 of width 21.\n",
      "Using motif -znf263 of width 21.\n",
      "Using motif +mafg of width 21.\n",
      "Using motif -mafg of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfat of width 10.\n",
      "Using motif -nfat of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esr1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -esr1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hinfp of width 12.\n",
      "Using motif -hinfp of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +zbtb33 of width 15.\n",
      "Using motif -zbtb33 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid5a of width 14.\n",
      "Using motif -arid5a of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2f1 of width 13.\n",
      "Using motif -nr2f1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif -znf410 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -elk_etv of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif -prox1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gli2 of width 12.\n",
      "Using motif -gli2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid3b of width 11.\n",
      "Using motif -arid3b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfkb of width 13.\n",
      "Using motif -nfkb of width 13.\n",
      "Using motif +myc of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dmbx1 of width 17.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy2 of width 11.\n",
      "Using motif -yy2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spz1 of width 11.\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Using motif -spz1 of width 11.\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Using motif +hic of width 9.\n",
      "Using motif -hic of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif +mtf1 of width 14.\n",
      "Using motif -mtf1 of width 14.\n",
      "Using motif -pou4 of width 16.\n",
      "Using motif +dux4 of width 11.\n",
      "Using motif -dux4 of width 11.\n",
      "Using motif +pax6 of width 14.\n",
      "Using motif -pax6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lef1 of width 15.\n",
      "Using motif +hoxa11 of width 12.\n",
      "Using motif -lef1 of width 15.\n",
      "Using motif -hoxa11 of width 12.\n",
      "Using motif +zfx of width 14.\n",
      "Using motif -zfx of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox17 of width 9.\n",
      "Using motif -sox17 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax2 of width 8.\n",
      "Using motif -pax2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +cux of width 10.\n",
      "Using motif -cux of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1 of width 10.\n",
      "Using motif -gfi1 of width 10.\n",
      "Using motif +e2f8 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -e2f8 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxd8 of width 17.\n",
      "Using motif -hoxd8 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa9 of width 11.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Using motif +t of width 16.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Using motif -t of width 16.\n",
      "Using motif +xbp1 of width 14.\n",
      "Using motif -xbp1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hsf of width 13.\n",
      "Using motif -hsf of width 13.\n",
      "Using motif +cenpb of width 15.\n",
      "Using motif -cenpb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esr2 of width 15.\n",
      "Using motif -esr2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +bcl6b of width 17.\n",
      "Using motif -bcl6b of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf21 of width 14.\n",
      "Using motif -tcf21 of width 14.\n",
      "Using motif +znf143 of width 16.\n",
      "Using motif -znf143 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +klf of width 11.\n",
      "Using motif -klf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +lhx3 of width 13.\n",
      "Using motif -lhx3 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +onecut of width 14.\n",
      "Using motif -onecut of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -mzf1 of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nr2e3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esrr of width 10.\n",
      "Using motif -esrr of width 10.\n",
      "Using motif +hoxc13 of width 11.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfyb of width 15.\n",
      "Using motif -nfyb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl2 of width 15.\n",
      "Using motif -mybl2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srf of width 16.\n",
      "Using motif -srf of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +sp4 of width 17.\n",
      "Using motif -sp4 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +plag1 of width 14.\n",
      "Using motif -plag1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebox of width 11.\n",
      "Using motif -ebox of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -bhlha15 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elf of width 12.\n",
      "Using motif -elf of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Using motif +rora of width 10.\n",
      "Using motif -rora of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif -grhl1 of width 12.\n",
      "Using motif +duxa of width 13.\n",
      "Using motif -duxa of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf2 of width 18.\n",
      "Using motif -irf2 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux of width 8.\n",
      "Using motif -dux of width 8.\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfi of width 6.\n",
      "Using motif -nfi of width 6.\n",
      "Using motif +tp53 of width 18.\n",
      "Using motif -tp53 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif -bcl6 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f2 of width 16.\n",
      "Using motif -e2f2 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nrf1 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rargvar of width 17.\n",
      "Using motif -rargvar of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nr5a2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -nr5a2 of width 15.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cdx of width 11.\n",
      "Using motif -cdx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou1f1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pou1f1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +znf740 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif -znf740 of width 10.\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f7 of width 14.\n",
      "Using motif -e2f7 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp63 of width 18.\n",
      "Using motif -tp63 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-2 of width 11.\n",
      "Using motif -homeobox-2 of width 11.\n",
      "Using motif +zic of width 14.\n",
      "Using motif -zic of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -hltf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +rar_rxr of width 17.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcf7 of width 12.\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif -tcf7 of width 12.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gata of width 9.\n",
      "Using motif -gata of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pou6f2 of width 10.\n",
      "Using motif -pou6f2 of width 10.\n",
      "Using motif +mzf1var of width 10.\n",
      "Using motif -mzf1var of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rel of width 10.\n",
      "Using motif -rel of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +usf of width 11.\n",
      "Using motif -usf of width 11.\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -sox10 of width 6.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spi1-c of width 14.\n",
      "Using motif -spi1-c of width 14.\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax5 of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +cebp of width 10.\n",
      "Using motif +zbtb18 of width 13.\n",
      "Using motif +rhox11 of width 17.\n",
      "Using motif -cebp of width 10.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Using motif -rhox11 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Using motif -pax5 of width 19.\n",
      "Using motif +sp1 of width 11.\n",
      "Using motif -sp1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif +pparg of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Using motif +rfx of width 16.\n",
      "Using motif -pparg of width 20.\n",
      "Using motif -rfx of width 16.\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp73 of width 18.\n",
      "Using motif -tp73 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -znf423 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfil3 of width 11.\n",
      "Using motif -nfil3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar of width 16.\n",
      "Using motif +phox2_prop of width 11.\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Using motif -rar of width 16.\n",
      "Using motif -hnf4g of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +glis of width 14.\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Using motif -glis of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif -yy1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tfap1 of width 12.\n",
      "Using motif -tfap1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +zbed1 of width 13.\n",
      "Read 1 motifs.\n",
      "Using motif -zbed1 of width 13.\n",
      "Using motif +sox8 of width 16.\n",
      "Using motif -sox8 of width 16.\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "chmod a+x ./scanMotifs*.sh\n",
    "for i in ./scanMotifs*sh; \n",
    "    do echo 'sleeping...';\n",
    "    echo $i;\n",
    "    $i;\n",
    "    sleep 5m;\n",
    "done\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Read in Score Files"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Motif Scores"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "chr1 2 alx1_alx4_arx.fimo\n",
      "chr1 3 ap-1.fimo\n",
      "chr1 4 ar_nr3c1_nr3c2.fimo\n",
      "chr1 5 arid3a.fimo\n",
      "chr1 6 arid3b.fimo\n",
      "chr1 7 arid5a.fimo\n",
      "chr1 8 arnt_mycn.fimo\n",
      "chr1 9 arntl_mitf.fimo\n",
      "chr1 10 ascl2_nhlh1.fimo\n",
      "chr1 11 atf7_batf3_creb5.fimo\n",
      "chr1 12 bcl6.fimo\n",
      "chr1 13 bcl6b.fimo\n",
      "chr1 14 bhlh.fimo\n",
      "chr1 15 bhlha15.fimo\n",
      "chr1 16 bhlhe23.fimo\n",
      "chr1 17 cdx.fimo\n",
      "chr1 18 cebp.fimo\n",
      "chr1 19 cenpb.fimo\n",
      "chr1 20 creb3-l1.fimo\n",
      "chr1 21 crem.fimo\n",
      "chr1 22 ctcf.fimo\n",
      "chr1 23 cux.fimo\n",
      "chr1 24 dbp_hlf_tef.fimo\n",
      "chr1 25 dmbx1.fimo\n",
      "chr1 26 dmrt3.fimo\n",
      "chr1 27 dux.fimo\n",
      "chr1 28 dux4.fimo\n",
      "chr1 29 duxa.fimo\n",
      "chr1 30 e2f1.fimo\n",
      "chr1 31 e2f2.fimo\n",
      "chr1 32 e2f4_e2f6.fimo\n",
      "chr1 33 e2f7.fimo\n",
      "chr1 34 e2f8.fimo\n",
      "chr1 35 ebf1.fimo\n",
      "chr1 36 ebox.fimo\n",
      "chr1 37 egr.fimo\n",
      "chr1 38 elf.fimo\n",
      "chr1 39 elk_etv.fimo\n",
      "chr1 40 esr1.fimo\n",
      "chr1 41 esr2.fimo\n",
      "chr1 42 esrr.fimo\n",
      "chr1 43 ewsr1-fli1.fimo\n",
      "chr1 44 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr1 45 fox.fimo\n",
      "chr1 46 gata.fimo\n",
      "chr1 47 gcm.fimo\n",
      "chr1 48 gfi1.fimo\n",
      "chr1 49 gfi1b.fimo\n",
      "chr1 50 gli2.fimo\n",
      "chr1 51 glis.fimo\n",
      "chr1 52 gmeb2.fimo\n",
      "chr1 53 grhl1.fimo\n",
      "chr1 54 gsc.fimo\n",
      "chr1 55 hes.fimo\n",
      "chr1 56 hic.fimo\n",
      "chr1 57 hinfp.fimo\n",
      "chr1 58 hltf.fimo\n",
      "chr1 59 hnf1.fimo\n",
      "chr1 60 hnf4a.fimo\n",
      "chr1 61 hnf4g.fimo\n",
      "chr1 62 homeobox-1.fimo\n",
      "chr1 63 homeobox-2.fimo\n",
      "chr1 64 homeobox-3.fimo\n",
      "chr1 65 homeobox-4.fimo\n",
      "chr1 66 homeobox-5.fimo\n",
      "chr1 67 hoxa11.fimo\n",
      "chr1 68 hoxa5.fimo\n",
      "chr1 69 hoxa9.fimo\n",
      "chr1 70 hoxb5_hoxd3.fimo\n",
      "chr1 71 hoxc13.fimo\n",
      "chr1 72 hoxc9.fimo\n",
      "chr1 73 hoxd8.fimo\n",
      "chr1 74 hsf.fimo\n",
      "chr1 75 id2_mlxip.fimo\n",
      "chr1 76 insm1.fimo\n",
      "chr1 77 irf1.fimo\n",
      "chr1 78 irf2.fimo\n",
      "chr1 79 irf7-8-9.fimo\n",
      "chr1 80 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr1 81 klf.fimo\n",
      "chr1 82 lef1.fimo\n",
      "chr1 83 lhx3.fimo\n",
      "chr1 84 lin54.fimo\n",
      "chr1 85 maf_nrl.fimo\n",
      "chr1 86 mafg.fimo\n",
      "chr1 87 mecom.fimo\n",
      "chr1 88 mef2a-b-d.fimo\n",
      "chr1 89 mef2c.fimo\n",
      "chr1 90 meis.fimo\n",
      "chr1 91 mga_tbx.fimo\n",
      "chr1 92 msc_myf6_tfap4.fimo\n",
      "chr1 93 mtf1.fimo\n",
      "chr1 94 myb.fimo\n",
      "chr1 95 mybl1.fimo\n",
      "chr1 96 mybl2.fimo\n",
      "chr1 97 myc.fimo\n",
      "chr1 98 myog_tcf12.fimo\n",
      "chr1 99 mzf1.fimo\n",
      "chr1 100 mzf1var.2.fimo\n",
      "chr1 101 nfat.fimo\n",
      "chr1 102 nfi.fimo\n",
      "chr1 103 nfil3.fimo\n",
      "chr1 104 nfkb.fimo\n",
      "chr1 105 nfya.fimo\n",
      "chr1 106 nfyb.fimo\n",
      "chr1 107 nkx2-3_nkx2-8.fimo\n",
      "chr1 108 nkx2-5.fimo\n",
      "chr1 109 nkx2-5var.2.fimo\n",
      "chr1 110 nr2_rxr.fimo\n",
      "chr1 111 nr2e1.fimo\n",
      "chr1 112 nr2e3.fimo\n",
      "chr1 113 nr2f1.fimo\n",
      "chr1 114 nr4a2.fimo\n",
      "chr1 115 nr5a2.fimo\n",
      "chr1 116 nrf1.fimo\n",
      "chr1 117 onecut.fimo\n",
      "chr1 118 pax1_pax9.fimo\n",
      "chr1 119 pax2.fimo\n",
      "chr1 120 pax3_pax7.fimo\n",
      "chr1 121 pax5.fimo\n",
      "chr1 122 pax6.fimo\n",
      "chr1 123 pbx1.fimo\n",
      "chr1 124 phox2_prop.fimo\n",
      "chr1 125 pknox_tgif.fimo\n",
      "chr1 126 plag1.fimo\n",
      "chr1 127 pou1f1.fimo\n",
      "chr1 128 pou2-3-5.fimo\n",
      "chr1 129 pou4.fimo\n",
      "chr1 130 pou6f2.fimo\n",
      "chr1 131 pparg.fimo\n",
      "chr1 132 pparg_rxra.fimo\n",
      "chr1 133 prdm1.fimo\n",
      "chr1 134 prox1.fimo\n",
      "chr1 135 rar.fimo\n",
      "chr1 136 rar_rxr.fimo\n",
      "chr1 137 rargvar.2.fimo\n",
      "chr1 138 rel.fimo\n",
      "chr1 139 rest.fimo\n",
      "chr1 140 rfx.fimo\n",
      "chr1 141 rhox11.fimo\n",
      "chr1 142 rora.fimo\n",
      "chr1 143 roravar.2.fimo\n",
      "chr1 144 rreb1.fimo\n",
      "chr1 145 runx.fimo\n",
      "chr1 146 rxravdr.fimo\n",
      "chr1 147 scrt.fimo\n",
      "chr1 148 six3.fimo\n",
      "chr1 149 smad3.fimo\n",
      "chr1 150 sox1-l1-21-4.fimo\n",
      "chr1 151 sox10.fimo\n",
      "chr1 152 sox17.fimo\n",
      "chr1 153 sox2.fimo\n",
      "chr1 154 sox3_sox6.fimo\n",
      "chr1 155 sox5_sox9_sry.fimo\n",
      "chr1 156 sox8.fimo\n",
      "chr1 157 sp1.fimo\n",
      "chr1 158 sp4.fimo\n",
      "chr1 159 spi1-c.fimo\n",
      "chr1 160 spib.fimo\n",
      "chr1 161 spz1.fimo\n",
      "chr1 162 srebpf.fimo\n",
      "chr1 163 srf.fimo\n",
      "chr1 164 stat1-3.fimo\n",
      "chr1 165 stat6.fimo\n",
      "chr1 166 t.fimo\n",
      "chr1 167 tbp.fimo\n",
      "chr1 168 tbx.fimo\n",
      "chr1 169 tcf21.fimo\n",
      "chr1 170 tcf7.fimo\n",
      "chr1 171 tcfl5.fimo\n",
      "chr1 172 tead.fimo\n",
      "chr1 173 tfap1.fimo\n",
      "chr1 174 tfap2-3.fimo\n",
      "chr1 175 tfcp2.fimo\n",
      "chr1 176 thap1.fimo\n",
      "chr1 177 tp53.fimo\n",
      "chr1 178 tp63.fimo\n",
      "chr1 179 tp73.fimo\n",
      "chr1 180 usf.fimo\n",
      "chr1 181 ventx.fimo\n",
      "chr1 182 xbp1.fimo\n",
      "chr1 183 yy1.fimo\n",
      "chr1 184 yy2.fimo\n",
      "chr1 185 zbed1.fimo\n",
      "chr1 186 zbtb18.fimo\n",
      "chr1 187 zbtb33.fimo\n",
      "chr1 188 zbtb7.fimo\n",
      "chr1 189 zeb1.fimo\n",
      "chr1 190 zfx.fimo\n",
      "chr1 191 zic.fimo\n",
      "chr1 192 znf143.fimo\n",
      "chr1 193 znf263.fimo\n",
      "chr1 194 znf354c.fimo\n",
      "chr1 195 znf410.fimo\n",
      "chr1 196 znf423.fimo\n",
      "chr1 197 znf740.fimo\n",
      "chr10 2 alx1_alx4_arx.fimo\n",
      "chr10 3 ap-1.fimo\n",
      "chr10 4 ar_nr3c1_nr3c2.fimo\n",
      "chr10 5 arid3a.fimo\n",
      "chr10 6 arid3b.fimo\n",
      "chr10 7 arid5a.fimo\n",
      "chr10 8 arnt_mycn.fimo\n",
      "chr10 9 arntl_mitf.fimo\n",
      "chr10 10 ascl2_nhlh1.fimo\n",
      "chr10 11 atf7_batf3_creb5.fimo\n",
      "chr10 12 bcl6.fimo\n",
      "chr10 13 bcl6b.fimo\n",
      "chr10 14 bhlh.fimo\n",
      "chr10 15 bhlha15.fimo\n",
      "chr10 16 bhlhe23.fimo\n",
      "chr10 17 cdx.fimo\n",
      "chr10 18 cebp.fimo\n",
      "chr10 19 cenpb.fimo\n",
      "chr10 20 creb3-l1.fimo\n",
      "chr10 21 crem.fimo\n",
      "chr10 22 ctcf.fimo\n",
      "chr10 23 cux.fimo\n",
      "chr10 24 dbp_hlf_tef.fimo\n",
      "chr10 25 dmbx1.fimo\n",
      "chr10 26 dmrt3.fimo\n",
      "chr10 27 dux.fimo\n",
      "chr10 28 dux4.fimo\n",
      "chr10 29 duxa.fimo\n",
      "chr10 30 e2f1.fimo\n",
      "chr10 31 e2f2.fimo\n",
      "chr10 32 e2f4_e2f6.fimo\n",
      "chr10 33 e2f7.fimo\n",
      "chr10 34 e2f8.fimo\n",
      "chr10 35 ebf1.fimo\n",
      "chr10 36 ebox.fimo\n",
      "chr10 37 egr.fimo\n",
      "chr10 38 elf.fimo\n",
      "chr10 39 elk_etv.fimo\n",
      "chr10 40 esr1.fimo\n",
      "chr10 41 esr2.fimo\n",
      "chr10 42 esrr.fimo\n",
      "chr10 43 ewsr1-fli1.fimo\n",
      "chr10 44 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr10 45 fox.fimo\n",
      "chr10 46 gata.fimo\n",
      "chr10 47 gcm.fimo\n",
      "chr10 48 gfi1.fimo\n",
      "chr10 49 gfi1b.fimo\n",
      "chr10 50 gli2.fimo\n",
      "chr10 51 glis.fimo\n",
      "chr10 52 gmeb2.fimo\n",
      "chr10 53 grhl1.fimo\n",
      "chr10 54 gsc.fimo\n",
      "chr10 55 hes.fimo\n",
      "chr10 56 hic.fimo\n",
      "chr10 57 hinfp.fimo\n",
      "chr10 58 hltf.fimo\n",
      "chr10 59 hnf1.fimo\n",
      "chr10 60 hnf4a.fimo\n",
      "chr10 61 hnf4g.fimo\n",
      "chr10 62 homeobox-1.fimo\n",
      "chr10 63 homeobox-2.fimo\n",
      "chr10 64 homeobox-3.fimo\n",
      "chr10 65 homeobox-4.fimo\n",
      "chr10 66 homeobox-5.fimo\n",
      "chr10 67 hoxa11.fimo\n",
      "chr10 68 hoxa5.fimo\n",
      "chr10 69 hoxa9.fimo\n",
      "chr10 70 hoxb5_hoxd3.fimo\n",
      "chr10 71 hoxc13.fimo\n",
      "chr10 72 hoxc9.fimo\n",
      "chr10 73 hoxd8.fimo\n",
      "chr10 74 hsf.fimo\n",
      "chr10 75 id2_mlxip.fimo\n",
      "chr10 76 insm1.fimo\n",
      "chr10 77 irf1.fimo\n",
      "chr10 78 irf2.fimo\n",
      "chr10 79 irf7-8-9.fimo\n",
      "chr10 80 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr10 81 klf.fimo\n",
      "chr10 82 lef1.fimo\n",
      "chr10 83 lhx3.fimo\n",
      "chr10 84 lin54.fimo\n",
      "chr10 85 maf_nrl.fimo\n",
      "chr10 86 mafg.fimo\n",
      "chr10 87 mecom.fimo\n",
      "chr10 88 mef2a-b-d.fimo\n",
      "chr10 89 mef2c.fimo\n",
      "chr10 90 meis.fimo\n",
      "chr10 91 mga_tbx.fimo\n",
      "chr10 92 msc_myf6_tfap4.fimo\n",
      "chr10 93 mtf1.fimo\n",
      "chr10 94 myb.fimo\n",
      "chr10 95 mybl1.fimo\n",
      "chr10 96 mybl2.fimo\n",
      "chr10 97 myc.fimo\n",
      "chr10 98 myog_tcf12.fimo\n",
      "chr10 99 mzf1.fimo\n",
      "chr10 100 mzf1var.2.fimo\n",
      "chr10 101 nfat.fimo\n",
      "chr10 102 nfi.fimo\n",
      "chr10 103 nfil3.fimo\n",
      "chr10 104 nfkb.fimo\n",
      "chr10 105 nfya.fimo\n",
      "chr10 106 nfyb.fimo\n",
      "chr10 107 nkx2-3_nkx2-8.fimo\n",
      "chr10 108 nkx2-5.fimo\n",
      "chr10 109 nkx2-5var.2.fimo\n",
      "chr10 110 nr2_rxr.fimo\n",
      "chr10 111 nr2e1.fimo\n",
      "chr10 112 nr2e3.fimo\n",
      "chr10 113 nr2f1.fimo\n",
      "chr10 114 nr4a2.fimo\n",
      "chr10 115 nr5a2.fimo\n",
      "chr10 116 nrf1.fimo\n",
      "chr10 117 onecut.fimo\n",
      "chr10 118 pax1_pax9.fimo\n",
      "chr10 119 pax2.fimo\n",
      "chr10 120 pax3_pax7.fimo\n",
      "chr10 121 pax5.fimo\n",
      "chr10 122 pax6.fimo\n",
      "chr10 123 pbx1.fimo\n",
      "chr10 124 phox2_prop.fimo\n",
      "chr10 125 pknox_tgif.fimo\n",
      "chr10 126 plag1.fimo\n",
      "chr10 127 pou1f1.fimo\n",
      "chr10 128 pou2-3-5.fimo\n",
      "chr10 129 pou4.fimo\n",
      "chr10 130 pou6f2.fimo\n",
      "chr10 131 pparg.fimo\n",
      "chr10 132 pparg_rxra.fimo\n",
      "chr10 133 prdm1.fimo\n",
      "chr10 134 prox1.fimo\n",
      "chr10 135 rar.fimo\n",
      "chr10 136 rar_rxr.fimo\n",
      "chr10 137 rargvar.2.fimo\n",
      "chr10 138 rel.fimo\n",
      "chr10 139 rest.fimo\n",
      "chr10 140 rfx.fimo\n",
      "chr10 141 rhox11.fimo\n",
      "chr10 142 rora.fimo\n",
      "chr10 143 roravar.2.fimo\n",
      "chr10 144 rreb1.fimo\n",
      "chr10 145 runx.fimo\n",
      "chr10 146 rxravdr.fimo\n",
      "chr10 147 scrt.fimo\n",
      "chr10 148 six3.fimo\n",
      "chr10 149 smad3.fimo\n",
      "chr10 150 sox1-l1-21-4.fimo\n",
      "chr10 151 sox10.fimo\n",
      "chr10 152 sox17.fimo\n",
      "chr10 153 sox2.fimo\n",
      "chr10 154 sox3_sox6.fimo\n",
      "chr10 155 sox5_sox9_sry.fimo\n",
      "chr10 156 sox8.fimo\n",
      "chr10 157 sp1.fimo\n",
      "chr10 158 sp4.fimo\n",
      "chr10 159 spi1-c.fimo\n",
      "chr10 160 spib.fimo\n",
      "chr10 161 spz1.fimo\n",
      "chr10 162 srebpf.fimo\n",
      "chr10 163 srf.fimo\n",
      "chr10 164 stat1-3.fimo\n",
      "chr10 165 stat6.fimo\n",
      "chr10 166 t.fimo\n",
      "chr10 167 tbp.fimo\n",
      "chr10 168 tbx.fimo\n",
      "chr10 169 tcf21.fimo\n",
      "chr10 170 tcf7.fimo\n",
      "chr10 171 tcfl5.fimo\n",
      "chr10 172 tead.fimo\n",
      "chr10 173 tfap1.fimo\n",
      "chr10 174 tfap2-3.fimo\n",
      "chr10 175 tfcp2.fimo\n",
      "chr10 176 thap1.fimo\n",
      "chr10 177 tp53.fimo\n",
      "chr10 178 tp63.fimo\n",
      "chr10 179 tp73.fimo\n",
      "chr10 180 usf.fimo\n",
      "chr10 181 ventx.fimo\n",
      "chr10 182 xbp1.fimo\n",
      "chr10 183 yy1.fimo\n",
      "chr10 184 yy2.fimo\n",
      "chr10 185 zbed1.fimo\n",
      "chr10 186 zbtb18.fimo\n",
      "chr10 187 zbtb33.fimo\n",
      "chr10 188 zbtb7.fimo\n",
      "chr10 189 zeb1.fimo\n",
      "chr10 190 zfx.fimo\n",
      "chr10 191 zic.fimo\n",
      "chr10 192 znf143.fimo\n",
      "chr10 193 znf263.fimo\n",
      "chr10 194 znf354c.fimo\n",
      "chr10 195 znf410.fimo\n",
      "chr10 196 znf423.fimo\n",
      "chr10 197 znf740.fimo\n",
      "chr11 2 alx1_alx4_arx.fimo\n",
      "chr11 3 ap-1.fimo\n",
      "chr11 4 ar_nr3c1_nr3c2.fimo\n",
      "chr11 5 arid3a.fimo\n",
      "chr11 6 arid3b.fimo\n",
      "chr11 7 arid5a.fimo\n",
      "chr11 8 arnt_mycn.fimo\n",
      "chr11 9 arntl_mitf.fimo\n",
      "chr11 10 ascl2_nhlh1.fimo\n",
      "chr11 11 atf7_batf3_creb5.fimo\n",
      "chr11 12 bcl6.fimo\n",
      "chr11 13 bcl6b.fimo\n",
      "chr11 14 bhlh.fimo\n",
      "chr11 15 bhlha15.fimo\n",
      "chr11 16 bhlhe23.fimo\n",
      "chr11 17 cdx.fimo\n",
      "chr11 18 cebp.fimo\n",
      "chr11 19 cenpb.fimo\n",
      "chr11 20 creb3-l1.fimo\n",
      "chr11 21 crem.fimo\n",
      "chr11 22 ctcf.fimo\n",
      "chr11 23 cux.fimo\n",
      "chr11 24 dbp_hlf_tef.fimo\n",
      "chr11 25 dmbx1.fimo\n",
      "chr11 26 dmrt3.fimo\n",
      "chr11 27 dux.fimo\n",
      "chr11 28 dux4.fimo\n",
      "chr11 29 duxa.fimo\n",
      "chr11 30 e2f1.fimo\n",
      "chr11 31 e2f2.fimo\n",
      "chr11 32 e2f4_e2f6.fimo\n",
      "chr11 33 e2f7.fimo\n",
      "chr11 34 e2f8.fimo\n",
      "chr11 35 ebf1.fimo\n",
      "chr11 36 ebox.fimo\n",
      "chr11 37 egr.fimo\n",
      "chr11 38 elf.fimo\n",
      "chr11 39 elk_etv.fimo\n",
      "chr11 40 esr1.fimo\n",
      "chr11 41 esr2.fimo\n",
      "chr11 42 esrr.fimo\n",
      "chr11 43 ewsr1-fli1.fimo\n",
      "chr11 44 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr11 45 fox.fimo\n",
      "chr11 46 gata.fimo\n",
      "chr11 47 gcm.fimo\n",
      "chr11 48 gfi1.fimo\n",
      "chr11 49 gfi1b.fimo\n",
      "chr11 50 gli2.fimo\n",
      "chr11 51 glis.fimo\n",
      "chr11 52 gmeb2.fimo\n",
      "chr11 53 grhl1.fimo\n",
      "chr11 54 gsc.fimo\n",
      "chr11 55 hes.fimo\n",
      "chr11 56 hic.fimo\n",
      "chr11 57 hinfp.fimo\n",
      "chr11 58 hltf.fimo\n",
      "chr11 59 hnf1.fimo\n",
      "chr11 60 hnf4a.fimo\n",
      "chr11 61 hnf4g.fimo\n",
      "chr11 62 homeobox-1.fimo\n",
      "chr11 63 homeobox-2.fimo\n",
      "chr11 64 homeobox-3.fimo\n",
      "chr11 65 homeobox-4.fimo\n",
      "chr11 66 homeobox-5.fimo\n",
      "chr11 67 hoxa11.fimo\n",
      "chr11 68 hoxa5.fimo\n",
      "chr11 69 hoxa9.fimo\n",
      "chr11 70 hoxb5_hoxd3.fimo\n",
      "chr11 71 hoxc13.fimo\n",
      "chr11 72 hoxc9.fimo\n",
      "chr11 73 hoxd8.fimo\n",
      "chr11 74 hsf.fimo\n",
      "chr11 75 id2_mlxip.fimo\n",
      "chr11 76 insm1.fimo\n",
      "chr11 77 irf1.fimo\n",
      "chr11 78 irf2.fimo\n",
      "chr11 79 irf7-8-9.fimo\n",
      "chr11 80 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr11 81 klf.fimo\n",
      "chr11 82 lef1.fimo\n",
      "chr11 83 lhx3.fimo\n",
      "chr11 84 lin54.fimo\n",
      "chr11 85 maf_nrl.fimo\n",
      "chr11 86 mafg.fimo\n",
      "chr11 87 mecom.fimo\n",
      "chr11 88 mef2a-b-d.fimo\n",
      "chr11 89 mef2c.fimo\n",
      "chr11 90 meis.fimo\n",
      "chr11 91 mga_tbx.fimo\n",
      "chr11 92 msc_myf6_tfap4.fimo\n",
      "chr11 93 mtf1.fimo\n",
      "chr11 94 myb.fimo\n",
      "chr11 95 mybl1.fimo\n",
      "chr11 96 mybl2.fimo\n",
      "chr11 97 myc.fimo\n",
      "chr11 98 myog_tcf12.fimo\n",
      "chr11 99 mzf1.fimo\n",
      "chr11 100 mzf1var.2.fimo\n",
      "chr11 101 nfat.fimo\n",
      "chr11 102 nfi.fimo\n",
      "chr11 103 nfil3.fimo\n",
      "chr11 104 nfkb.fimo\n",
      "chr11 105 nfya.fimo\n",
      "chr11 106 nfyb.fimo\n",
      "chr11 107 nkx2-3_nkx2-8.fimo\n",
      "chr11 108 nkx2-5.fimo\n",
      "chr11 109 nkx2-5var.2.fimo\n",
      "chr11 110 nr2_rxr.fimo\n",
      "chr11 111 nr2e1.fimo\n",
      "chr11 112 nr2e3.fimo\n",
      "chr11 113 nr2f1.fimo\n",
      "chr11 114 nr4a2.fimo\n",
      "chr11 115 nr5a2.fimo\n",
      "chr11 116 nrf1.fimo\n",
      "chr11 117 onecut.fimo\n",
      "chr11 118 pax1_pax9.fimo\n",
      "chr11 119 pax2.fimo\n",
      "chr11 120 pax3_pax7.fimo\n",
      "chr11 121 pax5.fimo\n",
      "chr11 122 pax6.fimo\n",
      "chr11 123 pbx1.fimo\n",
      "chr11 124 phox2_prop.fimo\n",
      "chr11 125 pknox_tgif.fimo\n",
      "chr11 126 plag1.fimo\n",
      "chr11 127 pou1f1.fimo\n",
      "chr11 128 pou2-3-5.fimo\n",
      "chr11 129 pou4.fimo\n",
      "chr11 130 pou6f2.fimo\n",
      "chr11 131 pparg.fimo\n",
      "chr11 132 pparg_rxra.fimo\n",
      "chr11 133 prdm1.fimo\n",
      "chr11 134 prox1.fimo\n",
      "chr11 135 rar.fimo\n",
      "chr11 136 rar_rxr.fimo\n",
      "chr11 137 rargvar.2.fimo\n",
      "chr11 138 rel.fimo\n",
      "chr11 139 rest.fimo\n",
      "chr11 140 rfx.fimo\n",
      "chr11 141 rhox11.fimo\n",
      "chr11 142 rora.fimo\n",
      "chr11 143 roravar.2.fimo\n",
      "chr11 144 rreb1.fimo\n",
      "chr11 145 runx.fimo\n",
      "chr11 146 rxravdr.fimo\n",
      "chr11 147 scrt.fimo\n",
      "chr11 148 six3.fimo\n",
      "chr11 149 smad3.fimo\n",
      "chr11 150 sox1-l1-21-4.fimo\n",
      "chr11 151 sox10.fimo\n",
      "chr11 152 sox17.fimo\n",
      "chr11 153 sox2.fimo\n",
      "chr11 154 sox3_sox6.fimo\n",
      "chr11 155 sox5_sox9_sry.fimo\n",
      "chr11 156 sox8.fimo\n",
      "chr11 157 sp1.fimo\n",
      "chr11 158 sp4.fimo\n",
      "chr11 159 spi1-c.fimo\n",
      "chr11 160 spib.fimo\n",
      "chr11 161 spz1.fimo\n",
      "chr11 162 srebpf.fimo\n",
      "chr11 163 srf.fimo\n",
      "chr11 164 stat1-3.fimo\n",
      "chr11 165 stat6.fimo\n",
      "chr11 166 t.fimo\n",
      "chr11 167 tbp.fimo\n",
      "chr11 168 tbx.fimo\n",
      "chr11 169 tcf21.fimo\n",
      "chr11 170 tcf7.fimo\n",
      "chr11 171 tcfl5.fimo\n",
      "chr11 172 tead.fimo\n",
      "chr11 173 tfap1.fimo\n",
      "chr11 174 tfap2-3.fimo\n",
      "chr11 175 tfcp2.fimo\n",
      "chr11 176 thap1.fimo\n",
      "chr11 177 tp53.fimo\n",
      "chr11 178 tp63.fimo\n",
      "chr11 179 tp73.fimo\n",
      "chr11 180 usf.fimo\n",
      "chr11 181 ventx.fimo\n",
      "chr11 182 xbp1.fimo\n",
      "chr11 183 yy1.fimo\n",
      "chr11 184 yy2.fimo\n",
      "chr11 185 zbed1.fimo\n",
      "chr11 186 zbtb18.fimo\n",
      "chr11 187 zbtb33.fimo\n",
      "chr11 188 zbtb7.fimo\n",
      "chr11 189 zeb1.fimo\n",
      "chr11 190 zfx.fimo\n",
      "chr11 191 zic.fimo\n",
      "chr11 192 znf143.fimo\n",
      "chr11 193 znf263.fimo\n",
      "chr11 194 znf354c.fimo\n",
      "chr11 195 znf410.fimo\n",
      "chr11 196 znf423.fimo\n",
      "chr11 197 znf740.fimo\n",
      "chr12 2 alx1_alx4_arx.fimo\n",
      "chr12 3 ap-1.fimo\n",
      "chr12 4 ar_nr3c1_nr3c2.fimo\n",
      "chr12 5 arid3a.fimo\n",
      "chr12 6 arid3b.fimo\n",
      "chr12 7 arid5a.fimo\n",
      "chr12 8 arnt_mycn.fimo\n",
      "chr12 9 arntl_mitf.fimo\n",
      "chr12 10 ascl2_nhlh1.fimo\n",
      "chr12 11 atf7_batf3_creb5.fimo\n",
      "chr12 12 bcl6.fimo\n",
      "chr12 13 bcl6b.fimo\n",
      "chr12 14 bhlh.fimo\n",
      "chr12 15 bhlha15.fimo\n",
      "chr12 16 bhlhe23.fimo\n",
      "chr12 17 cdx.fimo\n",
      "chr12 18 cebp.fimo\n",
      "chr12 19 cenpb.fimo\n",
      "chr12 20 creb3-l1.fimo\n",
      "chr12 21 crem.fimo\n",
      "chr12 22 ctcf.fimo\n",
      "chr12 23 cux.fimo\n",
      "chr12 24 dbp_hlf_tef.fimo\n",
      "chr12 25 dmbx1.fimo\n",
      "chr12 26 dmrt3.fimo\n",
      "chr12 27 dux.fimo\n",
      "chr12 28 dux4.fimo\n",
      "chr12 29 duxa.fimo\n",
      "chr12 30 e2f1.fimo\n",
      "chr12 31 e2f2.fimo\n",
      "chr12 32 e2f4_e2f6.fimo\n",
      "chr12 33 e2f7.fimo\n",
      "chr12 34 e2f8.fimo\n",
      "chr12 35 ebf1.fimo\n",
      "chr12 36 ebox.fimo\n",
      "chr12 37 egr.fimo\n",
      "chr12 38 elf.fimo\n",
      "chr12 39 elk_etv.fimo\n",
      "chr12 40 esr1.fimo\n",
      "chr12 41 esr2.fimo\n",
      "chr12 42 esrr.fimo\n",
      "chr12 43 ewsr1-fli1.fimo\n",
      "chr12 44 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr12 45 fox.fimo\n",
      "chr12 46 gata.fimo\n",
      "chr12 47 gcm.fimo\n",
      "chr12 48 gfi1.fimo\n",
      "chr12 49 gfi1b.fimo\n",
      "chr12 50 gli2.fimo\n",
      "chr12 51 glis.fimo\n",
      "chr12 52 gmeb2.fimo\n",
      "chr12 53 grhl1.fimo\n",
      "chr12 54 gsc.fimo\n",
      "chr12 55 hes.fimo\n",
      "chr12 56 hic.fimo\n",
      "chr12 57 hinfp.fimo\n",
      "chr12 58 hltf.fimo\n",
      "chr12 59 hnf1.fimo\n",
      "chr12 60 hnf4a.fimo\n",
      "chr12 61 hnf4g.fimo\n",
      "chr12 62 homeobox-1.fimo\n",
      "chr12 63 homeobox-2.fimo\n",
      "chr12 64 homeobox-3.fimo\n",
      "chr12 65 homeobox-4.fimo\n",
      "chr12 66 homeobox-5.fimo\n",
      "chr12 67 hoxa11.fimo\n",
      "chr12 68 hoxa5.fimo\n",
      "chr12 69 hoxa9.fimo\n",
      "chr12 70 hoxb5_hoxd3.fimo\n",
      "chr12 71 hoxc13.fimo\n",
      "chr12 72 hoxc9.fimo\n",
      "chr12 73 hoxd8.fimo\n",
      "chr12 74 hsf.fimo\n",
      "chr12 75 id2_mlxip.fimo\n",
      "chr12 76 insm1.fimo\n",
      "chr12 77 irf1.fimo\n",
      "chr12 78 irf2.fimo\n",
      "chr12 79 irf7-8-9.fimo\n",
      "chr12 80 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr12 81 klf.fimo\n",
      "chr12 82 lef1.fimo\n",
      "chr12 83 lhx3.fimo\n",
      "chr12 84 lin54.fimo\n",
      "chr12 85 maf_nrl.fimo\n",
      "chr12 86 mafg.fimo\n",
      "chr12 87 mecom.fimo\n",
      "chr12 88 mef2a-b-d.fimo\n",
      "chr12 89 mef2c.fimo\n",
      "chr12 90 meis.fimo\n",
      "chr12 91 mga_tbx.fimo\n",
      "chr12 92 msc_myf6_tfap4.fimo\n",
      "chr12 93 mtf1.fimo\n",
      "chr12 94 myb.fimo\n",
      "chr12 95 mybl1.fimo\n",
      "chr12 96 mybl2.fimo\n",
      "chr12 97 myc.fimo\n",
      "chr12 98 myog_tcf12.fimo\n",
      "chr12 99 mzf1.fimo\n",
      "chr12 100 mzf1var.2.fimo\n",
      "chr12 101 nfat.fimo\n",
      "chr12 102 nfi.fimo\n",
      "chr12 103 nfil3.fimo\n",
      "chr12 104 nfkb.fimo\n",
      "chr12 105 nfya.fimo\n",
      "chr12 106 nfyb.fimo\n",
      "chr12 107 nkx2-3_nkx2-8.fimo\n",
      "chr12 108 nkx2-5.fimo\n",
      "chr12 109 nkx2-5var.2.fimo\n",
      "chr12 110 nr2_rxr.fimo\n",
      "chr12 111 nr2e1.fimo\n",
      "chr12 112 nr2e3.fimo\n",
      "chr12 113 nr2f1.fimo\n",
      "chr12 114 nr4a2.fimo\n",
      "chr12 115 nr5a2.fimo\n",
      "chr12 116 nrf1.fimo\n",
      "chr12 117 onecut.fimo\n",
      "chr12 118 pax1_pax9.fimo\n",
      "chr12 119 pax2.fimo\n",
      "chr12 120 pax3_pax7.fimo\n",
      "chr12 121 pax5.fimo\n",
      "chr12 122 pax6.fimo\n",
      "chr12 123 pbx1.fimo\n",
      "chr12 124 phox2_prop.fimo\n",
      "chr12 125 pknox_tgif.fimo\n",
      "chr12 126 plag1.fimo\n",
      "chr12 127 pou1f1.fimo\n",
      "chr12 128 pou2-3-5.fimo\n",
      "chr12 129 pou4.fimo\n",
      "chr12 130 pou6f2.fimo\n",
      "chr12 131 pparg.fimo\n",
      "chr12 132 pparg_rxra.fimo\n",
      "chr12 133 prdm1.fimo\n",
      "chr12 134 prox1.fimo\n",
      "chr12 135 rar.fimo\n",
      "chr12 136 rar_rxr.fimo\n",
      "chr12 137 rargvar.2.fimo\n",
      "chr12 138 rel.fimo\n",
      "chr12 139 rest.fimo\n",
      "chr12 140 rfx.fimo\n",
      "chr12 141 rhox11.fimo\n",
      "chr12 142 rora.fimo\n",
      "chr12 143 roravar.2.fimo\n",
      "chr12 144 rreb1.fimo\n",
      "chr12 145 runx.fimo\n",
      "chr12 146 rxravdr.fimo\n",
      "chr12 147 scrt.fimo\n",
      "chr12 148 six3.fimo\n",
      "chr12 149 smad3.fimo\n",
      "chr12 150 sox1-l1-21-4.fimo\n",
      "chr12 151 sox10.fimo\n",
      "chr12 152 sox17.fimo\n",
      "chr12 153 sox2.fimo\n",
      "chr12 154 sox3_sox6.fimo\n",
      "chr12 155 sox5_sox9_sry.fimo\n",
      "chr12 156 sox8.fimo\n",
      "chr12 157 sp1.fimo\n",
      "chr12 158 sp4.fimo\n",
      "chr12 159 spi1-c.fimo\n",
      "chr12 160 spib.fimo\n",
      "chr12 161 spz1.fimo\n",
      "chr12 162 srebpf.fimo\n",
      "chr12 163 srf.fimo\n",
      "chr12 164 stat1-3.fimo\n",
      "chr12 165 stat6.fimo\n",
      "chr12 166 t.fimo\n",
      "chr12 167 tbp.fimo\n",
      "chr12 168 tbx.fimo\n",
      "chr12 169 tcf21.fimo\n",
      "chr12 170 tcf7.fimo\n",
      "chr12 171 tcfl5.fimo\n",
      "chr12 172 tead.fimo\n",
      "chr12 173 tfap1.fimo\n",
      "chr12 174 tfap2-3.fimo\n",
      "chr12 175 tfcp2.fimo\n",
      "chr12 176 thap1.fimo\n",
      "chr12 177 tp53.fimo\n",
      "chr12 178 tp63.fimo\n",
      "chr12 179 tp73.fimo\n",
      "chr12 180 usf.fimo\n",
      "chr12 181 ventx.fimo\n",
      "chr12 182 xbp1.fimo\n",
      "chr12 183 yy1.fimo\n",
      "chr12 184 yy2.fimo\n",
      "chr12 185 zbed1.fimo\n",
      "chr12 186 zbtb18.fimo\n",
      "chr12 187 zbtb33.fimo\n",
      "chr12 188 zbtb7.fimo\n",
      "chr12 189 zeb1.fimo\n",
      "chr12 190 zfx.fimo\n",
      "chr12 191 zic.fimo\n",
      "chr12 192 znf143.fimo\n",
      "chr12 193 znf263.fimo\n",
      "chr12 194 znf354c.fimo\n",
      "chr12 195 znf410.fimo\n",
      "chr12 196 znf423.fimo\n",
      "chr12 197 znf740.fimo\n",
      "chr13 2 alx1_alx4_arx.fimo\n",
      "chr13 3 ap-1.fimo\n",
      "chr13 4 ar_nr3c1_nr3c2.fimo\n",
      "chr13 5 arid3a.fimo\n",
      "chr13 6 arid3b.fimo\n",
      "chr13 7 arid5a.fimo\n",
      "chr13 8 arnt_mycn.fimo\n",
      "chr13 9 arntl_mitf.fimo\n",
      "chr13 10 ascl2_nhlh1.fimo\n",
      "chr13 11 atf7_batf3_creb5.fimo\n",
      "chr13 12 bcl6.fimo\n",
      "chr13 13 bcl6b.fimo\n",
      "chr13 14 bhlh.fimo\n",
      "chr13 15 bhlha15.fimo\n",
      "chr13 16 bhlhe23.fimo\n",
      "chr13 17 cdx.fimo\n",
      "chr13 18 cebp.fimo\n",
      "chr13 19 cenpb.fimo\n",
      "chr13 20 creb3-l1.fimo\n",
      "chr13 21 crem.fimo\n",
      "chr13 22 ctcf.fimo\n",
      "chr13 23 cux.fimo\n",
      "chr13 24 dbp_hlf_tef.fimo\n",
      "chr13 25 dmbx1.fimo\n",
      "chr13 26 dmrt3.fimo\n",
      "chr13 27 dux.fimo\n",
      "chr13 28 dux4.fimo\n",
      "chr13 29 duxa.fimo\n",
      "chr13 30 e2f1.fimo\n",
      "chr13 31 e2f2.fimo\n",
      "chr13 32 e2f4_e2f6.fimo\n",
      "chr13 33 e2f7.fimo\n",
      "chr13 34 e2f8.fimo\n",
      "chr13 35 ebf1.fimo\n",
      "chr13 36 ebox.fimo\n",
      "chr13 37 egr.fimo\n",
      "chr13 38 elf.fimo\n",
      "chr13 39 elk_etv.fimo\n",
      "chr13 40 esr1.fimo\n",
      "chr13 41 esr2.fimo\n",
      "chr13 42 esrr.fimo\n",
      "chr13 43 ewsr1-fli1.fimo\n",
      "chr13 44 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr13 45 fox.fimo\n",
      "chr13 46 gata.fimo\n",
      "chr13 47 gcm.fimo\n",
      "chr13 48 gfi1.fimo\n",
      "chr13 49 gfi1b.fimo\n",
      "chr13 50 gli2.fimo\n",
      "chr13 51 glis.fimo\n",
      "chr13 52 gmeb2.fimo\n",
      "chr13 53 grhl1.fimo\n",
      "chr13 54 gsc.fimo\n",
      "chr13 55 hes.fimo\n",
      "chr13 56 hic.fimo\n",
      "chr13 57 hinfp.fimo\n",
      "chr13 58 hltf.fimo\n",
      "chr13 59 hnf1.fimo\n",
      "chr13 60 hnf4a.fimo\n",
      "chr13 61 hnf4g.fimo\n",
      "chr13 62 homeobox-1.fimo\n",
      "chr13 63 homeobox-2.fimo\n",
      "chr13 64 homeobox-3.fimo\n",
      "chr13 65 homeobox-4.fimo\n",
      "chr13 66 homeobox-5.fimo\n",
      "chr13 67 hoxa11.fimo\n",
      "chr13 68 hoxa5.fimo\n",
      "chr13 69 hoxa9.fimo\n",
      "chr13 70 hoxb5_hoxd3.fimo\n",
      "chr13 71 hoxc13.fimo\n",
      "chr13 72 hoxc9.fimo\n",
      "chr13 73 hoxd8.fimo\n",
      "chr13 74 hsf.fimo\n",
      "chr13 75 id2_mlxip.fimo\n",
      "chr13 76 insm1.fimo\n",
      "chr13 77 irf1.fimo\n",
      "chr13 78 irf2.fimo\n",
      "chr13 79 irf7-8-9.fimo\n",
      "chr13 80 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr13 81 klf.fimo\n",
      "chr13 82 lef1.fimo\n",
      "chr13 83 lhx3.fimo\n",
      "chr13 84 lin54.fimo\n",
      "chr13 85 maf_nrl.fimo\n",
      "chr13 86 mafg.fimo\n",
      "chr13 87 mecom.fimo\n",
      "chr13 88 mef2a-b-d.fimo\n",
      "chr13 89 mef2c.fimo\n",
      "chr13 90 meis.fimo\n",
      "chr13 91 mga_tbx.fimo\n",
      "chr13 92 msc_myf6_tfap4.fimo\n",
      "chr13 93 mtf1.fimo\n",
      "chr13 94 myb.fimo\n",
      "chr13 95 mybl1.fimo\n",
      "chr13 96 mybl2.fimo\n",
      "chr13 97 myc.fimo\n",
      "chr13 98 myog_tcf12.fimo\n",
      "chr13 99 mzf1.fimo\n",
      "chr13 100 mzf1var.2.fimo\n",
      "chr13 101 nfat.fimo\n",
      "chr13 102 nfi.fimo\n",
      "chr13 103 nfil3.fimo\n",
      "chr13 104 nfkb.fimo\n",
      "chr13 105 nfya.fimo\n",
      "chr13 106 nfyb.fimo\n",
      "chr13 107 nkx2-3_nkx2-8.fimo\n",
      "chr13 108 nkx2-5.fimo\n",
      "chr13 109 nkx2-5var.2.fimo\n",
      "chr13 110 nr2_rxr.fimo\n",
      "chr13 111 nr2e1.fimo\n",
      "chr13 112 nr2e3.fimo\n",
      "chr13 113 nr2f1.fimo\n",
      "chr13 114 nr4a2.fimo\n",
      "chr13 115 nr5a2.fimo\n",
      "chr13 116 nrf1.fimo\n",
      "chr13 117 onecut.fimo\n",
      "chr13 118 pax1_pax9.fimo\n",
      "chr13 119 pax2.fimo\n",
      "chr13 120 pax3_pax7.fimo\n",
      "chr13 121 pax5.fimo\n",
      "chr13 122 pax6.fimo\n",
      "chr13 123 pbx1.fimo\n",
      "chr13 124 phox2_prop.fimo\n",
      "chr13 125 pknox_tgif.fimo\n",
      "chr13 126 plag1.fimo\n",
      "chr13 127 pou1f1.fimo\n",
      "chr13 128 pou2-3-5.fimo\n",
      "chr13 129 pou4.fimo\n",
      "chr13 130 pou6f2.fimo\n",
      "chr13 131 pparg.fimo\n",
      "chr13 132 pparg_rxra.fimo\n",
      "chr13 133 prdm1.fimo\n",
      "chr13 134 prox1.fimo\n",
      "chr13 135 rar.fimo\n",
      "chr13 136 rar_rxr.fimo\n",
      "chr13 137 rargvar.2.fimo\n",
      "chr13 138 rel.fimo\n",
      "chr13 139 rest.fimo\n",
      "chr13 140 rfx.fimo\n",
      "chr13 141 rhox11.fimo\n",
      "chr13 142 rora.fimo\n",
      "chr13 143 roravar.2.fimo\n",
      "chr13 144 rreb1.fimo\n",
      "chr13 145 runx.fimo\n",
      "chr13 146 rxravdr.fimo\n",
      "chr13 147 scrt.fimo\n",
      "chr13 148 six3.fimo\n",
      "chr13 149 smad3.fimo\n",
      "chr13 150 sox1-l1-21-4.fimo\n",
      "chr13 151 sox10.fimo\n",
      "chr13 152 sox17.fimo\n",
      "chr13 153 sox2.fimo\n",
      "chr13 154 sox3_sox6.fimo\n",
      "chr13 155 sox5_sox9_sry.fimo\n",
      "chr13 156 sox8.fimo\n",
      "chr13 157 sp1.fimo\n",
      "chr13 158 sp4.fimo\n",
      "chr13 159 spi1-c.fimo\n",
      "chr13 160 spib.fimo\n",
      "chr13 161 spz1.fimo\n",
      "chr13 162 srebpf.fimo\n",
      "chr13 163 srf.fimo\n",
      "chr13 164 stat1-3.fimo\n",
      "chr13 165 stat6.fimo\n",
      "chr13 166 t.fimo\n",
      "chr13 167 tbp.fimo\n",
      "chr13 168 tbx.fimo\n",
      "chr13 169 tcf21.fimo\n",
      "chr13 170 tcf7.fimo\n",
      "chr13 171 tcfl5.fimo\n",
      "chr13 172 tead.fimo\n",
      "chr13 173 tfap1.fimo\n",
      "chr13 174 tfap2-3.fimo\n",
      "chr13 175 tfcp2.fimo\n",
      "chr13 176 thap1.fimo\n",
      "chr13 177 tp53.fimo\n",
      "chr13 178 tp63.fimo\n",
      "chr13 179 tp73.fimo\n",
      "chr13 180 usf.fimo\n",
      "chr13 181 ventx.fimo\n",
      "chr13 182 xbp1.fimo\n",
      "chr13 183 yy1.fimo\n",
      "chr13 184 yy2.fimo\n",
      "chr13 185 zbed1.fimo\n",
      "chr13 186 zbtb18.fimo\n",
      "chr13 187 zbtb33.fimo\n",
      "chr13 188 zbtb7.fimo\n",
      "chr13 189 zeb1.fimo\n",
      "chr13 190 zfx.fimo\n",
      "chr13 191 zic.fimo\n",
      "chr13 192 znf143.fimo\n",
      "chr13 193 znf263.fimo\n",
      "chr13 194 znf354c.fimo\n",
      "chr13 195 znf410.fimo\n",
      "chr13 196 znf423.fimo\n",
      "chr13 197 znf740.fimo\n",
      "chr14 2 alx1_alx4_arx.fimo\n",
      "chr14 3 ap-1.fimo\n",
      "chr14 4 ar_nr3c1_nr3c2.fimo\n",
      "chr14 5 arid3a.fimo\n",
      "chr14 6 arid3b.fimo\n",
      "chr14 7 arid5a.fimo\n",
      "chr14 8 arnt_mycn.fimo\n",
      "chr14 9 arntl_mitf.fimo\n",
      "chr14 10 ascl2_nhlh1.fimo\n",
      "chr14 11 atf7_batf3_creb5.fimo\n",
      "chr14 12 bcl6.fimo\n",
      "chr14 13 bcl6b.fimo\n",
      "chr14 14 bhlh.fimo\n",
      "chr14 15 bhlha15.fimo\n",
      "chr14 16 bhlhe23.fimo\n",
      "chr14 17 cdx.fimo\n",
      "chr14 18 cebp.fimo\n",
      "chr14 19 cenpb.fimo\n",
      "chr14 20 creb3-l1.fimo\n",
      "chr14 21 crem.fimo\n",
      "chr14 22 ctcf.fimo\n",
      "chr14 23 cux.fimo\n",
      "chr14 24 dbp_hlf_tef.fimo\n",
      "chr14 25 dmbx1.fimo\n",
      "chr14 26 dmrt3.fimo\n",
      "chr14 27 dux.fimo\n",
      "chr14 28 dux4.fimo\n",
      "chr14 29 duxa.fimo\n",
      "chr14 30 e2f1.fimo\n",
      "chr14 31 e2f2.fimo\n",
      "chr14 32 e2f4_e2f6.fimo\n",
      "chr14 33 e2f7.fimo\n",
      "chr14 34 e2f8.fimo\n",
      "chr14 35 ebf1.fimo\n",
      "chr14 36 ebox.fimo\n",
      "chr14 37 egr.fimo\n",
      "chr14 38 elf.fimo\n",
      "chr14 39 elk_etv.fimo\n",
      "chr14 40 esr1.fimo\n",
      "chr14 41 esr2.fimo\n",
      "chr14 42 esrr.fimo\n",
      "chr14 43 ewsr1-fli1.fimo\n",
      "chr14 44 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr14 45 fox.fimo\n",
      "chr14 46 gata.fimo\n",
      "chr14 47 gcm.fimo\n",
      "chr14 48 gfi1.fimo\n",
      "chr14 49 gfi1b.fimo\n",
      "chr14 50 gli2.fimo\n",
      "chr14 51 glis.fimo\n",
      "chr14 52 gmeb2.fimo\n",
      "chr14 53 grhl1.fimo\n",
      "chr14 54 gsc.fimo\n",
      "chr14 55 hes.fimo\n",
      "chr14 56 hic.fimo\n",
      "chr14 57 hinfp.fimo\n",
      "chr14 58 hltf.fimo\n",
      "chr14 59 hnf1.fimo\n",
      "chr14 60 hnf4a.fimo\n",
      "chr14 61 hnf4g.fimo\n",
      "chr14 62 homeobox-1.fimo\n",
      "chr14 63 homeobox-2.fimo\n",
      "chr14 64 homeobox-3.fimo\n",
      "chr14 65 homeobox-4.fimo\n",
      "chr14 66 homeobox-5.fimo\n",
      "chr14 67 hoxa11.fimo\n",
      "chr14 68 hoxa5.fimo\n",
      "chr14 69 hoxa9.fimo\n",
      "chr14 70 hoxb5_hoxd3.fimo\n",
      "chr14 71 hoxc13.fimo\n",
      "chr14 72 hoxc9.fimo\n",
      "chr14 73 hoxd8.fimo\n",
      "chr14 74 hsf.fimo\n",
      "chr14 75 id2_mlxip.fimo\n",
      "chr14 76 insm1.fimo\n",
      "chr14 77 irf1.fimo\n",
      "chr14 78 irf2.fimo\n",
      "chr14 79 irf7-8-9.fimo\n",
      "chr14 80 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr14 81 klf.fimo\n",
      "chr14 82 lef1.fimo\n",
      "chr14 83 lhx3.fimo\n",
      "chr14 84 lin54.fimo\n",
      "chr14 85 maf_nrl.fimo\n",
      "chr14 86 mafg.fimo\n",
      "chr14 87 mecom.fimo\n",
      "chr14 88 mef2a-b-d.fimo\n",
      "chr14 89 mef2c.fimo\n",
      "chr14 90 meis.fimo\n",
      "chr14 91 mga_tbx.fimo\n",
      "chr14 92 msc_myf6_tfap4.fimo\n",
      "chr14 93 mtf1.fimo\n",
      "chr14 94 myb.fimo\n",
      "chr14 95 mybl1.fimo\n",
      "chr14 96 mybl2.fimo\n",
      "chr14 97 myc.fimo\n",
      "chr14 98 myog_tcf12.fimo\n",
      "chr14 99 mzf1.fimo\n",
      "chr14 100 mzf1var.2.fimo\n",
      "chr14 101 nfat.fimo\n",
      "chr14 102 nfi.fimo\n",
      "chr14 103 nfil3.fimo\n",
      "chr14 104 nfkb.fimo\n",
      "chr14 105 nfya.fimo\n",
      "chr14 106 nfyb.fimo\n",
      "chr14 107 nkx2-3_nkx2-8.fimo\n",
      "chr14 108 nkx2-5.fimo\n",
      "chr14 109 nkx2-5var.2.fimo\n",
      "chr14 110 nr2_rxr.fimo\n",
      "chr14 111 nr2e1.fimo\n",
      "chr14 112 nr2e3.fimo\n",
      "chr14 113 nr2f1.fimo\n",
      "chr14 114 nr4a2.fimo\n",
      "chr14 115 nr5a2.fimo\n",
      "chr14 116 nrf1.fimo\n",
      "chr14 117 onecut.fimo\n",
      "chr14 118 pax1_pax9.fimo\n",
      "chr14 119 pax2.fimo\n",
      "chr14 120 pax3_pax7.fimo\n",
      "chr14 121 pax5.fimo\n",
      "chr14 122 pax6.fimo\n",
      "chr14 123 pbx1.fimo\n",
      "chr14 124 phox2_prop.fimo\n",
      "chr14 125 pknox_tgif.fimo\n",
      "chr14 126 plag1.fimo\n",
      "chr14 127 pou1f1.fimo\n",
      "chr14 128 pou2-3-5.fimo\n",
      "chr14 129 pou4.fimo\n",
      "chr14 130 pou6f2.fimo\n",
      "chr14 131 pparg.fimo\n",
      "chr14 132 pparg_rxra.fimo\n",
      "chr14 133 prdm1.fimo\n",
      "chr14 134 prox1.fimo\n",
      "chr14 135 rar.fimo\n",
      "chr14 136 rar_rxr.fimo\n",
      "chr14 137 rargvar.2.fimo\n",
      "chr14 138 rel.fimo\n",
      "chr14 139 rest.fimo\n",
      "chr14 140 rfx.fimo\n",
      "chr14 141 rhox11.fimo\n",
      "chr14 142 rora.fimo\n",
      "chr14 143 roravar.2.fimo\n",
      "chr14 144 rreb1.fimo\n",
      "chr14 145 runx.fimo\n",
      "chr14 146 rxravdr.fimo\n",
      "chr14 147 scrt.fimo\n",
      "chr14 148 six3.fimo\n",
      "chr14 149 smad3.fimo\n",
      "chr14 150 sox1-l1-21-4.fimo\n",
      "chr14 151 sox10.fimo\n",
      "chr14 152 sox17.fimo\n",
      "chr14 153 sox2.fimo\n",
      "chr14 154 sox3_sox6.fimo\n",
      "chr14 155 sox5_sox9_sry.fimo\n",
      "chr14 156 sox8.fimo\n",
      "chr14 157 sp1.fimo\n",
      "chr14 158 sp4.fimo\n",
      "chr14 159 spi1-c.fimo\n",
      "chr14 160 spib.fimo\n",
      "chr14 161 spz1.fimo\n",
      "chr14 162 srebpf.fimo\n",
      "chr14 163 srf.fimo\n",
      "chr14 164 stat1-3.fimo\n",
      "chr14 165 stat6.fimo\n",
      "chr14 166 t.fimo\n",
      "chr14 167 tbp.fimo\n",
      "chr14 168 tbx.fimo\n",
      "chr14 169 tcf21.fimo\n",
      "chr14 170 tcf7.fimo\n",
      "chr14 171 tcfl5.fimo\n",
      "chr14 172 tead.fimo\n",
      "chr14 173 tfap1.fimo\n",
      "chr14 174 tfap2-3.fimo\n",
      "chr14 175 tfcp2.fimo\n",
      "chr14 176 thap1.fimo\n",
      "chr14 177 tp53.fimo\n",
      "chr14 178 tp63.fimo\n",
      "chr14 179 tp73.fimo\n",
      "chr14 180 usf.fimo\n",
      "chr14 181 ventx.fimo\n",
      "chr14 182 xbp1.fimo\n",
      "chr14 183 yy1.fimo\n",
      "chr14 184 yy2.fimo\n",
      "chr14 185 zbed1.fimo\n",
      "chr14 186 zbtb18.fimo\n",
      "chr14 187 zbtb33.fimo\n",
      "chr14 188 zbtb7.fimo\n",
      "chr14 189 zeb1.fimo\n",
      "chr14 190 zfx.fimo\n",
      "chr14 191 zic.fimo\n",
      "chr14 192 znf143.fimo\n",
      "chr14 193 znf263.fimo\n",
      "chr14 194 znf354c.fimo\n",
      "chr14 195 znf410.fimo\n",
      "chr14 196 znf423.fimo\n",
      "chr14 197 znf740.fimo\n",
      "chr15 2 alx1_alx4_arx.fimo\n",
      "chr15 3 ap-1.fimo\n",
      "chr15 4 ar_nr3c1_nr3c2.fimo\n",
      "chr15 5 arid3a.fimo\n",
      "chr15 6 arid3b.fimo\n",
      "chr15 7 arid5a.fimo\n",
      "chr15 8 arnt_mycn.fimo\n",
      "chr15 9 arntl_mitf.fimo\n",
      "chr15 10 ascl2_nhlh1.fimo\n",
      "chr15 11 atf7_batf3_creb5.fimo\n",
      "chr15 12 bcl6.fimo\n",
      "chr15 13 bcl6b.fimo\n",
      "chr15 14 bhlh.fimo\n",
      "chr15 15 bhlha15.fimo\n",
      "chr15 16 bhlhe23.fimo\n",
      "chr15 17 cdx.fimo\n",
      "chr15 18 cebp.fimo\n",
      "chr15 19 cenpb.fimo\n",
      "chr15 20 creb3-l1.fimo\n",
      "chr15 21 crem.fimo\n",
      "chr15 22 ctcf.fimo\n",
      "chr15 23 cux.fimo\n",
      "chr15 24 dbp_hlf_tef.fimo\n",
      "chr15 25 dmbx1.fimo\n",
      "chr15 26 dmrt3.fimo\n",
      "chr15 27 dux.fimo\n",
      "chr15 28 dux4.fimo\n",
      "chr15 29 duxa.fimo\n",
      "chr15 30 e2f1.fimo\n",
      "chr15 31 e2f2.fimo\n",
      "chr15 32 e2f4_e2f6.fimo\n",
      "chr15 33 e2f7.fimo\n",
      "chr15 34 e2f8.fimo\n",
      "chr15 35 ebf1.fimo\n",
      "chr15 36 ebox.fimo\n",
      "chr15 37 egr.fimo\n",
      "chr15 38 elf.fimo\n",
      "chr15 39 elk_etv.fimo\n",
      "chr15 40 esr1.fimo\n",
      "chr15 41 esr2.fimo\n",
      "chr15 42 esrr.fimo\n",
      "chr15 43 ewsr1-fli1.fimo\n",
      "chr15 44 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr15 45 fox.fimo\n",
      "chr15 46 gata.fimo\n",
      "chr15 47 gcm.fimo\n",
      "chr15 48 gfi1.fimo\n",
      "chr15 49 gfi1b.fimo\n",
      "chr15 50 gli2.fimo\n",
      "chr15 51 glis.fimo\n",
      "chr15 52 gmeb2.fimo\n",
      "chr15 53 grhl1.fimo\n",
      "chr15 54 gsc.fimo\n",
      "chr15 55 hes.fimo\n",
      "chr15 56 hic.fimo\n",
      "chr15 57 hinfp.fimo\n",
      "chr15 58 hltf.fimo\n",
      "chr15 59 hnf1.fimo\n",
      "chr15 60 hnf4a.fimo\n",
      "chr15 61 hnf4g.fimo\n",
      "chr15 62 homeobox-1.fimo\n",
      "chr15 63 homeobox-2.fimo\n",
      "chr15 64 homeobox-3.fimo\n",
      "chr15 65 homeobox-4.fimo\n",
      "chr15 66 homeobox-5.fimo\n",
      "chr15 67 hoxa11.fimo\n",
      "chr15 68 hoxa5.fimo\n",
      "chr15 69 hoxa9.fimo\n",
      "chr15 70 hoxb5_hoxd3.fimo\n",
      "chr15 71 hoxc13.fimo\n",
      "chr15 72 hoxc9.fimo\n",
      "chr15 73 hoxd8.fimo\n",
      "chr15 74 hsf.fimo\n",
      "chr15 75 id2_mlxip.fimo\n",
      "chr15 76 insm1.fimo\n",
      "chr15 77 irf1.fimo\n",
      "chr15 78 irf2.fimo\n",
      "chr15 79 irf7-8-9.fimo\n",
      "chr15 80 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr15 81 klf.fimo\n",
      "chr15 82 lef1.fimo\n",
      "chr15 83 lhx3.fimo\n",
      "chr15 84 lin54.fimo\n",
      "chr15 85 maf_nrl.fimo\n",
      "chr15 86 mafg.fimo\n",
      "chr15 87 mecom.fimo\n",
      "chr15 88 mef2a-b-d.fimo\n",
      "chr15 89 mef2c.fimo\n",
      "chr15 90 meis.fimo\n",
      "chr15 91 mga_tbx.fimo\n",
      "chr15 92 msc_myf6_tfap4.fimo\n",
      "chr15 93 mtf1.fimo\n",
      "chr15 94 myb.fimo\n",
      "chr15 95 mybl1.fimo\n",
      "chr15 96 mybl2.fimo\n",
      "chr15 97 myc.fimo\n",
      "chr15 98 myog_tcf12.fimo\n",
      "chr15 99 mzf1.fimo\n",
      "chr15 100 mzf1var.2.fimo\n",
      "chr15 101 nfat.fimo\n",
      "chr15 102 nfi.fimo\n",
      "chr15 103 nfil3.fimo\n",
      "chr15 104 nfkb.fimo\n",
      "chr15 105 nfya.fimo\n",
      "chr15 106 nfyb.fimo\n",
      "chr15 107 nkx2-3_nkx2-8.fimo\n",
      "chr15 108 nkx2-5.fimo\n",
      "chr15 109 nkx2-5var.2.fimo\n",
      "chr15 110 nr2_rxr.fimo\n",
      "chr15 111 nr2e1.fimo\n",
      "chr15 112 nr2e3.fimo\n",
      "chr15 113 nr2f1.fimo\n",
      "chr15 114 nr4a2.fimo\n",
      "chr15 115 nr5a2.fimo\n",
      "chr15 116 nrf1.fimo\n",
      "chr15 117 onecut.fimo\n",
      "chr15 118 pax1_pax9.fimo\n",
      "chr15 119 pax2.fimo\n",
      "chr15 120 pax3_pax7.fimo\n",
      "chr15 121 pax5.fimo\n",
      "chr15 122 pax6.fimo\n",
      "chr15 123 pbx1.fimo\n",
      "chr15 124 phox2_prop.fimo\n",
      "chr15 125 pknox_tgif.fimo\n",
      "chr15 126 plag1.fimo\n",
      "chr15 127 pou1f1.fimo\n",
      "chr15 128 pou2-3-5.fimo\n",
      "chr15 129 pou4.fimo\n",
      "chr15 130 pou6f2.fimo\n",
      "chr15 131 pparg.fimo\n",
      "chr15 132 pparg_rxra.fimo\n",
      "chr15 133 prdm1.fimo\n",
      "chr15 134 prox1.fimo\n",
      "chr15 135 rar.fimo\n",
      "chr15 136 rar_rxr.fimo\n",
      "chr15 137 rargvar.2.fimo\n",
      "chr15 138 rel.fimo\n",
      "chr15 139 rest.fimo\n",
      "chr15 140 rfx.fimo\n",
      "chr15 141 rhox11.fimo\n",
      "chr15 142 rora.fimo\n",
      "chr15 143 roravar.2.fimo\n",
      "chr15 144 rreb1.fimo\n",
      "chr15 145 runx.fimo\n",
      "chr15 146 rxravdr.fimo\n",
      "chr15 147 scrt.fimo\n",
      "chr15 148 six3.fimo\n",
      "chr15 149 smad3.fimo\n",
      "chr15 150 sox1-l1-21-4.fimo\n",
      "chr15 151 sox10.fimo\n",
      "chr15 152 sox17.fimo\n",
      "chr15 153 sox2.fimo\n",
      "chr15 154 sox3_sox6.fimo\n",
      "chr15 155 sox5_sox9_sry.fimo\n",
      "chr15 156 sox8.fimo\n",
      "chr15 157 sp1.fimo\n",
      "chr15 158 sp4.fimo\n",
      "chr15 159 spi1-c.fimo\n",
      "chr15 160 spib.fimo\n",
      "chr15 161 spz1.fimo\n",
      "chr15 162 srebpf.fimo\n",
      "chr15 163 srf.fimo\n",
      "chr15 164 stat1-3.fimo\n",
      "chr15 165 stat6.fimo\n",
      "chr15 166 t.fimo\n",
      "chr15 167 tbp.fimo\n",
      "chr15 168 tbx.fimo\n",
      "chr15 169 tcf21.fimo\n",
      "chr15 170 tcf7.fimo\n",
      "chr15 171 tcfl5.fimo\n",
      "chr15 172 tead.fimo\n",
      "chr15 173 tfap1.fimo\n",
      "chr15 174 tfap2-3.fimo\n",
      "chr15 175 tfcp2.fimo\n",
      "chr15 176 thap1.fimo\n",
      "chr15 177 tp53.fimo\n",
      "chr15 178 tp63.fimo\n",
      "chr15 179 tp73.fimo\n",
      "chr15 180 usf.fimo\n",
      "chr15 181 ventx.fimo\n",
      "chr15 182 xbp1.fimo\n",
      "chr15 183 yy1.fimo\n",
      "chr15 184 yy2.fimo\n",
      "chr15 185 zbed1.fimo\n",
      "chr15 186 zbtb18.fimo\n",
      "chr15 187 zbtb33.fimo\n",
      "chr15 188 zbtb7.fimo\n",
      "chr15 189 zeb1.fimo\n",
      "chr15 190 zfx.fimo\n",
      "chr15 191 zic.fimo\n",
      "chr15 192 znf143.fimo\n",
      "chr15 193 znf263.fimo\n",
      "chr15 194 znf354c.fimo\n",
      "chr15 195 znf410.fimo\n",
      "chr15 196 znf423.fimo\n",
      "chr15 197 znf740.fimo\n",
      "chr16 2 alx1_alx4_arx.fimo\n",
      "chr16 3 ap-1.fimo\n",
      "chr16 4 ar_nr3c1_nr3c2.fimo\n",
      "chr16 5 arid3a.fimo\n",
      "chr16 6 arid3b.fimo\n",
      "chr16 7 arid5a.fimo\n",
      "chr16 8 arnt_mycn.fimo\n",
      "chr16 9 arntl_mitf.fimo\n",
      "chr16 10 ascl2_nhlh1.fimo\n",
      "chr16 11 atf7_batf3_creb5.fimo\n",
      "chr16 12 bcl6.fimo\n",
      "chr16 13 bcl6b.fimo\n",
      "chr16 14 bhlh.fimo\n",
      "chr16 15 bhlha15.fimo\n",
      "chr16 16 bhlhe23.fimo\n",
      "chr16 17 cdx.fimo\n",
      "chr16 18 cebp.fimo\n",
      "chr16 19 cenpb.fimo\n",
      "chr16 20 creb3-l1.fimo\n",
      "chr16 21 crem.fimo\n",
      "chr16 22 ctcf.fimo\n",
      "chr16 23 cux.fimo\n",
      "chr16 24 dbp_hlf_tef.fimo\n",
      "chr16 25 dmbx1.fimo\n",
      "chr16 26 dmrt3.fimo\n",
      "chr16 27 dux.fimo\n",
      "chr16 28 dux4.fimo\n",
      "chr16 29 duxa.fimo\n",
      "chr16 30 e2f1.fimo\n",
      "chr16 31 e2f2.fimo\n",
      "chr16 32 e2f4_e2f6.fimo\n",
      "chr16 33 e2f7.fimo\n",
      "chr16 34 e2f8.fimo\n",
      "chr16 35 ebf1.fimo\n",
      "chr16 36 ebox.fimo\n",
      "chr16 37 egr.fimo\n",
      "chr16 38 elf.fimo\n",
      "chr16 39 elk_etv.fimo\n",
      "chr16 40 esr1.fimo\n",
      "chr16 41 esr2.fimo\n",
      "chr16 42 esrr.fimo\n",
      "chr16 43 ewsr1-fli1.fimo\n",
      "chr16 44 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr16 45 fox.fimo\n",
      "chr16 46 gata.fimo\n",
      "chr16 47 gcm.fimo\n",
      "chr16 48 gfi1.fimo\n",
      "chr16 49 gfi1b.fimo\n",
      "chr16 50 gli2.fimo\n",
      "chr16 51 glis.fimo\n",
      "chr16 52 gmeb2.fimo\n",
      "chr16 53 grhl1.fimo\n",
      "chr16 54 gsc.fimo\n",
      "chr16 55 hes.fimo\n",
      "chr16 56 hic.fimo\n",
      "chr16 57 hinfp.fimo\n",
      "chr16 58 hltf.fimo\n",
      "chr16 59 hnf1.fimo\n",
      "chr16 60 hnf4a.fimo\n",
      "chr16 61 hnf4g.fimo\n",
      "chr16 62 homeobox-1.fimo\n",
      "chr16 63 homeobox-2.fimo\n",
      "chr16 64 homeobox-3.fimo\n",
      "chr16 65 homeobox-4.fimo\n",
      "chr16 66 homeobox-5.fimo\n",
      "chr16 67 hoxa11.fimo\n",
      "chr16 68 hoxa5.fimo\n",
      "chr16 69 hoxa9.fimo\n",
      "chr16 70 hoxb5_hoxd3.fimo\n",
      "chr16 71 hoxc13.fimo\n",
      "chr16 72 hoxc9.fimo\n",
      "chr16 73 hoxd8.fimo\n",
      "chr16 74 hsf.fimo\n",
      "chr16 75 id2_mlxip.fimo\n",
      "chr16 76 insm1.fimo\n",
      "chr16 77 irf1.fimo\n",
      "chr16 78 irf2.fimo\n",
      "chr16 79 irf7-8-9.fimo\n",
      "chr16 80 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr16 81 klf.fimo\n",
      "chr16 82 lef1.fimo\n",
      "chr16 83 lhx3.fimo\n",
      "chr16 84 lin54.fimo\n",
      "chr16 85 maf_nrl.fimo\n",
      "chr16 86 mafg.fimo\n",
      "chr16 87 mecom.fimo\n",
      "chr16 88 mef2a-b-d.fimo\n",
      "chr16 89 mef2c.fimo\n",
      "chr16 90 meis.fimo\n",
      "chr16 91 mga_tbx.fimo\n",
      "chr16 92 msc_myf6_tfap4.fimo\n",
      "chr16 93 mtf1.fimo\n",
      "chr16 94 myb.fimo\n",
      "chr16 95 mybl1.fimo\n",
      "chr16 96 mybl2.fimo\n",
      "chr16 97 myc.fimo\n",
      "chr16 98 myog_tcf12.fimo\n",
      "chr16 99 mzf1.fimo\n",
      "chr16 100 mzf1var.2.fimo\n",
      "chr16 101 nfat.fimo\n",
      "chr16 102 nfi.fimo\n",
      "chr16 103 nfil3.fimo\n",
      "chr16 104 nfkb.fimo\n",
      "chr16 105 nfya.fimo\n",
      "chr16 106 nfyb.fimo\n",
      "chr16 107 nkx2-3_nkx2-8.fimo\n",
      "chr16 108 nkx2-5.fimo\n",
      "chr16 109 nkx2-5var.2.fimo\n",
      "chr16 110 nr2_rxr.fimo\n",
      "chr16 111 nr2e1.fimo\n",
      "chr16 112 nr2e3.fimo\n",
      "chr16 113 nr2f1.fimo\n",
      "chr16 114 nr4a2.fimo\n",
      "chr16 115 nr5a2.fimo\n",
      "chr16 116 nrf1.fimo\n",
      "chr16 117 onecut.fimo\n",
      "chr16 118 pax1_pax9.fimo\n",
      "chr16 119 pax2.fimo\n",
      "chr16 120 pax3_pax7.fimo\n",
      "chr16 121 pax5.fimo\n",
      "chr16 122 pax6.fimo\n",
      "chr16 123 pbx1.fimo\n",
      "chr16 124 phox2_prop.fimo\n",
      "chr16 125 pknox_tgif.fimo\n",
      "chr16 126 plag1.fimo\n",
      "chr16 127 pou1f1.fimo\n",
      "chr16 128 pou2-3-5.fimo\n",
      "chr16 129 pou4.fimo\n",
      "chr16 130 pou6f2.fimo\n",
      "chr16 131 pparg.fimo\n",
      "chr16 132 pparg_rxra.fimo\n",
      "chr16 133 prdm1.fimo\n",
      "chr16 134 prox1.fimo\n",
      "chr16 135 rar.fimo\n",
      "chr16 136 rar_rxr.fimo\n",
      "chr16 137 rargvar.2.fimo\n",
      "chr16 138 rel.fimo\n",
      "chr16 139 rest.fimo\n",
      "chr16 140 rfx.fimo\n",
      "chr16 141 rhox11.fimo\n",
      "chr16 142 rora.fimo\n",
      "chr16 143 roravar.2.fimo\n",
      "chr16 144 rreb1.fimo\n",
      "chr16 145 runx.fimo\n",
      "chr16 146 rxravdr.fimo\n",
      "chr16 147 scrt.fimo\n",
      "chr16 148 six3.fimo\n",
      "chr16 149 smad3.fimo\n",
      "chr16 150 sox1-l1-21-4.fimo\n",
      "chr16 151 sox10.fimo\n",
      "chr16 152 sox17.fimo\n",
      "chr16 153 sox2.fimo\n",
      "chr16 154 sox3_sox6.fimo\n",
      "chr16 155 sox5_sox9_sry.fimo\n",
      "chr16 156 sox8.fimo\n",
      "chr16 157 sp1.fimo\n",
      "chr16 158 sp4.fimo\n",
      "chr16 159 spi1-c.fimo\n",
      "chr16 160 spib.fimo\n",
      "chr16 161 spz1.fimo\n",
      "chr16 162 srebpf.fimo\n",
      "chr16 163 srf.fimo\n",
      "chr16 164 stat1-3.fimo\n",
      "chr16 165 stat6.fimo\n",
      "chr16 166 t.fimo\n",
      "chr16 167 tbp.fimo\n",
      "chr16 168 tbx.fimo\n",
      "chr16 169 tcf21.fimo\n",
      "chr16 170 tcf7.fimo\n",
      "chr16 171 tcfl5.fimo\n",
      "chr16 172 tead.fimo\n",
      "chr16 173 tfap1.fimo\n",
      "chr16 174 tfap2-3.fimo\n",
      "chr16 175 tfcp2.fimo\n",
      "chr16 176 thap1.fimo\n",
      "chr16 177 tp53.fimo\n",
      "chr16 178 tp63.fimo\n",
      "chr16 179 tp73.fimo\n",
      "chr16 180 usf.fimo\n",
      "chr16 181 ventx.fimo\n",
      "chr16 182 xbp1.fimo\n",
      "chr16 183 yy1.fimo\n",
      "chr16 184 yy2.fimo\n",
      "chr16 185 zbed1.fimo\n",
      "chr16 186 zbtb18.fimo\n",
      "chr16 187 zbtb33.fimo\n",
      "chr16 188 zbtb7.fimo\n",
      "chr16 189 zeb1.fimo\n",
      "chr16 190 zfx.fimo\n",
      "chr16 191 zic.fimo\n",
      "chr16 192 znf143.fimo\n",
      "chr16 193 znf263.fimo\n",
      "chr16 194 znf354c.fimo\n",
      "chr16 195 znf410.fimo\n",
      "chr16 196 znf423.fimo\n",
      "chr16 197 znf740.fimo\n",
      "chr17 2 alx1_alx4_arx.fimo\n",
      "chr17 3 ap-1.fimo\n",
      "chr17 4 ar_nr3c1_nr3c2.fimo\n",
      "chr17 5 arid3a.fimo\n",
      "chr17 6 arid3b.fimo\n",
      "chr17 7 arid5a.fimo\n",
      "chr17 8 arnt_mycn.fimo\n",
      "chr17 9 arntl_mitf.fimo\n",
      "chr17 10 ascl2_nhlh1.fimo\n",
      "chr17 11 atf7_batf3_creb5.fimo\n",
      "chr17 12 bcl6.fimo\n",
      "chr17 13 bcl6b.fimo\n",
      "chr17 14 bhlh.fimo\n",
      "chr17 15 bhlha15.fimo\n",
      "chr17 16 bhlhe23.fimo\n",
      "chr17 17 cdx.fimo\n",
      "chr17 18 cebp.fimo\n",
      "chr17 19 cenpb.fimo\n",
      "chr17 20 creb3-l1.fimo\n",
      "chr17 21 crem.fimo\n",
      "chr17 22 ctcf.fimo\n",
      "chr17 23 cux.fimo\n",
      "chr17 24 dbp_hlf_tef.fimo\n",
      "chr17 25 dmbx1.fimo\n",
      "chr17 26 dmrt3.fimo\n",
      "chr17 27 dux.fimo\n",
      "chr17 28 dux4.fimo\n",
      "chr17 29 duxa.fimo\n",
      "chr17 30 e2f1.fimo\n",
      "chr17 31 e2f2.fimo\n",
      "chr17 32 e2f4_e2f6.fimo\n",
      "chr17 33 e2f7.fimo\n",
      "chr17 34 e2f8.fimo\n",
      "chr17 35 ebf1.fimo\n",
      "chr17 36 ebox.fimo\n",
      "chr17 37 egr.fimo\n",
      "chr17 38 elf.fimo\n",
      "chr17 39 elk_etv.fimo\n",
      "chr17 40 esr1.fimo\n",
      "chr17 41 esr2.fimo\n",
      "chr17 42 esrr.fimo\n",
      "chr17 43 ewsr1-fli1.fimo\n",
      "chr17 44 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr17 45 fox.fimo\n",
      "chr17 46 gata.fimo\n",
      "chr17 47 gcm.fimo\n",
      "chr17 48 gfi1.fimo\n",
      "chr17 49 gfi1b.fimo\n",
      "chr17 50 gli2.fimo\n",
      "chr17 51 glis.fimo\n",
      "chr17 52 gmeb2.fimo\n",
      "chr17 53 grhl1.fimo\n",
      "chr17 54 gsc.fimo\n",
      "chr17 55 hes.fimo\n",
      "chr17 56 hic.fimo\n",
      "chr17 57 hinfp.fimo\n",
      "chr17 58 hltf.fimo\n",
      "chr17 59 hnf1.fimo\n",
      "chr17 60 hnf4a.fimo\n",
      "chr17 61 hnf4g.fimo\n",
      "chr17 62 homeobox-1.fimo\n",
      "chr17 63 homeobox-2.fimo\n",
      "chr17 64 homeobox-3.fimo\n",
      "chr17 65 homeobox-4.fimo\n",
      "chr17 66 homeobox-5.fimo\n",
      "chr17 67 hoxa11.fimo\n",
      "chr17 68 hoxa5.fimo\n",
      "chr17 69 hoxa9.fimo\n",
      "chr17 70 hoxb5_hoxd3.fimo\n",
      "chr17 71 hoxc13.fimo\n",
      "chr17 72 hoxc9.fimo\n",
      "chr17 73 hoxd8.fimo\n",
      "chr17 74 hsf.fimo\n",
      "chr17 75 id2_mlxip.fimo\n",
      "chr17 76 insm1.fimo\n",
      "chr17 77 irf1.fimo\n",
      "chr17 78 irf2.fimo\n",
      "chr17 79 irf7-8-9.fimo\n",
      "chr17 80 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr17 81 klf.fimo\n",
      "chr17 82 lef1.fimo\n",
      "chr17 83 lhx3.fimo\n",
      "chr17 84 lin54.fimo\n",
      "chr17 85 maf_nrl.fimo\n",
      "chr17 86 mafg.fimo\n",
      "chr17 87 mecom.fimo\n",
      "chr17 88 mef2a-b-d.fimo\n",
      "chr17 89 mef2c.fimo\n",
      "chr17 90 meis.fimo\n",
      "chr17 91 mga_tbx.fimo\n",
      "chr17 92 msc_myf6_tfap4.fimo\n",
      "chr17 93 mtf1.fimo\n",
      "chr17 94 myb.fimo\n",
      "chr17 95 mybl1.fimo\n",
      "chr17 96 mybl2.fimo\n",
      "chr17 97 myc.fimo\n",
      "chr17 98 myog_tcf12.fimo\n",
      "chr17 99 mzf1.fimo\n",
      "chr17 100 mzf1var.2.fimo\n",
      "chr17 101 nfat.fimo\n",
      "chr17 102 nfi.fimo\n",
      "chr17 103 nfil3.fimo\n",
      "chr17 104 nfkb.fimo\n",
      "chr17 105 nfya.fimo\n",
      "chr17 106 nfyb.fimo\n",
      "chr17 107 nkx2-3_nkx2-8.fimo\n",
      "chr17 108 nkx2-5.fimo\n",
      "chr17 109 nkx2-5var.2.fimo\n",
      "chr17 110 nr2_rxr.fimo\n",
      "chr17 111 nr2e1.fimo\n",
      "chr17 112 nr2e3.fimo\n",
      "chr17 113 nr2f1.fimo\n",
      "chr17 114 nr4a2.fimo\n",
      "chr17 115 nr5a2.fimo\n",
      "chr17 116 nrf1.fimo\n",
      "chr17 117 onecut.fimo\n",
      "chr17 118 pax1_pax9.fimo\n",
      "chr17 119 pax2.fimo\n",
      "chr17 120 pax3_pax7.fimo\n",
      "chr17 121 pax5.fimo\n",
      "chr17 122 pax6.fimo\n",
      "chr17 123 pbx1.fimo\n",
      "chr17 124 phox2_prop.fimo\n",
      "chr17 125 pknox_tgif.fimo\n",
      "chr17 126 plag1.fimo\n",
      "chr17 127 pou1f1.fimo\n",
      "chr17 128 pou2-3-5.fimo\n",
      "chr17 129 pou4.fimo\n",
      "chr17 130 pou6f2.fimo\n",
      "chr17 131 pparg.fimo\n",
      "chr17 132 pparg_rxra.fimo\n",
      "chr17 133 prdm1.fimo\n",
      "chr17 134 prox1.fimo\n",
      "chr17 135 rar.fimo\n",
      "chr17 136 rar_rxr.fimo\n",
      "chr17 137 rargvar.2.fimo\n",
      "chr17 138 rel.fimo\n",
      "chr17 139 rest.fimo\n",
      "chr17 140 rfx.fimo\n",
      "chr17 141 rhox11.fimo\n",
      "chr17 142 rora.fimo\n",
      "chr17 143 roravar.2.fimo\n",
      "chr17 144 rreb1.fimo\n",
      "chr17 145 runx.fimo\n",
      "chr17 146 rxravdr.fimo\n",
      "chr17 147 scrt.fimo\n",
      "chr17 148 six3.fimo\n",
      "chr17 149 smad3.fimo\n",
      "chr17 150 sox1-l1-21-4.fimo\n",
      "chr17 151 sox10.fimo\n",
      "chr17 152 sox17.fimo\n",
      "chr17 153 sox2.fimo\n",
      "chr17 154 sox3_sox6.fimo\n",
      "chr17 155 sox5_sox9_sry.fimo\n",
      "chr17 156 sox8.fimo\n",
      "chr17 157 sp1.fimo\n",
      "chr17 158 sp4.fimo\n",
      "chr17 159 spi1-c.fimo\n",
      "chr17 160 spib.fimo\n",
      "chr17 161 spz1.fimo\n",
      "chr17 162 srebpf.fimo\n",
      "chr17 163 srf.fimo\n",
      "chr17 164 stat1-3.fimo\n",
      "chr17 165 stat6.fimo\n",
      "chr17 166 t.fimo\n",
      "chr17 167 tbp.fimo\n",
      "chr17 168 tbx.fimo\n",
      "chr17 169 tcf21.fimo\n",
      "chr17 170 tcf7.fimo\n",
      "chr17 171 tcfl5.fimo\n",
      "chr17 172 tead.fimo\n",
      "chr17 173 tfap1.fimo\n",
      "chr17 174 tfap2-3.fimo\n",
      "chr17 175 tfcp2.fimo\n",
      "chr17 176 thap1.fimo\n",
      "chr17 177 tp53.fimo\n",
      "chr17 178 tp63.fimo\n",
      "chr17 179 tp73.fimo\n",
      "chr17 180 usf.fimo\n",
      "chr17 181 ventx.fimo\n",
      "chr17 182 xbp1.fimo\n",
      "chr17 183 yy1.fimo\n",
      "chr17 184 yy2.fimo\n",
      "chr17 185 zbed1.fimo\n",
      "chr17 186 zbtb18.fimo\n",
      "chr17 187 zbtb33.fimo\n",
      "chr17 188 zbtb7.fimo\n",
      "chr17 189 zeb1.fimo\n",
      "chr17 190 zfx.fimo\n",
      "chr17 191 zic.fimo\n",
      "chr17 192 znf143.fimo\n",
      "chr17 193 znf263.fimo\n",
      "chr17 194 znf354c.fimo\n",
      "chr17 195 znf410.fimo\n",
      "chr17 196 znf423.fimo\n",
      "chr17 197 znf740.fimo\n",
      "chr18 2 alx1_alx4_arx.fimo\n",
      "chr18 3 ap-1.fimo\n",
      "chr18 4 ar_nr3c1_nr3c2.fimo\n",
      "chr18 5 arid3a.fimo\n",
      "chr18 6 arid3b.fimo\n",
      "chr18 7 arid5a.fimo\n",
      "chr18 8 arnt_mycn.fimo\n",
      "chr18 9 arntl_mitf.fimo\n",
      "chr18 10 ascl2_nhlh1.fimo\n",
      "chr18 11 atf7_batf3_creb5.fimo\n",
      "chr18 12 bcl6.fimo\n",
      "chr18 13 bcl6b.fimo\n",
      "chr18 14 bhlh.fimo\n",
      "chr18 15 bhlha15.fimo\n",
      "chr18 16 bhlhe23.fimo\n",
      "chr18 17 cdx.fimo\n",
      "chr18 18 cebp.fimo\n",
      "chr18 19 cenpb.fimo\n",
      "chr18 20 creb3-l1.fimo\n",
      "chr18 21 crem.fimo\n",
      "chr18 22 ctcf.fimo\n",
      "chr18 23 cux.fimo\n",
      "chr18 24 dbp_hlf_tef.fimo\n",
      "chr18 25 dmbx1.fimo\n",
      "chr18 26 dmrt3.fimo\n",
      "chr18 27 dux.fimo\n",
      "chr18 28 dux4.fimo\n",
      "chr18 29 duxa.fimo\n",
      "chr18 30 e2f1.fimo\n",
      "chr18 31 e2f2.fimo\n",
      "chr18 32 e2f4_e2f6.fimo\n",
      "chr18 33 e2f7.fimo\n",
      "chr18 34 e2f8.fimo\n",
      "chr18 35 ebf1.fimo\n",
      "chr18 36 ebox.fimo\n",
      "chr18 37 egr.fimo\n",
      "chr18 38 elf.fimo\n",
      "chr18 39 elk_etv.fimo\n",
      "chr18 40 esr1.fimo\n",
      "chr18 41 esr2.fimo\n",
      "chr18 42 esrr.fimo\n",
      "chr18 43 ewsr1-fli1.fimo\n",
      "chr18 44 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr18 45 fox.fimo\n",
      "chr18 46 gata.fimo\n",
      "chr18 47 gcm.fimo\n",
      "chr18 48 gfi1.fimo\n",
      "chr18 49 gfi1b.fimo\n",
      "chr18 50 gli2.fimo\n",
      "chr18 51 glis.fimo\n",
      "chr18 52 gmeb2.fimo\n",
      "chr18 53 grhl1.fimo\n",
      "chr18 54 gsc.fimo\n",
      "chr18 55 hes.fimo\n",
      "chr18 56 hic.fimo\n",
      "chr18 57 hinfp.fimo\n",
      "chr18 58 hltf.fimo\n",
      "chr18 59 hnf1.fimo\n",
      "chr18 60 hnf4a.fimo\n",
      "chr18 61 hnf4g.fimo\n",
      "chr18 62 homeobox-1.fimo\n",
      "chr18 63 homeobox-2.fimo\n",
      "chr18 64 homeobox-3.fimo\n",
      "chr18 65 homeobox-4.fimo\n",
      "chr18 66 homeobox-5.fimo\n",
      "chr18 67 hoxa11.fimo\n",
      "chr18 68 hoxa5.fimo\n",
      "chr18 69 hoxa9.fimo\n",
      "chr18 70 hoxb5_hoxd3.fimo\n",
      "chr18 71 hoxc13.fimo\n",
      "chr18 72 hoxc9.fimo\n",
      "chr18 73 hoxd8.fimo\n",
      "chr18 74 hsf.fimo\n",
      "chr18 75 id2_mlxip.fimo\n",
      "chr18 76 insm1.fimo\n",
      "chr18 77 irf1.fimo\n",
      "chr18 78 irf2.fimo\n",
      "chr18 79 irf7-8-9.fimo\n",
      "chr18 80 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr18 81 klf.fimo\n",
      "chr18 82 lef1.fimo\n",
      "chr18 83 lhx3.fimo\n",
      "chr18 84 lin54.fimo\n",
      "chr18 85 maf_nrl.fimo\n",
      "chr18 86 mafg.fimo\n",
      "chr18 87 mecom.fimo\n",
      "chr18 88 mef2a-b-d.fimo\n",
      "chr18 89 mef2c.fimo\n",
      "chr18 90 meis.fimo\n",
      "chr18 91 mga_tbx.fimo\n",
      "chr18 92 msc_myf6_tfap4.fimo\n",
      "chr18 93 mtf1.fimo\n",
      "chr18 94 myb.fimo\n",
      "chr18 95 mybl1.fimo\n",
      "chr18 96 mybl2.fimo\n",
      "chr18 97 myc.fimo\n",
      "chr18 98 myog_tcf12.fimo\n",
      "chr18 99 mzf1.fimo\n",
      "chr18 100 mzf1var.2.fimo\n",
      "chr18 101 nfat.fimo\n",
      "chr18 102 nfi.fimo\n",
      "chr18 103 nfil3.fimo\n",
      "chr18 104 nfkb.fimo\n",
      "chr18 105 nfya.fimo\n",
      "chr18 106 nfyb.fimo\n",
      "chr18 107 nkx2-3_nkx2-8.fimo\n",
      "chr18 108 nkx2-5.fimo\n",
      "chr18 109 nkx2-5var.2.fimo\n",
      "chr18 110 nr2_rxr.fimo\n",
      "chr18 111 nr2e1.fimo\n",
      "chr18 112 nr2e3.fimo\n",
      "chr18 113 nr2f1.fimo\n",
      "chr18 114 nr4a2.fimo\n",
      "chr18 115 nr5a2.fimo\n",
      "chr18 116 nrf1.fimo\n",
      "chr18 117 onecut.fimo\n",
      "chr18 118 pax1_pax9.fimo\n",
      "chr18 119 pax2.fimo\n",
      "chr18 120 pax3_pax7.fimo\n",
      "chr18 121 pax5.fimo\n",
      "chr18 122 pax6.fimo\n",
      "chr18 123 pbx1.fimo\n",
      "chr18 124 phox2_prop.fimo\n",
      "chr18 125 pknox_tgif.fimo\n",
      "chr18 126 plag1.fimo\n",
      "chr18 127 pou1f1.fimo\n",
      "chr18 128 pou2-3-5.fimo\n",
      "chr18 129 pou4.fimo\n",
      "chr18 130 pou6f2.fimo\n",
      "chr18 131 pparg.fimo\n",
      "chr18 132 pparg_rxra.fimo\n",
      "chr18 133 prdm1.fimo\n",
      "chr18 134 prox1.fimo\n",
      "chr18 135 rar.fimo\n",
      "chr18 136 rar_rxr.fimo\n",
      "chr18 137 rargvar.2.fimo\n",
      "chr18 138 rel.fimo\n",
      "chr18 139 rest.fimo\n",
      "chr18 140 rfx.fimo\n",
      "chr18 141 rhox11.fimo\n",
      "chr18 142 rora.fimo\n",
      "chr18 143 roravar.2.fimo\n",
      "chr18 144 rreb1.fimo\n",
      "chr18 145 runx.fimo\n",
      "chr18 146 rxravdr.fimo\n",
      "chr18 147 scrt.fimo\n",
      "chr18 148 six3.fimo\n",
      "chr18 149 smad3.fimo\n",
      "chr18 150 sox1-l1-21-4.fimo\n",
      "chr18 151 sox10.fimo\n",
      "chr18 152 sox17.fimo\n",
      "chr18 153 sox2.fimo\n",
      "chr18 154 sox3_sox6.fimo\n",
      "chr18 155 sox5_sox9_sry.fimo\n",
      "chr18 156 sox8.fimo\n",
      "chr18 157 sp1.fimo\n",
      "chr18 158 sp4.fimo\n",
      "chr18 159 spi1-c.fimo\n",
      "chr18 160 spib.fimo\n",
      "chr18 161 spz1.fimo\n",
      "chr18 162 srebpf.fimo\n",
      "chr18 163 srf.fimo\n",
      "chr18 164 stat1-3.fimo\n",
      "chr18 165 stat6.fimo\n",
      "chr18 166 t.fimo\n",
      "chr18 167 tbp.fimo\n",
      "chr18 168 tbx.fimo\n",
      "chr18 169 tcf21.fimo\n",
      "chr18 170 tcf7.fimo\n",
      "chr18 171 tcfl5.fimo\n",
      "chr18 172 tead.fimo\n",
      "chr18 173 tfap1.fimo\n",
      "chr18 174 tfap2-3.fimo\n",
      "chr18 175 tfcp2.fimo\n",
      "chr18 176 thap1.fimo\n",
      "chr18 177 tp53.fimo\n",
      "chr18 178 tp63.fimo\n",
      "chr18 179 tp73.fimo\n",
      "chr18 180 usf.fimo\n",
      "chr18 181 ventx.fimo\n",
      "chr18 182 xbp1.fimo\n",
      "chr18 183 yy1.fimo\n",
      "chr18 184 yy2.fimo\n",
      "chr18 185 zbed1.fimo\n",
      "chr18 186 zbtb18.fimo\n",
      "chr18 187 zbtb33.fimo\n",
      "chr18 188 zbtb7.fimo\n",
      "chr18 189 zeb1.fimo\n",
      "chr18 190 zfx.fimo\n",
      "chr18 191 zic.fimo\n",
      "chr18 192 znf143.fimo\n",
      "chr18 193 znf263.fimo\n",
      "chr18 194 znf354c.fimo\n",
      "chr18 195 znf410.fimo\n",
      "chr18 196 znf423.fimo\n",
      "chr18 197 znf740.fimo\n",
      "chr19 2 alx1_alx4_arx.fimo\n",
      "chr19 3 ap-1.fimo\n",
      "chr19 4 ar_nr3c1_nr3c2.fimo\n",
      "chr19 5 arid3a.fimo\n",
      "chr19 6 arid3b.fimo\n",
      "chr19 7 arid5a.fimo\n",
      "chr19 8 arnt_mycn.fimo\n",
      "chr19 9 arntl_mitf.fimo\n",
      "chr19 10 ascl2_nhlh1.fimo\n",
      "chr19 11 atf7_batf3_creb5.fimo\n",
      "chr19 12 bcl6.fimo\n",
      "chr19 13 bcl6b.fimo\n",
      "chr19 14 bhlh.fimo\n",
      "chr19 15 bhlha15.fimo\n",
      "chr19 16 bhlhe23.fimo\n",
      "chr19 17 cdx.fimo\n",
      "chr19 18 cebp.fimo\n",
      "chr19 19 cenpb.fimo\n",
      "chr19 20 creb3-l1.fimo\n",
      "chr19 21 crem.fimo\n",
      "chr19 22 ctcf.fimo\n",
      "chr19 23 cux.fimo\n",
      "chr19 24 dbp_hlf_tef.fimo\n",
      "chr19 25 dmbx1.fimo\n",
      "chr19 26 dmrt3.fimo\n",
      "chr19 27 dux.fimo\n",
      "chr19 28 dux4.fimo\n",
      "chr19 29 duxa.fimo\n",
      "chr19 30 e2f1.fimo\n",
      "chr19 31 e2f2.fimo\n",
      "chr19 32 e2f4_e2f6.fimo\n",
      "chr19 33 e2f7.fimo\n",
      "chr19 34 e2f8.fimo\n",
      "chr19 35 ebf1.fimo\n",
      "chr19 36 ebox.fimo\n",
      "chr19 37 egr.fimo\n",
      "chr19 38 elf.fimo\n",
      "chr19 39 elk_etv.fimo\n",
      "chr19 40 esr1.fimo\n",
      "chr19 41 esr2.fimo\n",
      "chr19 42 esrr.fimo\n",
      "chr19 43 ewsr1-fli1.fimo\n",
      "chr19 44 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr19 45 fox.fimo\n",
      "chr19 46 gata.fimo\n",
      "chr19 47 gcm.fimo\n",
      "chr19 48 gfi1.fimo\n",
      "chr19 49 gfi1b.fimo\n",
      "chr19 50 gli2.fimo\n",
      "chr19 51 glis.fimo\n",
      "chr19 52 gmeb2.fimo\n",
      "chr19 53 grhl1.fimo\n",
      "chr19 54 gsc.fimo\n",
      "chr19 55 hes.fimo\n",
      "chr19 56 hic.fimo\n",
      "chr19 57 hinfp.fimo\n",
      "chr19 58 hltf.fimo\n",
      "chr19 59 hnf1.fimo\n",
      "chr19 60 hnf4a.fimo\n",
      "chr19 61 hnf4g.fimo\n",
      "chr19 62 homeobox-1.fimo\n",
      "chr19 63 homeobox-2.fimo\n",
      "chr19 64 homeobox-3.fimo\n",
      "chr19 65 homeobox-4.fimo\n",
      "chr19 66 homeobox-5.fimo\n",
      "chr19 67 hoxa11.fimo\n",
      "chr19 68 hoxa5.fimo\n",
      "chr19 69 hoxa9.fimo\n",
      "chr19 70 hoxb5_hoxd3.fimo\n",
      "chr19 71 hoxc13.fimo\n",
      "chr19 72 hoxc9.fimo\n",
      "chr19 73 hoxd8.fimo\n",
      "chr19 74 hsf.fimo\n",
      "chr19 75 id2_mlxip.fimo\n",
      "chr19 76 insm1.fimo\n",
      "chr19 77 irf1.fimo\n",
      "chr19 78 irf2.fimo\n",
      "chr19 79 irf7-8-9.fimo\n",
      "chr19 80 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr19 81 klf.fimo\n",
      "chr19 82 lef1.fimo\n",
      "chr19 83 lhx3.fimo\n",
      "chr19 84 lin54.fimo\n",
      "chr19 85 maf_nrl.fimo\n",
      "chr19 86 mafg.fimo\n",
      "chr19 87 mecom.fimo\n",
      "chr19 88 mef2a-b-d.fimo\n",
      "chr19 89 mef2c.fimo\n",
      "chr19 90 meis.fimo\n",
      "chr19 91 mga_tbx.fimo\n",
      "chr19 92 msc_myf6_tfap4.fimo\n",
      "chr19 93 mtf1.fimo\n",
      "chr19 94 myb.fimo\n",
      "chr19 95 mybl1.fimo\n",
      "chr19 96 mybl2.fimo\n",
      "chr19 97 myc.fimo\n",
      "chr19 98 myog_tcf12.fimo\n",
      "chr19 99 mzf1.fimo\n",
      "chr19 100 mzf1var.2.fimo\n",
      "chr19 101 nfat.fimo\n",
      "chr19 102 nfi.fimo\n",
      "chr19 103 nfil3.fimo\n",
      "chr19 104 nfkb.fimo\n",
      "chr19 105 nfya.fimo\n",
      "chr19 106 nfyb.fimo\n",
      "chr19 107 nkx2-3_nkx2-8.fimo\n",
      "chr19 108 nkx2-5.fimo\n",
      "chr19 109 nkx2-5var.2.fimo\n",
      "chr19 110 nr2_rxr.fimo\n",
      "chr19 111 nr2e1.fimo\n",
      "chr19 112 nr2e3.fimo\n",
      "chr19 113 nr2f1.fimo\n",
      "chr19 114 nr4a2.fimo\n",
      "chr19 115 nr5a2.fimo\n",
      "chr19 116 nrf1.fimo\n",
      "chr19 117 onecut.fimo\n",
      "chr19 118 pax1_pax9.fimo\n",
      "chr19 119 pax2.fimo\n",
      "chr19 120 pax3_pax7.fimo\n",
      "chr19 121 pax5.fimo\n",
      "chr19 122 pax6.fimo\n",
      "chr19 123 pbx1.fimo\n",
      "chr19 124 phox2_prop.fimo\n",
      "chr19 125 pknox_tgif.fimo\n",
      "chr19 126 plag1.fimo\n",
      "chr19 127 pou1f1.fimo\n",
      "chr19 128 pou2-3-5.fimo\n",
      "chr19 129 pou4.fimo\n",
      "chr19 130 pou6f2.fimo\n",
      "chr19 131 pparg.fimo\n",
      "chr19 132 pparg_rxra.fimo\n",
      "chr19 133 prdm1.fimo\n",
      "chr19 134 prox1.fimo\n",
      "chr19 135 rar.fimo\n",
      "chr19 136 rar_rxr.fimo\n",
      "chr19 137 rargvar.2.fimo\n",
      "chr19 138 rel.fimo\n",
      "chr19 139 rest.fimo\n",
      "chr19 140 rfx.fimo\n",
      "chr19 141 rhox11.fimo\n",
      "chr19 142 rora.fimo\n",
      "chr19 143 roravar.2.fimo\n",
      "chr19 144 rreb1.fimo\n",
      "chr19 145 runx.fimo\n",
      "chr19 146 rxravdr.fimo\n",
      "chr19 147 scrt.fimo\n",
      "chr19 148 six3.fimo\n",
      "chr19 149 smad3.fimo\n",
      "chr19 150 sox1-l1-21-4.fimo\n",
      "chr19 151 sox10.fimo\n",
      "chr19 152 sox17.fimo\n",
      "chr19 153 sox2.fimo\n",
      "chr19 154 sox3_sox6.fimo\n",
      "chr19 155 sox5_sox9_sry.fimo\n",
      "chr19 156 sox8.fimo\n",
      "chr19 157 sp1.fimo\n",
      "chr19 158 sp4.fimo\n",
      "chr19 159 spi1-c.fimo\n",
      "chr19 160 spib.fimo\n",
      "chr19 161 spz1.fimo\n",
      "chr19 162 srebpf.fimo\n",
      "chr19 163 srf.fimo\n",
      "chr19 164 stat1-3.fimo\n",
      "chr19 165 stat6.fimo\n",
      "chr19 166 t.fimo\n",
      "chr19 167 tbp.fimo\n",
      "chr19 168 tbx.fimo\n",
      "chr19 169 tcf21.fimo\n",
      "chr19 170 tcf7.fimo\n",
      "chr19 171 tcfl5.fimo\n",
      "chr19 172 tead.fimo\n",
      "chr19 173 tfap1.fimo\n",
      "chr19 174 tfap2-3.fimo\n",
      "chr19 175 tfcp2.fimo\n",
      "chr19 176 thap1.fimo\n",
      "chr19 177 tp53.fimo\n",
      "chr19 178 tp63.fimo\n",
      "chr19 179 tp73.fimo\n",
      "chr19 180 usf.fimo\n",
      "chr19 181 ventx.fimo\n",
      "chr19 182 xbp1.fimo\n",
      "chr19 183 yy1.fimo\n",
      "chr19 184 yy2.fimo\n",
      "chr19 185 zbed1.fimo\n",
      "chr19 186 zbtb18.fimo\n",
      "chr19 187 zbtb33.fimo\n",
      "chr19 188 zbtb7.fimo\n",
      "chr19 189 zeb1.fimo\n",
      "chr19 190 zfx.fimo\n",
      "chr19 191 zic.fimo\n",
      "chr19 192 znf143.fimo\n",
      "chr19 193 znf263.fimo\n",
      "chr19 194 znf354c.fimo\n",
      "chr19 195 znf410.fimo\n",
      "chr19 196 znf423.fimo\n",
      "chr19 197 znf740.fimo\n",
      "chr2 2 alx1_alx4_arx.fimo\n",
      "chr2 3 ap-1.fimo\n",
      "chr2 4 ar_nr3c1_nr3c2.fimo\n",
      "chr2 5 arid3a.fimo\n",
      "chr2 6 arid3b.fimo\n",
      "chr2 7 arid5a.fimo\n",
      "chr2 8 arnt_mycn.fimo\n",
      "chr2 9 arntl_mitf.fimo\n",
      "chr2 10 ascl2_nhlh1.fimo\n",
      "chr2 11 atf7_batf3_creb5.fimo\n",
      "chr2 12 bcl6.fimo\n",
      "chr2 13 bcl6b.fimo\n",
      "chr2 14 bhlh.fimo\n",
      "chr2 15 bhlha15.fimo\n",
      "chr2 16 bhlhe23.fimo\n",
      "chr2 17 cdx.fimo\n",
      "chr2 18 cebp.fimo\n",
      "chr2 19 cenpb.fimo\n",
      "chr2 20 creb3-l1.fimo\n",
      "chr2 21 crem.fimo\n",
      "chr2 22 ctcf.fimo\n",
      "chr2 23 cux.fimo\n",
      "chr2 24 dbp_hlf_tef.fimo\n",
      "chr2 25 dmbx1.fimo\n",
      "chr2 26 dmrt3.fimo\n",
      "chr2 27 dux.fimo\n",
      "chr2 28 dux4.fimo\n",
      "chr2 29 duxa.fimo\n",
      "chr2 30 e2f1.fimo\n",
      "chr2 31 e2f2.fimo\n",
      "chr2 32 e2f4_e2f6.fimo\n",
      "chr2 33 e2f7.fimo\n",
      "chr2 34 e2f8.fimo\n",
      "chr2 35 ebf1.fimo\n",
      "chr2 36 ebox.fimo\n",
      "chr2 37 egr.fimo\n",
      "chr2 38 elf.fimo\n",
      "chr2 39 elk_etv.fimo\n",
      "chr2 40 esr1.fimo\n",
      "chr2 41 esr2.fimo\n",
      "chr2 42 esrr.fimo\n",
      "chr2 43 ewsr1-fli1.fimo\n",
      "chr2 44 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr2 45 fox.fimo\n",
      "chr2 46 gata.fimo\n",
      "chr2 47 gcm.fimo\n",
      "chr2 48 gfi1.fimo\n",
      "chr2 49 gfi1b.fimo\n",
      "chr2 50 gli2.fimo\n",
      "chr2 51 glis.fimo\n",
      "chr2 52 gmeb2.fimo\n",
      "chr2 53 grhl1.fimo\n",
      "chr2 54 gsc.fimo\n",
      "chr2 55 hes.fimo\n",
      "chr2 56 hic.fimo\n",
      "chr2 57 hinfp.fimo\n",
      "chr2 58 hltf.fimo\n",
      "chr2 59 hnf1.fimo\n",
      "chr2 60 hnf4a.fimo\n",
      "chr2 61 hnf4g.fimo\n",
      "chr2 62 homeobox-1.fimo\n",
      "chr2 63 homeobox-2.fimo\n",
      "chr2 64 homeobox-3.fimo\n",
      "chr2 65 homeobox-4.fimo\n",
      "chr2 66 homeobox-5.fimo\n",
      "chr2 67 hoxa11.fimo\n",
      "chr2 68 hoxa5.fimo\n",
      "chr2 69 hoxa9.fimo\n",
      "chr2 70 hoxb5_hoxd3.fimo\n",
      "chr2 71 hoxc13.fimo\n",
      "chr2 72 hoxc9.fimo\n",
      "chr2 73 hoxd8.fimo\n",
      "chr2 74 hsf.fimo\n",
      "chr2 75 id2_mlxip.fimo\n",
      "chr2 76 insm1.fimo\n",
      "chr2 77 irf1.fimo\n",
      "chr2 78 irf2.fimo\n",
      "chr2 79 irf7-8-9.fimo\n",
      "chr2 80 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr2 81 klf.fimo\n",
      "chr2 82 lef1.fimo\n",
      "chr2 83 lhx3.fimo\n",
      "chr2 84 lin54.fimo\n",
      "chr2 85 maf_nrl.fimo\n",
      "chr2 86 mafg.fimo\n",
      "chr2 87 mecom.fimo\n",
      "chr2 88 mef2a-b-d.fimo\n",
      "chr2 89 mef2c.fimo\n",
      "chr2 90 meis.fimo\n",
      "chr2 91 mga_tbx.fimo\n",
      "chr2 92 msc_myf6_tfap4.fimo\n",
      "chr2 93 mtf1.fimo\n",
      "chr2 94 myb.fimo\n",
      "chr2 95 mybl1.fimo\n",
      "chr2 96 mybl2.fimo\n",
      "chr2 97 myc.fimo\n",
      "chr2 98 myog_tcf12.fimo\n",
      "chr2 99 mzf1.fimo\n",
      "chr2 100 mzf1var.2.fimo\n",
      "chr2 101 nfat.fimo\n",
      "chr2 102 nfi.fimo\n",
      "chr2 103 nfil3.fimo\n",
      "chr2 104 nfkb.fimo\n",
      "chr2 105 nfya.fimo\n",
      "chr2 106 nfyb.fimo\n",
      "chr2 107 nkx2-3_nkx2-8.fimo\n",
      "chr2 108 nkx2-5.fimo\n",
      "chr2 109 nkx2-5var.2.fimo\n",
      "chr2 110 nr2_rxr.fimo\n",
      "chr2 111 nr2e1.fimo\n",
      "chr2 112 nr2e3.fimo\n",
      "chr2 113 nr2f1.fimo\n",
      "chr2 114 nr4a2.fimo\n",
      "chr2 115 nr5a2.fimo\n",
      "chr2 116 nrf1.fimo\n",
      "chr2 117 onecut.fimo\n",
      "chr2 118 pax1_pax9.fimo\n",
      "chr2 119 pax2.fimo\n",
      "chr2 120 pax3_pax7.fimo\n",
      "chr2 121 pax5.fimo\n",
      "chr2 122 pax6.fimo\n",
      "chr2 123 pbx1.fimo\n",
      "chr2 124 phox2_prop.fimo\n",
      "chr2 125 pknox_tgif.fimo\n",
      "chr2 126 plag1.fimo\n",
      "chr2 127 pou1f1.fimo\n",
      "chr2 128 pou2-3-5.fimo\n",
      "chr2 129 pou4.fimo\n",
      "chr2 130 pou6f2.fimo\n",
      "chr2 131 pparg.fimo\n",
      "chr2 132 pparg_rxra.fimo\n",
      "chr2 133 prdm1.fimo\n",
      "chr2 134 prox1.fimo\n",
      "chr2 135 rar.fimo\n",
      "chr2 136 rar_rxr.fimo\n",
      "chr2 137 rargvar.2.fimo\n",
      "chr2 138 rel.fimo\n",
      "chr2 139 rest.fimo\n",
      "chr2 140 rfx.fimo\n",
      "chr2 141 rhox11.fimo\n",
      "chr2 142 rora.fimo\n",
      "chr2 143 roravar.2.fimo\n",
      "chr2 144 rreb1.fimo\n",
      "chr2 145 runx.fimo\n",
      "chr2 146 rxravdr.fimo\n",
      "chr2 147 scrt.fimo\n",
      "chr2 148 six3.fimo\n",
      "chr2 149 smad3.fimo\n",
      "chr2 150 sox1-l1-21-4.fimo\n",
      "chr2 151 sox10.fimo\n",
      "chr2 152 sox17.fimo\n",
      "chr2 153 sox2.fimo\n",
      "chr2 154 sox3_sox6.fimo\n",
      "chr2 155 sox5_sox9_sry.fimo\n",
      "chr2 156 sox8.fimo\n",
      "chr2 157 sp1.fimo\n",
      "chr2 158 sp4.fimo\n",
      "chr2 159 spi1-c.fimo\n",
      "chr2 160 spib.fimo\n",
      "chr2 161 spz1.fimo\n",
      "chr2 162 srebpf.fimo\n",
      "chr2 163 srf.fimo\n",
      "chr2 164 stat1-3.fimo\n",
      "chr2 165 stat6.fimo\n",
      "chr2 166 t.fimo\n",
      "chr2 167 tbp.fimo\n",
      "chr2 168 tbx.fimo\n",
      "chr2 169 tcf21.fimo\n",
      "chr2 170 tcf7.fimo\n",
      "chr2 171 tcfl5.fimo\n",
      "chr2 172 tead.fimo\n",
      "chr2 173 tfap1.fimo\n",
      "chr2 174 tfap2-3.fimo\n",
      "chr2 175 tfcp2.fimo\n",
      "chr2 176 thap1.fimo\n",
      "chr2 177 tp53.fimo\n",
      "chr2 178 tp63.fimo\n",
      "chr2 179 tp73.fimo\n",
      "chr2 180 usf.fimo\n",
      "chr2 181 ventx.fimo\n",
      "chr2 182 xbp1.fimo\n",
      "chr2 183 yy1.fimo\n",
      "chr2 184 yy2.fimo\n",
      "chr2 185 zbed1.fimo\n",
      "chr2 186 zbtb18.fimo\n",
      "chr2 187 zbtb33.fimo\n",
      "chr2 188 zbtb7.fimo\n",
      "chr2 189 zeb1.fimo\n",
      "chr2 190 zfx.fimo\n",
      "chr2 191 zic.fimo\n",
      "chr2 192 znf143.fimo\n",
      "chr2 193 znf263.fimo\n",
      "chr2 194 znf354c.fimo\n",
      "chr2 195 znf410.fimo\n",
      "chr2 196 znf423.fimo\n",
      "chr2 197 znf740.fimo\n",
      "chr3 1 alx1_alx4_arx.fimo\n",
      "chr3 2 ap-1.fimo\n",
      "chr3 3 ar_nr3c1_nr3c2.fimo\n",
      "chr3 4 arid3a.fimo\n",
      "chr3 5 arid3b.fimo\n",
      "chr3 6 arid5a.fimo\n",
      "chr3 7 arnt_mycn.fimo\n",
      "chr3 8 arntl_mitf.fimo\n",
      "chr3 9 ascl2_nhlh1.fimo\n",
      "chr3 10 atf7_batf3_creb5.fimo\n",
      "chr3 11 bcl6.fimo\n",
      "chr3 12 bcl6b.fimo\n",
      "chr3 13 bhlh.fimo\n",
      "chr3 14 bhlha15.fimo\n",
      "chr3 15 bhlhe23.fimo\n",
      "chr3 16 cdx.fimo\n",
      "chr3 17 cebp.fimo\n",
      "chr3 18 cenpb.fimo\n",
      "chr3 19 creb3-l1.fimo\n",
      "chr3 20 crem.fimo\n",
      "chr3 21 ctcf.fimo\n",
      "chr3 22 cux.fimo\n",
      "chr3 23 dbp_hlf_tef.fimo\n",
      "chr3 24 dmbx1.fimo\n",
      "chr3 25 dmrt3.fimo\n",
      "chr3 26 dux.fimo\n",
      "chr3 27 dux4.fimo\n",
      "chr3 28 duxa.fimo\n",
      "chr3 29 e2f1.fimo\n",
      "chr3 30 e2f2.fimo\n",
      "chr3 31 e2f4_e2f6.fimo\n",
      "chr3 32 e2f7.fimo\n",
      "chr3 33 e2f8.fimo\n",
      "chr3 34 ebf1.fimo\n",
      "chr3 35 ebox.fimo\n",
      "chr3 36 egr.fimo\n",
      "chr3 37 elf.fimo\n",
      "chr3 38 elk_etv.fimo\n",
      "chr3 39 esr1.fimo\n",
      "chr3 40 esr2.fimo\n",
      "chr3 41 esrr.fimo\n",
      "chr3 42 ewsr1-fli1.fimo\n",
      "chr3 43 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr3 44 fox.fimo\n",
      "chr3 45 gata.fimo\n",
      "chr3 46 gcm.fimo\n",
      "chr3 47 gfi1.fimo\n",
      "chr3 48 gfi1b.fimo\n",
      "chr3 49 gli2.fimo\n",
      "chr3 50 glis.fimo\n",
      "chr3 51 gmeb2.fimo\n",
      "chr3 52 grhl1.fimo\n",
      "chr3 53 gsc.fimo\n",
      "chr3 54 hes.fimo\n",
      "chr3 55 hic.fimo\n",
      "chr3 56 hinfp.fimo\n",
      "chr3 57 hltf.fimo\n",
      "chr3 58 hnf1.fimo\n",
      "chr3 59 hnf4a.fimo\n",
      "chr3 60 hnf4g.fimo\n",
      "chr3 61 homeobox-1.fimo\n",
      "chr3 62 homeobox-2.fimo\n",
      "chr3 63 homeobox-3.fimo\n",
      "chr3 64 homeobox-4.fimo\n",
      "chr3 65 homeobox-5.fimo\n",
      "chr3 66 hoxa11.fimo\n",
      "chr3 67 hoxa5.fimo\n",
      "chr3 68 hoxa9.fimo\n",
      "chr3 69 hoxb5_hoxd3.fimo\n",
      "chr3 70 hoxc13.fimo\n",
      "chr3 71 hoxc9.fimo\n",
      "chr3 72 hoxd8.fimo\n",
      "chr3 73 hsf.fimo\n",
      "chr3 74 id2_mlxip.fimo\n",
      "chr3 75 insm1.fimo\n",
      "chr3 76 irf1.fimo\n",
      "chr3 77 irf2.fimo\n",
      "chr3 78 irf7-8-9.fimo\n",
      "chr3 79 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr3 80 klf.fimo\n",
      "chr3 81 lef1.fimo\n",
      "chr3 82 lhx3.fimo\n",
      "chr3 83 lin54.fimo\n",
      "chr3 84 maf_nrl.fimo\n",
      "chr3 85 mafg.fimo\n",
      "chr3 86 mecom.fimo\n",
      "chr3 87 mef2a-b-d.fimo\n",
      "chr3 88 mef2c.fimo\n",
      "chr3 89 meis.fimo\n",
      "chr3 90 mga_tbx.fimo\n",
      "chr3 91 msc_myf6_tfap4.fimo\n",
      "chr3 92 mtf1.fimo\n",
      "chr3 93 myb.fimo\n",
      "chr3 94 mybl1.fimo\n",
      "chr3 95 mybl2.fimo\n",
      "chr3 96 myc.fimo\n",
      "chr3 97 myog_tcf12.fimo\n",
      "chr3 98 mzf1.fimo\n",
      "chr3 99 mzf1var.2.fimo\n",
      "chr3 100 nfat.fimo\n",
      "chr3 101 nfi.fimo\n",
      "chr3 102 nfil3.fimo\n",
      "chr3 103 nfkb.fimo\n",
      "chr3 104 nfya.fimo\n",
      "chr3 105 nfyb.fimo\n",
      "chr3 106 nkx2-3_nkx2-8.fimo\n",
      "chr3 107 nkx2-5.fimo\n",
      "chr3 108 nkx2-5var.2.fimo\n",
      "chr3 109 nr2_rxr.fimo\n",
      "chr3 110 nr2e1.fimo\n",
      "chr3 111 nr2e3.fimo\n",
      "chr3 112 nr2f1.fimo\n",
      "chr3 113 nr4a2.fimo\n",
      "chr3 114 nr5a2.fimo\n",
      "chr3 115 nrf1.fimo\n",
      "chr3 116 onecut.fimo\n",
      "chr3 117 pax1_pax9.fimo\n",
      "chr3 118 pax2.fimo\n",
      "chr3 119 pax3_pax7.fimo\n",
      "chr3 120 pax5.fimo\n",
      "chr3 121 pax6.fimo\n",
      "chr3 122 pbx1.fimo\n",
      "chr3 123 phox2_prop.fimo\n",
      "chr3 124 pknox_tgif.fimo\n",
      "chr3 125 plag1.fimo\n",
      "chr3 126 pou1f1.fimo\n",
      "chr3 127 pou2-3-5.fimo\n",
      "chr3 128 pou4.fimo\n",
      "chr3 129 pou6f2.fimo\n",
      "chr3 130 pparg.fimo\n",
      "chr3 131 pparg_rxra.fimo\n",
      "chr3 132 prdm1.fimo\n",
      "chr3 133 prox1.fimo\n",
      "chr3 134 rar.fimo\n",
      "chr3 135 rar_rxr.fimo\n",
      "chr3 136 rargvar.2.fimo\n",
      "chr3 137 rel.fimo\n",
      "chr3 138 rest.fimo\n",
      "chr3 139 rfx.fimo\n",
      "chr3 140 rhox11.fimo\n",
      "chr3 141 rora.fimo\n",
      "chr3 142 roravar.2.fimo\n",
      "chr3 143 rreb1.fimo\n",
      "chr3 144 runx.fimo\n",
      "chr3 145 rxravdr.fimo\n",
      "chr3 146 scrt.fimo\n",
      "chr3 147 six3.fimo\n",
      "chr3 148 smad3.fimo\n",
      "chr3 149 sox1-l1-21-4.fimo\n",
      "chr3 150 sox10.fimo\n",
      "chr3 151 sox17.fimo\n",
      "chr3 152 sox2.fimo\n",
      "chr3 153 sox3_sox6.fimo\n",
      "chr3 154 sox5_sox9_sry.fimo\n",
      "chr3 155 sox8.fimo\n",
      "chr3 156 sp1.fimo\n",
      "chr3 157 sp4.fimo\n",
      "chr3 158 spi1-c.fimo\n",
      "chr3 159 spib.fimo\n",
      "chr3 160 spz1.fimo\n",
      "chr3 161 srebpf.fimo\n",
      "chr3 162 srf.fimo\n",
      "chr3 163 stat1-3.fimo\n",
      "chr3 164 stat6.fimo\n",
      "chr3 165 t.fimo\n",
      "chr3 166 tbp.fimo\n",
      "chr3 167 tbx.fimo\n",
      "chr3 168 tcf21.fimo\n",
      "chr3 169 tcf7.fimo\n",
      "chr3 170 tcfl5.fimo\n",
      "chr3 171 tead.fimo\n",
      "chr3 172 tfap1.fimo\n",
      "chr3 173 tfap2-3.fimo\n",
      "chr3 174 tfcp2.fimo\n",
      "chr3 175 thap1.fimo\n",
      "chr3 176 tp53.fimo\n",
      "chr3 177 tp63.fimo\n",
      "chr3 178 tp73.fimo\n",
      "chr3 179 usf.fimo\n",
      "chr3 180 ventx.fimo\n",
      "chr3 181 xbp1.fimo\n",
      "chr3 182 yy1.fimo\n",
      "chr3 183 yy2.fimo\n",
      "chr3 184 zbed1.fimo\n",
      "chr3 185 zbtb18.fimo\n",
      "chr3 186 zbtb33.fimo\n",
      "chr3 187 zbtb7.fimo\n",
      "chr3 188 zeb1.fimo\n",
      "chr3 189 zfx.fimo\n",
      "chr3 190 zic.fimo\n",
      "chr3 191 znf143.fimo\n",
      "chr3 192 znf263.fimo\n",
      "chr3 193 znf354c.fimo\n",
      "chr3 194 znf410.fimo\n",
      "chr3 195 znf423.fimo\n",
      "chr3 196 znf740.fimo\n",
      "chr4 1 alx1_alx4_arx.fimo\n",
      "chr4 2 ap-1.fimo\n",
      "chr4 3 ar_nr3c1_nr3c2.fimo\n",
      "chr4 4 arid3a.fimo\n",
      "chr4 5 arid3b.fimo\n",
      "chr4 6 arid5a.fimo\n",
      "chr4 7 arnt_mycn.fimo\n",
      "chr4 8 arntl_mitf.fimo\n",
      "chr4 9 ascl2_nhlh1.fimo\n",
      "chr4 10 atf7_batf3_creb5.fimo\n",
      "chr4 11 bcl6.fimo\n",
      "chr4 12 bcl6b.fimo\n",
      "chr4 13 bhlh.fimo\n",
      "chr4 14 bhlha15.fimo\n",
      "chr4 15 bhlhe23.fimo\n",
      "chr4 16 cdx.fimo\n",
      "chr4 17 cebp.fimo\n",
      "chr4 18 cenpb.fimo\n",
      "chr4 19 creb3-l1.fimo\n",
      "chr4 20 crem.fimo\n",
      "chr4 21 ctcf.fimo\n",
      "chr4 22 cux.fimo\n",
      "chr4 23 dbp_hlf_tef.fimo\n",
      "chr4 24 dmbx1.fimo\n",
      "chr4 25 dmrt3.fimo\n",
      "chr4 26 dux.fimo\n",
      "chr4 27 dux4.fimo\n",
      "chr4 28 duxa.fimo\n",
      "chr4 29 e2f1.fimo\n",
      "chr4 30 e2f2.fimo\n",
      "chr4 31 e2f4_e2f6.fimo\n",
      "chr4 32 e2f7.fimo\n",
      "chr4 33 e2f8.fimo\n",
      "chr4 34 ebf1.fimo\n",
      "chr4 35 ebox.fimo\n",
      "chr4 36 egr.fimo\n",
      "chr4 37 elf.fimo\n",
      "chr4 38 elk_etv.fimo\n",
      "chr4 39 esr1.fimo\n",
      "chr4 40 esr2.fimo\n",
      "chr4 41 esrr.fimo\n",
      "chr4 42 ewsr1-fli1.fimo\n",
      "chr4 43 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr4 44 fox.fimo\n",
      "chr4 45 gata.fimo\n",
      "chr4 46 gcm.fimo\n",
      "chr4 47 gfi1.fimo\n",
      "chr4 48 gfi1b.fimo\n",
      "chr4 49 gli2.fimo\n",
      "chr4 50 glis.fimo\n",
      "chr4 51 gmeb2.fimo\n",
      "chr4 52 grhl1.fimo\n",
      "chr4 53 gsc.fimo\n",
      "chr4 54 hes.fimo\n",
      "chr4 55 hic.fimo\n",
      "chr4 56 hinfp.fimo\n",
      "chr4 57 hltf.fimo\n",
      "chr4 58 hnf1.fimo\n",
      "chr4 59 hnf4a.fimo\n",
      "chr4 60 hnf4g.fimo\n",
      "chr4 61 homeobox-1.fimo\n",
      "chr4 62 homeobox-2.fimo\n",
      "chr4 63 homeobox-3.fimo\n",
      "chr4 64 homeobox-4.fimo\n",
      "chr4 65 homeobox-5.fimo\n",
      "chr4 66 hoxa11.fimo\n",
      "chr4 67 hoxa5.fimo\n",
      "chr4 68 hoxa9.fimo\n",
      "chr4 69 hoxb5_hoxd3.fimo\n",
      "chr4 70 hoxc13.fimo\n",
      "chr4 71 hoxc9.fimo\n",
      "chr4 72 hoxd8.fimo\n",
      "chr4 73 hsf.fimo\n",
      "chr4 74 id2_mlxip.fimo\n",
      "chr4 75 insm1.fimo\n",
      "chr4 76 irf1.fimo\n",
      "chr4 77 irf2.fimo\n",
      "chr4 78 irf7-8-9.fimo\n",
      "chr4 79 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr4 80 klf.fimo\n",
      "chr4 81 lef1.fimo\n",
      "chr4 82 lhx3.fimo\n",
      "chr4 83 lin54.fimo\n",
      "chr4 84 maf_nrl.fimo\n",
      "chr4 85 mafg.fimo\n",
      "chr4 86 mecom.fimo\n",
      "chr4 87 mef2a-b-d.fimo\n",
      "chr4 88 mef2c.fimo\n",
      "chr4 89 meis.fimo\n",
      "chr4 90 mga_tbx.fimo\n",
      "chr4 91 msc_myf6_tfap4.fimo\n",
      "chr4 92 mtf1.fimo\n",
      "chr4 93 myb.fimo\n",
      "chr4 94 mybl1.fimo\n",
      "chr4 95 mybl2.fimo\n",
      "chr4 96 myc.fimo\n",
      "chr4 97 myog_tcf12.fimo\n",
      "chr4 98 mzf1.fimo\n",
      "chr4 99 mzf1var.2.fimo\n",
      "chr4 100 nfat.fimo\n",
      "chr4 101 nfi.fimo\n",
      "chr4 102 nfil3.fimo\n",
      "chr4 103 nfkb.fimo\n",
      "chr4 104 nfya.fimo\n",
      "chr4 105 nfyb.fimo\n",
      "chr4 106 nkx2-3_nkx2-8.fimo\n",
      "chr4 107 nkx2-5.fimo\n",
      "chr4 108 nkx2-5var.2.fimo\n",
      "chr4 109 nr2_rxr.fimo\n",
      "chr4 110 nr2e1.fimo\n",
      "chr4 111 nr2e3.fimo\n",
      "chr4 112 nr2f1.fimo\n",
      "chr4 113 nr4a2.fimo\n",
      "chr4 114 nr5a2.fimo\n",
      "chr4 115 nrf1.fimo\n",
      "chr4 116 onecut.fimo\n",
      "chr4 117 pax1_pax9.fimo\n",
      "chr4 118 pax2.fimo\n",
      "chr4 119 pax3_pax7.fimo\n",
      "chr4 120 pax5.fimo\n",
      "chr4 121 pax6.fimo\n",
      "chr4 122 pbx1.fimo\n",
      "chr4 123 phox2_prop.fimo\n",
      "chr4 124 pknox_tgif.fimo\n",
      "chr4 125 plag1.fimo\n",
      "chr4 126 pou1f1.fimo\n",
      "chr4 127 pou2-3-5.fimo\n",
      "chr4 128 pou4.fimo\n",
      "chr4 129 pou6f2.fimo\n",
      "chr4 130 pparg.fimo\n",
      "chr4 131 pparg_rxra.fimo\n",
      "chr4 132 prdm1.fimo\n",
      "chr4 133 prox1.fimo\n",
      "chr4 134 rar.fimo\n",
      "chr4 135 rar_rxr.fimo\n",
      "chr4 136 rargvar.2.fimo\n",
      "chr4 137 rel.fimo\n",
      "chr4 138 rest.fimo\n",
      "chr4 139 rfx.fimo\n",
      "chr4 140 rhox11.fimo\n",
      "chr4 141 rora.fimo\n",
      "chr4 142 roravar.2.fimo\n",
      "chr4 143 rreb1.fimo\n",
      "chr4 144 runx.fimo\n",
      "chr4 145 rxravdr.fimo\n",
      "chr4 146 scrt.fimo\n",
      "chr4 147 six3.fimo\n",
      "chr4 148 smad3.fimo\n",
      "chr4 149 sox1-l1-21-4.fimo\n",
      "chr4 150 sox10.fimo\n",
      "chr4 151 sox17.fimo\n",
      "chr4 152 sox2.fimo\n",
      "chr4 153 sox3_sox6.fimo\n",
      "chr4 154 sox5_sox9_sry.fimo\n",
      "chr4 155 sox8.fimo\n",
      "chr4 156 sp1.fimo\n",
      "chr4 157 sp4.fimo\n",
      "chr4 158 spi1-c.fimo\n",
      "chr4 159 spib.fimo\n",
      "chr4 160 spz1.fimo\n",
      "chr4 161 srebpf.fimo\n",
      "chr4 162 srf.fimo\n",
      "chr4 163 stat1-3.fimo\n",
      "chr4 164 stat6.fimo\n",
      "chr4 165 t.fimo\n",
      "chr4 166 tbp.fimo\n",
      "chr4 167 tbx.fimo\n",
      "chr4 168 tcf21.fimo\n",
      "chr4 169 tcf7.fimo\n",
      "chr4 170 tcfl5.fimo\n",
      "chr4 171 tead.fimo\n",
      "chr4 172 tfap1.fimo\n",
      "chr4 173 tfap2-3.fimo\n",
      "chr4 174 tfcp2.fimo\n",
      "chr4 175 thap1.fimo\n",
      "chr4 176 tp53.fimo\n",
      "chr4 177 tp63.fimo\n",
      "chr4 178 tp73.fimo\n",
      "chr4 179 usf.fimo\n",
      "chr4 180 ventx.fimo\n",
      "chr4 181 xbp1.fimo\n",
      "chr4 182 yy1.fimo\n",
      "chr4 183 yy2.fimo\n",
      "chr4 184 zbed1.fimo\n",
      "chr4 185 zbtb18.fimo\n",
      "chr4 186 zbtb33.fimo\n",
      "chr4 187 zbtb7.fimo\n",
      "chr4 188 zeb1.fimo\n",
      "chr4 189 zfx.fimo\n",
      "chr4 190 zic.fimo\n",
      "chr4 191 znf143.fimo\n",
      "chr4 192 znf263.fimo\n",
      "chr4 193 znf354c.fimo\n",
      "chr4 194 znf410.fimo\n",
      "chr4 195 znf423.fimo\n",
      "chr4 196 znf740.fimo\n",
      "chr5 1 alx1_alx4_arx.fimo\n",
      "chr5 2 ap-1.fimo\n",
      "chr5 3 ar_nr3c1_nr3c2.fimo\n",
      "chr5 4 arid3a.fimo\n",
      "chr5 5 arid3b.fimo\n",
      "chr5 6 arid5a.fimo\n",
      "chr5 7 arnt_mycn.fimo\n",
      "chr5 8 arntl_mitf.fimo\n",
      "chr5 9 ascl2_nhlh1.fimo\n",
      "chr5 10 atf7_batf3_creb5.fimo\n",
      "chr5 11 bcl6.fimo\n",
      "chr5 12 bcl6b.fimo\n",
      "chr5 13 bhlh.fimo\n",
      "chr5 14 bhlha15.fimo\n",
      "chr5 15 bhlhe23.fimo\n",
      "chr5 16 cdx.fimo\n",
      "chr5 17 cebp.fimo\n",
      "chr5 18 cenpb.fimo\n",
      "chr5 19 creb3-l1.fimo\n",
      "chr5 20 crem.fimo\n",
      "chr5 21 ctcf.fimo\n",
      "chr5 22 cux.fimo\n",
      "chr5 23 dbp_hlf_tef.fimo\n",
      "chr5 24 dmbx1.fimo\n",
      "chr5 25 dmrt3.fimo\n",
      "chr5 26 dux.fimo\n",
      "chr5 27 dux4.fimo\n",
      "chr5 28 duxa.fimo\n",
      "chr5 29 e2f1.fimo\n",
      "chr5 30 e2f2.fimo\n",
      "chr5 31 e2f4_e2f6.fimo\n",
      "chr5 32 e2f7.fimo\n",
      "chr5 33 e2f8.fimo\n",
      "chr5 34 ebf1.fimo\n",
      "chr5 35 ebox.fimo\n",
      "chr5 36 egr.fimo\n",
      "chr5 37 elf.fimo\n",
      "chr5 38 elk_etv.fimo\n",
      "chr5 39 esr1.fimo\n",
      "chr5 40 esr2.fimo\n",
      "chr5 41 esrr.fimo\n",
      "chr5 42 ewsr1-fli1.fimo\n",
      "chr5 43 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr5 44 fox.fimo\n",
      "chr5 45 gata.fimo\n",
      "chr5 46 gcm.fimo\n",
      "chr5 47 gfi1.fimo\n",
      "chr5 48 gfi1b.fimo\n",
      "chr5 49 gli2.fimo\n",
      "chr5 50 glis.fimo\n",
      "chr5 51 gmeb2.fimo\n",
      "chr5 52 grhl1.fimo\n",
      "chr5 53 gsc.fimo\n",
      "chr5 54 hes.fimo\n",
      "chr5 55 hic.fimo\n",
      "chr5 56 hinfp.fimo\n",
      "chr5 57 hltf.fimo\n",
      "chr5 58 hnf1.fimo\n",
      "chr5 59 hnf4a.fimo\n",
      "chr5 60 hnf4g.fimo\n",
      "chr5 61 homeobox-1.fimo\n",
      "chr5 62 homeobox-2.fimo\n",
      "chr5 63 homeobox-3.fimo\n",
      "chr5 64 homeobox-4.fimo\n",
      "chr5 65 homeobox-5.fimo\n",
      "chr5 66 hoxa11.fimo\n",
      "chr5 67 hoxa5.fimo\n",
      "chr5 68 hoxa9.fimo\n",
      "chr5 69 hoxb5_hoxd3.fimo\n",
      "chr5 70 hoxc13.fimo\n",
      "chr5 71 hoxc9.fimo\n",
      "chr5 72 hoxd8.fimo\n",
      "chr5 73 hsf.fimo\n",
      "chr5 74 id2_mlxip.fimo\n",
      "chr5 75 insm1.fimo\n",
      "chr5 76 irf1.fimo\n",
      "chr5 77 irf2.fimo\n",
      "chr5 78 irf7-8-9.fimo\n",
      "chr5 79 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr5 80 klf.fimo\n",
      "chr5 81 lef1.fimo\n",
      "chr5 82 lhx3.fimo\n",
      "chr5 83 lin54.fimo\n",
      "chr5 84 maf_nrl.fimo\n",
      "chr5 85 mafg.fimo\n",
      "chr5 86 mecom.fimo\n",
      "chr5 87 mef2a-b-d.fimo\n",
      "chr5 88 mef2c.fimo\n",
      "chr5 89 meis.fimo\n",
      "chr5 90 mga_tbx.fimo\n",
      "chr5 91 msc_myf6_tfap4.fimo\n",
      "chr5 92 mtf1.fimo\n",
      "chr5 93 myb.fimo\n",
      "chr5 94 mybl1.fimo\n",
      "chr5 95 mybl2.fimo\n",
      "chr5 96 myc.fimo\n",
      "chr5 97 myog_tcf12.fimo\n",
      "chr5 98 mzf1.fimo\n",
      "chr5 99 mzf1var.2.fimo\n",
      "chr5 100 nfat.fimo\n",
      "chr5 101 nfi.fimo\n",
      "chr5 102 nfil3.fimo\n",
      "chr5 103 nfkb.fimo\n",
      "chr5 104 nfya.fimo\n",
      "chr5 105 nfyb.fimo\n",
      "chr5 106 nkx2-3_nkx2-8.fimo\n",
      "chr5 107 nkx2-5.fimo\n",
      "chr5 108 nkx2-5var.2.fimo\n",
      "chr5 109 nr2_rxr.fimo\n",
      "chr5 110 nr2e1.fimo\n",
      "chr5 111 nr2e3.fimo\n",
      "chr5 112 nr2f1.fimo\n",
      "chr5 113 nr4a2.fimo\n",
      "chr5 114 nr5a2.fimo\n",
      "chr5 115 nrf1.fimo\n",
      "chr5 116 onecut.fimo\n",
      "chr5 117 pax1_pax9.fimo\n",
      "chr5 118 pax2.fimo\n",
      "chr5 119 pax3_pax7.fimo\n",
      "chr5 120 pax5.fimo\n",
      "chr5 121 pax6.fimo\n",
      "chr5 122 pbx1.fimo\n",
      "chr5 123 phox2_prop.fimo\n",
      "chr5 124 pknox_tgif.fimo\n",
      "chr5 125 plag1.fimo\n",
      "chr5 126 pou1f1.fimo\n",
      "chr5 127 pou2-3-5.fimo\n",
      "chr5 128 pou4.fimo\n",
      "chr5 129 pou6f2.fimo\n",
      "chr5 130 pparg.fimo\n",
      "chr5 131 pparg_rxra.fimo\n",
      "chr5 132 prdm1.fimo\n",
      "chr5 133 prox1.fimo\n",
      "chr5 134 rar.fimo\n",
      "chr5 135 rar_rxr.fimo\n",
      "chr5 136 rargvar.2.fimo\n",
      "chr5 137 rel.fimo\n",
      "chr5 138 rest.fimo\n",
      "chr5 139 rfx.fimo\n",
      "chr5 140 rhox11.fimo\n",
      "chr5 141 rora.fimo\n",
      "chr5 142 roravar.2.fimo\n",
      "chr5 143 rreb1.fimo\n",
      "chr5 144 runx.fimo\n",
      "chr5 145 rxravdr.fimo\n",
      "chr5 146 scrt.fimo\n",
      "chr5 147 six3.fimo\n",
      "chr5 148 smad3.fimo\n",
      "chr5 149 sox1-l1-21-4.fimo\n",
      "chr5 150 sox10.fimo\n",
      "chr5 151 sox17.fimo\n",
      "chr5 152 sox2.fimo\n",
      "chr5 153 sox3_sox6.fimo\n",
      "chr5 154 sox5_sox9_sry.fimo\n",
      "chr5 155 sox8.fimo\n",
      "chr5 156 sp1.fimo\n",
      "chr5 157 sp4.fimo\n",
      "chr5 158 spi1-c.fimo\n",
      "chr5 159 spib.fimo\n",
      "chr5 160 spz1.fimo\n",
      "chr5 161 srebpf.fimo\n",
      "chr5 162 srf.fimo\n",
      "chr5 163 stat1-3.fimo\n",
      "chr5 164 stat6.fimo\n",
      "chr5 165 t.fimo\n",
      "chr5 166 tbp.fimo\n",
      "chr5 167 tbx.fimo\n",
      "chr5 168 tcf21.fimo\n",
      "chr5 169 tcf7.fimo\n",
      "chr5 170 tcfl5.fimo\n",
      "chr5 171 tead.fimo\n",
      "chr5 172 tfap1.fimo\n",
      "chr5 173 tfap2-3.fimo\n",
      "chr5 174 tfcp2.fimo\n",
      "chr5 175 thap1.fimo\n",
      "chr5 176 tp53.fimo\n",
      "chr5 177 tp63.fimo\n",
      "chr5 178 tp73.fimo\n",
      "chr5 179 usf.fimo\n",
      "chr5 180 ventx.fimo\n",
      "chr5 181 xbp1.fimo\n",
      "chr5 182 yy1.fimo\n",
      "chr5 183 yy2.fimo\n",
      "chr5 184 zbed1.fimo\n",
      "chr5 185 zbtb18.fimo\n",
      "chr5 186 zbtb33.fimo\n",
      "chr5 187 zbtb7.fimo\n",
      "chr5 188 zeb1.fimo\n",
      "chr5 189 zfx.fimo\n",
      "chr5 190 zic.fimo\n",
      "chr5 191 znf143.fimo\n",
      "chr5 192 znf263.fimo\n",
      "chr5 193 znf354c.fimo\n",
      "chr5 194 znf410.fimo\n",
      "chr5 195 znf423.fimo\n",
      "chr5 196 znf740.fimo\n",
      "chr6 1 alx1_alx4_arx.fimo\n",
      "chr6 2 ap-1.fimo\n",
      "chr6 3 ar_nr3c1_nr3c2.fimo\n",
      "chr6 4 arid3a.fimo\n",
      "chr6 5 arid3b.fimo\n",
      "chr6 6 arid5a.fimo\n",
      "chr6 7 arnt_mycn.fimo\n",
      "chr6 8 arntl_mitf.fimo\n",
      "chr6 9 ascl2_nhlh1.fimo\n",
      "chr6 10 atf7_batf3_creb5.fimo\n",
      "chr6 11 bcl6.fimo\n",
      "chr6 12 bcl6b.fimo\n",
      "chr6 13 bhlh.fimo\n",
      "chr6 14 bhlha15.fimo\n",
      "chr6 15 bhlhe23.fimo\n",
      "chr6 16 cdx.fimo\n",
      "chr6 17 cebp.fimo\n",
      "chr6 18 cenpb.fimo\n",
      "chr6 19 creb3-l1.fimo\n",
      "chr6 20 crem.fimo\n",
      "chr6 21 ctcf.fimo\n",
      "chr6 22 cux.fimo\n",
      "chr6 23 dbp_hlf_tef.fimo\n",
      "chr6 24 dmbx1.fimo\n",
      "chr6 25 dmrt3.fimo\n",
      "chr6 26 dux.fimo\n",
      "chr6 27 dux4.fimo\n",
      "chr6 28 duxa.fimo\n",
      "chr6 29 e2f1.fimo\n",
      "chr6 30 e2f2.fimo\n",
      "chr6 31 e2f4_e2f6.fimo\n",
      "chr6 32 e2f7.fimo\n",
      "chr6 33 e2f8.fimo\n",
      "chr6 34 ebf1.fimo\n",
      "chr6 35 ebox.fimo\n",
      "chr6 36 egr.fimo\n",
      "chr6 37 elf.fimo\n",
      "chr6 38 elk_etv.fimo\n",
      "chr6 39 esr1.fimo\n",
      "chr6 40 esr2.fimo\n",
      "chr6 41 esrr.fimo\n",
      "chr6 42 ewsr1-fli1.fimo\n",
      "chr6 43 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr6 44 fox.fimo\n",
      "chr6 45 gata.fimo\n",
      "chr6 46 gcm.fimo\n",
      "chr6 47 gfi1.fimo\n",
      "chr6 48 gfi1b.fimo\n",
      "chr6 49 gli2.fimo\n",
      "chr6 50 glis.fimo\n",
      "chr6 51 gmeb2.fimo\n",
      "chr6 52 grhl1.fimo\n",
      "chr6 53 gsc.fimo\n",
      "chr6 54 hes.fimo\n",
      "chr6 55 hic.fimo\n",
      "chr6 56 hinfp.fimo\n",
      "chr6 57 hltf.fimo\n",
      "chr6 58 hnf1.fimo\n",
      "chr6 59 hnf4a.fimo\n",
      "chr6 60 hnf4g.fimo\n",
      "chr6 61 homeobox-1.fimo\n",
      "chr6 62 homeobox-2.fimo\n",
      "chr6 63 homeobox-3.fimo\n",
      "chr6 64 homeobox-4.fimo\n",
      "chr6 65 homeobox-5.fimo\n",
      "chr6 66 hoxa11.fimo\n",
      "chr6 67 hoxa5.fimo\n",
      "chr6 68 hoxa9.fimo\n",
      "chr6 69 hoxb5_hoxd3.fimo\n",
      "chr6 70 hoxc13.fimo\n",
      "chr6 71 hoxc9.fimo\n",
      "chr6 72 hoxd8.fimo\n",
      "chr6 73 hsf.fimo\n",
      "chr6 74 id2_mlxip.fimo\n",
      "chr6 75 insm1.fimo\n",
      "chr6 76 irf1.fimo\n",
      "chr6 77 irf2.fimo\n",
      "chr6 78 irf7-8-9.fimo\n",
      "chr6 79 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr6 80 klf.fimo\n",
      "chr6 81 lef1.fimo\n",
      "chr6 82 lhx3.fimo\n",
      "chr6 83 lin54.fimo\n",
      "chr6 84 maf_nrl.fimo\n",
      "chr6 85 mafg.fimo\n",
      "chr6 86 mecom.fimo\n",
      "chr6 87 mef2a-b-d.fimo\n",
      "chr6 88 mef2c.fimo\n",
      "chr6 89 meis.fimo\n",
      "chr6 90 mga_tbx.fimo\n",
      "chr6 91 msc_myf6_tfap4.fimo\n",
      "chr6 92 mtf1.fimo\n",
      "chr6 93 myb.fimo\n",
      "chr6 94 mybl1.fimo\n",
      "chr6 95 mybl2.fimo\n",
      "chr6 96 myc.fimo\n",
      "chr6 97 myog_tcf12.fimo\n",
      "chr6 98 mzf1.fimo\n",
      "chr6 99 mzf1var.2.fimo\n",
      "chr6 100 nfat.fimo\n",
      "chr6 101 nfi.fimo\n",
      "chr6 102 nfil3.fimo\n",
      "chr6 103 nfkb.fimo\n",
      "chr6 104 nfya.fimo\n",
      "chr6 105 nfyb.fimo\n",
      "chr6 106 nkx2-3_nkx2-8.fimo\n",
      "chr6 107 nkx2-5.fimo\n",
      "chr6 108 nkx2-5var.2.fimo\n",
      "chr6 109 nr2_rxr.fimo\n",
      "chr6 110 nr2e1.fimo\n",
      "chr6 111 nr2e3.fimo\n",
      "chr6 112 nr2f1.fimo\n",
      "chr6 113 nr4a2.fimo\n",
      "chr6 114 nr5a2.fimo\n",
      "chr6 115 nrf1.fimo\n",
      "chr6 116 onecut.fimo\n",
      "chr6 117 pax1_pax9.fimo\n",
      "chr6 118 pax2.fimo\n",
      "chr6 119 pax3_pax7.fimo\n",
      "chr6 120 pax5.fimo\n",
      "chr6 121 pax6.fimo\n",
      "chr6 122 pbx1.fimo\n",
      "chr6 123 phox2_prop.fimo\n",
      "chr6 124 pknox_tgif.fimo\n",
      "chr6 125 plag1.fimo\n",
      "chr6 126 pou1f1.fimo\n",
      "chr6 127 pou2-3-5.fimo\n",
      "chr6 128 pou4.fimo\n",
      "chr6 129 pou6f2.fimo\n",
      "chr6 130 pparg.fimo\n",
      "chr6 131 pparg_rxra.fimo\n",
      "chr6 132 prdm1.fimo\n",
      "chr6 133 prox1.fimo\n",
      "chr6 134 rar.fimo\n",
      "chr6 135 rar_rxr.fimo\n",
      "chr6 136 rargvar.2.fimo\n",
      "chr6 137 rel.fimo\n",
      "chr6 138 rest.fimo\n",
      "chr6 139 rfx.fimo\n",
      "chr6 140 rhox11.fimo\n",
      "chr6 141 rora.fimo\n",
      "chr6 142 roravar.2.fimo\n",
      "chr6 143 rreb1.fimo\n",
      "chr6 144 runx.fimo\n",
      "chr6 145 rxravdr.fimo\n",
      "chr6 146 scrt.fimo\n",
      "chr6 147 six3.fimo\n",
      "chr6 148 smad3.fimo\n",
      "chr6 149 sox1-l1-21-4.fimo\n",
      "chr6 150 sox10.fimo\n",
      "chr6 151 sox17.fimo\n",
      "chr6 152 sox2.fimo\n",
      "chr6 153 sox3_sox6.fimo\n",
      "chr6 154 sox5_sox9_sry.fimo\n",
      "chr6 155 sox8.fimo\n",
      "chr6 156 sp1.fimo\n",
      "chr6 157 sp4.fimo\n",
      "chr6 158 spi1-c.fimo\n",
      "chr6 159 spib.fimo\n",
      "chr6 160 spz1.fimo\n",
      "chr6 161 srebpf.fimo\n",
      "chr6 162 srf.fimo\n",
      "chr6 163 stat1-3.fimo\n",
      "chr6 164 stat6.fimo\n",
      "chr6 165 t.fimo\n",
      "chr6 166 tbp.fimo\n",
      "chr6 167 tbx.fimo\n",
      "chr6 168 tcf21.fimo\n",
      "chr6 169 tcf7.fimo\n",
      "chr6 170 tcfl5.fimo\n",
      "chr6 171 tead.fimo\n",
      "chr6 172 tfap1.fimo\n",
      "chr6 173 tfap2-3.fimo\n",
      "chr6 174 tfcp2.fimo\n",
      "chr6 175 thap1.fimo\n",
      "chr6 176 tp53.fimo\n",
      "chr6 177 tp63.fimo\n",
      "chr6 178 tp73.fimo\n",
      "chr6 179 usf.fimo\n",
      "chr6 180 ventx.fimo\n",
      "chr6 181 xbp1.fimo\n",
      "chr6 182 yy1.fimo\n",
      "chr6 183 yy2.fimo\n",
      "chr6 184 zbed1.fimo\n",
      "chr6 185 zbtb18.fimo\n",
      "chr6 186 zbtb33.fimo\n",
      "chr6 187 zbtb7.fimo\n",
      "chr6 188 zeb1.fimo\n",
      "chr6 189 zfx.fimo\n",
      "chr6 190 zic.fimo\n",
      "chr6 191 znf143.fimo\n",
      "chr6 192 znf263.fimo\n",
      "chr6 193 znf354c.fimo\n",
      "chr6 194 znf410.fimo\n",
      "chr6 195 znf423.fimo\n",
      "chr6 196 znf740.fimo\n",
      "chr7 1 alx1_alx4_arx.fimo\n",
      "chr7 2 ap-1.fimo\n",
      "chr7 3 ar_nr3c1_nr3c2.fimo\n",
      "chr7 4 arid3a.fimo\n",
      "chr7 5 arid3b.fimo\n",
      "chr7 6 arid5a.fimo\n",
      "chr7 7 arnt_mycn.fimo\n",
      "chr7 8 arntl_mitf.fimo\n",
      "chr7 9 ascl2_nhlh1.fimo\n",
      "chr7 10 atf7_batf3_creb5.fimo\n",
      "chr7 11 bcl6.fimo\n",
      "chr7 12 bcl6b.fimo\n",
      "chr7 13 bhlh.fimo\n",
      "chr7 14 bhlha15.fimo\n",
      "chr7 15 bhlhe23.fimo\n",
      "chr7 16 cdx.fimo\n",
      "chr7 17 cebp.fimo\n",
      "chr7 18 cenpb.fimo\n",
      "chr7 19 creb3-l1.fimo\n",
      "chr7 20 crem.fimo\n",
      "chr7 21 ctcf.fimo\n",
      "chr7 22 cux.fimo\n",
      "chr7 23 dbp_hlf_tef.fimo\n",
      "chr7 24 dmbx1.fimo\n",
      "chr7 25 dmrt3.fimo\n",
      "chr7 26 dux.fimo\n",
      "chr7 27 dux4.fimo\n",
      "chr7 28 duxa.fimo\n",
      "chr7 29 e2f1.fimo\n",
      "chr7 30 e2f2.fimo\n",
      "chr7 31 e2f4_e2f6.fimo\n",
      "chr7 32 e2f7.fimo\n",
      "chr7 33 e2f8.fimo\n",
      "chr7 34 ebf1.fimo\n",
      "chr7 35 ebox.fimo\n",
      "chr7 36 egr.fimo\n",
      "chr7 37 elf.fimo\n",
      "chr7 38 elk_etv.fimo\n",
      "chr7 39 esr1.fimo\n",
      "chr7 40 esr2.fimo\n",
      "chr7 41 esrr.fimo\n",
      "chr7 42 ewsr1-fli1.fimo\n",
      "chr7 43 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr7 44 fox.fimo\n",
      "chr7 45 gata.fimo\n",
      "chr7 46 gcm.fimo\n",
      "chr7 47 gfi1.fimo\n",
      "chr7 48 gfi1b.fimo\n",
      "chr7 49 gli2.fimo\n",
      "chr7 50 glis.fimo\n",
      "chr7 51 gmeb2.fimo\n",
      "chr7 52 grhl1.fimo\n",
      "chr7 53 gsc.fimo\n",
      "chr7 54 hes.fimo\n",
      "chr7 55 hic.fimo\n",
      "chr7 56 hinfp.fimo\n",
      "chr7 57 hltf.fimo\n",
      "chr7 58 hnf1.fimo\n",
      "chr7 59 hnf4a.fimo\n",
      "chr7 60 hnf4g.fimo\n",
      "chr7 61 homeobox-1.fimo\n",
      "chr7 62 homeobox-2.fimo\n",
      "chr7 63 homeobox-3.fimo\n",
      "chr7 64 homeobox-4.fimo\n",
      "chr7 65 homeobox-5.fimo\n",
      "chr7 66 hoxa11.fimo\n",
      "chr7 67 hoxa5.fimo\n",
      "chr7 68 hoxa9.fimo\n",
      "chr7 69 hoxb5_hoxd3.fimo\n",
      "chr7 70 hoxc13.fimo\n",
      "chr7 71 hoxc9.fimo\n",
      "chr7 72 hoxd8.fimo\n",
      "chr7 73 hsf.fimo\n",
      "chr7 74 id2_mlxip.fimo\n",
      "chr7 75 insm1.fimo\n",
      "chr7 76 irf1.fimo\n",
      "chr7 77 irf2.fimo\n",
      "chr7 78 irf7-8-9.fimo\n",
      "chr7 79 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr7 80 klf.fimo\n",
      "chr7 81 lef1.fimo\n",
      "chr7 82 lhx3.fimo\n",
      "chr7 83 lin54.fimo\n",
      "chr7 84 maf_nrl.fimo\n",
      "chr7 85 mafg.fimo\n",
      "chr7 86 mecom.fimo\n",
      "chr7 87 mef2a-b-d.fimo\n",
      "chr7 88 mef2c.fimo\n",
      "chr7 89 meis.fimo\n",
      "chr7 90 mga_tbx.fimo\n",
      "chr7 91 msc_myf6_tfap4.fimo\n",
      "chr7 92 mtf1.fimo\n",
      "chr7 93 myb.fimo\n",
      "chr7 94 mybl1.fimo\n",
      "chr7 95 mybl2.fimo\n",
      "chr7 96 myc.fimo\n",
      "chr7 97 myog_tcf12.fimo\n",
      "chr7 98 mzf1.fimo\n",
      "chr7 99 mzf1var.2.fimo\n",
      "chr7 100 nfat.fimo\n",
      "chr7 101 nfi.fimo\n",
      "chr7 102 nfil3.fimo\n",
      "chr7 103 nfkb.fimo\n",
      "chr7 104 nfya.fimo\n",
      "chr7 105 nfyb.fimo\n",
      "chr7 106 nkx2-3_nkx2-8.fimo\n",
      "chr7 107 nkx2-5.fimo\n",
      "chr7 108 nkx2-5var.2.fimo\n",
      "chr7 109 nr2_rxr.fimo\n",
      "chr7 110 nr2e1.fimo\n",
      "chr7 111 nr2e3.fimo\n",
      "chr7 112 nr2f1.fimo\n",
      "chr7 113 nr4a2.fimo\n",
      "chr7 114 nr5a2.fimo\n",
      "chr7 115 nrf1.fimo\n",
      "chr7 116 onecut.fimo\n",
      "chr7 117 pax1_pax9.fimo\n",
      "chr7 118 pax2.fimo\n",
      "chr7 119 pax3_pax7.fimo\n",
      "chr7 120 pax5.fimo\n",
      "chr7 121 pax6.fimo\n",
      "chr7 122 pbx1.fimo\n",
      "chr7 123 phox2_prop.fimo\n",
      "chr7 124 pknox_tgif.fimo\n",
      "chr7 125 plag1.fimo\n",
      "chr7 126 pou1f1.fimo\n",
      "chr7 127 pou2-3-5.fimo\n",
      "chr7 128 pou4.fimo\n",
      "chr7 129 pou6f2.fimo\n",
      "chr7 130 pparg.fimo\n",
      "chr7 131 pparg_rxra.fimo\n",
      "chr7 132 prdm1.fimo\n",
      "chr7 133 prox1.fimo\n",
      "chr7 134 rar.fimo\n",
      "chr7 135 rar_rxr.fimo\n",
      "chr7 136 rargvar.2.fimo\n",
      "chr7 137 rel.fimo\n",
      "chr7 138 rest.fimo\n",
      "chr7 139 rfx.fimo\n",
      "chr7 140 rhox11.fimo\n",
      "chr7 141 rora.fimo\n",
      "chr7 142 roravar.2.fimo\n",
      "chr7 143 rreb1.fimo\n",
      "chr7 144 runx.fimo\n",
      "chr7 145 rxravdr.fimo\n",
      "chr7 146 scrt.fimo\n",
      "chr7 147 six3.fimo\n",
      "chr7 148 smad3.fimo\n",
      "chr7 149 sox1-l1-21-4.fimo\n",
      "chr7 150 sox10.fimo\n",
      "chr7 151 sox17.fimo\n",
      "chr7 152 sox2.fimo\n",
      "chr7 153 sox3_sox6.fimo\n",
      "chr7 154 sox5_sox9_sry.fimo\n",
      "chr7 155 sox8.fimo\n",
      "chr7 156 sp1.fimo\n",
      "chr7 157 sp4.fimo\n",
      "chr7 158 spi1-c.fimo\n",
      "chr7 159 spib.fimo\n",
      "chr7 160 spz1.fimo\n",
      "chr7 161 srebpf.fimo\n",
      "chr7 162 srf.fimo\n",
      "chr7 163 stat1-3.fimo\n",
      "chr7 164 stat6.fimo\n",
      "chr7 165 t.fimo\n",
      "chr7 166 tbp.fimo\n",
      "chr7 167 tbx.fimo\n",
      "chr7 168 tcf21.fimo\n",
      "chr7 169 tcf7.fimo\n",
      "chr7 170 tcfl5.fimo\n",
      "chr7 171 tead.fimo\n",
      "chr7 172 tfap1.fimo\n",
      "chr7 173 tfap2-3.fimo\n",
      "chr7 174 tfcp2.fimo\n",
      "chr7 175 thap1.fimo\n",
      "chr7 176 tp53.fimo\n",
      "chr7 177 tp63.fimo\n",
      "chr7 178 tp73.fimo\n",
      "chr7 179 usf.fimo\n",
      "chr7 180 ventx.fimo\n",
      "chr7 181 xbp1.fimo\n",
      "chr7 182 yy1.fimo\n",
      "chr7 183 yy2.fimo\n",
      "chr7 184 zbed1.fimo\n",
      "chr7 185 zbtb18.fimo\n",
      "chr7 186 zbtb33.fimo\n",
      "chr7 187 zbtb7.fimo\n",
      "chr7 188 zeb1.fimo\n",
      "chr7 189 zfx.fimo\n",
      "chr7 190 zic.fimo\n",
      "chr7 191 znf143.fimo\n",
      "chr7 192 znf263.fimo\n",
      "chr7 193 znf354c.fimo\n",
      "chr7 194 znf410.fimo\n",
      "chr7 195 znf423.fimo\n",
      "chr7 196 znf740.fimo\n",
      "chr8 1 alx1_alx4_arx.fimo\n",
      "chr8 2 ap-1.fimo\n",
      "chr8 3 ar_nr3c1_nr3c2.fimo\n",
      "chr8 4 arid3a.fimo\n",
      "chr8 5 arid3b.fimo\n",
      "chr8 6 arid5a.fimo\n",
      "chr8 7 arnt_mycn.fimo\n",
      "chr8 8 arntl_mitf.fimo\n",
      "chr8 9 ascl2_nhlh1.fimo\n",
      "chr8 10 atf7_batf3_creb5.fimo\n",
      "chr8 11 bcl6.fimo\n",
      "chr8 12 bcl6b.fimo\n",
      "chr8 13 bhlh.fimo\n",
      "chr8 14 bhlha15.fimo\n",
      "chr8 15 bhlhe23.fimo\n",
      "chr8 16 cdx.fimo\n",
      "chr8 17 cebp.fimo\n",
      "chr8 18 cenpb.fimo\n",
      "chr8 19 creb3-l1.fimo\n",
      "chr8 20 crem.fimo\n",
      "chr8 21 ctcf.fimo\n",
      "chr8 22 cux.fimo\n",
      "chr8 23 dbp_hlf_tef.fimo\n",
      "chr8 24 dmbx1.fimo\n",
      "chr8 25 dmrt3.fimo\n",
      "chr8 26 dux.fimo\n",
      "chr8 27 dux4.fimo\n",
      "chr8 28 duxa.fimo\n",
      "chr8 29 e2f1.fimo\n",
      "chr8 30 e2f2.fimo\n",
      "chr8 31 e2f4_e2f6.fimo\n",
      "chr8 32 e2f7.fimo\n",
      "chr8 33 e2f8.fimo\n",
      "chr8 34 ebf1.fimo\n",
      "chr8 35 ebox.fimo\n",
      "chr8 36 egr.fimo\n",
      "chr8 37 elf.fimo\n",
      "chr8 38 elk_etv.fimo\n",
      "chr8 39 esr1.fimo\n",
      "chr8 40 esr2.fimo\n",
      "chr8 41 esrr.fimo\n",
      "chr8 42 ewsr1-fli1.fimo\n",
      "chr8 43 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr8 44 fox.fimo\n",
      "chr8 45 gata.fimo\n",
      "chr8 46 gcm.fimo\n",
      "chr8 47 gfi1.fimo\n",
      "chr8 48 gfi1b.fimo\n",
      "chr8 49 gli2.fimo\n",
      "chr8 50 glis.fimo\n",
      "chr8 51 gmeb2.fimo\n",
      "chr8 52 grhl1.fimo\n",
      "chr8 53 gsc.fimo\n",
      "chr8 54 hes.fimo\n",
      "chr8 55 hic.fimo\n",
      "chr8 56 hinfp.fimo\n",
      "chr8 57 hltf.fimo\n",
      "chr8 58 hnf1.fimo\n",
      "chr8 59 hnf4a.fimo\n",
      "chr8 60 hnf4g.fimo\n",
      "chr8 61 homeobox-1.fimo\n",
      "chr8 62 homeobox-2.fimo\n",
      "chr8 63 homeobox-3.fimo\n",
      "chr8 64 homeobox-4.fimo\n",
      "chr8 65 homeobox-5.fimo\n",
      "chr8 66 hoxa11.fimo\n",
      "chr8 67 hoxa5.fimo\n",
      "chr8 68 hoxa9.fimo\n",
      "chr8 69 hoxb5_hoxd3.fimo\n",
      "chr8 70 hoxc13.fimo\n",
      "chr8 71 hoxc9.fimo\n",
      "chr8 72 hoxd8.fimo\n",
      "chr8 73 hsf.fimo\n",
      "chr8 74 id2_mlxip.fimo\n",
      "chr8 75 insm1.fimo\n",
      "chr8 76 irf1.fimo\n",
      "chr8 77 irf2.fimo\n",
      "chr8 78 irf7-8-9.fimo\n",
      "chr8 79 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr8 80 klf.fimo\n",
      "chr8 81 lef1.fimo\n",
      "chr8 82 lhx3.fimo\n",
      "chr8 83 lin54.fimo\n",
      "chr8 84 maf_nrl.fimo\n",
      "chr8 85 mafg.fimo\n",
      "chr8 86 mecom.fimo\n",
      "chr8 87 mef2a-b-d.fimo\n",
      "chr8 88 mef2c.fimo\n",
      "chr8 89 meis.fimo\n",
      "chr8 90 mga_tbx.fimo\n",
      "chr8 91 msc_myf6_tfap4.fimo\n",
      "chr8 92 mtf1.fimo\n",
      "chr8 93 myb.fimo\n",
      "chr8 94 mybl1.fimo\n",
      "chr8 95 mybl2.fimo\n",
      "chr8 96 myc.fimo\n",
      "chr8 97 myog_tcf12.fimo\n",
      "chr8 98 mzf1.fimo\n",
      "chr8 99 mzf1var.2.fimo\n",
      "chr8 100 nfat.fimo\n",
      "chr8 101 nfi.fimo\n",
      "chr8 102 nfil3.fimo\n",
      "chr8 103 nfkb.fimo\n",
      "chr8 104 nfya.fimo\n",
      "chr8 105 nfyb.fimo\n",
      "chr8 106 nkx2-3_nkx2-8.fimo\n",
      "chr8 107 nkx2-5.fimo\n",
      "chr8 108 nkx2-5var.2.fimo\n",
      "chr8 109 nr2_rxr.fimo\n",
      "chr8 110 nr2e1.fimo\n",
      "chr8 111 nr2e3.fimo\n",
      "chr8 112 nr2f1.fimo\n",
      "chr8 113 nr4a2.fimo\n",
      "chr8 114 nr5a2.fimo\n",
      "chr8 115 nrf1.fimo\n",
      "chr8 116 onecut.fimo\n",
      "chr8 117 pax1_pax9.fimo\n",
      "chr8 118 pax2.fimo\n",
      "chr8 119 pax3_pax7.fimo\n",
      "chr8 120 pax5.fimo\n",
      "chr8 121 pax6.fimo\n",
      "chr8 122 pbx1.fimo\n",
      "chr8 123 phox2_prop.fimo\n",
      "chr8 124 pknox_tgif.fimo\n",
      "chr8 125 plag1.fimo\n",
      "chr8 126 pou1f1.fimo\n",
      "chr8 127 pou2-3-5.fimo\n",
      "chr8 128 pou4.fimo\n",
      "chr8 129 pou6f2.fimo\n",
      "chr8 130 pparg.fimo\n",
      "chr8 131 pparg_rxra.fimo\n",
      "chr8 132 prdm1.fimo\n",
      "chr8 133 prox1.fimo\n",
      "chr8 134 rar.fimo\n",
      "chr8 135 rar_rxr.fimo\n",
      "chr8 136 rargvar.2.fimo\n",
      "chr8 137 rel.fimo\n",
      "chr8 138 rest.fimo\n",
      "chr8 139 rfx.fimo\n",
      "chr8 140 rhox11.fimo\n",
      "chr8 141 rora.fimo\n",
      "chr8 142 roravar.2.fimo\n",
      "chr8 143 rreb1.fimo\n",
      "chr8 144 runx.fimo\n",
      "chr8 145 rxravdr.fimo\n",
      "chr8 146 scrt.fimo\n",
      "chr8 147 six3.fimo\n",
      "chr8 148 smad3.fimo\n",
      "chr8 149 sox1-l1-21-4.fimo\n",
      "chr8 150 sox10.fimo\n",
      "chr8 151 sox17.fimo\n",
      "chr8 152 sox2.fimo\n",
      "chr8 153 sox3_sox6.fimo\n",
      "chr8 154 sox5_sox9_sry.fimo\n",
      "chr8 155 sox8.fimo\n",
      "chr8 156 sp1.fimo\n",
      "chr8 157 sp4.fimo\n",
      "chr8 158 spi1-c.fimo\n",
      "chr8 159 spib.fimo\n",
      "chr8 160 spz1.fimo\n",
      "chr8 161 srebpf.fimo\n",
      "chr8 162 srf.fimo\n",
      "chr8 163 stat1-3.fimo\n",
      "chr8 164 stat6.fimo\n",
      "chr8 165 t.fimo\n",
      "chr8 166 tbp.fimo\n",
      "chr8 167 tbx.fimo\n",
      "chr8 168 tcf21.fimo\n",
      "chr8 169 tcf7.fimo\n",
      "chr8 170 tcfl5.fimo\n",
      "chr8 171 tead.fimo\n",
      "chr8 172 tfap1.fimo\n",
      "chr8 173 tfap2-3.fimo\n",
      "chr8 174 tfcp2.fimo\n",
      "chr8 175 thap1.fimo\n",
      "chr8 176 tp53.fimo\n",
      "chr8 177 tp63.fimo\n",
      "chr8 178 tp73.fimo\n",
      "chr8 179 usf.fimo\n",
      "chr8 180 ventx.fimo\n",
      "chr8 181 xbp1.fimo\n",
      "chr8 182 yy1.fimo\n",
      "chr8 183 yy2.fimo\n",
      "chr8 184 zbed1.fimo\n",
      "chr8 185 zbtb18.fimo\n",
      "chr8 186 zbtb33.fimo\n",
      "chr8 187 zbtb7.fimo\n",
      "chr8 188 zeb1.fimo\n",
      "chr8 189 zfx.fimo\n",
      "chr8 190 zic.fimo\n",
      "chr8 191 znf143.fimo\n",
      "chr8 192 znf263.fimo\n",
      "chr8 193 znf354c.fimo\n",
      "chr8 194 znf410.fimo\n",
      "chr8 195 znf423.fimo\n",
      "chr8 196 znf740.fimo\n",
      "chr9 1 alx1_alx4_arx.fimo\n",
      "chr9 2 ap-1.fimo\n",
      "chr9 3 ar_nr3c1_nr3c2.fimo\n",
      "chr9 4 arid3a.fimo\n",
      "chr9 5 arid3b.fimo\n",
      "chr9 6 arid5a.fimo\n",
      "chr9 7 arnt_mycn.fimo\n",
      "chr9 8 arntl_mitf.fimo\n",
      "chr9 9 ascl2_nhlh1.fimo\n",
      "chr9 10 atf7_batf3_creb5.fimo\n",
      "chr9 11 bcl6.fimo\n",
      "chr9 12 bcl6b.fimo\n",
      "chr9 13 bhlh.fimo\n",
      "chr9 14 bhlha15.fimo\n",
      "chr9 15 bhlhe23.fimo\n",
      "chr9 16 cdx.fimo\n",
      "chr9 17 cebp.fimo\n",
      "chr9 18 cenpb.fimo\n",
      "chr9 19 creb3-l1.fimo\n",
      "chr9 20 crem.fimo\n",
      "chr9 21 ctcf.fimo\n",
      "chr9 22 cux.fimo\n",
      "chr9 23 dbp_hlf_tef.fimo\n",
      "chr9 24 dmbx1.fimo\n",
      "chr9 25 dmrt3.fimo\n",
      "chr9 26 dux.fimo\n",
      "chr9 27 dux4.fimo\n",
      "chr9 28 duxa.fimo\n",
      "chr9 29 e2f1.fimo\n",
      "chr9 30 e2f2.fimo\n",
      "chr9 31 e2f4_e2f6.fimo\n",
      "chr9 32 e2f7.fimo\n",
      "chr9 33 e2f8.fimo\n",
      "chr9 34 ebf1.fimo\n",
      "chr9 35 ebox.fimo\n",
      "chr9 36 egr.fimo\n",
      "chr9 37 elf.fimo\n",
      "chr9 38 elk_etv.fimo\n",
      "chr9 39 esr1.fimo\n",
      "chr9 40 esr2.fimo\n",
      "chr9 41 esrr.fimo\n",
      "chr9 42 ewsr1-fli1.fimo\n",
      "chr9 43 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chr9 44 fox.fimo\n",
      "chr9 45 gata.fimo\n",
      "chr9 46 gcm.fimo\n",
      "chr9 47 gfi1.fimo\n",
      "chr9 48 gfi1b.fimo\n",
      "chr9 49 gli2.fimo\n",
      "chr9 50 glis.fimo\n",
      "chr9 51 gmeb2.fimo\n",
      "chr9 52 grhl1.fimo\n",
      "chr9 53 gsc.fimo\n",
      "chr9 54 hes.fimo\n",
      "chr9 55 hic.fimo\n",
      "chr9 56 hinfp.fimo\n",
      "chr9 57 hltf.fimo\n",
      "chr9 58 hnf1.fimo\n",
      "chr9 59 hnf4a.fimo\n",
      "chr9 60 hnf4g.fimo\n",
      "chr9 61 homeobox-1.fimo\n",
      "chr9 62 homeobox-2.fimo\n",
      "chr9 63 homeobox-3.fimo\n",
      "chr9 64 homeobox-4.fimo\n",
      "chr9 65 homeobox-5.fimo\n",
      "chr9 66 hoxa11.fimo\n",
      "chr9 67 hoxa5.fimo\n",
      "chr9 68 hoxa9.fimo\n",
      "chr9 69 hoxb5_hoxd3.fimo\n",
      "chr9 70 hoxc13.fimo\n",
      "chr9 71 hoxc9.fimo\n",
      "chr9 72 hoxd8.fimo\n",
      "chr9 73 hsf.fimo\n",
      "chr9 74 id2_mlxip.fimo\n",
      "chr9 75 insm1.fimo\n",
      "chr9 76 irf1.fimo\n",
      "chr9 77 irf2.fimo\n",
      "chr9 78 irf7-8-9.fimo\n",
      "chr9 79 isl2_nkx3-1_nkx3-2.fimo\n",
      "chr9 80 klf.fimo\n",
      "chr9 81 lef1.fimo\n",
      "chr9 82 lhx3.fimo\n",
      "chr9 83 lin54.fimo\n",
      "chr9 84 maf_nrl.fimo\n",
      "chr9 85 mafg.fimo\n",
      "chr9 86 mecom.fimo\n",
      "chr9 87 mef2a-b-d.fimo\n",
      "chr9 88 mef2c.fimo\n",
      "chr9 89 meis.fimo\n",
      "chr9 90 mga_tbx.fimo\n",
      "chr9 91 msc_myf6_tfap4.fimo\n",
      "chr9 92 mtf1.fimo\n",
      "chr9 93 myb.fimo\n",
      "chr9 94 mybl1.fimo\n",
      "chr9 95 mybl2.fimo\n",
      "chr9 96 myc.fimo\n",
      "chr9 97 myog_tcf12.fimo\n",
      "chr9 98 mzf1.fimo\n",
      "chr9 99 mzf1var.2.fimo\n",
      "chr9 100 nfat.fimo\n",
      "chr9 101 nfi.fimo\n",
      "chr9 102 nfil3.fimo\n",
      "chr9 103 nfkb.fimo\n",
      "chr9 104 nfya.fimo\n",
      "chr9 105 nfyb.fimo\n",
      "chr9 106 nkx2-3_nkx2-8.fimo\n",
      "chr9 107 nkx2-5.fimo\n",
      "chr9 108 nkx2-5var.2.fimo\n",
      "chr9 109 nr2_rxr.fimo\n",
      "chr9 110 nr2e1.fimo\n",
      "chr9 111 nr2e3.fimo\n",
      "chr9 112 nr2f1.fimo\n",
      "chr9 113 nr4a2.fimo\n",
      "chr9 114 nr5a2.fimo\n",
      "chr9 115 nrf1.fimo\n",
      "chr9 116 onecut.fimo\n",
      "chr9 117 pax1_pax9.fimo\n",
      "chr9 118 pax2.fimo\n",
      "chr9 119 pax3_pax7.fimo\n",
      "chr9 120 pax5.fimo\n",
      "chr9 121 pax6.fimo\n",
      "chr9 122 pbx1.fimo\n",
      "chr9 123 phox2_prop.fimo\n",
      "chr9 124 pknox_tgif.fimo\n",
      "chr9 125 plag1.fimo\n",
      "chr9 126 pou1f1.fimo\n",
      "chr9 127 pou2-3-5.fimo\n",
      "chr9 128 pou4.fimo\n",
      "chr9 129 pou6f2.fimo\n",
      "chr9 130 pparg.fimo\n",
      "chr9 131 pparg_rxra.fimo\n",
      "chr9 132 prdm1.fimo\n",
      "chr9 133 prox1.fimo\n",
      "chr9 134 rar.fimo\n",
      "chr9 135 rar_rxr.fimo\n",
      "chr9 136 rargvar.2.fimo\n",
      "chr9 137 rel.fimo\n",
      "chr9 138 rest.fimo\n",
      "chr9 139 rfx.fimo\n",
      "chr9 140 rhox11.fimo\n",
      "chr9 141 rora.fimo\n",
      "chr9 142 roravar.2.fimo\n",
      "chr9 143 rreb1.fimo\n",
      "chr9 144 runx.fimo\n",
      "chr9 145 rxravdr.fimo\n",
      "chr9 146 scrt.fimo\n",
      "chr9 147 six3.fimo\n",
      "chr9 148 smad3.fimo\n",
      "chr9 149 sox1-l1-21-4.fimo\n",
      "chr9 150 sox10.fimo\n",
      "chr9 151 sox17.fimo\n",
      "chr9 152 sox2.fimo\n",
      "chr9 153 sox3_sox6.fimo\n",
      "chr9 154 sox5_sox9_sry.fimo\n",
      "chr9 155 sox8.fimo\n",
      "chr9 156 sp1.fimo\n",
      "chr9 157 sp4.fimo\n",
      "chr9 158 spi1-c.fimo\n",
      "chr9 159 spib.fimo\n",
      "chr9 160 spz1.fimo\n",
      "chr9 161 srebpf.fimo\n",
      "chr9 162 srf.fimo\n",
      "chr9 163 stat1-3.fimo\n",
      "chr9 164 stat6.fimo\n",
      "chr9 165 t.fimo\n",
      "chr9 166 tbp.fimo\n",
      "chr9 167 tbx.fimo\n",
      "chr9 168 tcf21.fimo\n",
      "chr9 169 tcf7.fimo\n",
      "chr9 170 tcfl5.fimo\n",
      "chr9 171 tead.fimo\n",
      "chr9 172 tfap1.fimo\n",
      "chr9 173 tfap2-3.fimo\n",
      "chr9 174 tfcp2.fimo\n",
      "chr9 175 thap1.fimo\n",
      "chr9 176 tp53.fimo\n",
      "chr9 177 tp63.fimo\n",
      "chr9 178 tp73.fimo\n",
      "chr9 179 usf.fimo\n",
      "chr9 180 ventx.fimo\n",
      "chr9 181 xbp1.fimo\n",
      "chr9 182 yy1.fimo\n",
      "chr9 183 yy2.fimo\n",
      "chr9 184 zbed1.fimo\n",
      "chr9 185 zbtb18.fimo\n",
      "chr9 186 zbtb33.fimo\n",
      "chr9 187 zbtb7.fimo\n",
      "chr9 188 zeb1.fimo\n",
      "chr9 189 zfx.fimo\n",
      "chr9 190 zic.fimo\n",
      "chr9 191 znf143.fimo\n",
      "chr9 192 znf263.fimo\n",
      "chr9 193 znf354c.fimo\n",
      "chr9 194 znf410.fimo\n",
      "chr9 195 znf423.fimo\n",
      "chr9 196 znf740.fimo\n",
      "chrX 1 alx1_alx4_arx.fimo\n",
      "chrX 2 ap-1.fimo\n",
      "chrX 3 ar_nr3c1_nr3c2.fimo\n",
      "chrX 4 arid3a.fimo\n",
      "chrX 5 arid3b.fimo\n",
      "chrX 6 arid5a.fimo\n",
      "chrX 7 arnt_mycn.fimo\n",
      "chrX 8 arntl_mitf.fimo\n",
      "chrX 9 ascl2_nhlh1.fimo\n",
      "chrX 10 atf7_batf3_creb5.fimo\n",
      "chrX 11 bcl6.fimo\n",
      "chrX 12 bcl6b.fimo\n",
      "chrX 13 bhlh.fimo\n",
      "chrX 14 bhlha15.fimo\n",
      "chrX 15 bhlhe23.fimo\n",
      "chrX 16 cdx.fimo\n",
      "chrX 17 cebp.fimo\n",
      "chrX 18 cenpb.fimo\n",
      "chrX 19 creb3-l1.fimo\n",
      "chrX 20 crem.fimo\n",
      "chrX 21 ctcf.fimo\n",
      "chrX 22 cux.fimo\n",
      "chrX 23 dbp_hlf_tef.fimo\n",
      "chrX 24 dmbx1.fimo\n",
      "chrX 25 dmrt3.fimo\n",
      "chrX 26 dux.fimo\n",
      "chrX 27 dux4.fimo\n",
      "chrX 28 duxa.fimo\n",
      "chrX 29 e2f1.fimo\n",
      "chrX 30 e2f2.fimo\n",
      "chrX 31 e2f4_e2f6.fimo\n",
      "chrX 32 e2f7.fimo\n",
      "chrX 33 e2f8.fimo\n",
      "chrX 34 ebf1.fimo\n",
      "chrX 35 ebox.fimo\n",
      "chrX 36 egr.fimo\n",
      "chrX 37 elf.fimo\n",
      "chrX 38 elk_etv.fimo\n",
      "chrX 39 esr1.fimo\n",
      "chrX 40 esr2.fimo\n",
      "chrX 41 esrr.fimo\n",
      "chrX 42 ewsr1-fli1.fimo\n",
      "chrX 43 figla_id4_snai2_tcf3_tcf4.fimo\n",
      "chrX 44 fox.fimo\n",
      "chrX 45 gata.fimo\n",
      "chrX 46 gcm.fimo\n",
      "chrX 47 gfi1.fimo\n",
      "chrX 48 gfi1b.fimo\n",
      "chrX 49 gli2.fimo\n",
      "chrX 50 glis.fimo\n",
      "chrX 51 gmeb2.fimo\n",
      "chrX 52 grhl1.fimo\n",
      "chrX 53 gsc.fimo\n",
      "chrX 54 hes.fimo\n",
      "chrX 55 hic.fimo\n",
      "chrX 56 hinfp.fimo\n",
      "chrX 57 hltf.fimo\n",
      "chrX 58 hnf1.fimo\n",
      "chrX 59 hnf4a.fimo\n",
      "chrX 60 hnf4g.fimo\n",
      "chrX 61 homeobox-1.fimo\n",
      "chrX 62 homeobox-2.fimo\n",
      "chrX 63 homeobox-3.fimo\n",
      "chrX 64 homeobox-4.fimo\n",
      "chrX 65 homeobox-5.fimo\n",
      "chrX 66 hoxa11.fimo\n",
      "chrX 67 hoxa5.fimo\n",
      "chrX 68 hoxa9.fimo\n",
      "chrX 69 hoxb5_hoxd3.fimo\n",
      "chrX 70 hoxc13.fimo\n",
      "chrX 71 hoxc9.fimo\n",
      "chrX 72 hoxd8.fimo\n",
      "chrX 73 hsf.fimo\n",
      "chrX 74 id2_mlxip.fimo\n",
      "chrX 75 insm1.fimo\n",
      "chrX 76 irf1.fimo\n",
      "chrX 77 irf2.fimo\n",
      "chrX 78 irf7-8-9.fimo\n",
      "chrX 79 isl2_nkx3-1_nkx3-2.fimo\n",
      "chrX 80 klf.fimo\n",
      "chrX 81 lef1.fimo\n",
      "chrX 82 lhx3.fimo\n",
      "chrX 83 lin54.fimo\n",
      "chrX 84 maf_nrl.fimo\n",
      "chrX 85 mafg.fimo\n",
      "chrX 86 mecom.fimo\n",
      "chrX 87 mef2a-b-d.fimo\n",
      "chrX 88 mef2c.fimo\n",
      "chrX 89 meis.fimo\n",
      "chrX 90 mga_tbx.fimo\n",
      "chrX 91 msc_myf6_tfap4.fimo\n",
      "chrX 92 mtf1.fimo\n",
      "chrX 93 myb.fimo\n",
      "chrX 94 mybl1.fimo\n",
      "chrX 95 mybl2.fimo\n",
      "chrX 96 myc.fimo\n",
      "chrX 97 myog_tcf12.fimo\n",
      "chrX 98 mzf1.fimo\n",
      "chrX 99 mzf1var.2.fimo\n",
      "chrX 100 nfat.fimo\n",
      "chrX 101 nfi.fimo\n",
      "chrX 102 nfil3.fimo\n",
      "chrX 103 nfkb.fimo\n",
      "chrX 104 nfya.fimo\n",
      "chrX 105 nfyb.fimo\n",
      "chrX 106 nkx2-3_nkx2-8.fimo\n",
      "chrX 107 nkx2-5.fimo\n",
      "chrX 108 nkx2-5var.2.fimo\n",
      "chrX 109 nr2_rxr.fimo\n",
      "chrX 110 nr2e1.fimo\n",
      "chrX 111 nr2e3.fimo\n",
      "chrX 112 nr2f1.fimo\n",
      "chrX 113 nr4a2.fimo\n",
      "chrX 114 nr5a2.fimo\n",
      "chrX 115 nrf1.fimo\n",
      "chrX 116 onecut.fimo\n",
      "chrX 117 pax1_pax9.fimo\n",
      "chrX 118 pax2.fimo\n",
      "chrX 119 pax3_pax7.fimo\n",
      "chrX 120 pax5.fimo\n",
      "chrX 121 pax6.fimo\n",
      "chrX 122 pbx1.fimo\n",
      "chrX 123 phox2_prop.fimo\n",
      "chrX 124 pknox_tgif.fimo\n",
      "chrX 125 plag1.fimo\n",
      "chrX 126 pou1f1.fimo\n",
      "chrX 127 pou2-3-5.fimo\n",
      "chrX 128 pou4.fimo\n",
      "chrX 129 pou6f2.fimo\n",
      "chrX 130 pparg.fimo\n",
      "chrX 131 pparg_rxra.fimo\n",
      "chrX 132 prdm1.fimo\n",
      "chrX 133 prox1.fimo\n",
      "chrX 134 rar.fimo\n",
      "chrX 135 rar_rxr.fimo\n",
      "chrX 136 rargvar.2.fimo\n",
      "chrX 137 rel.fimo\n",
      "chrX 138 rest.fimo\n",
      "chrX 139 rfx.fimo\n",
      "chrX 140 rhox11.fimo\n",
      "chrX 141 rora.fimo\n",
      "chrX 142 roravar.2.fimo\n",
      "chrX 143 rreb1.fimo\n",
      "chrX 144 runx.fimo\n",
      "chrX 145 rxravdr.fimo\n",
      "chrX 146 scrt.fimo\n",
      "chrX 147 six3.fimo\n",
      "chrX 148 smad3.fimo\n",
      "chrX 149 sox1-l1-21-4.fimo\n",
      "chrX 150 sox10.fimo\n",
      "chrX 151 sox17.fimo\n",
      "chrX 152 sox2.fimo\n",
      "chrX 153 sox3_sox6.fimo\n",
      "chrX 154 sox5_sox9_sry.fimo\n",
      "chrX 155 sox8.fimo\n",
      "chrX 156 sp1.fimo\n",
      "chrX 157 sp4.fimo\n",
      "chrX 158 spi1-c.fimo\n",
      "chrX 159 spib.fimo\n",
      "chrX 160 spz1.fimo\n",
      "chrX 161 srebpf.fimo\n",
      "chrX 162 srf.fimo\n",
      "chrX 163 stat1-3.fimo\n",
      "chrX 164 stat6.fimo\n",
      "chrX 165 t.fimo\n",
      "chrX 166 tbp.fimo\n",
      "chrX 167 tbx.fimo\n",
      "chrX 168 tcf21.fimo\n",
      "chrX 169 tcf7.fimo\n",
      "chrX 170 tcfl5.fimo\n",
      "chrX 171 tead.fimo\n",
      "chrX 172 tfap1.fimo\n",
      "chrX 173 tfap2-3.fimo\n",
      "chrX 174 tfcp2.fimo\n",
      "chrX 175 thap1.fimo\n",
      "chrX 176 tp53.fimo\n",
      "chrX 177 tp63.fimo\n",
      "chrX 178 tp73.fimo\n",
      "chrX 179 usf.fimo\n",
      "chrX 180 ventx.fimo\n",
      "chrX 181 xbp1.fimo\n",
      "chrX 182 yy1.fimo\n",
      "chrX 183 yy2.fimo\n",
      "chrX 184 zbed1.fimo\n",
      "chrX 185 zbtb18.fimo\n",
      "chrX 186 zbtb33.fimo\n",
      "chrX 187 zbtb7.fimo\n",
      "chrX 188 zeb1.fimo\n",
      "chrX 189 zfx.fimo\n",
      "chrX 190 zic.fimo\n",
      "chrX 191 znf143.fimo\n",
      "chrX 192 znf263.fimo\n",
      "chrX 193 znf354c.fimo\n",
      "chrX 194 znf410.fimo\n",
      "chrX 195 znf423.fimo\n",
      "chrX 196 znf740.fimo\n"
     ]
    }
   ],
   "source": [
    "###  create pickles for each chromosome\n",
    "! if [ ! -d ./pickles ]; then mkdir ./pickles; fi\n",
    "motif_dir = '/home/jenhan/analysis/cobinding_motif_analysis/fimo_motifs/'\n",
    "\n",
    "peakDirectory =  './group_by_chromosome/'\n",
    "scaler = preprocessing.MinMaxScaler()\n",
    "\n",
    "\n",
    "for chrom in sorted(os.listdir(peakDirectory)):\n",
    "    peak_frame = pd.read_csv('./group_by_chromosome/' + chrom + '/' + chrom + '_tile_peaks.tsv', sep='\\t')\n",
    "    columns = peak_frame.columns.values\n",
    "    columns[0] = 'ID'\n",
    "    peak_frame.columns = columns\n",
    "    peak_frame['Factors'] = ['atac' if not 'tile' in x else 'tile' for x in peak_frame['ID'].values]\n",
    "    \n",
    "#     peak_start_dict = dict(zip(peak_frame['ID'].values, peak_frame['start'].values))\n",
    "\n",
    "    motif_score_frame = peak_frame[['ID', 'Factors', 'chr']]\n",
    "    motif_score_frame.index=motif_score_frame['ID'].values\n",
    "\n",
    "#     motif_sequence_frame = peak_frame[['ID', 'Factors', 'chr']]\n",
    "#     motif_sequence_frame.index=motif_score_frame['ID'].values\n",
    "\n",
    "#     motif_strand_frame = peak_frame[['ID', 'Factors', 'chr']]\n",
    "#     motif_strand_frame.index=motif_score_frame['ID'].values\n",
    "\n",
    "#     motif_start_frame = peak_frame[['ID', 'Factors', 'chr']]\n",
    "#     motif_start_frame.index=motif_score_frame['ID'].values\n",
    "\n",
    "#     motif_end_frame = peak_frame[['ID', 'Factors', 'chr']]\n",
    "#     motif_end_frame.index=motif_score_frame['ID'].values\n",
    "\n",
    "#     motif_count_frame = peak_frame[['ID', 'Factors', 'chr']]\n",
    "#     motif_count_frame.index = motif_count_frame['ID'].values\n",
    "\n",
    "    counter=0\n",
    "    for m in sorted(os.listdir(motif_dir)):\n",
    "        counter+=1\n",
    "\n",
    "        if '.fimo' in m:\n",
    "            print(chrom, counter,m)\n",
    "            motif_results = './fimo_results/' + chrom + '_' + m.replace('.fimo','') +'.txt'\n",
    "            fimo_result_frame=pd.read_csv(motif_results, \n",
    "                                          skiprows=1,\n",
    "                                          names=['motif_name', \n",
    "                                                 'peak_id', \n",
    "                                                 'start', \n",
    "                                                 'stop', \n",
    "                                                 'strand', \n",
    "                                                 'score', \n",
    "                                                 'pvalue', \n",
    "                                                 'sequence'],\n",
    "                                          sep='\\t')\n",
    "            motif_name = m.replace('.fimo','')\n",
    "            id_score_dict = {}\n",
    "#             id_strand_dict = {}\n",
    "#             id_sequence_dict = {}\n",
    "#             id_start_dict = {}\n",
    "#             id_end_dict = {}\n",
    "#             id_count_dict = {}\n",
    "            \n",
    "            ids = fimo_result_frame['peak_id'].values\n",
    "            scores = fimo_result_frame['score'].values\n",
    "#             strands = fimo_result_frame['strand'].values\n",
    "#             sequences = fimo_result_frame['sequence']\n",
    "#             starts = fimo_result_frame['start']\n",
    "#             ends = fimo_result_frame['stop']\n",
    "\n",
    "            for i in range(len(ids)):\n",
    "                currentScore = float(scores[i])\n",
    "#                 currentSequence = sequences[i]\n",
    "#                 currentStrand = strands[i]\n",
    "#                 currentStart = int(starts[i])\n",
    "#                 currentEnd = int(ends[i])\n",
    "                if currentScore < 0.0:\n",
    "                    currentScore = 0.0\n",
    "                if ids[i] in id_score_dict:\n",
    "                    if currentScore > id_score_dict[ids[i]]:\n",
    "                        id_score_dict[ids[i]] = currentScore\n",
    "#                         id_sequence_dict[ids[i]] = currentSequence\n",
    "#                         id_strand_dict[ids[i]] = currentStrand\n",
    "#                         id_start_dict[ids[i]] = currentStart\n",
    "#                         id_end_dict[ids[i]] = currentEnd\n",
    "#                         id_count_dict[ids[i]] += 1\n",
    "                else:\n",
    "                    id_score_dict[ids[i]] = currentScore\n",
    "#                     id_sequence_dict[ids[i]] = currentSequence\n",
    "#                     id_strand_dict[ids[i]] = currentStrand\n",
    "#                     id_start_dict[ids[i]] = currentStart\n",
    "#                     id_end_dict[ids[i]] = currentEnd\n",
    "#                     id_count_dict[ids[i]] = 0\n",
    "\n",
    "            motif_score_frame[motif_name] = [id_score_dict[x] if x in id_score_dict else 0 for x in motif_score_frame['ID'].values]\n",
    "#             motif_sequence_frame[motif_name] = [id_sequence_dict[x] if x in id_sequence_dict else '?' for x in motif_sequence_frame['ID'].values]\n",
    "#             motif_strand_frame[motif_name] = [id_strand_dict[x] if x in id_score_dict else '' for x in motif_strand_frame['ID'].values]\n",
    "#             motif_start_frame[motif_name] = [id_start_dict[x] + peak_start_dict[x] if x in id_start_dict else -1 for x in motif_start_frame['ID'].values]\n",
    "#             motif_end_frame[motif_name] = [id_end_dict[x] + peak_start_dict[x] if x in id_end_dict else -1 for x in motif_end_frame['ID'].values]\n",
    "#             motif_count_frame[motif_name] = [id_count_dict[x] if x in id_count_dict else 0 for x in motif_count_frame['ID'].values]\n",
    "\n",
    "    motif_score_frame.to_pickle('./pickles/' + chrom + '_motif_score_frame.pickle')\n",
    "\n",
    "#     motif_sequence_frame.to_pickle('./pickles/' + chrom + '_motif_sequence_frame.pickle')\n",
    "\n",
    "#     motif_strand_frame.to_pickle('./pickles/' + chrom + '_motif_strand_frame.pickle')\n",
    "\n",
    "#     motif_start_frame.to_pickle('./pickles/' + chrom + '_motif_start_frame.pickle')\n",
    "\n",
    "#     motif_end_frame.to_pickle('./pickles/' + chrom + '_motif_end_frame.pickle')\n",
    "\n",
    "#     motif_count_frame.to_pickle('./pickles/' + chrom + '_motif_count_frame.pickle')\n",
    "\n",
    "    normed_motif_frame = pd.DataFrame(scaler.fit_transform(motif_score_frame.ix[:,3:]))\n",
    "    normed_motif_frame.columns = motif_score_frame.columns.values[3:]\n",
    "    normed_motif_frame.index = motif_score_frame.index.values\n",
    "    \n",
    "    normed_motif_frame.to_pickle('./pickles/' + chrom + '_normed_motif_frame.pickle')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Logistic Regression on all Chromosomes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "training on  chr1 testing on  chr1\n",
      "training on  chr1 testing on  chr10\n",
      "chr1 chr10 0.970243041902 0.403462094146\n",
      "training on  chr1 testing on  chr11\n",
      "chr1 chr11 0.971258706813 0.437662054684\n",
      "training on  chr1 testing on  chr12\n",
      "chr1 chr12 0.967603345275 0.33508062234\n",
      "training on  chr1 testing on  chr13\n",
      "chr1 chr13 0.960285500933 0.332330641053\n",
      "training on  chr1 testing on  chr14\n",
      "chr1 chr14 0.971966768158 0.355216655196\n",
      "training on  chr1 testing on  chr15\n",
      "chr1 chr15 0.970657885872 0.391986424239\n",
      "training on  chr1 testing on  chr16\n",
      "chr1 chr16 0.971054003027 0.385842652165\n",
      "training on  chr1 testing on  chr17\n",
      "chr1 chr17 0.967248962819 0.378665413736\n",
      "training on  chr1 testing on  chr18\n",
      "chr1 chr18 0.966060376908 0.345431730266\n",
      "training on  chr1 testing on  chr19\n",
      "chr1 chr19 0.975728063484 0.434762225123\n",
      "training on  chr1 testing on  chr2\n",
      "chr1 chr2 0.969383593667 0.389530358\n",
      "training on  chr1 testing on  chr3\n",
      "chr1 chr3 0.970662351663 0.354604328071\n",
      "training on  chr1 testing on  chr4\n",
      "chr1 chr4 0.970697058741 0.396148199192\n",
      "training on  chr1 testing on  chr5\n",
      "chr1 chr5 0.97047677004 0.375254492803\n",
      "training on  chr1 testing on  chr6\n",
      "chr1 chr6 0.970642793732 0.391861268733\n",
      "training on  chr1 testing on  chr7\n",
      "chr1 chr7 0.971386112585 0.40956333689\n",
      "training on  chr1 testing on  chr8\n",
      "chr1 chr8 0.970849051244 0.402651887057\n",
      "training on  chr1 testing on  chr9\n",
      "chr1 chr9 0.966431638342 0.382035491052\n",
      "training on  chr1 testing on  chrX\n",
      "chr1 chrX 0.974715018621 0.308888248322\n",
      "training on  chr10 testing on  chr1\n",
      "chr10 chr1 0.971358014461 0.363035647552\n",
      "training on  chr10 testing on  chr10\n",
      "training on  chr10 testing on  chr11\n",
      "chr10 chr11 0.971598981879 0.423162000276\n",
      "training on  chr10 testing on  chr12\n",
      "chr10 chr12 0.967174643149 0.329657300849\n",
      "training on  chr10 testing on  chr13\n",
      "chr10 chr13 0.961246870988 0.326112309729\n",
      "training on  chr10 testing on  chr14\n",
      "chr10 chr14 0.971987853192 0.336614868833\n",
      "training on  chr10 testing on  chr15\n",
      "chr10 chr15 0.970689741786 0.383212640139\n",
      "training on  chr10 testing on  chr16\n",
      "chr10 chr16 0.970508828159 0.373098876632\n",
      "training on  chr10 testing on  chr17\n",
      "chr10 chr17 0.967818654899 0.370450414579\n",
      "training on  chr10 testing on  chr18\n",
      "chr10 chr18 0.966621204442 0.343127284104\n",
      "training on  chr10 testing on  chr19\n",
      "chr10 chr19 0.976071373697 0.422049702912\n",
      "training on  chr10 testing on  chr2\n",
      "chr10 chr2 0.969335443028 0.382506354698\n",
      "training on  chr10 testing on  chr3\n",
      "chr10 chr3 0.969809428949 0.352277906323\n",
      "training on  chr10 testing on  chr4\n",
      "chr10 chr4 0.970721766506 0.38843811649\n",
      "training on  chr10 testing on  chr5\n",
      "chr10 chr5 0.970810682696 0.367341306838\n",
      "training on  chr10 testing on  chr6\n",
      "chr10 chr6 0.970104123834 0.381607325657\n",
      "training on  chr10 testing on  chr7\n",
      "chr10 chr7 0.970489631291 0.399113316741\n",
      "training on  chr10 testing on  chr8\n",
      "chr10 chr8 0.970620244254 0.39406127624\n",
      "training on  chr10 testing on  chr9\n",
      "chr10 chr9 0.966338747021 0.368555200727\n",
      "training on  chr10 testing on  chrX\n",
      "chr10 chrX 0.973420470483 0.296165887261\n",
      "training on  chr11 testing on  chr1\n",
      "chr11 chr1 0.966876172556 0.365523489682\n",
      "training on  chr11 testing on  chr10\n",
      "chr11 chr10 0.966992845012 0.3934791855\n",
      "training on  chr11 testing on  chr11\n",
      "training on  chr11 testing on  chr12\n",
      "chr11 chr12 0.964495707823 0.336981867122\n",
      "training on  chr11 testing on  chr13\n",
      "chr11 chr13 0.957453570894 0.330909233599\n",
      "training on  chr11 testing on  chr14\n",
      "chr11 chr14 0.968494578343 0.346523577718\n",
      "training on  chr11 testing on  chr15\n",
      "chr11 chr15 0.967857530224 0.392769989791\n",
      "training on  chr11 testing on  chr16\n",
      "chr11 chr16 0.967429810288 0.381107209378\n",
      "training on  chr11 testing on  chr17\n",
      "chr11 chr17 0.96580444885 0.393945554903\n",
      "training on  chr11 testing on  chr18\n",
      "chr11 chr18 0.96181924585 0.340203269621\n",
      "training on  chr11 testing on  chr19\n",
      "chr11 chr19 0.974313759601 0.438760648304\n",
      "training on  chr11 testing on  chr2\n",
      "chr11 chr2 0.964701951829 0.397064033362\n",
      "training on  chr11 testing on  chr3\n",
      "chr11 chr3 0.966089957476 0.35188184723\n",
      "training on  chr11 testing on  chr4\n",
      "chr11 chr4 0.968661769467 0.405974087829\n",
      "training on  chr11 testing on  chr5\n",
      "chr11 chr5 0.967851855999 0.387268902835\n",
      "training on  chr11 testing on  chr6\n",
      "chr11 chr6 0.965686626422 0.383585110605\n",
      "training on  chr11 testing on  chr7\n",
      "chr11 chr7 0.969311939668 0.422731823083\n",
      "training on  chr11 testing on  chr8\n",
      "chr11 chr8 0.968600761042 0.412467123027\n",
      "training on  chr11 testing on  chr9\n",
      "chr11 chr9 0.965047555128 0.391079051292\n",
      "training on  chr11 testing on  chrX\n",
      "chr11 chrX 0.970210427797 0.322298918623\n",
      "training on  chr12 testing on  chr1\n",
      "chr12 chr1 0.969669388781 0.36620091381\n",
      "training on  chr12 testing on  chr10\n",
      "chr12 chr10 0.969469226554 0.397289843289\n",
      "training on  chr12 testing on  chr11\n",
      "chr12 chr11 0.971090005177 0.437531104733\n",
      "training on  chr12 testing on  chr12\n",
      "training on  chr12 testing on  chr13\n",
      "chr12 chr13 0.959171111477 0.326341000831\n",
      "training on  chr12 testing on  chr14\n",
      "chr12 chr14 0.971371272519 0.353203391124\n",
      "training on  chr12 testing on  chr15\n",
      "chr12 chr15 0.970168280494 0.388432429832\n",
      "training on  chr12 testing on  chr16\n",
      "chr12 chr16 0.970109944442 0.384523533393\n",
      "training on  chr12 testing on  chr17\n",
      "chr12 chr17 0.967026686896 0.379093792589\n",
      "training on  chr12 testing on  chr18\n",
      "chr12 chr18 0.96291431507 0.345013726914\n",
      "training on  chr12 testing on  chr19\n",
      "chr12 chr19 0.975077022996 0.434604187588\n",
      "training on  chr12 testing on  chr2\n",
      "chr12 chr2 0.968146091995 0.387889303227\n",
      "training on  chr12 testing on  chr3\n",
      "chr12 chr3 0.967883395987 0.350863404497\n",
      "training on  chr12 testing on  chr4\n",
      "chr12 chr4 0.969435640466 0.392951874235\n",
      "training on  chr12 testing on  chr5\n",
      "chr12 chr5 0.969511321232 0.374246421686\n",
      "training on  chr12 testing on  chr6\n",
      "chr12 chr6 0.968907717874 0.386949592727\n",
      "training on  chr12 testing on  chr7\n",
      "chr12 chr7 0.970268624234 0.410863723892\n",
      "training on  chr12 testing on  chr8\n",
      "chr12 chr8 0.969724690492 0.400947592572\n",
      "training on  chr12 testing on  chr9\n",
      "chr12 chr9 0.965258140592 0.375203507021\n",
      "training on  chr12 testing on  chrX\n",
      "chr12 chrX 0.972739423997 0.304723645159\n",
      "training on  chr13 testing on  chr1\n",
      "chr13 chr1 0.97056558428 0.368173108822\n",
      "training on  chr13 testing on  chr10\n",
      "chr13 chr10 0.970888350295 0.400410615658\n",
      "training on  chr13 testing on  chr11\n",
      "chr13 chr11 0.972204699279 0.433898178904\n",
      "training on  chr13 testing on  chr12\n",
      "chr13 chr12 0.966819501329 0.332598427522\n",
      "training on  chr13 testing on  chr13\n",
      "training on  chr13 testing on  chr14\n",
      "chr13 chr14 0.972263384968 0.349163499286\n",
      "training on  chr13 testing on  chr15\n",
      "chr13 chr15 0.970353254718 0.38583476617\n",
      "training on  chr13 testing on  chr16\n",
      "chr13 chr16 0.971115306065 0.378556478003\n",
      "training on  chr13 testing on  chr17\n",
      "chr13 chr17 0.967427064639 0.375418552555\n",
      "training on  chr13 testing on  chr18\n",
      "chr13 chr18 0.96547026857 0.339367690773\n",
      "training on  chr13 testing on  chr19\n",
      "chr13 chr19 0.975862978457 0.430691348814\n",
      "training on  chr13 testing on  chr2\n",
      "chr13 chr2 0.968794045171 0.389112092842\n",
      "training on  chr13 testing on  chr3\n",
      "chr13 chr3 0.970661544745 0.35168135067\n",
      "training on  chr13 testing on  chr4\n",
      "chr13 chr4 0.971134642549 0.390676901716\n",
      "training on  chr13 testing on  chr5\n",
      "chr13 chr5 0.970334582796 0.369951473999\n",
      "training on  chr13 testing on  chr6\n",
      "chr13 chr6 0.970682282184 0.384605352727\n",
      "training on  chr13 testing on  chr7\n",
      "chr13 chr7 0.971106357501 0.406460593786\n",
      "training on  chr13 testing on  chr8\n",
      "chr13 chr8 0.970941046437 0.396081040031\n",
      "training on  chr13 testing on  chr9\n",
      "chr13 chr9 0.966251892941 0.376765132081\n",
      "training on  chr13 testing on  chrX\n",
      "chr13 chrX 0.974082238203 0.304552078995\n",
      "training on  chr14 testing on  chr1\n",
      "chr14 chr1 0.968356314634 0.367614670185\n",
      "training on  chr14 testing on  chr10\n",
      "chr14 chr10 0.968257156146 0.399407812415\n",
      "training on  chr14 testing on  chr11\n",
      "chr14 chr11 0.970424256087 0.431328522902\n",
      "training on  chr14 testing on  chr12\n",
      "chr14 chr12 0.965480176554 0.337361057934\n",
      "training on  chr14 testing on  chr13\n",
      "chr14 chr13 0.957843759755 0.327660300643\n",
      "training on  chr14 testing on  chr14\n",
      "training on  chr14 testing on  chr15\n",
      "chr14 chr15 0.968257475017 0.389899581578\n",
      "training on  chr14 testing on  chr16\n",
      "chr14 chr16 0.970220482595 0.378760115597\n",
      "training on  chr14 testing on  chr17\n",
      "chr14 chr17 0.966424406774 0.372809408683\n",
      "training on  chr14 testing on  chr18\n",
      "chr14 chr18 0.964173815649 0.344921237845\n",
      "training on  chr14 testing on  chr19\n",
      "chr14 chr19 0.975295497213 0.434142742148\n",
      "training on  chr14 testing on  chr2\n",
      "chr14 chr2 0.967575520677 0.383676260481\n",
      "training on  chr14 testing on  chr3\n",
      "chr14 chr3 0.968760193756 0.352469270242\n",
      "training on  chr14 testing on  chr4\n",
      "chr14 chr4 0.96965394768 0.394169942118\n",
      "training on  chr14 testing on  chr5\n",
      "chr14 chr5 0.969687533566 0.373509809076\n",
      "training on  chr14 testing on  chr6\n",
      "chr14 chr6 0.96900571876 0.385665486837\n",
      "training on  chr14 testing on  chr7\n",
      "chr14 chr7 0.969901655393 0.405657297572\n",
      "training on  chr14 testing on  chr8\n",
      "chr14 chr8 0.969424718643 0.397961485317\n",
      "training on  chr14 testing on  chr9\n",
      "chr14 chr9 0.964860814046 0.374841555729\n",
      "training on  chr14 testing on  chrX\n",
      "chr14 chrX 0.972426951294 0.304138730636\n",
      "training on  chr15 testing on  chr1\n",
      "chr15 chr1 0.969729853631 0.372273987666\n",
      "training on  chr15 testing on  chr10\n",
      "chr15 chr10 0.969224251076 0.405269757196\n",
      "training on  chr15 testing on  chr11\n",
      "chr15 chr11 0.971411959764 0.440949936903\n",
      "training on  chr15 testing on  chr12\n",
      "chr15 chr12 0.965830711615 0.33779502343\n",
      "training on  chr15 testing on  chr13\n",
      "chr15 chr13 0.959561445979 0.330876507137\n",
      "training on  chr15 testing on  chr14\n",
      "chr15 chr14 0.970089993278 0.349720711514\n",
      "training on  chr15 testing on  chr15\n",
      "training on  chr15 testing on  chr16\n",
      "chr15 chr16 0.96934852629 0.387419414406\n",
      "training on  chr15 testing on  chr17\n",
      "chr15 chr17 0.966403628419 0.384858578275\n",
      "training on  chr15 testing on  chr18\n",
      "chr15 chr18 0.964423438228 0.349037315121\n",
      "training on  chr15 testing on  chr19\n",
      "chr15 chr19 0.975526358227 0.438187558531\n",
      "training on  chr15 testing on  chr2\n",
      "chr15 chr2 0.967787371725 0.392153308275\n",
      "training on  chr15 testing on  chr3\n",
      "chr15 chr3 0.96871919067 0.354934977129\n",
      "training on  chr15 testing on  chr4\n",
      "chr15 chr4 0.96997965557 0.396100793254\n",
      "training on  chr15 testing on  chr5\n",
      "chr15 chr5 0.968850360042 0.376978430507\n",
      "training on  chr15 testing on  chr6\n",
      "chr15 chr6 0.969365730818 0.386566518494\n",
      "training on  chr15 testing on  chr7\n",
      "chr15 chr7 0.969477485077 0.408903394853\n",
      "training on  chr15 testing on  chr8\n",
      "chr15 chr8 0.970161972196 0.406452183735\n",
      "training on  chr15 testing on  chr9\n",
      "chr15 chr9 0.964730802532 0.384110945676\n",
      "training on  chr15 testing on  chrX\n",
      "chr15 chrX 0.972872504821 0.312495980686\n",
      "training on  chr16 testing on  chr1\n",
      "chr16 chr1 0.968687046321 0.36816515837\n",
      "training on  chr16 testing on  chr10\n",
      "chr16 chr10 0.96910536429 0.393271794528\n",
      "training on  chr16 testing on  chr11\n",
      "chr16 chr11 0.97025178186 0.433447887601\n",
      "training on  chr16 testing on  chr12\n",
      "chr16 chr12 0.965836367772 0.331735851397\n",
      "training on  chr16 testing on  chr13\n",
      "chr16 chr13 0.958627701738 0.323850286155\n",
      "training on  chr16 testing on  chr14\n",
      "chr16 chr14 0.970164588396 0.35259697118\n",
      "training on  chr16 testing on  chr15\n",
      "chr16 chr15 0.969627180414 0.385521337514\n",
      "training on  chr16 testing on  chr16\n",
      "training on  chr16 testing on  chr17\n",
      "chr16 chr17 0.965700732546 0.372377100109\n",
      "training on  chr16 testing on  chr18\n",
      "chr16 chr18 0.964670824897 0.338803317902\n",
      "training on  chr16 testing on  chr19\n",
      "chr16 chr19 0.975297032766 0.42771611676\n",
      "training on  chr16 testing on  chr2\n",
      "chr16 chr2 0.967344491044 0.384062244366\n",
      "training on  chr16 testing on  chr3\n",
      "chr16 chr3 0.967335843939 0.346059265002\n",
      "training on  chr16 testing on  chr4\n",
      "chr16 chr4 0.970047362476 0.394105881381\n",
      "training on  chr16 testing on  chr5\n",
      "chr16 chr5 0.969110544013 0.375263884502\n",
      "training on  chr16 testing on  chr6\n",
      "chr16 chr6 0.969212515721 0.386489332106\n",
      "training on  chr16 testing on  chr7\n",
      "chr16 chr7 0.970135805125 0.409474269125\n",
      "training on  chr16 testing on  chr8\n",
      "chr16 chr8 0.968769983944 0.397544425419\n",
      "training on  chr16 testing on  chr9\n",
      "chr16 chr9 0.966340771255 0.380536416131\n",
      "training on  chr16 testing on  chrX\n",
      "chr16 chrX 0.973682423804 0.304793402763\n",
      "training on  chr17 testing on  chr1\n",
      "chr17 chr1 0.967574528551 0.3632836435\n",
      "training on  chr17 testing on  chr10\n",
      "chr17 chr10 0.967768611787 0.396854292951\n",
      "training on  chr17 testing on  chr11\n",
      "chr17 chr11 0.971279816288 0.452254933123\n",
      "training on  chr17 testing on  chr12\n",
      "chr17 chr12 0.965040179574 0.333028576602\n",
      "training on  chr17 testing on  chr13\n",
      "chr17 chr13 0.958753936601 0.328020215219\n",
      "training on  chr17 testing on  chr14\n",
      "chr17 chr14 0.969137360646 0.348254825183\n",
      "training on  chr17 testing on  chr15\n",
      "chr17 chr15 0.968437022857 0.394169807772\n",
      "training on  chr17 testing on  chr16\n",
      "chr17 chr16 0.968082844615 0.38069752139\n",
      "training on  chr17 testing on  chr17\n",
      "training on  chr17 testing on  chr18\n",
      "chr17 chr18 0.962694370099 0.343709811406\n",
      "training on  chr17 testing on  chr19\n",
      "chr17 chr19 0.973995679158 0.43703847388\n",
      "training on  chr17 testing on  chr2\n",
      "chr17 chr2 0.966826309065 0.390567440208\n",
      "training on  chr17 testing on  chr3\n",
      "chr17 chr3 0.966381898311 0.350604290831\n",
      "training on  chr17 testing on  chr4\n",
      "chr17 chr4 0.96929390301 0.401808132028\n",
      "training on  chr17 testing on  chr5\n",
      "chr17 chr5 0.968394854642 0.379849754152\n",
      "training on  chr17 testing on  chr6\n",
      "chr17 chr6 0.967813173806 0.385520025746\n",
      "training on  chr17 testing on  chr7\n",
      "chr17 chr7 0.969340480718 0.416344876669\n",
      "training on  chr17 testing on  chr8\n",
      "chr17 chr8 0.969258963676 0.406324263406\n",
      "training on  chr17 testing on  chr9\n",
      "chr17 chr9 0.964854511085 0.385889227987\n",
      "training on  chr17 testing on  chrX\n",
      "chr17 chrX 0.971247801374 0.316140274213\n",
      "training on  chr18 testing on  chr1\n",
      "chr18 chr1 0.971348973011 0.360847906632\n",
      "training on  chr18 testing on  chr10\n",
      "chr18 chr10 0.969997439411 0.396166616098\n",
      "training on  chr18 testing on  chr11\n",
      "chr18 chr11 0.97074382235 0.420520912282\n",
      "training on  chr18 testing on  chr12\n",
      "chr18 chr12 0.966789319637 0.322298668998\n",
      "training on  chr18 testing on  chr13\n",
      "chr18 chr13 0.959585585239 0.322380146591\n",
      "training on  chr18 testing on  chr14\n",
      "chr18 chr14 0.971882436933 0.340756373311\n",
      "training on  chr18 testing on  chr15\n",
      "chr18 chr15 0.968941389269 0.383123879544\n",
      "training on  chr18 testing on  chr16\n",
      "chr18 chr16 0.970593640647 0.373284033165\n",
      "training on  chr18 testing on  chr17\n",
      "chr18 chr17 0.966614059561 0.369615062604\n",
      "training on  chr18 testing on  chr18\n",
      "training on  chr18 testing on  chr19\n",
      "chr18 chr19 0.974904602807 0.42380168875\n",
      "training on  chr18 testing on  chr2\n",
      "chr18 chr2 0.967870376979 0.378073959618\n",
      "training on  chr18 testing on  chr3\n",
      "chr18 chr3 0.969178979188 0.346165151778\n",
      "training on  chr18 testing on  chr4\n",
      "chr18 chr4 0.969931779865 0.384134304675\n",
      "training on  chr18 testing on  chr5\n",
      "chr18 chr5 0.970030785897 0.364113258417\n",
      "training on  chr18 testing on  chr6\n",
      "chr18 chr6 0.969809137345 0.377060985914\n",
      "training on  chr18 testing on  chr7\n",
      "chr18 chr7 0.970526538219 0.394771590287\n",
      "training on  chr18 testing on  chr8\n",
      "chr18 chr8 0.970161746366 0.390735236987\n",
      "training on  chr18 testing on  chr9\n",
      "chr18 chr9 0.965211308252 0.370902652185\n",
      "training on  chr18 testing on  chrX\n",
      "chr18 chrX 0.973405818711 0.292704984748\n",
      "training on  chr19 testing on  chr1\n",
      "chr19 chr1 0.967655178573 0.363936602301\n",
      "training on  chr19 testing on  chr10\n",
      "chr19 chr10 0.967633959872 0.390761534903\n",
      "training on  chr19 testing on  chr11\n",
      "chr19 chr11 0.970046820902 0.442727927779\n",
      "training on  chr19 testing on  chr12\n",
      "chr19 chr12 0.964037709271 0.33151993147\n",
      "training on  chr19 testing on  chr13\n",
      "chr19 chr13 0.957889432549 0.325790525079\n",
      "training on  chr19 testing on  chr14\n",
      "chr19 chr14 0.970085465418 0.348954710236\n",
      "training on  chr19 testing on  chr15\n",
      "chr19 chr15 0.967947109236 0.388003664083\n",
      "training on  chr19 testing on  chr16\n",
      "chr19 chr16 0.969393650117 0.377071912146\n",
      "training on  chr19 testing on  chr17\n",
      "chr19 chr17 0.965648547934 0.379634221467\n",
      "training on  chr19 testing on  chr18\n",
      "chr19 chr18 0.96335337542 0.337267200748\n",
      "training on  chr19 testing on  chr19\n",
      "training on  chr19 testing on  chr2\n",
      "chr19 chr2 0.966448759528 0.385141622939\n",
      "training on  chr19 testing on  chr3\n",
      "chr19 chr3 0.967121210006 0.341358663064\n",
      "training on  chr19 testing on  chr4\n",
      "chr19 chr4 0.969717415978 0.393918245804\n",
      "training on  chr19 testing on  chr5\n",
      "chr19 chr5 0.967118080084 0.381732708388\n",
      "training on  chr19 testing on  chr6\n",
      "chr19 chr6 0.968223231326 0.380211291428\n",
      "training on  chr19 testing on  chr7\n",
      "chr19 chr7 0.967994626309 0.405378951162\n",
      "training on  chr19 testing on  chr8\n",
      "chr19 chr8 0.968130554081 0.400282479742\n",
      "training on  chr19 testing on  chr9\n",
      "chr19 chr9 0.964020735427 0.380057128271\n",
      "training on  chr19 testing on  chrX\n",
      "chr19 chrX 0.971900780125 0.310402844089\n",
      "training on  chr2 testing on  chr1\n",
      "chr2 chr1 0.970832552193 0.372647380072\n",
      "training on  chr2 testing on  chr10\n",
      "chr2 chr10 0.969645046396 0.405949286908\n",
      "training on  chr2 testing on  chr11\n",
      "chr2 chr11 0.972045917825 0.45149723124\n",
      "training on  chr2 testing on  chr12\n",
      "chr2 chr12 0.966805133945 0.340894377956\n",
      "training on  chr2 testing on  chr13\n",
      "chr2 chr13 0.960301373061 0.333335136701\n",
      "training on  chr2 testing on  chr14\n",
      "chr2 chr14 0.971060669881 0.353787286428\n",
      "training on  chr2 testing on  chr15\n",
      "chr2 chr15 0.970706080132 0.39715499611\n",
      "training on  chr2 testing on  chr16\n",
      "chr2 chr16 0.970101433337 0.387706022974\n",
      "training on  chr2 testing on  chr17\n",
      "chr2 chr17 0.967617641308 0.386502336926\n",
      "training on  chr2 testing on  chr18\n",
      "chr2 chr18 0.964325486074 0.345429514244\n",
      "training on  chr2 testing on  chr19\n",
      "chr2 chr19 0.975410069247 0.442988889794\n",
      "training on  chr2 testing on  chr2\n",
      "training on  chr2 testing on  chr3\n",
      "chr2 chr3 0.968613660536 0.359241836497\n",
      "training on  chr2 testing on  chr4\n",
      "chr2 chr4 0.970735294521 0.404079570333\n",
      "training on  chr2 testing on  chr5\n",
      "chr2 chr5 0.970508678109 0.381231995386\n",
      "training on  chr2 testing on  chr6\n",
      "chr2 chr6 0.969578158715 0.392243163282\n",
      "training on  chr2 testing on  chr7\n",
      "chr2 chr7 0.971702903565 0.41953454071\n",
      "training on  chr2 testing on  chr8\n",
      "chr2 chr8 0.970666701322 0.407834295282\n",
      "training on  chr2 testing on  chr9\n",
      "chr2 chr9 0.967108647326 0.389621870454\n",
      "training on  chr2 testing on  chrX\n",
      "chr2 chrX 0.974492415304 0.318999718696\n",
      "training on  chr3 testing on  chr1\n",
      "chr3 chr1 0.97168107254 0.364444635883\n",
      "training on  chr3 testing on  chr10\n",
      "chr3 chr10 0.971318832365 0.397235314299\n",
      "training on  chr3 testing on  chr11\n",
      "chr3 chr11 0.971930918987 0.425087573904\n",
      "training on  chr3 testing on  chr12\n",
      "chr3 chr12 0.967422876734 0.325529290058\n",
      "training on  chr3 testing on  chr13\n",
      "chr3 chr13 0.961298156795 0.326317318671\n",
      "training on  chr3 testing on  chr14\n",
      "chr3 chr14 0.972155827617 0.346540487035\n",
      "training on  chr3 testing on  chr15\n",
      "chr3 chr15 0.970862775365 0.38304326245\n",
      "training on  chr3 testing on  chr16\n",
      "chr3 chr16 0.971734255829 0.374104671315\n",
      "training on  chr3 testing on  chr17\n",
      "chr3 chr17 0.967679232861 0.372515847402\n",
      "training on  chr3 testing on  chr18\n",
      "chr3 chr18 0.966582555958 0.341104195862\n",
      "training on  chr3 testing on  chr19\n",
      "chr3 chr19 0.975532671058 0.42506776537\n",
      "training on  chr3 testing on  chr2\n",
      "chr3 chr2 0.969065956055 0.382030381614\n",
      "training on  chr3 testing on  chr3\n",
      "training on  chr3 testing on  chr4\n",
      "chr3 chr4 0.9715027522 0.389769905248\n",
      "training on  chr3 testing on  chr5\n",
      "chr3 chr5 0.970764227846 0.362653337324\n",
      "training on  chr3 testing on  chr6\n",
      "chr3 chr6 0.971783469021 0.381052645615\n",
      "training on  chr3 testing on  chr7\n",
      "chr3 chr7 0.971553981552 0.40083080951\n",
      "training on  chr3 testing on  chr8\n",
      "chr3 chr8 0.971741541861 0.393500490627\n",
      "training on  chr3 testing on  chr9\n",
      "chr3 chr9 0.967260325482 0.37240465599\n",
      "training on  chr3 testing on  chrX\n",
      "chr3 chrX 0.973276985838 0.30503485169\n",
      "training on  chr4 testing on  chr1\n",
      "chr4 chr1 0.969544660743 0.366897685068\n",
      "training on  chr4 testing on  chr10\n",
      "chr4 chr10 0.968069918799 0.398867044854\n",
      "training on  chr4 testing on  chr11\n",
      "chr4 chr11 0.971594805286 0.449102101371\n",
      "training on  chr4 testing on  chr12\n",
      "chr4 chr12 0.964976136605 0.332837590844\n",
      "training on  chr4 testing on  chr13\n",
      "chr4 chr13 0.95851550662 0.326441833206\n",
      "training on  chr4 testing on  chr14\n",
      "chr4 chr14 0.970454069721 0.349258923671\n",
      "training on  chr4 testing on  chr15\n",
      "chr4 chr15 0.969362965967 0.392324782583\n",
      "training on  chr4 testing on  chr16\n",
      "chr4 chr16 0.969589794414 0.380392404801\n",
      "training on  chr4 testing on  chr17\n",
      "chr4 chr17 0.967290461583 0.382262907871\n",
      "training on  chr4 testing on  chr18\n",
      "chr4 chr18 0.963159951134 0.340742737438\n",
      "training on  chr4 testing on  chr19\n",
      "chr4 chr19 0.975750373008 0.435138287961\n",
      "training on  chr4 testing on  chr2\n",
      "chr4 chr2 0.968158746289 0.391025997164\n",
      "training on  chr4 testing on  chr3\n",
      "chr4 chr3 0.96845689395 0.353383730281\n",
      "training on  chr4 testing on  chr4\n",
      "training on  chr4 testing on  chr5\n",
      "chr4 chr5 0.969889931316 0.379115295981\n",
      "training on  chr4 testing on  chr6\n",
      "chr4 chr6 0.968910549762 0.386278020563\n",
      "training on  chr4 testing on  chr7\n",
      "chr4 chr7 0.970428843932 0.413621415717\n",
      "training on  chr4 testing on  chr8\n",
      "chr4 chr8 0.970333912159 0.408628303982\n",
      "training on  chr4 testing on  chr9\n",
      "chr4 chr9 0.966712461871 0.384851993261\n",
      "training on  chr4 testing on  chrX\n",
      "chr4 chrX 0.974224482554 0.312696336886\n",
      "training on  chr5 testing on  chr1\n",
      "chr5 chr1 0.968554050366 0.370893209225\n",
      "training on  chr5 testing on  chr10\n",
      "chr5 chr10 0.967710251971 0.402521077541\n",
      "training on  chr5 testing on  chr11\n",
      "chr5 chr11 0.971132927918 0.450528987376\n",
      "training on  chr5 testing on  chr12\n",
      "chr5 chr12 0.964004639553 0.341190351716\n",
      "training on  chr5 testing on  chr13\n",
      "chr5 chr13 0.958249876582 0.331127252274\n",
      "training on  chr5 testing on  chr14\n",
      "chr5 chr14 0.970368156227 0.354696034081\n",
      "training on  chr5 testing on  chr15\n",
      "chr5 chr15 0.968826921223 0.394829800231\n",
      "training on  chr5 testing on  chr16\n",
      "chr5 chr16 0.969573131816 0.38658558534\n",
      "training on  chr5 testing on  chr17\n",
      "chr5 chr17 0.966708423606 0.385781867458\n",
      "training on  chr5 testing on  chr18\n",
      "chr5 chr18 0.963238162306 0.34947832669\n",
      "training on  chr5 testing on  chr19\n",
      "chr5 chr19 0.974992213757 0.438738141824\n",
      "training on  chr5 testing on  chr2\n",
      "chr5 chr2 0.967323876825 0.392182561185\n",
      "training on  chr5 testing on  chr3\n",
      "chr5 chr3 0.966006897418 0.35407556579\n",
      "training on  chr5 testing on  chr4\n",
      "chr5 chr4 0.969837696825 0.4005204685\n",
      "training on  chr5 testing on  chr5\n",
      "training on  chr5 testing on  chr6\n",
      "chr5 chr6 0.967906964885 0.393570368341\n",
      "training on  chr5 testing on  chr7\n",
      "chr5 chr7 0.968276971415 0.416233318426\n",
      "training on  chr5 testing on  chr8\n",
      "chr5 chr8 0.968730455644 0.413217130793\n",
      "training on  chr5 testing on  chr9\n",
      "chr5 chr9 0.964466810704 0.391039748777\n",
      "training on  chr5 testing on  chrX\n",
      "chr5 chrX 0.971709927299 0.31816905664\n",
      "training on  chr6 testing on  chr1\n",
      "chr6 chr1 0.970786806856 0.372975810627\n",
      "training on  chr6 testing on  chr10\n",
      "chr6 chr10 0.970496972487 0.40499351533\n",
      "training on  chr6 testing on  chr11\n",
      "chr6 chr11 0.971422262535 0.436687725779\n",
      "training on  chr6 testing on  chr12\n",
      "chr6 chr12 0.966448267573 0.337716437158\n",
      "training on  chr6 testing on  chr13\n",
      "chr6 chr13 0.96114485728 0.332039676563\n",
      "training on  chr6 testing on  chr14\n",
      "chr6 chr14 0.971598089388 0.352930433277\n",
      "training on  chr6 testing on  chr15\n",
      "chr6 chr15 0.970721578931 0.391142237548\n",
      "training on  chr6 testing on  chr16\n",
      "chr6 chr16 0.971332885768 0.386561713374\n",
      "training on  chr6 testing on  chr17\n",
      "chr6 chr17 0.967436767027 0.379726328227\n",
      "training on  chr6 testing on  chr18\n",
      "chr6 chr18 0.965759429524 0.346278866044\n",
      "training on  chr6 testing on  chr19\n",
      "chr6 chr19 0.975621061458 0.433789869468\n",
      "training on  chr6 testing on  chr2\n",
      "chr6 chr2 0.968750404331 0.389681034683\n",
      "training on  chr6 testing on  chr3\n",
      "chr6 chr3 0.970136034257 0.358208812592\n",
      "training on  chr6 testing on  chr4\n",
      "chr6 chr4 0.970429966112 0.398638616167\n",
      "training on  chr6 testing on  chr5\n",
      "chr6 chr5 0.970225141742 0.378183118246\n",
      "training on  chr6 testing on  chr6\n",
      "training on  chr6 testing on  chr7\n",
      "chr6 chr7 0.971694173876 0.413412712188\n",
      "training on  chr6 testing on  chr8\n",
      "chr6 chr8 0.970910494998 0.407588253313\n",
      "training on  chr6 testing on  chr9\n",
      "chr6 chr9 0.96713842305 0.384728284166\n",
      "training on  chr6 testing on  chrX\n",
      "chr6 chrX 0.974485847501 0.309221253737\n",
      "training on  chr7 testing on  chr1\n",
      "chr7 chr1 0.968524527013 0.364240683884\n",
      "training on  chr7 testing on  chr10\n",
      "chr7 chr10 0.967937212223 0.392135691763\n",
      "training on  chr7 testing on  chr11\n",
      "chr7 chr11 0.971956945012 0.450386818731\n",
      "training on  chr7 testing on  chr12\n",
      "chr7 chr12 0.965674072502 0.333037195769\n",
      "training on  chr7 testing on  chr13\n",
      "chr7 chr13 0.958572098336 0.327396648525\n",
      "training on  chr7 testing on  chr14\n",
      "chr7 chr14 0.969449171944 0.350406006051\n",
      "training on  chr7 testing on  chr15\n",
      "chr7 chr15 0.968601920108 0.383801032052\n",
      "training on  chr7 testing on  chr16\n",
      "chr7 chr16 0.969517268125 0.383467668591\n",
      "training on  chr7 testing on  chr17\n",
      "chr7 chr17 0.966292793657 0.382646498698\n",
      "training on  chr7 testing on  chr18\n",
      "chr7 chr18 0.963459852046 0.336757936399\n",
      "training on  chr7 testing on  chr19\n",
      "chr7 chr19 0.974142264336 0.433282985997\n",
      "training on  chr7 testing on  chr2\n",
      "chr7 chr2 0.966569803112 0.391925584906\n",
      "training on  chr7 testing on  chr3\n",
      "chr7 chr3 0.968302509707 0.35074326326\n",
      "training on  chr7 testing on  chr4\n",
      "chr7 chr4 0.968720294712 0.399117637125\n",
      "training on  chr7 testing on  chr5\n",
      "chr7 chr5 0.969503280394 0.374381600573\n",
      "training on  chr7 testing on  chr6\n",
      "chr7 chr6 0.968014674805 0.384740615265\n",
      "training on  chr7 testing on  chr7\n",
      "training on  chr7 testing on  chr8\n",
      "chr7 chr8 0.970668326636 0.408736849233\n",
      "training on  chr7 testing on  chr9\n",
      "chr7 chr9 0.965126535728 0.386153199198\n",
      "training on  chr7 testing on  chrX\n",
      "chr7 chrX 0.973141043141 0.316713029806\n",
      "training on  chr8 testing on  chr1\n",
      "chr8 chr1 0.969797413419 0.364784738099\n",
      "training on  chr8 testing on  chr10\n",
      "chr8 chr10 0.967995454855 0.396482221124\n",
      "training on  chr8 testing on  chr11\n",
      "chr8 chr11 0.971332064867 0.445985456711\n",
      "training on  chr8 testing on  chr12\n",
      "chr8 chr12 0.965710387614 0.330876383259\n",
      "training on  chr8 testing on  chr13\n",
      "chr8 chr13 0.958690709124 0.32832520428\n",
      "training on  chr8 testing on  chr14\n",
      "chr8 chr14 0.970557151402 0.348020654937\n",
      "training on  chr8 testing on  chr15\n",
      "chr8 chr15 0.970013988405 0.391345184372\n",
      "training on  chr8 testing on  chr16\n",
      "chr8 chr16 0.968779378658 0.380651960408\n",
      "training on  chr8 testing on  chr17\n",
      "chr8 chr17 0.967098956136 0.382279937136\n",
      "training on  chr8 testing on  chr18\n",
      "chr8 chr18 0.963926482471 0.343564005033\n",
      "training on  chr8 testing on  chr19\n",
      "chr8 chr19 0.974286701535 0.436173570765\n",
      "training on  chr8 testing on  chr2\n",
      "chr8 chr2 0.967315596355 0.389752177633\n",
      "training on  chr8 testing on  chr3\n",
      "chr8 chr3 0.969022090468 0.353062477524\n",
      "training on  chr8 testing on  chr4\n",
      "chr8 chr4 0.969817321435 0.400326750221\n",
      "training on  chr8 testing on  chr5\n",
      "chr8 chr5 0.968513835467 0.381847456559\n",
      "training on  chr8 testing on  chr6\n",
      "chr8 chr6 0.968505839713 0.387379131463\n",
      "training on  chr8 testing on  chr7\n",
      "chr8 chr7 0.971049072889 0.41620567595\n",
      "training on  chr8 testing on  chr8\n",
      "training on  chr8 testing on  chr9\n",
      "chr8 chr9 0.965374515849 0.389829551172\n",
      "training on  chr8 testing on  chrX\n",
      "chr8 chrX 0.973954216494 0.313695129254\n",
      "training on  chr9 testing on  chr1\n",
      "chr9 chr1 0.969346683652 0.366146308348\n",
      "training on  chr9 testing on  chr10\n",
      "chr9 chr10 0.968002049911 0.394669541099\n",
      "training on  chr9 testing on  chr11\n",
      "chr9 chr11 0.971191560114 0.450854169663\n",
      "training on  chr9 testing on  chr12\n",
      "chr9 chr12 0.965667440864 0.332005440987\n",
      "training on  chr9 testing on  chr13\n",
      "chr9 chr13 0.958161603425 0.33134150716\n",
      "training on  chr9 testing on  chr14\n",
      "chr9 chr14 0.969819653838 0.350546412423\n",
      "training on  chr9 testing on  chr15\n",
      "chr9 chr15 0.968892496949 0.389276209355\n",
      "training on  chr9 testing on  chr16\n",
      "chr9 chr16 0.969517602648 0.388893251127\n",
      "training on  chr9 testing on  chr17\n",
      "chr9 chr17 0.966821527264 0.384738845003\n",
      "training on  chr9 testing on  chr18\n",
      "chr9 chr18 0.96401566896 0.342551762489\n",
      "training on  chr9 testing on  chr19\n",
      "chr9 chr19 0.974550523965 0.433746408029\n",
      "training on  chr9 testing on  chr2\n",
      "chr9 chr2 0.967349671772 0.393807124494\n",
      "training on  chr9 testing on  chr3\n",
      "chr9 chr3 0.968017017316 0.352572701011\n",
      "training on  chr9 testing on  chr4\n",
      "chr9 chr4 0.96868110967 0.400987177695\n",
      "training on  chr9 testing on  chr5\n",
      "chr9 chr5 0.968663207981 0.380685406517\n",
      "training on  chr9 testing on  chr6\n",
      "chr9 chr6 0.968083441079 0.389753261905\n",
      "training on  chr9 testing on  chr7\n",
      "chr9 chr7 0.970868849978 0.417681843386\n",
      "training on  chr9 testing on  chr8\n",
      "chr9 chr8 0.969955188603 0.412692691094\n",
      "training on  chr9 testing on  chr9\n",
      "training on  chr9 testing on  chrX\n",
      "chr9 chrX 0.971781515986 0.311491385979\n",
      "training on  chrX testing on  chr1\n",
      "chrX chr1 0.963422976829 0.352042464698\n",
      "training on  chrX testing on  chr10\n",
      "chrX chr10 0.962664230517 0.37883812101\n",
      "training on  chrX testing on  chr11\n",
      "chrX chr11 0.966710593085 0.439264892321\n",
      "training on  chrX testing on  chr12\n",
      "chrX chr12 0.958914034832 0.320378379622\n",
      "training on  chrX testing on  chr13\n",
      "chrX chr13 0.951592325675 0.314792048184\n",
      "training on  chrX testing on  chr14\n",
      "chrX chr14 0.963264425726 0.336115763388\n",
      "training on  chrX testing on  chr15\n",
      "chrX chr15 0.965052517718 0.370547017716\n",
      "training on  chrX testing on  chr16\n",
      "chrX chr16 0.964824199527 0.366424182534\n",
      "training on  chrX testing on  chr17\n",
      "chrX chr17 0.963645988341 0.369227776226\n",
      "training on  chrX testing on  chr18\n",
      "chrX chr18 0.957551770139 0.322074327762\n",
      "training on  chrX testing on  chr19\n",
      "chrX chr19 0.971541367455 0.423761742924\n",
      "training on  chrX testing on  chr2\n",
      "chrX chr2 0.961308083784 0.375677884484\n",
      "training on  chrX testing on  chr3\n",
      "chrX chr3 0.962063286041 0.336783167908\n",
      "training on  chrX testing on  chr4\n",
      "chrX chr4 0.966198119644 0.382369846051\n",
      "training on  chrX testing on  chr5\n",
      "chrX chr5 0.964194649593 0.362407100444\n",
      "training on  chrX testing on  chr6\n",
      "chrX chr6 0.962571076724 0.370370478658\n",
      "training on  chrX testing on  chr7\n",
      "chrX chr7 0.965873718236 0.406779350625\n",
      "training on  chrX testing on  chr8\n",
      "chrX chr8 0.966256607185 0.395319969175\n",
      "training on  chrX testing on  chr9\n",
      "chrX chr9 0.961113270877 0.369724494588\n",
      "training on  chrX testing on  chrX\n"
     ]
    }
   ],
   "source": [
    "chromosome_coeff_dict = {} # store coefficients for each chromosome {trainingChromosome:{testChromosome:[coefficients]}}\n",
    "chromosome_probas_dict = {} # probabilities for each locus {trainingChromosome:{testChromosome:[probas]}}\n",
    "chromosome_rocAUC_dict = {} # ROC AUC for each chromosome {trainingChromosome:{testChromosome:auc}}\n",
    "chromosome_prAUC_dict = {}\n",
    "peakDirectory = './group_by_chromosome/'\n",
    "chromosomes = sorted(os.listdir(peakDirectory))\n",
    "for training_chromosome in chromosomes:\n",
    "    training_features = pd.read_pickle('./pickles/' + training_chromosome + '_normed_motif_frame.pickle')\n",
    "    training_labels = [False if 'tile' in x else True for x in training_features.index.values]\n",
    "\n",
    "    #  train classifier\n",
    "    lr_classifier = sklearn.linear_model.LogisticRegression(penalty='l1')\n",
    "    lr_classifier.fit(training_features, training_labels)\n",
    "\n",
    "    # initialize sub dictionaries to store results\n",
    "    chromosome_probas_dict[training_chromosome] = {}\n",
    "    chromosome_rocAUC_dict[training_chromosome] = {}\n",
    "    chromosome_prAUC_dict[training_chromosome] = {}\n",
    "    \n",
    "    coefficients = lr_classifier.coef_\n",
    "\n",
    "    chromosome_coeff_dict[training_chromosome] = coefficients.flatten()\n",
    "    for chrom in chromosomes:\n",
    "        print('training on ', training_chromosome, 'testing on ', chrom)\n",
    "        if not chrom == training_chromosome:\n",
    "            test_features = pd.read_pickle('./pickles/' + chrom + '_normed_motif_frame.pickle')\n",
    "            test_labels = [False if 'tile' in x else True for x in test_features.index.values]\n",
    "\n",
    "            probas_lr = lr_classifier.predict_proba(test_features)\n",
    "\n",
    "            fpr, tpr, thresholds_roc_lr = sklearn.metrics.roc_curve(test_labels, probas_lr[:, 1])\n",
    "            \n",
    "            precision, recall, thresholds = sklearn.metrics.precision_recall_curve(test_labels, probas_lr[:, 1])\n",
    "            \n",
    "            roc_auc = auc(fpr, tpr) # calculate area under the curve\n",
    "            pr_auc = auc(recall, precision)\n",
    "            \n",
    "            \n",
    "            chromosome_probas_dict[training_chromosome][chrom] = probas_lr[:,1]\n",
    "            chromosome_rocAUC_dict[training_chromosome][chrom] = roc_auc\n",
    "            chromosome_prAUC_dict[training_chromosome][chrom] = pr_auc\n",
    "\n",
    "            print(training_chromosome, chrom, roc_auc, pr_auc)\n",
    "motifs = training_features.columns.values"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "pickle.dump(chromosome_coeff_dict, open('chromosome_coeff_dict.pickle','wb'))\n",
    "pickle.dump(chromosome_probas_dict, open('chromosome_probas_dict.pickle','wb'))\n",
    "pickle.dump(chromosome_rocAUC_dict, open('chromosome_rocAUC_dict.pickle','wb'))\n",
    "pickle.dump(chromosome_prAUC_dict, open('chromosome_prAUC_dict.pickle','wb'))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Summarize Performance"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Visualize Correlations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/bioinformatics/software/anaconda3/lib/python3.4/site-packages/seaborn-0.7.0-py3.4.egg/seaborn/matrix.py:134: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison\n",
      "  if xticklabels == []:\n",
      "/bioinformatics/software/anaconda3/lib/python3.4/site-packages/seaborn-0.7.0-py3.4.egg/seaborn/matrix.py:142: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison\n",
      "  if yticklabels == []:\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "<matplotlib.text.Text at 0x7fe615a2c860>"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAB+0AAAa0CAYAAAAyJ4sJAAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAAewgAAHsIBbtB1PgAAIABJREFUeJzs3Xe4Y2W1+PHvGURAQYqCCoKgwnLsggVEpSgWFBXbtQwq\nXHsBC3oV9We9l3vtXUSvFbGDqFesCHbBimVcjEpRKYqIAoqg5PfH2iF7QpKTc+aUOTnfz/PkSc5k\n5827s0v2ZL1rvVOdTgdJkiRJkiRJkiRJkrTwVix2ByRJkiRJkiRJkiRJWq4M2kuSJEmSJEmSJEmS\ntEgM2kuSJEmSJEmSJEmStEgM2kuSJEmSJEmSJEmStEgM2kuSJEmSJEmSJEmStEgM2kuSJEmSJEmS\nJEmStEgM2kuSJEmSJEmSJEmStEgM2kuSJEmSJEmSJEmStEgM2kuSJEmSJEmSJEmStEgM2kuSJEmS\nJEmSJEmStEgM2kuSJEmSJEmSJEmStEgM2kuSJEmSJEmSJEmStEgM2kuSJEmSJEmSJEmStEgM2kuS\nJEmSJEmSJEmStEgM2kuSJEmSJEmSJEmStEgM2kuSJEmSJEmSJEmStEgM2kuSJEmSJEmSJEmStEgM\n2kuSJEmSJEmSJEmStEgM2kuSJEmSJEmSJEmStEiutdgdkDQ7EXFV8/CszLzZIvflLGAHoJOZGyxm\nXyRpfRMRAaxu/vxCZu7f9/x3gbs0f94oM/8wy/e5L3DisPdZSBGxEfD35s9fZuatFqsvkiRJkiRJ\nkrS+m7egfUTsBXxtjEWvAi4Gfg98F/hYZp40X/3S+iciVgIPBO4F3AzYGrgucBlwHvAL4KvAp2Yb\nyJhgncXuQGN96cecioiTgXuOWKRDBaX+DPwK+DZ1Djt9/nun9Z3ntvkXEd8B7tr8+f8y89WzbOcI\noPva72TmnnPRvwGGnSs70zw/V++zGNanvsy5iLgp8HBgbyCAbajj/ErgUuAc4GfUYIrjMvPKxemp\nJEmSJEmSpPXZQmTaT/dj7RSwZXO7LfCk5kf4VZl55nx3TounyTw8EngwtR/07yvXa263BB4KvCEi\n3g0ckZmXLmRfl7uIOBe4EbBjZp4zYJFfUcHrSQvOdJh+nTZpbtsCewEviogPAU/NzL+PfOUciIj7\nAF8ATs7Mfef7/TQ9z20L6igqaN8BDqEXeJ+pQ+htp3fOQb9m6kxg86YP/1yE95+xiNicGrB0eWZe\nZ8AiVwG/bB7/esE6tkAi4trA64GnUNfT/cf5Rs3t+sCuwOOA30bEwQsxODUiDgPeCByVmU+f7/eT\nJEmSJEmStG4Wqjz+r4CHUMGLfhsBNwbuTv1ovjWwB3BKROyRmb9foD5qAUXEQ4APUdloHarawvHA\nl4HfAZdQAzluDzwM2BO4NvBM4L4Rsd+Q4LHmWETchArYDw1eZ+Z+C9ejBdcNuj6X2j/7n7sucHPg\nAcAjgQ2Ag6gA3EMWoH93mX4RLRTPbQvuY8AbqM/0ps3n13+cjhQR+1KVEAAuAj4+t12cXmY+ZqHf\ncw7cedSTTUb5RJbEj4gNqMFSe9OruPIR4GRqgMKl1HG9A3VN+zjq+nZ74MSIuF9mjlONal3cmckb\nSCdJkiRJkiRNrIUK2v8jM1ePeP5HwOcj4rXA/wG7A9sBbwIesQD90wKKiAcCn2r+7ADvA16QmRcN\nWPxk4M0RcS/gg1Tw+BbAFyPiTpl52QJ0ebkzKFx+m5m/GPLcqcBHIuI9wJeowP0BEXH3zPzmPPfL\n7bOe8Ny28DLz8oj4APDs5p+eyDUH10znSc19B3h/Zl4xV/2bcMv53HMYvYD9r4H9MvPsAcv9ADg+\nIl4JfAK4L7Ah8J6IuOU8l8pfzttHkiRJkiRJWnJWLHYH2jLzz8CjqXlAp4CHRsSNF7dXmktN1vYH\nWv90ZGY+cUhQ62qZ+VVqbvE/U/vGLsy+DLJmxh/+x5SZJwPH0asqshAVCMymXA94bltURzX3U8CD\nI+L6474wIrZk7YoY75rLjk245fzd8PjW40OHBOyv1kx78WgqA78D7AgcOF+da/brW8xX+5IkSZIk\nSZLm3kJl2o8tM8+OiG9Tc0NDZTJ9ZNCyEbEd8FTgXsDO1BzBfwF+D3yFypj7+XTvGRE7AgcD+wBB\nlbW+CrgQ+DFVKvcjmfmvIa9/GfCy5s8HUpm2L6Ky924M/DQzd+17ze2b97w7VZZ3U2qwwgXA6cCn\ngQ9Pl4UVEXeifjzei6pOsClVjvkcKpPzPZmZI9b7N82fH8vMR0fEhsBjqakKbkuVeP5L06cPZOYH\nR/VnDC+myhh3gG9k5kvGfWFm/joiXgK8HTiD2j4DNT9YPxm4DzVv9PWBy4E/At8DjsvM40a8/mQq\nkNYBNqM+1/8BDmj6/+bMfG6z7Pvo/YB/G+qzfxX1A/3WwAmZ+dAB7zEn++8ozZy7j6bm1r5D05+N\ngL9Sn+FXqPluzx3w2rOo0r5dU8BZNV03AE/o7g+tZTuZucGI/uxCHRf7NMtvTpULPxf4OrWPnTbi\n9Vc1D7+XmXs0//awps07tdr7BVU2+6h5zmQc5HSqTD7ANqMWjIgpKmD4SCoAdsPmqT827ZwAfKh/\nHSJiL6C/tPLerc8H4OnAO5rHL87MI4f0YUd65wGAew0r29yUfu8eN6/LzBf0Pb8h8JhmnXal9rd/\nAn8Avg98KjM/MajtAe8V1FzR9wR2oo7Bi6jj6wvAe4cFyibt3Na3vV+XmS+IiEcCL6WC/BsCW2Tm\nX/tetwnw78D9gdtR58F/Ne3/EPgscExmjpxDPSIeADyKOsa2Ba4D/I0q938qcOyokvQRsRF1jjyA\n+uy3bvp8CbWdvk59V42qyDNUZp4REV+jzisbNu/1hjFf/jjqnNgBvpaZa0asw2OBB1FTG9yA3rn0\nl1R2/7sy8/zZrENEfJdeEPxGmfmHIX14FvBw6jtjE+A84LvUue4bM3i/zanP6f7U99ZW1DXhxcDP\nqapH78nMvwzow9/7mtu479yze2ae2rfsLzNzaKn8iNid2hb3pK6butcyZ1P7/rsz81dDXntL6pwP\ndSwf3Hz3PQ54QrN+12na+zHwvswceF05A7u0Hn9vnBdk5sUR8XTqs/4pdQwO1PR/FfXdvSu1v11B\nfTecCnxy0DVMRDwFeGfzZ4f63n5qRDy1+bdrbIeI2JW6Hr0bvXPtlcD5wE+oc/5HpztPSJIkSZIk\nSVo3613QvvErekH77QYtEBHPpoKoG7J2lun1m9vtgWdHxBsy8z+GvVFEHNa0c22uma26HXATKhB/\naETsn5lDA8Wt17+JCph1+v69+57/DTyf3lzZXRsAN6UysB4E/EdEPDAzfz2g3xsC76Z+lO5/jxs0\nt92oz+D1mfnCEf3tABs0P+KfQC9Y3bUVFQzZJyLulZmPZxaa7Mf2a18wbNkR3kUFxIYGsyPiEdRn\ncz3WXo8NqQD8zYHHRMQPgAcPClj3vW5j4EQq6D1K9wfyj1Dzmw/Nfp6r/XeUiLg18BnqR/hu/7q2\nBO5KTUVxWEQclJmfGbA+3ddMtf5tkGkzvZv9/nnUft7fly2pwMrTI+KDwJNGBNu7++u1gWOoAFa7\nvc2p4MPdqKzb+2TmVddsZt5s3Hr852ELRcRNgU9SxymsvQ47UOeCA4AXRMQjM/P0vibay/efSzrA\nF1uP7wkMDNpTg0ba7e3NNQcEdO3TWvbE9hPNQKRP0ZsbvNveRtQ+eDPgkRFxOPDwzPztoDeIiBVN\nXw8fsF7bNLc7A8+PiCMy800DmpnIcxtN3yPi4cBH6a3LNfbviLgnNXDlhu3XNrr714HAiyPiwYOm\nfoiITalt2q0Y0W5jU2pA1Erg8RHxWeCRmfmPvjZWUgHgHQe0sQUVkNyNOg+9IjNfNWzlp3EUvf3z\niYwftD+k9fidgxZo9u0T6A1i6j9/7UGdbw6LiMdk5hfG7XTLwOuFVh9uSB2Xt+xbbsfm9qimBPt3\np3ujiLg39T3VrUjQf/2wd3N7TrNv/GBIX2HwuaffqO/CjahpJB41pC83oAaLPCcijszMl3FN/cf7\nVsDnqO+3/u/XewP3joi9M/Mpw/o1hvb187aMONe3ZeYx0y0TEbtR3w03bf6puw7Xpo67m1Hb+zvU\nufS8viZGfW/3X4++gSr1P+h6dEfq3P0QetejIysKSJIkSZIkSZq99ao8fks7U/camT0R8QLqB/lr\nUaVGX0v9qLgbFVw4ksqoWkEFdgb+eN/MP/xGeoHT9zXt3KW5PRFY3Ty3G5VxP52VwNOogNb+VEDi\naa33PJheQOdC4AhgX+COVEDpaVQWVYfK+v9ME8Tq9zEqYN+hsk9fQQXfdqWyy18HXNZ8Bi9oAqbT\neT8VfHgnNe9qt6130wvgroqIA8Zoa5B7UwHNDvDzzDx1pg1kZmeagP2BVDBrMyqQ9XEqqHsnKqjy\ndCojr7tNT4mI60zztk+gguhvAe7RtHX0kGXvSwXsj6G3PV7R18c52X9HiYjr0guUdaiMvidT+9gd\nmz5+pHluU+CjTRZ8235UVmw3YNOh9ofbNrdPz6A/b6T2+xXUfvl64H7U57MPVaniwuY9Hgd8aIxm\nXwM8FDiWGuTSbet/qPNGp/n7GeP2c47s1Xp8yqAFmiDv16lt3gG+RW2fe1CDKR5HBeg6VEbnVyLi\nZq0mTqW2wX3pBVu+T2/b3C4zz6SymKeAuzVZ/YPs29z/pVl27xHr1g2KXgZ8s7U+O1OVPXZq+vJ/\nVBB7j+b2FGof7FAB95MiYosh7/E2alATVFbpy6n9dTfqnPpWKnt3Y+D1EXHoiP52vZ8lfm5r2ZA6\nf6ymqhrsRm3Dy7oLRMRdqEEb2zR9+gKVJX4Xav86mArudqhBTKdExLYD3uvt1HmgQ2UGP4WqDnNH\nap1fCvy2ef4AattdrRlcdgIVfOxQg4ge3fRjN2pA3FuorP0VwMsj4jFjfAaDHE9Vqpmqt467T/eC\n5nO6bdO3CxhwTouI6wGfB7ZvlvsedW1wD+pzOICaq7xDDUL4RETs1N/OHDieCth3qGosz6L26e72\nPBX4f9Q5cahmsNBx1ICVDlUZaBX1/bgr9X35lea5bYHPRcRm3dc3gzK655nu1A9XUIOubktVdPjJ\nDNfrUc37/bFZh32bvtyPukb7G3Vd+NKIePkYbR7TvP6t1HF+R+q4fx+94/2JEbEu05e0KzL8Z0QM\nrTAzExFxK+rcvwPVzxOAg6gBCHejrhF/0jy3B3Uu3bTVxEep7dC97uxQ10Hdbfag1ns9DXh28+cF\nwH9Q5/g7UN9jz6C+/zvArZnBd74kSZIkSZKkmVtfM+2j9fjMtZ6oHzT/k/oR8XxgrwElU0+KiPcA\n36GyDA+NiI8OCKS8hF5m0Yszsz+w/YOIOI4qFbstsFdE7JmZ3xrR98MYUg690Q4g3jczf9T3/Lci\n4t1UUP5h1I/0B1LZjgBExKOpIG/3M7jbgOynrzZ9P4UK8jwvIo4dkK17dV+ooNP9M/OkAW2toAIV\nHeoH5M8OaWeUdjBzWCbvrDWBhfZ8xE/LzHf3Lfa9iPgAcBL1I/jNqKD68xnuMODwIRm9/Z4NvCUz\nnz3oyTnef0c5iN6P/qcDd8/My1vPnw58ISL+Qv24vxH1GTypu0C3XxFxWet1azLznBn0g4jYAzi0\n6ctlwD0z88d9i309Ij5CBcS2Ah4RER/KzP8b0uxKKihzcGb2B/i/HhGXUvOCd/fXt86kz7MVEd1g\nXocq4f/FIYu+nV4Q8PX9ZeapAPyHm8E2L6AyRN9CBTnJzL8Dv+jbNpcNyJb+EjUFw6bU59WfMQsV\nIOtQ87EfBtwlIjYakDF9Aypw0wFO7quE8CGqwkEHOCwz1wreAqdGxHupYNqjqOPu1cAz+97jfk1/\nO1RQep/M/GNfW1+MiA9T55BNgP+KiOOHZe4zAee2Pg+jStzfrb90OVw95cIHqKxcgP/OzBf3Lfb9\niPgQlc17IHXMvRl4RKudLagAe4eavuLumXlJXztfi4ijqAEANweeEBEvbpV2vy81r3YH+GxmPqTv\n9T8GToyIY6jz3QbU1DLHjvVJtGTmPyPif6mBcFDb9JsjXgKt8x3wv0OmwDmEKtfeHRizV2Ze0Xr+\ndODzzbF4MFWG/blUUH1ORE0B0s0a/yNwl77s6u72/Di1TqMqnzyHOh90GHyt8hPg+Ij4MjXwbBtq\nQNHruwt0zzMR0R1U2ZnN1AbNIMb7NX35LbBnZv6+b7EvR8Snqe/sDYAXNd+HvxzS7IOpff/emdm/\n/b8SvbLz3eN96LQO03hbc5uiAuGnRsSrgc+t45QsH6a3fZ6Sme/pe/57zX7+MWqAxi7UwKbDAZpz\nwl8iYvvWa/48qJIGa1+P3mvAMt9srkePo757bhcRDxjxvSxJkiRJkiRpHax3mfZNxuZdmz+v4JoB\nkBfRy8R/xrA5TjPzLHoZ7VNcMzi0gspuPZYKbPUHd7vt/IW1Awj7Dlqu9T7b0vx4OsTK5v5PAwL2\n3ffsNH1/FTUfcX/W2mGtxy8YVq40M79HBbGnqG09qhTsplSwuT+o1fXe1uPdhiwznZ1bjweu+zp6\nLFVKF2pu4mHb9HLW/rH6kCYjdJApKkD25jHef4rKtHzpiGXmZP8dw8XA/1IByNf1Bezb3t56PGrf\nXhfPplem938GBOyBmtebGtDQ9fQRbV4HOH5AwL6ru79OUYGGuciC3D4ibt13u01E7B4Rj4+Iz1EV\nGLoDJQYO3ImIW9Ar6f+TAQH7tiOo6UKmgPv3ZduPoz1o4J4D+nIreuXTj6YqFFybyuDstze97Xh1\n+e+I2JfKnO4GZvsD9gA0UxQ8ldo3p4CDB1S5OKL1eNWAgH23rdOoigpTVOD+SYOWa0zCua1tO+AV\ngwL2jfvSG/i2hiHno+Z75unU3NVT1FQSN2otsjO9gX3fGRCw77ZzETXw58VUELQdsFzZevylYSvU\nlF9/BjV9xkuGLTeGo6kKK1PAw5ss+YGafe+RzZ9XMeQaAPgTtY90z6VXDFluPs+lj209fu2Acujd\n7fk0Kit9lN9QlSc+T2WxD9OeKmC+vhva1zLPGxCwB6AJvr+P2q7XYvrj/XUDAvZdc3G8Q03H0K2G\nApXN/yngwoj4TES8ICL2GHFtcQ0RsT9V1acDfGJAwB6AZnDJk6lKPVPAk5rBCGNrBvdE816/HxLU\n777X84BXUgNYZjw4Q5IkSZIkSdJ41qtM+2Ye0g9TAeYOcHRm/rX1/BRNpikV+Omfg7vfJ6ig9cat\n1wFXB5CeN2bX2mVQB5UQ7uoAp2fmb0YsczkVZNoiIm42bNkmaPvy/n9vgip3af68jFrHUY6ll/G3\n/5BluuW1/3dEO6tby24zzXsOc/3W4wtn2cYo7SzO941aMDN/FBFJ/Wi9BVV2dlAp8w7w6SYgMp0O\n8JVhwa253H+nk5kfpcrkTqe7b3cHnMypZnDMA1r/9P5pXvIRKqNzCrhXRFwrM68xRUZj6P6amec3\nVQQ2pypNbEEF39bFqABX19lU2f73jMi2fBC9c9wHRzWWmVc1WeXdeZwPYLwBJF0nUYH4Daigff86\ndINxF2Tm6oj4KVUaeW+q3H3bPq3H7cEA7ePu/aM6k5l/jYgTqNL5G1Ol10+Aq6cM2JP6XH42bHBH\nywfpTT1xAFVWu9+knNu6pqgA86gy1e3t8aHmu26gzLwgIk6iAv0bNPcfaJ5uD/S5bURsOGyfzsyv\nAl8d8FS7jTuN6DPDBlnNRGaeExFfoL7rNqGC3QPnqaeqCGxGM33AsAoizcCgcabrmM9zaTtoPnTb\nZ+aFEXEiVY1h4HdWZr5lzPcc97pnViJiR6qUPtT34XSl14+lF6zfn8HXbwt1vHfPzd2piJ5Gfc9A\n7VMPpPedfXlEfI8a6HTCiAoBMLNz6UUR8X/Av1EDFfZh7fPydP3vRMQV1HGydUTcJDN/N2TZX9E3\nzY8kSZIkSZKkubdQQfuNIuLWQ57bkPrhdE/qB9kb0Zs/94i+ZW9DrwTzWaOCEVAZ1RHxcyqbavOI\n2GFUae8mG/fG1A+g7czcrdvrMuo9gVE/yEIF0R5GffanRMQLgeOactfj6AY+ugMEhmX9df2YyhTf\nANghIjYfkqH5t2l+TP5r6/F0c8AP0553dbpswNloB4W+N8byP6CXkXp7hsw/zvTbdNxl53X/nU4T\nFN2SyqTuZky35zmfUabemG5F7S8d4LxhmZRdTbD9XCpItGHz+v4pHdpzuI/yV+rzpunDugbtxxm4\nsQNwJFVi/r8yc82AZfZsPf71GG221/P2Yyx/tcy8pAkY7UmV7e93r+b+G637OzJ4Xvtu4PA3TVWE\nrtmsz+Obx7enCdpTA2e623ZgBYq2zDw7Iv5EBcxXRsQGQ8qbT8K5re0PmXnxiOdncx68b/O4vX+t\nBs6jvhMDODkiXgycMuYgJqjvO6jt+oRm4NJrZlNKfQbeSW+A2hMZHrT/977XjK2ZKmIL1j6Xtq8N\nprtOmMl7bQd0Kwb8re/YG+Q06hpjJu9xQ3oDnLp2bD2es/VpaV/L/GjIsdvWndpjCrhFRGwy5Lrp\nz82gx2Hm4ngHrs5Cf05EvIGqIvIw1q66ATU4aa/mdmQz7cBLc/B0N7M5l/5b8/j2zCBo3ziJGlS3\nEfCNiHgRVcHmH6NfJkmSJEmSJGk+LFTQfmcqCD+dTnP7DDVX9WV9z7ezve4YESODngNsD6wV9Gyy\n+w+lsl9vw7p/JgPLObc8jyo9fWOqzPGHgKMj4jtUZuuXgVNHBEXa85SeOV1nMvMfEXEBvc9uW6A/\naN8BLpqmnU5EjFpkHO333XzoUrMQEZtQ8zJ3TfvZAGe1Hm83Yrnptum4y875/jtKEyB7OBUgvTu9\nwE+/DmsH7+fSjPbXxln0PqvtuGbQvmu6IHz7852L9Xt4Zh7f/49NNYGtqPPHgdTgo8cDj4qIgzLz\nk30vae8HJ8zguJpi7c9zXF+igkFbRsRtMvNnTb+nqEBSh96Ala9T58O15rVvKnx0Syl/oa/9bekN\naPjJDM8T7fVpfy4HzvD42JAa8NU/KGTJn9v6dOc0H2Wmx9xZrcdXnwebOeKfTJX83pD63joJ+FNE\nnEKVBv/SkIEp3TZ+HhH/RU0LAnVcPD4ifkN9350EfHnYNAiz9HnqPLkDcIeI2DUzf9heoJkWojtH\n/G+BE0c12BwrjwIeRw0u2WzIovNxLu0eFx3g3DGWHzhdTltTSv1gap3uQmVbD7LefDdk5qURcRF1\nrl1BHe/9r+uw8N8LZOZvqekhXtxUELg7dc7dg/peaA+S2w/YOyKeN2Aqkfa5NNfhXDquZwN3pgam\n3pSqZvC3iPg2vevR789gkI4kSZIkSZKkdbBQc9p3RtyuAv4M/IwqBX73zDxwSCbhlmO2OegGfUHL\niLgb8AuqrPIdqGz0Ua8fx59HPdlkSt+FKkF8RdP2xlQW6yuB7wDnRcSbmwy7fu2A0KVj9qm93LDA\n7XRZbnOhXTZ6rsvttj+XK8bI2oPxPheYZpvOYNk53X9HiYjrUsGrj1FZp90y0Ouyb8/GfO2vLEIg\nYWCAJzOvyswLM/PkzDyMCtT8hcrC/WATIGzbkt7nPtP9YFiwcJRh89rvSmULQwVgoYL2UJmXu7eW\nbZfG7w/az3a/hrW377ocHx0m99zWb7rz0UyPuaHHW2b+H7XPfI3e57wV8FDgrVRg8RcRcXgzcOoa\nMvMl1Fz3Z7TauBk1P/Yx1PfdKRHxiDH6Oq3mvNAutT9o/vMntx4fPepcEhHXo4KXH6YqEmzKwp5L\n21Uc+gcyDjJymzfXFadS1QX2oq4/Jum7YSGO96Ey86zMPCYzn5aZd6AqgTwSOJ6aqqRDDYJ5c0Tc\nv/u6ZmBIe53m7fqg1ddfU0H7Y4Arm7Y2Ae4NvJqq1PH7iHh9M3BLkiRJkiRJ0jxaqEz7n2fm7aZf\nbFrtDKlvA0+Z4euvzkBrysEeD9yA+qHyN8B/U0GrP7YHDUTE45lmjvQhfRwoM88FDomIl1DlVO9P\n78dzqOkCngU8MSIOzsyPt17e/iF93Cyx9uCMmWZ3z6XT6ZVh3h14+xy2PZ+fy0w+s3HbWaf9dwxH\nUZ91h/ox/i3AJ6ms2ova88TPIuN/Jpby/jormfmTiHgVNdfxRsB/0CsHD7VO3TLwq4CfzKD5cafR\naDuNCvRuQQVg39H8e7c0/h+75cqbObFXA7ekSuR3M/C7Qfsr6AX4u9rrszczm4agXaq6va0/QQ1i\nmolxyknPl/k8t/Wb7piY6TE38nhrynjfOyJuR1WSuD81ZUf3dbcEXgM8IyIO6FZy6GvjWODYiNgP\nOAC4H3DzVh/vAdyjyex/WGb+tb+NGXoPNRhvQ+DREfHcbin1Jsv8sc1y/2T0/Oc0z+9Lfa5XAG+k\nrh3OpnUujYiNmN3xORPjBNKHXtc1geFPUXPJd6jj73+oAV7nUutzVbNsUFMkzFfwfll8NzTTEX0K\n+FRE3B74LL2KFi+kqfLQVPzoUOvYoSo6XDKDtxo1Zcao/v2Wqn7xYnrXo/ekNyXCjYDnAE9uKsd8\nejbvI0mSJEmSJGl6CxW0nyvtEsQrMvMX69DWU6mSoB1gDXCnzByW7bXBkH9fJ03w/q3AW5tAwt2p\nrOiDqMEEmwDHRMTqzOxOL9D+YXbcrNt2pt6g+ewXyjeA51M/0N97xBzUs9H+XDaMiA0z88ppXrPQ\nn8tc7r9DNeV5H0MvKPLgzBw4121T2n0+LeX9dV18ngraQy843tVepwvnaz/oaoJBX6WmSmjPa98N\nRJ7S95KvAyupAPwrmn/bp1n2W5nZP2f7X+gNODp/VLn0abQ/l3/N9+cyx+bz3DZTFwM3bB5vBpw/\nzfJjHW+ZeTo1OOEVEbEFtV8/hAr0bUSV1/5cRNwyMy8f0saXqax1IuKmwH2oLOR9m0X2pYLk65R1\nn5kXRMQJ1D6/GTXv9/ubpw+ksp87wAmZecGwdprA9cPoVQXaPzP7B610zct1Amtn148zB/uo8+x9\nqEo/HeBvwN26A3YGmK/16Vp23w3NgK4n0qtWskffueISehUIzm0C6gvVt98Bb6YqAGxEfVc8gBpY\nthVwXeDf4PAXAAAgAElEQVRjEXG7zMyF6pckSZIkSZK0nCxUefy50s2knAJ2Xse22uWeXzsiYA9V\nyndeZeYVmXlSZh4O7AR8pnlqA+AlrUXb2dY3ZxpNyeJuAOcq4Hdz0N3Z+iq90s7bUMGaGYuIo5rp\nA27c/bdm7u0/tBab9rNh7e06kyz22ZrL/XeUvellLn57WMC+Md/7dvtzHfe9Fnq7zIduQGmKChC2\ntTPCd1mY7vCl5v5GEbFzRGxIDRKCa2bOd0vk3zUirh0RN6F3PPWXxoe5W5/F+Fzmyryd22ZhRt8R\nzOJ4y8yLM/NTmXkQNcDjV81T21NzpY/TxtmZ+e7M3I+qUnA5dbw8NCJWjtPGNI5qPX78kMfvmqaN\n9nXCySMC9jB/59Lu99oUlfU8nVHbvL0+Hx0RsIeF/W4Y51pmC3rTeVwJnDcfnZqJiJjNdCXtQVIb\nsPZAhPXiHJiZ/8jMr2Tmc6jr0e55/1rAEYvVL0mSJEmSJGnSLamgfWb+il5g5PpNqdHZav/4PeqH\na6hswgXTZLI+s/VPt209PrW5nwJuGxHTZd7dmV7p6tUDsmQXTFOeuB0kOXKmP3pHxL2p+YifBfy8\n7/Xfaz3eY4zm7tx6fNpM+jEbc7z/jjKTffth89QHADLzl1QAewrYJiJ2GrV8ROxABT0BLlnCGX3d\n8scd4I99z53aenzvhenO1UF7qNLHd6WXtXty37LdoFJ3Xvu9Ws8NCtrP1fp8n/q8poDbR8RW69DW\nglqAc9tMzPQ8eJfW4xmfBzPzbNaeyuC2w5Yd0cZXgGPXpY0BbZ4EnEHtT3ePiG0i4vrUPtoBfpWZ\nX52mmfXhXHo2vbL714uIbadZfvcRz81mfcYtXT9T3f10CrhDU21olO5+2gFOH6OSzryIiA0i4iMR\nsQa4OCLuNMMm2tdsVzbl87sW47thpGZA66Gtf1rnY1OSJEmSJEnSYEsqaN/4ZOvxoUOXAiJiq4j4\ndUR8OCL273u6PfdsfyZsu41DgFvRKzW+4Uw629fW/hHx3oj4/hg/vLdLx14daM/Mi+gF2TYGHj1N\nO+2sx+PG7es8eg1wDvVD/fbUPMcbjX5JiYg70gvsdIDXZ2Z7ztdPtR4fMk1be1EZZB3g7Mz84Xjd\nX2dztf+OMu6+fUPgubTmFo6I6Uoiz2ZKjeNbj0duF3r7awdYynPntjOtT+177jjgX9QxsH9EjMxo\njYhXRsRpEfGyiLjBiEWHbpvMPAfoDoC4O72S/Rc0Ayvay54L/Kb58x7Ans3jcwfNV07NP991UERc\nb0QfiYj3R8QpEfH8phJI930vBr7S/LkB8Ixp2omIODsi3h0Re45adoHM57ltJtrnwVURMWqO852o\nQRxQZdi/1HruWRHxsYg4cYz3vMb3VURsFhH/FRGfj4gjZ9PGHOgOpFhBlcp/CL3j5KiBr1jbuOfS\n7ajzefdcOmdTD2VmB/hW658ePKIf29Ob9mKQcdfn9qw9xcp01z0zLqWfmecB323+3Izpp0RYL65l\nmlL229CrDvD/ZtjEo5r7DtesctI+lx4cEdcd1VAzeOBrEfG8ac4119gfI+LBzbn4tIjYepo+z8ex\nKUmSJEmSJKnPUgzav4kqjToFPD4iBgatm2DQR6jA7KOA6FvkJ63H/zakjYcAbwM+Ry/b7Kaz7jls\nCzwBuCO9eeyHaZfw/Xbfc925sqeA/xyWvRwR9wcOav68jOlLAc+7JjD3WHqlkB8AnBQRtxr1uog4\ngCpBfQPqx+7PZuZ/9i32Mar8/xRwt4gYGPRrMnjf1vqnN8xiVWZrrvbfUdr79j6DAr3NoJETgXOB\n9ry5g/bv9g/2synr/yZ6QernRsRugxZqApfPb/78V/O6JSci7sPaAzLa+1p37uCPNn9eiwruDsyq\njogHUp/JbsBTgP5pPNrbZrpy1l+itsGd6ZXG75/PvqtbIn8vKnO3w9rZ+lfLzO9QQcUpYEvgg035\n/WuIiCdR56R7AI9pMtTbXtvcTwEvioi9h7SzNXV8bE8NBNlm0HILaZ7PbTPpx9epqgVTwI7Afw15\n342o+eNXNO97dN80MXelAqn3iYjDp3nb9vfVd5r7S6n5sO8HHBoR9xj24mb/P7D5819cc6DLbL2f\n2h5Q59HuYJp/0JvjfpT2uXS/pkT7WppA+YnUFAHdUvYbjDEwbyY+2nr8wkFB1mZwxtGMDqC31+fA\nQQM6IuI2wGeBL7f++UZDBn90zz/XiojZXBu1r2X+p/ksr6G5Futuu78C75nFe82lI6jphqaAB0TE\neyJi02leQ0Q8gBrc0/Xa9vPN9As/aNrdGnjfsEE3EfEs6tp1L+ARzRRBbdN9b28PPA7YFXjLqME9\njL4elSRJkiRJkjRH5iwbbKFk5uqIeBH1Y+cKoJuF/HEqYHt9Kij+DGAHKhjxY+DtfU19gPrBcgp4\nTERcCXyY+qHz5lTw5YHUD5TPaB5PAbtHxFOooMJv+kqbTucDwHOoOYAPBH4YER9o+nchlTm/PfBQ\nKisQKvDx+r7P4PMR8X7qh9RtgFMj4o1UoO1SqjT3g6gBAt2AzDObzLZFl5nfioj9gBOoIN8ewI8j\n4gvAZ6h5XS8CNgfuQP0w3S3z3KECGE8Y0O4/IuIJwBepwMVbImIPKpj/W2ru2N2pqQe6+8ZJmfm2\n/rbmyxzuv6N8HTiTCvhvAZwSEa8C1gBbAftRAeANqc/1dc17QQVOXgtc1JTzh7Xn2X1Ls69dAfw5\nM9tZ9MPW+fTm/V8GbAKcHBFvo4LAF1H78H2Ap1KlgzvAqzLzxzNY54WwfUTcesC/b0Ctw3bUeeKh\n9KakeENmnjzgNYdSgfMdqLLPp0fEm6ny5FdSgyceSZ0nusfwszPz8nYjmfmXiLiI2q43johjqf1/\ncyr4e2Zr8S9RpdejaX9QpmfXKdQxtju1zWBwafyuJzR935w69/wgIt4KnN70fyfqfLVfs/yVrD0F\nSHd9vtK87pnU+fBLEXE0FUT8A3BDap99Gr0g9xfG2Q8Xwnyd22bhCVT58esChzf77XupCgobN+/9\nDHpVZH4BvKSvjVdSmenXAV7TDEb5WLMOFwPXA25JDcLYs2nnRzQVMjKzExFHAB+k9qEvR8THqEFw\nv6cydrei9rFDqEEnHeCozOwGv9dJZv45Ij5OfdfvSQVaO8AnMvPPI19cvtr0dTtqf/t6RLyaCtDf\nALgvNaVBdzDM0dQ+CvDaiHgT8KfM/E1/wzP0QWrwTlDXCKdFxH8D3Qoxt6XOKbcB/ht4EYOz7T9J\nne+vQ227r0bE66nvnhtTWfyPo7bvwU3729HbBz5MVef4XdPer+kNKDs2It5LXVeubgaPjJSZx0XE\nR6njYFvg+xHxBuCb1EDD7Vt9mqK231My88Lp2h7DsGoE08rM70XE04F3NP06BHhIRHyS6vuZ1LXY\nptR3w82o9egOluoAL22mcOh3EFWB4HrUdeAtIuLtwM+o75qbUdtmn2b5f1Dn9X7dfW4KuEfzvf1D\naoqEd1ADHw4FbkF9/rdqrkd/Sl2PbkJ9Pz2M+k6D2i/ePNaHJEmSJEmSJGnGFiJoP+fzoWbmGyLi\nH1TG0sZUgP2xfYt1mttXqIzOK/ra+FoTJOtmxD6etbOJOlTg6kGZeUlEfJP60f9awDubZfaml5E6\nTr+vjIj7UUGNO1DB+/8ZsniHCjQ/spkzuN8TqVK3T6ECH68e0salwNMz85gRXZuvOWuHaoJbd6Cy\nQB9D/Rj9wObWr7stzwFenJnHDlim2+5JEfEg4ENU0OwxzW1Qex9j+nLtc24u9t+Wa2y7zLwqIh5H\nZX9el9rP+j+zi4EDMvMnEfEReuXSH9bc3k/vs3k3cBgVfN2Z+sGfZpmxgqWZ+cqIuAp4KRUA+o/m\n1tahBgP8v8x8DeuXKeCNYy7bobJBX5aZAwMcTTDxblSZ57tQwZFBFR+6bT0zMz8+5P2OojI/obKJ\nH0VvsEc7aH8y9flem9ovOlxzPvuu7nmtmz36T9bOvO1fn183JeqPA3YBbs3gyh4dKvj+uMz81oDn\nyczDIuJSKki5AfD05tbfTofKtn/isH4xQee2xljrk5m/iIh9qO2xHXD/5jbovb9Gfc/0Dwg5ozmX\nfoQKUN+bwfNsd9v5NvCwpnx4t41jImIb4EhqkNBB9Kq/DGrjaGpg21w6il7Qd4PmfcaqOtN8Zz+O\nGjSyCRUU/2jfYn+irhNWN+fS7lQNj25uR7H2/jvjfTIz/9mqyHATKpj9jr7FOtTnfBwVtB/03XBh\nM+jwfdRncY/m1m7jbOB+mfnHZn0Ob9p6dnN7Ib1s8aOA7tQte9AbgPJC1r42GrXOB1EB+kOo/WzQ\nVArd8+CTR5wHp3ufdVn2GjLz6Ij4FfAW6jt2S2oAx5OHvKS7j58FPDczTxjS7i+bqhTHUQH621Pf\nwYPaOxc4KDNPG9DOHyPiOHoDvw5rve4DmfmH1vXoral9+3X97bRecxbw8Mz8/ZBlJEmSJEmSJK2j\n+S6P32nd5lRmvp3KiH8lFSz4A5W9eQk1d/MHqR+e75uZfxrSxnOoANfXgD9Tgak/URmpjwD2bc0r\n/Gjqh/s/Ne/xbeoH066x1jEzf0tl5D2amr/0DCqw/i8q8/AsKiPzSUBk5veGtHNVZj4DuBM1iODn\nwF+ooNwFwDeozMmdpgnYz2Qbzen2zMzfZebjqEy9w6lM3jOobXEllZH6Y6qE84OBW4wR1CIzT6R+\n7H4RFZS8gPpcLqYySo8G9szMx/QHqvrMZF1n9LnMxf7LiO3RBER3owLrZ1Pr/zcqi+5lwC6tTMj3\nU/PynkmVkv4VvbmGycykSlyfRu2rfwVWUyW4B/Vn2Dq/msrsfQOV8XdRs85/pCpXHAnsPE3Afqb7\n4Fzsr51pbldR2+1XVADkGcDNhgXsuzLzvMzcg8piPJbKWr2U2lZ/oPbdF1H7/ahj+OXUoJ3fUFmX\nf6SyPc/ve7/LqDL23X6fl5lnDOnbmdSgoe6ypzbl30etz2oq+PN4ajDHOdQ+dzlwHlUB4FnUNh46\nAKBp68XUvvI6qlz0hdS+8hdqH34XcLfMXDXiGJ60c9uM+piZ36cG2Tyref9zqf3jEqrqxoeA/TPz\n3pl50ZA2TmraeC61/c6hBov9izoP/AI4hgpa32NQhnxmvoHK6j6SOtdd0PSj+zmcRmXv7paZT2sH\n/edCZn6X+qy7n91PM3PsEt9NyfI7U5/Xb6nj8zKqisSLqe/p7pQA7wJeRX2PX059zu1S/9Ntw6HP\nNZVPbk2dv39EHQuXN+/1UWC/zHwJtX277VzjGi8zP0wF6o+jzhFXUtvyVJps/dZ54eXU4IDfUdv9\nZ9Tx123rc1RVh581fbm46dvqcdc5M/+VmU+ipmN4F7VPda9lzqcGTx5BXcuMCtgv+PGemSdl5m2o\nCiJvpM7Zv6N3TXcpddx9k6qWc3/qWB8YsG+1+1PqmPl3qmpH+1x6LjUg7+nUvnfyiKYOpgL+v6WO\nufOogYB/a97nN1Rln1XAp6j9tdv3y6hrgk9TA6NWZuYPkSRJkiRJkjRvpjqdOY+nS5IkSZIkSZIk\nSZKkMcx3pr0kSZIkSZIkSZIkSRrCoL0kSZIkSZIkSZIkSYvEoL0kSZIkSZIkSZIkSYvEoL0kSZIk\nSZIkSZIkSYvEoL0kSZIkSZIkSZIkSYvEoL0kSZIkSZIkSZIkSYvEoL0kSZIkSZIkSZIkSYvEoL0k\nSZIkSZIkSZIkSYvEoL0kSZIkSZIkSZIkSYvEoL0kSZIkSZIkSZIkSYvEoL0kSZIkSZIkSZIkSYvE\noL0kSZIkSZIkSZIkSYvkWovdAUnSutvkjs/sLHYfFtrff/S2qcXug6RFtezOe5IkSZIkSdIM+Bv6\nEmKmvSRJkiRJkiRJkiRJi8RMe0maBFOOwZIkSZIkSZIkSVqKDNpL0iSYssqNJEmSJEmSJEnSUmTQ\nXpImgZn2kiRJkiRJkiRJS5JBe0maBGbaS5IkSZIkSZIkLUkG7SVpEphpL0mSJEmSJEmStCQZtJek\nSWCmvSRJkiRJkiRJ0pJk0F6SJoGZ9pIkSZIkSZIkSUuSQXtJmgQrNljsHkiSJEmSJEmSJGkWDNpL\n0iSwPL4kSZIkSZIkSdKSZNBekiaB5fElSZIkSZIkSZKWJIP2kjQJzLSXJEmSJEmSJElakgzaS9Ik\nMNNekiRJkiRJkiRpSTJoL0mTwKC9JC2YNWvWcMkllyx2N7TEbbbZZuy8886L3Q1JkiRJkiStBwza\nS9IkWGF5fElaCGvWrGGXXXZZ7G5oQpxxxhkG7iVJkiRJkmTQXpImgpn2krQguhn2xxxzDCtXrlzk\n3mipWr16NatWrbJigyRJkiRJkgCD9pI0GabMtJekhbRy5Up23XXXxe7GgnFKgPmxevXqxe7CRHHK\nAUmSJEmStFQZtJekSWCmvSRpnjglwPxZtWrVYndh4hx//PHssMMOi90NjcFBFpIkSZIk9Ri0l6RJ\nYKa9JGmeLNcpAc455xwOPPDAxe6GZshttrScccYZBu4lSZIkScKgvSRNBjPtJUnzbLlNCdC13AYr\nSAth9erVrFq1ymk3JEmSJElqGLSXpElgpr0kSfNiuQ5WkCRJkiRJ0sIxaC9Jk8BMe0mSNEfWrFlj\nBrTm1erVq9e6l+bTZptt5jQMkiRJktZ7Bu0laRKYaS9JkubAmjVr2GWXXRa7G1omVq1atdhd0DJx\nxhlnGLiXJEmStF4zaC9Jk8BMe0mSNAe6GfbHHHMMK1euXOTeSNK6Wb16NatWrbJ6iCRJkqT1nkF7\nSZoEZtpLkqQ5tHLlSnbdddfF7saCcUoAabI5FYM0mZz+QpIkTRKD9pI0Ccy0lyRJmhWnBJAmn1Mx\nSJPL6S8kSdKkMGgvSZPAoL0kSdKsOCWA5sI555zDZZddttjdkKRl48wzz+SlL30pp556qtVyJC1J\nS/X68QY3uAFbb731YndjxqzOoqXAoL0kTQLL48+5iNgdOBS4G3BD4HIggeOBt2bm32bR5gbAY4F/\nA3YFtmraPQf4GvDOzLR2pyRJi2C5TQkAy3NagPn4cfT888/n8MMPn9M2JUnjsZKGJGlcVmfR+s6g\nvSRNAjPt51REHAG8GugA/wDOBDYD7gzcBTg4IvbNzHNn0OZWwIlNGx3gL8AZwCbASuDWwFMi4rDM\nPGoOV0eSJOkanBZA0ro6/vjj2WGHHRa7G5IkLTmrV69m1apVvOpVr2KnnXZa7O7MyFLMtO9+3stt\nwLKWHoP2kjQJzLSfMxHxIHoB+zcCL8vMy5rn7gwcC+wCfALYcwZNf4AK2P8DeEZmvrf1njcC3g08\nAHhbRPwgM0+bg9WRJEkaaDlOC7CUfxxdys4//3z+/ve/z+g1W2yxBVtuueU89UjrqluWfCmW9NVo\nlg6WpIW1//77L7tqX5KGM2gvSZPATPu59DoqYP+ZzFyrzmlmnhYRjwB+AOweEY/IzE9M12BE7EgF\n5DvAm9sB+6bd8yPi0cD5VOb90wCD9pIkad4tx2kB/HF04VjRYbJZlnwyWUFh/eAACkmTaLGm51q9\nevVa9wvNc7rGZdBekiaBmfZzIiL2BG5BBddfN2iZzPxxRJwE7As8gcq4n7bp1uNvDGn30ohI4I59\ny0uSJElL0nKs6KCZO+ecczjwwAMXuxtquC3WH869LGmSrA+DORdzwKHndI3DoL0kTYApg/ZzZZ/m\n/lLgOyOW+zJwL+CeY7Z7fuvxxiOW25AaMHDemO1KkiRJ673lWNFBM+fgDo3jnHPOmfipGbpTUJx6\n6qkTPf+ymafS8rJcB3N2p+ia5PO55o5Be0maAFMrDNrPkds192sy86oRy/2yub9ORERm5jTt/hw4\nE9gJeAzwqf4FImJ74JbNn58dv8uSJEmSVBar7Oy6WKxStXPBoOPCWrNmzbKqBLAcpqAw81RafhzM\nKQ1n0F6SJoCZ9nPmJlSm+++mWa79/E2AkUH7zPxnRDwZOAF4SES8HXgTFcjfGNgTeD2wAZXF/8FZ\n9V6SJEkaYDnOH7ocg6nrQ9nZdbFUA5QGHRfOcs3SnERmnkqSdE0G7SVpAhi0nzObNfd/m2a59vOb\nDV2qJTO/GhF3B14EPAV4Wt8iZwJHAK/PzM44bUqSJEnTWR8CuYsVjF1uwdTlHNBcjJLpi13CfDkO\nTOlablmaS7GCxriWcqWNYZbzsSlJWjcG7SVpAhi0nzObNPdXTLPcP1qPrzOD9u8MrASmgL9SGfsb\nAzsC21IZ9ycCp8+gTUmSJGmo5RjIXe4ZnMsxoLmYJdMXs0LAchuYshytDwOv5tNSrbAxHY9NSdJs\nGLSXpAlg0H7OdDPorz3NchsPeM1IEfFOKsP+T8AjgOO7GfURcQPgZcAzgHtFxP0z8+sz6bgkSZI0\nynIL5Gr5cGCKJtly3L+XMo9NSdK6MGgvSZPAmP1c6f6v6rrTLLdp6/Ffp2s0Iu5FBew7wFMz87j2\n85l5IfCsiNgOeAjwDuA243ZakiRJkpY7B6Zokrl/S5I0+VYsdgckSetuampq2d3mydnUEIibTLPc\njq3Hvxmj3YOa+0sy81MjlusG81dGxO3HaFeSJEmSJEmSJC1xZtpL0gSwPP6c+QnwKGCXiLhWZv5z\nyHK3a+4vzsyzxmj3xs39edMsd2Hfa34yRtuSJEmSJEkTZc2aNUuuzPzq1avXul9KNttsM3beeefF\n7oYkLWsG7SVpAhi0nzNfBI4ErgPcEzhpyHL3o0rdf37Mdv/c3O8wzXLt5y8as21JkiRJkqSJsWbN\nGnbZZZfF7sasrVq1arG7MCtnnHGGgXtJWkQG7SVpAhi0nxuZ+eOI+BFwR+AFDAjaR8S9gd2ooP3/\njtn0KcAjgY0j4kGZ+Zkhyx3Y3F8K/GgmfZckSZIkSZoE3Qz7Y445hpUrVy5ybybf6tWrWbVq1ZKr\nbCBJk8agvSRNAmP2c+k5wNeA/SLi7cALM/MSgIi4F/AhKmB/XGae3H1RROxAzW/fAV6Rma9utfkB\n4EXATYD3RsSTgU9n5lXNa7cCXg7ct3n9azPzyvlcSUmSJEmSpPXZypUr2XXXXRe7G5IkLQiD9pI0\nAcy0nzuZ+fWIeBrwduCpwCERcRawOXAjKqj+TeDgvpdOASua51f0tfm3iLg/8BlgR+CTwCUR8Tvg\n2sBOrde+uy/gL0mSJEmSJEmSJtiK6ReRJK3vpqamlt1tPmXm0VSJ/A8A51GB9msDJwNPAvbOzEsH\nvLQzos2fA7cBDgW+CvwNuDlwQ+DXwPuBe2bmU+doNSRJkiRJkiRJ0hJgpr0kTQAz7edeZv4MOGQG\ny58NbDDNMn+nMvjfvm69kyRJkiRJkiRJk8KgvSRNAmP2kiRJkiRJkiRJS5JBe0maAGbaS5IkSZIk\nSZIkLU0G7SVpAhi0lyRJkiRJkiRJWpoM2kvSBDBoL0mSJEmSJEmStDQZtJekCWDQXpIkSZIkSZIk\naWkyaC9Jk8CYvSRJkiRJkrSkrVmzhksuuWRB33P16tVr3S+0zTbbjJ133nlR3luS1icG7SVpAphp\nL0mSJEmSJC1da9asYZdddlm091+1atWivfcZZ5xh4F7SsmfQXpImgEF7SZIkSZIkaenqZtgfc8wx\nrFy5cpF7szBWr17NqlWrFry6gCStjwzaS9IEMGgvSZIkSZIkLX0rV65k1113XexuSJIWmEF7SZoE\nxuwlSZIkSZIkSZKWJIP2kjQBzLSXJEmSJEmSJElamgzaS9IEMGgvSZIkSZIkSZK0NBm0l6QJsGLF\nisXugiRJkiRJkiRJkmbBoL0kTQIT7SVJkiRJkiRJkpYkg/aSNAEsjy9JkiRJkiRJkrQ0GbSXpAlg\n0F6SJEmSJEmSJGlpMmgvSRPAmL0kSZIkSZIkSdLSZNBekiaAmfaSJEmSJEmSJElLk0F7SZoAxuwl\nSZIkSZIkSZKWJoP2kjQBzLSXJEmSJEmSJElamgzaS9IEMGYvSZIkSZIkSZK0NBm0l6QJsGKFUXtJ\nkiRJkiRJkqSlyKC9JE0AM+0lSZIkSZIkSZKWJoP2kjQBnNNekiRJkiRJkiRpaTJoL0kTwJi9JEmS\nJEmSJEnS0mTQXpImgJn2kiRJkiRJkiRJS5NBe0maAAbtJUmSJEmSJEmSliaD9pI0AYzZS5IkSZIk\nSZIkLU0G7SVpAphpL0mSJEmSJEmStDQZtJekCWDMXpIkSZIkSZIkaWkyaC9JE8BMe0mSJEmSJEmS\npKXJoL0kTQBj9nMvInYHDgXuBtwQuBxI4HjgrZn5t1m2uwHwJGAVcEtgE+Bc4OvAUZl52rr3XpIk\nSZIkSZIkLRUrFrsDkqR1NzU1texu8ykijgC+DfwbsA1wJnApcGfgSOCHEbHtLNrdEvgO8A5gD+Ay\n4PfATYGDge9GxNPnYh0kSZIkSZIkSdLSYNBekibA1NTyu82XiHgQ8GqgA7wR2Dozb5WZ2wO7A78G\ndgE+MYvmPw3cCfghcIfMvGlm7gJs3zw3BbwxIlau+5pIkiRJkiRJkqSlwKC9JE2Axc56n7BM+9dR\nAfvPZObhmXlZ94mmdP0jmud3j4hHjNtoRDwWuAdVCn+/zPxpq90LgMcAXwI+QpXjlyRJkiRJkiRJ\ny4Bz2kvSBJjnavHLRkTsCdyCCsq/btAymfnjiDgJ2Bd4AuNn3D+rafc1mfnnAe1eDtxvFt2WJEmS\nJEmSJElLmEF7SZoA8z3H+zKyT3N/KTX3/DBfBu4F3HOcRiNiW+DOzZ+zKasvSZIkSZIkSZImlEF7\nSZoAK1YYtJ8jt2vu12TmVSOW+2Vzf52IiMzMadq9EzVf/QWZeV5E3AQ4BNgd2Aq4APgq8N7MvHT2\n3ZckSZIkSZIkSUuNQXtJmgAm2s+Zm1Al7H83zXLt528CTBe0v3X3dRHxIOBDwKZ9yxwAvDAiHpyZ\np43ZX0mSJEmSJEmStMStWOwOSJLW3dTU1LK7zZPNmvu/TbNc+/nNhi7Vc/3mfmvgw8DJwD2AzYEb\nANdofAMAACAASURBVP8OXAjcCPhsRNxwzP5KkiRJkiRJkqQlzkx7SZoAzmk/ZzZp7q+YZrl/tB5f\nZ4x2u4H97YHPZeaD+55/f0T8DPgOFdh/PnD4GO1KkiRJkiRJkqQlzkx7SZoAU1PL7zZPuhn0155m\nuY0HvGaUTuvxfw5aIDO/D5wITAEPG6NNSZIkSZIkSZI0AQzaS9IEWOxS9RNUHv+S5v660yzXno/+\nrzNoF+AnI5b7RnO/Q0T0z3kvSZIkSZIkSZImkOXxJWkCWB1/zpwN7AHcZJrldmw9/s0Y7f52zPe/\nqPV4U+DSMV8nSZIkSZIkSZKWKDPtJWkCLHbW+wRl2nez4HeJiFED227X3F+cmWeN0e5PW493GrHc\nFq3HF4/RriRJkiRJkiRJWuIM2kvSBOif73053ObJF5v76wD3HLHc/ah56j8/ZrvfoldG/5Ejlrtr\nc39GZl4+ZtuSJEmSJEmSJGkJ+//s3XuwZWdd5+HvPt3mBiGGkAAhxISJeekUpiQYK0BVABMGnNGM\noQyoeEsUBYeJxpKL4BTjFAVYFQcRIkqBISSoiEJNpsShKJKQwYimhDAykJ9dYBJSXgj3JG2ufeaP\nvXs805xr9znn7bX281C71tlnvevd76lU58LnvGuJ9gAjsDCZzN1rK1TVLUk+lWSS5JXLjWmtnZ/k\nabO371rnvA8k+YPZvP+xtfaoZeY9NckPZvrLAH+64cUDAAAAAACDJNoDjEDvXe8j2mmfJJdlGs6f\n21q7orV29L4TrbXzklw9O/+BqrphybmTW2sPtdYebK392jLz/tckX0vymCQfbq39myXXfneSP0ty\neJIvJXnL5v9YAAAAAADAoWi15/UCMBBb+Iz3uVNVN7bWXpbkiiQvTXJJa+22JMckeVymwf7jSS7e\n79JJpr8Mt5hlfimuqv6ptfbvk1yb5HuT/F1r7fNJDkty8uy6ryT5oaq6a/N/MgAAAAAA4FBkpz3A\nCCxM5u+1larqHUmemuSqJP+Y5JRM4/oNSV6S5NlVdc8yly6uMe8nkpyR5DeSfDbTXwI4Lsn/mX1v\nV1X91ab8EAAAAAAAwCDYaQ8wAnbab76q+kySSzYw/vYkO9Yx7stJXjN7AQAAAAAAc060BxgBzR4A\nAAAAAGCYRHuAEZhEtQcAAAAAABgi0R5gBLb6Ge8AAAAAAABsDdEeYAQ80x4AAAAAAGCYRHuAEdDs\nAQAAAAAAhkm0BxiBBdUeAAAAAABgkER7gBHQ7AEAAAAAAIZJtAcYAc+0BwAAAAAAGCbRHmAENHsA\nAAAAAIBhEu0BRsAz7QEAAAAAAIZJtAcYAckeAAAAAABgmER7gBHwTHsAAAAAAIBhEu0BRmDHgmgP\nAAAAAAAwRKI9wAjYaA8AAAAAADBMoj3ACLg9PgAAAAAAwDCJ9gAj4O74AAAAAAAAwyTaA4yAnfYA\nAAAAAADDJNoDjIBkDwAAAAAAMEyiPcAILNhpDwAAAAAAMEiiPcAIaPYAAAAAAADDJNoDjIBn2gMA\nAAAAAAyTaA8wApo9AAAAAADAMIn2ACPgmfYAAAAAAADDJNoDjIBmDwAAAAAAMEyiPcAIeKY9AAAA\nAADAMIn2ACOw0HsBAAAAAAAAHBDRHmAE7LQHAAAAAAAYJtEeYAQWNHsAAAAAAIBBEu0BRkC0BwAA\nAAAAGCbRHmAE3B4fAAAAAABgmER7gBGw0x4AAAAAAGCYRHuAEbDRHgAAAAAAYJhEe4ARWFDtN11r\n7ZwklyZ5RpLHJrkvSSX5YJK3VtWeTfqcY5J8Nsnjk9xWVU/ajHkBAAAAAIBhWOi9AAAO3sIcvrZS\na+01SW5K8qIkJyT5+yT3JDk7yRuTfLK1duImfdxbMw32i5s0HwAAAAAAMCCiPcAITCbz99oqrbUL\nkrw+04j+5iTHV9UZVfXEJOck+XyS05O8f5M+68eTPJTE7RIAAAAAAGAOifYAI7Awmczdawtdnmmw\nv7aqfqWq7t13oqpuTnLR7Pw5rbWLDvRDWmvHJvm92VzvPqgVAwAAAAAAgyXaA4xA713vY9lp31p7\nZpLTZm8vX25MVd2S5LrZ258+iI+7ItNb7/9Rkr84iHkAAAAAAIAB29l7AQAcvJ0L7qy+SZ4zO96T\n5C9XGfeRJOclOfdAPqS19oIkP5Lkn5L8pyQ/eCDzAAAAAAAAw2enPcAI9N71Ppad9knOnB13V9Xe\nVcbdOjse1VprG/mA1tpxSd6e6W3xX1ZVX934MgEAAAAAgLGw0x5gBGy03zQnZRrT71xj3NLzJyWp\nDXzG25Mcn+S9VfXfN7Y8AAAAAABgbER7gBGYRLXfJEfPjnvWGLf0/NErjtpPa+2FSX44yT9kelt8\nAAAAAABgzon2ACNgp/2mOXJ2fGCNcfcv+fqo9UzcWjs+yRWZ7uT/+ar6+saXBwAAAAAAjI1oDzAC\nov2m2beD/rA1xh2xzDVreXuS45JcVVV/ttGFAQAAAAAA4yTaA4zAZKLab5K7Z8dHrDHukUu+/uZa\nk7bWfjTJC5LcmeTSA1saAAAAAAAwRqI9wAjYab9pbk/y9CQnrTHulCVff2G1ga21E5L8dpKHk1xc\nVXcvM8xfQQAAAAAAmFOiPcAI2Gi/aT6d5EeSnN5a21lVD60w7szZ8etVddsacz4/09viJ8lHWmur\njT2ltbZ39vW7q+qSdawZAAAAAAAYMNEeYAQWVPvN8uEkb0xyVJJzk1y3wrjnJ1lM8qF1zPlgkq+v\nMeaw2Wfuzb/ebv/edcwNAAAAAAAMnGgPMAJuj785quqW1tqnkjw1ySuzTLRvrZ2f5GmZRvt3rWPO\nP0zyh6uNaa39VJIrk9xRVU86gKUDAAAAAAADJdoDjICN9pvqsiTXJ3lua+2KJK/e9xz61tp5Sa7O\nNNh/oKpu2HdRa+3kTJ9vv5jk16vq9du9cAAAAAAAYHhEe4ARWIhqv1mq6sbW2suSXJHkpUkuaa3d\nluSYJI/LNMp/PMnF+106SbIwO7+wXesFAAAAAACGTVQAGIHJZP5eW6mq3pHpLfKvSvKPSU7J9Lnz\nNyR5SZJnV9U9y1y6eBAfu3iQ1wMAAAAAAANkpz3ACHim/earqs8kuWQD429PsuMAP+uqTH9BAAAA\nAAAAmDOiPcAILHioPQAAAAAAwCCJ9gAjoNkDAAAAAAAMk2gPMAJ22gMAAAAAAAyTaA8wApo9AAAA\nAADAMIn2ACOw0HsBAAAAAAAAHBDRHmAEJrbaAwAAAAAADJJoDzACkj0AAAAAAMAwifYAI7Bgpz0A\nAAAAAMAgifYAIyDZAwAAAAAADJNoDzACNtoDAAAAAAAMk2gPMAIT1R4AAAAAAGCQRHuAEdgh2gMA\nAAAAAAySaA8wApI9AAAAAADAMIn2ACPg9vgAAAAAAADDJNoDjMBC7wUAAAAAAABwQER7gBGw0x4A\nAAAAAGCYRHuAEZDsAQAAAAAAhkm0BxgBG+0BAAAAAACGSbQHGIEFe+0BAAAAAAAGSbQHGAE77QEA\nAAAAAIZJtAcYgYmd9gAAAAAAAIMk2gOMgJ32AAAAAAAAwyTaA4yAZ9oDAAAAAAAMk2gPMAJ22gMA\nAAAAAAyTaA8wAqI9AAAAAADAMIn2ACMwcXt8AAAAAACAQRLtAUZgQbMHAAAAAAAYJNEeYATstAcA\nAAAAABgm0R5gBDzTHgAAAAAAYJhEe4ARsNMeAAAAAABgmER7gBHwTHsAAAAAAIBhEu0BRsBOewAA\nAAAAgGES7QFGwDPtN19r7ZwklyZ5RpLHJrkvSSX5YJK3VtWeA5z3BUl+MsnZSR6T5P4kf5/ko0mu\nqKrPH/zqAQAAAACAoVjovQAADt5kDl9bqbX2miQ3JXlRkhMyjer3ZBra35jkk621Ezc455GttQ8l\n+ZMkP5jkuCS3JXkwyVOS/FKSv22t/fAm/RgAAAAAAMAAiPYAI7Awmczda6u01i5I8voki0nenOT4\nqjqjqp6Y5Jwkn09yepL3b3Dq30/y/CR7k7w2yTFV1arquCTnzuY9IslVrbUnbsoPAwAAAAAAHPJE\ne4AR6L3rfWQ77S/PNNhfW1W/UlX37jtRVTcnuWh2/pzW2kXrmbC1dkamu/YXk7yhqt5UVfcvmfcv\nkrx49vaIJBdvyk8CAAAAAAAc8kR7gBGYTCZz99oKrbVnJjlt9vby5cZU1S1Jrpu9/el1Tv20JHcl\neTjJO1eY96+T3D57+93rnBcAAAAAABi4nb0XAMDB28K7xc+b58yO9yT5y1XGfSTJeZne1n5NVXV1\nkqtba5OqWlxl6IOz4+HrmRcAAAAAABg+O+0BRqD3repHdHv8M2fH3VW1d5Vxt86OR7XW2nonXy3Y\nt9aOS3LK7O1n1zsnAAAAAAAwbHbaA4yBnfab5aRMnzt/5xrjlp4/KUltwme/OtN/Lj+Y5B2bMB8A\nAAAAADAAoj3ACExU+81y9Oy4Z41xS88fveKodWqtXZjkskx/YeAtVbX7YOcEAAAAAACGQbQHGAHP\ntN80R86OD6wx7v4lXx91MB/YWvuJJO/M9H4Jf57pjnsAAAAAAGBOiPYAI6DZb5p9O+gPW2PcEctc\ns2Gttf+c5Ncz3WH/4SQXVdXeA50PAAAAAAAYHtEeYAxU+81y9+z4iDXGPXLJ19/c6Ie01g5L8vtJ\nfizTYH9lkp+vqoc3OhcAAAAAADBsoj3ACHim/aa5PcnTk5y0xrhTlnz9hY18QGvtUZneBv/pSR5O\n8sqqevNG5gAAAAAAAMZjofcCADh4k8n8vbbIp2fH01trq/1i25mz49er6rb1Tt5aOzLJhzIN9ncn\nuUCwBwAAAACA+SbaA4zAZA5fW+TDs+NRSc5dZdzzM72t/YfWO3FrbUeSa5M8I8lXk5xbVX9+gOsE\nAAAAAABGQrQHGIPeBX0k1b6qbknyqdknvHK5Ma2185M8bfb2XRuY/r8kOS/JvUmeV1WfXn04AAAA\nAAAwDzzTHmAEPNN+U12W5Pokz22tXZHk1VV1d5K01s5LcnWmu+w/UFU37LuotXZyps+3X0zy61X1\n+iXnTk3yqtm5V1XV32zTzwIAAAAAABziRHuAEdjCZ7zPnaq6sbX2siRXJHlpkktaa7clOSbJ4zIN\n7x9PcvF+l04yvYPNYr71TjaX5l//mfuy1tpL1rGOpx7gjwAAAAAAAAyIaA8wApr95qqqd7TWbkry\ny0meneSUTG9rf0OSa5JcWVWLy1y63PeS5NuXnDtjHUtYaR4AAAAAAGBkRHuAMVDtN11VfSbJJRsY\nf3uSHSucuzjfujMfAAAAAABAtAcYA8+0BwAAAAAAGCbRHmAEPNMeAAAAAABgmER7gBHQ7AEAAAAA\nAIZJtAcYA9UeAAAAAABgkER7gBHwTHsAAAAAAIBhEu0BRsAz7QEAAAAAAIZJtAcYAc0eAAAAAABg\nmER7gDFQ7QEAAAAAAAZJtAcYAc+0BwAAAAAAGCbRHmAEPNMeAAAAAABgmER7gBEQ7QEAAAAAAIZJ\ntAcYAbfHBwAAAAAAGCbRHmAE7LQHAAAAAAAYJtEeYAQ0ewAAAAAAgGES7QHGQLUHAAAAAAAYJNEe\nYAQ80x4AAAAAAGCYRHuAEfBMewAAAAAAgGES7QFGQLMHAAAAAAAYJtEeYATstAcAAAAAABgm0R5g\nFFR7AAAAAACAIRLtAUbATnsAAAAAAIBhEu0BRkCzBwAAAAAAGCbRHmAE7LQHAAAAAAAYJtEeYAQm\n9toDAAAAAAAMkmgPMAaaPQAAAAAAwCCJ9gAjoNkDAAAAAAAMk2gPMAKeaQ8AAAAAADBMoj3ACHim\nPQAAAAAAwDCJ9gBjoNkDAAAAAAAMkmgPMAKaPQAAAAAAwDCJ9gAj4Jn2AAAAAAAAwyTaA4yAZ9oD\nAAAAAAAMk2gPMAJ22m++1to5SS5N8owkj01yX5JK8sEkb62qPYfSvAAAAAAAwDAt9F4AABxqWmuv\nSXJTkhclOSHJ3ye5J8nZSd6Y5JOttRMPlXkBAAAAAIDhEu0BRmAymb/XVmmtXZDk9UkWk7w5yfFV\ndUZVPTHJOUk+n+T0JO8/FOYFAAAAAACGTbQHGIHJHP5vC12eaVi/tqp+paru3Xeiqm5OctHs/Dmt\ntYsOgXkBAAAAAIABE+0BRmBhMn+vrdBae2aS02ZvL19uTFXdkuS62duf7jkvAAAAAAAwfKI9wBhM\n5vC1NZ4zO96T5C9XGfeR2SrO7TwvAAAAAAAwcKI9wAj0vlX9iG6Pf+bsuLuq9q4y7tbZ8ajWWus4\nLwAAAAAAMHA7ey8AgIM32dJHvM+VkzJ9rvyda4xbev6kJNVpXgAAAAAAYOBEe4AR0Ow3zdGz4541\nxi09f/SKo7Z+XgAAAAAAYOBEe4AxUO03y5Gz4wNrjLt/yddHdZwXAAAAAAAYONEeYAS28Bnv82bf\nTvfD1hh3xDLX9JgXAAAAAAAYONEeYAQ8037T3D07PmKNcY9c8vU3O84LAAAAAAAMnGgPMAJH7LTV\nfpPcnuTpSU5aY9wpS77+Qsd5AQAAAACAgVvovQAAOIR8enY8vbW22i+2nTk7fr2qbus4LwAAAAAA\nMHCiPQD8qw/PjkclOXeVcc9PspjkQ53nBQAAAAAABk60B4CZqrolyaeSTJK8crkxrbXzkzxt9vZd\nPecFAAAAAACGT7QHgP/fZZnudn9ua+2K1trR+0601s5LcvXs/Aeq6oYl505urT3UWnuwtfZrmzUv\nAAAAAAAwbqI9ACxRVTcmeVmSvUlemuRLrbXPtdb+IclHkpyQ5ONJLt7v0kmm/1zd99qseQEAAAAA\ngBET7QFgP1X1jiRPTXJVkn9MckqSw5LckOQlSZ5dVfcsc+niFs0LAAAAAACM1M7eCwCAQ1FVfSbJ\nJRsYf3uSHZs9LwAAAAAAMG522gMAAAAAAABAJ6I9AAAAAAAAAHQi2gMAAAAAAABAJ6I9AAAAAAAA\nAHQi2gMAAAAAAABAJ6I9AAAAAAAAAHQi2gMAAAAAAABAJ6I9AAAAAAAAAHQi2gMAAAAAAABAJ6I9\nAAAAAAAAAHQi2gMAAAAAAABAJ6I9AAAAAAAAAHQi2gMAAAAAAABAJ6I9AAAAAAAAAHQi2gMAAAAA\nAABAJ6I9AAAAAAAAAHQi2gMAAAAAAABAJ6I9AAAAAAAAAHQi2gMAAAAAAABAJ6I9AAAAAAAAAHQi\n2gMAAAAAAABAJ6I9AAAAAAAAAHQi2gMAAAAAAABAJ6I9AAAAAAAAAHQi2gMAAAAAAABAJ6I9AAAA\nAAAAAHQi2gMAAAAAAABAJ6I9AAAAAAAAAHQi2gMAAAAAAABAJ6I9AAAAAAAAAHQi2gMAAAAAAABA\nJ6I9AAAAAAAAAHQi2gMAAAAAAABAJ6I9AAAAAAAAAHQi2gMAAAAAAABAJ6I9AAAAAAAAAHQi2gMA\nAAAAAABAJ6I9AAAAAAAAAHQi2gMAAAAAAABAJ6I9AAAAAAAAAHQi2gMAAAAAAABAJ6I9AAAAAAAA\nAHQi2gMAAAAAAABAJ6I9AAAAAAAAAHQi2gMAAAAAAABAJ6I9AAAAAAAAAHQi2gMAAAAAAABAJ6I9\nAAAAAAAAAHQi2gMAAAAAAABAJ6I9AAAAAAAAAHQi2gMAAAAAAABAJ6I9AAAAAAAAAHQi2gMAAAAA\nAABAJ6I9AAAAAAAAAHQi2gMAAAAAAABAJ6I9AAAAAAAAAHQi2gMAAAAAAABAJ6I9AAAAAAAAAHQy\nWVxc7L0GAAAAAAAAAJhLdtoDAAAAAAAAQCeiPQAAAAAAAAB0ItoDAAAAAAAAQCeiPQAAAAAAAAB0\nItoDAAAAAAAAQCc7ey8AgE2x2HsBAFtosv837nto/v6+d+zZL++9hD4WdvRewfbb8W29V9DH4t7e\nK9h+kzn9Pfp5/GudJDsP670Ctss8/tmefMu/rs2Hw4/qvYI+Hry/9wq2396He6+A7XLYEb1X0MdR\nx/Rewfb72j/1XkEfDz3QewV9zOP/t5DkXz7xG3P6L2nDNIf/FQEAAAAAAAAAhwbRHgAAAAAAAAA6\nEe0BAAAAAAAAoBPRHgAAAAAAAAA6Ee0BAAAAAAAAoBPRHgAAAAAAAAA6Ee0BAAAAAAAAoBPRHgAA\nAAAAAAA6Ee0BAAAAAAAAoBPRHgAAAAAAAAA6Ee0BAAAAAAAAoBPRHgAAAAAAAAA6Ee0BAAAAAAAA\noBPRHgAAAAAAAAA6Ee0BAAAAAAAAoBPRHgAAAAAAAAA6Ee0BAAAAAAAAoJOdvRcAAAAAAAAAwLi0\n1p6V5Poki0lOrao7Oi/pkCXaAwAAAAAAADBYrbUTk1ya5HlJnpTkiCRfTfLJJH+Y5L1Vtdhvhatz\ne3wAAAAAAAAABqm1dkGSzyV5ZZIzk9yf5I4kxyV5fpL3JLmutXZkt0WuQbQHAAAAAAAAYHBaa09J\n8sdJHpnkb5J8T1WdUFXfmeTYJP9tNvTcJJf3WeXaRHsAAAAAAAAAhuh1SQ7L9Fb4/66qPrXvRFXd\nW1WvSPI/k0yS/FRr7fA+y1ydZ9oDAAAAAAAAsKbW2kKSH0/y4kxvRf/oTIP5TUneVlXXr3LtsUle\nk+SCJCcleSjJrUmurKrfXWb8bUlOTvLqJNckeVuSZyU5tqr2bU4/LcmeJNdW1V0rfPT7Mr1N/pFJ\nWpL/ve4feJvYaQ8AAAAAAADAqlprj0hyXZJ3Jzk/yWKSLyR5VJILk3y0tfaGFS4/KdPb11+W6a73\nL2Ya0c9O8juttd9c5prF2WuS5E+T/ECSL80+M0lSVU+tqkcm+dlVlv7gkq8PyZ32oj0AAAAAAAAA\na3lnps+G/3KS76uqE6tqV5Ljkrx1NuZVrbUfWuba30uyO8kpVXV6VT05yROSfGJ2/tLW2hNW+Nzn\nJXlMkidX1a6qOm3/AVW1uMq6v2d2fDjJ360yrhvRHgAAAAAAAIAVtdbOTPKiTHe+v7yqPrbvXFXd\nV1W/mGTf8+RftcwUhye5oKruXHLdXUl+dfZ2Ick5y1w3yfSW+D9XVV9Y5vxa6z4xycWzdb+3qr6x\n0Tm2g2faAwAAAAAAALCaF86O92Z6q/rlXJbkO5Ms92z5t1fV/ct8/7NLvn78CvN+qaquX9cql2it\nHZHk/UmOSfK1JK/d6BzbRbQHAAAAAAAAYDVnz463VtXDyw2oqhuT3LjC9Tev8P2vLPn62GXOLyb5\n3LpWuERr7dFJrk3y9CT3J3lBVf3DRufZLqI9wJzbvXt37r777t7LAFjRWWed1XsJAAAAADDvnpBp\nQF9uF/16fHO5b1bV3tbavreTFa7d0Ge21k5L8meZ7vrfk+RFS2/nfygS7QHm2O7du3P66af3XgbA\nqhYXF3svAQAAAADm3RGz40MdPvve9Q5srT0ryZ8kOS7JPyf5D1X111u1sM0i2gPMsX077K+55prs\n2rWr82oAAAAAAIBD1J7Z8du7rmIVrbUfS3Jlpg38lkyD/Rf7rmp9RHsAsmvXLrefBgAAAAAAVvLF\nJE9JcnzvhSyntfaCJFclWUjyP5L8WFXtWf2qQ4doDwAAAAAAAMBqPpnk+5Oc1lo7pqq+sf+A1tpz\nk7wwyWKSX9iuhbXW/m2SP8g02L+rqn5uuz57syz0XgAAAAAAAAAAh7Q/nh13JHnpCmNekeRnknxv\nVT20HYtqrT02yTVJvi3J+4YY7BPRHgAAAAAAAIBVVNXfJvmjJJMkr2utvbi1NkmS1tqRrbU3JTk/\n0132b9zGpb0hyWOS3JHpLwwMktvjAwAAAAAAALCWn01yQpLnJLk6yW+11r6c5DuSHJFpsH9TVb1v\nEz9zstKJ1tqjk/zE7O0jk9zUWltrvrdX1Ts2aW2bRrQHAAAAAAAAYFVVtSfJ+a21n8w0ln9Xkicl\n+WqSP0/y21X1sf0uW5y9DtRq1x+d6e36F5M8evZay+MOYi1bRrQHAAAAAAAAYF2q6j1J3rOOcR/L\nNKqvNW7ZR7pX1alrXHf7euYfAs+0BwAAAAAAAIBORHsAAAAAAAAA6ES0BwAAAAAAAIBORHsAAAAA\nAAAA6ES0BwAAAAAAAIBORHsAAAAAAAAA6ES0BwAAAAAAAIBORHsAAAAAAAAA6ES0BwAAAAAAAIBO\nRHsAAAAAAAAA6ES0BwAAAAAAAIBORHsAAAAAAAAA6ES0BwAAAAAAAIBORHsAAAAAAAAA6ES0BwAA\nAAAAAIBORHsAAAAAAAAA6ES0BwAAAAAAAIBORHsAAAAAAAAA6ES0BwAAAAAAAIBORHsAAAAAAAAA\n6ES0BwAAAAAAAIBORHsAAAAAAAAA6ES0BwAAAAAAAIBOdvZeAAAAbNSxZ7+89xK23dduflvvJXQx\nj3+tc/gjeq+gjwf+pfcKtt+OHb1X0MfOI3uvoI/JpPcKtt/iYu8V9LEwh3+2H3qg9wr6WNzbewV9\nzOufbebDww/3XkEfO76t9wrYLkc+qvcK+rjnK71XAGuy0x4AAAAAAAAAOhHtAQAAAAAAAKAT0R4A\nAAAAAAAAOhHtAQAAAAAAAKAT0R4AAAAAAAAAOhHtAQAAAAAAAKAT0R4AAAAAAAAAOhHtAQAAAAAA\nAKAT0R4AAAAAAAAAOhHtAQAAAAAAAKAT0R4AAAAAAAAAOhHtAQAAAAAAAKAT0R4AAAAAAAAAOhHt\nAQAAAAAAAKAT0R4AAAAAAAAAOhHtAQAAAAAAAKCTnb0XAAAAAAAAAMC4tNaeleT6JItJTq2qOzov\n6ZAl2gMAAAAAAAAweK21s5K8J8kZSRarakfnJa2LaA8AAAAAAADAYLXWdiT5tSSvTTKIUL+UZ9oD\nAAAAAAAAMEittV1J/irJ65J8PdNb8g+KaA8AAAAAAADAUL03yVOTfDjJdyX5X32Xs3Fujw8AAAAA\nAADAmlprC0l+PMmLk5yZ5NFJvprkpiRvq6oVd7m31o5N8pokFyQ5KclDSW5NcmVV/e4y429LcnKS\nVye5JsnbkjwrybFVtXRz+n1Jfrmq3jK77qB+xh7stAcAAAAAAABgVa21RyS5Lsm7k5yfZDHJlP4I\ngQAAIABJREFUF5I8KsmFST7aWnvDCpeflORvklyWZJLki0mOTHJ2kt9prf3mMtcszl6TJH+a5AeS\nfGn2mUtduC/YD5VoDwAAAAAAAMBa3pnk3CRfTvJ9VXViVe1KclySt87GvKq19kPLXPt7SXYnOaWq\nTq+qJyd5QpJPzM5f2lp7wgqf+7wkj0ny5KraVVWnLT1ZVf98UD/VIUC0BwAAAAAAAGBFrbUzk7wo\n053vL6+qj+07V1X3VdUvJvnU7FuvWmaKw5NcUFV3LrnuriS/Onu7kOScZa6bZHpL/J+rqv132I+G\nZ9oDAAAAAAAAsJoXzo73Znqr+uVcluQ7k9y1zLm3V9X9y3z/s0u+fvwK836pqq5f1yoHSrQHAAAA\nAAAAYDVnz463VtXDyw2oqhuT3LjC9Tev8P2vLPn62GXOLyb53LpWOGBujw8AAAAAAADAap6QaUBf\nbhf9enxzuW9W1d4lbycrXHugnzkYdtoDkCTZvXt37r777t7LAPgWZ511Vu8lAAAAAMC8O2J2fKjD\nZ9/b4TO3lWgPQO64445ceOGFvZcBsKzFxcXeSwAAAACAebdndvz2rqsYKdEegNx77/SX1K655prs\n2rWr82oAAAAAAIBDzBeTPCXJ8b0XMkaiPQD/z65du9yGGgAAAAAA2N8nk3x/ktNaa8dU1Tf2H9Ba\ne26SFyZZTPIL27y+QVvovQAAAAAAAAAADml/PDvuSPLSFca8IsnPJPneqnpoW1Y1EnbaAwAAAAAA\nALCiqvrb1tofJfmRJK9rrd2Z5A+qarG1dmSS1yU5P9Nd9m/crnW11g5LcvR+3z5qyfnj9jt3X1Xd\nu+UL2yDRHgAAAAAAAIC1/GySE5I8J8nVSX6rtfblJN+R5IhMg/2bqup9m/iZkzXO/2iSK1e59q79\nvvfuJJcc5Jo2ndvjAwAAAAAAALCqqtpTVecnuTjJR5M8nORJSb6R5INJvq+qXrvfZYuz14Faz/WL\nG3wdcuy0BwAAAAAAAGBdquo9Sd6zjnEfS7JjHeOW3WheVaeu49qrkly11rhDnZ32AAAAAAAAANCJ\naA8AAAAAAAAAnYj2AAAAAAAAANCJaA8AAAAAAAAAnYj2AAAAAAAAANCJaA8AAAAAAAAAnYj2AAAA\nAAAAANCJaA8AAAAAAAAAnYj2AAAAAAAAANCJaA8AAAAAAAAAnYj2AAAAAAAAANCJaA8AAAAAAAAA\nnYj2AAAAAAAAANCJaA8AAAAAAAAAnYj2AAAAAAAAANCJaA8AAAAAAAAAnYj2AAAAAAAAANCJaA8A\nAAAAAAAAnYj2AAAAAAAAANCJaA8AAAAAAAAAnYj2AAAAAAAAANCJaA8AAAAAAAAAnYj2AAAAAAAA\nANDJzt4LAACADVvY0XsF2+7Ys1/eewldfO3mt/VewrY79pmv6L2EPiaT3ivYfvP4MyfJQw/0XkEf\nOw/rvQK2y475+/eU7PmX3ivo4+hH915BH/fP4V/vOfzvjyTz++8q82gyh/s75/XP9eLe3ivo47Aj\ne68A1jSHfycGAAAAAAAAgEODaA8AAAAAAAAAnYj2AAAAAAAAANCJaA8AAAAAAAAAnYj2AAAAAAAA\nANCJaA8AAAAAAAAAnYj2AAAAAAAAANCJaA8AAAAAAAAAnYj2AAAAAAAAANCJaA8AAAAAAAAAnYj2\nAAAAAAAAANCJaA8AAAAAAAAAnYj2AAAAAAAAANCJaA8AAAAAAAAAnYj2AAAAAAAAANCJaA8AAAAA\nAAAAnYj2AAAAAAAAANDJzt4LAAAAAAAAAGBcWmvPSnJ9ksUkp1bVHZ2XdMgS7QEAAAAAAAAYtNba\nGUl+Kcn5SR6f5BtJPv9/2bv7aMvr+j707zPDowgy+JDoEATC8OEkykomwWXzIJFAvLZe4qQN2kLb\n69Vr1RotaQk+xNBKlpJlbY1yfYouDY53qY0xjUmTlRQppPW6YsRYG+HjrKIZNEZAuDAyERw494+9\nJz2Z7PM0wvnOcF6vtfb67f37fb/7995rYAbmfb7fneRDSd7b3XsHxluW7fEBAAAAAAAAOGxV1QuT\n3JjkhUken+SWJJuTPD3Jryb546o6aVzC5SntAQAAAAAAADgsVdUFSd6VSUn/qiSP7+7v7+7HJ7ko\nyZ4k80neNi7l8myPDwAAAAAAAMBhp6o2J3lrkrkkv9jdb1p8vbs/WlUnJPkHSb46IOKqKO0BAAAA\nAAAAWFFVbUpySZKLk5yd5KQkdyb5ZJKru/u6ZeZuSfKaJBcmOTnJviQ3J3lfd79zxvgvJzklk9Xz\nO5NcneTcJFu6e/+O8ucnqSRfT/LmWfft7vcled/aPun6sj0+AAAAAAAAAMuqquOSfCLJ+zMpyxcy\n+e74E5LsSHJtVb1hieknJ/lMkkszWRV/a5Jjk5yT5O1VNatwX5g+5pJ8NMlzktw2ved+PzMd89vd\n/e3v4OMNZaU9AAAAAAAAACt5T5JnJLkjyc929/VJUlXHJPmVJD+X5PKq+uPu/q0D5r4rya4kz+ju\nr0znPT7Jf0zy9CSvqKp/192ztrB/VpLHJTmru2854No50+ON0/e8MMnPJjk1yQNJ/izJzu7+fw/6\nU68DK+0BAAAAAAAAWFJVnZ3keZmsan/5/sI+Sbr7W939yiSfnZ66fMZbHJ3kwv2F/XTe7UlePX25\nKZPy/kBzmWyJ/+IZhX2SzE+Pd1XVf0ryW0n+UZIfSfLjSV6a5L9V1buqam5VH3YAK+0BAAAAAAAA\nWM5F0+O9mWxVP8ulSbYluX3GtXd0930zzn9h0fMnLvG+t3X3dQeerKpHZ/LDAAtJ/nWS70rysiS/\nn+SrmRT6r8tkC/0XZfK997+0xD2GUtoDAAAAAAAAsJz929Df3N0PzBrQ3TckuWGJ+Z9e4vw3Fj3f\nMuP6QpKblph7/PQ4l8kPC/xod//xouv/PcnPVtXHkvx0kn9ZVW/t7juWeL9hlPYA/LXdu3ePjgDw\nt2zfvn10BAAAAADY6LZmUqDPWkW/GvfMOtndD1bV/pdLbV+/1D0XFh3/8IDCfrErMyntj0ny95L8\n+opp15nSHoC/tmPHjtERAP6WhYWFlQcBAAAAAA+nY6bHfQPufe8S5/csev65ZeZ/dvoej0ry1Icq\n1ENJaQ/A37Bz587Mz8+PjgEAAAAAABw69k6PJw5NsUh331tV/1+Sx6wwbmE67lFJHr0u4dZIaQ/A\n3zA/P28ragAAAAAAYLFbkzwlyeNHBznA55P8WJLTVhi3v9i/6+GNc3A2jQ4AAAAAAAAAwCHtxunx\njKqaubK9qi6oql+rqndX1XotHv+9JHNJnlVVM1fRV9X35X+tsP/Tdcq1Jkp7AAAAAAAAAJbzkelx\nc5KXLDHmsiQvTPK07t63LqmSDya5P8kJSX5uiTGvnB73Jvn99Qi1Vkp7AAAAAAAAAJbU3Z9P8qFM\nVrVfUVUXV9VcklTVsVV1VZLzkywkeeM65ro1yb+b5vo3VfXP96/yr6qjq+qKJP/XNNebuvvu9cq2\nFr7THgAAAAAAAICVvCjJE5I8M8kHkrylqu5I8uQkx2RSjF/V3R9+CO85t4oxr02yNcklSd6W5Mqq\n+to016OmuXYm+eWHMNdDSmkPAAAAAAAAwLK6e2+S86vqnyT5x0memuT0JHdm8t3yb+3u6w+YtjB9\nHKwV53f3QpJ/WlW/kckPFvxwku9NcleSa5O8u7t/9zvI8LBT2gMAAAAAAACwKt19TZJrVjHu+iSb\nVzFu5le6d/dpa8z18SQfX8ucQ4XvtAcAAAAAAACAQZT2AAAAAAAAADCI0h4AAAAAAAAABlHaAwAA\nAAAAAMAgSnsAAAAAAAAAGERpDwAAAAAAAACDKO0BAAAAAAAAYBClPQAAAAAAAAAMorQHAAAAAAAA\ngEGU9gAAAAAAAAAwiNIeAAAAAAAAAAZR2gMAAAAAAADAIEp7AAAAAAAAABhEaQ8AAAAAAAAAgyjt\nAciXvvSl0REAAAAAAAA2JKU9wAZ2/PHHJ0le97rXDU4CAAAAAACwMSntATawbdu25Ytf/GJ27tw5\nOgoAAAAAAMCGdMToAACMtW3btuzZs2d0DAAAAAAAgA3JSnsAAAAAAAAAGERpDwAAAAAAAACDKO0B\nAAAAAAAAYBClPQAAAAAAAAAMorQHAAAAAAAAgEGOGB0AAADWbPORoxOsv6OPG51giC0/etnoCOvu\nrv/2ptERhtjyE68bHWH9PfjA6ARjzI0OMMjCg6MTrL+jHzU6wRjfvm90gvV33JbRCcZYWBidYIwj\njxmdYP1t3jw6wRgb8c+uBzfov9ebNuA/43Mb9D9K990/OsEYG/HvkTjsWGkPAAAAAAAAAIMo7QEA\nAAAAAABgEKU9AAAAAAAAAAyitAcAAAAAAACAQZT2AAAAAAAAADCI0h4AAAAAAAAABlHaAwAAAAAA\nAMAgSnsAAAAAAAAAGERpDwAAAAAAAACDKO0BAAAAAAAAYBClPQAAAAAAAAAMorQHAAAAAAAAgEGU\n9gAAAAAAAAAwiNIeAAAAAAAAAAZR2gMAAAAAAADAIEp7AAAAAAAAABhEaQ8AAAAAAAAAgxwxOgAA\nAAAAAAAAjyxVdW6S65IsJDmtu3cPjnTIUtoDAAAAAAAAcNiqqm1J/nmSn0zy5CRHJ7kzyZ8m+UiS\nD3T3vnEJl2d7fAAAAAAAAAAOS1X1wiSfT/KKJPNJbkvyxUyK+59K8t4kn6qqxw4LuQKlPQAAAAAA\nAACHnar64STvSnJkkk8k+d7uPqO7n9rdJyV5fpK/SvKDSX59XNLlKe0BAAAAAAAAOBy9PJPOe2+S\ni7r7zxdf7O7/kOTfJplL8uyqOmX9I67Md9oDAAAAAAAAsKKq2pTkkiQXJzk7yUmZfHf8J5Nc3d3X\nLTN3S5LXJLkwyclJ9iW5Ocn7uvudM8Z/OckpSV6VZGeSq5Ocm2RLd+9fnF7T483dfecSt/6TxW+b\nZPdKn3O9WWkPAAAAAAAAwLKq6rhMtqB/f5LzkywkuSXJCUl2JLm2qt6wxPSTk3wmyaWZrHq/Ncmx\nSc5J8vaqevOMOQvTx1ySjyZ5TibfV3/LojF/OT0es0z0Ixc9/9oy44ZR2gMAAAAAAACwkvckeUaS\nO5Kc191P6u75JI9N8rbpmMur6rkz5r4rya4kp3b3md19VpKtST41vf6Kqtq6xH2fleRxSc7q7vnu\nPmPRtY9Pj/NVdfYy85Pkfyb5wvIfcQylPQAAAAAAAABLmhbiz8tk5fvLu/v6/de6+1vd/cokn52e\nunzGWxyd5MLu/sqiebcnefX05aYkT58xby6TLfFf3N23zLj+/iS/Px33W1X13Ko6oao2V9Wp05X/\nL8rkO+9f2t0PrvpDryPfaQ8AAAAAAADAci6aHu/NZKv6WS5Nsi3J7TOuvaO775txfvHK9ycu8b63\ndfd1sy5094NV9b9P7/3SJL95wJCFJL+R5E3d/ScHzj9UKO0BAAAAAAAAWM450+PN3f3ArAHdfUOS\nG5aY/+klzn9j0fMtM64vJLlphWxbk/xgkidMx/95Jj9csDXJY5L8cJKfqqobrbQH4LCza9eu7Nmz\nZ3QMYIPbvn376AgAAAAAsNFtzaQQn7WKfjXumXVyulJ+/8u5JeYuec+q+r4kf5TkxCQfTnJZd391\n0fXzk7w3yS8n+dEkf2/NydeB0h6AmXbt2pUzzzxzdAyALCwsjI4AAAAAABvdMdPjvgH3vneZa+/O\nZIX+55JccuBK+u7+z1V1YZIbk/xvVfW87v7wwxf14CjtAZhp/wr7nTt3Zn5+fnAaAAAAAABgoL3T\n44lDUyxSVacm+ZFMdgC4Zqmt77v7c1X1pSSnJXl+JivyDylKewCWNT8/b2tqAAAAAADY2G5N8pQk\njx8dZJEnLnr+FyuMvSPJ6QfMOWQo7QEAAAAAAABYzo1Jnp3kjKp6THfffeCAqrogyUWZrHx/2Tpk\numvR81NXGHtKJrnufNjSfAc2jQ4AAAAAAAAAwCHtI9Pj5iQvWWLMZUlemORp3b1vHTJ1kq8nmUvy\nD5YaVFVPT/Ld05fXr0OuNVPaAwAAAAAAALCk7v58kg9lUpBfUVUXV9VcklTVsVV1VZLzM1nN/sZ1\nyrT4Xj9UVf9PVW1dPKaqzpvmTpLbk7x7PbKtle3xAQAAAAAAAFjJi5I8Ickzk3wgyVuq6o4kT05y\nTCaF/VXd/eGH8J5zy13s7rdW1fck+fkkz0vy/Kr68yTfTPKkJFumub6aZEd337Xkmw1kpT0AAAAA\nAAAAy+ruvd19fpIXJLk2yQNJTk9yd5KPJTmvu197wLSF6eNgrTi/uy9Lck6S9yS5KclJSc5Msi+T\n7fAvS/L93f0n30GOh5WV9gAAAAAAAACsSndfk+SaVYy7PsnmVYybudC8u09bQ6Ybk/yz1Y4/1Fhp\nDwAAAAAAAACDKO0BAAAAAAAAYBClPQAAAAAAAAAMorQHAAAAAAAAgEGU9gAAAAAAAAAwiNIeAAAA\nAAAAAAZR2gMAAAAAAADAIEp7AAAAAAAAABhEaQ8AAAAAAAAAgyjtAQAAAAAAAGAQpT0AAAAAAAAA\nDKK0BwAAAAAAAIBBlPYAAAAAAAAAMIjSHgAAAAAAAAAGUdoDAAAAAAAAwCBKewAAAAAAAAAYRGkP\nAAAAAAAAAIMo7QEAAAAAAABgEKU9AAAAAAAAAAyitAcAAAAAAACAQZT2AAAAAAAAADCI0h4AAAAA\nAAAABjlidAAAAFizhQdHJ1h/9//V6ARjzM2NTrDutvzE60ZHGOKu/3Ll6AjrbstPvn50hDE2bbx/\nr5MkDy6MTsB62bQB/7rtwX2jE4xx5NGjE4yx957RCdbfURv013rf/aMTrL9Hnzg6AevliCNHJxhj\no37u+zbo36lwWLHSHgAAAAAAAAAGUdoDAAAAAAAAwCBKewAAAAAAAAAYRGkPAAAAAAAAAIMo7QEA\nAAAAAABgEKU9AAAAAAAAAAyitAcAAAAAAACAQZT2AAAAAAAAADCI0h4AAAAAAAAABlHaAwAAAAAA\nAMAgSnsAAAAAAAAAGERpDwAAAAAAAACDKO0BAAAAAAAAYBClPQAAAAAAAAAMorQHAAAAAAAAgEGU\n9gDk+OOPHx0BAAAAAABgQ1LaA5Bt27blYx/72OgYAAAAAAAAG84RowMAcGg45ZRTRkcAAAAAAAAe\nIarq3CTXJVlIclp37x4c6ZCltAcAAAAAAADgsFVVZyV5WZKfSHJakqOS3JXkvyf5zSTv6+77hgVc\nge3xAQAAAAAAADgsVdWlST6f5OVJvj/JPUn+PMmJSX4yyduTfLaqnjQs5AqU9gAAAAAAAAAcdqrq\nOUnenEnv/VuZbMO/tbvPTHJ8kssz2Z6/kuwcFnQFSnsAAAAAAAAADke/mEkp/7kkF3X37v0Xuvvb\n3f1vk/xakrkk51bV6WNiLs932gMAAAAAAACwoqralOSSJBcnOTvJSUnuTPLJJFd393XLzN2S5DVJ\nLkxycpJ9SW7O5Pvm3zlj/JeTnJLkVZmskr86yblJtnT3pqqaS/KFJLcl+Wh371vi1n+Q5MXT59+b\n5JbVf+L1obQHAAAAAAAAYFlVdVyS303yjExWt389kwL8lCQ7kuyoqqu6+zUzpp+cSfF+ynTOrUlO\nT3JOknOqalt3/8sD5ixMH3NJPprkh5L8zyR3JUl3LyT5P1cRfW7R879Yxfh1Z3t8AAAAAAAAAFby\nnkwK+zuSnNfdT+ru+SSPTfK26ZjLq+q5M+a+K8muJKd295ndfVaSrUk+Nb3+iqrausR9n5XkcUnO\n6u757j5jtYGr6qgkPzd9+aUkN6127npS2gMAAAAAAACwpKo6O8nzMln5/vLuvn7/te7+Vne/Msln\np6cun/EWRye5sLu/smje7UlePX25KcnTZ8yby2RL/Bd396q2ta+quap6XFX9dJJPJPnxJLcneX53\nP7ia91hvtscHAAAAAAAAYDkXTY/3ZrJV/SyXJtmWSUF+oHd0930zzn9h0fMnLvG+t3X3dasJWVV/\nmuTsRaduTnLF9P7fWM17jKC0BwAAAAAAAGA550yPN3f3A7MGdPcNSW5YYv6nlzi/uEjfMuP6Qta2\npf3+sY9P8oQkleTiJHuq6uqlso9me3wAAAAAAAAAlrM1kwJ91ir61bhn1skDtqufW2Luqu/Z3f+w\nu3+gu7cmeWySn88k+79P8hurfZ/1ZqU9AKuya9eu7NmzZ3QMYAPavn376AgAAAAAsNEdMz3uG3Dv\new9mUnffk+RXq2pXkt9JcmFV/dPu/vWHNN1DQGkPwIp27dqVM888c3QMYINaWFgYHQEAAAAANrq9\n0+OJQ1MchO7+T1V1S5LTkvxMEqU9AIef/Svsd+7cmfn5+cFpAAAAAACAdXZrkqdk8l3xh4SqOinJ\n9iTfneR3u/uuZYZ/PcnpSb5rPbKtldIegFWbn5+3TTUAAAAAAGw8NyZ5dpIzquox3X33gQOq6oIk\nFyVZSPKydcj01CR/ML3fC5Jcs8zY75mOu20dcq3ZptEBAAAAAAAAADikfWR63JzkJUuMuSzJC5M8\nrbv3rUOmTyb5xvT5v6iqmQvWq+pZSU6evvzEOuRaM6U9AAAAAAAAAEvq7s8n+VCSuSRXVNXFVTWX\nJFV1bFVdleT8TFazv3GdMn07yeunmX4gye9U1bb916tqU1W9IMkHp6f+Msl71yPbWtkeHwAAAAAA\nAICVvCjJE5I8M8kHkrylqu5I8uQkx2RS2F/V3R9+CO85t9zF7n5bVT0xyS8kuSBJV9XuJN9KcmqS\no6a5/iLJhd295yHM9pCx0h4AAAAAAACAZXX33u4+P5Pvj782yQNJTk9yd5KPJTmvu197wLSF6eNg\nrTi/u1+T5IeSvDvJTUlOTHJaknuS/FGSVyd5Snff+B3keFhZaQ8AAAAAAADAqnT3NUmuWcW465Ns\nXsW4mQvNu/u0NWT6XJKXrnb8ocZKewAAAAAAAAAYRGkPAAAAAAAAAIMo7QEAAAAAAABgEKU9AAAA\nAAAAAAyitAcAAAAAAACAQZT2AAAAAAAAADCI0h4AAAAAAAAABlHaAwAAAAAAAMAgSnsAAAAAAAAA\nGERpDwAAAAAAAACDKO0BAAAAAAAAYBClPQAAAAAAAAAMorQHAAAAAAAAgEGU9gAAAAAAAAAwiNIe\nAAAAAAAAAAZR2gMAAAAAAADAIEp7AAAAAAAAABhEaQ8AAAAAAAAAgyjtAQAAAAAAAGAQpT0AAAAA\nAAAADKK0BwAAAAAAAIBBlPYAAAAAAAAAMIjSHgAAAAAAAAAGOWJ0AAAAWLO5Dfizp5s3j04wxtzc\n6ATr78EHRicYYstPvn50hHV317W/NDrCEFsuuHJ0hDE26u/jG9GmDfhn11HHjU4wxgb9MzuPOmF0\ngvV331+NTjDGkUeNTrD+9t49OsEQR52wZXSEdXf/t+8fHWGMjfj/2ElyzAb9bxUOKxvwbzsBAAAA\nAAAA4NCgtAcAAAAAAACAQZT2AAAAAAAAADCI0h4AAAAAAAAABlHaAwAAAAAAAMAgSnsAAAAAAAAA\nGERpDwAAAAAAAACDKO0BAAAAAAAAYBClPQAAAAAAAAAMorQHAAAAAAAAgEGU9gAAAAAAAAAwiNIe\nAAAAAAAAAAZR2gMAAAAAAADAIEp7AAAAAAAAABhEaQ8AAAAAAAAAgyjtAQAAAAAAAGAQpT0AAAAA\nAAAADHLE6AAAAAAAAAAAPLJU1blJrkuykOS07t49ONIhS2kPAAAAAAAAwGGrquaT/FySZyY5eXp6\nd5LfSfKm7r5jVLbVsD0+AAAAAAAAAIelqnpFks8leUmSbUn+cvqoJJcl+UJV/eC4hCtT2gMAAAAA\nAABw2Kmqv5vk3yfZnOT3MtmGf1t3b8ukwP+vSR6X5Her6vhxSZentAcAAAAAAADgcPT6JHNJdiXZ\n0d237r/Q3V9K8ncz2Sb/u5L8qyEJV8F32gMAAAAAAACwoqralOSSJBcnOTvJSUnuTPLJJFd393XL\nzN2S5DVJLszke+f3Jbk5yfu6+50zxn85ySlJXpVkZ5Krk5ybZEt3b6qqJyTZnmQhya919/0Hvkd3\nf7Oq3pbkTUn+cZIrDuqDP8ystAcAAAAAAABgWVV1XJJPJHl/kvMzKctvSXJCkh1Jrq2qNywx/eQk\nn0lyaSYr429NcmySc5K8varePGPOwvQxl+SjSZ6T5LbpPZPkyYvGfn6Z6NfvH19V37vMuGGU9gAA\nAAAAAACs5D1JnpHkjiTndfeTuns+yWOTvG065vKqeu6Mue/KZAv7U7v7zO4+K8nWJJ+aXn9FVW1d\n4r7PyuR76c/q7vnuPmPGmOOWyX3Houffv8y4YZT2AAAAAAAAACypqs5O8rxMVr6/vLv3r15Pd3+r\nu1+Z5LPTU5fPeIujk1zY3V9ZNO/2JK+evtyU5Okz5s1lsiX+i7v7lgOu7V70/PuWif89i54v9YMB\nQyntAQAAAAAAAFjORdPjvZlsVT/LpUlenOSNM669o7vvm3H+C4ueP3GJ972tu6878GR3fz3J5zIp\n9l9SVUcvMf9Vi54/eokxQx0xOgAAAAAAAAAAh7Rzpsebu/uBWQO6+4YkNywx/9NLnP/GoudbZlxf\nSHLTMrmuTPIbSZ6U5A+q6pLuvjVJquqUJFcl+bEk38xkC/3Ny7zXMEp7ANZs165d2bNnz+gYwAax\nffv20REAAAAAYKPbmkmBfvtBzr9n1snufrCq9r+cW2Lukvfs7t+sqiuT/GIm5fyXquqWJMdMM9+X\n5J8keUcmpf0hWW4o7QFYk127duXMM88cHQPYQBYWFkZHAAAAAICN7pjpcd+Ae9+73MXuvqKq/kuS\nVyb5O0lOTvIXSd6d5C1JdiX54HT4N2a9x2hKewDWZP8K+507d2Z+fn5wGgAAAAAAYB3snR5PHJpi\nCdPvvP9b33ufJFV1RpIjM9kp4M/WM9dqKe0BOCjz8/O2rAYAAAAAgI3h1iRPSfL40UHavC+JAAAg\nAElEQVQOwo9Pj3uT3DQyyFKU9gAAAAAAAAAs58Ykz05yRlU9prvvPnBAVV2Q5KJMVrS/bD1CVdVx\nSeaTPNDdn11i2D+cHj/e3SO291/RptEBAAAAAAAAADikfWR63JzkJUuMuSzJC5M8bR3L8Q8m+eMk\nH6qqv9V9V9VPJTk/kx8keMc6ZVozpT0AAAAAAAAAS+ruzyf5UJK5JFdU1cVVNZckVXVsVV2V/1WO\nv3Edo711ejwjyfur6sT9F6rq708zLyT5YHf/0TrmWhOlPQAAAAAAAAAreVGSTyQ5OskHktxWVTcl\n+UaSX8ikHL+quz/8EN5zbrmL3f2JJL8yfXlJkq9V1c1VdXuS/5DkMUl+J5MdAA5ZSnsAAAAAAAAA\nltXde7v7/CQvSHJtkgeSnJ7k7iQfS3Jed7/2gGkL08fBWnF+d786yY4kv5fka0mePM32+0me390/\n3d3f/g4yPOyOGB0AAAAAAAAAgMNDd1+T5JpVjLs+yeZVjJu50Ly7T1tDpt9O8turHX+osdIeAAAA\nAAAAAAZR2gMAAAAAAADAIEp7AAAAAAAAABhEaQ8AAAAAAAAAgyjtAQAAAAAAAGAQpT0AAAAAAAAA\nDKK0BwAAAAAAAIBBlPYAAAAAAAAAMIjSHgAAAAAAAAAGUdoDAAAAAAAAwCBKewAAAAAAAAAYRGkP\nAAAAAAAAAIMo7QEAAAAAAABgEKU9AAAAAAAAAAyitAcAAAAAAACAQZT2AAAAAAAAADCI0h4AAAAA\nAAAABlHaAwAAAAAAAMAgSnsAAAAAAAAAGERpDwAAAAAAAACDKO0BAAAAAAAAYBClPQAAAAAAAAAM\ncsToAAAcWm666aa/cQQAAAAAAODho7QHIEly/PHHJ0kuueSSwUkAVmHhwdEJ1t8Rx45OMMa++0cn\nWH9zowMMsmnjffAtF1w5OsIQd/3h60ZHGGJD/npv2jw6wRibjhydYP09+MDoBKynB749OsH627xR\nfz/bgJ/7qI35/1333/vN0RHW39Eb89c6e+8ZnWCMY44bnQBWpLQHIEmybdu2fPGLX8yePXuSTFba\nK/ABAAAAAAAeXkp7AP7atm3bRkcAAAAAAADYUDaNDgAAAAAAAAAAG5XSHgAAAAAAAAAGUdoDAAAA\nAAAAwCBKewAAAAAAAAAYRGkPAAAAAAAAAIMo7QEAAAAAAABgEKU9AAAAAAAAAAyitAcAAAAAAACA\nQZT2AAAAAAAAADCI0h4AAAAAAAAABlHaAwAAAAAAAMAgSnsAAAAAAAAAGERpDwAAAAAAAACDKO0B\nAAAAAAAAYJAjRgcAAAAAAAAA4JGlqs5Ncl2ShSSndffuwZEOWUp7AAAAAAAAAA4rVfXkJF9a47R/\n3d2vfzjyfCeU9gAAAAAAAAAcbu5P8qerHPvUTL46/v6HL87BU9oDAAAAAAAAcFjp7q8l2b7SuKp6\nTpLfTnJnkvc83LkOxqbRAQAAAAAAAADgoVZVj07yjiQLSS7v7jsGR5rJSnsAAAAAAAAAVlRVm5Jc\nkuTiJGcnOSmTFeyfTHJ1d1+3zNwtSV6T5MIkJyfZl+TmJO/r7nfOGP/lJKckeVWSnUmuTnJuki3d\nvdrF6b+SZGuS/9rd713lnHVnpT0AAAAAAAAAy6qq45J8Isn7k5yfyer1W5KckGRHkmur6g1LTD85\nyWeSXJpkLsmtSY5Nck6St1fVm2fMWZg+5pJ8NMlzktw2vedq8v5Ikpck+XaSf7aaOaMo7QEAAAAA\nAABYyXuSPCPJHUnO6+4ndfd8kscmedt0zOVV9dwZc9+VZFeSU7v7zO4+K5MV8J+aXn9FVW1d4r7P\nSvK4JGd193x3n7HKvG+dHv/v7r5plXOGUNoDAAAAAAAAsKSqOjvJ8zJZ+f7y7r5+/7Xu/lZ3vzLJ\nZ6enLp/xFkcnubC7v7Jo3u1JXj19uSnJ02fMm8tkS/wXd/eqVthP8/6jJNuT3JPkl1c7bxTfaQ8A\nAAAAAADAci6aHu/NZKv6WS5Nsi3J7TOuvaO775tx/guLnj9xife9rbuvW1XKJFU1l+SKTH7A4C3d\nfedq546itAcAAAAAAABgOedMjzd39wOzBnT3DUluWGL+p5c4/41Fz7fMuL6QZK1b2/9MJj88sDfJ\n1WucO4Tt8QEAAAAAAABYztZMCvRZq+hX455ZJ7v7wUUv55aYu9Z7vjKTrB/q7m+sNPhQYKU9AAdl\n9+7doyMAG8T27dtHRwAAAACAje6Y6XHfgHvfu9qBVXVGkh/LpLT/wMOW6CGmtAfgoOzYsWN0BGCD\nWFhYGB0BAAAAADa6vdPjiUNTrOzi6fH27r5+aJI1UNoDcNB27tyZ+fn50TEAAAAAAICH161JnpLk\n8aODrOA5mayy/8+jg6yF0h6AgzY/P2/bagAAAAAAeOS7Mcmzk5xRVY/p7rsPHFBVFyS5KJPS/GXr\nnC9V9dgk+0uLw2aVfZJsGh0AAAAAAAAAgEPaR6bHzUlessSYy5K8MMnTunvfuqT6m34gydz0+Z8N\nuP9BU9oDAAAAAAAAsKTu/nySD2VSil9RVRdX1VySVNWxVXVVkvMzWWX/xkExn7roeQ/KcFBsjw8A\nAAAAAADASl6U5AlJnpnkA0neUlV3JHlykmMyKeyv6u4PP4T3nFt5yF/77kXP73wIMzzsrLQHAAAA\nAAAAYFndvbe7z0/ygiTXJnkgyelJ7k7ysSTndfdrD5i2MH0crLXMf8x07N7u/k7uue6stAcAAAAA\nAABgVbr7miTXrGLc9Uk2r2LczIXm3X3aGnO9NMlL1zLnUGGlPQAAAAAAAAAMorQHAAAAAAAAgEGU\n9gAAAAAAAAAwiNIeAAAAAAAAAAZR2gMAAAAAAADAIEp7AAAAAAAAABhEaQ8AAAAAAAAAgyjtAQAA\nAAAAAGAQpT0AAAAAAAAADKK0BwAAAAAAAIBBlPYAAAAAAAAAMIjSHgAAAAAAAAAGUdoDAAAAAAAA\nwCBKewDWZPfu3aMjAAAAAAAAPGIo7QFYtd27d2fHjh2jYwAAAAAAADxiKO0BWLV77713dAQAAAAA\nAIBHFKU9AAAAAAAAAAyitAcAAAAAAACAQZT2AAAAAAAAADCI0h4AAAAAAAAABlHaAwAAAAAAAMAg\nSnsAAAAAAAAAGERpDwAAAAAAAACDHDE6AAAArNkRR41OsP7m5kYnGGMj/lovPDg6wRgPLoxOsP42\nbx6dYIgtF1w5OsIQd/3h60ZHWHdbLvzV0RHG+OZdoxOwXjZtzN/HN6SjHzU6wRj33j06wfo7dmP+\nf9em404YHWHdPXjv6ASDbD5ydAJgCVbaAwAAAAAAAMAgSnsAAAAAAAAAGERpDwAAAAAAAACDKO0B\nAAAAAAAAYBClPQAAAAAAAAAMorQHAAAAAAAAgEGU9gAAAAAAAAAwiNIeAAAAAAAAAAZR2gMAAAAA\nAADAIEp7AAAAAAAAABhEaQ8AAAAAAAAAgyjtAQAAAAAAAGAQpT0AAAAAAAAADKK0BwAAAAAAAIBB\nlPYAAAAAAAAAMIjSHgAAAAAAAAAGUdoDAAAAAAAAwCBHjA4AAAAAAAAAwCNLVZ2b5LokC0lO6+7d\ngyMdspT2AAAAAAAAADyiVNUvJLlq+vL/6O5rRuZZjtIeAAAAAAAAgEeMqppP8m8yWeV/yPOd9gAA\nAAAAAAA8IlTVpiS/nuSoJA8MjrMqSnsAAAAAAAAAHileleSHk1yb5KuDs6yK7fEBAAAAAAAAWNF0\nFfslSS5OcnaSk5LcmeSTSa7u7uuWmbslyWuSXJjk5CT7ktyc5H3d/c4Z47+c5JRMSvidSa5Ocm6S\nLd09c3F6VT0lyS8luT/Jzyf5+EF8zHVnpT0AAAAAAAAAy6qq45J8Isn7k5yfyffF35LkhCQ7klxb\nVW9YYvrJST6T5NIkc0luTXJsknOSvL2q3jxjzsL0MZfko0mek+S26T1n5ducybb4Rya5srv/x5o/\n5CBKewAA4P9n7+6j7Trr+8B/ryTbko1sC9sELFfYxvLjS8At6phhpqt2cc2wGMCx0sZAcdMGiMMw\nBC8zQ81LiDrTBkhrShg8vCQkfkGwgBXqdApNmNYYm5YxiwJr6gnyz0qA2p4m2ObFCAnZlnXnj3OU\nXO6cc1+ke8/jK30+a521z9n7efb+Hl7kl+959gYAAACAhXw0ycVJHk5yaVWdWVXTSU5L8oHhmOta\na1eMmPuRJLuTnF1V51fVBUk2J7lrePxNrbXNY6774iSnJ7mgqqar6rwx496R5HkZ/Djg3Uv8bl0p\n7QEAAAAAAAAYq7V2YZJXZLDy/Y1VdcehY1W1v6quSfKN4a7rRpzihCSXV9UDs+Y9lORtw49rkrxg\nxLypDG6Jf3VVjVxhP8z3VzMo7R9N8g+q6uBiv9uTgWfaAwAAAAAAADCfK4fbvRncqn6Ua5NsTfLQ\niGMfqqpHR+z/5qz3zxhz3ger6vZxwVpr6zK4Lf66JG+vql3jxj5ZKe0BAAAAAAAAmM9Fw+09VfXE\nqAFVdWeSO8fM/+qY/d+b9X7TiOMzSRYq4d+Z5MIkX0nyzxYY+6SktAdgWe3evTt79uzpHQM4imzb\ntq13BAAAAAA41m3OoEAftYp+MX40amdVHWytHfo4NWbu2Gu21p6XwS3292dwW/yZw8zXldIegGWz\ne/funH/++b1jAEeZmZlV+ffZAAAAAHA0WT/cHuhw7b2jdrbWjsvgtvhrk7ytqu6daKplpLQHYNkc\nWmG/c+fOTE9Pd04DAAAAAAAsk33D7aldU/y0f5zkOUm+WFXv7ZzliCjtAVh209PTbmcNAAAAAABH\nj/szKMjP6B1kllcNt3+rtXZwgbE3tdZuGr4/u6ruW7lYS6e0BwAAAAAAAGA+X0/ykiTntdZOqapH\n5g5orb0oyZVJZpK8YQKZfpTkhwuMOTnJVJKfJHksg2wLFfwTp7QHAAAAAAAAYD6fTvKODJ4f//ok\nvzlizFuSXJbkP1XVgdbaigaqqr+20JjW2reTbEnyhqq6ZUUDHYE1vQMAAAAAAAAA8ORVVXcn+WQG\nq9Z3tNZe3VqbSpLW2obW2nsyKOxnkry7X9LVyUp7AAAAAAAAABbyuiRPS/LCJB9L8luttYeTPDPJ\n+gwK+/dU1aeW8ZpTy3iuJy0r7QEAAAAAAACYV1Xtq6rLkvxSktuSPJHk3CSPJLk1yaVV9Y4502aG\nr8N1pPMPneNJzUp7AAAAAAAAABZl+Gz4BZ8PX1V3JFm7iHEjF5pX1TlLT7f855gEK+0BAAAAAAAA\noBOlPQAAAAAAAAB0orQHAAAAAAAAgE6U9gAAAAAAAADQidIeAAAAAAAAADpR2gMAAAAAAABAJ0p7\nAAAAAAAAAOhEaQ8AAAAAAAAAnSjtAQAAAAAAAKATpT0AAAAAAAAAdKK0BwAAAAAAAIBOlPYAAAAA\nAAAA0InSHgAAAAAAAAA6UdoDAAAAAAAAQCdKewAAAAAAAADoRGkPAAAAAAAAAJ0o7QEAAAAAAACg\nE6U9AAAAAAAAAHSitAcAAAAAAACATpT2AAAAAAAAANCJ0h4AAAAAAAAAOlnXOwAAq9d99933U593\n7drVKQkAAAAAAMDqpLQH4LBt3769dwQAAAAAAIBVTWkPwBHZuXNnpqenkwxW2l911VWdEwEcpWZm\neidgUk44sXcCWFlr1vZO0MWmy9/fO8LE/eD/uKZ3hC42vfx9vSNM3oHHeifo47j1vRP08dhPeieY\nvP0/7p2gjw1P6Z1g8o7F/30nOe7443pHmLhHp47Rp0evP6l3gj6O0X8GYXVR2gNwRKanp7Nt27be\nMQAAAAAAAFalY/SnRAAAAAAAAADQn9IeAAAAAAAAADpR2gMAAAAAAABAJ0p7AAAAAAAAAOhEaQ8A\nAAAAAAAAnSjtAQAAAAAAAKATpT0AAAAAAAAAdKK0BwAAAAAAAIBOlPYAAAAAAAAA0InSHgAAAAAA\nAAA6UdoDAAAAAAAAQCdKewCW7Prrr+8dAQAAAAAA4KigtAdgyZ7+9Kf3jgAAAAAAAHBUUNoDAAAA\nAAAAQCdKewAAAAAAAADoRGkPAAAAAAAAAJ2s6x0AAAAAAAAAgKNLa+2SJLcnmUlyTlXd1znSk5bS\nHgAAAAAAAIBVr7W2LcktSZ6dZKaq1i5iztokv5jklUn+WpJNSfYl2Z3kj5L871X15ysWOm6PDwAA\nAAAAAMAq1lpb21rbkeSuJNNLmHdGkv8rye8muSzJU5J8K8lUkm1J3pHkm621v7HsoWdR2gMAAAAA\nAACwKrXWppN8JcmOJD/M4Jb8i5k3leQPkvxXSX6S5OokG6vqgqo6JcnPJXkoySlJfr+1dtIKxE+i\ntAcAAAAAAABg9fp4kucl+XyS5yb50iLnvSTJf5NkJsmvVtXvVtXBQwer6rNJfjWDVfdPS/J3ljP0\nbJ5pDwAAAAAAAMCCWmtrklyV5NVJLkzy1CTfT/LlJDdU1dhV7q21TUnenuTyJGclOZDkniQ3VtWH\nR4z/TpItSd6aZGeSG5JckmRTVc1enL4/yZur6v3DeYv9On91mP34DIr/UT4zzLk2g+fd37LYky+F\nlfYAAAAAAAAAzGt4e/gvJLkpg+e/z2Tw/PeTk2xPcltr7V1jpp+V5GtJrs1g5fr9STYkuSjJB1tr\n7x0xZ2b4msqgPH9ZkgeH15xt+6HCfimq6t1VdXqSU6rq0TFjnkhyaPX9CUu9xmIp7QEAAAAAAABY\nyEeTXJzk4SSXVtWZVTWd5LQkHxiOua61dsWIuR9JsjvJ2VV1flVdkGRzkruGx9/UWts85rovTnJ6\nkguqarqqzpt9sKq+eyRfqqpmxh1rrV2YwUr8JPnmkVxnPkp7AAAAAAAAAMYaltevyGDl+xur6o5D\nx6pqf1Vdk+Qbw13XjTjFCUkur6oHZs17KMnbhh/XJHnBiHlTGdwS/+qqmrvCfhJ+bbj9YZJPrNRF\nPNMeAAAAAAAAgPlcOdzuzeBW9aNcm2RrkodGHPvQmFvQz169/owx532wqm5fVMpl1Fq7JsnfzeCH\nCu+sqh+s1LWU9gAAAAAAAADM56Lh9p7hc97/f6rqziR3jpn/1TH7vzfr/aYRx2eS7FpUwmXUWntr\nkt8YXv93quqDK3k9pT0Ay27Xron/9RM4im3btq13BAAAAAA41m3OoMAetYp+MX40amdVHWytHfo4\nNWbu4V5zyVpra5N8MMkvZ/B9b0ryhpW+rtIegGWzcePGJMlVV13VOQlwNJmZmekdAQAAAACOdeuH\n2wMdrr13EhdprZ2S5PeT/O0MCvt/WlU7JnFtpT0A81rKqvmtW7fm3nvvzZ49e1YwEQAAAAAAMGH7\nhttTu6ZYIa21s5L82yQtyU+SvKaqPjWp6yvtARjpcFfNb926dSXiAAAAAAAA/dyf5DlJzugdZLm1\n1s5I8u+SnJ/kz5L8XFX9x0lmUNoDMNLsVfO7du1yy3sAAAAAADh2fT3JS5Kc11o7paoemTugtfai\nJFdmcGv5FX8O/HJorW3MXxb2307yt6rq/knnWDPpCwKwemzdujXbtm3L9PR07ygAAAAAAEA/nx5u\n1yZ5/Zgxb0ny2iTPr6oDE0l15D6Y5LkZrLD/2z0K+8RKewAAAAAAAADmUVV3t9Y+meSVSXa01h5I\n8omqmmmtbUiyI8llGayyf/ekcrXWjk+ycc7uE2cdP23Osf1VtXd47G8meXUGmX+5qr6zglHnpbQH\nAAAAAAAAYCGvS/K0JC9M8rEkv9VaezjJM5Osz6D8fk9VfWoZrzm1wPFXJblxnrkPzdl3U5LXDN+/\nedb+32it/cYC1/ovVfXSBcYcFqU9AAAAAAAAAPOqqn1JLmut/WKSv5/BbeXPTfL9JH+Y5H+rqjvm\nTJsZvg7XYuYv5fyzx5466/OFi5h78hKusyRKewAAAAAAAAAWpapuSXLLIsbdkWTtIsatGbP/nEXM\nvTnJzQuNGzP3hYczbyWM/A8AAAAAAAAAAFh5SnsAAAAAAAAA6ERpDwAAAAAAAACdKO0BAAAAAAAA\noBOlPQAAAAAAAAB0orQHAAAAAAAAgE6U9gAAAAAAAADQidIeAAAAAAAAADpR2gMAAAAAAABAJ0p7\nAAAAAAAAAOhEaQ8AAAAAAAAAnSjtAQAAAAAAAKATpT0AAAAAAAAAdKK0BwAAAAAAAIBOlPYAAAAA\nAAAA0InSHgAAAAAAAAA6UdoDAAAAAAAAQCdKewAAAAAAAADoRGkPAAAAAAAAAJ0o7QEAAAAAAACg\nE6U9AAAAAAAAAHSitAcAAAAAAACATtb1DgAAAEs2dQz+9nTN2t4J+lh7DH7vxx/tnaCPNcfgP56u\nmeqdoI81x/VO0MePf9A7wcRtevn7ekfo4gf/+treESbuWP3vOlPH6J/jx+L33nBy7wR97H2kd4LJ\nW39i7wRdPP7Y470jTN6x+GdZkuz7Ue8EfWzY2DsBLOgY/LedAAAAAAAAAPDkoLQHAAAAAAAAgE6U\n9gAAAAAAAADQidIeAAAAAAAAADpR2gMAAAAAAABAJ0p7AAAAAAAAAOhEaQ8AAAAAAAAAnSjtAQAA\nAAAAAKATpT0AAAAAAAAAdKK0BwAAAAAAAIBOlPYAAAAAAAAA0InSHgAAAAAAAAA6UdoDAAAAAAAA\nQCdKewAAAAAAAADoRGkPAAAAAAAAAJ0o7QEAAAAAAACgE6U9AAAAAAAAAHSyrncAAAAAAAAAAI4u\nrbVLktyeZCbJOVV1X+dIT1pKewAAAAAAAABWvdbatiS3JHl2kpmqWjvP2C8muXgJp/9OVZ17ZAlH\nU9oDAAAAAAAAsGq11tYm+bUk70gytqif494kJy9i3JlJzkjy2OGlW5jSHgAAAAAAAIBVqbU2neRj\nSbYleTjJf0py6ULzqurqRZz7pCT/z/DjPzuCmPNas1InBgAAAAAAAIAV9vEkz0vy+STPTfKlZTz3\nu5I8M8m/r6rfW8bz/hQr7QEAAAAAAABYUGttTZKrkrw6yYVJnprk+0m+nOSGqrp9nrmbkrw9yeVJ\nzkpyIMk9SW6sqg+PGP+dJFuSvDXJziQ3JLkkyaaqmr04fX+SN1fV+4fzjug7zrr+f53kf8zgtvi/\nsiwnHcNKewAAAAAAAADmNbxV/BeS3JTksiQzSb6VwXPhtye5rbX2rjHTz0rytSTXJplKcn+SDUku\nSvLB1tp7R8yZGb6mknwmycuSPDi85mzbDxX2y6W1ti7J7wyvfX1V3bOc559LaQ8AAAAAAADAQj6a\n5OIMnht/aVWdWVXTSU5L8oHhmOtaa1eMmPuRJLuTnF1V51fVBUk2J7lrePxNrbXNY6774iSnJ7mg\nqqar6rzZB6vqu0f0rUZ7Q5LnJPmzJL+xAuf/KUp7AAAAAAAAAMZqrV2Y5BUZrHx/Y1XdcehYVe2v\nqmuSfGO467oRpzghyeVV9cCseQ8ledvw45okLxgxbyqDW+JfXVVzV9iviNbaxiS/lsF3/fWq+slK\nX9Mz7QEAAAAAAACYz5XD7d4MblU/yrVJtiZ5aMSxD1XVoyP2f3PW+2eMOe+DVXX7olIuj2syWNn/\nrSQ3TuKCSnsAAAAAAAAA5nPRcHtPVT0xakBV3ZnkzjHzvzpm//dmvd804vhMkl2LSrgMWmvrk/zq\n8LrXV9XMJK7r9vgAAAAAAAAAzGdzBkX2qFX0i/GjUTur6uCsj1Nj5h7uNQ/H30tyRpIfJ7l5Uhe1\n0h6AZbV79+7s2bOndwzgKLJt27beEQAAAADgWLd+uD3Q4dp7J3it12Tw44TPTOJZ9oco7QFYNrt3\n787555/fOwZwlJmZmcgdqAAAAACA8fYNt6d2TbGCWmtnJ/lvMyjtf3+S11baA7BsDq2w37lzZ6an\npzunAQAAAAAAlsn9SZ6Twa3jj1YvHW4fT/LFSV5YaQ/AspuennY7awAAAAAAOHp8PclLkpzXWjul\nqh6ZO6C19qIkV2awUv0NE863HP674fZrVbVv3pHLbM0kLwYAAAAAAADAqvPp4XZtktePGfOWJK9N\n8vyqOjCRVMtrWwY/OPjjSV/YSnsAAAAAAAAAxqqqu1trn0zyyiQ7WmsPJPlEVc201jYk2ZHksgxK\n73dPKldr7fgkG+fsPnHW8dPmHNtfVXtHnGdTks0Z5K/lzrkQpT0AAAAAAAAAC3ldkqcleWGSjyX5\nrdbaw0memWR9BoX3e6rqU8t4zakFjr8qyY3zzH1ozr6bkrxmxNifmfX+e4tKtozcHh8AAAAAAACA\neVXVvqq6LMkvJbktyRNJzk3ySJJbk1xaVe+YM21m+Dpci5k/s8TXKKfMOv7jI8h7WKy0BwAAAAAA\nAGBRquqWJLcsYtwdSdYuYtzIheZVdc4i5t6c5OaFxi3iPF/JIrKuFCvtAQAAAAAAAKATpT0AAAAA\nAAAAdKK0BwAAAAAAAIBOlPYAAAAAAAAA0InSHgAAAAAAAAA6UdoDAAAAAAAAQCdKewAAAAAAAADo\nRGkPAAAAAAAAAJ0o7QEAAAAAAACgE6U9AAAAAAAAAHSitAcAAAAAAACATpT2AAAAAAAAANCJ0h4A\nAAAAAAAAOlHaAwAAAAAAAEAnSnsAAAAAAAAA6ERpDwAAAAAAAACdKO0BAAAAAAAAoBOlPQAAAAAA\nAAB0orQHYMlOOumk3hEAAAAAAACOCkp7AJZsy5YtufXWW3vHAAAAAAAAWPWU9gAcli1btvSOAAAA\nAAAAsOop7QEAAAAAAACgE6U9AAAAAAAAAHSyrncAAABYsqmp3gkm78BjvRP0se8nvRNM3kmbeifo\n4+CB3gkm7/iTeifo4+ATvRMwKcfoX7s2vfx9vSNM3A/+9bW9I3Sx6Yobekfo41j8//Zx63sn6ONY\n/Puz/ft6J+hi3VN/pneEiXtszdreEfo46dTeCfp4fH/vBLAgK+0BAAAAAAAAoBOlPQAAAAAAAAB0\norQHAAAAAAAAgE6U9gAAAAAAAADQidIeAAAAAAAAADpR2gMAAAAAAABAJ0p7AKYyvT4AACAASURB\nVAAAAAAAAOhEaQ8AAAAAAAAAnSjtAQAAAAAAAKATpT0AAAAAAAAAdKK0BwAAAAAAAIBOlPYAAAAA\nAAAA0InSHgAAAAAAAAA6Wdc7AACr3+7du7Nnz57s2rWrdxQAAAAAAIBVRWkPwBG57777sn379t4x\nAAAAAAAAViW3xwfgiOzduzdJsnPnzuzcubNzGgAAAAAAgNXFSnsAlsX09HTvCAAAAAAAAKuO0h4A\nAAAAAACAZdVauyTJ7UlmkpxTVfd1jvSkpbQHAAAAAAAAYNVrrW1LckuSZyeZqaq1i5z3kiRXJ3l+\nktOT/DjJ3Uk+meSjVXVgZRIPeKY9AAAAAAAAAKtWa21ta21HkruSLOl5vq21DyX5XJLLk2xM8idJ\nHk/yN5N8MMkdrbWNy5v4pyntAQAAAAAAAFiVWmvTSb6SZEeSH2ZwS/7Fzn1Tkl9JcjDJW5KcXlU/\nW1VPT/KSJA8neUGS317u3LMp7QEAAAAAAABYrT6e5HlJPp/kuUm+tJhJrbUNSf6XJDNJPlBV/6Kq\nHjt0vKr+zyS/nGQqyZWttYuWO/ghnmkPAAAAAAAAwIJaa2uSXJXk1UkuTPLUJN9P8uUkN1TV2FXu\nrbVNSd6ewW3oz0pyIMk9SW6sqg+PGP+dJFuSvDXJziQ3JLkkyaaqmr04fX+SN1fV+4fzFvt1fj7J\nKRmU9u8dNaCq/lVr7U+SPCvJP0zy1cWefCmstAcAAAAAAABgXq21k5J8IclNSS7LoOz+VpKTk2xP\ncltr7V1jpp+V5GtJrs1g5fr9STYkuSjJB1tro0rzmeFrKslnkrwsyYPDa862/VBhv0QvHG6rqh6Y\nZ9y/G2a49DCusShKewAAAAAAAAAW8tEkF2fwnPdLq+rMqppOclqSDwzHXNdau2LE3I8k2Z3k7Ko6\nv6ouSLI5yV3D429qrW0ec90XJzk9yQVVNV1V580+WFXfPczvc2EGPwr45gLj7hluzxveUn/ZKe0B\nAAAAAAAAGKu1dmGSV2RQcr+xqu44dKyq9lfVNUm+Mdx13YhTnJDk8tkr2qvqoSRvG35ck+QFI+ZN\nZXBL/Kurau4K+yN11nA73yr72cfXJDlzmTMk8Ux7AAAAAAAAAOZ35XC7N4Nb1Y9ybZKtSR4acexD\nVfXoiP2zV7k/Y8x5H6yq2xeVcmk2Drf7Fhg3+/jGsaOOgNIeAAAAAAAAgPlcNNzeU1VPjBpQVXcm\nuXPM/K+O2f+9We83jTg+k2TXohIu3aFb3T+2wLjZPzY4cSWCKO0BWHb33Xdf7wjAUWTbtm29IwAA\nAADAsW5zBgX6qFX0i/GjUTur6mBr7dDHqTFzD/eaC9mX5KQkxy8wbv2cOctOaQ/Astu+fXvvCMBR\nZGZmpncEAAAAADjWHSquD3S49t4VOu+eDEr7kxYY95RZ70f++OBIKe0BWBE7d+7M9PR07xgAAAAA\nAMCRO7TC/NSuKZbXf07y9CRnLTDu7OH28ST3r0QQpT0AK2J6etotrQEAAAAA4Ohwf5LnJDmjd5Bl\n9H8neUEG32s+Fw6336yqx1ciyJqVOCkAAAAAAAAAR42vD7fntdZOGTWgtfai1trvtNZ+u7W2GhaP\nf364Pa+1du6oAa21tUkuSzKT5HMrFURpDwAAAAAAAMB8Pj3crk3y+jFj3pLktUmeX1UHJpLqyHwu\nyXeH768bM+aXkvxMkieS3LRSQVbDLxwAAAAAAAAA6KSq7m6tfTLJK5PsaK09kOQTVTXTWtuQZEf+\nckX6uyeVq7V2fJKNc3afOOv4aXOO7a+qvUlSVY+31v5RkpuTvK61dl+S66vq0dbaVJJfSPK+DL7T\n+6vqT1fqe1hpDwAAAAAAAMBCXpfkC0lOSPKxJA+21nYl+V6Sf5RBuf2eqvrUMl5zaoHjr0ry0JzX\nW2bNnXvsA7MnV9XHkvzT4cd/kuTh1tofJ3kwyScz+AHAZ5K89Ui/yHyU9gAAAAAAAADMq6r2VdVl\nGdwy/rYMbhl/bpJHktya5NKqesecaTPD1+FazPyZJb5+SlX9epKLM3gEwA+TPCvJwST/JskvVNWV\nVfXEEXyHBbk9PgAAAAAAAACLUlW3JLllEePuSLJ2EeNGLjSvqnMWMffmDG5vf0Sq6j8k+Q9Hep7D\nZaU9AAAAAAAAAHSitAcAAAAAAACATpT2AAAAAAAAANCJ0h4AAAAAAAAAOlHaAwAAAAAAAEAnSnsA\nAAAAAAAA6ERpDwAAAAAAAACdKO0BAAAAAAAAoBOlPQAAAAAAAAB0orQHAAAAAAAAgE6U9gAAAAAA\nAADQidIeAAAAAAAAADpR2gMAAAAAAABAJ0p7AAAAAAAAAOhEaQ8AAAAAAAAAnSjtAQAAAAAAAKAT\npT0AAAAAAAAAdKK0BwAAAAAAAIBOlPYAAAAAAAAA0InSHgAAAAAAAAA6UdoDAAAAAAAAQCdKewAA\nAAAAAADoZF3vAAAcPTZu3Ng7AnCsOOHE3gkmb+Zg7wR9bHxq7wSTNzPTO0Efx53QO8HkHXyidwIm\nac3a3gkm77j1vRP0MTXVO8HEbbriht4RuvjBH7yxd4QuNr30+t4RJu+Jx3sn6OPEk3snmLypY3Od\n44EDB3pHmLy9j/RO0MdTNvVO0MfxG3ongAUdm38FAmBFbN26NbfeemvvGAAAAAAAAKuG0h6AZbVl\ny5beEQAAAAAAAFYNpT0AAAAAAAAAdKK0BwAAAAAAAIBOlPYAAAAAAAAA0InSHgAAAAAAAAA6UdoD\nAAAAAAAAQCdKewAAAAAAAADoRGkPAAAAAAAAAJ0o7QEAAAAAAACgE6U9AAAAAAAAAHSitAcAAAAA\nAACATpT2AAAAAAAAANCJ0h4AAAAAAAAAOlHaAwAAAAAAAEAnSnsAAAAAAAAA6ERpDwAAAAAAAACd\nrOsdAAAAAAAAAICjS2vtkiS3J5lJck5V3dc50pOW0h4AAAAAAACAVa+1ti3JLUmenWSmqtYuYs5T\nklyd5IrhvI1JHkry5SQ3VNWdK5d4wO3xAQAAAAAAAFi1WmtrW2s7ktyVZHoJ856d5O4k1yf5G0kO\nJvmTJCcn+TtJbm+t/fryJ/5pSnsAAAAAAAAAVqXW2nSSryTZkeSHGdySfzHzNiT5XJJnZrCy/qVV\n9bSq+tkkZyT5x0mmkuxorf3CCkT/C0p7AAAAAAAAAFarjyd5XpLPJ3luki8tct5rMijsZ5K8oqr+\n6NCBqnq0qv5JkvdlUNz/ZmttallTz+KZ9gAAAAAAAAAsqLW2JslVSV6d5MIkT03y/fzl89/HrnJv\nrW1K8vYklyc5K8mBJPckubGqPjxi/HeSbEny1iQ7k9yQ5JIkm6pq9uL0/UneXFXvH85b7Nf574fb\nb1bVF8eM+c0k12RQ7l+c5I7FnnwprLQHAAAAAAAAYF6ttZOSfCHJTUkuy2CF+rcyeP779iS3tdbe\nNWb6WUm+luTaDFau359kQ5KLknywtfbeEXNmhq+pJJ9J8rIkDw6vOdv2Q4X9Ep09PP/d4wZU1YMZ\n/LAgGXznFaG0BwAAAAAAAGAhH81gtfnDSS6tqjOrajrJaUk+MBxzXWvtihFzP5Jkd5Kzq+r8qrog\nyeYkdw2Pv6m1tnnMdV+c5PQkF1TVdFWdN/tgVX33iL5VctICxx8ebn/2CK8zltIeAAAAAAAAgLFa\naxcmeUUGK9PfWFV/cZv4qtpfVdck+cZw13UjTnFCksur6oFZ8x5K8rbhxzVJXjBi3lQGt8S/uqrm\nrrA/Uv95eP7pBcadNdyO+1HBEVPaAwAAAAAAADCfK4fbvRncqn6Ua5NcneTdI459qKoeHbH/m7Pe\nP2PMeR+sqtsXlXJpPj/cPqu19vJRA1prr0hy7vDjU1YgQ5Jk3UqdGAAAAAAAAICjwkXD7T1V9cSo\nAVV1Z5I7x8z/6pj935v1ftOI4zNJdi0q4dL9bpK3JDkzyU2ttddW1R8kSWttXZJfTvLPk/xZBj8o\nWLtCOay0BwAAAAAAAGBemzMo0B86zPk/GrWzqg7O+jg1Zu7hXnNeVfXjJNszeGb9qUn+ZWvtz1tr\nu5LsSXJDks8l+eRwyp6VyJFYaQ/ACtm1a6V++AYca7Zt29Y7AgAAAAAc69YPtwc6XHvvSp24qr7a\nWntukuuSvDTJXxke+mKSG6vq06213x3u+96IUywLpT0Ay2rjxo1JkquuuqpzEuBoMTMz0zsCAAAA\nABzr9g23p3ZNsQKq6rtJ3jx8jdIyuMvAH69UBqU9AMtq69atuffee7Nnz4rdJQYAAAAAAJis+5M8\nJ8kZvYNMUmttQ5K/Pvz4H1fqOkp7AJbd1q1be0cAAAAAAACWz9eTvCTJea21U6rqkbkDWmsvSnJl\nBqvS3zDhfEvWWluT5NwkW6vqD8cM+/kkJyR5NMlnVyrLmpU6MQAAAAAAAABHhU8Pt2uTvH7MmLck\neW2S51fVgYmkOjKXJ7k3yWdbaxfPPdhaOynJOzP4EcLOqlqxWwxbaQ8AAAAAAADAWFV1d2vtk0le\nmWRHa+2BJJ+oqpnhLeR3JLksg4L73ZPK1Vo7PsnGObtPnHX8tDnH9lfV3uH7zyb5Vgar7W9urf18\nVX1jOO9ZSW5Mcn6SBzMo71eM0h4AAAAAAACAhbwuydOSvDDJx5L8Vmvt4STPTLI+g8L+PVX1qWW8\n5tQCx1+VQbk+bu5Dc/bdlOQ1SVJVB1prr07yR0m2JPna8McIjyU5Zzj+z5K8rKr+fOnRF8/t8QEA\nAAAAAACYV1Xtq6rLkvxSktuSPJHBKvVHktya5NKqesecaTPD1+FazPyZJb7+QlV9Jcnzkvx2kj9O\ncnKSM5LcneR/TfLsQ6vvV5KV9gAAAAAAAAAsSlXdkuSWRYy7I8naRYwbudC8qs4ZtX/OmJuT3LzQ\nuAXO8Z0k/8ORnONIWWkPAAAAAAAAAJ0o7QEAAAAAAACgE6U9AAAAAAAAAHSitAcAAAAAAACATpT2\nAAAAAAAAANCJ0h4AAAAAAAAAOlHaAwAAAAAAAEAnSnsAAAAAAAAA6ERpDwAAAAAAAACdKO0BAAAA\nAAAAoBOlPQAAAAAAAAB0orQHAAAAAAAAgE6U9gAckW9/+9u9IwAAAAAAAKxaSnsADsvGjRuTJO98\n5zs7JwEAAAAAAFi9lPYAHJatW7fm3nvvzc6dO3tHAQAAAAAAWLXW9Q4AwOq1devW7Nmzp3cMAAAA\nAACAVctKewAAAAAAAADoRGkPAAAAAAAAAJ0o7QEAAAAAAACgE6U9AAAAAAAAAHSitAcAAAAAAACA\nTpT2AAAAAAAAANCJ0h4AAAAAAAAAOlnXOwAAACzZ44/2TjB5MzO9E/Tx6E96J5i849b3TtDHvh/1\nTjB5J57cO0EfTzzeOwGT8tgx+Gd4kkxN9U4weQce652gi00vvb53hC5+8Ln/uXeEidv08vf1jtDH\nsfjn+Imn9k7QxcEDT/SOMHkbntI7QR+P7u2doI+Dx+i/U2FVsdIeAAAAAAAAADpR2gMAAAAAAABA\nJ0p7AAAAAAAAAOhEaQ8AAAAAAAAAnSjtAQAAAAAAAKATpT0AAAAAAAAAdKK0BwAAAAAAAIBOlPYA\nAAAAAAAA0InSHgAAAAAAAAA6UdoDAAAAAAAAQCdKewAAAAAAAADoRGkPAAAAAAAAAJ0o7QEAAAAA\nAACgE6U9AAAAAAAAAHSitAcAAAAAAACATpT2AAAAAAAAANCJ0h4AAAAAAAAAOlnXOwAAAAAAAAAA\nR5fW2iVJbk8yk+Scqrqvc6QnLaU9AAAAAAAAAKtea21bkluSPDvJTFWtXcScDUmuTvJzSZ6b5JQk\n+5J8O8ltST5UVX+6YqHj9vgAAAAAAAAArGKttbWttR1J7koyvYR5z0lyT5L3JbkkyfFJ/iTJwSQX\nJnlzkrtba39v2UPPorQHAAAAAAAAYFVqrU0n+UqSHUl+mMEt+Rcz78Qkn03yV5L8v0leXlWnVNWz\nq+qpGZT4f5pkfZLfa609ayXyJ0p7AAAAAAAAAFavjyd5XpLPZ3B7+y8tct4vJtmSZCbJFVX1b2Yf\nrKp/n+TvDz8el+QfLEvaETzTHgAAAAAAAIAFtdbWJLkqyaszuH38U5N8P8mXk9xQVWNXubfWNiV5\ne5LLk5yV5EAGt6a/sao+PGL8dzIo1d+aZGeSGzJY/b6pqmYvTt+f5M1V9f7hvMV+nceS/MskP6mq\nr40aUFV3tdb2JHlKkhVbaa+0BwAAAAAAAGBerbWTknwuycUZrE7/bpJvZVCsb0+yvbX2nqp6+4jp\nZ2VQvG8Zzrk/yblJLkpyUWtta1X9T3PmzAxfU0k+k+SvZ3C7+h/MGbe9qr671O9TVb+X5PfmG9Na\nm5r18b8s9RqL5fb4AAAAAAAAACzkoxkU9g8nubSqzqyq6SSnJfnAcMx1rbUrRsz9SJLdSc6uqvOr\n6oIkm5PcNTz+ptba5jHXfXGS05NcUFXTVXXe7IOHU9gvwa8k2Th8/4crdRGlPQAAAAAAAABjtdYu\nTPKKDFa+v7Gq7jh0rKr2V9U1Sb4x3HXdiFP8f+zde9TldX0f+vczwx0GGRQTA14wPHx9rNJmWoxp\neiC6sMaUcErSgMgkrZdDrCvqwsaDNzKuJlFSq9GEoHZZQRgv2BrjOa6m9gQJ6PF4OWobGobPzAlQ\nLknkKk4GuQzznD9+e+zjZD+XmXme/Z3L67XWXr+99+/7/f3ee8FihvXe3+8+PMk5VXXXnHn3Jnnr\n6OWqJC8cM28qw5b4F1XVrXv9QZagtXZMa+3vtdZ+N8kfZPjMv1VVX1ype9oeHwAAAAAAAICFnDc6\nbsuwVf04FyeZTnLvmHMfrKpHx7x/85znT5vnuvdU1fVLSrkXWmtvTPK7c976myRXJfl3VfW1lby3\n0h4AAAAAAACAhZw+Ot5SVU+MG1BVNya5cZ7535jn/fvnPF875vxskk1LSrj37k3yX5Mcm+ELBEcn\n+YUkaa3dPXeXgOWmtAdg2WzZsiVbt27tHQM4wKxbt653BAAAAAA42J2YoUAft4p+Kb437s2q2tFa\n2/lyap65e3rP3VJVn0jyiSRpre3clv+9SV6Z5J+01s6oqs0rcW+lPQDL4o477si5557bOwZwAJqd\nne0dAQAAAAAOdkeMjts73HvbpG9YVbNJ/rS1dmaG1fcnJ/lwkhetxP2U9gAsi23bhj8zN27cmJmZ\nmc5pAAAAAACAZfTw6Hhc1xQTVlV/01q7Ksm/TnJGa+34qnpgue+jtAdgWc3MzNjKGgAAAAAADix3\nJnlekhN6B1kurbV/mOG36++uqq8uMPQ7c54/NYnSHoB+brvttt4RAAAAAACAyftWkpclOaW19qSq\nemjXAa21lyQ5L8lsktdNON+e+HiSZyb50yQvXmDc0+c8v2clgqxaiYsCcGBZs2ZNkuTSSy/tnAQA\nAAAAAOjg06Pj6iSvnWfMm5O8OskLqmr7RFLtnf9jdPxHrbUXjhvQWjsiyStGL29aia3xEyvtAViC\n6enpbN68OV//+tezfv363nEAAAAAAIAJqqqbWmufSvLyJBtaa3cl+URVzbbWjkyyIclZGVbZv3tS\nuVprhyVZs8vbR805/+Rdzj1SVdtGz9+T5J8nOTbJH7bW3pjkP1bV7GjuaUn+IMmzM3yu317+TzBQ\n2gOwJNPT09m6dWvvGAAAAAAAQB+vyfCb7i9Kck2S97fW7suwxfwRGYrty6rq2mW859Qi5y9IcuUC\nc+/d5b2rkrwqSarqrtba2Un+Q5IfSXJtku+21v4qw2/dH5fhMz2R5J1V9R/25AMshe3xAQAAAAAA\nAFhQVT1cVWcleWWS6zKU2c9O8lCSzyZ5cVW9fZdps6PHnlrK/NndfPxAVX05yXOSvCXJl5JsT3JK\nhsXvNyf5dxm2+1+xVfaJlfYAAAAAAAAALFFVXZ3k6iWMuyHJ6iWMG7vQvKpOXsLcjyX52GLjFrnG\nQxm2yn/P3lxnb1hpDwAAAAAAAACdKO0BAAAAAAAAoBOlPQAAAAAAAAB0orQHAAAAAAAAgE6U9gAA\nAAAAAADQidIeAAAAAAAAADpR2gMAAAAAAABAJ0p7AAAAAAAAAOhEaQ8AAAAAAAAAnSjtAQAAAAAA\nAKATpT0AAAAAAAAAdKK0BwAAAAAAAIBOlPYAAAAAAAAA0InSHgAAAAAAAAA6UdoDAAAAAAAAQCdK\newAAAAAAAADoRGkPAAAAAAAAAJ0o7QEAAAAAAACgE6U9AAAAAAAAAHSitAcAAAAAAACATpT2AAAA\nAAAAANCJ0h4AAAAAAAAAOlHaAwAAAAAAAEAnh/QOAAAAu23HE70TMCmrVvdOMHmrD8LPnCSHHd47\nweQ9+v3eCfo4WP8dP/yo3gkm75G/6Z2gjyOP7Z1g8g49oneCPp54vHeCLtb+/O/2jjBxD/6fF/eO\n0MXas9/XO8LkPbqtd4IuDj36mN4RJu7x+w/Sv4sfdRD+PSVJZnf0TgCLstIeAAAAAAAAADpR2gMA\nAAAAAABAJ0p7AAAAAAAAAOhEaQ8AAAAAAAAAnSjtAQAAAAAAAKATpT0AAAAAAAAAdKK0BwAAAAAA\nAIBOlPYAAAAAAAAA0InSHgAAAAAAAAA6UdoDAAAAAAAAQCdKewAAAAAAAADoRGkPAAAAAAAAAJ0o\n7QEAAAAAAACgE6U9AAAAAAAAAHSitAcAAAAAAACATpT2AAAAAAAAANCJ0h4AAAAAAAAAOjmkdwAA\nAAAAAAAADiyttTOTXJ9kNsnJVXVH50j7LKU9AAAAAAAAAPu91tq6JFcneW6S2apavcR5RyZ5fZJf\nSnJKktVJ7k7yX5J8sKpuWZnEA6U9AAAAAAAAAPut1trqJO9I8vYMhfvuzD05yRcylPU7ktye4Wfm\np5O0JL/aWju/qj63nJnn8pv2AAAAAAAAAOyXWmszSb6WZEOS72bYkn+pc49I8sdJfjzJnyT58ao6\npaqeneTUJF9JcliSK1trT1nu7Dsp7QEAAAAAAADYX308yU9kWC3//CRf2o25/ypDOf9nSc6uqv+x\n80RV3ZrkF5PckOQzSX5suQLvyvb4AAAAAAAAACyqtbYqyfokFyY5LcnxSR7IsCL98qqad5V7a21t\nkrclOSfJSUm2J7klyZVV9aEx429P8owkb0myMcnlSc5Msraq5i5OfyTJm6rqA6N5S/0sU0lel2Q2\nyYaqemzXMVX1nSQvWtIF94KV9gAAAAAAAAAsqLV2dJIvJrkqyVkZyu5bkxyb5Nwk17XW3jXP9JOS\nfDPJxUmmktyZ5Mgkpye5orX23jFzZkePqQwr3c9Ocs/onnOdu7Ow300vTPK0JNsybJHfjZX2ACyL\n2267rXcEAAAAAABg5XwkyRlJ7kvyS1V1Q/KD34X/nSSvT3JJa+3rVfVHu8z9cJItSc6oqrtG805I\n8rkM5fkbWmvvq6q7x9z3pUmekuQ5oy3rf8hoNfyeOH10/POqery1dmqSVyf5exm+iHB3kv+U5Jqq\nenwP77EkVtoDsFfWrFmTJLn00ks7JwEAAAAAAFZCa+20JOdnWPn+azsL+ySpqkeq6o1Jvj1665Ix\nlzg8yTk7C/vRvHuTvHX0clWG8n5XUxm2xL9oXGG/l/7O6HhXa+2iDL9r/+sZdhF4QYbdAz6S5KbW\n2o8v871/iNIegL0yPT2dzZs3Z+PGjb2jAAAAAAAAK+O80XFbhq3qx7k4yUVJ3j3m3Aer6tEx7988\n5/nT5rnuPVV1/ZJS7p4nZ/gSwvOTXJHk4xlW3x+d5MeS/KskDyc5NcnnRzsKrAjb4wOw16anp7N1\n69beMQAAAAAAgJWxcyv5W6rqiXEDqurGJDfOM/8b87x//5zna8ecn02yaUkJd9+aDCv5p5NcUVWv\nn3PukSTvb63dnuQPMxT3r0ly+UoEUdoDsOy2bNmixAeWzbp163pHAAAAAICD3YkZCvR793D+98a9\nWVU7Wms7X07NM3dP77mY2TnH3xo3oKr+qLX23zNspf+LUdoDsD+44447cu655/aOARxAZmdnFx8E\nAAAAAKyknVvDb+9w720rdN2dqw/vrarvLDDuS0mel2Eb/RWhtAdgWW3bNvzZuXHjxszMzHROAwAA\nAAAALIOHR8fjuqZYXncucdwDo+MxKxVEaQ/AipiZmbGlNQAAAAAAHBjuzLDa/ITeQZbRTaPjU1pr\nR1fVfCv6d35R4cGVCqK0BwAAAAAAAGAh30rysiSntNaeVFUP7TqgtfaSJOdl+I3410043574Qoas\nqzL8Xv3V84z7ydG4/7pSQVat1IUBAAAAAAAAOCB8enRcneS184x5c5JXJ3lBVW2fSKq9UFV/maG4\nn0ry5tba31rw3lr7qST/YPTyMyuVxUp7AAAAAAAAAOZVVTe11j6V5OVJNrTW7kryiaqaba0dmWRD\nkrMyrEh/96RytdYOS7Jml7ePmnP+ybuce2SXbfAvSfKiJM9N8oettYuq6q9Hc38mycdH425Jcs0y\nRv8hSnsAAAAAAAAAFvOaJE/NUHJfk+T9rbX7kjwzyREZCvvLquraZbzn1CLnL0hy5QJz793lvauS\nvGrni9GXEV6eZGOSf5LkjtbarUmOTfKjGT7T7Un+16p6dHfDL5Xt8QEAAAAAAABYUFU9XFVnJXll\nkuuSPJHk2UkeSvLZJC+uqrfvMm129NhTS5k/u5uPH1JVn0vy/CR/kOS2JCdlWK3/zSTvSPL8qvr/\n9uIzLMpKewAAAAAAAACWpKquTnL1EsbdkGT1EsaNXWheVScvYe7HknxssXFLuM7tSd6wt9fZU1ba\nAwAAAAAAAEAnSnsAAAAAAAAA6ERpDwAAAAAAAACdKO0BAAAAAAAAoBOlPQAAAAAAAAB0orQHAAAA\nAAAAgE6U9gAAAAAAAADQidIeAAAAAAAAADpR2gMAAAAAAABAJ0p7AAAAAAAAAOhEaQ8AAAAAAAAA\nnSjtAQAAAAAAAKATpT0AAAAAAAAAdKK0BwAAAAAAAIBOlPYAAAAAAAAAa+L6jgAAIABJREFU0InS\nHgAAAAAAAAA6UdoDAAAAAAAAQCdKewAAAAAAAADoRGkPAAAAAAAAAJ0o7QEAAAAAAACgE6U9AAAA\nAAAAAHSitAcAAAAAAACATg7pHQAAAGBeU1O9E0ze7I7eCfrY/ljvBJN36GG9E/SxanXvBH1se6h3\ngsk78pjeCfo4GP9Z79jeO0EfRx3bO0Efj32/d4KJW3v2+3pH6OLBz7+pd4SJW/tz7+kdoYtDDz+0\nd4SJe/yQg+8zJzk4/78rSR57pHcCWJSV9gAAAAAAAADQidIeAAAAAAAAADpR2gMAAAAAAABAJ0p7\nAAAAAAAAAOhEaQ8AAAAAAAAAnSjtAQAAAAAAAKATpT0AAAAAAAAAdKK0BwAAAAAAAIBOlPYAAAAA\nAAAA0InSHgAAAAAAAAA6UdoDAAAAAAAAQCdKewAAAAAAAADoRGkPAAAAAAAAAJ0o7QEAAAAAAACg\nE6U9AAAAAAAAAHSitAcAAAAAAACATpT2AAAAAAAAANDJIb0DAAAAAAAAAHBgaa2dmeT6JLNJTq6q\nOzpH2mcp7QEAAAAAAADY77XW1iW5Oslzk8xW1eolzDkiyWuSnJvktCRPSrItyV8k+b+SXFFVd65Y\n6NgeHwAAAAAAAID9WGttdWttQ5KvJpnZjXknJ/mzJL+X5Gcy7ApwS5KHkvxEkkuS/Hlr7ZzlzjyX\n0h4AAAAAAACA/VJrbSbJ15JsSPLdDFvyL2XeqiSfTXJKkgeS/EJVPbWqTquqZyU5NcnXkxyT5JOt\ntaevQPwkSnsAltltt93WOwIAAAAAAHDw+HiGVfFfSPL8JF9a4rwzMmyHP5vkHVX1ubknq+ovklww\nenlEklcuS9ox/KY9AMtizZo1SZJLL720cxIAAAAAAGAljFanr09yYYbC+/gMq9S/kuTyqpp3lXtr\nbW2StyU5J8lJSbZn2Ir+yqr60Jjxtyd5RpK3JNmY5PIkZyZZW1VzF6c/kuRNVfWB0bwlf5w5z8cW\n/VV1W2vtwSTH7TJ+WVlpD8CymJ6ezubNm7Nx48beUQAAAAAAgGXWWjs6yReTXJXkrAwr1G9NcmyS\nc5Nc11p71zzTT0ryzSQXJ5lKcmeSI5OcnuSK1tp7x8yZHT2mknwmydlJ7hndc65zdxb2u+mv5zw/\nYoFxh46Of7UH91gSpT0Ay2Z6ejozMzO9YwAAAAAAAMvvIxm2lL8vyYur6seqaibJk5P8/mjMJa21\nfzpm7oeTbEnyrKo6taqek+TEJF8dnX9Da+3Eee770iRPSfKcqpqpqlPmnqyq7+zh57kxyUOj568Y\nN6C19lMZftM+ST6/h/dZlNIeAAAAAAAAgHm11k5Lcn6Gle+/VlU37DxXVY9U1RuTfHv01iVjLnF4\nknOq6q458+5N8tbRy1VJXjhm3lSGLfEvqqpdV9jvlap6MMkbkjyR4UsD72ytPaO1trq1dlxr7Z8l\n+XSGz/zRqvrT5bz/XH7THgAAAAAAAICFnDc6bsuwVf04FyeZTnLvmHMfrKpHx7x/85znT5vnuvdU\n1fVLSrmbquqa1tpdSd6c5DdGj7n+e5LfqqoPr8T9d1LaAwAAAAAAALCQ00fHW6rqiXEDqurGDFvO\nj/ONed6/f87ztWPOzybZtKSEe6C1tirJTyY5dXSvBzL81v2aJE9P8qwkP9Na+89V9T9WKoft8QEA\nAAAAAABYyIkZSu1xq+iX4nvj3qyqHXNeTs0zd0/vuRSfT/KuDL35WVV1QlU9v6qelaGw/+MMPwvw\nzdZaW6kQVtoDMDFbtmzJ1q1be8cA9jPr1q3rHQEAAAAADnZHjI7bO9x720pctLX2miQ/m2RHkvOr\n6od2A6iqO1trFyR5dpJ1Sd6f5GUrkUVpD8BEbNmyJaeeemrvGMB+aHZ2tncEAAAAADjYPTw6Htc1\nxfJaPzr++a6F/U5VtaO19rkkfz/JS1prx1XVd5c7iNIegInYucJ+48aNmZmZ6ZwGAAAAAADYDXcm\neV6SE3oHWUZPy7Dl/18uMu6+0XEqyY8mUdoDsH+bmZmx1TUAAAAAAOxfvpVha/hTWmtPqqqHdh3Q\nWntJkvMyFOGvm3C+PfFghiL+WYuMe8ac5w+sRJBVK3FRAAAAAAAAAA4Ynx4dVyd57Txj3pzk1Ule\nUFXbJ5Jq79wwOp7aWjtt3IDW2qokPz96uaWq7lmJIFbaAwAAAAAAADCvqrqptfapJC9PsqG1dleS\nT1TVbGvtyCQbkpyVYZX9uyeVq7V2WJI1u7x91JzzT97l3CNVtW30/PcyfAFhTZLPtdZeVVXXz5l7\nYpL3Jfk7GT7Xby1z/B9Q2gMAAAAAAACwmNckeWqSFyW5Jsn7W2v3JXlmkiMyFNuXVdW1y3jPqUXO\nX5DkygXm3rvLe1cleVWSVNXdrbVzMuwi8Iwk17XWHszwG/dHZ/hcU0meSPKvq2rjnnyApbA9PgAA\nAAAAAAALqqqHq+qsJK9Mcl2GMvvZSR5K8tkkL66qt+8ybXb02FNLmT+7m48fqKobkjwnyduTfDnJ\njiSnJjk+yc1Jrkiyrqp+cy8+w6KstAcAAAAAAABgSarq6iRXL2HcDUlWL2Hc2IXmVXXyEuZ+LMnH\nFhu3yDUeTHLZ6NGFlfYAAAAAAAAA0InSHgAAAAAAAAA6UdoDAAAAAAAAQCdKewAAAAAAAADoRGkP\nAAAAAAAAAJ0o7QEAAAAAAACgE6U9AAAAAAAAAHSitAcAAAAAAACATpT2AAAAAAAAANCJ0h4AAAAA\nAAAAOlHaAwAAAAAAAEAnSnsAAAAAAAAA6ERpDwAAAAAAAACdKO0BAAAAAAAAoBOlPQAAAAAAAAB0\norQHAAAAAAAAgE6U9gAAAAAAAADQidIeAAAAAAAAADpR2gMAAAAAAABAJ0p7AAAAAAAAAOhEaQ8A\nAAAAAAAAnSjtAQAAAAAAAKATpT0AAAAAAAAAdHJI7wAAALDbDjuid4LJe+KJ3gmYlB2zvRP0ccxx\nvRNM3sMP9U7Qx2FH9k7Qx5FTvRNM3mPf752gjyOO6p1g8h55uHeCPqYO0vVQRx2Ef2Y/uq13gi7W\n/tx7ekeYuAf/05t7R+jixFd9sneEyTvksN4JmKRj1vZOAIs6SP9mCQAAAAAAAAD9Ke0BAAAAAAAA\noBOlPQAAAAAAAAB0orQHAAAAAAAAgE6U9gAAAAAAAADQidIeAAAAAAAAADpR2gMAAAAAAABAJ0p7\nAAAAAAAAAOhEaQ8AAAAAAAAAnSjtAQAAAAAAAKATpT0AAAAAAAAAdKK0BwAAAAAAAIBOlPYAAAAA\nAAAA0MkhvQMAcGDatGnTgq8BAAAAAABQ2gOwzNasWZMkWb9+feckAAAAAAAA+z6lPQDLanp6Ops3\nb87WrVt/6P1NmzYp8gEAAAAAAHahtAdg2U1PT/eOAAAAAAAAsF9Q2gMAAAAAAACwrFprZya5Psls\nkpOr6o7OkfZZSnsAAAAAAAAA9juttQ1JNixx+B9V1S+sZJ49pbQHAAAAAAAAYH/2WJKbFxlz6ySC\n7AmlPQAAAAAAAAD7s7+sqnW9Q+ypVb0DAAAAAAAAAMDBykp7AAAAAAAAABbVWluVZH2SC5OcluT4\nJA8k+UqSy6vq+gXmrk3ytiTnJDkpyfYktyS5sqo+NGb87UmekeQtSTYmuTzJmUnWVtUBtTj9gPow\nAAAAAAAAACy/1trRSb6Y5KokZyWZzfA78ccmOTfJda21d80z/aQk30xycZKpJHcmOTLJ6UmuaK29\nd8yc2dFjKslnkpyd5J7sw79Nv6estAcAAAAAAABgMR9JckaS+5L8UlXdkCSttSOS/E6S1ye5pLX2\n9ar6o13mfjjJliRnVNVdo3knJPlckhcmeUNr7X1VdfeY+740yVOSPKeq5i3sW2s/kuSVSX46yVOT\nbEtyc5Jrq+pLe/iZJ8JKewAAAAAAAADm1Vo7Lcn5GVa+/9rOwj5JquqRqnpjkm+P3rpkzCUOT3LO\nzsJ+NO/eJG8dvVyVobzf1VSGLfEvWqiwT/KjSf4iyW8n+bkk/2A073VJbmitfbK1dviiH7QTpT0A\nAAAAAAAACzlvdNyWYav6cS5OclGSd48598GqenTM+zfPef60ea57T1Vdv0i+wzN8aeDnkjw5yVFJ\nfirJ5zN80eC8DKv990m2xwcAAAAAAABgIaePjrdU1RPjBlTVjUlunGf+N+Z5//45z9eOOT+bZNMC\nua5L8niSB5J8uKpm55z7epJzWmufSPLyJL/cWvu9qvrWAtfrQmkPwD5py5Yt2bp1a+8YwD5g3bp1\nvSMAAAAAwMHuxAwF+r17OP97496sqh2ttZ0vp+aZO+89q+rLSb68yL1/PcNK+6kkr0iitAeAxWzZ\nsiWnnnpq7xjAPmJ2dnbxQQAAAADASjpidNze4d7b9mZyVf1la+2WJDNJ/u7yRFpeSnsA9jk7V9hv\n3LgxMzMzndMAAAAAAMBB7+HR8biuKfbcgxlW2h/TO8g4SnsA9lkzMzO2xQYAAAAAgP7uTPK8JCf0\nDrKHnpphe//7ewcZR2kPAAAAAAAAwEK+leRlSU5prT2pqh7adUBr7SUZfjt+NsnrVjpQa+25SX47\nybOSvH70+/bjxj0tySmjl//vSufaE6t6BwAAAAAAAABgn/bp0XF1ktfOM+bNSV6d5AVVtX0Cme5O\n8o+TnJbkbQuMe0eGrfFnk3xiArl2m9IeAAAAAAAAgHlV1U1JPpWh/N7QWruwtTaVJK21I1trlyU5\nK0Mx/u4JZXooyftGmX62tXZVa+0pO8+31o5rrb03yb8c5fr9qto8iWy7y/b4AAAAAAAAACzmNRl+\nG/5FSa5J8v7W2n1JnpnkiAzF+GVVde0y3nNqkfPvTHJSkl8ZPS5srd06yvLsDH34bJJ/n+TXlzHX\nslLaAwAAAAAAALCgqno4yVmttV9J8stJnp+hGH8gyR8n+b2qumGXabOjx55acH5VPZHkla21jUn+\ntyQ/meTpSXYkuTPJl5J8ZL7fu99XKO0BAAAAAAAAWJKqujrJ1UsYd0OS1UsYN/Yn3avq5N3IdF2S\n65Y6fl/jN+0BAAAAAAAAoBOlPQAAAAAAAAB0orQHAAAAAAAAgE6U9gAAAAAAAADQidIeAAAAAAAA\nADpR2gMAAAAAAABAJ0p7AAAAAAAAAOhEaQ8AAAAAAAAAnSjtAQAAAAAAAKATpT0AAAAAAAAAdKK0\nBwAAAAAAAIBOlPYAAAAAAAAA0InSHgAAAAAAAAA6UdoDAAAAAAAAQCdKewAAAAAAAADoRGkPAAAA\nAAAAAJ0o7QEAAAAAAACgE6U9AAAAAAAAAHSitAcAAAAAAACATpT2AAAAAAAAANCJ0h4AAAAAAAAA\nOjmkdwAADi6bNm1aljEAAAAAAAAHAqU9ABOxZs2aJMn69es7JwEOCEc9qXeCyVt9aO8EfUwdhJuD\nrVrdOwETctixa3tH6OKxbX/TO0IXq44+tneEiTv0sIPzz67HH3u8d4SJO+T4H+kdoYvt27f3jtDF\nju1P9I4wcYcefUzvCF0cevjB99/xE1/1yd4Rurj7oxf0jjBxx5//0d4RmKDZxx/tHQEWpbQHYCKm\np6ezefPmbN26ddGxmzZtUu4DAAAAAAAHBaU9ABMzPT3dOwIAAAAAAMA+5SDcaxIAAAAAAAAA9g1K\newAAAAAAAADoRGkPAAAAAAAAAJ0o7QEAAAAAAACgE6U9AAAAAAAAAHSitAcAAAAAAACATpT2AAAA\nAAAAANCJ0h4AAAAAAAAAOlHaAwAAAAAAAEAnSnsAAAAAAAAA6ERpDwAAAAAAAACdKO0BAAAAAAAA\noBOlPQAAAAAAAAB0orQHAAAAAAAAgE6U9gAAAAAAAADQySG9AwAAAAAAAABwYGmtnZnk+iSzSU6u\nqjs6R9pnKe0BAAAAAAAA2K+01o5L8u0kz0xyS5J1VfXIInP+WZJPj17+ZlVtWNmUS2N7fAAAAAAA\nAAD2K1X13SQXJNmepCX53YXGt9Z+NMmHMqz8/3KSd65wxCVT2gMAAAAAAACw36mqryb5jSRTSS5q\nrf38AsOvTHJ8koeSXFhVsxOIuCRKewAAAAAAAAD2V7+T5LoMxf2/b639yK4DWmuvS/LSDKvsL6qq\nOycbcWF+0x4AAAAAAACARbXWViVZn+TCJKdlWLn+QJKvJLm8qq5fYO7aJG9Lck6SkzJsa39Lkiur\n6kNjxt+e5BlJ3pJkY5LLk5yZZG1V/WBxelXNttbWJ/lvSU5I8rEkPzvnOqcm+TcZCvuPVtV/3LNP\nv3KstAcAAAAAAABgQa21o5N8MclVSc7KUILfmuTYJOcmua619q55pp+U5JtJLs6wIv7OJEcmOT3J\nFa21946ZMzt6TCX5TJKzk9wzuucPqarvJPmV0cuXtNbeNMq8Osk1SY5KUknesDufeVKU9gAAAAAA\nAAAs5iNJzkhyX5IXV9WPVdVMkicn+f3RmEtaa/90zNwPJ9mS5FlVdWpVPSfJiUm+Ojr/htbaifPc\n96VJnpLkOVU1U1WnjBtUVf8lyXsylPy/3Vr7u0nekeGLAY8muaCqvr97H3kylPYAAAAAAAAAzKu1\ndlqS8zOsfP+1qrph57mqeqSq3pjk26O3LhlzicOTnFNVd82Zd2+St45erkrywjHzpjJsiX9RVf2t\nFfZjvCPJ15IcluSzSd4+yvzWqvpvS5jfhd+0BwAAAAAAAGAh542O2zJsVT/OxUmmk9w75twHq+rR\nMe/fPOf50+a57j1Vdf1SQlbV9tbaBRm+QPDM0dtfqKr3L2V+L0p7AAAAAAAAABZy+uh4S1U9MW5A\nVd2Y5MZ55n9jnvfvn/N87Zjzs0k2LSnh/3TP6PGk0fxtuzl/4myPDwAAAAAAAMBCTsxQgI9bRb8U\n3xv3ZlXtmPNyap65u3vPP8iw4v/B0etfaK39i928xkRZaQ/APmvTpt398hxwIFq3bl3vCAAAAABw\nsDtidNze4d5LXinfWrswyT9PsiPJ+Ul+McmvJvlAa+3Gqrp1ZSLuHaU9APucNWvWJEnWr1/fOQmw\nL5idne0dAQAAAAAOdg+Pjsd1TbGA1topSa7IsCPAB6rqT1pr/0+SlyQ5OcnHW2s/vcvq/n2C0h6A\nfc709HQ2b96crVu39o4CAAAAAAAkdyZ5XpITegcZp7V2aJJrk6xJ8mdJ3pokVbWttfYrSW5M8oIk\n70zyG51izktpD8A+aXp6uncEAAAAAABg8K0kL0tySmvtSVX10K4DWmsvSXJehpXur5twvn+b5CeS\nPJLkwqp6bOeJqvpKa+3fJvnfk7yltfafq+orE863oFW9AwAAAAAAAACwT/v06Lg6yWvnGfPmJK9O\n8oKq2j6RVElaaz+f5PUZvizwtqr68zHDLk1yU4ZF7de01tZMKt9SKO0BAAAAAAAAmFdV3ZTkU0mm\nkmxorV3YWptKktbaka21y5KclaE4f/ekcrXWTkzy0dF9/6Sq3j9uXFU9nuSXkzyW5FlJrphUxqVQ\n2gMAAAAAAACwmNck+WKSw5Nck+Se1tqmJPdn2Hp+NsllVXXtMt5zar4TrbVVST6Z5MlJHkzyLxa6\nUFX9WYbftJ9K8orW2iuWLeVeUtoDAAAAAAAAsKCqeriqzkryyiTXJXkiybOTPJTks0leXFVv32Xa\n7OixpxaavyHJTyfZkeRXq+qvlnC9f5Pk/x49v7y19vS9yLZsDukdAAAAAAAAAID9Q1VdneTqJYy7\nIcnqJYwbu9C8qk5eZN6GDMX9klXVjiT/y+7MmQQr7QEAAAAAAACgE6U9AAAAAAAAAHSitAcAAAAA\nAACATpT2AAAAAAAAANCJ0h4AAAAAAAAAOlHaAwAAAAAAAEAnSnsAAAAAAAAA6ERpDwAAAAAAAACd\nKO0BAAAAAAAAoBOlPQAAAAAAAAB0orQHAAAAAAAAgE6U9gAAAAAAAADQidIeAAAAAAAAADpR2gMA\nAAAAAABAJ0p7AAAAAAAAAOhEaQ8AAAAAAAAAnSjtAQAAAAAAAKATpT0AAAAAAAAAdKK0BwAAAAAA\nAIBOlPYAAAAAAAAA0InSHgAAAAAAAAA6UdoDAAAAAAAAQCdKewAAAAAAAADo5JDeAQAAYLc9+Ne9\nEzApq1b3TjB5U1O9E/RxyKG9E0zcY48/1jtCH4cf2TtBFzu29U4weY9OHaRrRQ7C/44/djD+eZ0k\n2x7qnaCPI4/pnWDiHr//+70jdPH4Qfj3sxxyWO8EXRx//kd7R5i4B659Ve8IXaz92ct6RwDmcZD+\n3xMAAAAAAAAA9Ke0BwAAAAAAAIBOlPYAAAAAAAAA0InSHgAAAAAAAAA6UdoDAAAAAAAAQCdKewAA\nAAAAAADoRGkPAAAAAAAAAJ0o7QEAAAAAAACgE6U9AAAAAAAAAHSitAcAAAAAAACATpT2AAAAAAAA\nANCJ0h4AAAAAAAAAOlHaAwAAAAAAAEAnSnsAAAAAAAAA6ERpDwAAAAAAAACdKO0BAAAAAAAAoBOl\nPQAAAAAAAAB0ckjvAAAAAAAAAAAcWFprZya5PslskpOr6o7OkfZZSnsAAAAAAAAA9nuttXVJrk7y\n3CSzVbV6D69zcpKbkhyV5E+r6sXLl/JvU9oDAAAAAAAAsN9qra1O8o4kb0+yR0X9Lq7MUNjPLsO1\nFuU37QEAAAAAAADYL7XWZpJ8LcmGJN/NsCX/3lzvDUnOSLJ979MtjdIeAAAAAAAAgP3Vx5P8RJIv\nJHl+ki/t6YVaa6ckeVeSx5J8IsnUcgRcjO3xAQAAAAAAAFhUa21VkvVJLkxyWpLjkzyQ5CtJLq+q\neVe5t9bWJnlbknOSnJRhJfstSa6sqg+NGX97kmckeUuSjUkuT3JmkrVVNXdx+iNJ3lRVHxjN29PP\nNpVhW/wjk1yWobifCCvtAQAAAAAAAFhQa+3oJF9MclWSszL83vutSY5Ncm6S61pr75pn+klJvpnk\n4gyr1+/MUI6fnuSK1tp7x8yZHT2mknwmydlJ7hndc65zdxb2e+niJD+d5OYk71yG6y2Z0h4A4P9n\n797jLZ/rxY+/9owYzJShSERkvE0nilJOp1SOki6KioRyS3d+nBxETboola5OFye5JN2OSodSJ0SX\no46SnOhN0XErhtw1mJn9++Pz3WZZs/Z9r/Xda39fz8djP9bM9/v97PV+77X2+n739/25SJIkSZIk\nSZJG8yXKWu+3ATtk5hMycyGwDvDZ6pgjIuJVHdp+EbgGeFJmbp6ZWwAbAJdU+w+OiA2Ged6dgMcC\nW2TmwszcrHVnZt4yqayAiNgc+ABl9P8bM/OhyX7P8bBoL0mSJEmSJEmSJEkaVkRsBexBGfn+jsy8\naGhfZi7JzEOAy6pNR3T4FqsBu2TmjS3tFgNHVf+dBWzXod0AZUr8gzKzfYT9lKimxT8VmAMcn5m/\n6cbzjMQ17SVJkiRJkiRJkiRJI9m9eryPMlV9J4cCC4DFHfZ9PjMf6LD9ypZ/rz/M9701My8cU5QT\nczilw8DvgGO7+DzDsmgvSZIkSZIkSZIkSRrJttXjHzJzWacDMvNi4OJh2v/PMNtvb/n3/A77B4Gr\nxhThBETEQkqh/iHKtPhLu/VcI7FoL0kat+uvv77uECQ1yDbbbFN3CJIkSZIkSVLTbUApoHcaRT8W\nd3famJnLI2LovwPDtJ3oc44oImZRpsVfFXh/Zl7ejecZC4v2kqRx23XXXesOQVKDDA4O1h2CJEmS\nJEmS1HRzqsc6RqLf16XvewRlBoHLgA926TnGxKK9JGlCzjjjDBYuXFh3GJIkSZIkSZIkqfvurx7X\nqjWKKRIRTwEWUfLaZ7gp/3vFor0kaUIWLlzolNWSJEmSJEmSJDXDDcBTgcfVHcgUeQ1lWvxVgf9t\nmaK/kxdExPLq3+/LzPdPdTAW7SVJkiRJkiRJkiRJI/kNsDOwWUQ8JjPvaj8gIl4E7A4MAm/rcXzj\n9QBw5yjHzKm+lgL3VtuWdCMYi/aSJEmSJEmSJEmSpJF8EzgamA28BTi+wzGHAzsCv8vMpaOMXq9V\nZh5P5xweFhGLKFPo/ywzd+hmPBbtJUmSJEmSJEmSJEnDyswrIuLrwOuARRFxI3BmZg5GxOqU4vaO\nlFH2H+5VXBGxKjCvbfMaLfvXadu3JDPv63pg42TRXpIkSZIkSZIkSZI0mgOBdYEXAl8BPhURtwEb\nU6aRHwQ+kpnfmMLnHBhl/57AKSO0Xdy27VRg/0nGNOVm1R2AJEmSJEmSJEmSJGl6y8z7M3NHYD/g\nfGAZsClwF/AdYIfMPLqt2WD1NVFjaT84zq+pfv5Jc6S9JEmSJEmSJEmSJGlMMvN04PQxHHcRMHsM\nx3UcaJ6Zm4yh7WnAaaMdNxGZeSxwbDe+dztH2kuSJEmSJEmSJEmSVBOL9pIkSZIkSZIkSZIk1cSi\nvSRJkiRJkiRJkiRJNbFoL0mSJEmSJEmSJElSTSzaS5IkSZIkSZIkSZJUE4v2kiRJkiRJkiRJkiTV\nxKK9JEmSJEmSJEmSJEk1sWgvSZIkSZIkSZIkSVJNLNpLkiRJkiRJkiRJklQTi/aSJEmSJEmSJEmS\nJNXEor0kSZIkSZIkSZIkSTWxaC9JkiRJkiRJkiRJUk0s2kuSJEmSJEmSJEmSVBOL9pIkSZIkSZIk\nSZIk1cSivSRJkiRJkiRJkiRJNbFoL0mSJEmSJEmSJElSTSzaS5IkSZIkSZIkSZJUE4v2kiRJkiRJ\nkiRJkiTVxKK9JEmSJEmSJEmSJEk1sWgvSZIkSZIkSZIkSVJNLNpLkiRJkiRJkiRJklQTi/aSJEmS\nJEmSJEmSJNXEor0kSZIkSZIkSZIkSTVZpe4AJEmSpHFb+mDdEfTe6o+uO4J6DC6vO4Lea+L7G2CV\nR9UdQe8NDNQdQT3uv7vuCOoxu4Hv8Tlr1h1BPZr4Hl9zrbojqMfc+XVHUI8H7qs7gt5bo6HX4k29\nLlUjzH/JR+oOoRZ3nHdk3SHUYv7LPl53CNKoHGkvSZIkSZIkSZJM76AWAAAgAElEQVQkSVJNLNpL\nkiRJkiRJkiRJklQTi/aSJEmSJEmSJEmSJNXEor0kSZIkSZIkSZIkSTWxaC9JkiRJkiRJkiRJUk0s\n2kuSJEmSJEmSJEmSVBOL9pIkSZIkSZIkSZIk1cSivSRJkiRJkiRJkiRJNbFoL0mSJEmSJEmSJElS\nTSzaS5IkSZIkSZIkSZJUE4v2kiRJkiRJkiRJkiTVxKK9JEmSJEmSJEmSJEk1sWgvSZIkSZIkSZIk\nSVJNLNpLkiRJkiRJkiRJklQTi/aSpDGbN29e3SFIkiRJkiRJkiTNKBbtJUljtmDBAr7zne/UHYYk\nSZIkSZIkSdKMYdFekjQuG220Ud0hSJIkSZIkSZIkzRir1B2AJEmSJEmSJEmSJGlmiYjnAxcCg8Am\nmXl9zSFNWxbtJUmSJEmSJEmSJEl9LyK2AU4HngIMZubscbSdC3wSOKDatG9mnj71Ua7Mor0kSZIk\nSZIkSZIkqW9FxGzgGOBoYMyF+pb2LwBOATamzAzQU65pL0mSJEmSJEmSJEnqSxGxEPglsAi4kzIl\n/1jbzomITwLnAxsB3wIGuhHnSCzaS5IkSZIkSZIkSZL61VeBrYEfAlsCPx1H2zcBhwC3AC8F/nXK\noxsDp8eXJEmSJEmSJEmSJI0qImYBewN7AVsBawN/A34BnJiZw45yj4j5wLuBXYANgaXAH4BTMvML\nHY7/M2X0+5HAGcCJwPOB+ZnZOjh9CXBYZn66ajeelGYB5wD7ZebtEbHxeBpPFUfaS5IkSZIkSZIk\nSZJGFBFrAhcApwI7UtZ+vxZ4NLArcH5EHDdM8w2BXwOHUqafvwFYHdgW+FxEnNChzWD1NQCcBbwc\nuLV6zla7DhXsJ+BrmblLZt4+wfZTwqK9JEmSJEmSJEmSJGk0XwK2B24DdsjMJ2TmQmAd4LPVMUdE\nxKs6tP0icA3wpMzcPDO3ADYALqn2HxwRGwzzvDsBjwW2yMyFmblZ687MvGWiCWXmrRNtO5Us2kuS\nJEmSJEmSJEmShhURWwF7UEa+vyMzLxral5lLMvMQ4LJq0xEdvsVqwC6ZeWNLu8XAUdV/ZwHbdWg3\nQJkS/6DMbB9hP2O4pr0kSZIkSZIkSZIkaSS7V4/3Uaaq7+RQYAGwuMO+z2fmAx22X9ny7/WH+b63\nZuaFY4qyT1m0lyRJkiRJkiRJkiSNZNvq8Q+ZuazTAZl5MXDxMO3/Z5jtrWvJz++wfxC4akwR9jGL\n9pKkSbnmmmu455576g5D0gy2zTbb1B2CJEmSJEmS1HQbUAronUbRj8XdnTZm5vKIGPrvwDBtJ/qc\nfcOivSRpwq655ho233zzusOQNMMNDg7WHYIkSZIkSZLUdHOqx6U1PPd9NTxnT1m0lyRN2NAI+zPO\nOIOFCxfWHI0kSZIkSZIkSeqS+6vHtWqNYoayaC9JmrSFCxc6fbUkSZIkSZIkSTPXDcBTgcfVHchM\nZNFekiRJkiRJkiRJkjSS3wA7A5tFxGMy8672AyLiRcDuwCDwth7H19dm1R2AJEmSJEmSJEmSJGla\n+2b1OBt4yzDHHA4cADwrM5f2JKoZwpH2kiRJkiRJkiRJkqRhZeYVEfF14HXAooi4ETgzMwcjYnVg\nEbAjZZT9h3sVV0SsCsxr27xGy/512vYtycz7Rti/dsu/57bvz8zbJxHusCzaS5IkSZIkSZIkSZJG\ncyCwLvBC4CvApyLiNmBjYA6lYP+RzPzGFD7nwCj79wROGaHt4rZtpwL7t/y/fX9r2xOrr1Zdmcne\n6fElSZIkSZIkSZIkSSPKzPszc0dgP+B8YBmwKXAX8B1gh8w8uq3ZYPU1UWNpPzjOr4m2XT6JPEbk\nSHtJkiRJkiRJkiRJ0phk5unA6WM47iJg9hiO6zjQPDM3GUPb04DTRjtuhPajxtcLjrSXJEmSJEmS\nJEmSJKkmFu0lSZIkSZIkSZIkSaqJRXtJkiRJkiRJkiRJkmpi0V6SJEmSJEmSJEmSpJpYtJckSZIk\nSZIkSZIkqSYW7SVJkiRJkiRJkiRJqolFe0mSJEmSJEmSJEmSamLRXpIkSZIkSZIkSZKkmli0lyRJ\nkiRJkiRJkiSpJhbtJUmSJEmSJEmSJEmqiUV7SZIkSZIkSZIkSZJqYtFekiRJkiRJkiRJkqSaWLSX\nJEmSJEmSJEmSJKkmFu0lSZIkSZIkSZIkSaqJRXtJkiRJkiRJkiRJkmpi0V6SJEmSJEmSJEmSpJpY\ntJckSZIkSZIkSZIkqSYW7SVJkiRJkiRJkiRJqolFe0mSJEmSJEmSJEmSamLRXpIkSZIkSZIkSZKk\nmli0lyRJkiRJkiRJkiSpJqvUHYAkqT9dddVVdYcgSZIkSZIkSZLU9yzaS5LGZd68eQDsvffeNUci\nqdFmza47gt679/a6I6jHqqvXHUHvzX5U3RHU44G/1x1B781Zs+4I6tHUvJuoiedrgNXn1R1B7z20\npO4I6tHE6xSA5YN1R9B7g8vrjqAeDzbwd3vu/LojqMXgQw/UHYJ6ZP7LPl53CLW449x31R2CNCqn\nx5ckjcuCBQu4+uqr+fWvf80ZZ5xRdziSJEmSJEmSJEl9zZH2kqRxW7BgQd0hSJIkSZIkSZIkzQiO\ntJckSZIkSZIkSZIkqSYW7SVJkiRJkiRJkiRJqolFe0mSJEmSJEmSJEmSamLRXpIkSZIkSZIkSZKk\nmli0lyRJkiRJkiRJkiSpJhbtJUmSJEmSJEmSJEmqiUV7SZIkSZIkSZIkSZJqYtFekiRJkiRJkiRJ\nkqSaWLSXJEmSJEmSJEmSJKkmFu0lSZIkSZIkSZIkSaqJRXtJkiRJkiRJkiRJkmpi0V6SJEmSJEmS\nJEmSpJpYtJckSZIkSZIkSZIkqSYW7SVJkiRJkiRJkiRJqolFe0mSJEmSJEmSJEmSarJK3QFIkiRJ\nkiRJkiRJkmaWiHg+cCEwCGySmdfXHNK0ZdFekiRJkiRJkiRJktT3ImIb4HTgKcBgZs4e4diNgevG\n8e3Xysy7JxliRxbtJUmSJEmSJEmSJEl9KyJmA8cARwPDFuqHMQj8Cbh3lGOWTSy60Vm0lyRJkiRJ\nkiRJkiT1pYhYCHwF2Aa4DfgdsMM4v82BmXnxVMc2VrPqemJJkiRJkiRJkiRJkibpq8DWwA+BLYGf\n1hvO+DnSXpIkSZIkSZIkSZI0qoiYBewN7AVsBawN/A34BXBiZl44Qtv5wLuBXYANgaXAH4BTMvML\nHY7/M7ARcCRwBnAi8Hxgfma2Dk5fAhyWmZ+u2k0qxzo40l6SJEmSJEmSJEmSNKKIWBO4ADgV2JGy\nzvu1wKOBXYHzI+K4YZpvCPwaOBQYAG4AVge2BT4XESd0aDNYfQ0AZwEvB26tnrPVrkMF+37lSHtJ\nkiRJkiRJkiRJ0mi+BGxPWTf+tZl5EUBEzAGOB94JHBERv8rM77a1/SJwDbB9Zt5YtXsccDawHXBw\nRHwiM2/q8Lw7AY8FtsjM9oI9mXnLFOS2SkTsAbwM2ASYDdwI/Aj4SmY+MAXPMSxH2kuSJEmSJEmS\nJEmShhURWwF7UEa+v2OoYA+QmUsy8xDgsmrTER2+xWrALkMF+6rdYuCo6r+zKMX7dgOUKfEP6lSw\nn0LfAL5Gmfb/OcCzgVcDJwEZEU/r4nNbtJckSZIkSZIkSZIkjWj36vE+ylT1nRwKHAR8uMO+zw8z\nWv3Kln+vP8z3vTUzLxxTlJPz/yij7FerHo8GlgAbAedFxHrdemKnx5ckSZIkSZIkSZIkjWTb6vEP\nmbms0wGZeTFw8TDt/2eY7be3/Ht+h/2DwFVjinD87gCOqZ7jq5l5fcu+64GPRMTlwLnAupQi/sHd\nCMSivSRJkiRJkiRJkiRpJBtQituLJ9j+7k4bM3N5RAz9d2CYthN9zhFl5t3AcaMc84OI+BHwYuB1\nWLSXJE1XV13VrU5ukgTbbLNN3SFIkiRJkiRJTTenelxaw3PfV8NztvoxpWi/TkRskJk3TfUTWLSX\nJE3YvHnzANh7771rjkTSTDY4OFh3CJIkSZIkSVLT3V89rlVrFPW4o+Xfc7vxBBbtJUkTtmDBAq6+\n+mruueeeukORJEmSJEmSJEndcwPwVOBxdQdSg3Vb/n17N57Aor0kaVIWLFhQdwiSJEmSJEmSJKm7\nfgPsDGwWEY/JzLvaD4iIFwG7A4PA23oc37hFxInAVsClmXnYCIc+v3q8PjNv60Yss7rxTSVJkiRJ\nkiRJkiRJM8Y3q8fZwFuGOeZw4ADgWZm5tCdRTc7awHOBN0XEpp0OiIhnAy+idEQ4o1uBONJekiRJ\nkiRJkiRJkjSszLwiIr4OvA5YFBE3Amdm5mBErA4sAnakFLc/3Ku4ImJVYF7b5jVa9q/Ttm9JZt5X\n/ftDwG7AmsD3I+LAzPxZ1W4AeCXwRWAAuA44YeozKCzaS5IkSZIkSZIkSZJGcyBlffcXAl8BPhUR\ntwEbA3MoBfuPZOY3pvA5B0bZvydwyghtF7dtOxXYHyAzfx8Re1ByWQBcHBF/paxbvwGwFiWnBHbJ\nzDsnksBYOD2+JEmSJEmSJEmSJGlEmXl/Zu4I7AecDywDNgXuAr4D7JCZR7c1G6y+Jmos7QfH+fWw\nzDwbWAh8DLiMMkp/c+BB4ALg7cBWmXnNJHIY1cDg4GR+RpKkacIPc0kz2Uq9aVff7ojmfe49tKTu\nCOqx6up1R9B7sx9VdwT1WL6s7gh6b86adUdQj9lO+tcYs2bXHUE9lvXD0p1TbNlDdUdQjyZepwAs\nub/uCHpvzhqjHzMT/f3euiPovbnz646gHqusWncEvXf/XXVHUI+GXovfce676g6hFnNWGXWEuqYR\nR9pLkiRJkiRJkiRJklQTi/aSJEmSJEmSJEmSJNXEor0kSZIkSZIkSZIkSTWxaC9JkiRJkiRJkiRJ\nUk0s2kuSJEmSJEmSJEmSVBOL9pIkSZIkSZIkSZIk1cSivSRJkiRJkiRJkiRJNbFoL0mSJEmSJEmS\nJElSTSzaS5IkSZIkSZIkSZJUE4v2kiRJkiRJkiRJkiTVxKK9JEmSJEmSJEmSJEk1sWgvSZIkSZIk\nSZIkSVJNLNpLkiRJkiRJkiRJklQTi/aSJEmSJEmSJEmSJNXEor0kSZIkSZIkSZIkSTWxaC9JkiRJ\nkiRJkiRJUk0s2kuSJEmSJEmSJEmSVBOL9pIkSZIkSZIkSZIk1cSivSRJkiRJkiRJkiRJNbFoL0mS\nJEmSJEmSJElSTSzaS5IkSZIkSZIkSZJUE4v2kiRJkiRJkiRJkiTVZGBwcLDuGCRJkiRJkiRJkiRJ\naiRH2kuSJEmSJEmSJEmSVBOL9pIkSZIkSZIkSZIk1cSivSRJkiRJkiRJkiRJNbFoL0mSJEmSJEmS\nJElSTSzaS5IkSZIkSZIkSZJUE4v2kiRJkiRJkiRJkiTVxKK9JEmSJEmSJEmSJEk1sWgvSZIkSZIk\nSZIkSVJNLNpLkiRJkiRJkiRJklQTi/aSJEmSJEmSJEmSJNXEor0kSZIkSZIkSZIkSTWxaC9JkiRJ\nkiRJkiRJUk0s2kuSJEmSJEmSJEmSVBOL9pIkSZIkSZIkSZIk1cSivSRJkiRJkiRJkiRJNbFoL0mS\nJEmSJEmSJElSTSzaS5IkSZIkSZIkSZJUE4v2kiRJkiRJkiRJkiTVxKK9JEmSJEmSJEmSJEk1sWgv\nSZIkSZIkSZIkSVJNLNpLkiRJkiRJkiRJklQTi/aSJEmSJEmSJEmSJNXEor0kSZIkSdIMFxEbRcSG\ndcchSZI0FhHxxLpjkKReWqXuACRJkqTpIiJWAwJYDbg6M+8aY7uDgMdn5vu7Gd90ERFrAp8FBjPz\ngLrjmUoR8VjgZcCWwFzgDuC3wHljfT9MZxGxEeU9/hfgysxc3rZ/D+B1wPrA34CLgJMy845exzpV\nImJT4MnAzcBVHXJ+I7AbsB5wK3ABcHJm3tPrWLshIp4M/AOwAeU9PRu4B7gd+D3lZ7K0vggnJyLe\nCzwIfCkzb6s7nmnuz5TPtHVqjkOasIhYHXgCMA9YA7gfuBu4ITMfqjM29UZErA2sC9yamX+rOx51\nT0SsBTwe+Ftm3lp3PN0UEdsArweeRrleux24FDgzM6+uM7aJiIhjgfdl5uAkvsfrgROBtacssGmo\nupZdkpkfrTsWdUdEDP0ddrt/r2g0A4ODE/7clCRJkmaE6gbwx4E3AqtXm5cBXwWOzsybR2l/GbBV\nZs7uaqDTRESsAyymFO37KufqRu9ngZsy819btg8AxwL/Aszp0PR+4GPAB9uLvv2ges1OA3Zu2fwn\n4K2ZeX5EzAL+E3hJtW8AGPpjcTHw8sy8tFfxToWIeAJwBvD8ls3XAa/LzEsjYjbwH8Au1b7WnP8K\nvCIzf9OreKdSRMwD3gXsA2w8yuH3U177z2fmT7sd21SLiOWU1+0O4IjMPLnmkKat6md1Z2b2/c3v\nqvPYq4GnUzraXUkpbIzawSgizgW2yMwndzfKqRER11I6pnw0M79cdzx1iIhtgX2BHYDN6Dxz6EOU\n98G5wKmZ+aeeBdgF1XXJ9sCOjNzx6lLgnJnQ0azqZPZGVnS0+0FmXtCyf3Pgi5Sfy5ArgA9l5rd6\nGetUqjqMPiEzf9e2fQ5wIOWzrr0z6VnAVzLzgR6HOyWq0dP7saIj6TmZ+ZOW/QF8CngR5foM4BrK\ndfgZvY128iLi28C1mfmuDvtWo7yv92nZ3HpNOkj52+Vdmbms27FOleqa42fAXpl5wzjbzgM+R+nE\nQL/9vTleM+n6bCJa7i0sz8y+GWQcEY8GlmXmfcPsXw84mtIhvrXD7PWUv1FPyMw7ux6o+o5Fe0mS\nJDVadVP0AsoNwIG23YPAncBumXnRCN/Don0fiIg1gF8BCykj51/Wsu+LlBujQ++B+yiv/WMoN0mh\nvB/OyMw39izoKRARqwCXAFuz8nv8fuAFwIuBD1JyvA64DdiIMroJ4CbgHzLz7h6EPGnVDdDLKDeD\n23NeDDwF2Bv4ZLXtWuAW4InA0PThNwNP7bebKRGxNaVotR4r5w5wL/AA8GjgUS3bBykdlQ7op9Gq\n1Y3OZZRC1iDwc+DNmXlVrYF1WUS8YQLNTqX8zr+tdWNmnj4VMfVKRPwTpcPNum277gTek5mfG6V9\nX52zWzqmQCmAvDkz/1BjSD1TFW5OAnavNnX6TGs3SPlM+DRwZD8VuYZExGuB4+nc6ar1ZzD0vniQ\nUgB4T2b+tcvhdUU1ovbLPPK8BKVIu6ha2uNS4HF0vl5/b2Z+qPuRTq2IeCVwCnBWZr6pZfuTgXOA\nzatNnV73PwCvzMw/9iLWqRIRuwBfY+VOsl/OzDdFxMbAr4H5dH6t35eZH+h+pFOn+hz/bWZu02Hf\n1yifcQPAcsrregflHLcZKwr4J2fmQT0LepJazl13AQdl5n+Msd1zgK8AT6Lk/vPMfF634pwOLNr3\n7b2F5cBlmfmMDvueBpxH+T3udO0ySJkB6yWZeU0341T/6ZueK5IkSVKX7MWKkbjnAd+gjNZ6JmUE\nyHzgvIh4TWaeW0+ImiIHUIq191AKPgBExD8Cb6L88Xw28AHKH+CD1f6nAIdUx+wdEd/ss/fCPsA2\nwN8pM0pcSrkpvhOlo8LhlPf7DcAumXn5UMOI2A74OqWYfSDwiZ5GPnH7AVtQXtOTgB9Rbpi8E3ge\n8FbK7/69lJH33x9qGBE7UYrX6wMHAX0zVWU1Wu+HwGMpozg+S1ni4DbKa3gk8FxK/t8GNqX8PPak\njObci9JR5ZW9jn2SrgDeC3yekt9vI+LfKMWemTp18qmsKNyMx+qUAtGQQaBvivYR8XjKzBCPofxO\nL6UUOValnK8/GxFbAW/rx1lRRrCY8hn+PMr7++OU0cV/rzes7qk6nJ0HbEd5rf8I/JhS0LqR0gHl\nAUrhby6lwLMV5bNsPeAwStF7d/pINU3yIh5ZwLuN0qFsU8qo5K8Da1E6pj2DMtvE/sArImLnzLys\nhtAnrCpQf4nyewzlOg3KEgjHRMT3KefjdSmd7E5lRUe711N+Lu+LiO/3U+4RsQXl745VKcXZoe2P\nonS+GyrY/y/wU8qyRWsBz6Zcty0EfhQRWw430nO6qUbYn8GKmc3uoFyTzgX2j4hLKZ1J16YU7s9m\nRUfSV1BmnXhvRHw3M6/obfRTLyKeB+xBORd/nTKa/uaW/esDx1Cu2w6IiNMz82e1BDt+uwNfoLyW\n34iILwMHD3feqmb9eh/lWnUVSmek99FH1+EAEXHB6Ed1NLet7WBm/vNUxKSuWqkgX3UeP5tyLQLl\n79Afs6IzzvMps4hsAnwvIrbqpw7T6j6L9pIkSWq6PSk3Sj6RmYe3bD+zuin+LeAfKTcbdsjMX9UR\npKbE6yiv9f6ZeVbL9r2rx29n5mvbG2XmlcCbI+Jmyo30gyg3U/vF7pS8922bPvbbEXE7cET1/51a\nC/YAmXlJROxPudHwCvqnaL8bJecjMvPjQxsj4j8pM2u8nnIz/LDWgj1AZv4wIg4HTgZeRn/dLDyc\nUrD/GfDSzLy3Zd+fgZ9GxJmUIu0VmZmUQtgpEfFiyg3jl0fEa/ttquHMPCcifkIZnfoWSkeb/SLi\nU8CJmXl7nfF10TLKTBhjsTHl92Jc09ROM2+nFK0WUzrhnJWZy6pC/THAaygdrB5F6ag1U9xMKdgf\nR/kZHAXsWxV4T5nMmsHT2IGU66/bgDdl5tljaVQtfbIfZSaVV0fEbpn57e6FOXUiYntWFOw/CRzf\nuo53ROxBOTdtmJl7VNtWp5zz3kM5r50TEU/JzLt6Hf8kvJXS+eJPwD7VtccA8Gbg3yjXXbtRZtB5\nfuu5LSKOA/4LeE513Ft7HPtkvJVSsP8F5Rp1yD6U1/J+YO/M/G57w6rD6X9QPtcPBj7c9Winxtso\nBfpfAXtm5nXwcD7fpHy+PYny99eerZ2vIuIYygj911LeG+/oaeTdsUf1eG5mvr59Z2b+BXh7RCwB\nDqV8LvZF0T4z/yMifkbpKLgTpWPRP0XEnu1/b0TEppQOs8+ifP5dQfks+B395wWs6FQ5ltlhhqxS\ntR0yE8/rTbEPpaPREuDVmfmDtv0fjohnA9+jfNYfQOngIgEW7SVJkqRtKEWP97XvyMybI+KfKQXL\n5wBnR8Szxrsu33QTEX03VewUeTLwt7aCPZRRO4OUEfYj+QilKPqsLsTWTU8H7hqmCHsSZUTLnZn5\n406NM/OCiLiDMktBv9iKMmPGp1s3ZuaDEfEx4DuU3/tTOrQFOI0yanthN4PsgpdTRma+sa1g3+pI\nyk3i/0dLcSMzfxQRb6LcKN+veuwrVc5vj4ivAp+hfL6/Fzg8Ir5OWf932KVO+syRwLGUgs81lA4o\n/ztSg2oaz7szc5MexNctL6F8Xh+QmecMbaxu7O8eEYdQOhftGxE3ZuaimuKcctVI2kOqjjf/Djy1\nenx31cnwtBk28n6oo93emfmjsTaqpsP/UkTcTemItD9lZpF+MFSEXNRp+u/M/EZErA2cGBG7Z+Y3\nq9f8qxHxXeAHwD9RPt+P7VnUk/fPlNd6v8y8BKDqiPKFam3zA4A1gX9pP7dl5pKIeDtlnfd+m0L7\nxZRz9hsy85aW7S+l/DyO6VSwB8jM/46IvSgdEXejf4r2L6LkttdQwR4ezudQSuF+kDLi/BGzpWTm\n8og4jNI56/nMDE+h5PvBUY77ACtmi+ob1XIdO0fE2yidYLcALomIozLzUwARsS/len0u5WfxMcpS\nH/068vh+YA3KTEBnUpYdG80iSoH3+C7G1TUTXLKp1dzRD+krL6a8l9/foWAPQGb+MiLeTLk+eQ0W\n7dXCor0kSZKabm3g/4abVrK6GbgL8N/AAkrh/rmZeX8vg5xi4+n1P5M8hjKtaruhtfNGXAM7Mx+I\niP+jFP/7ydrA1cPsG+qAMtpI3ZtZMU1rP5hP+b3udMPv59XjrZl5d6fG1Y3hP1OmLewnGwHXtd4I\nb5eZ10fETZQb5+37zqpmX1hpzdV+kpm/AJ5ZrZG8iPLZvR9l5P1iSkesi4DfA0npzNNXI5oy86PV\nzBGnUaYDvzQiPgB8eIZNC99uAXBva8G+VWZ+uhp5fBxlWu0rM/MbPY2wy6obvdtQRqweQ5ka/N+A\n4yPiLMro25/0+XUKwJbA4vEU7Nt8i/Jz2XbqQuq65wD3MfIML/9O6ZiyH6XACZROHRGxH+V8vyv9\nVbTfhNK5sNMI4tMoM6f8nfK5vZLM/F1E3Ek5B/aTDYDrM7P92nSD6vFrIzXOzJ9ExF8pn4v94snA\nDZn5pw77hjqP3jBcB+nMvKm6Fn9itwLssTWrx8tHOigz74yIaynLZPSdzPxcRPyYslb9tsAJ1QxP\n91E6nQxQ/kZ7Q3UN18+2pHQK3p7S+ewE4H0jdUKIiEXAkszsp8/tVqfizACthn5PzxzluLOBOymd\nMKWHzao7AEmSJKlmSyjF3GFVayK/DLgdeBqj/wE23V1J+cP6AspN3fF+faz3IU+JvwAbRcSctu1/\nrh43YATV+rrrU94H/eQBVqwd2m7NtsfhzKX8rvSLBxk+p6HXb/Eo32MOpUgwEy0BnjDMvtsZ5TOx\nX2TmmZRRXa+ldNYYoKwluSdlRMtPgVuBpRFxz3DfZ7rKzKso630fU216P2UEW7/NEDEea1A+y4eV\nmR+hFDYHgC9HxDN7EVgvZebSzPwMpQD2XsoU8vOANwL/CfwtIn4eESdFxL9ExL4R8ZoaQ56IuUzi\nfFt1xLmV/vo8exylYPnAcAdk5lLKdcvWHfb9iTKqs986nK0B/HWYfX+sHv8ySueqvwCrTWlU3beM\nMtK+3VCHwrEscXA3/TUob01W5NfunrbH4dzH8Ne1/WZo+Yux5DNAH9dyMvNqSsekYynv/Z0oBXuA\nLwJPmwEFezLzusx8AWXGk6WUmZF+U3W2m+kGJvE1kwzNanjzSAdV57SbKJ3NpYf17Qe9JEmSNEWu\nBdaJiJVufrbKzD9Spi5bCrwiIk7qRXBd8gbKH5NbAV/IzFFmaPwAABwtSURBVGPH80V/rfHd6seU\nG8PvbNv+LcrNgpeM0v7NlLWUL5360Lrq/4AnRcTGHfbtUD1uHBEdOy1U2zcGbuxSfN1wHfD4iFhp\n1F11g+R3jDCzQkQsoIzYG+ta4dPFjcAmETHsCLRq36Z0uCkeEWtRXusRi6L9JDMHM/OszNyeMlvE\nscBvqt2tNwvXqCnEScnM5Zl5HPBMyvTQz6TcHD6iWhN6prkDeGJEjHY/623A+ZRCyNkj/U70s8y8\nJzM/SPm9PZAyK9AAZdmEf6RMK/5RyjroX68rzgm6jfJarzqRxhGxGuVzfLQOWtPJPZTC/WjmA48e\nZt+q9N+Ix7sYpuNky3T4wxV6qT7r1mdsRe7p5I90vj77bfU44rJEEfF4yvm8n65V7gIeP8y+offA\ncPuHPIER3g99Zmj2p2ePdFBEPJbyWt8y0nHTXbV8yXdZ8TfFAGWk8VnDzXrXr6qOdU8HfkFZiu2S\niPhA1Ql8pjmZ8lp+i9JpbLxfz+h9yF01NOPZcB2kW63DzO0krgmyaC9JkqSm+yHlj8yTImLEG6XV\nWsj7UUbFHBARZ1bT8PaVzPwN8CHgsZTp7JriU5R1zo+LiCMj4lEAmfk94DzgQxGx0nr1ETE3Io4C\nPkm5Ef6lHsY8Fc6n/O13crUWLgARsRVlit3rKOsvfmyYQtjHKHl3mrZ2uvoJ5ff6U51ujmXm0zNz\nz04Nq+M/S8n54m4G2QUXUl7rL3QqdFVFrDMoP5v/bts3i7IO/GqMslREv8rMP1adj7aljLh/EXAY\npajZzx2xqNazfxbwPsp74DjgFxHRT8tajMVvKbNg7DfSQVVh4DWU5Q/WB34SEU/qenQ1ycwlmfnl\nzHwupbDzTsqI+5vp35Fsv6KMyn3HaAcO411V+5+PduA0ciWlI+mwnQgj4lWUz6+Vpg+vOqBu2Gnf\nNHcFMDciXjrM/kOBj4/Q/jWUTpX9du76FuXz+t/bztknU65X3z5cw+pa5STKEk8/7GaQU+x3lPf4\nKzvs2796XDsiXtipcbV9Pv33Wg/nFErB7phROtp9nPJa/7InUXVJRBwGXELpaHY3cA3l9TwvIk6Y\naCet6arq9L895Xz0EPBuynJGT6s1sKl3KKWT+GuALTLz/8bzBVxfb/hT7lzKNderRjooIp5L6aTU\nabkQNdjA4GC/db6UJEmSpk5EPIFy42cu8Dfge8CBI03BGRG7UabIfxTlj8y5wNqZOXu4NtNNRAzd\n+NkaOCwzPz2OtutQRq4N9lPOABFxAGUKxgHKlJTfo4y6vYNS7FoA/Bcr1pbchDJ946OrNl/JzDf2\nNurJqUaNX0F5v94P/IFS9FpIyekIyvvgdZSbqadTCj1Potx8eQalgL1t1eFj2ouILSg5z6K8lp/O\nzNNGaTMfeAVlOsunUzrnPCMzR1xndDqppkW/nHJj9xrKFOFXUApXTwHewopRHztm5oVVu60po3A3\nq/btWnVmmfYiYjnw28xswrSjY1LdDD6NMpvKEuA9mXlC9bO6MzPXHvEbTGPVmt0nU4oc7wS+OtJU\n4hGxIWWWlc0pM0i8idJpbat+OX9N9j1efbatBzwmM/um4BMRO1NufC+ndKQ6ITNHnfElIjalnNcO\nrDZtn5l9UbivClofp1xj7ZGZP2nb/wpKZ8u1KOe1w1r2bU/5vd8IOCkz39qjsCctIg6hdIy8CXhV\nZv56HG23A75NeY8fmZl9s4RT1fH3csoyF78DFgE/yMyHIuIg4HPA54EPZuYtVZs1gVcDh1PO6/cD\nW2bmn3ufwfhFxDsoHQT/Rlna5aeUa9LXAwdTrsE3p1y/7ZGZ/9PSdivKGtAbAUdXS6H0hepz/EE6\nT5f9eEqHybdn5hfa2j2XUuh9CeVa/MWZeX6Xw51yEbEe5fPpRZS/PX5JWaroVuDfKEu7DFKuWffK\nzN/XFGrXVJ0oT6Usa/QQpXPlBzNz2Qy5PnsBpaP4LZTPpDEvb9Ov9xaq1+2PrLjeGLIK8I3q38/I\nzJU6JVTXKt+n3Hv4WGYe2c1Y1V8s2kuSJKnxImIX4JuUKUXvz8y5Y2jzYsqI1cdWm/rqj0x4uMD3\nVUpRZ9ehG4JjaNeXf1gPiYgXUW4YBitPITswzLYHKaPSjx5lTdVpqaWzQvtI+vMoheoNgMsoo11a\n8xsa9XNCZh7e7TinUkQcCpxAyefczNxllOP3ptxQhJL3RzLz3d2NcupFxMGU4sdw72WAD2TmopY2\nz6SMagU4PjOP6nqgU8SifWfVKMz3AEdROnFcBLyA/r8pPJsyS8QzKe/v+yjF6JE62j2OUrjfkpbf\niX45fzX5PR4Rn6csTTP0uiVlNPqNlGLlg5RrtzUpI8yfyorORwOUgsh7exnzZETEXErRamNKzley\nouPVQkpxd4CydMCWLYXcnYFzqn13AVv3SxEXHi5e/wF4IiXvMzNzn1HabEcpej2XUiC5hTLCs6+m\nyI+ITYAfUArVQ59pv6e8x7ekFHQGWbHMw2Mp13IDwAPA6zLz7B6HPWERMYfSQWEzVr5GGVrn/NmU\n13Y55Wcx1JF0c0reiymv9R29iXryqs/x0RxTLXfT2u4ySge8AeC0zBxxlpnpqOpsdDJlGnAoM3gd\nk5lLW47Zi9JJZR7l79J3Z+aneh1rt1WzKRxOWappVcrsQfsA/0ufX58BRMQngUOAc0b7u6utXV/e\nW6h+r0e7L3BMZn64rd0nKJ1I1wTuBRZmZj8tc6Ium4lraEiSJEnjkpnfq4pWi1hxs3e0Nj+KiC0p\nN5X2oRRF+kpmXgVMpAhwD6NMTTydZeZ/RcRTgR2B3YBtKTfDV6sOGSps3km5Wf4DymjOfptu9mGZ\neXJEXE6ZZjiAv1KmTj41M5cD10fE8yhTrT6npektwIerdRn7SmZ+MiKuoYzkum0MTf5Kee1vB96f\nmZ/tZnzdkpmfiYg/Au9n5d/vyykF+2+3bb+KMuvEZzLzgh6EqS6rboYviojvUjqjvKDeiKZGNSLt\nZZQ1cf8ReHC0jlSZubgarfg5YK9qc991vmqizHxrRFxFKXA8BtiCcg4bztD5+xbgXzPzK10OcUpl\n5r3V+/tcSuH+H1ixrvlQbjezckfLa6r911JGqf65NxFPjcz8e5X3OZRR1GNZemoDVnyu3Q28tt8K\n9gCZeV1EPB34F+BtlOU8nl19DbJiaYv1WpoNUjpdHpGZV/Q24snJzCURsRPwHUoxeshdwMGZeWFE\nXESZUvwllI4LT2XF+/8eYPd+KthXjh3DMZ2WZPozpbPCJygzgvWNqoPGJ4GDWDHD2Rsy80ftx2bm\nVyPil5TRyVsDJ1SdkfbNzL/0MOyuqq5XPhoR/0kZdb8tKzrNzgRHUc5DsyJi3cy8te6AemC0pYc6\n7X8mpWD/ILCPBXu1c6S9JEmSNEnVOuHzqjXZ1KeqEW7zKDdD78nM+2oOqRbVus/rU5YMyH6cWaBd\nRAyMlkf1e7wl8IvMfKg3kXVXRKxLKfwAXD/W2TT6SURsTCnczpibulMtIh5Fudm/J3B3Zj693ogm\nrxqttjPwtPYRTKO0ex5wGLBWZnZcM3m6iYifAFdn5kF1x1KXaiT2bsALKYW+DSnn69UpI+7voayn\nezllfe9z+/lzvMp3f+CllBHGUPL7IXByZt7bdvwsYBfK6Mal9KmIWAN4A+Uz/cujHPscymxR36NM\nLTzq0gnTXfW5th2lkPcPlJHJD1+XAjdQOpP+MDM7TbPeN6r37Pas6Eh6cXshPiIOBPZmxTXpRZTO\nhY0pcEXEU4Cb+rFDSkRcSXl9ByjTpu892nVodb3yccryN4OU1/3NmXlWl8Ptuep34AjKoIFVmQEj\n7Zum+htkNHdk5t1t7U4E1gY+NBOXgtDkWbSXJEmSJEmSJEnSpFVThy8F3ktZemnMRahqSv0vUzqu\nLM/MGTtbdNUx4zXAA5l5fN3xSKqfRXtJkiSpRXWD4drMHNM0+TNBE3MG825S3k3MGZqZdxNzhubm\nLUlSv2nCOTsirgP2zMxLJth+A+BrwD/10zrnkjRZM7aXkiRJkjRBdwB9O6XqBDUxZzDvJmliztDM\nvJuYMzQ373GJiDWBzwKDmXlA3fH0QhNzhmbm3cScwbxpUN4zKOcmnLO3zsw7J9o4M2+KiBdQpo9v\njBn0Hh+zJuYMzc1bo5tVdwCSJEnSNHMR8OSI2LTuQHqoiTmDeTcp7ybmDM3Mu4k5Q3PzHq85wL7V\nV1M0MWdoZt5NzBnMe996w+ipmZLzjD9nT6Zg3/I9lmdmo4r2zJz3+Hg0MWdobt4ahUV7SZIk6ZHe\nAfwa+GFEvLDuYHqkiTmDeTcp7ybmDM3Mu4k5Q3PzliSp3zTunB0RyyPij3XHIUnTndPjS5IkSY+0\nN/BdYCfgxxFxK3AVsBi4b4R2/TytWRNzBvNuUt5NzBmamXcTc4bm5i1JUr9p4jm7CUsCSNKkWbSX\nJEmSHukjwGD17wFgPWDdUdoMVG369SZKE3MG84bm5N3EnKGZeTcxZ2hu3pIk9ZsmnrMvAl4eEZtm\n5rV1ByNJ05VFe0mSJOmRrmfFTZSmaGLOYN5N0sScoZl5NzFnaEjeEbGs7hh6rYk5QzPzbmLOYN5N\n0sSch9GIc3abdwDrU5YEOCgzL6w7oG5o4nu8iTlDc/NW91m0lyRJklpk5pPqjqHXmpgzmHeTNDFn\naGbeTcwZGpX3QN0B1KCJOUMz825izmDeTdLEnFfSoHN2q6YsCdDE93gTc4bm5q0us2gvSZIkSZKk\nfnAlsBC4EPjpBNqvARw+pRF1XxNzhmbm3cScwbyblHcTc1bRlCUBmvgeb2LO0Ny81WUW7SVJkiRJ\nktQP3gBcAmwF7JWZt4yncUSsQ//dIG1iztDMvJuYM5h3k/JuYs4qmrIkQBPf403MGZqbt7rMor0k\nSZLUQURsAGxHWXtvPmOY/iwz39/tuLqpiTmDedOgvJuYMzQz7ybmDDM/78z8TUR8CFgEnArsXG9E\n3dfEnKGZeTcxZzBvGpR3E3MeyUw/Z7dqypIATXyPNzFnaG7e6j6L9pIkSVKbiDgBOBiYNc6mfXkT\nBZqZM5g3Dcq7iTlDM/NuYs7QqLw/CLwCeHFEHJKZn647oB5oYs7QzLybmDOYd5PybmLOK2nQObuJ\nmvgeb2LO0Ny81UXjPSlIkiRJM1pEvAU4FJhNGe0wnq++1MScwbxpUN5NzBmamXcTc4Zm5Z2Zy4B9\ngMuBPSJivZpD6rom5gzNzLuJOYN506C8m5hzuyads5uoie/xJuYMzc1b3eVIe0mSJOmR9qseLwGO\nAq4Ebs/M5fWF1HVNzBnMu0l5NzFnaGbeTcwZGpZ3Zl4FbDOBpvew4mfVV5qYMzQz7ybmDOY9gaZ9\nm3cTc27TqHN2q6YsCdDE93gTc4bm5q3uGRgcHKw7BkmSJGnaiIi7gNWBDTJzcd3x9EITcwbzpkF5\nNzFnaGbeTcwZmpu3JEn9pqnn7IkuCZCZs7sTkSRNP460lyRJkh5pFeDaJt1AoZk5g3k3Ke8m5gzN\nzLuJOUNz85Ykqd807pzdsiSAJGkEFu0lSZKkR7qRcfb+nwGamDOYd5M0MWdoZt5NzBkamndELKcU\nPjarO5ZeaWLO0My8m5gzmHeT8m5izpUmnrMbuSRAE9/jTcwZmpu3pl7TTg6SJEnSaL4HbBoRG9Yd\nSA81MWcw7ybl3cScoZl5NzFnaG7edwAP1R1EjzUxZ2hm3k3MGcy7SZqYMzTznL0FsBR4ZWZelJmL\nZ3rBvtLE93gTc4bm5q0pZtFekiRJeqTjgP/f3v2F+l3XcRx/nm1gYDD/LUvFwqR3V7spJbxQJqsu\ngiUEG5kEkoQXYnjT1lUY+If+XAmKaBCJYpNBeFfawm4MJBOF8o1QOIXAlYKwP8L0dPE74tk8Zzvq\n9/f77vd7PR8wzrbz+f14P+WD3/H9cH7fQ8Cvq+qcsYeZkcRmsDupO7EZMrsTmyG3+xngi1V1xdiD\nzFBiM2R2JzaD3Undic2Qec2OeyTAisQ9ntgMud0a2NLy8vLYM0iSJEmjqKrL1/nWZcCDwDnAA8Bz\nwNtner/ufnG46aYjsRnsXsPCdic2Q2Z3YjPkdq+lqi4BDgAXAT/s7j+PPNLUJTZDZndiM9hNUHdC\ns9fsiapqYKm7vzT2LLOUsMdPldgMud0ans+0lyRJUrJ/b2DNLzf4XsvMx7+vE5vB7tNZtO7EZsjs\nTmyG3O613AT8Hvgm8HRVvQH8EzgMHDnN65a7+wczmG8aEpshszuxGexO6k5o9po98SRwR1Vd1t2v\njz3MDCXs8VMlNkNutwY2r/+TlyRJkoawNPYAI0hsBruTJDZDZndiM+R2r+VeJocYMPnvcjHwmTO8\nZmnlNfN6gzSxGTK7E5vBbsjpTmj2mj1xN/AdJo8E2NXd74w90Iwk7PFTJTZDbrcG5qG9JEmSku0Y\ne4ARJDaD3UkSmyGzO7EZcrvXcogPbpCmSGyGzO7EZrA7SUJz3DX7NI8EuInJIwFeqqqFfyTAioQ9\nfqrEZsjt1sB8pr0kSZL0MVTVlu4+MfYcs5TYDHaPPccsJTZDZndiM+R2S5I0b+b1ml1V7w74dsvd\n7Q+eSoqxaewBJEmSpLNRVd1QVc9W1a51ltxfVS9U1Z6ZDjZFic1gd1J3YjNkdic2Q263JEnzZoGv\n2UsD/5KkGB7aS5IkSaeoqruBA8DVwLZ1li0B24HHququWc02LYnNYDdB3YnNkNmd2Ay53ZIkzZsF\nv2bvGPDX9TOeXZJG5UeLSJIkSatU1Q5g38of/w68ss7S/cDngZ3Avqr6Y3c/M4MRB5fYDHav/DGi\nO7EZMrsTmyG3+31VdSnwNeBzwPls4Cfzuvtn055rmhKbIbM7sRnsJqg7rXnRr9mfZMZ5fSTAmaTt\ncchshtxuDcdDe0mSJOlkt618va+7f7Teou5+Cniqqu4FfgzcDpz1N1HWkdgMdid1JzZDZndiM+R2\nU1W/YtLxUT9Ncm5vkCY2Q2Z3YjPYTVB3YjPZ1+wbgL3APd395BpL7q+qq1e+/7vZTjcdiXs8sRly\nuzUsPx5fkiRJOtk1wDEmNxM24k7g+Mrr5lViM9id1J3YDJndic0Q2l1VtwJ3AJsJeUZuYjNkdic2\ng90EdSc2r0i9Zi/yIwHWlLjHE5sht1vD8yftJUmSpJNdAPyru49vZHF3H6uqV4ErpjvWVCU2g91J\n3YnNkNmd2Ay53TevfP0r8BPgH8D/uvu98UaausRmyOxObAa7k7oTmyHwmr3ojwQ4jcQ9ntgMud0a\nmIf2kiRJ0sneBLZudHFVLQEXAW9NbaLpS2wGuzdkQboTmyGzO7EZcru/DJwAvt3dh8ceZkYSmyGz\nO7EZ7E7qTmyGzGt26iMBEvd4YjPkdmtgfjy+JEmSdLIXgYur6roNrr8RuBB4aXojTV1iM9id1J3Y\nDJndic2Q272FyU8rJt0cTWyGzO7EZrA7qTuxGTKv2ZGPBCBzjyc2Q263BuahvSRJknSyR5k8V+yJ\nqtpdVZvXWlRVW6tqL/AQsAw8MsMZh5bYDHYndSc2Q2Z3YjPkdr9O3r2txGbI7E5sBruTJDZD5jX7\nAuDQR3kkAPDqyuvmWeIeT2yG3G4NbGl5eXnsGSRJkqSzRlVtAg4C1zK5OXIEeBk4zOTjzs4DtgFX\nApuZ3HA52N07Rxl4AInNYDdB3YnNkNmd2AzR3b8A7gC+0N2vjz3PLCQ2Q2Z3YjPYTVB3YjNkXrOr\n6j/AcndfssH1S8AbwLvd/dmpDjdFiXs8sRlyuzU8D+0lSZKkU1TVucDDwJ5Vf736H85Lq37/OHBL\ndx+dxWzTktgMdhPUndgMmd2JzZDZXVXnA38DXgF2dfc7I480dYnNkNmd2Ax2E9Sd2Py+tGt2Vf0B\n2Alc391nfEZ9VX2PyScL/Km7vz7t+aYlcY8nNkNut4bnob0kSZK0jqraDuwGrgIuBT4FHAVeA54H\n9nf3PD9b8EMSm8FugroTmyGzO7EZFre7qi5f51uXAQ8C5wAPAM8Bb5/p/br7xeGmm47EZsjsTmwG\nu9ewsN2JzRuxqNfsU1XV94HfAP8FbgMOdPe7a6zbCtwK/JTJnri5u387w1E/tsQ9ntgMud2aDQ/t\nJUmSJEmSdFarqg/d3P8Elrt7y4DvNxWJzZDZndgMdg9kLroTm/WBhEcCJO7xxGbI7dZsuBkkSZIk\nSZJ0tls685KFk9gMmd2JzWB3ksRmreju96rqW3zwSIBPA1/lDI8EmN2Eg0jc44nNkNutGfDQXpIk\nSZIkSWe7HWMPMILEZsjsTmwGu5MkNmuV7j4CfLeq7mExHwmQuMcTmyG3WzPgx+NLkiRJkiRpYVXV\nlu4+MfYcs5TYDJndic1g99hzzFJis7Ik7vHEZsjt1sZtGnsASZIkSZIk6eOqqhuq6tmq2rXOkvur\n6oWq2jPTwaYosRkyuxObwe6k7sRmZUnc44nNkNut4XhoL0mSJEmSpLlUVXcDB4CrgW3rLFsCtgOP\nVdVds5ptWhKbIbM7sRnsJqg7sVlZEvd4YjPkdmtYHtpLkiRJkiRp7lTVDmAfkxugLwCvrLN0P/D0\nyrp9VXXdbCYcXmIzZHYnNoPdBHUnNitL4h5PbIbcbg3PQ3tJkiRJkiTNo9tWvt7X3V/p7r+stai7\nn+rubwA/Z3KT9PZZDTgFic2Q2Z3YDHYndSc2K0viHk9shtxuDcxDe0mSJEmSJM2ja4BjwN4Nrr8T\nOL7yunmV2AyZ3YnNYHdSd2KzsiTu8cRmyO3WwDy0lyRJkiRJ0jy6ADjU3cc3sri7jwGvrrxuXiU2\nQ2Z3YjPYndSd2KwsiXs8sRlyuzUwD+0lSZIkSZI0j94Etm50cVUtARcBb01toulLbIbM7sRmsHtD\nFqQ7sVlZEvd4YjPkdmtgHtpLkiRJkiRpHr0IXFxV121w/Y3AhcBL0xtp6hKbIbM7sRnsTupObFaW\nxD2e2Ay53RqYh/aSJEmSJEmaR48CS8ATVbW7qjavtaiqtlbVXuAhYBl4ZIYzDi2xGTK7E5vB7qTu\nxGZlSdzjic2Q262BLS0vL489gyRJkiRJkvSRVNUm4CBwLZMbn0eAl4HDwAngPGAbcCWwmcnN1IPd\nvXOUgQeQ2AyZ3YnNYDdB3YnNypK4xxObIbdbw/PQXpIkSZIkSXOpqs4FHgb2rPrr1Te7llb9/nHg\nlu4+OovZpiWxGTK7E5vBboK6E5uVJXGPJzZDbreG5aG9JEmSJEmS5lpVbQd2A1cBlwKfAo4CrwHP\nA/u7e6GeG5rYDJndic1gN0Hdic3KkrjHE5sht1vD8NBekiRJkiRJkiRJkqSRbBp7AEmSJEmSJEmS\nJEmSUnloL0mSJEmSJEmSJEnSSDy0lyRJkiRJkiRJkiRpJB7aS5IkSZIkSZIkSZI0Eg/tJUmSJEmS\nJEmSJEkaiYf2kiRJkiRJkiRJkiSNxEN7SZIkSZIkSZIkSZJG4qG9JEmSJEmSJEmSJEkj8dBekiRJ\nkiRJkiRJkqSReGgvSZIkSZIkSZIkSdJIPLSXJEmSJEmSJEmSJGkkHtpLkiRJkiRJkiRJkjQSD+0l\nSZIkSZIkSZIkSRqJh/aSJEmSJEmSJEmSJI3EQ3tJkiRJkiRJkiRJkkbiob0kSZIkSZIkSZIkSSPx\n0F6SJEmSJEmSJEmSpJH8Hx1a4GO+QXqHAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<matplotlib.figure.Figure at 0x7fe61ba650f0>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "###Show Correlation Between Test Validation Sets\n",
    "chromosomes = ['chr1', 'chr2', 'chr3', 'chr4', 'chr5', 'chr6', 'chr7', 'chr8', 'chr9', 'chr10',\n",
    "               'chr11', 'chr12', 'chr13', 'chr14', 'chr15', 'chr16', 'chr17', 'chr18', 'chr19', 'chrX'\n",
    "              ]\n",
    "\n",
    "    \n",
    "chromosome_index_dict = dict(zip(chromosomes, range(len(chromosomes))))\n",
    "\n",
    "corr_matrix = np.zeros((len(chromosomes), len(chromosomes)))\n",
    "for i in range(len(chromosomes) - 1):\n",
    "    training_chromosome_1 = chromosomes[i]\n",
    "    index_1 = chromosome_index_dict[training_chromosome_1]\n",
    "    for j in range(i+1, len(chromosomes)):\n",
    "        training_chromosome_2 = chromosomes[j]\n",
    "        index_2 = chromosome_index_dict[training_chromosome_2]\n",
    "        pearson, pvalue = scipy.stats.pearsonr(chromosome_coeff_dict[training_chromosome_1],\n",
    "                                              chromosome_coeff_dict[training_chromosome_2])\n",
    "        corr_matrix[index_1][index_2] = pearson\n",
    "        corr_matrix[index_2][index_1] = pearson\n",
    "\n",
    "cg = sns.clustermap(data=corr_matrix, cmap='Blues', xticklabels = chromosomes, yticklabels = chromosomes)\n",
    "plt.setp(cg.ax_heatmap.yaxis.get_majorticklabels(), rotation=0);\n",
    "plt.title(\"Pearson Correlation Between Cross Validation Sets\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Visualize rocAUC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(array([ 0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12, 13, 14, 15, 16,\n",
       "        17, 18, 19]), <a list of 20 Text xticklabel objects>)"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAABWUAAAQOCAYAAACzV1oHAAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAAewgAAHsIBbtB1PgAAIABJREFUeJzs3Xu4pVddH/DvmRkCBCIkZiaACYZCs+QWrtFELoIQSUhI\nQiXcxApoLKCmWihVam1BNNKiSGnA0oJcRGzBQK4kBELkTqEQJFx+KAgiJjNnSJCEEHAyp3+8e8zp\nMDNnn5m91z6Xz+d59vPuy3rX+1vz7r3Pme+sWe/cwsJCAAAAAADoY8OsCwAAAAAAWE+EsgAAAAAA\nHQllAQAAAAA6EsoCAAAAAHQklAUAAAAA6EgoCwAAAADQkVAWAAAAAKAjoSwAAAAAQEdCWQAAAACA\njoSyAAAAAAAdCWUBAAAAADoSygIAAAAAdCSUBQAAAADoSCgLAAAAANCRUBYAAAAAoCOhLAAAAABA\nR0JZAAAAAICOhLIAAAAAAB0JZQEAAAAAOhLKAgAAAAB0tGnWBaxErbXbJPlPSV6YZGOS/1RVLznA\nPo9PcnaSH09yRJKbk1SSdyR5VVXddCD9AwAAAACrg5myu2mtPSDJJ5L8RoY/n4UJ9PmiJB9O8pQk\nW5L8TZIbkxyX5Jwkn2yt3e1AjwMAAAAArHxC2ZHW2obW2m8k+ViS+yX530nmJtDvaUlemiHcfUWS\nzVV1n6o6KsnxSb6U5JgkbzvQYwEAAAAAK59Q9lanJPmdJN9N8qyqeuqE+n15hkD2gqp6QVV9e9cL\nVfXxJGeOXj++tXbmhI4JAAAAAKxQQtlbbUjy0SQPqKo3TaLD1trDktxr9PDle2pTVVcluWL08JmT\nOC4AAAAAsHIJZW/1wSQPr6qvTLDPR4+2Nyb5yD7aXZ5hqYRHTvDYAAAAAMAKJJQdqapvVNXOCXd7\n7Gj7V0v0/YXR9uDWWptwDQAAAADACiKUna4jM6wX+3dLtFv8+pHTKwcAAAAAmDWh7HQdMtretES7\nxa8fstdWAAAAAMCqJ5SdrtuPtt9bot13F90/eEq1AAAAAAArgFB2unbNgD1oiXa328M+AAAAAMAa\nJJSdrhtG2zss0e6Oi+5/a0q1AAAAAAArwKZZF7DGfTXJCVn64l1HL7r/5SnVsjClfgEAAABgPZib\nVEdC2en6dJKnJjmmtbapqnbspd2xo+03q+or0yhkx45bMjc3sfcNAAAAAKwrGzdObtEBoex0XZbk\nnAwX73pkkiv20u6kDDNZL5lWIddfb6laAAAAANhfmzcfMrG+rCk7RVV1VZJPZZja/MI9tWmtPTbJ\nQ0YPX9epNAAAAABgRsyUHWmt3TnJxkVPLf6//ge31n5wt12uq6qF1trdM6wDu5DkxVX10t3a/VqS\n9yU5sbV2bpJfr6obRsd8TJI3j/Y9r6qunNiAAAAAAIAVyUzZW12VZH7Rbdvo+V2zXOd3ux216PUN\ni27/n6p6f5LnJtmZ5DlJtrXWPt9a+/sklyfZkuSDSZ41lVEBAAAAACuKUPZWOzPMWB3ntnO3fRf2\n1XFVvTbJg5K8Mck1SY5OclCSK5OcleRRVXXjZIYBAAAAAKxkcwsL+8wTWSPm529wogEAAABgP23e\nfMjc0q3GY6YsAAAAAEBHQlkAAAAAgI6EsgAAAAAAHQllAQAAAAA6EsoCAAAAAHQklAUAAAAA6Ego\nCwAAAADQkVAWAAAAAKAjoSwAAAAAQEdCWQAAAACAjoSyAAAAAAAdCWUBAAAAADoSygIAAAAAdCSU\nBQAAAADoSCgLAAAAANCRUBYAAAAAoCOhLAAAAABAR5tmXQDQz8mPPylbt83PuowDdsSWzXnXJZfO\nugwAgIk747Sfyvb5bbMu44AdvnlL3nnBu2ddBgCsWELZde5xJz0uW7eu/l/6jjhiSy679LKx2590\n8knZum0NjHvLllz6rvHDya3b5vOM56/+MPNPfv+kWZfACnbKEx6f+fnV/48PmzdvzsUXXjLrMmBF\nOf30k9fM5/v889816zJWvDNOe1zmt6/+39c2H74l77xg/N9Tt89vy4f+9NVTrKiPhz39ebMuAVgh\nTjvp0Znfdu2syzhgm7fcJRdc+r5Zl8EaIpRd57Zu3ZZHPfH1sy7jgF35jmcvq/3Wbdty2rPfPqVq\n+rng9U+adQmrwuNPOTnb1sBf4rds3pxLLh7/L/GPP/XkzM9vn2JFfWzefHguuWj8cc/Pz+d3X/fx\nKVbUx4t+/rhltT/1tLUTRl90wfhh9BNOe3zmt6+BcR++ORcuY9ynraFw8oJlhJPz8/O56NL3TLGi\nPk496bGzLmFVmN++LR84709mXcYBe8S/eMasS1gVznjCiWtnhvCFl4/d/oxTH7t2xn3R+N/Pp5/y\nmGzftnWKFfVx+JYjcv7F7x27/emP/8nMr4Fxb95yRM6/5Iqx289vuzZ//rSTp1hRHz/9Vv+gymQJ\nZYE1b9v8fJ77kr+YdRkH7DW/9RPLaj8/vz3/7g8/MqVq+nnZr54w6xJWhfn5+fy3t35i1mUcsF9+\n2kOX1X5++3ze8I6PTamafp75xB9bVvv5+fn874s/NKVq+nnyKQ+bdQmrwumnn5TtayCEP3zz5px/\n/ur/HztMx/b5bfnwa3931mUcsB//xRctq/32+W35wO//uylV088jnv+yZbXfvm1rrvgP/2pK1fTz\nk7/935fVfn7b1rzrl54ypWr6Ofnc/zXrElaFJzzuUZnfugZmCB9xl1x42ZWzLmNNEsoCAMAKtn1+\nPu+55J2zLuOAPfbxZ8y6BADoZn7rtXnjiav/H6B/7vLVPxFgpdow6wIAAAAAANYTM2UBAAAAgAN2\n6ok/sWaWbbjo8ukugyiUBQAAAAAO2PzWa/Nf73//WZdxwM7+zGemfgzLFwAAAAAAdCSUBQAAAADo\nSCgLAAAAANCRUBYAAAAAoCOhLAAAAABAR0JZAAAAAICOhLIAAAAAAB0JZQEAAAAAOhLKAgAAAAB0\nJJQFAAAAAOhIKAsAAAAA0JFQFgAAAACgI6EsAAAAAEBHQlkAAAAAgI6EsgAAAAAAHQllAQAAAAA6\nEsoCAAAAAHQklAUAAAAA6EgoCwAAAADQkVAWAAAAAKAjoSwAAAAAQEdCWQAAAACAjoSyAAAAAAAd\nCWUBAAAAADoSygIAAAAAdCSUBQAAAADoSCgLAAAAANCRUBYAAAAAoCOhLAAAAABAR0JZAAAAAICO\nhLIAAAAAAB0JZQEAAAAAOhLKAgAAAAB0JJQFAAAAAOhIKAsAAAAA0JFQFgAAAACgI6EsAAAAAEBH\nQlkAAAAAgI6EsgAAAAAAHQllAQAAAAA6EsoCAAAAAHQklAUAAAAA6EgoCwAAAADQkVAWAAAAAKAj\noSwAAAAAQEdCWQAAAACAjoSyAAAAAAAdCWUBAAAAADoSygIAAAAAdCSUBQAAAADoSCgLAAAAANCR\nUBYAAAAAoCOhLAAAAABAR0JZAAAAAICOhLIAAAAAAB0JZQEAAAAAOhLKAgAAAAB0JJQFAAAAAOhI\nKAsAAAAA0JFQFgAAAACgI6EsAAAAAEBHQlkAAAAAgI6EsgAAAAAAHQllAQAAAAA6EsoCAAAAAHQk\nlAUAAAAA6EgoCwAAAADQkVAWAAAAAKAjoSwAAAAAQEdCWQAAAACAjoSyAAAAAAAdCWUBAAAAADoS\nygIAAAAAdCSUBQAAAADoSCgLAAAAANCRUBYAAAAAoCOhLAAAAABAR0JZAAAAAICOhLIAAAAAAB0J\nZQEAAAAAOhLKAgAAAAB0JJQFAAAAAOhIKAsAAAAA0JFQFgAAAACgI6EsAAAAAEBHQlkAAAAAgI6E\nsgAAAAAAHQllAQAAAAA6EsoCAAAAAHQklAUAAAAA6EgoCwAAAADQkVAWAAAAAKAjoSwAAAAAQEdC\nWQAAAACAjoSyAAAAAAAdCWUBAAAAADoSygIAAAAAdCSUBQAAAADoSCgLAAAAANCRUBYAAAAAoCOh\nLAAAAABAR0JZAAAAAICOhLIAAAAAAB0JZQEAAAAAOhLKAgAAAAB0JJQFAAAAAOhIKAsAAAAA0JFQ\nFgAAAACgI6EsAAAAAEBHQlkAAAAAgI6EsgAAAAAAHQllAQAAAAA6EsoCAAAAAHQklAUAAAAA6Ego\nCwAAAADQkVAWAAAAAKAjoSwAAAAAQEdCWQAAAACAjoSyAAAAAAAdCWUBAAAAADoSygIAAAAAdCSU\nBQAAAADoSCgLAAAAANCRUBYAAAAAoCOhLAAAAABAR5tmXcBK01o7PsnZSX48yRFJbk5SSd6R5FVV\nddN+9Lkxyc8keUqSByc5bNTv3yZ5X5LXVNXnJzIAAAAAAGBFM1N2kdbai5J8OEN4uiXJ3yS5Mclx\nSc5J8snW2t2W2edhoz7fkOSkJLdN8sUk80nuneSXk1zVWnvOZEYBAAAAAKxkQtmR1tppSV6aZCHJ\nK5Jsrqr7VNVRSY5P8qUkxyR52zK7fmOGUPe7Sc6qqsOq6v5Vda8kRya5OMltkvy31tpxkxkNAAAA\nALBSCWVv9fIMgewFVfWCqvr2rheq6uNJzhy9fnxr7cxxOmytHZ3klNF+r6yq1y9+vaquTfK0JDcl\nmUvy3AmMAwAAAABYwYSySVprD0tyr9HDl++pTVVdleSK0cNnjtv1ovsf2Eu/N2ZYs3Zut/YAAAAA\nwBoklB08erS9MclH9tHu8gzh6SPH7PfaRfdvt492t8kwm/aaMfsFAAAAAFYpoezg2NH2r6pq5z7a\nfWG0Pbi1Ns6s1s9muFjYXJKn76lBa+2oJD8yenjhGH0CAAAAAKvYplkXsEIcmWGm6t8t0W7x60dm\nWHZgr6pqR2vtF5Ocn+SM1tq5Sf4wQ1B7uyQPS/L7STZmmIX7pv2qHgAAAABYNcyUHRwy2t60RLvF\nrx+y11aLVNV7kzw8yduT/KsMQe73knwrybuS3D7Ji5KcWlULy6gZAAAAAFiFhLKD24+231ui3XcX\n3T94Gf0fl+TeGZYx+FaSzyX5cpKdSe6WYcbsfZbRHwAAAACwSgllB7tmwB60RLvFF+taalZtkqS1\n9pok/z3JXZOcmeTQqrpfVd0ryV2S/M8kpyb5SGtt3AuIAQAAAACrlDVlBzeMtndYot0dF93/1lKd\nttYek2HJgoUkz6mq8xa/XlXbk/xKa+2HkpyR5NVJ7jdu0ctx6KEHZ25u7vuen9vw/c+tRnMb5nLY\nYUudvltt2MOfxWq0YW6Z414j53vDMs/3nt77q9HcMs/3eh33en2fr9vvtfU67vX6Pl+n455bI9Mo\n5jbE53sM6/Xn9/od9xSL6Whubpmf73X6fb5ev9fW7fk27lVtuePeH0LZwVeTnJDh4l37cvSi+18e\no9+fHW1vqKo/30e78zKEsvdurT2gqj49Rt/LsmnTxj0+vzY+KsM4Nm5cxt9Y1tDAlzXuNcT5HqP5\nGhn3nPf5eNbI+fa9tv4Y99Lm1sgHfC5zzvc41sbp9nvL+HtMrZa+fL7XG+NeX4x7OoSyg08neWqS\nY1prm6pqx17aHTvafrOqvjJGv3cdba9Zot323faZeCi7Y8cte/zX57VyZbGFJLfcsnN5O6wFC8sc\n9xrifI/RfI2Me8H7fDxr5Hz7Xlt/jHtpC2vkA76QBed7HGvjdPu9Zfw9plZLXz7f641xry/GfatJ\nBrVC2cFlSc7JcPGuRya5Yi/tTsrwU/OSMfu9frS9+xLtFr9+3Zh9L8v11+95CdyFnWvjl4CFnQu5\n7rpvj91+5xr5rW/nwjLHvUbO985lnu+FNXK+F5Z5vtfruNfr+3zdfq+t13Gv1/f5Oh33whr5e9DC\nzvh8j2G9/vxev+OeYjEdLSws8/O9Tr/P1+v32ro938a9qu1t3Js3HzKxY6zP+ce7qaqrknwqw/8d\neeGe2rTWHpvkIaOHrxuz678YbW/XWjttH+2eONreOKoDAAAAAFijhLK3+rUMs2BPbK2d21r7p+h7\ndMGuN49eP6+qrlz02t1baztaa//YWvvN3fp8Y5K/yxD2vr619i9aaxsW7XtYa+2/JnncqO//UlX/\nOKXxAQAAAAArgFB2pKren+S5SXYmeU6Sba21z7fW/j7J5Um2JPlgkmfttutchj/HXbfFfd6U5OQk\nf5Pk0CRvT3J9a+2zrbW/SrItyS9nCGT/R1W9dErDAwAAAABWCKHsIlX12iQPyjDD9ZokRyc5KMmV\nSc5K8qiqunEPu+51wYyq+myS+yU5O8l7k9yU5J5JjkjypSRvSPLIqnrOhIYBAAAAAKxgLvS1m6q6\nOsmzl9H+q0k2LtHmO0nOHd0AAAAAgHXMTFkAAAAAgI6EsgAAAAAAHQllAQAAAAA6EsoCAAAAAHQk\nlAUAAAAA6EgoCwAAAADQkVAWAAAAAKAjoSwAAAAAQEdCWQAAAACAjoSyAAAAAAAdCWUBAAAAADoS\nygIAAAAAdCSUBQAAAADoSCgLAAAAANCRUBYAAAAAoCOhLAAAAABAR0JZAAAAAICOhLIAAAAAAB0J\nZQEAAAAAOhLKAgAAAAB0JJQFAAAAAOhIKAsAAAAA0JFQFgAAAACgI6EsAAAAAEBHQlkAAAAAgI6E\nsgAAAAAAHQllAQAAAAA6EsoCAAAAAHQklAUAAAAA6EgoCwAAAADQkVAWAAAAAKAjoSwAAAAAQEdC\nWQAAAACAjoSyAAAAAAAdCWUBAAAAADoSygIAAAAAdCSUBQAAAADoSCgLAAAAANCRUBYAAAAAoCOh\nLAAAAABAR0JZAAAAAICOhLIAAAAAAB0JZQEAAAAAOhLKAgAAAAB0JJQFAAAAAOhIKAsAAAAA0JFQ\nFgAAAACgI6EsAAAAAEBHQlkAAAAAgI6EsgAAAAAAHQllAQAAAAA6EsoCAAAAAHQklAUAAAAA6Ego\nCwAAAADQkVAWAAAAAKAjoSwAAAAAQEdCWQAAAACAjoSyAAAAAAAdCWUBAAAAADoSygIAAAAAdCSU\nBQAAAADoSCgLAAAAANCRUBYAAAAAoCOhLAAAAABAR0JZAAAAAICOhLIAAAAAAB0JZQEAAAAAOhLK\nAgAAAAB0JJQFAAAAAOhIKAsAAAAA0JFQFgAAAACgI6EsAAAAAEBHQlkAAAAAgI6EsgAAAAAAHQll\nAQAAAAA6EsoCAAAAAHQklAUAAAAA6EgoCwAAAADQkVAWAAAAAKAjoSwAAAAAQEdCWQAAAACAjoSy\nAAAAAAAdCWUBAAAAADoSygIAAAAAdCSUBQAAAADoSCgLAAAAANCRUBYAAAAAoCOhLAAAAABAR0JZ\nAAAAAICOhLIAAAAAAB0JZQEAAAAAOhLKAgAAAAB0JJQFAAAAAOhIKAsAAAAA0JFQFgAAAACgI6Es\nAAAAAEBHQlkAAAAAgI6EsgAAAAAAHQllAQAAAAA6EsoCAAAAAHQklAUAAAAA6EgoCwAAAADQkVAW\nAAAAAKAjoSwAAAAAQEdCWQAAAACAjoSyAAAAAAAdCWUBAAAAADoSygIAAAAAdCSUBQAAAADoSCgL\nAAAAANCRUBYAAAAAoCOhLAAAAABAR0JZAAAAAICOhLIAAAAAAB0JZQEAAAAAOhLKAgAAAAB0JJQF\nAAAAAOhIKAsAAAAA0JFQFgAAAACgI6EsAAAAAEBHQlkAAAAAgI6EsgAAAAAAHQllAQAAAAA6EsoC\nAAAAAHQklAUAAAAA6EgoCwAAAADQkVAWAAAAAKAjoSwAAAAAQEdCWQAAAACAjoSyAAAAAAAdCWUB\nAAAAADoSygIAAAAAdCSUBQAAAADoSCgLAAAAANCRUBYAAAAAoCOhLAAAAABAR0JZAAAAAICOhLIA\nAAAAAB0JZQEAAAAAOhLKAgAAAAB0JJQFAAAAAOhIKAsAAAAA0JFQFgAAAACgI6EsAAAAAEBHQlkA\nAAAAgI6EsgAAAAAAHQllAQAAAAA6EsoCAAAAAHQklAUAAAAA6EgoCwAAAADQkVAWAAAAAKCjTbMu\nYKVprR2f5OwkP57kiCQ3J6kk70jyqqq6aT/73ZjkrCTPSPIjSW6f5O+TvD/JH1XVxw+8egAAAABg\npTNTdpHW2ouSfDjJU5JsSfI3SW5MclySc5J8srV2t/3o99AkH0ny6iQnJPl2kq8n+eEkz0ry0dba\n8yYxBgAAAABgZRPKjrTWTkvy0iQLSV6RZHNV3aeqjkpyfJIvJTkmydv2o/t3Jnlokk8meWBV/XBV\nHZPkqNFrc0le0Vq794GPBAAAAABYyaYeyrbW5lprYx+ntXbP1trcNGvai5dnCGQvqKoXVNW3d70w\nWlrgzNHrx7fWzhy309bazyR5RIalCk6sqs8s6ndrkqcneXeSt2ZYLgEAAAAAWMOmGsq21n4qyeeT\nnDFm+00ZAspPtNZ+ZJq17XbchyW51+jhy/fUpqquSnLF6OEzl9H9r2QIc/9zVV2/h35vrqqTquqZ\nVXXlMvoFAAAAAFahqYWyrbVfSXJJkn+e5Glj7nZWknskeWCSj43C0h4ePdremGHt1725PMNSA48c\np9PR+rPHjR7uz7IHAAAAAMAas2kanbbWTk7yygwzROcy/n/Lf2eGgPRJSQ5JckFr7cFV9dVp1LnI\nsaPtX1XVzn20+8Joe3BrrVVVLdHvQzOMf2tVXdNaOzLJszOsUXtYkq1J3pvk9VV14/6XDwAAAACs\nFhOfKdta25jk3NHD7yb5haoaa2ZpVV1TVU9O8pwMge6dk7x60jXuwZGj4/3dEu0Wv37kGP3ed9d+\nowuJfTbJf0zyuAwzaE9N8odJvthaO27PXQAAAAAAa8k0li84PcnRGULOs6rq9cvtoKpem+RXM8wy\nPam19qCJVvj9Dhltb1qi3eLXD9lrq1v94Gi7OclbklyZ4aJfd0pyeJKfT7I9yV2SXNhac6EvAAAA\nAFjjprF8wRNG2/dV1Vv2t5OqelVr7akZ/qv/zyT51CSK24vbj7bfW6LddxfdP3iMfncFt0cluaiq\nTt/t9Te01q7OsI7t5iT/NskLxugXAAAAAFilpjFT9kEZZskue4bsHrwmw2zZaV/wa9cM2IOWaHe7\nPeyzLwuL7v/OnhpU1SeSvCvDOH96jD4BAAAAgFVsGjNl7z7afnQCfX14tL3nBPralxtG2zss0e6O\ni+5/axn9Jsmn99HuAxnWl717a+2O07jo16GHHpy5ubnve35uw/c/txrNbZjLYYctdfputWEPfxar\n0Ya5ZY57jZzvDcs833t6769Gc8s83+t13Ov1fb5uv9fW67jX6/t8nY57bhrTKGZgbkN8vsewXn9+\nr99xT7GYjubmlvn5Xqff5+v1e23dnm/jXtWWO+79MY1Qdldw+Y0J9HXdaHunCfS1L19NckKWvnjX\n0Yvuf3mMfr825vGvW3T/jkkmHspu2rRxj8+vjY/KMI6NG5fxN5Y1NPBljXsNcb7HaL5Gxj3nfT6e\nNXK+fa+tP8a9tLk18gGfy5zzPY61cbr93jL+HlOrpS+f7/XGuNcX456OaYSyNyS5c4Yg9R8OsK9D\nF/U5TZ9O8tQkx7TWNlXVjr20O3a0/WZVfWWMfj+z6P49knxhL+3uvOj+N8fod9l27Lhlj//6vLCH\ntqvRQpJbbtm5vB3WgoVljnsNcb7HaL5Gxr3gfT6eNXK+fa+tP8a9tIU18gFfyILzPY61cbr93jL+\nHlOrpS+f7/XGuNcX477VJIPaaYSy8xlCxn+W5G8PsK/7jrbbDrCfpVyW5JwMF+96ZJIr9tLupAw/\nNS8Zs98PZVjm4JAkT07ykr20+7HR9otVdfOYfS/L9dfveQnchZ1r45eAhZ0Lue66b4/dfuca+a1v\n58Iyx71GzvfOZZ7vhTVyvheWeb7X67jX6/t83X6vrddxr9f3+Tod98Ia+XvQws74fI9hvf78Xr/j\nnmIxHS0sLPPzvU6/z9fr99q6Pd/GvartbdybNx8ysWNMYx7urrVTT5tAX0/erc+pqKqrknwqw/8d\neeGe2rTWHpvkIaOHrxuz3+8l+dNRv7/UWvuBPfR7jyRPyBD2/vmyiwcAAAAAVpVphLKXZgghf6G1\ndtf97aS19uAkT8sQVl44odr25ddGxzqxtXZua+2fou/W2mOSvHn0+nlVdeWi1+7eWtvRWvvH1tpv\n7qHflyS5PsnhSS5rrd1z0b4PTHJxkttmmA38yskPCwAAAABYSaYRyv55hgtX3SHJW1trt19uB621\nI5O8PcnGJFuTvHOiFe5BVb0/yXOT7EzynCTbWmufb639fZLLk2xJ8sEkz9pt17kMf467brv3e22S\nUzJc+OxHk3yxtfbF1tpXknwySRu9dkZVzU9+ZAAAAADASjLxULaqvpVhfda5JI9I8hettfuNu39r\n7YwkH01ydIaZqb9ZVXteEHXCquq1SR6U5I1JrhnVcFCSK5OcleRRVXXjHnbd54IZVfXRJPdJ8rIk\nn0tylyQ/mOSzo+fuXVUfm8ggAAAAAIAVbRoX+kqSP0jy8CSnZ1iH9VOttcuS/FmS/5Pky1W1I0lG\nM2nvmyHA/bkk988Q6CbJ66rq9VOqcY+q6uokz15G+69mmNG7VLvtSV40ugEAAAAA69RUQtmqWmit\nPT3J/0jy9Ayh5cmjW5Kktfad0fMH7bb7XIaZp69K8m+mUR8AAAAAwKxMa6Zsquo7SZ7RWjs/w+zQ\nB+zW5OC97Pr+JC+uqvdNqzYAAAAAgFmZWii7S1W9LcnbWms/luRRSX4sw5qqh2a4qNZ1Sf42wzqy\n76mqz0+7JgAAAACAWZl6KLvL6EJWLmYFAAAAAKxrG2ZdAAAAAADAeiKUBQAAAADoaOLLF7TWTtuP\n3W5J8o0kX6+qr024JAAAAACAFWMaa8q+M8nC/u7cWvtSkjcl+f2q+s7EqgIAAAAAWAGmtXzB3AHc\n7pnkxUk+31q735TqAwAAAACYiWnMlH3xfuwzl+TOSe6R5JFJ7pTk7kne31p7aFV9eYL1AQAAAADM\nzMRD2aran1D2n7TWbpfkV5O8JEM4+6YkD59AaQAAAAAAMzet5Qv2W1XdXFW/l+TpGWbQntBae9Rs\nqwIAAAAAmIwVF8ruUlVvT3LB6OFTZlkLAAAAAMCkrNhQduQNGc2WnXEdAAAAAAATsdJD2U+OtkfN\ntAoAAABI4HnqAAAgAElEQVQAgAlZ6aHsP4y2d5xpFQAAAAAAE7LSQ9k7j7Y3zLQKAAAAAIAJWemh\n7ENH26/MsggAAAAAgElZ6aHsWUkWknx41oUAAAAAAEzCig1lW2tnJ/mp0cM3z7IWAAAAAIBJ2TTr\nAnbXWntIkhckeXKGWbIXVdXHZ1sVAAAAAMBkTDyUba19cj93vUOSuyS54+jxXJK/TvILk6gLAAAA\nAGAlmMZM2QdmmOG6P+YW3b88yc9W1fyBlwQAAAAAsDJMa/mCuaWbfJ9vJbk2yQeT/FlVvWeyJQEA\nAAAAzN7EQ9mqWrEXDwMAAAAAmDUBKgAAAABARys+lG2t3aa1duas6wAAAAAAmIRprSl7wFprD0jy\n7CRPT3JokrfNtiIAAAAAgAO3okLZ1tqdk/xMhjD2gaOn55IszKwoAAAAAIAJWhGhbGvtxAxB7OlJ\nbjt6em60vSbJn82iLgAAAACASZtZKNtaOzrJs5L8XJKjRk/vCmJvSHJekj9JckVVmSkLAAAAAKwJ\nXUPZ1tptkzwpw6zYn8gQwu4KYheSXJnkNUkurKqbe9YGAAAAANBDl1C2tXZchiD2qUl+YPT0rjD2\n/yZ5yOj+y6vqkh41AQAAAADMwtRC2dba4Ul+NsMSBfcdPb0riJ3PsDTBH1fV1a21ndOqAwAAAABg\nJZl4KNtaOyXDrNhTktwmtwaxO5JckuSPk1xcVTsmfWwAAAAAgJVuGjNlL8ywPuyuMPZTSd6c5E+r\natsUjgcAAAAAsGpMc03Z/5vkuVX1iSkeAwAAAABgVZlmKPvgJB9srV2W5K1J3llVN0/xeAAAAAAA\nK96GKfT5pCSXZljC4KAkpyZ5S5JrW2t/1Fo7YQrHBAAAAABYFSYeylbVeVV1SpIfTvIfk3w1w/qy\nP5DkrAyzZ7/QWntha+0ukz4+AAAAAMBKNo2ZskmSqvp6Vf12Vf2zJCcm+V9JvpchoD0myTlJvtZa\nu2RaNQAAAAAArDRTC2UXq6r3VtXTkvxQkl9L8pkM4ezGJI/LsNRBkjy7tfawHjUBAAAAAMzCNC/0\n9X2q6rokr0zyytbacRmWM3hKkkNGTZ6Y5Imtta9kWIf2LVVVPWsEAAAAAJimLjNl96SqPl5Vv5jk\nrkl+PsmHM8yenUtyjyT/PsnnWmsfb62dPas6AQAAAAAmaWah7C5VdVNV/XFVPTzJfZL8QZL53BrQ\nPiTJK2ZYIgAAAADAxMw8lF2sqr5QVS/IsPbsk5NcmmTnbKsCAAAAAJicrmvKjquqdiR5e5K3t9aO\nSvLM2VYEAAAAADAZ3UPZ1tpckvtW1dV7eX1Lknsn+UBV7ayqryX57Z41AgAAAABMS9flC1prZyb5\n6yTv3kezf57kiiRfaa09rUthAAAAAACddAtlW2vPT/JnSe6R5PAlms8lOTLJn7TWXjjt2gAAAAAA\neukSyrbWjk3yexnC1uuS/M/W2sa9NP98kpcm+dqo/e+01h7Qo04AAAAAgGnrNVP2V5JszBC43r+q\nnldVt+ypYVVdV1W/leTBSa4e1Xh2pzoBAAAAAKaqVyj7k0kWkvx6VV0zzg5V9Y0kv5Vhtuyjp1gb\nAAAAAEA3vULZu422n1jmfh8bbe86wVoAAAAAAGamVyj77dF2YZn7bRptvzPBWgAAAAAAZqZXKPul\n0fZhy9zv1NH2bydYCwAAAADAzPQKZc/PsDbsy1prPzTODq2145Ock2F27SVTrA0AAAAAoJtNSzeZ\niNcleWGSeyS5urX2hiTvSfLXSa5LsiPJYUl+MElL8oQkZ4zq+4ckr+pUJwAAAADAVHUJZatqa2vt\nGUneluROSc4e3fZlLsl3kzyjqq6ZcokAAAAAAF30Wr4gVXVRkkck+ViGwHWp24eSPLyqLu5VIwAA\nAADAtPVaviBJUlWfSHJCa+3YJCcmuU+SI5LcNslNSbYmuTrJu6vqCz1rAwAAAADooWsou0tV/WWS\nv5zFsQEAAAAAZqnb8gUAAAAAAMxopmxr7UczrC973ySHJTk4yY1JvpHk00muqKrPzaI2AAAAAIBp\n6hrKttYem+RVSY4Zo+0Hkzyvqj479cIAAAAAADrptnxBa+1fJrk0QyA7N8bt4Uk+3lp7VK8aAQAA\nAACmrUso21o7MskfjY63I8mbkzwtyQOSHJnk8CRHJXlQkp9N8vYkC0lul+RtrbU79agTAAAAAGDa\nei1f8EsZAtatSX6qqj6zl3Zfz7Cm7FtaayckuSTDmrPPS3JOj0IBAAAAAKap1/IFJ2aY+fpb+whk\n/z9V9ZEkL86wlMEpU6wNAAAAAKCbXqHs0aPtu5e530WjbZtcKQAAAAAAs9MrlD1ktP3mMvebH21/\nYIK1AAAAAADMTK9QdlcYe+Qy97vLaPsPE6wFAAAAAGBmeoWyXxhtn7zM/Z4y2n5ugrUAAAAAAMxM\nr1D24gwX7Hpha+0pSzVOktba05O8KMMFwi6cYm0AAAAAAN1s6nScVyd5fpLDk/xpa+35Sd6R5OoM\n68benOTgJFuSPCDJE5PcP0OQ+/Ukr+lUJwAAAADAVHUJZavqxtba6Ukuy3DRr4eMbvsyl+QbSU6p\nqpumXCIAAAAAQBe9li9IVX00ybFJzktyS4bQdW+37yZ5Y5L7VdVf9qoRAAAAAGDaei1fkCSpqq8m\neVJr7fAkj8iwRMHhSW6f5NsZljK4KsmHquqbPWsDAAAAAOihayi7S1Vtz7Cm7DtmcXwAAAAAgFnp\nEsq21v5gdPd3R4EsAAAAAMC61GtN2bOT/Osk/9jpeAAAAAAAK1KvUPba0XZzp+MBAAAAAKxIvULZ\nc5PMJfn1TscDAAAAAFiRel3o6/cyCmVba1uSvCzJR6pqZ6fjAwAAAACsCL1C2VeOtucl+ekkpyT5\nTmvt2iTbk9y0xP4LVfWYKdYHAAAAANBFr1D2l5Ms7PbcwUnuMbrty9we9gUAAAAAWJV6hbLJEK6O\n8xwAAAAAwJrVJZStql4XFAMAAAAAWNGEpQAAAAAAHQllAQAAAAA6EsoCAAAAAHQklAUAAAAA6Ego\nCwAAAADQkVAWAAAAAKAjoSwAAAAAQEdCWQAAAACAjoSyAAAAAAAdCWUBAAAAADoSygIAAAAAdLSp\n9wFba0cmuW+SQ5McNO5+VfWmqRUFAAAAANBJt1C2tfajSc5N8uD92H0hiVAWAAAAAFj1uoSyrbX7\nJLkiye2TzPU4JgAAAADAStRrpuxvJDl4dP/qJBcm+Vry/9i78zBbrrpe+N9OQiCTmMQAIpPgy483\nSGQQZLgMGiYn1EiYEQICRu9FURSuooAiCKLoxQiiTIbxMl0QQcSbNyggF1GCKJefCIQpIghBEiIJ\nyen3j9rNaU/6nO7O6b367O7P53n6qe6qVVVrde2qXfu71147/zFo/wAAAAAAh4RRoexdMw1BcHZ3\nP3zQPgEAAAAADjmHDdrPtWfT3xy0PwAAAACAQ9KoUPay2fTTg/YHAAAAAHBIGhXK/uNsesKg/QEA\nAAAAHJJGhbJ/lGQpyWmD9gcAAAAAcEgaFcq+OMnbkjy5qu40aJ8AAAAAAIecI0bspLuvqKofSfKs\nJOdU1SuTvCHJR5JcvMFtfHKOVQQAAAAAGGJIKFtVFyY5fPbn1ZI8dPazUcsZVFcAAAAAgHkaFXRe\nc5+/lwbtFwAAAADgkDIqlP3jTL1dAQAAAAB2tVFjyj58xH4AAAAAAA51h/Q4rVV1RJKTk3yhuz+z\n3fUBAAAAADhYh213BdZxbJLzkrx+uysCAAAAALAVDvVQ9q6z6c23tRYAAAAAAFtk6PAFVXVikocl\nuV2SE5Mcvp+iS0lOSnKzTF8QdtGQCgIAAAAAzNmwULaq7pXk1UmO2+AqS6t+f+PW1wgAAAAAYLwh\noWxV3TDJa5Ick/8cth7IRUn+McmbkzxnTlUDAAAAABhq1Jiyj830pV2XJ3lqkkpyje5e2f/ybPnV\nk9w2ydmz+S/u7qd3938MqicAAAAAwFyNGr7g1EzB65O7+zf2V6i7v5bkb5M8rKp+Isnzq+pbuvsp\nY6oJAAAAADBfo3rKfuts+uaNrtDdz0/ypiS/XFV3mEutAAAAAAAGGxXKHjWbfmaNZcuz6TFrLDsr\n0xi0Pz2PSgEAAAAAjDYqlL1oNv2mNZZ9YTY9cY1lH5xN77rlNQIAAAAA2AajQtl/mk1PW2PZ52bT\nu62x7Oqz6VqBLQAAAADAwhkVyv55pmEIfrWqfq6qrrtq2Ydny36iqo7cZ70fnk0vHlBHAAAAAIC5\nO2LQfp6f5GeSHJvkWUnuk71DErwxUw/aWyR5T1X9YZILk3xXkjMzjTn7d4PqCQAAAAAwV0N6ynb3\nvyS5f6Yer0tJPrtq8SuTfGA2/zuS/F6Slyd5bJKVnrO/O6KeAAAAAADzNmr4gnT3nyW5WZJfTXLu\nqvmXJ/mBJH+TKZhd/XNpkp/r7j8ZVU8AAAAAgHkaNXxBkq/3mH3KGvM/k+S7quq/JLlVkmOSfDrJ\n27r78yPrCAAAAAAwT0ND2fV09zuTvHO76wEAAAAAMC/Dhi8AAAAAAGAbespW1dGZxpC9Z5KTk1wn\nybHdfa19yt29u/9idP0AAAAAAOZpaChbVQ9K8ltJVgewS0mW9yl3qyRvq6q/SHK/7v73cbUEAAAA\nAJifYcMXVNVPJzk7UyC7NPv51/0Uv8ds+d2TvGZIBQEAAAAABhgSylbVzZI8M1PQ+ukkj0pyUpKb\n7meV5yR5xqz8qVX1gyPqCQAAAAAwb6N6yj4myZFJ/jnJrbv7hd39hewzbMGK7v5ad/9SkjdkCmYf\nOqieAAAAAABzNSqUvWemAPZXZmHsRp01m95266sEAAAAADDeqFD2urPpX25yvZ5Nr72FdQEAAAAA\n2DajQtlrzKaXb3K9y2bTNYc5AAAAAABYNKNC2X+dTW++yfVuOZt+bgvrAgAAAACwbUaFsu/J9IVd\n/3WjK1TVUpJfzNRL9q/nVC8AAAAAgKFGhbIvnU1/uKp+v6qOOVDhqvrmJG9IctfZrJfNs3IAAAAA\nAKMcMWIn3f3WqnpLku9L8pgk96+qP09ywUqZqnp8kpMyDVlw1yRXmy16a3e/ZUQ9AQAAAADmbUgo\nO/PAJK9Ocu8kxye532z+ypd4PXNV2aXZ9O2rygEAAAAALLxRwxekuy/q7u9L8qAk780Uxi6t8bOc\n5F1JHtLd9+ruS0bVEQAAAABg3kb2lE2SdPerkryqqr4hyXckOTHJMUkuTvK5JH/f3V8ZXS8AAAAA\ngBGGh7IruvvLSf5qu/YPAAAAALAdhg1fAAAAAACAUBYAAAAAYKgtH76gqj621dtMstzdN5nDdq+k\nqm6f5LFJ7pjk2km+mqSTvCHJc7fqi8eq6ppJPpTkm5Oc39033ortAgAAAACHtnmMKXujJMtJlrZw\nm8tbuK39qqpfTPK02f4uTfLxJMcluW2S2yU5o6q+p7sv2ILdPTdTIDukbQAAAADAoWGeX/T1tST/\nJ8lH57iPLVNV98neQPY5SZ7c3V+ZLbttklckuWmS1yS50xbs6yFJLs82ftkaAAAAADDePALBLyY5\nIcnVMoWX103y5iSv6+6/msP+tsqzMwWyb+rux69e0N1/U1WnJ/nbJLevqtO7+zVXZSdVdXySP5jt\n6yVJfvygag0AAAAALJR5fNHXdZL8UKYepZcmuXGS/5bk3Kr6ZFU9s6puMYf9XmVVdack3zb789lr\nlenu85KcM/vz4Qexu7OSXCvJq5K86yC2AwAAAAAsoC0PZbv78u7+k+6+f6YvyvrxJOdm6hl6vSSP\nT3JeVX2wqp5QVdff6jpcBd89m16c5K8PUO7tmcbKvctV2UlVnZbkAUn+NVNQDQAAAADsMvPoKft1\n3X1Rd7+ou09NcoMkT0jywUzB5s2TPD3Jx6vqHVX1qNlH+7fDKbPpR7p7zwHKfXg2PbqqajM7qKoT\nkzwvUzh9Znd/cfPVBAAAAAAW3VxD2dW6+4Lu/s3uvmWSWyR5VpJPz+pw5yTPT/IvVfX6qrpvVV19\nVN0y9eBdntXnQFYvv94m9/G8JCcleUV3v3GT6wIAAAAAO8SwUHa17v7H7n5id98w09ABL0ry5SRH\nJvnhJK9O8tmqemFVfc+AKh03m16yTrnVy4/bb6l9VNX9ktw3yQUxbAEAAAAA7GrbEsqu1t3v6O4f\nzzT+7I8mOTvJF5JcM9MXar29qj4152ocNZtetk65S1f9fvRGNlxVJ2X6cq/lJI/p7i9tvnoAAAAA\nwE5xxHZXYEV3X5bkDVV1fqaxWx+daRzapSTXnfPuV3rAHrlOuWussc56npfkxCQv7e4/3WzFtsrx\nxx+dpaWlK81fOuzK8xbR0mFLOeGEYzZc/rA1/heL6LClTbZ7hxzvwzZ5vNd67C+ipU0e793a7t36\nON+117Xd2u7d+jjfpe1e2vZuFFtj6bA4vzdgtz5/7952z7EyAy0tbfL83qXX8916Xdu1x1u7F9pm\n231VHBKhbFWdkuShSR6QvQHsylF8X6bes/N00Wy63n/72FW/f3m9jVbVA5Oclmks2sdetaptjSOO\nOHzN+TvjVJnacfjhm3jFsoMavql27yCO9waK75B2L3mcb8wOOd6ua7uPdq9vaYec4EtZcrw3Ymcc\nbvctG19jbnUZy/m922j37qLd87FtoWxVXSfJgzOFsbeYzV55RvpEkpcnObu7e0B1PpHkDln/y7tu\ntOr3jx2oYFVdK8n/SHJFkjO6+6I1ig17Br788ivWfPd5eVQF5mw5yRVX7NncCjvB8ibbvYM43hso\nvkPavexxvjE75Hi7ru0+2r2+5R1ygi9n2fHeiJ1xuN23bHyNudVlLOf3bqPdu4t277WVQe3QULaq\njsrUc/ShSU7NNKbtSlL470lemymI/cuR9UrygUy9dG9aVUd09+X7KXfKbPql7j5/nW3eO9OwBck0\nLu6Byt6oqlaO9Eu6+xEbqPOmXHjh2qMtLO/ZGTcBy3uW88UvfmXD5ffskLu+PcubbPcOOd57Nnm8\nl3fI8V7e5PHere3erY/zXXtd263t3q2P813a7uUd8jpoeU+c3xuwW5+/d2+751iZgZaXN3l+79Lr\n+W69ru3a463dC21/7T7ppOO2bB9DQtmqOjVTEHta9g4RsJTka0nelml4gjd196Vrb2Hu3pbkGZm+\nvOsuSc7ZT7l7Z3or8y0b2ObXkqz3pV5Hzva5J3uHQ9j4Ix0AAAAAWDhzC2Wr6uQkP5bkQUm+ZTZ7\npVfsezMFsa/q7i/Mqw4b1d3nVdX7k9wqyS9kjVC2qu6e5DaZQtkXbmCbr0zyygOVqaqHJXlxkk92\n942vQtUBAAAAgAWz5aFsVf1Mpl6xt5zNWgliP5oppDy7uz+y1fvdAo9L8v8luUdVnZXkiSvjwM56\n+p6dKZB9fXefu7JSVd0g0/iyy0me2t1PG11xAAAAAGBxzKOn7G9n9r1LSd6d5K1J/qS7PzSHfW2Z\n7v7LqjozyVlJfiLJI6rq/CTXTHKdTG16Z5Iz9ll1KdPYuMuzKQAAAADAfs1zTNnDk9x59vP0db7o\naj3L3T338W+7+wVV9e4kP5vkbklulGmM13OTvCzJi7t7rRGLD2YU4+WDXB8AAAAAWCDzCjqX1i9y\naOruf0jyiE2U/0SmAPqq7OulSV56VdYFAAAAABbTPELZp85hmwAAAAAAO8KWh7LdLZQFAAAAANgP\nX0wFAAAAADCQUBYAAAAAYCChLAAAAADAQEJZAAAAAICBhLIAAAAAAAMJZQEAAAAABhLKAgAAAAAM\nJJQFAAAAABhIKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBAQlkAAAAAgIGEsgAAAAAAAwllAQAAAAAG\nEsoCAAAAAAwklAUAAAAAGEgoCwAAAAAwkFAWAAAAAGAgoSwAAAAAwEBCWQAAAACAgYSyAAAAAAAD\nCWUBAAAAAAYSygIAAAAADCSUBQAAAAAYSCgLAAAAADCQUBYAAAAAYCChLAAAAADAQEJZAAAAAICB\nhLIAAAAAAAMJZQEAAAAABhLKAgAAAAAMJJQFAAAAABhIKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBA\nQlkAAAAAgIGEsgAAAAAAAwllAQAAAAAGEsoCAAAAAAwklAUAAAAAGEgoCwAAAAAwkFAWAAAAAGAg\noSwAAAAAwEBCWQAAAACAgYSyAAAAAAADCWUBAAAAAAYSygIAAAAADCSUBQAAAAAYSCgLAAAAADCQ\nUBYAAAAAYCChLAAAAADAQEJZAAAAAICBhLIAAAAAAAMJZQEAAAAABhLKAgAAAAAMJJQFAAAAABhI\nKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBAQlkAAAAAgIGEsgAAAAAAAwllAQAAAAAGEsoCAAAAAAwk\nlAUAAAAAGEgoCwAAAAAwkFAWAAAAAGAgoSwAAAAAwEBCWQAAAACAgYSyAAAAAAADCWUBAAAAAAYS\nygIAAAAADCSUBQAAAAAYSCgLAAAAADCQUBYAAAAAYCChLAAAAADAQEJZAAAAAICBhLIAAAAAAAMJ\nZQEAAAAABhLKAgAAAAAMJJQFAAAAABhIKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBAQlkAAAAAgIGE\nsgAAAAAAAwllAQAAAAAGEsoCAAAAAAwklAUAAAAAGEgoCwAAAAAwkFAWAAAAAGAgoSwAAAAAwEBC\nWQAAAACAgYSyAAAAAAADCWUBAAAAAAYSygIAAAAADCSUBQAAAAAYSCgLAAAAADCQUBYAAAAAYCCh\nLAAAAADAQEJZAAAAAICBhLIAAAAAAAMJZQEAAAAABhLKAgAAAAAMJJQFAAAAABhIKAsAAAAAMJBQ\nFgAAAABgIKEsAAAAAMBAQlkAAAAAgIGEsgAAAAAAAwllAQAAAAAGEsoCAAAAAAwklAUAAAAAGEgo\nCwAAAAAwkFAWAAAAAGAgoSwAAAAAwEBCWQAAAACAgYSyAAAAAAADCWUBAAAAAAYSygIAAAAADCSU\nBQAAAAAYSCgLAAAAADCQUBYAAAAAYCChLAAAAADAQEJZAAAAAICBhLIAAAAAAAMJZQEAAAAABhLK\nAgAAAAAMJJQFAAAAABhIKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBAQlkAAAAAgIGEsgAAAAAAAwll\nAQAAAAAGEsoCAAAAAAwklAUAAAAAGEgoCwAAAAAwkFAWAAAAAGAgoSwAAAAAwEBCWQAAAACAgYSy\nAAAAAAADCWUBAAAAAAYSygIAAAAADCSUBQAAAAAYSCgLAAAAADCQUBYAAAAAYCChLAAAAADAQEJZ\nAAAAAICBjtjuChxqqur2SR6b5I5Jrp3kq0k6yRuSPLe7L7mK2z0tyY8luW2Sb0pyaZKPJ/nfSc7q\n7o8efO0BAAAAgEOdnrKrVNUvJnl3kvsnuVam0PTiTEHqM5L8XVVdd5PbPKqq3pLktUl+MMmJSc5P\n8rUk357kZ5J8sKruu0XNAAAAAAAOYULZmaq6T5KnJVlO8pwkJ3X3yd19/SS3T/LRJDdN8ppNbvpF\nSe6dZE+SX0pyze6u7j4xyV1m271GkpdW1fW3pDEAAAAAwCFLKLvXszMFsm/q7sd391dWFnT33yQ5\nfbb89lV1+kY2WFUnZ+p1u5zk6d39G9196artvivJg2d/XiPJGVvSEgAAAADgkCWUTVJVd0rybbM/\nn71Wme4+L8k5sz8fvsFN3ybJ55NckeSP9rPd9yb5xOzPW25wuwAAAADAghLKTr57Nr04yV8foNzb\nkyxlGnZgXd19dndfO8nVu/uTByj6tdn06hvZLgAAAACwuISyk1Nm0490954DlPvwbHp0VdVGN97d\ny/tbVlUnJrnR7M8PbXSbAAAAAMBiEspOrpdp3NdPr1Nu9fLrbdG+n5jkiCSXJ3nBFm0TAAAAADhE\nCWUnx82ml6xTbvXy4/ZbaoOq6keSPC5TIPy73f2Rg90mAAAAAHBoE8pOjppNL1un3KWrfj/6YHZY\nVQ9N8qpMY9S+NVOPWQAAAABghxPKTlZ6wB65TrlrrLHOplXVLyd5aaZhC96W5PR1xrIFAAAAAHaI\nI7a7AoeIi2bTY9Ypd+yq37+82Z1U1ZFJXpTkQZmGLHhxksd09xWb3dZmHX/80VlaWrrS/KXDrjxv\nES0dtpQTTljv8O112Br/i0V02NIm271Djvdhmzzeaz32F9HSJo/3bm33bn2c79rr2m5t9259nO/S\ndi/tkG4US4fF+b0Bu/X5e/e2e46VGWhpaZPn9y69nu/W69quPd7avdA22+6rQig7+USSO2T9L++6\n0arfP7aZHVTVN2QapuAOSa5I8gvd/ZzNbONgHHHE4WvO3xmnytSOww/fxCuWHdTwTbV7B3G8N1B8\nh7R7yeN8Y3bI8XZd2320e31LO+QEX8qS470RO+Nwu2/Z+Bpzq8tYzu/dRrt3F+2eD6Hs5ANJHpDk\nplV1RHdfvp9yp8ymX+ru8ze68ao6KslbMgWyFyV5QHe/9SDqu2mXX37Fmu8+L4+sxBwtJ7niik2M\nALGDGr6pdu8gjvcGiu+Qdi97nG/MDjnermu7j3avb3mHnODLWXa8N2JnHG73LRtfY251Gcv5vdto\n9+6i3XttZVArlJ28LckzMn15112SnLOfcvfO9Kz5lo1uuKoOT/KmJHdM8sUkp3b3Bw6qtlfBhReu\nPQTu8p6dcROwvGc5X/ziVzZcfs8Ouevbs7zJdu+Q471nk8d7eYcc7+VNHu/d2u7d+jjftde13dru\n3fo436XtXt4hr4OW98T5vQG79fl797Z7jpUZaHl5k+f3Lr2e79br2q493tq90PbX7pNOOm7L9rE7\n+x/vo7vPS/L+TJ8d+YW1ylTV3ZPcZvbnCzex+ackOTXJV5LcazsCWQAAAADg0CGU3etxmXrB3qOq\nzqqqr0ffVXVqkrNny1/f3eeuWnaDqrq8qr5WVU9avcGq+tYkT5it94Tu/tsB7QAAAAAADmGGL5jp\n7p0na60AACAASURBVL+sqjOTnJXkJ5I8oqrOT3LNJNfJFKy+M8kZ+6y6lCncXs6VQ+7HZu//+Myq\netQG6nGrq9gEAAAAAGABCGVX6e4XVNW7k/xskrsluVGmYQfOTfKyJC/u7rUGx9jfgBnfuGrZyRuo\nws4YeAMAAAAA2C+h7D66+x+SPGIT5T+R5PD9LDsjV+5ZCwAAAADsYsaUBQAAAAAYSCgLAAAAADCQ\nUBYAAAAAYCChLAAAAADAQEJZAAAAAICBhLIAAAAAAAMJZQEAAAAABhLKAgAAAAAMJJQFAAAAABhI\nKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBAQlkAAAAAgIGEsgAAAAAAAwllAQAAAAAGEsoCAAAAAAwk\nlAUAAAAAGEgoCwAAAAAwkFAWAAAAAGAgoSwAAAAAwEBCWQAAAACAgYSyAAAAAAADCWUBAAAAAAYS\nygIAAAAADCSUBQAAAAAYSCgLAAAAADCQUBYAAAAAYCChLAAAAADAQEJZAAAAAICBhLIAAAAAAAMJ\nZQEAAAAABhLKAgAAAAAMJJQFAAAAABhIKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBAQlkAAAAAgIGE\nsgAAAAAAAwllAQAAAAAGEsoCAAAAAAwklAUAAAAAGEgoCwAAAAAwkFAWAAAAAGAgoSwAAAAAwEBC\nWQAAAACAgYSyAAAAAAADCWUBAAAAAAYSygIAAAAADCSUBQAAAAAYSCgLAAAAADCQUBYAAAAAYCCh\nLAAAAADAQEJZAAAAAICBhLIAAAAAAAMJZQEAAAAABhLKAgAAAAAMJJQFAAAAABhIKAsAAAAAMJBQ\nFgAAAABgIKEsAAAAAMBAQlkAAAAAgIGEsgAAAAAAAwllAQAAAAAGEsoCAAAAAAwklAUAAAAAGEgo\nCwAAAAAwkFAWAAAAAGAgoSwAAAAAwEBCWQAAAACAgYSyAAAAAAADCWUBAAAAAAYSygIAAAAADCSU\nBQAAAAAYSCgLAAAAADCQUBYAAAAAYCChLAAAAADAQEJZAAAAAICBhLIAAAAAAAMJZQEAAAAABhLK\nAgAAAAAMJJQFAAAAABhIKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBAQlkAAAAAgIGEsgAAAAAAAwll\nAQAAAAAGEsoCAAAAAAwklAUAAAAAGEgoCwAAAAAwkFAWAAAAAGAgoSwAAAAAwEBCWQAAAACAgYSy\nAAAAAAADCWUBAAAAAAYSygIAAAAADCSUBQAAAAAYSCgLAAAAADCQUBYAAAAAYCChLAAAAADAQEJZ\nAAAAAICBhLIAAAAAAAMJZQEAAAAABhLKAgAAAAAMJJQFAAAAABhIKAsAAAAAMJBQFgAAAABgIKEs\nAAAAAMBAQlkAAAAAgIGEsgAAAAAAAwllAQAAAAAGEsoCAAAAAAwklAUAAAAAGEgoCwAAAAAwkFAW\nAAAAAGAgoSwAAAAAwEBCWQAAAACAgYSyAAAAAAADCWUBAAAAAAYSygIAAAAADCSUBQAAAAAYSCgL\nAAAAADCQUBYAAAAAYCChLAAAAADAQEJZAAAAAICBhLIAAAAAAAMJZQEAAAAABhLKAgAAAAAMJJQF\nAAAAABhIKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBAQlkAAAAAgIGEsgAAAAAAAwllAQAAAAAGEsoC\nAAAAAAwklAUAAAAAGEgoCwAAAAAwkFAWAAAAAGAgoSwAAAAAwEBCWQAAAACAgYSyAAAAAAADCWUB\nAAAAAAYSygIAAAAADCSUBQAAAAAYSCgLAAAAADCQUBYAAAAAYCChLAAAAADAQEJZAAAAAICBhLIA\nAAAAAAMdsd0VONRU1e2TPDbJHZNcO8lXk3SSNyR5bndfcihtFwAAAABYLHrKrlJVv5jk3Unun+Ra\nST6e5OIkt03yjCR/V1XXPVS2CwAAAAAsHqHsTFXdJ8nTkiwneU6Sk7r75O6+fpLbJ/lokpsmec2h\nsF0AAAAAYDEJZfd6dqbg9E3d/fju/srKgu7+mySnz5bfvqpOPwS2CwAAAAAsIKFskqq6U5Jvm/35\n7LXKdPd5Sc6Z/fnw7dwuAAAAALC4hLKT755NL07y1wco9/YkS0nuss3bBQAAAAAWlFB2csps+pHu\n3nOAch+eTY+uqtrG7QIAAAAAC0ooO7lepnFdP71OudXLr7eN2wUAAAAAFpRQdnLcbHrJOuVWLz9u\nv6Xmv10AAAAAYEEJZSdHzaaXrVPu0lW/H72N2wUAAAAAFpRQdrLSU/XIdcpdY411tmO7AAAAAMCC\nOmK7K3CIuGg2PWadcseu+v3L27jdTTv++KOztLR0pflLh1153iJaOmwpJ5yw3r95r8PW+F8sosOW\nNtnuHXK8D9vk8V7rsb+IljZ5vHdru3fr43zXXtd2a7t36+N8l7Z7aYd0o1g6LM7vDditz9+7t91z\nrMxAS0ubPL936fV8t17Xdu3x1u6Fttl2XxVLy8vLc93BIqiqVyR5QJL3d/dtDlDu9CSvzvTlXTfp\n7vO3Y7sAAAAAwOLaIe+7H7QPzKY3raoD9R4+ZTb90gaD03ltFwAAAABYUELZydtm06OT3OUA5e6d\nqTfrW7Z5uwAAAADAghLKJunu85K8P8lSkl9Yq0xV3T3JyhAEL9zO7QIAAAAAi0sou9fjMvVWvUdV\nnVVVx60sqKpTk5w9W/767j531bIbVNXlVfW1qnrSVm0XAAAAANiZfNHXKlX16CRnZQqrL0tyfpJr\nJrlOpuD0nUm+v7svXrXODZN8fLb8qd39q1uxXQAAAABgZ9JTdpXufkGSWyV5aZJ/SXKjJEcmOTfJ\no5LcbT/B6QGT7YPYLgAAAACww+gpCwAAAAAwkJ6yAAAAAAADCWUBAAAAAAYSygIAAAAADCSUBQAA\nAAAYSCgLAAAAADCQUBYAAAAAYCChLAAAAADAQEJZAAAAAICBhLIAAAAAAAMJZQEAAAAABhLKAgAA\nAAAMJJQFAAAAABhIKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBAR2x3BQB2iqq6QZI93f3p7a4LAAdW\nVUcluW6S45IcneSSJF9O8qnu/tp21g3mrapOSHKtJJ/r7i9ud31GqqpfSfLV7n7WdtdlhKr6liTH\nJvlCd//bdtcH5qWqvjHJdZJ8sbs/t931YXtU1fW7+1PbXY+NWlpeXt7uOrDLVNUxSZ6bZLm7H7nd\n9bkqqurqSSrJ1ZP8U3f/+wbXe3SS63T3r86zfvNSVTdOcpMkFyT5v929Z5/lD0tyWpJrJ/lcknOS\nvLC7Lxpd1+1QVXuSXNjdJ253Xeapqm6d5EFJviOzm/wk70vyiu7+p+2s2zxV1Tcl+f4kt8jU7guT\nnJfkzzZ6DTjUzF6YXpbkj3brC7XZmymV5F+SfGiN69r9kzwgyTcn+WKSdyR5QXdfOLqu81BVN0ly\n8yQrL9oPT3JRpvP6HzNd6y/fvhpuraq6bZKHJ/meJN+WtT819rUkH0ryp0le0t0fHVbBQ0hVnZjk\n85nebFy4jhyz+80fTXLLTPdrH8r0PLXuuVtVf5rkZt19k/nWcuvNzumHZe/92lu7+5xVy2+a5A+S\n3GXVah9M8uvd/ZqRdd0us/u1L3X3Cdtdl6uqqr4hyRXd/ZX9LL92kl/K9Py1+r70k0leluS3uvtL\nc6/onMzuya7b3X+/z/xrJPnxTOf+vvdrr0tydndfOri6B62qPpbpfu1Z3f2i7a7PaFV1/SRnZO/9\n2pu7+9xVyyvJ7yS5R5Kl2eyPJHlad79sbG23VlUtZbpe3z0Hvl97X6b/y8K+7q6qpyZ5Sndf5ZCy\nqh6U5PcW6foulGW4VTf5y919+HbXZzNmvWqenelm96jZ7CuSvDzJL3X3Beus//4kpyxgu6+b6Qbu\nrqtmfzzJA7r7fVV1eJLXJrnPbNlSkpWLy2eT/GB3/92o+m6XHXKT//okH+vux6+x7OqZXsg9dNXs\n1cd6OdMbLo/v7ivmXdetNus19Nwkn+nuX1g1fynJU5P8XJJrrLHqJUl+M9ON3541lh+yZo/Z5Uwv\nWJ7Q3S/c5ioNM3suemmS7101+6NJzuzu/11VhyX5kyT3ni1b/Vj/fJIf6O73jarvVqqq45I8PtO5\nfMN1il+S6f/wvO7+q3nXbV5mbX5BkvvNZi0doPiK5UzP8b+b5ImLeF07GAt+v3anTPcl19pn0ZeS\n/HJ3//466y/q/dqDkrwoydX2WfS07n5yVV0v0wv3k3Llc2A5ya9096/Pv6bba4fcr+1J8v7uvs0a\ny74jyZ9levyvda1bTnJ+knt390fmWc95qKofSvLiJK/r7ketmn+TJG9OctPZrNVtX3n+/nCSH+ru\nfx5R162y6n4tSd6Z5DHd/eFtrNIwVXWfJK/Mle/BX9Tdj6qqGyb52yTHZ+3r2lO6+9fmX9OtV1Wn\nJ3lm1r5XW+vxfVmm1+y/3N2fnXP1ttzscf7OJA/ebE/X2X3e72fqOJRFev4WyjLcot7kz4KZczK9\nU7XWBf9LSU7r7nccYBsLd5M/C+Len+mdyX3b/fkkJyd5SJLnzOZ9LMm/Jrl+kuvN5l2Q5NsX6R35\nqvqxq7DaSzIFGD+5emZ3//FW1GmE2ZPhed196zWWvTJToLGUZE+mG9sLM930f1v2hlYv7O5HD6v0\nFqiqo5O8N8n/m6nn6/evWvYHmXpdrDz+v5LpfL9mpneqk6ndL+vuhw2r9BaYHe8rMr3bvpzkXZlu\n9P/vtlZszqrqiCTvSXKrXPm6dkmSuyW5Z5KnZfq/fDzJvyW5QaaPxSXJZ5LcvLu/PKDKW6aqbpWp\nB+i1s/aL9YuTXJrkG/Kfg53lTG9APnLRPto/O97vSHL7TG3+5yR/keka9ulMx/zSTC/4jk1yoySn\nZOqVcu1MbX9dd99v323vZAt8v3adTL1ir5npeF+e6TnryFmR5SR/mOQn9/dG2oLer90kU4/XleBi\npbfUcZnafMckj87U2+xjme5ZVu7XHpTkxpmeD27X3e8fVvGDVFXnrF/qSu6W6XHxzlXzlrv71C2p\n1AD7u1+b3bd3puerJPnzTNe7lfu1u2Zvb8LO9DhfmGt6Vd0sU6/XI5O8o7u/ezb/apke/yuB7D8k\n+atMn3D5xiTfleQ7Z8vOT3KL/fUyPhTNjvfnkvxHpoDuskwdhX69u/9jO+s2T7Mesv+YvffbF2a6\nNzk203XtJzOdz/fPFMy+MXvv134wU8/SK5Lcurs/OLLuB2v2abYn5z+/7vq3TK+vb5ypx/CrMj2+\nK8ltMn0qZDnTc/f3LtK1PPlPbz78e5JHd/drN7jeHZOcnen+bSnJu7r7zvOq51ZbuI8iwTZ6cPb2\nFP2zJK/O9BHH78x0g3t8kj+rqvt2959uTxXn4owkN8t0gXxBppu7pST/Lcmdk5yZ6X9zcaaes29Z\nWbGq7pXpRfw3Z3ohsEhjd70ke9913IyjMr17v2I5ycKEsvtTVXfOdMOznOkG4PGre4ZX1TcneVKm\nx8Mjq+qPu/uda27s0PTITG8wXJSpd1WSpKrukORRmdr9xiS/lqlnyvJs+clJfnpW5iFV9T8X8Pz/\nYJJfSfK8JP8lyXlVdVamnlU7dZzBhya5daYXN8/O1HPsaknulSmA//lM1/ZPJblPd39gZcWqun2m\nc+D6s7K/PbTmB2H2cc+3JfmmTB9hfW6msPLfMrXniZkeA2cmeX2mm/47J3lgpoDywZmCrh8aXfeD\n9ONJ7pCpnY/q7jduZKXZp0DOyPSm449W1Wnd/fr5VZMt8lOZXqR+PtO9yuu6+4qqOiXT89R9M12z\nr5bp2r9TnJkpkP1okod293tmHQoek+SsTPdhp2V6o/2u3X3xyopV9fQkb8/e4PbMwXU/GHfL3vu1\njfSAX3HEbN0VO6Wn0kMzBVJfTfKj3f3WfZY/o6q+K8mbMgWYj0zy/LFVPChnZgpk351paIYVD83U\nnkuSPKS7/9e+K87u6V6bKdR8bJJnzL22W+uCTM/JT890nfvvSR4+C+9efDAf+T6E/WSmAPa9SR7Y\n3R9Pvn4s/2em/8ONkrxmtvzrb7RV1ZMy9bA9PdN18L8OrflBqKq7ZG8g+5wkz1w9Ru5saK0XJrle\nd99/Nu+oTNf4X850Lry5qk5esCHW7pfpenRCkldX1YuSPHZ/bzzMPtX2lEz3r0dkerPiKVmszEEo\nC5vwwEw3bL/d3T+/av4rqurZmZ4M7pDpAvI93f3e7ajkHJyWqd1P6O5nr8ysqj/J1HP4QZku/D+7\nOpBNku5+W1X9fKYnje/Pgl0gZ67I1CNuI26Y6X+1MAOLb8L9Z9M/7e4H7buwu/8lyU9V1VeTPC5T\nCLJIoewDMh27R3T361bNf8hs+vruPn3flbr7Q0keU1UXZLp5enSmXogLpbvfXFXnZvqI1E9kCprP\nqKrfyTQu0xe2s35zcL9Mx/vh+4yh+Pqq+kKSJ8z+vtfqQDZJZkHHIzL1PPrBLFAomyls/qZM5+b3\nrQ5lMvUc+quqekWmN5I+2N2dqVfpi6vqnpnC6B+oqtMXbOzJlfP7Id395xtdaTZcwR9V1Zcztf0R\nmcLqhXAVP/Gx2rHrFzkk3TvT8X5kd795ZeZs7Mn7VdVPZzpvH15Vn+7uJ29TPbfaqZnafUZ3vydJ\nZiHN82fjLT4yyTFJfm6fcz/d/dWq+qlMPRAXpnfRzCWZvqjv8iSvyPTJhvU8OVNo+cw51mu73DPT\n4+BX1whkkyTd/X+q6jGZrmf3zWKFsvfM1Gvwx7r7X1fN/75M7X7SWoFsknT3X1fVgzO9fjktixfK\nZta796dnz9V/mOTbZ9NfnL0efekO6zl7j0zH9cErgWzy9WP5uEzB7HKmziL/6ZMP3b2nqn4202N8\n9RB8i2AlQH7yWkMvdPerZ8Ou/V5V3a+7/+fsuL+8qv5XkrcmuVOSn8k0BNtC6O7XVtU7M3Vwulem\n+647VdUD970fr+m7bl6e5HaZwusPZnpD8u+zYISybFpV7aox1Va5daaA7in7LujuC6rq1Ewv0u+Y\n5I1VdbvNjoVyiDolU4/g3109s7svq6rfTPKGTP+XF6+xbjKN2fi8TB8LXyRPzPQkdmSmgeJ/trv/\n4UArzD5y8eXu/tYB9Rvt5Ew3PU9bp9yvZW8v6kVyk0zf1Pq6febfPFO71xuL6jcyBV63m0Pdhpi9\nSP+pqnp5kv+R6Zr3K0l+vqpelenLMfY7PMuCuWWSf99PsPiCTOf/l7r7L9ZaubvPqaoLM50Xi+QH\nMr2Yfdi+ocwqT8z0JszPZFVvue7+86p6VKY3IM+YTRfFLZJ8fjOB7D5ek6mn4W23rkpDvCQ7p/ff\nZvw/SS5eHciu1t2/O+tR9PQkT6qqD3X3q4fWcD6+NdN1ba03RF+a6c22/8jUO/5Kuvvvq+pL2fux\n90Vxi0z3oHfJ9AbMb2UaQ3K/H8mvqicn+Wp3L0xYsQkrQ4e9Yp1yb8w0FNO3z7c6W+5bknyyuz+2\nxvxk6hm5X919blV9NtN1YmHNgvVbZ+pJ+qRMn2w5K8kzq+p1mXoEn9vdl2xjNbfCTZJ8qtf+ws2V\ne7RP7e81d3d/pqo+kenTQIvkjpmGSztQh6Y/zPQG4xmZwukkU3BfVWck+ackP5IFCmWTpKexcL+3\nqn4yU/tvluQ9VfXfu/t3kqSqHp4pm1gZxuI3M42juzBDsay21jfOwnqWtuBnEZ2Q5BP7G3+ou7+a\n6YuuPpJpDLo3zsapXHTHZ7r5Wesi967Z9HP7G1dx9q7l+Zk+8rowuvtZmUKpv830sd33VdUvzT4m\nsRsdM5t+4ECFeho3+GOZzoFFcs1MY3Xta2U8wQOOsdrTN/l+ItP5stC6+93d/Z2Zegn/c6YeSGck\nOaeqPltVL6uqR1XVHavqxNnHYxfNCdl/D/iVG/v1eshfkOkj0ovkBkk+vrq3yb66+5OZ2n6PNZa9\nLtO3/F5pzOlD3LGZ6n2VzHobfi4L9jy2ym67Xzs601h7+9Xdv5HpBe1SkhdV1XceqPyCODrTl6uu\n5f9v777DJKuq9Y9/e4YcBIYsSJYFShRBfl6yREGCIkFASSKXJBgICihIkig5SxL0gqCAiSzcSzIg\nSWQBkiRIHDIMjFO/P9Yu+kx1pe6uOtWnzvt5nn461DlV+62q7j61ap+1q4saPd/iFOfniZ6EheHu\nT7j7WsQbSZOJN5buSQWrMqpOnmm6+HB6HjxL8Y5b/kO8uVir+jqknVO136APJqe5+2R3P4UoXB5K\ntOiZlViQ+lrgVTO73czOMbNvm9mOZrZlD4c8EjMz+NjWerPmcyNvM7hAd1HMTRSbJzXawN0nE6+x\nV6xz2T+JswYKO1HIY0HOTwF/Jv4vnWBmvzOzK4izcGclMq7h7gcWtSALKsrKyDxEvCNxM/HOy3A/\njst/yB3xHi1ekKX+ixsTL/6Wp/W71EXwPoMFuVrVF7kvtbiOGYjZGYXisdjRqsQ70ACHE+/UFW3W\nbydUC5btHNQMULz/L88DC5lZ7cquT6bPC9BEWkhofkZR+Blr3P0y4t3pLxNvwAwQi4RsS5zq+L/E\n82KymbU6IB5rJtH4uTxzzedGZiH+L/Sj94CPNrjsFYpXnHwZ+JiZTddyyzrSwjkL0fp/3VhzPvF7\newXxwmy4H0NWdS+IicTj3er/0B7ATcTfgqvTgjJF9joN/ldlZsY3XJgwvcE2P+0VtcacVJxageg1\n+knieO1H6f9zmVTfdGv0NzxrTop3fP4YsIiZ1a5Gf2/63PQMlrQQ4GK035pszHP3N939CKKN2q7A\nncTf/umI1nq7EDMOzyda8RTJ6wwusFqr+veu0eVVH6XJ374x6k2iMNvKHMTCrPVMR8HPlnH3R4hZ\nw4cRb8hsQLQeATgbWN7d7+jR8DqmaC+aZWz4KvFLsRxwlrsfNpwPitlXFGL235wWK1c35O6PEb1r\nJgNfMLNz8hhcFz0BzGdmQ05nS++y30+TWYRm9nHixWwhD37cfYq7H0Us+nNv+nyPmR1Q0BmCI1Wd\nFf2ZZhulhYQWI1Z0LpIbiVlGe9f8/AriwHbDFvt/g5g1+ZfOD6133L3i7le6+xpE7+jDgHvSxdnZ\ndEU7K+Ap6r+oA1gnfV7YzOoWONLPFwae6dL4uuUZYNFmxad02WLUmXliZrMTuZvOQhyD/kQU2Ue6\nyMd30v63t9pwjNmPeK5vCSzl7k8N54NYDK6I7iXeDN6p2UapZ/CWxAr08wN/NLNFuj667nkAmMXM\nPt/g8v2IhQ0b2ZL4P9b0zJCxLB2Dr0H8zn4AfI8402n5ng4sX78l/i9v3mwjM1uNKGbVOy18LLuC\nqGGcW/NG2/nEY75nox1Tgf4c4iyo67o5yF5w9/fc/afuvhrxf3xvYsbscxT3DIj7idff9RYY3Tl9\nnmBma9fbOf18Dor3d+0hInfD1x9mtjkxWWJI64ZUr1iw3mVFk/5X/5rBY+4BovXKlY3OYC4aFWVl\n2Nz9HuBIYrGQC3s7mlxdR/wROMfMmr5zlfou7kScXrOLmV2W+pcV0R+J3D+pN9vA3Vdw923r7Zi2\nP5V4l+62bg6y21I/2VWInsLjiF50d5jZkr0cV44uIGZTHNyiGH08cbB7dy6j6pyfEAfzR5nZgWY2\nLYC7XwP8ATjSzIb0izWzWczsIGJl1ApwXo5jzpW7P5beXFuZOAhcD/gW8UZb0d58uon4PT4/LZQA\ngMXq7CcSb0a9AxzXYLbdccTjXaTF7ABuIXKfVW/WaJoR+jPib/6dNZeNI3oNT0/xXtycQ2Q61sxO\nMrMFW+0AsYiEmZ1NnCVRIf6fFUaaHVktTF5gZnP2cjw5+gXxeJ9iZjun53VdHqtSr0f03lsUuL1J\nUXOsu5rIfbaZDZnl7O4npzMghjCzVYn+fBWgbi/eokhvJp5InNJ7NzGJ5E9m9gMzG99878KZxczW\nyH4QM/pfAQ6pN6ECPlwg5zzi8b4hv+F2xKnEbNnPAXeb2aZmNm2aUbcP8DUzO9XMPmyjZWYzWyx8\n+DfijMZ3KNYincPm7k+6++nuvpm7f4yYFf0JYtZhkfyK+Lt2vpntbmafNLOVzOwE4PvEMfoT6fKp\n+r6nY7qfEs/z31EsvyZyX2Rma9VeaGZfIN6IqFCzwHD6O3AVxTxOHcJisba7iEkBbxCtIucA/mBm\nJ4z0LKixZKBSKfSMZumRdFBzN3HA8y13P7nFLtl95yQOGCruXpiDIzP7KPFCdBbgVeAaYNdmvbnM\n7ItEC4NpiRknswATCpZ7KWL2xTiin+jJ7n5Ri33mIFYl35c4lWwKsJLXrJpYVGnGxUXEgf57RGPx\nEywW+nrN3Sc0vYIxLGV4n/q9yOYjCjJ7uvtZNfutRsxIqa56vb6739Tl4XaUme1CnAozQJyWfw0x\nK3QiUYz/OPHipfo8XpQ4jeYjaZ9L3P1r+Y56dNLjfa+7l6r3XprB/wDxt/kd4GFiZt3SxGN5APH/\nbRtilsbFxO/EIsRsspWI5/nK6Y3KQkitV+4j3jh5lOip+QAxC/QTwO4MnvK6rrvfkvZbkSh0LZEu\n2yK9YVEYZnYmMaO9+j/biZkozxDPgfeJU/1mJmaXLMNg3gHgCHc/NM8xd4qZnUQs8vQbd990GPsV\n9XhtPPGmwqeJx/ttYLYWx2tzE2dMLEvmdM+C5Z6R+Fv2MSLDZe6+Q4t9ViXeZF6N6LH5AjGrupAt\nDGqlN5G/y+DCrfcCOwAP0h/Ha61eyB/s7kfX7Hci8HXib91bwNLuXqiz2cxsUWJ1+SUZ/B3/O/H3\nfFnieK3CYMuZuYjXMQNE+6Jt3P3qnIc9KiU+XpuBOA5bgqHP9+rp7J8h/o5NIZ4H1eO1JYnH/CXi\n79rEfEY9emY2C3F8tjCR+yEGj9eWJvoIDxDtmZZ19xfSfhsRb6wNEK0fVnT3J/MefyekN1YuIt44\nHSBqT9sSr9FOJ3onV4j7ZTt3/3uPhjpqmikrI5Kmke9AvLjbOvtuZL9y9+eIzB8Q7zZu3WKxNrik\nLwAAIABJREFUBNz9KmLxr1eIP6qFm6Xi7g8D+xN/DJcHvtTGbhsTMyurp4sd2y8FWYCU5dPAj4gX\nMcea2c29HVVHTUcczNR+zEA8D+q9iDmVODACuLhoBVkAdz8f2IiYMTUv0ZfrDGIlXyMKWRsQvw/7\nE71WZyP+JhwD7Jj7oGVE3P1R4hTHCnGAuxLRg3AccVbEScBBxOlRyxMzwC8jDvqrM9BOLFJBFj7s\nk/2d9O2SxCzn3xOrNB9O9GerFiBvyew6nniRO0D8PS9UQRbA3f+beKPwDSLHUsSqxHsTRfhD0ue9\niNN+q3lfBL5W1IJschBRlL3JzObp9WC6LR2jbsxgX8X32zhee4koTF5KMU/xxd3fJXI/TYy/nTO0\nFgDWIo5l3gC+3C8FWfhw1mx14da/EG+2/am3o+qokSzW92ni/977wA5FK8hCLO5GTPo4hFjcbhai\nMPclBv92jyOO5eZlcNHWPwCrFK0gW2YeC2lvQBRms8/rN4Bd0rHKscRjO44oyq9PHLcPEK2YtipS\nQRY+PNNlY6IF0QBxjLo1sAlRoB4gis8bVwuyyaPpsseBDQtckP0CUWytLjp7LLB6mgH+jrvvRNRl\n3iImSf3ZzPbtzWhHTzNlJXdpivm2AK1mXI5FZrYM8ANgCXdv2l82s8+8xIv5HYDxRZp5UWVmmxAL\nXj3k7ju32HZd4HqiGH24uxfqlM/hSDPILiJmVUHxZ178oI3NbnL3qU6HMbNfEQdBJwI/TC+KCynN\nslqXaCS/MvGOdO3pr68RBwu/By5190L2bCrrzIsqixXX9yIO3v9N9F670N2npMs/QZz6nj3d7wXg\naI9FZQopnZ59OFGoyLoP+FF6QzG7/cxEseoUdy/0G1BpNuEXgbWJA/kFiRV8ZyRmzL5JvAi6jyjQ\n/9YLvKJvmaVZkhsRC4Ec3Wr7zH6rE61ZZnf3un0KxzIzm4lY/+F9d/9pi20/S/xuXwMc5+5F65Pd\nttSC5QDiGH46in+8Vq8neq2J7j7VAkdmdhrx5vqRRZ5ZVpV+z1cljtc+SUyAmZV40/VNoqfmA8B1\naYJNIZnZH4FH3H23Xo+lF9Lv7xoMHq/dVltoNbNdge2JHuETgVuJ45bCvfFQlY5ZdgY+T0yQgThG\nuQ443wcXcaxuP46YEPYbd5+c41A7Is2MPgnYjcE3xr/q7tc32H4J4H+IN9wqxBkvO7p7odY+UFFW\nJEepd+GsaQGNQjKzgVYzTlLOZYE7yvBiNvUf/SHxZsMb7r5Cb0eUv1S8erafZthkpdOIPjzI75fG\n8ulF3ftFO3jJm8XiP9WDfG/1N7Ao0qzJ6gv7p2tmW4iI9J10vLIlMMndf9zr8YiISDCzhxic5XwT\nsH2rY9P0Ovx44synCnGs/g13v7LLw+0YFWVFRERERERERESkJ9IZfJOBQ4EfD2cSRGp58FNitvwU\ndx+yQPlYVZiBytiVfnked/clWm7cR5RbuctAuZW7DJS7XLmHK7VvOJVY8GqXXo8nL8qt3GWg3Mpd\nBsqt3AXxFLCtu9813B3d/VozW4FYC+S/Oj6yLtJCX9IJE4mFbspGuctFuctFuctFuaWZGYiF/Hbs\n7TByp9zlotzlotzlotzlUtTcK46kIFuV+gevBRzRsRHlQEVZ6YRbgcXNbLFeDyRnyl0uyl0uyl0u\nyi0iIiIi0iPu/loHrmOKu7ezcPWYoaKsdMJewF+B68yscKvUjoJyK3cZKLdyl4Fylyu3iIiIiIxh\nZjbFzB7r9Ti6TT1lpRO2B34NbADcaGYvAv8AXgKarVBetB4ntZRbuZW7MeUuJuVW7jLkFhEREZGx\nrRRttlSUlU44BqiujDcAzAvM02KfgbRPkV/UKbdyK3d9yl1cyq3cfZfbzP7T6zH0gnKXi3KXi3KX\ni3KXS1lz13ErsImZLebuj/d6MN2ioqx0wtMMvqgrE+UuF+UuF+UuF+XubwO9HkCPKHe5KHe5KHe5\nKHe5lDV3rb2A+Yk2W7u5+y29HlA3qCgro+bui/R6DL2g3OWi3OWi3OWi3H3vIWBp4Bbgf0ew/0zA\ndzs6onwot3IPh3IXi3Ir93Aod7GUNXetUrTZUlFWRERERPrZV4G7gOWA7dz9heHsbGZzUswXN8qt\n3G1T7sJRbuVum3IXTllz1ypFm61xvR6AiIiIiEi3uPs9wJHAXMCFvR1NfpRbuctAuZW7DJRbuUvq\n6czHU+nj6RYf1W0KQzNlpWPMbAFgVaLvxxy00QvF3Q/v9ri6TbmVu9U+yl1cyq3crfZR7sI4AvgC\nsL6ZfdPdT+71gHKi3MpdBsqt3GWg3MpdKmVpszVQqZRhfQfpNjM7AdiHYc6+dvfx3RlRPpRbuduh\n3MWk3MrdDuUuDjNbGrgUeA/Yot3TAdNpgC8RPcqUuyCUW7nb3E+5lbswlFu529yv0LnLRjNlZdTM\nbHdgv16PI2/KXS7KXS7KXS7KXQ7u/g/gUyPY9U1gpw4PJzfKPWzKXUDKPWzKXUDKPWzKLWOeZsrK\nqJnZ3cDKRDPqg4jVAl9x9yk9HViXKbdyo9x9S7mVG+UWEREREempfm+zpZmy0glLAZOBzdz9pV4P\nJkfKrdxloNzKXQbKXa7cIiIiIjLGjbTNFlCYouxwg4nUMw3weAlf0Cl3uSh3uSh3uSh3iZjZFDN7\nrNfjyJtyl4tyl4tyl4tyl0uJc1fbbI0nZscO56MwNFNWOuEZylngV+5yUe5yUe5yUe5ymQh80OtB\n9IByl4tyl4tyl4tyl0tZc1f74vZ1m60yHohL510DLGZmC/Z6IDlT7nJR7nJR7nJR7nK5FVjczBbr\n9UByptzlotzlotzlotzlUtbc2TZbt7r7S/1WkAUVZaUzjgKeBs43s+l7PZgcKbdyl4FyK3cZKHe5\ncu8F/BW4zszW7vVgcqTcyl0Gyq3cZaDcyl0GpWizNVCpVHo9BikQM1uowUULAmcD0wNnAn8G3mh1\nfe5+f+dG1z3KPYRyK7dyN6DcY5tyD9HXuesxs/2JfmMbAGsCLwL/AF4C3m6ya8Xdd+n+CLtDuZUb\n5VbuoZS7gJRbuSlHbgcG3H3JXo+lm9RTVobriTa2Ob7N66pQnOegcjem3M0p99in3I0pd3PKXUzH\nEBkgXuTMC8zTYp+BtE9hX9yg3KDcyl2fcheXciu3ctfXD7mvAfYzswXd/ZleD6ZbinxALb1RqJXs\nOki5y0W5y0W5y0W55WkGX9yUiXKXi3KXi3KXi3KXS1lzHwV8iWiztam7T+r1gLpBRVkZrjL1MMlS\n7nJR7nJR7nJR7pJz90V6PYZeUO5yUe5yUe5yUe5yKUPuJm22tifabD1gZn3ZZks9ZSU3ZjaNu0/u\n9TjyptzlotzlotzlotwiIiIiIp1lZv/p4NVV3L0wE1DH9XoA0j/MbHMzu9PMNm2wyRlmdq+ZbZ3r\nwLpMuZW7hnL3EeVW7hrKLSIiIiLSWQMd/iiMwlSPZWwzs6OAA9K3czfYbABYDrjMzJZz9+/nMrgu\nUm5AuWspt3IXmnIDyl2r73IDmNkCwKrA/MActHEQ7+6Hd3tc3abcyt1qH+UuLuVW7lb7KHdx9Xnu\n0rbZUlFWRs3M1gYOTN/+DXi0waaXAwsD6wIHmtn17n5rDkPsCuUGlLse5VZu5S4Y5QZKlBvAzE4A\n9mH4Z40V5cVNXcqt3G1S7gJSbuVuk3IXUL/nHs1xZdHbbKl9gXTCXunzqe6+krvfVm8jd7/B3dcH\njiXe1dknrwF2iXIr9xDKrdx5DbBLlFu5h+i33Ga2O7AfMJ4+Ph2ulnIr9zA+Cku5lXsYH4Wl3Mo9\njI/C6/c2W5opK53wWeBdBk9/bOUw4gXdZ7s2onwod3uUu9iUuz3KXWzK3Z5+yb1T+nwXcBDwEPCK\nu0/p3ZByodzKrdz9S7mVW7n7V1lzl6LNloqy0gkTgMfd/b12Nnb3d83sKWCx7g6r65S7Dcqt3AWl\n3G1QbuUuqKWAycBm7v5SrweTI+VW7jJQbuUuA+VW7r5XljZbal8gnfAqMFu7G5vZADAXMLFrI8qH\ncrdBuZW7oJS7Dcqt3AU1DVGMLs0Lm0S5y0W5y0W5y0W5y6WsuUvRZktFWemE+4F5zWzNNrf/CjAn\n8ED3hpQL5W6PchebcrdHuYtNudvTL7mfoZzHwMpdLspdLspdLspdLmXNPZI2W+9RsDZbZXxgpfMu\nJd6RuMLMtjKz8fU2MrPZzOwA4FygAlyS4xi7QbmVewjlVu4cx9gNyq3cQ/Rh7muAxcxswV4PJGfK\nXS7KXS7KXS7KXS5lzT0BeHo4bbaAp9J+hTFQqVR6PQYpODMbB9wMrEG8WHsbeBh4ieh9MjvRlHkJ\nBlcMvNnd1+3JgDtEuZUb5VZu5VbuAitx7jmAvxK9yTZ190k9HlIulFu5ezykXCi3cvd4SLlQbuXu\n8ZByYWbPAxV3/2ib2w8ALwL/cff5ujq4DlJRVjrCzGYGzgO2zvw4++QayHz9C2BXd38nj7F1k3Ir\nN8oNyg3KrdwF1e+5zWyhBhctCJwNTA+cCfwZeKPV9bn7/Z0bXfco9xDKrdzK3YByj23KPYRy92Hu\neszsOmLxrnXaWbjLzLYjzua6yd3X6/b4OkVFWekoM1sO2ApYGVgAmAF4B/gXcA9wubsXvRfdEMqt\n3Ci3cvcZ5VZu+iC3mf2ng1dXcfdpOnh9XaPcHaHcY5xyd4Ryj3HK3RHKXUBm9lXgQuBlYtGvK919\nyP1jZrMBuwM/IIrWO7n7xTkOdVQK+wDJ2JTeiSnsuzEjpdzlotzlotzlotx9Z6D1Jn1JuctFuctF\nuctFuculrLnr+RmwM9Fm6+fA22bWTputwhRkQUVZEREREelfa/d6AD2i3OWi3OWi3OWi3OVS1txD\nuPsUM9uYwTZbswCfpkWbrfxG2BlqXyAiIiIiUsPMpnH3yb0eR96Uu1yUu1yUu1yUu1z6OXe/ttkC\nGNfrAYiIiIiI5M3MNjezO81s0wabnGFm95rZ1g0uLyTlVu4ayt1HlFu5ayh3Hylrbog2W+5+sLtv\n4O7LuPsS7r6cu2/s7ocUtSALKsqKiIiISMmY2VHAlcAqRD+yegaA5YDLzOzIvMbWTcqt3HUot3IX\nmnIrdx3K3Se5y0BFWREREREpDTNbGziQePFyL/Bog00vB25M2x1oZmvmM8LuUG7lbrCpcit3YSm3\ncjfYVLn7IHdZqCgrIiIiImWyV/p8qruv5O631dvI3W9w9/WBY4kXOPvkNcAuUW7lHkK5lTuvAXaJ\nciv3EMrdN7lLQUVZERERESmTzwLvAge0uf1hwHtpvyJT7vYod7Epd3uUu9iUuz3KLWOeirIiIiIi\nUiYTgKfd/b12Nnb3d4Gn0n5FptxtUG7lLijlboNyK3dBlTV3KagoKyIiIiJl8iowW7sbm9kAMBcw\nsWsjyodyt0G5lbuglLsNyq3cBVXW3KWgoqyIiIiIlMn9wLzDWADjK8CcwAPdG1IulLs9yl1syt0e\n5S425W6PcsuYp6KsiIiIiJTJpcQCGFeY2VZmNr7eRmY2m5kdAJwLVIBLchxjNyi3cg+h3Mqd4xi7\nQbmVewjl7pvcpTBQqVR6PQYRERERkVyY2TjgZmAN4kXL28DDwEvAZGB2YG5gCWA88ULoZndftycD\n7hDlVm6UW7mVW7kLTLnLlbssVJQVERERkVIxs5mB84CtMz/OHhQPZL7+BbCru7+Tx9i6SbmVG+UG\n5QblVu6CUu5y5S4DFWVFREREpJTMbDlgK2BlYAFgBuAd4F/APcDl7t53PdmUW7lRbuXuM8qt3Ch3\n3+buZyrKioiIiIiIiIiIiORIC32JiIiIiIiIiIiI5EhFWREREREREREREZEcqSgrIiIiIiIiIiIi\nkiMVZUVERERERERERERypKKsiIiIiIiIiIiISI5UlBURERERERERERHJkYqyIiIiIiIiIiIiIjlS\nUVZEREREREREREQkRyrKioiIiIiIiIiIiORIRVkRERERERERERGRHKkoKyIiIiIiIiIiIpIjFWVF\nREREREREREREcjRNrwcgIiIiIvkws8WBR9O3B7v7UR2+/l2Ac9O3q7n7HZ28/iIzs88BN6Rvt3f3\ny3o5HhERERHpLRVlRURERIbBzL4GXDDKq3nK3RftxHhGoJI+inr9uTOzGYDNgY2ATwPzALMB7wDP\nAvcBvwV+6e6TWlxdX903IiIiIjIyal8gIiIiMnyVUX5MyX/I4O7/BGYFPgL8uAs3cUH1+vtllqyZ\n7Qw8BlwGbA8sBUwAxhNZlwK2AS4BnjKz7Xs0VBEREREpEM2UFRERERmeS4ArGly2HHAHUXi9FNi9\nwXY9KcoCuPs7XbzuKcTs0cIzs/HAGcDXicdzEnAxcBXgwERgdmAlYAdgU2IG7cVmtqy7H9CLcYuI\niIhIMagoKyIiIjIMzQqPZvZu5tvJ3SyAStcdxWBB9kHgS+7+WM02rwNPAVeZ2QbAz4lC7XfM7GF3\nH22bCxERERHpU2pfICIiIiKSYWb/BXyHKMg+CaxdpyA7FXe/jmhjUPVjM5u5a4MUERERkULTTFkR\nERGRHjKzzwE3pG83B24Hjk9fzwZs4u6/q9lnLWBnYFVgfmA64DXgfuB/gAvdfXKd21oceDR9e7C7\nH5W5bFfgnPTtsu7+dzPbENgb+AzRh/Y14C/AKe7+hzrXvwtwbvp2tWxfWTP7GfAV4BV3n9vMpkkZ\ndgE+kTK8ANwCHOPu/2hyn30K2A9YE5gbeDHdb8e5+99SH9jz0uZrufttja6rge8BA0RRdnd3f7Wd\nndz9ejO7EpiJ6EE75DGoybEO8E2iBcLcxMzbZvfvSJ4riwF7AesACwEzA68CfweuBc5x9+wM7+p+\nSwCPpG/3cffT0ni/Cyyfbu8Zop3DMe7+etpvaeAAYDVgwZTp/4DD3f2+JvfFdMTM5M2AZYm+vW+n\n27gZONPdvcn+cwJ7AhsARjxfPyAWYrsbuMDdb26y/zhgO2BL4FPAXMD7wPPArcB57v7nBvtWn9uv\nuvtcZjYbUdT/IrAw8Bbxu3mcu9+Q9hkP7ArsBCwBzAj8k2h7ckK939/M7S1K/F6uAywKTEv8DvwV\nuNTdr2q0r4iIiIwdKsqKiIiIjA0VohB4BVFshJres2Y2QBROd8nsUzUX8Ln0sauZrefubza5rUY/\nqwDjzewIojiZ3XZOYENgQzPbw93PGsH1jzezGYGrgXVrtl2A6M/6RTNb293/UnslZrYDcD5xHFvd\nd0Fga+BLZrYjMEvmNpsWRutc/4JExgrwQLWI1i53/3Kbt3MA0SIhK3v/7unuZzbYveVzJd3GnsCJ\nRNEuez/Pkz7WIVotbOzu99e5jerncangfk7NNksQBdg1zWx14rl3FVFgrJoL2AJY38w+6+4P1Bmn\nAb8jCozZcc6WPpYB9jSz77n7cXX2XyXtP6Fm//HA4mmc25nZWe6+R5395wV+SxRjs/tPB3wcWBL4\nupmd4u771u6f2Wecmc0O/JEoLFfNSDzX1zGzLwG/J+6nz9fc3jLA0enzDnVup/qYnsTUz3+AjxFF\n9y3M7Hqi3cbb9a5DRERExga1LxAREREZO9YFPkvMAv0oMQv21szl+xIF2Wqf082JQsz8wNrAdWm7\nlYHTRjGObYiC7GnAikSxa/H0synp9k9IBaiROAFYBfg2UTCbg5h9eVm6fCbg1NqdUvHuXKLY9j5w\nKFE0m5uYYfkoUbBdcoTjgihyDqSvfzGK62lmE6L49lPisZpA3A8HA/8h7t/j0uzPRpo+V8xsa+I+\nnAZ4DtgNWCzd1nLp9icThfAbWtzWx4nnwunErOYJ6bb/li7/DLAH8fjdRcySnTNlOjltMxNwRO0V\np+fQzURBdgpwCvDptP9CRHHyceIxP8bMvlaz/zTA5cRz6B1ihuoniWLwx4jfkTvT5t8ws53q7H8d\ngwXZy9L45yLu1y2Ae9Pme5vZD5rcTxAzlz+S9psnZTiQwUL6UcAhxPNsP2CRdFsbEa0yAL5iZivW\nua+2Ix7T8cAT6b5Zknj+rwb8Mt3Oeuk+ERERkTFMM2VFRERExo6vAQe4+ykNLv8WUXR5F9jA3Z/P\nXPaimd0FPEQU375iZvu6+8QRjGN/YH93Pz7zs9eJPqmLAN8AZiBmdQ63cDk7UVhe093vyvz8QTP7\nKlEcWxpYxczmc/d/Z7Y5gJi9WAG+7e6nZy77jZn9H3Gq+u7DHFPWZzJf/2kU19PM1sCP3D1b4Hsd\nONrM5ifaDcxInIp/WZ39oclzJbUCOJm4n94iWkk8WXNbB5vZv4ki6FzAYel26/lv4HR3/2bmZ3en\nx+vB9P1PiPtrw8yp968B3zKz1YhC6/pmNt7d/5O5nh8SBeVKynNC5rLXgMvM7NZ0Ox8Bjjezn7v7\n+2mb1YjCZwX4Xs39MRF4zsz+QLQPWDJlyS7AthdRpK4AZ7j73jXZrzGzW4jC7KLAQWZ2rrs/V+d+\nmp1of7Csu/8r8/PjzGzldNnSwEHAFu5+bWab681sL+A36fuNGSx6Y2azEAXZCrG43Mo1bTXuBO40\ns7OJNhAbmtlm7n51nXGKiIjIGKCZsiIiIiJjwwAxS7LuKeupKPN74jTrC2oKsgCkQtWV6dtxRK/S\nkXiipiCbdU3m60+O4LorwCU1BVkA3L3CYFFqqutPrRu+kL59BTi7zv6vEbNvR3OMO1/m68dHcT3N\n/Bs4vMFl2fwLN9im6XMF2JSYpQlwWk1BNut0oufqALBN6nNa77Y+IIqnU3H3h4gs1ZnFP2rQC/XG\n9Hk64g0DAMxsWmK2ZyWN46R6g3T3Z4Ez0u1MIGYaVy2Y+fqhBvt/QMyk/hSwfs3Fu6bPk4DvN9j/\nTeCY9O20TL2gW1aF6NH7rzqX3ZjZ5m81BdmqmxhsSbB0zWU7EkVfgAOb9Dk+kHjTBgbbnIiIiMgY\npKKsiIiIyNhQAW7PzACciru/5e67ufumdWbzZT2V+brZKenN/LrJZdmC0xwjvP5fjeD6FyfyVIBb\nmyyE9Dti0aORyt7ma6O4nmaurZktmvVs5ut5G2zT9LlCtLKoavhYpiJ4dUGxOYhepvVu669NZlxX\nxzuFaEPQbBuIHrFVKzB4f1/r7kP64mZkFzBbPfN19rHeI80SHsLdH3H3+1LhHgAzm5tox1ABbnH3\nN0Zw+7Wub/DzZ1tt4+6TiEXYYOr7CaJdBUTLiWtoID1OtxIF7HXSmxkiIiIyBql9gYiIiMjY8UwH\nrmNS5ut6Mx/b8WiTy97NfF23ANal618s8/UjjXZ29ylmdjeDs2qHK1sYHOn910qz/O9lvp6pyXbN\nnivZRaYebLhVyM4u/SRwX51tnmyyf3W8L7j7ey22gakf09GMs+qPRCG/2j/2fjM7nyjyPtziOtu+\nfXd/1szeJBaRazZD/MkGP8/eB0802b+6Xe3vVrXH7MvEYnkzN7mOh4nWIjMSLRe6NeNbRERERkFF\nWREREZGx4+VWG5jZAsSiTWsRp27PAzQr0IzEWx2+vk5c/4TM16+02PapFpc3kz0tfEIbtzUS77be\npKVmz5W5qrfj7q1uK5tvQoNt3mljPO1sU2uuzNcvNdvQ3SeaWfXU/gmZn79vZpsRbT3mI/rG/pjo\nf/wsMSv1SuC6OrOT27795BVgVhrfT9C9+6o6S3x+4M1h7DcfKsqKiIiMSWpfICIiIjJ2NC3WmNm2\nxCzRQ4gFjhYhZlNWaj760QyZryc13CqMpqj8dObrkfTMzUuz58qM6XOjmatZ7c7M7YYZM1+3M9bq\n4z7VON39XmAJYiG4Rxn8PfgosBPRp9fNbONR3n51m7zvp+xt1v6ut/qYNfeRioiISFs0U1ZERESk\nAMzsU8BFxCn1U4CfAZcTp3W/SvSahFgQ6LQeDLHbsv1TW7VNGM3M4ewCZKvTvL/uWPV2+txO8TBb\nmOz2DOlab2e+bmes06fPQ8aZZgQfDxxvZh8HNgI2IPrrTk+0v7jazLZy96tGePvV+yrv+wlirDMB\nD7j7Cj24fREREekwFWVFREREimEf4titAnzX3euuVJ85xbvfZE/Znr3hVuFjo7id24gC8LTAV8xs\n/yaLcg2Ren3uBpzVRuuAbqmeij+9mc3i7s2KiNlT+LvRqqGZbMuAeZptaGZzEotXVWgxTnd/lJgx\ne4qZzQp8Czg07X+Smf0qLXLW9u0n1RYCed9PEAuaLQrM3YPbFhERkS5Q+wIRERGRYqgu9DMFOKPJ\ndsvlMJZeyPaJXbTRRmm1+VVHeiPu/grwS6KANw/w38O8ih8BJwCPm9nKIx3HKN2b+Xr5Ftsuk/m6\n3iJf3dT1cbr7m+5+GHAu8ZguCCxd53qa3r6ZLUws8jWs2++g6n01n5mpMCsiItIHVJQVERERKYbq\nKfnvuXvdnqpmNhewbeZHA10fVX6cwZ6eazbZbiNicaPROJqYLTsAHGlmbfWWNbNNgb3Tt5OAB0c5\njpG6MfP1Fo02MrNpgM+nb59z94e7OqqhHgReIO7nTcxsfJNtN898fXP1CzNb3cz2NbP1W9zWA5mv\npwNw99eAv6bbX93Mmi3gVff2c3R95uvtmm1oZtuY2SZmNmOz7URERKS3VJQVERERKYZn0ueZ6hUJ\n02nzlzN1v8u+mVGXCtE3k2Y7pkXPpmJmcwAnMXX/2ZHc1t+Bg9O3swK3mNk6zfYxs62I+398uv3t\ne9i+4PfEzOIBYDczazSzeH/iOVIBzslpbB9KbSHOS9/OA3y73nZmtgTw9fTt4+6eLVCeDpzIYKuC\nRtZOn98HHsv8/Kz0eVrgiAa3PxdxXwG8AVzW5Ha65efA68Rj+v1Gj6mZrQScD1wDnJnf8ERERGS4\n1FNWREREpBiuAdZIX//czL4J3A/MBqwHHEQsBLQVgzP5djCzq4FX3f31nMfbDScQMzsHgHNSsexq\nYhGk/wKOIfp+XgB8YzQ35O7Hm9lHgO+n67wx3ZeXEqeSv0zc9ysDuwLrE8XNd4Ht3P0zilOjAAAE\ntklEQVT/RnP7o+HuU8xsD+C3xCn3/2tmBxMzaN8i2j/sBOxZ3QU4rhdjJR6zbYmFuI42s/mIx+9p\nYA5gXeAw4rk9haGP64+BS4AlgbvM7HjgdqJf7EzAx4HdgS8Sj88FNT12LwB2Bv4fsLuZzQKcAvwz\n7b868ENg/rT/fi169HaFu7+RfucvJJ6Pd5nZocB1RL/lBYiM+xELkr1J3G8iIiIyRmmmrIiIiEgx\nnA78JX29DHATUXh6jJgRNx2wIbFQ1T/Tdiumr2/IdaRd4u63EEW8ClEwOxl4krgffk0UzjYjCqad\nuL1DgS2BJ9JtbkrMhn0EeDX9/HKiKF4B7gA+4+6/7sTtj4a7/54ovE4i7pefEoXOV4lT9vdKm94L\nrOvu79W7nhzG+TbwOaDaOmFfomfrROBxYgbvfMA7wDbufnPN/pcCRxIF26WIWaIPE4tx/Yt4g2Ir\n4vG5llj0K7v/FOJxvSNtsz3wp8z+lxGF3Q+Ab7n7hZ3K3kTdtiPufjHRHuN9YoG2M4n76CXicTyU\nKMK/AHzB3Z/IYawiIiIyQirKioiIiHRWpebzcPZpKJ2+vxZRgPoH0V/1PaIAdTiwnLvfk4pMXwbu\nJgpZLwPZWZvNbqvS5ribZWx2HcO5/rrbufv3iBmBNxKFsw+IYuNpwLLufnuDsY6Iu/+KWBjqy8DF\nwN+J+/QDonB4D7Hw2pruvrq7N+sj26n7ty3ufhFgREuH+4nT3ycBzxGzaHcEVnL350YwjtrtRryN\nuz9FLFC3B/G4vkAUHicSBeSjgMXd/ZcN9j8EWIVoRVDNOZmYQf0I8DPg8+6+eb3is7u/4u6rEQXZ\na4n7Z1K6ngeAnwCfcPeTR5JvmNs03c7dTyeKz9XH9DXiufgy8YbM/oC5+21t3I6IiIj00EClMqrj\nVBERERGRMcXMTiRmXFaAFdz9gRa7iIiIiIjkSjNlRURERKTfLJT5+sWejUJEREREpAHNlBURERGR\nQjCzzwEbET11v5FOe6/dZjxx+vncwPPuvkC+oxQRERERaU0zZUVERESkKJYlFmpaD/hOg232Igqy\nFeCKnMYlIiIiIjIsmikrIiIiIoVgZh8hFjmbP/3oXGJxpyeBeYiFmg4ExhMLIC3n7s/kP1IRERER\nkeZUlBURERGRwjCzlYDfErNhB+psUiEKsl9091vzHJuIiIiISLtUlBURERGRQjGz2YG9gU2AJYGZ\ngTeAR4E/AKe7+8u9G6GIiIiISHMqyoqIiIiIiIiIiIjkSAt9iYiIiIiIiIiIiORIRVkRERERERER\nERGRHKkoKyIiIiIiIiIiIpIjFWVFREREREREREREcqSirIiIiIiIiIiIiEiOVJQVERERERERERER\nyZGKsiIiIiIiIiIiIiI5UlFWREREREREREREJEcqyoqIiIiIiIiIiIjkSEVZERERERERERERkRyp\nKCsiIiIiIiIiIiKSIxVlRURERERERERERHKkoqyIiIiIiIiIiIhIjlSUFREREREREREREcmRirIi\nIiIiIiIiIiIiOVJRVkRERERERERERCRHKsqKiIiIiIiIiIiI5EhFWREREREREREREZEcqSgrIiIi\nIiIiIiIikqP/D5LaOIuf3MNEAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<matplotlib.figure.Figure at 0x7fe615ad2a58>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "### Show AUCs as a training X test X AUC heatmap\n",
    "chromosomes = ['chr1', 'chr2', 'chr3', 'chr4', 'chr5', 'chr6', 'chr7', 'chr8', 'chr9', 'chr10',\n",
    "               'chr11', 'chr12', 'chr13', 'chr14', 'chr15', 'chr16', 'chr17', 'chr18', 'chr19', 'chrX'\n",
    "              ]\n",
    "roc_matrix = np.zeros((len(chromosomes), len(chromosomes)))\n",
    "for i in range(len(chromosomes)):\n",
    "    training_chromosome = chromosomes[i]\n",
    "    for j in range(len(chromosomes)):\n",
    "        if not i==j:\n",
    "            test_chromosome = chromosomes[j]\n",
    "            roc_matrix[i][j] = chromosome_rocAUC_dict[training_chromosome][test_chromosome]\n",
    "\n",
    "pr_matrix = np.zeros((len(chromosomes), len(chromosomes)))\n",
    "for i in range(len(chromosomes)):\n",
    "    training_chromosome = chromosomes[i]\n",
    "    for j in range(len(chromosomes)):\n",
    "        if not i==j:\n",
    "            test_chromosome = chromosomes[j]\n",
    "            pr_matrix[i][j] = chromosome_prAUC_dict[training_chromosome][test_chromosome]\n",
    "            \n",
    "\n",
    "data = pd.DataFrame({'Training Chromosome': chromosomes, \n",
    "                         'mean aucROC':roc_matrix.sum(axis=1)/(len(chromosomes)-1)})\n",
    "\n",
    "sns.barplot(data=data, x='Training Chromosome', y='mean aucROC', palette = 'coolwarm')\n",
    "plt.ylabel('Mean rocAUC')\n",
    "# plt.gca().set_xticklabels(chromosomes)\n",
    "plt.xticks(rotation=90)\n",
    "# plt.xlabel('Training Chromosome')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Visualize prAUC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(0, 1)"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAABWUAAAQOCAYAAACzV1oHAAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAAewgAAHsIBbtB1PgAAIABJREFUeJzs3WmUZWdd7/FfpTsh6aSFDAUREkwWmD8EDGMgEEYBQcEI\nuHIBlSvDBWFdBb0gC9CriAiKKINGEQcIkctVMCiTBiREBgERCJfxgQVEjQwpIDEdAkk6fe6LvYuU\nbXdXdeWcp051Pp+1au1TZ+/97Od05dV3PXnOwmQyCQAAAAAAfRy00RMAAAAAALghEWUBAAAAADoS\nZQEAAAAAOhJlAQAAAAA6EmUBAAAAADoSZQEAAAAAOhJlAQAAAAA6EmUBAAAAADoSZQEAAAAAOhJl\nAQAAAAA6EmUBAAAAADoSZQEAAAAAOhJlAQAAAAA6EmUBAAAAADoSZQEAAAAAOhJlAQAAAAA6EmUB\nAAAAADoSZQEAAAAAOhJlAQAAAAA6EmUBAAAAADrautETmEdVdXCS5yV5VpItSZ7XWnv+9RzztCRP\nS3LPJDdL8p0kLcmbkvxea+3K6zM+AAAAALA5WCm7m6q6Q5J/TvKcDP8+kymM+dwk/5jkUUlumuRL\nSa5IcmqSFyX5aFXd/Po+BwAAAACYf6LsqKoOqqrnJPlQktsn+cskC1MY94wkL8gQd1+aZLG1dnJr\n7fgkpyX5QpKTkrzh+j4LAAAAAJh/oux1HprkN5JcleTxrbVHT2ncl2QIsm9urT2ztfat5ROttQ8n\nOXM8f1pVnTmlZwIAAAAAc0qUvc5BST6Y5A6ttddOY8CqOj3JrcdfX7Kna1prFyY5f/z1cdN4LgAA\nAAAwv0TZ67wvyb1aaxdNccz7j8crknxgH9e9M8NWCfeZ4rMBAAAAgDkkyo5aa99ore2a8rCnjMfP\nrzL2Z8fjtqqqKc8BAAAAAJgjouxsHZdhv9iLV7lu5fnjZjcdAAAAAGCjibKztX08XrnKdSvPb9/r\nVQAAAADApifKztZh4/HqVa67asXrbTOaCwAAAAAwB0TZ2VpeAXvIKtcduod7AAAAAIADkCg7WzvG\n4+GrXHfEiteXz2guAAAAAMAc2LrREzjA/UuSe2T1L+86YcXrL85oLpMZjQsAAAAANwQL0xpIlJ2t\njyd5dJKTqmpra23nXq47ZTxe1lq7aBYT2bnz2iwsTO2/GwAAAAC4QdmyZXqbDoiys3Vekhdl+PKu\n+yQ5fy/XPSTDSta3z2oil15qq1oAAAAAWK/Fxe1TG8uesjPUWrswyccyLG1+1p6uqaoHJrnL+Ouf\ndpoaAAAAALBBrJQdVdVNkmxZ8dbK/9d/W1Udvdst32ytTarqlhn2gZ0k+bXW2gt2u+4Xkrw7yYOq\n6qwkz26t7Rif+YAk54z3nttau2BqHwgAAAAAmEtWyl7nwiRLK34uGd9fXuW6tNvP8SvOH7Ti5z9p\nrb0nyVOT7ErylCSXVNVnqurLSd6Z5KZJ3pfk8TP5VAAAAADAXBFlr7Mrw4rVtfzs2u3eyb4Gbq29\nKsmdkpyd5CtJTkhySJILkjwpyf1aa1dM52MAAAAAAPNsYTLZZ0/kALG0tMMfGgAAAADWaXFx+8Lq\nV62NlbIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAA\nAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAA\nHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAd\nibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2J\nsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmy\nAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIA\nAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAA\nAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAA\nAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAA\nAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAA\nHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAd\nibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2J\nsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmy\nAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIA\nAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAA\nAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAA\nAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAA\nAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAA\nHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAd\nibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2J\nsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmyAAAAAAAdibIAAAAAAB2JsgAAAAAAHYmy\nAAAAAAAdbd3oCcybqjotydOS3DPJzZJ8J0lL8qYkv9dau3IdY25J8pNJHpXkzkmOGsf91yTvTvKH\nrbXPTOUDAAAAAABzbWEymWz0HOZGVT03yQuSTJJcleSiJNuT3DzJQpLPJfnB1tqX92PMo5L8bZJT\nx3H/I8m/JzksyQkZVitfk+TprbVXTumj/BdLSzv8oQEAAABgnRYXty9MayzbF4yq6oxcF2RfmmSx\ntXZya+34JKcl+UKSk5K8YT+HPjtDkL0qyZNaa0e11n6gtXbrJMcleVuSg5P8flWdOp1PAwAAAADM\nK1H2Oi/JEGTf3Fp7ZmvtW8snWmsfTnLmeP60qjpzLQNW1QlJHjre9/LW2p+tPN9a+2qSxyS5MsNK\n3KdO4XMAAAAAAHNMlE1SVacnufX460v2dE1r7cIk54+/Pm6tQ694/d69jHtFhj1rF3a7HgAAAAA4\nAImyg/uPxyuSfGAf170zQzy9zxrH/eqK14fu47qDM6ym/coaxwUAAAAANilRdnDKePx8a23XPq77\n7HjcVlVrWdX6qSRfyhByf2JPF1TV8UluM/76ljWMCQAAAABsYls3egJz4rgMK1UvXuW6leePy7Dt\nwF611nZW1ZOT/E2Sh1fVWUleliHUHprk9CS/k2RLhlW4r13X7AEAAACATcNK2cH28XjlKtetPL99\nr1et0Fp7V5J7JXljkp/JEHKvTnJ5kr9NcliS5yZ5WGttsh9zBgAAAAA2IVF2cNh4vHqV665a8Xrb\nfox/apLbZtjG4PIkn07yxSS7ktw8w4rZk/djPAAAAABgkxJlB8srYA9Z5bqVX9a12qraJElV/WGS\nP0ryvUnOTHJka+32rbVbJzk2yZ8keViSD1TVWr9ADAAAAADYpOwpO9gxHg9f5bojVry+fLVBq+oB\nGbYsmCR5Smvt3JXnW2tfT/JzVXWLJA9P8gdJbr/WSe+PI4/cloWFhVkMDQAAAADsB1F28C9J7pHh\ny7v25YQVr7+4hnEfOx53tNb+ah/XnZshyt62qu7QWvv4GsbeL1u3bpn2kAAAAADAOoiyg48neXSS\nk6pqa2tt516uO2U8XtZau2gN437vePzKKtd9fbd7ph5ld+681kpZAAAAAFinLVumtxOsKDs4L8mL\nMnx5132SnL+X6x6SYSuCt69x3EvH4y1XuW7l+W+ucez9cumla9oCFwAAAADYg8XF7VMbyxd9JWmt\nXZjkY0kWkjxrT9dU1QOT3GX89U/XOPQ/jMdDq+qMfVz3iPF4xTgPAAAAAOAAJcpe5xcyrIJ9UFWd\nVVXfTd/jF3adM54/t7V2wYpzt6yqnVV1TVX98m5jnp3k4gyx98+q6pFVddCKe4+qqlckefA49m+3\n1q6Z0ecDAAAAAObAwmQy2eg5zI2qenKSszLE6quTXJTkxkmOzRBN35fkoa21K1bc831JvjSe/7XW\n2vN3G/N2Sd6c4UvCFpLsyBBqD0ly4visSZI/bq09ZVafbWlphz80AAAAAKzT4uL2qX1hk5WyK7TW\nXpXkThlWuH4lQ0g9JMkFSZ6U5H4rg+wKew2erbVPJbl9kqcleVeSK5PcKsnNknwhyWuS3GeWQRYA\nAAAAmB9Wyt5AWCkLAAAAAOtnpSwAAAAAwCYlygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAA\ndCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0\nJMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQk\nygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTK\nAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoC\nAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIA\nAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAA\nAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAA\nAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAA\ndCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0\nJMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQk\nygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTK\nAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoC\nAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIA\nAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAA\nAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAA\nAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAA\ndCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0\nJMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQk\nygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTK\nAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoC\nAAAAAHQkygIAAAAAdCTKAgAAAAB0JMoCAAAAAHS0daMnMG+q6rQkT0tyzyQ3S/KdJC3Jm5L8Xmvt\nynWOuyXJk5L8VJLbJDksyZeTvCfJK1trH77+swcAAAAA5t3CZDLZ6DnMjap6bpIXJJkkuSrJRUm2\nJ7l5koUkn0vyg621L+/nuEcmOS/JXcexLx7HPyFDGJ8k+bnW2h9M43PsydLSDn9oAAAAAFinxcXt\nC9May/YFo6o6I9cF2ZcmWWytndxaOz7JaUm+kOSkJG9Yx/B/nSHIfjTJHVtr39daOynJ8eO5hSQv\nrarbXv9PAgAAAADMM1H2Oi/JEGTf3Fp7ZmvtW8snxq0FzhzPn1ZVZ6510Kr6yST3zrBVwYNaa59Y\nMe7XkvxEknckeX2G7RIAAAAAgAOYPWWTVNXpSW6dIbq+ZE/XtNYurKrzk/xgksdl7Stmf24c98Wt\ntUv3MO53kjxkHdMGAAAAADYhK2UH9x+PVyT5wD6ue2eGrQbus5ZBq+rmSU4df13PtgcAAAAAwAHG\nStnBKePx8621Xfu47rPjcVtVVWutrTLuXTNE3K+11r5SVccleUKGPWqPSvK1JO9K8mettSvWP30A\nAAAAYLOwUnZwXIYtBi5e5bqV549bw7i3W75v/CKxTyX51SQPzrCC9mFJXpbkc1V16p6HAAAAAAAO\nJKLsYPt4vHKV61ae377Xq65z9HhcTPK6JBdk+NKvGyc5JskTk3w9ybFJ3lJVvugLAAAAAA5wti8Y\nHDYer17luqtWvN62hnGXw+3xSd7aWvux3c6/pqo+mWEf28Ukv5jkmWsYFwAAAADYpOZupWxV3WQD\nHru8AvaQVa47dA/37Mtkxevf2NMFrbV/TvK3Gfae/fE1jAkAAAAAbGIzXSlbVbfPsGfqb7fWzlvj\nbRdU1UVJntxau2RWc9vNjvF4+CrXHbHi9eX7MW6SfHwf1703w/6yt6yqI2bxpV9HHrktCwsL0x4W\nAAAAANhPM4uyVfXIJH+e5EZJ/i3JqlG2qh6T5JQkP5DkI1X1wNZam9UcV/iXJPfI6l/edcKK119c\nw7j/tsbnf3PF6yOSTD3Kbt26ZdpDAgAAAADrMJMoW1V3S/L6cfyFJHdb462fTPJP4/W3SPKOqrpz\na+0bs5jnCh9P8ugkJ1XV1tbazr1cd8p4vKy1dtEaxv3EitcnJvnsXq5buWXDZWsYd7/t3HmtlbIA\nAAAAsE5btkxvJ9hZrZT94yQHZ9hT9QVJXriWm1prn0hyWlW9MMmzM6xcfVmSx85onsvOS/KiDF/e\ndZ8k5+/luodk+ExvX+O478+wzcH2JP8tyfP3ct3dx+PnWmvfWePY++XSS9eyBS4AAAAAsCeLi9un\nNtbUv+irqn4ow/YDkyTPaq39yv6Gxtbac5O8OMMq28dU1fdPe567Pe/CJB8bn/esPV1TVQ9Mcpfx\n1z9d47hXJ/k/47j/s6q+Zw/jnpjkRzP8e/3Vfk8eAAAAANhUph5lkzxiPH60tfY712Oc5yb5VIag\nOeuVsknyCxnC6IOq6qyq+m76rqoHJDlnPH9ua+2CFeduWVU7q+qaqvrlPYz7/CSXJjkmyXlVdasV\n994xydsy7Lt7SZKXT/9jAQAAAADzZBZR9q4Z4uUfXZ9BWmu7kpyVIcredwrzWu1570ny1CS7kjwl\nySVV9Zmq+nKSdya5aZL3JXn8brcuZPh3XP7ZfdyvJnlokm9k2Cv3c1X1uaq6KMlHk9R47uGttaXp\nfzIAAAAAYJ7MIsqeOB7fN4Wx3j0eawpjraq19qokd0pydpKvJDkhySFJLkjypCT3a61dsYdbJ6uM\n+8EkJyf5rSSfTnJskqMzrAT+rSS3ba19aCofAgAAAACYawuTyT574n6rqquTbElydGvtsus51k2S\nfDPJ1a21Q6cxvxuqpaUd0/1DAwAAAMANyOLi9oVpjTWLlbJXjsfDpzDW8hdjfXsKYwEAAAAAbLhZ\nRNlLxuMtpzDWSbuNCQAAAACwqc0iyn5mPD50CmM9crcxAQAAAAA2tVlE2fOSLCR50rgn7LpU1a2S\nPC7Dl2i9dTpTAwAAAADYWLOIsm/MsK/sMUleVVX7vQFuVX1Pkr9McmiSy5OcO9UZAgAAAABskKlH\n2dbaJUlekWG17I8n+euqOnat91fVXZK8L8mdMqySfWFr7ZvTnicAAAAAwEZYmEwmUx+0qm6U5Pwk\n98gQVnckOSfJXyT5SGvt27tdf0ySe2fYruBHMsTihSRvT3JGa23X1Cd5A7O0tGP6f2gAAAAAuIFY\nXNy+3zsC7M1MomySVNXRGbYduPf41vKDrk1ySZLLkmxJclSGrQ6+O6fx+OYkj9k94LI+oiwAAAAA\nrN+miLJJUlUHJXnG+HPT3U4vP3j3D/OlJL/eWnvNzCZ2AyTKAgAAAMD6bZoou6yqDknyiCT3S3L3\nJMcmOTLJriTfTPKvST6Y5J1JzmutCYhTJsoCAAAAwPptuijLxhNlAQAAAGD9phllD5rWQAAAAAAA\nrE6UBQAAAADoaOu0B6yqU9Zx27VJvpFkqbV27ZSnBAAAAAAwN6YeZZNcmGS9+5deU1XvSfLa1tqf\nT3FOAAAAAABzYVbbFyys8+eQJA9IcnZV/VNV3WJG8wMAAAAA2BCzWCl79jruWUhykyQnJrn9+Ptd\nk7y/qu7WWrtkivMDAAAAANgwC5PJencamI2qunmSX0ny5AzbILyttXbGxs5q81ta2jFff2gAAAAA\n2EQWF7cvTGusuYuyy6rq55P8boYwe2pr7aMbPKVNTZQFAAAAgPWbZpSd1Z6y11tr7WVJ3jP++tiN\nnAsAAAAAwLTMbZQd/VGG/WVP3+iJAAAAAABMw7xH2Q+NxxM2chIAAAAAANMy71H2G+Pxxhs6CwAA\nAACAKZn3KHvEePzWhs4CAAAAAGBK5j3K3mE8XryhswAAAAAAmJJ5j7L/Pckk1+0tCwAAAACwqc1t\nlK2qRyQ5c/z19Rs5FwAAAACAadm60RPYXVUtJvlfSZ4xvvXB1tr5GzglAAAAAICpmXqUrapz13nr\n4UlukeQ2SRbGn68neeyUpgYAAAAAsOFmsVL24Rn2gV2vhfH4mSSPbK198fpPCQAAAABgPsxq+4KF\n1S/Zo6uSvD/J/03y6tbatdObEgAAAADAxptFlD1xHfdMklzeWrts2pMBAAAAAJgnC5PJ9dlpgM1i\naWmHPzQAAAAArNPi4vb17g7wXxw0rYFmqapO3eg5AAAAAABMw6z2lL3equqYJI9N8vgkJ2eO5woA\nAAAAsFZzFTqraiHJjyR5QpKHJjk4w5eG+V/vAQAAAIADwlxE2ar6/gwh9rFJvnd8e3mPhquSvG0j\n5gUAAAAAMG0bFmWraluSR2WIsfcc314OsZMk/5Dkz5O8sbX2H/1nCAAAAAAwfd2jbFWdniHEnpnk\n8PHt5Rj76SRnJ3l9a+3i3nMDAAAAAJi1LlG2qo5N8tMZvrTr+8e3l0Ps15LcLMPq2Ge11t7eY04A\nAAAAABthZlG2qrYkOSPDqtgHJ9mS60LsNUnemuTVSf4uydWzmgcAAAAAwDyZepStqpOTPDHJTyU5\nZnx7Ocb+vwwh9nWtta+vuGfa0wAAAAAAmEuzWCn7yQxbESyH2KUkr09ydmvtYzN4HgAAAADApjHL\nPWU/kuSXkryrtXbtDJ8DAAAAALBpHDTDse+c5DVJfreqTpvhcwAAAAAANo1ZRNlnJvlMhu0Ljk3y\ns0neX1Wtqp5dVbeYwTMBAAAAADaFhclkMpOBx9WxT0pyZpIjxrcnSXYleVeGL/x6U2vt6qraNZ77\n0dba22cyoRu4paUds/lDAwAAAMANwOLi9oXVr1qbmUXZZVV1eJLHJHlikruPby8/9LIkf5HkKRFl\nZ0qUBQAAAID121RRdqWqul2S/5Hkp5IcPb69cgKvTPKi1trF3SZ1AyHKAgAAAMD6bdoou6yqDk7y\niAyrZx+QYW/b5YlMkrw3yTlJ3thau7z7BA9AoiwAAAAArN+mj7IrVdUtM8TZxyU5fnx7eVJXJXlb\nknNaa2/uP7sDhygLAAAAAOt3QEXZZVW1kOTBGQLtGUkOHk9Nkkxaa1s3am4HAlEWAAAAANbvgIyy\nK1XVMRlWzj4hyW0yRNktGzqpTU6UBQAAAID1O+Cj7EpVdXqSJ7TWnrjRc9nMRFkAAAAAWL9NF2Wr\n6nfHly9srX195g/kvxBlAQAAAGD9phllD5rWQKt4WpKnJ7mm0/MAAAAAAOZSryj71fG42Ol5AAAA\nAABzqVeUPSvJQpJnd3oeAAAAAMBc2trpOb+ZMcpW1U2T/FaSD7TWdnV6PgAAAADAXOj1RV+vGF9+\nT5IfT7ItybczbGvw9SRXrjLEpLX2gNnN8MDni74AAAAAYP2m+UVfvVbK/myS3aPgtiQnjj/7srCH\newEAAAAANqVeUTYZ4upa3gMAAAAAOGB12b6AjWf7AgAAAABYv2luX3DQtAYCAAAAAGB1oiwAAAAA\nQEc995T9rqo6McldMnzJ142THJLkO0m+meSLST7UWvvaRswNAAAAAGCWuu4pW1WPTvJLSU5ew+Uf\nSPLrrbXzZjurGwZ7ygIAAADA+m3KPWWr6uVJXpchyC6s4eceSd5eVb/Sa44AAAAAALPWZaVsVT08\nybnjr5NqwkGQAAAgAElEQVQk705yQZIvZNiy4OoMWxgsJrl1kvsnufeK63+4tfaOmU/0AGalLAAA\nAACs3zRXyvbaU/ap43EpyRmttQ+tcv3zqupeSf46yZFJnp5ElAUAAAAANr1e2xfcMcOK12esIcgm\nSVpr70vyixm2MrjzDOcGAAAAANBNryh74/H49/t53/KXfN14n1cBAAAAAGwSvaLs0ni8Zj/vu2o8\nfm2KcwEAAAAA2DC9ouw/jsc77ud9tx+P753iXAAAAAAANkyvKPvSJNcm+eWqWtOXi1XVQpLnJNmZ\n5BUznBsAAAAAQDddomxr7YNJHpfkrkneW1U/VFV7fXZVnZbkHUlOT/LE1to/95gnAAAAAMCsLUwm\nk5k/pKpelGRbklsn+eEkkyRXJvl0kkuSfCvJjZIcneS2SY4ab31Pki+tMvyktfbEGUz7gLK0tGP2\nf2gAAAAAOEAtLm5fmNZYvaLsrgwh9rvPHY97evi+zu1Ra23LOqd2gyHKAgAAAMD6TTPKrml/1ynZ\n06T39UHW+iHFRgAAAABg0+gVZW+VZFenZwEAAAAAzK0u2xew8WxfAAAAAADrN83tCw6a1kAAAAAA\nAKxOlAUAAAAA6EiUBQAAAADoSJQFAAAAAOhIlAUAAAAA6EiUBQAAAADoSJQFAAAAAOhIlAUAAAAA\n6EiUBQAAAADoSJQFAAAAAOhoa+8HVtVxSW6X5Mgkh6z1vtbaa2c2KQAAAACATrpF2aq6W5Kzktx5\nHbdPkoiyAAAAAMCm1yXKVtXJSc5PcliShR7PBAAAAACYR71Wyj4nybbx9SeTvCXJvyX5dqfnAwAA\nAADMhV5R9r4ZtiA4p7X2uE7PBAAAAACYOwd1es7NxuNvd3oeAAAAAMBc6hVlrx6PF3d6HgAAAADA\nXOoVZT81Ho/q9DwAAAAAgLnUK8r+SZKFJI/s9DwAAAAAgLnUK8q+Osl5SX61qk7v9EwAAAAAgLmz\nMJlMujyoqg5N8uIkP5Pk9UnelOTzSa5Yy/2ttX+d3ewOfEtLO/r8oQEAAADgALS4uH1hWmNtndZA\n+1JVlybZMv56cJLHjj9rNUmnuQIAAAAAzFKv0Hnj3X6fWlUGAAAAANhMekXZ12ZY7QoAAAAAcIPW\nbU9ZNpY9ZQEAAABg/TbdnrLrVVVbk5yc5ButtX/f6PkAAAAAAFxfB230BFZxRJILk5y70RMBAAAA\nAJiGeY+y9x2Pt9vQWQAAAAAATEnX7Quq6ugkP53kbkmOTrJlL5cuJFlMcpsMXxC2o8sEAQAAAABm\nrFuUraoHJ/mLJNvXeMvKjXP/ZvozAgAAAADor0uUrarvS/KGJIfnP8fWfdmR5FNJ3prkpTOaGgAA\nAABAV732lH1ahi/t2pnk15JUkkNba8vPn4znb5Tk1CTnjO+/urX2wtbatzvNEwAAAABgpnptX/CA\nDOH1V1trv7m3i1pr1yT5SJKfrqqnJHllVd2itfa8PtMEAAAAAJitXitlTxyPb13rDa21VyZ5c5L/\nXVX3mMmsAAAAAAA66xVlDxuP/76Hc5PxePgezp2VYQ/ap89iUgAAAAAAvfWKsjvG4zF7OPeN8Xj0\nHs59Yjzed+ozAgAAAADYAL2i7OfG4yP3cO6S8Xi/PZy70XjcU7AFAAAAANh0ekXZd2TYhuD5VfWM\nqrr5inOfHc89paoO2e2+h4/HKzrMEQAAAABg5rZ2es4rk/x8kiOSvDjJGbluS4K/ybCC9geSfLCq\n/jjJpUnunuSpGfac/WineQIAAAAAzFSXlbKtta8keVSGFa8LSb664vTrk3x8fP8OSX4/yeuSPC3J\n8srZl/eYJwAAAADArPXaviCttb9Lcpskz09ywYr3dyZ5WJIPZwizK3+uSvKM1tpbes0TAAAAAGCW\nFiaTyUbP4buq6l5J7pTk8CQXJzmvtba0sbM6MCwt7ZifPzQAAAAAbDKLi9sXpjXWXEVZZkeUBQAA\nAID1m2aU7bZ9AQAAAAAAydbeD6yqbRn2kP2hJCcnOTbJEa21m+523QNba3/fe34AAAAAALPUNcpW\n1U8k+Z0kKwPsQpLJbtfdKcl5/5+9Ow+z7Crrxf+tJEQIRAzQCSCT4GXxAwmDoAwyhiEBEhAJM0JA\nQEARFIWLyKAIoigoRjTKJBBExoQQRvMLMikioxd9ZUoEA+mKTAm5QIa6f6xTdtmp6urqqlqnT9Xn\n8zz17O491btrn7PPPt+99tqttfcneWBVfXtclQAAAAAAm2dY9wWttV9J8tr0QHZu8nPuCrPffTL9\nbkneNKRAAAAAAIABhoSyrbUbJXlRetD61SSPTbIjyQ1XWOQlSV44mf+o1tqxI+oEAAAAANhso1rK\nPj7JwUm+kOSWVfWKqvqv7NZtwaKquqiqfjPJ29KD2UcMqhMAAAAAYFONCmXvkR7APnsSxu6tEyfD\nW298SQAAAAAA440KZa85Gf79GperyfCIDawFAAAAAGBqRoWyl58ML17jcj+YDJft5gAAAAAAYNaM\nCmXPnQxvssblbj4Z7tzAWgAAAAAApmZUKPsP6Q/s+qW9XaC1NpfkmemtZD+6SXUBAAAAAAw1KpR9\nzWR4v9ban7XWrrinmVtr10jytiR3mox63WYWBwAAAAAwytzCwpjuWltrpyW5V3rL128leW+Sc5I8\ndTLu6Ul2pHdZcKckl0tvXXt6Vd1nSJFb2Pz8+frlBQAAAIB9tGPHoXMbta6DNmpFe+EhSd6Y5Ogk\nhyV54GT8Ylj4oiXzLm7g+5bMBwAAAAAw84a1lF3UWntwkqckuVWW7z7h0vQ+ZF9eVSePrG0r01IW\nAAAAAPbdRraUHR7KLmqt/XCSmyW5apIrJrkgyc4kn6mq706lqC1MKAsAAAAA+25LhLKMJZQFAAAA\ngH23kaHsct0HAAAAAACwSYSyAAAAAAADHbTRK2ytfWmj15lkoapusAnrBQAAAAAYasND2STXS7KQ\nZMP6WJisDwAAAABg5m1GKLvooiT/mOSLm/g7AAAAAABmymaEst9IcpUkl0ty+yTXTHJakrdU1Qc3\n4fcBAAAAAMyMuYWFje0ZoLV2UJJjkjw8ybFJLp9d3Q/8Z5I3JHldVX12Q38xezQ/f74uIAAAAABg\nH+3YceiGdde64aHsUq21Q5Mcn+RhSe6U5IDsCmg/l+R1SU6uqq9sWhEkEcoCAAAAwHrMTCi7VGvt\nmunh7MOSHDkZvTD5+XB6QPvmqvrmkIK2GaEsAAAAAOy7mQxll2qt3STJI5I8JMm1J6MX0h8OdnqS\nk5O8o6q+P7y4LUooCwAAAAD7buZD2aVaa3dKD2h/LsmVJ6MXknwnyVuTvL6qzhhYz22SPDnJ7ZIc\nkeR7SSrJ25K8rKou3KDfc+X0LhyukeSsqrr+Rqx3JUJZAAAAANh3WyqUXdRaOzjJvZPcL/1BYVfL\nrv5nz6mqa6+07AbW8Mwkz5/83u8nOSvJoUmumWQuyb8nuWtVnbMBv+uv0x+GtpDkbKEsAAAAAOy/\nNjKUPWCjVrReVfWDqnpbkpcmeUmSsyeT5tJD0U3VWjsuuwLZlyTZUVU3noTBt0nyxSQ3TPKmDfpd\nD09ycfr2AQAAAADbxEHTLiBJWmtHpndh8ODsCmAXw8qPJ3ntgDJenB7InlpVT1s6oar+qbV2fJJ/\nTnKb1trxVbVP4Wxr7bAkfzH5Xa9O8gvrqhoAAAAAmClTC2Vba1dP8rD0MPamk9GLQezZSV6f5LVV\nVQNquX2SH08PSl+83DxV9anW2hlJ7prkUdn3FrMnJjk8yd8k+XCEsgAAAACwrQwNZVtrV0hy//Qg\n9qj07hMWg9hvJ3lzehD79yPrSnKXyfCCJB/dw3zvS6/7jvvyS1pr909vDfz1JL+c5Nh9WQ8AAAAA\nMLuGhLKttaPSg9j7J7niZPRckouSvCe9e4JTq+r7I+pZxpGT4eer6tI9zPdvk+EhrbW2lla8rbWr\nJnl5emvcJ1TVN1pr+1YtAAAAADCzNi2Uba3dOMnPJ3lokh+djF5sFfux9CD2b6rqvzarhjW4VnpY\n+tVV5ls6/VpJ1tK1wsuT7Ejy+qo6ZW3lAQAAAABbxYaHsq21p6S3ir35ZNRiEPvFJG9I757g8xv9\ne9fp0MnwwlXmWzr90BXn2k1r7YFJHpDknPRuCwAAAACAbWozWsr+UXqr00uSfCTJu5K8o6o+twm/\na6NcYTL8wSrzLe1e4ZC9WXFrbUf6w70Wkjy+qr619vIAAAAAgK1iM/uUPTDJHSY/L1hn/6kLVbWZ\ntS62gD14lfkuv8wyq3l5kqsmeU1VvXOthW2Uww47JHNzc6vPCAAAAABsqs0KOmct/Tt/MrziHudK\nrrTk399ZbaWttYekP9zsq0mevG+lbYyDDjpwmr8eAAAAAJjYjFD2eZuwzs12dpLbpj+8a0+ut+Tf\nX9rTjK21w5P8SXo3DidU1fnLzDYsvL744ku0lAUAAACAfXTggQds2Lo2PJStqlkMZT+d5MFJbtha\nO6iqLl5hviMnw29V1VmrrPPo9G4LkuR9q3TfcL3W2qWTf7+6qh69FzWvyTe/ube9LQAAAAAAu9ux\n49ANW9fGxbuz7T2T4SFJ7riH+Y5Of2DX6XuxzouSfGuVn8Wk9NIl4767xtoBAAAAgBkyt7CwMO0a\n9guttX9Ocosk762qo5eZfrck700PZY+qqjM34Hc+MsmrkpxVVddf7/r2ZH7+fDsaAAAAAPbRjh2H\nbljfoFrK7vLU9MD17q21E1tr/90eubV2VJLXTqa/dWkg21q7Tmvt4tbaRa21Z40uGgAAAACYLULZ\niar6+yRPSO9K4BeT7Gyt/Wtr7Zwk70tyeJIPJTlht0Xn0v+Oiz8AAAAAACsSIi5RVSeld2HwmiRf\nS3K9JAcnOTPJY5PcuaouWGbR9XQNsLDO5QEAAACAGaJP2W1Cn7IAAAAAsO/0KQsAAAAAMKOEsgAA\nAAAAAwllAQAAAAAGEsoCAAAAAAwklAUAAAAAGOigaRcA03D0MUfn3J07p13Guh1x+OF597vePe0y\nAAAAAFgDoSzb0rk7d+a4R7952mWs26mvfMC0S2A/dq/7HJP5+fOmXca67dhxtZx+2rumXQYAAABs\nGKEswBY1P39env7Sj067jHV70VNuO+0SAAAAYEMJZQEAZsxx9z0m8/Pz0y5j3Xbs2JFTT9ESHgCA\n7Ucou83d8+h75txzt0Dfqkccnve8+z3TLgPYD9z72HttmbDqne84fdplsJ+an5/P377zw9MuY90e\neO/bT7sEgKm6333ulvPmZ//72NV2HJ63n/b+aZex37vvve6a+Z3nTruMddtx+BE55fQzpl0GzDyh\n7DZ37rk7c+effeW0y1i3M9/26GmXwH7sXvc+Jju3QEh3+I4dOf2dWpStZn5+Pi94xT9Nu4x1e+Zj\nbj3tEmbCscfdK/Pnzf77e8fVduQdpwrhWd5973t0ztsCn2NX27Ejp5yy9w8ovd9x98z8ebMfVu24\n2uF5+6kaD7C88+Z35oN/+PRpl7Fud/i1F027hJkwv/PcvOtJD5p2Get2zIlvnHYJsCUIZYEtb+f8\nfJ7w2x+Ydhnr9vJn32naJcB+Z/68+bz6bf847TLW7VE/+9PTLoH92Hnz83n/6W+fdhnrdrd73W9N\n88+ftzMffOvrNqmace5w/4evaf77HXePrdNy8tT37vX89zv27ltnu9/xvmmXAcAMEMoCwBZwn+O2\nTrcNp2kxCmxj583vzIdP/rNpl7Fut3/oE9c0/3nzO/ORk16wSdWMc7vHPXPaJQAwI4SyALAFzM/P\n50/f8PFpl7Fuv/SQW027BAAAgE0nlAUAAACGue+9j8p5W+CBV1c7/Iic8s6/m3YZwIwSysI2csy9\njs65O2f/9uYjDt+Rd52+9w8KAQAA9h/n7Tw3Z/zW46ddxrrd9Xf+YtolsB879p53zvy5X592Geu2\n44ir5x3vOXPaZWxJQlnYRs7dOZ+H/9rsh5mv+8Ojp10CAAAArGj+3K/nNXe//bTLWLdHvu/D0y5h\nyzpg2gUAAAAAAGwnWsoCADAT7nvfYzI/P/vd8OzYsSOnnPKuaZcBAEMcd/RdMr9zC9zGf/jVc+q7\n//9pl8EWIpQFAGAmzM/P57R3v3/aZazbfY6+27RLAIBh5nd+PW95yDHTLmPdfu4NLqjujfvc/U5b\npi/d0973gU39HUJZAAAAAGDd5s/9ev7kpjeddhnr9uTPfnbTf4c+ZQEAAAAABhLKAgAAAAAMJJQF\nAAAAABhIKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBAQlkAAAAAgIGEsgAAAAAAAwllAQAAAAAGEsoC\nAAAAAAwklAUAAAAAGEgoCwAAAAAwkFAWAAAAAGAgoSwAAAAAwEBCWQAAAACAgYSyAAAAAAADCWUB\nAAAAAAYSygIAAAAADCSUBQAAAAAYSCgLAAAAADCQUBYAAAAAYCChLAAAAADAQEJZAAAAAICBhLIA\nAAAAAAMJZQEAAAAABhLKAgAAAAAMJJQFAAAAABhIKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBAQlkA\nAAAAgIGEsgAAAAAAAwllAQAAAAAGEsoCAAAAAAwklAUAAAAAGEgoCwAAAAAwkFAWAAAAAGAgoSwA\nAAAAwEBCWQAAAACAgYSyAAAAAAADCWUBAAAAAAYSygIAAAAADCSUBQAAAAAYSCgLAAAAADCQUBYA\nAAAAYCChLAAAAADAQEJZAAAAAICBhLIAAAAAAAMJZQEAAAAABhLKAgAAAAAMJJQFAAAAABhIKAsA\nAAAAMJBQFgAAAABgIKEsAAAAAMBAQlkAAAAAgIGEsgAAAAAAAwllAQAAAAAGEsoCAAAAAAwklAUA\nAAAAGEgoCwAAAAAwkFAWAAAAAGAgoSwAAAAAwEBCWQAAAACAgYSyAAAAAAADCWUBAAAAAAYSygIA\nAAAADCSUBQAAAAAYSCgLAAAAADCQUBYAAAAAYCChLAAAAADAQEJZAAAAAICBhLIAAAAAAAMJZQEA\nAAAABhLKAgAAAAAMJJQFAAAAABhIKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBAQlkAAAAAgIGEsgAA\nAAAAAwllAQAAAAAGEsoCAAAAAAwklAUAAAAAGEgoCwAAAAAwkFAWAAAAAGAgoSwAAAAAwEBCWQAA\nAACAgYSyAAAAAAADCWUBAAAAAAYSygIAAAAADCSUBQAAAAAYSCgLAAAAADCQUBYAAAAAYCChLAAA\nAADAQEJZAAAAAICBhLIAAAAAAAMJZQEAAAAABhLKAgAAAAAMJJQFAAAAABhIKAsAAAAAMJBQFgAA\nAABgIKEsAAAAAMBAQlkAAAAAgIGEsgAAAAAAAwllAQAAAAAGEsoCAAAAAAwklAUAAAAAGEgoCwAA\nAAAwkFAWAAAAAGAgoSwAAAAAwEBCWQAAAACAgYSyAAAAAAADCWUBAAAAAAYSygIAAAAADCSUBQAA\nAAAYSCgLAAAAADCQUBYAAAAAYCChLAAAAADAQEJZAAAAAICBhLIAAAAAAAMJZQEAAAAABhLKAgAA\nAAAMJJQFAAAAABhIKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBAQlkAAAAAgIGEsgAAAAAAAwllAQAA\nAAAGOmjaBexvWmu3SfLkJLdLckSS7yWpJG9L8rKqunAf13v/JD+f5NZJrpbk+0m+nOTvkpxYVV9c\nf/UAAAAAwP5OS9klWmvPTPKRJA9Kcnh6aHpBepD6wiSfaK1dc43rvEJr7fQkb05ybJKrJjkryUVJ\nfiLJU5J8trX2gA3aDAAAAABgPyaUnWitHZfk+UkWkrwkyY6qunFVXTvJbZJ8MckNk7xpjat+ZZKj\nk1ya5DeTXLmqWlVdNckdJ+u9fJLXtNauvSEbAwAAAADst4Syu7w4PZA9taqeVlXfXZxQVf+U5PjJ\n9Nu01o7fmxW21m6c3up2IckLqur3qur7S9b74SQPm/z38klO2JAtAQAAAAD2W0LZJK212yf58cl/\nX7zcPFX1qSRnTP77qL1c9U8mmU9ySZK/WmG9H0ty9uS/N9/L9QIAAAAAM0oo291lMrwgyUf3MN/7\nksyldzuwqqp6bVUdkeSHquo/9jDrRZPhD+3NegEAAACA2SWU7Y6cDD9fVZfuYb5/mwwPaa21vV15\nVS2sNK21dtUk15v893N7u04AAAAAYDYJZbtrpff7+tVV5ls6/Vob9LufkeSgJBcnOWmD1gkAAAAA\n7KeEst2hk+GFq8y3dPqhK861l1prP5vkqemB8B9X1efXu04AAAAAYP8mlO2uMBn+YJX5vr/k34es\n5xe21h6R5G/S+6h9V3qLWQAAAABgixPKdostYA9eZb7LL7PMmrXWfivJa9K7LXhPkuNX6csWAAAA\nANgiDpp2AfuJ8yfDK64y35WW/Ps7a/0lrbWDk7wyyUPTuyx4VZLHV9Ula13XWh122CGZm5u7zPi5\nAy47bhbNHTCXq1xltd23ywHL/C1m0QFza9zuLbK/D1jj/l7utT+L5ta4v7frdm/X1/m2Pa5t1+3e\nrq/zbbrdc1ukGcXcAfH+3gvb9fN7+273JhYz0NzcGt/f2/R4vl2Pa9t2f9vumbbW7d4XQtnu7CS3\nzeoP77rekn9/aS2/oLX2w+ndFNw2ySVJfqOqXrKWdazHQQcduOz4rfFW6dtx4IFr+MayhTZ8Tdu9\nhdjfezH7FtnuOa/zvbNF9rfj2vZju1c3t0Xe4HOZs7/3xtbY3c5b9n6JTatlLO/v7cZ2by+2e3MI\nZbtPJ3lwkhu21g6qqotXmO/IyfBbVXXW3q68tXaFJKenB7LnJ3lwVb1rHfWu2cUXX7Ls1eeFkUVs\nooUkl1yyhh4gttCGr2m7txD7ey9m3yLbveB1vne2yP52XNt+bPfqFrbIG3whC/b33tgau9t5y94v\nsWm1jOX9vd3Y7u3Fdu+ykUGtULZ7T5IXpj+8645JzlhhvqPTPzVP39sVt9YOTHJqktsl+UaSo6rq\n0+uqdh9885vLd4G7cOnWOAlYuHQh3/jGd/d6/ku3yFnfpQtr3O4tsr8vXeP+Xtgi+3thjft7u273\ndn2db9vj2nbd7u36Ot+m272wRb4HLVwa7++9sF0/v7fvdm9iMQMtLKzx/b1Nj+fb9bi2bfe37Z5p\nK233jh2Hbtjv2J7tj3dTVZ9K8sn0e0d+Y7l5Wmt3S/KTk/++Yg2rf26So5J8N8k9pxHIAgAAAAD7\nD6HsLk9NbwV799baia21/46+W2tHJXntZPpbq+rMJdOu01q7uLV2UWvtWUtX2Fr7sSRPnyz39Kr6\n5wHbAQAAAADsx3RfMFFVf99ae0KSE5P8YpJHt9bOSnLlJFdPD1Y/lOSE3RadSw+3F3LZkPvJ2fU3\nfkJr7bF7Ucct9nETAAAAAIAZIJRdoqpOaq19JMmvJrlzkuuldztwZpLXJXlVVS3XOcZKHWb8yJJp\nN96LErZGxxsAAAAAwIqEsrupqn9J8ug1zH92kgNXmHZCLtuyFgAAAADYxvQpCwAAAAAwkFAWAAAA\nAGAgoSwAAAAAwEBCWQAAAACAgYSyAAAAAAADCWUBAAAAAAYSygIAAAAADCSUBQAAAAAYSCgLAAAA\nADCQUBYAAAAAYCChLAAAAADAQEJZAAAAAICBhLIAAAAAAAMJZQEAAAAABhLKAgAAAAAMJJQFAAAA\nABhIKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBAQlkAAAAAgIGEsgAAAAAAAwllAQAAAAAGEsoCAAAA\nAAwklAUAAAAAGEgoCwAAAAAwkFAWAAAAAGAgoSwAAAAAwEBCWQAAAACAgYSyAAAAAAADCWUBAAAA\nAAYSygIAAAAADCSUBQAAAAAYSCgLAAAAADCQUBYAAAAAYCChLAAAAADAQEJZAAAAAICBhLIAAAAA\nAAMJZQEAAAAABhLKAgAAAAAMJJQFAAAAABhIKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBAQlkAAAAA\ngIGEsgAAAAAAAwllAQAAAAAGEsoCAAAAAAwklAUAAAAAGEgoCwAAAAAwkFAWAAAAAGAgoSwAAAAA\nwEBCWQAAAACAgYSyAAAAAAADCWUBAAAAAAYSygIAAAAADCSUBQAAAAAYSCgLAAAAADCQUBYAAAAA\nYCChLAAAAADAQEJZAAAAAICBhLIAAAAAAAMJZQEAAAAABhLKAgAAAAAMJJQFAAAAABhIKAsAAAAA\nMJBQFgAAAABgIKEsAAAAAMBAQlkAAAAAgIGEsgAAAAAAAwllAQAAAAAGEsoCAAAAAAwklAUAAAAA\nGEgoCwAAAAAwkFAWAAAAAGAgoSwAAAAAwEBCWQAAAACAgYSyAAAAAAADCWUBAAAAAAYSygIAAAAA\nDCSUBQAAAAAYSCgLAAAAADCQUBYAAAAAYCChLAAAAADAQEJZAAAAAICBhLIAAAAAAAMJZQEAAAAA\nBhLKAgAAAAAMJJQFAAAAABhIKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBAQlkAAAAAgIGEsgAAAAAA\nAwllAQAAAAAGEsoCAAAAAAwklAUAAAAAGEgoCwAAAAAwkFAWAAAAAGAgoSwAAAAAwEBCWQAAAACA\ngYSyAAAAAAADCWUBAAAAAAYSygIAAAAADCSUBQAAAAAYSCgLAAAAADCQUBYAAAAAYCChLAAAAADA\nQEJZAAAAAICBhLIAAAAAAAMJZQEAAAAABhLKAgAAAAAMJJQFAAAAABhIKAsAAAAAMJBQFgAAAABg\nIKEsAAAAAMBAQlkAAAAAgIGEsgAAAAAAAwllAQAAAAAGEsoCAAAAAAwklAUAAAAAGEgoCwAAAAAw\nkFAWAAAAAGAgoSwAAAAAwEBCWQAAAACAgYSyAAAAAAADCWUBAAAAAAYSygIAAAAADCSUBQAAAAAY\nSCgLAAAAADCQUBYAAAAAYCChLAAAAADAQEJZAAAAAICBhLIAAAAAAAMJZQEAAAAABhLKAgAAAAAM\nJKWya0sAACAASURBVJQFAAAAABhIKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBAQlkAAAAAgIGEsgAA\nAAAAAwllAQAAAAAGEsoCAAAAAAwklAUAAAAAGEgoCwAAAAAwkFAWAAAAAGAgoSwAAAAAwEBCWQAA\nAACAgYSyAAAAAAADCWUBAAAAAAY6aNoF7G9aa7dJ8uQkt0tyRJLvJakkb0vysqq6cH9aLwAAAAAw\nW7SUXaK19swkH0nyoCSHJ/lykguS3DrJC5N8orV2zf1lvQAAAADA7BHKTrTWjkvy/CQLSV6SZEdV\n3biqrp3kNkm+mOSGSd60P6wXAAAAAJhNQtldXpwenJ5aVU+rqu8uTqiqf0py/GT6bVprx+8H6wUA\nAAAAZpBQNklr7fZJfnzy3xcvN09VfSrJGZP/Pmqa6wUAAAAAZpdQtrvLZHhBko/uYb73JZlLcscp\nrxcAAAAAmFFC2e7IyfDzVXXpHub7t8nwkNZam+J6AQAAAIAZJZTtrpXer+tXV5lv6fRrTXG9AAAA\nAMCMEsp2h06GF64y39Lph6441+avFwAAAACYUULZ7gqT4Q9Wme/7S/59yBTXCwAAAADMKKFst9hS\n9eBV5rv8MstMY70AAAAAwIw6aNoF7CfOnwyvuMp8V1ry7+9Mcb1rdthhh2Rubu4y4+cOuOy4WTR3\nwFyucpXV/sy7HLDM32IWHTC3xu3eIvv7gDXu7+Ve+7Nobo37e7tu93Z9nW/b49p23e7t+jrfpts9\nt0WaUcwdEO/vvbBdP7+373ZvYjEDzc2t8f29TY/n2/W4tm33t+2eaWvd7n0xt7CwsKm/YBa01k5O\n8uAkn6yqn9zDfMcneWP6w7tuUFVnTWO9AAAAAMDs2iLX3dft05PhDVtre2o9fORk+K29DE43a70A\nAAAAwIwSynbvmQwPSXLHPcx3dHpr1tOnvF4AAAAAYEYJZZNU1aeSfDLJXJLfWG6e1trdkix2QfCK\naa4XAAAAAJhdQtldnpreWvXurbUTW2uHLk5orR2V5LWT6W+tqjOXTLtOa+3i1tpFrbVnbdR6AQAA\nAICtyYO+lmitPS7Jielh9Q+SnJXkykmunh6cfijJvavqgiXLXDfJlyfTn1dVv70R6wUAAAAAtiYt\nZZeoqpOS3CLJa5J8Lcn1khyc5Mwkj01y5xWC0z0m2+tYLwAAAACwxWgpCwAAAAAwkJayAAAAAAAD\nCWUBAAAAAAYSygIAAAAADCSUBQAAAAAYSCgLAAAAADCQUBYAAAAAYCChLAAAAADAQEJZAAAAAICB\nhLIAAAAAAAMJZQEAAAAABhLKAgAAAAAMJJQFAAAAABhIKAsAAAAAMJBQFgAAAABgIKEsAAAAAMBA\nB027AICtorV2nSSXVtVXp10LAHvWWrtCkmsmOTTJIUkuTPKdJF+pqoumWRtsttbaVZIcnmRnVX1j\n2vWM1Fp7dpLvVdXvT7uWEVprP5rkSkn+q6rOm3Y9sFlaaz+S5OpJvlFVO6ddD9PRWrt2VX1l2nXs\nrbmFhYVp18A201q7YpKXJVmoqsdMu5590Vr7oSQtyQ8l+feq+vZeLve4JFevqt/ezPo2S2vt+klu\nkOScJP9aVZfuNv2RSe6f5IgkO5OckeQVVXX+6FqnobV2aZJvVtVVp13LZmqt3TLJQ5PcLJOT/CQf\nT3JyVf37NGvbTK21qyW5d5Kbpm/3N5N8Ksm79/YYsL+ZfDH9QZK/2q5f1CYXU1qSryX53DLHtQcl\neXCSayT5RpIPJDmpqr45utbN0Fq7QZKbJFn80n5gkvPT39f/J/1Yf/H0KtxYrbVbJ3lUkrsm+fEs\nf9fYRUk+l+SdSV5dVV8cVuB+pLV21STz6RcbZ64hx+R88+eS3Dz9fO1z6Z9Tq753W2vvTHKjqrrB\n5la58Sbv6Udm1/nau6rqjCXTb5jkL5Lccclin03yu1X1ppG1TsvkfO1bVXWVadeyr1prP5zkkqr6\n7grTj0jym+mfX0vPS/8jyeuS/GFVfWvTC90kk3Oya1bVZ3Ybf/kkv5D+3t/9fO0tSV5bVd8fXO66\ntda+lH6+9vtV9cpp1zNaa+3aSU7IrvO106rqzCXTW5KXJrl7krnJ6M8neX5VvW5stRurtTaXfry+\nW/Z8vvbx9L/LzH7vbq09L8lzq2qfQ8rW2kOT/OksHd+Fsgy35CR/oaoOnHY9azFpVfPi9JPdK0xG\nX5Lk9Ul+s6rOWWX5TyY5cga3+5rpJ3B3WjL6y0keXFUfb60dmOTNSY6bTJtLsnhw+XqSY6vqE6Pq\nnZYtcpL/1iRfqqqnLTPth9K/yD1iyeil+3oh/YLL06rqks2udaNNWg29LMl/VtVvLBk/l+R5SX4t\nyeWXWfTCJH+QfuJ36TLT91uT1+xC+heWp1fVK6Zc0jCTz6LXJDlmyegvJnlCVf1da+2AJO9IcvRk\n2tLX+nyS+1TVx0fVu5Faa4cmeVr6e/m6q8x+Yfrf4eVV9cHNrm2zTLb5pCQPnIya28PsixbSP+P/\nOMkzZvG4th4zfr52+/TzksN3m/StJL9VVX+2yvKzer720CSvTHK53SY9v6qe01q7VvoX9x257Htg\nIcmzq+p3N7/S6doi52uXJvlkVf3kMtNuluTd6a//5Y51C0nOSnJ0VX1+M+vcDK21+yZ5VZK3VNVj\nl4y/QZLTktxwMmrpti9+fv9bkvtW1RdG1LpRlpyvJcmHkjy+qv5tiiUN01o7Lskbctlz8FdW1WNb\na9dN8s9JDsvyx7XnVtXvbH6lG6+1dnySF2X5c7XlXt8/SP/O/ltV9fVNLm/DTV7nH0rysLW2dJ2c\n5/1ZesOhzNLnt1CW4Wb1JH8SzJyRfqVquQP+t5Lcv6o+sId1zNxJ/iSI+2T6lcndt3s+yY2TPDzJ\nSybjvpTk3CTXTnKtybhzkvzELF2Rb639/D4s9ur0AOOJS0dW1V9vRE0jTD4MP1VVt1xm2hvSA425\nJJemn9h+M/2k/8ezK7R6RVU9bljRG6C1dkiSjyX5/9Jbvt57ybS/SG91sfj6/276+/3K6Veqk77d\nr6uqRw4regNM9vcl6VfbF5J8OP1E/1+nWtgma60dlOQfktwilz2uXZjkzknukeT56X+XLyc5L8l1\n0m+LS5L/THKTqvrOgJI3TGvtFuktQI/I8l/WL0jy/SQ/nP8Z7CykX4B8zKzd2j/Z3x9Icpv0bf5C\nkvenH8O+mr7Pv5/+he9KSa6X5Mj0VilHpG/7W6rqgbuveyub4fO1q6e3ir1y+v6+OP0z6+DJLAtJ\n/jLJE1e6kDaj52s3SG/xuhhcLLaWOjR9m2+X5HHprc2+lH7Osni+9tAk10//PPipqvrksMLXqbV2\nxupzXcad018XH1oybqGqjtqQogZY6Xxtct5e6Z9XSfLe9OPd4vnanbKrNWGlv85n5pjeWrtReqvX\ng5N8oKruMhl/ufTX/2Ig+y9JPph+h8uPJPnpJLeaTDsryU1XamW8P5rs751J/m96QPeD9IZCv1tV\n/3eatW2mSQvZ/5Nd59vfTD83uVL6ce2J6e/nB6UHs6dk1/nasektSy9Jcsuq+uzI2tdrcjfbc/I/\nv3edl/79+vrpLYb/Jv313ZL8ZPpdIQvpn93HzNKxPPkfFx++neRxVfXmvVzudklem37+Npfkw1V1\nh82qc6PN3K1IMEUPy66Wou9O8sb0WxxvlX6Ce1iSd7fWHlBV75xOiZvihCQ3Sj9AnpR+cjeX5JeT\n3CHJE9L/Nhekt5w9fXHB1to907/EXyP9i8As9d316uy66rgWV0i/er9oIcnMhLIraa3dIf2EZyH9\nBOBpS1uGt9aukeRZ6a+Hx7TW/rqqPrTsyvZPj0m/wHB+euuqJElr7bZJHpu+3ack+Z30likLk+k3\nTvIrk3ke3lr72xl8/382ybOTvDzJzyT5VGvtxPSWVVu1n8FHJLll+pebF6e3HLtcknumB/C/nn5s\n/0qS46rq04sLttZuk/4euPZk3j8aWvk6TG73fE+Sq6Xfwvqy9LDyvPTteUb6a+AJSd6aftJ/hyQP\nSQ8oH5YedN13dO3r9AtJbpu+nY+tqlP2ZqHJXSAnpF90/LnW2v2r6q2bVyYb5EnpX1Ln089V3lJV\nl7TWjkz/nHpA+jH7cunH/q3iCemB7BeTPKKq/mHSoODxSU5MPw+7f/qF9jtV1QWLC7bWXpDkfdkV\n3D5hcO3rcefsOl/bmxbwiw6aLLtoq7RUekR6IPW9JD9XVe/abfoLW2s/neTU9ADzMUn+fGyJ6/KE\n9ED2I+ldMyx6RPr2XJjk4VX19t0XnJzTvTk91HxykhduerUb65z0z+QXpB/n/neSR03Cu1et55bv\n/dgT0wPYjyV5SFV9Ofnvffm36X+H6yV502T6f19oa609K72F7fHpx8FfGlr5OrTW7phdgexLkrxo\naR+5k661XpHkWlX1oMm4K6Qf438r/b1wWmvtxjPWxdoD049HV0nyxtbaK5M8eaULD5O72p6bfv56\nUPrFiudmtjIHoSyswUPST9j+qKp+fcn4k1trL07/MLht+gHkrlX1sWkUuQnun77dT6+qFy+ObK29\nI73l8EPTD/y/ujSQTZKqek9r7dfTPzTunRk7QE5ckt4ibm9cN/1vNTMdi6/BgybDd1bVQ3efWFVf\nS/Kk1tr3kjw1PQSZpVD2wen77tFV9ZYl4x8+Gb61qo7ffaGq+lySx7fWzkk/eXpceivEmVJVp7XW\nzky/ReoX04PmE1prL03vl+m/plnfJnhg+v5+1G59KL61tfZfSZ4++f89lwaySTIJOh6d3vLo2MxQ\nKJseNl8t/b15r6WhTHrLoQ+21k5Ov5D02aqq9Falr2qt3SM9jL5Pa+34Get7cvH9/fCqeu/eLjTp\nruCvWmvfSd/2R6eH1TNhH+/4WOpKq8+yXzo6fX8/pqpOWxw56Xvyga21X0l/3z6qtfbVqnrOlOrc\naEelb/cJVfUPSTIJaf580t/iY5JcMcmv7fbeT1V9r7X2pPQWiDPTumjiwvQH9V2c5OT0OxtW85z0\n0PJFm1jXtNwj/XXw28sEskmSqvrH1trj049nD8hshbL3SG81+PNVde6S8fdK3+5nLRfIJklVfbS1\n9rD07y/3z+yFspm07v2VyWf1Xyb5icnwmZPvo6/ZYi1n756+Xx+2GMgm/70vn5oezC6kNxb5H3c+\nVNWlrbVfTX+NL+2CbxYsBsjPWa7rhap646TbtT9trT2wqv52st9f31p7e5J3Jbl9kqekd8E2E6rq\nza21D6U3cLpn+nnX7VtrD9n9fLz1Z928PslPpYfXn02/IPmZzBihLGvWWttWfaotccv0gO65u0+o\nqnNaa0elf0m/XZJTWms/tda+UPZTR6a3CP7jpSOr6gettT9I8rb0v8urllk26X02vjz9tvBZ8oz0\nD7GD0zuK/9Wq+pc9LTC55eI7VfVjA+ob7cbpJz3PX2W+38muVtSz5AbpT2p9y27jb5K+3av1RfV7\n6YHXT21CbUNMvqQ/qbX2+iR/kn7Me3aSX2+t/U36wzFW7J5lxtw8ybdXCBZPSn//f6uq3r/cwlV1\nRmvtm+nvi1lyn/Qvs4/cPZRZ4hnpF2GekiWt5arqva21x6ZfgDxhMpwVN00yv5ZAdjdvSm9peOuN\nK2mIV2frtP5bi/+V5IKlgexSVfXHkxZFL0jyrNba56rqjUMr3Bw/ln5cW+6C6GvSL7b93/TW8ZdR\nVZ9prX0ru257nxU3TT8HvWP6BZg/TO9DcsVb8ltrz0nyvaqambBiDRa7Djt5lflOSe+K6Sc2t5wN\n96NJ/qOqvrTM+KS3jFxRVZ3ZWvt6+nFiZk2C9VumtyR9VvqdLScmeVFr7S3pLYLPrKoLp1jmRrhB\nkq/U8g/cXDxH+8pK37mr6j9ba2en3w00S26X3l3anho0/WX6BcYT0sPpJD24b62dkOTfk/xsZiiU\nTZLqfeEe01p7Yvr23yjJP7TW/ndVvTRJWmuPSs8mFrux+IP0fnRnpiuWpZZ74iysZm4DfmbRVZKc\nvVL/Q1X1vfQHXX0+vQ+6Uyb9VM66w9JPfpY7yH14Mty5Ur+Kk6uWZ6Xf8jozqur300Opf06/bffj\nrbXfnNwmsR1dcTL89J5mqt5v8JfS3wOz5MrpfXXtbrE/wT32sVr9Sb5np79fZlpVfaSqbpXeSvgL\n6S2QTkhyRmvt662117XWHttau11r7aqT22NnzVWycgv4xRP71VrIn5N+i/QsuU6SLy9tbbK7qvqP\n9G2/+zLT3pL+lN/L9Dm9n7tSet37ZNLacGdm7HNsie12vnZIel97K6qq30v/QjuX5JWttVvtaf4Z\ncUj6w1WXs/hQo6+tcovz19L7JJwZVfXlqrpz+oWki9MvLH1iElhtR4uNZ/b48OHJ6+A/M3vnLZek\nX1zc3eL3kL25Vfs72QKN06rq4qr6k/Tg8tnpXfQcmv5A6nck+UZr7cOttZNaa7/WWntUa+0BUyx5\nX1wxu/bt7s7fbbiS72bXA7pnxY70sPn7K81QVRenf8e+xTLTvph+18DMNhSq/kDOWyb5p/TPpT9s\nrZ3eWntT+l24h6Zv4x2r6hmzGsgmQln2zefSr0ickX7lZa0/fzC+5A3xvazyhWzS/+K907/83Syr\nX6WeBT/IrkBud4tfcudXWcfl01tnzJTqDzu6TfoV6CT57fQrdbPW6ncjLAaWe3NSM5fZ+3z5WpLr\ntNZ2f7LrWZPhj2YPJg8SukbWEfzsb6rq5PSr08enX4CZS39IyEPSb3X8YPrr4uLW2monxPub72fl\n1/IVdxuu5Erpnwtb0feSXHOFaf+V2Qsnz0ty7dbawavOuYzJg3Ouk9U/6/Y3r0h/374p/YvZWn8u\n81T3GfHN9P292ufQE5P8Xfqx4JTJA2Vm2bezwmfVkpbxKz6YcHKB7RrZu1BrvzMJp26e3tfoTdLP\n135n8vm8nSxedFvpGL7UVTN75+dfSHK91truT6P/1GS4xztYJg8CvH72vmuy/V5VnV9Vz0/vRu0X\nknw0/dh/cHrXeo9Jb3H4ivSueGbJt7PrAau7WzzerTR90TWzh2Pffur89GB2NYelP5h1OQdnxu+W\nqap/T281/Lz0CzL3TO96JEn+IsnNquojUypvw8zal2b2Dz+f/qY4MsmfV9Xz1vKT2exXNOmt/67a\n+pOrV1RVX0jvu+biJMe21k4aUdwm+nKSq7fWLnM72+Qq+2eyh1aErbX/lf5ldiZPfqrq0qp6QfpD\nfz41GX6itfb0GW0huK8WW0X/9J5mmjxI6PrpT3SeJe9Pb2X0y7uNf1P6ie3Rqyz/+PRWkx/f+NKm\np6oWquotVXXH9L6jn5fkE5PJS1vTzdpdAWdn+S91SXLXyfC6rbVlA47J+Osm+eom1bdZvprkx/YU\nPk2mXT/LtDxprf1I+nbvsRXifuhj6SH7vj7k42mT5T+82oz7maemv9YfkORGVXX2Wn7SHwY3iz6V\nfjH4hD3NNOkz+AHpT6C/RpIzW2vX2/TqNs9nk1yptXavFaY/Nf3Bhit5QPrn2B7vDNmfTc7B75j+\nnr0oyTPT73S62VQLG+ud6Z/L99vTTK21n0kPs5a7LXx/9qb0DOMvd7vQ9or0ff6klRacBPQnpd8F\n9Z7NLHIaqup7VfXKqvqZ9M/xX05vMXtOZvcOiM+kf/9e7gGjj54Mr9Jau8tyC0/GH5bZO659Ln27\nV/z+0Vq7X3pjict03TDJK6613LRZM/msfnt2nXPPpXe98paV7mCeNUJZ1qyqPpHkd9MfFvLq6VYz\n1HvSDwIntdb2eOVq0u/iCem31zymtXbypP+yWXRm+na/dLnWBlV186p6yHILTuZ/WfpVur/fzCI3\n26Q/2Z9K71P4gPS+6D7SWrvhNOsa6FXprSmetUoY/eL0k91/HFLVxnlp+sn8C1prz2itXe7/tXff\nYbJVVd7Hv30vIknJSTAQZIkKBkR9HZAggghiGkEFVASVUYyjBMWEiihpQDEhYgIdzBgBgcEZFYwI\niixRBMSASBAk4+33j33Krtu3urq6u0KfPt/P8/Rzq7vOqdq/quq+p1btszZAZp4JfAd4T0Qs0y82\nIlaJiMMoK6OOAx8f4piHKjN/W324tjXlIPCpwBsoH7TV7cOncym/x6dUCyUAEGV19uMoH0bdDhw9\nxWy7oynPd50WswM4n5L7I51mjVYzQj9L+Zv/w0nXLaL0Gr4v9Xtz8zFKpvdHxPERseF0O0BZRCIi\nPko5S2Kc8v9ZbVSzI1uFyVMjYs1RjmeIPk95vk+MiJdWr+uOsqxK/VRK772NgO93KWrOd1+j5P5o\nRCwzyzkzT6jOgFhGRDyR0p9vHOjYi7cuqg8Tj6Oc0nsRZRLJjyLi7RGxuPvetbNKRDy5/Ysyo/8G\n4K2dJlTAvxbI+Tjl+T5neMPtiw9QZss+BbgoIvaIiPtUM+peA7w4Ij4QEf9qoxURK0dZ+PDnlDMa\nb6dei3TOWGZelZknZeYzM/OBlFnRD6fMOqyTr1D+rp0SEQdGxCMiYquIOBZ4C+UY/ffV9Uv1fa+O\n6T5BeZ1/i3r5KiX3pyJi+8lXRsQzKB9EjDNpgeHq78CXqedx6jKiLNZ2IWVSwC2UVpGrA9+JiGNn\nexbUfDI2Pl7rGc0akeqg5iLKAc8bMvOEaXZp33dNygHDeGbW5uAoIh5AeSO6CnAjcCZwQLfeXBHx\nHEoLg/tQZpysAqxRs9wPo8y+WETpJ3pCZn5qmn1Wp6xK/jrKqWRLgK1y0qqJdVXNuPgU5UD/Tkpj\n8WOjLPR1c2au0fUG5rEqw9107kW2HqUg86rM/Mik/bahzEhprXq9c2aeO+Dh9lVE7E85FWaMclr+\nmZRZoTdRivEPpbx5ab2ON6KcRnP/ap/PZOaLhzvquame74szs1G996oZ/JdS/jbfDlxOmVm3OeW5\nPITy/9vzKbM0Pk35nXgIZTbZVpTX+dbVB5W1ULVe+QXlg5MrKD01L6XMAn04cCATp7zulJnnV/s9\nhlLo2rS67tnVBxa1EREfpsxob/2fnZSZKNdSXgN3U071W5kyu+SRTOQdA96dmW8b5pj7JSKOpyzy\n9I3M3GMG+9X1eG0x5UOFx1Ge79uAVac5XlubcsbEFrSd7lmz3CtS/pY9kJLh9Mzcd5p9nkj5kHkb\nSo/N6yizqmvZwmCy6kPkNzGxcOvFwL7AL1kYx2vTvZE/PDPfO2m/44CXUf7W/QPYPDNrdTZbRGxE\nWV1+MyZ+x39F+Xu+BeV4bZyJljNrUd7HjFHaFz0/M7825GHPSYOP11agHIdtyrKv99bp7E+g/B1b\nQnkdtI7XNqM859dT/q7dNJxRz11ErEI5PnswJfdlTByvbU7pIzxGac+0RWZeV+23K+WDtTFK64fH\nZOZVwx5/P1QfrHyK8sHpGKX29ALKe7STKL2TxymPy96Z+asRDXXOnCmrWammke9LeXO3V/unkQtV\nZv6JkvkeyqeNe02zWAKZ+WXK4l83UP6o1m6WSmZeDhxM+WP4KOC5Pey2G2VmZet0sfcvlIIsQJXl\nccC7KG9i3h8R5412VH21POVgZvLXCpTXQac3MR+gHBgBfLpuBVmAzDwF2JUyY2pdSl+uD1FW8g1K\nIWsXyu/DwZReq6tS/iYcBbxk6IPWrGTmFZRTHMcpB7hbUXoQLqKcFXE8cBjl9KhHUWaAn0456G/N\nQDuuTgVZ+Fef7DdW325GmeX8bcoqzUdQ+rO1CpDnt+26mPImd4zy97xWBVmAzPwPygeFt1ByPIyy\nKvGrKUX4t1b/HkQ57beV96/Ai+takK0cRinKnhsR64x6MINWHaPuxkRfxbt7OF67nlKYPI16nuJL\nZt5ByX0NZfy9nKG1AbA95VjmFuB5C6UgC/+aNdtauPUnlA/bfjTaUfXVbBbrexzl/727gX3rVpCF\nsrgbZdLHWymL261CKcw9l4m/3Ysox3LrMrFo63eAx9etINtkWRbS3oVSmG1/Xd8C7F8dq7yf8twu\nohTld6Yct49RWjHtWaeCLPzrTJfdKC2IxijHqHsBu1MK1GOU4vNurYJs5YrquiuBp9W4IPsMSrG1\ntejs+4Ftqxngt2fmfpS6zD8ok6R+HBGvG81o586Zshq6aor5CwCmm3E5H0XEI4G3A5tmZtf+sm37\nrEt5M78vsLhOMy9aImJ3yoJXl2XmS6fZdifgbEox+ojMrNUpnzNRzSD7FGVWFdR/5sXbe9js3Mxc\n6nSYiPgK5SDoOOAd1ZviWqpmWe1EaSS/NeUT6cmnv95MOVj4NnBaZtayZ1NTZ160RFlx/SDKwftf\nKL3XPpmZS6rrH0459b39dL/rgPdmWVSmlqrTs4+gFCra/QJ4V/WBYvv2K1OKVSdmZq0/gKpmEz4H\n2IFyIL8hZQXfFSkzZm+lvAn6BaVA/82s8Yq+TVbNktyVshDIe6fbvm2/bSmtWVbLzI59CueziFiJ\nsv7D3Zn5iWm2fRLld/tM4OjMrFuf7J5VLVgOoRzDL0/9j9c69USf7KbMXGqBo4j4IOXD9ffUeWZZ\nS/V7/kTK8dojKBNg7kf50PVWSk/NS4Gzqgk2tRQR/wP8JjNfPuqxjEL1+/tkJo7Xvje50BoRBwD7\nUHqE3wRcQDluqd0HDy3VMctLgadTJshAOUY5CzglJxZxbG2/iDIh7BuZee8Qh9oX1czo44GXM/HB\n+Isy8+wptt8U+G/KB27jlDNeXpKZtVr7wKKsNERV78L7VQto1FJEjE0346TKuQXwgya8ma36j76D\n8mHDLZn56NGOaPiq4tUfF9IMm3bVaUT/OshfKI3lqzd1d9ft4GXYoiz+0zrIz+n+BtZFNWuy9cb+\nmkmzLSRpwamOV/4duCsz3zfq8UiSioi4jIlZzucC+0x3bFq9Dz+GcubTOOVY/RWZ+aUBD7dvLMpK\nkiRJkiRJGonqDL57gbcB75vJJIiq5cEnKLPll2TmMguUz1e1Gajmr+qX58rM3HTajRcQc5u7lIqB\nGAAAIABJREFUCcxt7iYwd7Nyz1TVvuEDlAWv9h/1eIbF3OZuAnObuwnMbe6auBp4QWZeONMdM/Pr\nEfFoylog/9b3kQ2QC32pH26iLHTTNOZuFnM3i7mbxdzqZgXKQn4vGe0whs7czWLuZjF3s5i7Weqa\n+zGzKci2VP2Dtwfe3bcRDYFFWfXDBcAmEbHxqAcyZOZuFnM3i7mbxdySJEnSiGTmzX24jSWZ2cvC\n1fOGRVn1w0HAT4GzIqJ2q9TOgbnN3QTmNncTmLtZuSVJkjSPRcSSiPjtqMcxaPaUVT/sA3wV2AX4\nbkT8Ffg1cD3QbYXyuvU4mczc5jb31MxdT+Y2dxNyS5IkaX5rRJsti7Lqh6OA1sp4Y8C6wDrT7DNW\n7VPnN3XmNre5OzN3fZnb3Asud0T8c9RjGAVzN4u5m8XczWLuZmlq7g4uAHaPiI0z88pRD2ZQLMqq\nH65h4k1dk5i7WczdLOZuFnMvbGOjHsCImLtZzN0s5m4WczdLU3NPdhCwPqXN1ssz8/xRD2gQLMpq\nzjLzIaMewyiYu1nM3SzmbhZzL3iXAZsD5wP/O4v9VwLe1NcRDYe5zT0T5q4Xc5t7JsxdL03NPVkj\n2mxZlJUkSdJC9iLgQmBLYO/MvG4mO0fEmtTzzY25zd0zc9eOuc3dM3PXTlNzT9aINluLRj0ASZIk\naVAy82fAe4C1gE+OdjTDY25zN4G5zd0E5jZ3Q13T9nV19XXNNF+tbWrDmbLqm4jYAHgipe/H6vTQ\nCyUzjxj0uAbN3Oaebh9z15e5zT3dPuaujXcDzwB2jojXZuYJox7QkJjb3E1gbnM3gbnN3ShNabM1\nNj7ehPUdNGgRcSzwGmY4+zozFw9mRMNhbnP3wtz1ZG5z98Lc9RERmwOnAXcCz+71dMDqNMDrKT3K\nzF0T5jZ3j/uZ29y1YW5z97hfrXM3jTNlNWcRcSDw+lGPY9jM3SzmbhZzN4u5myEzfw08dha73grs\n1+fhDI25Z8zcNWTuGTN3DZl7xsytec+ZspqziLgI2JrSjPowymqBN2TmkpEObMDMbW7MvWCZ29yY\nW5IkSRqphd5my5my6oeHAfcCz8zM60c9mCEyt7mbwNzmbgJzNyu3JEmS5rnZttkCalOUnWkwqZPl\ngCsb+IbO3M1i7mYxd7OYu0EiYklE/HbU4xg2czeLuZvF3M1i7mZpcO5Wm63FlNmxM/mqDWfKqh+u\npZkFfnM3i7mbxdzNYu5muQm4Z9SDGAFzN4u5m8XczWLuZmlq7lZf3AXdZquJB+LqvzOBjSNiw1EP\nZMjM3SzmbhZzN4u5m+UCYJOI2HjUAxkyczeLuZvF3M1i7mZpau72NlsXZOb1C60gCxZl1R9HAtcA\np0TEfUc9mCEyt7mbwNzmbgJzNyv3QcBPgbMiYodRD2aIzG3uJjC3uZvA3OZugka02RobHx8f9RhU\nIxHxoCmu2hD4KHBf4MPAj4Fbpru9zLykf6MbHHMvw9zmNvcUzD2/mXsZCzp3JxFxMKXf2C7AdsBf\ngV8D1wO3ddl1PDP3H/wIB8Pc5sbc5l6WuWvI3OamGbkTGMvMzUY9lkGyp6xm6vc9bHNMj7c1Tn1e\ng+aemrm7M/f8Z+6pmbs7c9fTUZQMUN7krAusM80+Y9U+tX1zg7nB3ObuzNz1ZW5zm7uzhZD7TOD1\nEbFhZl476sEMSp0PqDUatVrJro/M3SzmbhZzN4u5dQ0Tb26axNzNYu5mMXezmLtZmpr7SOC5lDZb\ne2TmXaMe0CBYlNVMNamHSTtzN4u5m8XczWLuhsvMh4x6DKNg7mYxd7OYu1nM3SxNyN2lzdY+lDZb\nl0bEgmyzZU9ZDU1ELJeZ9456HMNm7mYxd7OYu1nMLUmSJPVXRPyzjzc3npm1mYC6aNQD0MIREc+K\niB9GxB5TbPKhiLg4IvYa6sAGzNzmnsTcC4i5zT2JuSVJkqT+GuvzV23Upnqs+S0ijgQOqb5de4rN\nxoAtgdMjYsvMfMtQBjdA5gbMPZm5zV1r5gbMPdmCyw0QERsATwTWB1anh4P4zDxi0OMaNHObe7p9\nzF1f5jb3dPuYu74WeO7GttmyKKs5i4gdgEOrb38OXDHFpmcADwZ2Ag6NiLMz84IhDHEgzA2YuxNz\nm9vcNWNuoEG5ASLiWOA1zPyssbq8uenI3ObukblryNzm7pG5a2ih557LcWXd22zZvkD9cFD17wcy\nc6vM/F6njTLznMzcGXg/5VOd1wxrgANibnMvw9zmHtYAB8Tc5l7GQssdEQcCrwcWs4BPh5vM3Oae\nwVdtmdvcM/iqLXObewZftbfQ22w5U1b98CTgDiZOf5zOOylv6J40sBENh7l7Y+56M3dvzF1v5u7N\nQsm9X/XvhcBhwGXADZm5ZHRDGgpzm9vcC5e5zW3uhaupuRvRZsuirPphDeDKzLyzl40z846IuBrY\neLDDGjhz98Dc5q4pc/fA3OauqYcB9wLPzMzrRz2YITK3uZvA3OZuAnObe8FrSpst2xeoH24EVu11\n44gYA9YCbhrYiIbD3D0wt7lrytw9MLe5a2o5SjG6MW9sKuZuFnM3i7mbxdzN0tTcjWizZVFW/XAJ\nsG5EbNfj9i8E1gQuHdyQhsLcvTF3vZm7N+auN3P3ZqHkvpZmHgObu1nM3SzmbhZzN0tTc8+mzdad\n1KzNVhOfWPXfaZRPJL4QEXtGxOJOG0XEqhFxCHAyMA58ZohjHARzm3sZ5jb3EMc4COY29zIWYO4z\ngY0jYsNRD2TIzN0s5m4WczeLuZulqbnXAK6ZSZst4Opqv9oYGx8fH/UYVHMRsQg4D3gy5c3abcDl\nwPWU3ierUZoyb8rEioHnZeZOIxlwn5jb3Jjb3OY2d401OPfqwE8pvcn2yMy7RjykoTC3uUc8pKEw\nt7lHPKShMLe5RzykoYiIPwPjmfmAHrcfA/4K/DMz1xvo4PrIoqz6IiJWBj4O7NX24/YX11jb5c8D\nB2Tm7cMY2yCZ29yYG8wN5jZ3TS303BHxoCmu2hD4KHBf4MPAj4Fbpru9zLykf6MbHHMvw9zmNvcU\nzD2/mXsZ5l6AuTuJiLMoi3ft2MvCXRGxN+VsrnMz86mDHl+/WJRVX0XElsCewNbABsAKwO3AH4Cf\nAWdkZt170S3D3ObG3OZeYMxtbhZA7oj4Zx9vbjwzl+vj7Q2MufvC3POcufvC3POcufvC3DUUES8C\nPgn8jbLo15cyc5nHJyJWBQ4E3k4pWu+XmZ8e4lDnpLZPkOan6pOY2n4aM1vmbhZzN4u5m8XcC87Y\n9JssSOZuFnM3i7mbxdzN0tTcnXwWeCmlzdbngNsiopc2W7UpyIJFWUmSJC1cO4x6ACNi7mYxd7OY\nu1nM3SxNzb2MzFwSEbsx0WZrFeBxTNNma3gj7A/bF0iSJEmTRMRymXnvqMcxbOZuFnM3i7mbxdzN\nspBzL9Q2WwCLRj0ASZIkadgi4lkR8cOI2GOKTT4UERdHxF5TXF9L5jb3JOZeQMxt7knMvYA0NTeU\nNluZeXhm7pKZj8zMTTNzy8zcLTPfWteCLFiUlSRJUsNExJHAl4DHU/qRdTIGbAmcHhHvGdbYBsnc\n5u7A3OauNXObuwNzL5DcTWBRVpIkSY0RETsAh1LevFwMXDHFpmcA3622OzQithvOCAfD3OaeYlNz\nm7u2zG3uKTY19wLI3RQWZSVJktQkB1X/fiAzt8rM73XaKDPPycydgfdT3uC8ZlgDHBBzm3sZ5jb3\nsAY4IOY29zLMvWByN4JFWUmSJDXJk4A7gEN63P6dwJ3VfnVm7t6Yu97M3Rtz15u5e2NuzXsWZSVJ\nktQkawDXZOadvWycmXcAV1f71Zm5e2Buc9eUuXtgbnPXVFNzN4JFWUmSJDXJjcCqvW4cEWPAWsBN\nAxvRcJi7B+Y2d02ZuwfmNndNNTV3I1iUlSRJUpNcAqw7gwUwXgisCVw6uCENhbl7Y+56M3dvzF1v\n5u6NuTXvWZSVJElSk5xGWQDjCxGxZ0Qs7rRRRKwaEYcAJwPjwGeGOMZBMLe5l2Fucw9xjINgbnMv\nw9wLJncjjI2Pj496DJIkSdJQRMQi4DzgyZQ3LbcBlwPXA/cCqwFrA5sCiylvhM7LzJ1GMuA+Mbe5\nMbe5zW3uGjN3s3I3hUVZSZIkNUpErAx8HNir7cftB8VjbZc/DxyQmbcPY2yDZG5zY24wN5jb3DVl\n7mblbgKLspIkSWqkiNgS2BPYGtgAWAG4HfgD8DPgjMxccD3ZzG1uzG3uBcbc5sbcCzb3QmZRVpIk\nSZIkSZKGyIW+JEmSJEmSJGmILMpKkiRJkiRJ0hBZlJUkSZIkSZKkIbIoK0mSJEmSJElDZFFWkiRJ\nkiRJkobIoqwkSZIkSZIkDZFFWUmSJEmSJEkaIouykiRJkiRJkjREFmUlSZIkSZIkaYgsykqSJEmS\nJEnSEFmUlSRJkiRJkqQhsigrSZIkSZIkSUO03KgHIEmSpOGIiE2AK6pvD8/MI/t8+/sDJ1ffbpOZ\nP+jn7ddZRDwFOKf6dp/MPH2U45EkSdJoWZSVJEmagYh4MXDqHG/m6szcqB/jmYXx6quutz90EbEC\n8CxgV+BxwDrAqsDtwB+BXwDfBL6YmXdNc3ML6rGRJEnS7Ni+QJIkaebG5/i1ZPhDhsz8HXA/4P7A\n+wZwF6e2bn+hzJKNiJcCvwVOB/YBHgasASymZH0Y8HzgM8DVEbHPiIYqSZKkGnGmrCRJ0sx8BvjC\nFNdtCfyAUng9DThwiu1GUpQFyMzbB3jbSyizR2svIhYDHwJeRnk+7wI+DXwZSOAmYDVgK2BfYA/K\nDNpPR8QWmXnIKMYtSZKkerAoK0mSNAPdCo8RcUfbt/cOsgCqgTuSiYLsL4HnZuZvJ23zd+Bq4MsR\nsQvwOUqh9o0RcXlmzrXNhSRJkhYo2xdIkiRJbSLi34A3UgqyVwE7dCjILiUzz6K0MWh5X0SsPLBB\nSpIkqdacKStJkjRCEfEU4Jzq22cB3weOqS6vCuyemd+atM/2wEuBJwLrA8sDNwOXAP8NfDIz7+1w\nX5sAV1TfHp6ZR7ZddwDwserbLTLzVxHxNODVwBMofWhvBn4CnJiZ3+lw+/sDJ1ffbtPeVzYiPgu8\nELghM9eOiOWqDPsDD68yXAecDxyVmb/u8pg9Fng9sB2wNvDX6nE7OjN/XvWB/Xi1+faZ+b2pbmsK\nbwbGKEXZAzPzxl52ysyzI+JLwEqUHrTLPAeTcuwIvJbSAmFtyszbbo/vbF4rGwMHATsCDwJWBm4E\nfgV8HfhYZrbP8G7ttynwm+rb12TmB6vxvgl4VHV/11LaORyVmX+v9tscOATYBtiwyvR/wBGZ+Ysu\nj8XylJnJzwS2oPTtva26j/OAD2dmdtl/TeBVwC5AUF6v91AWYrsIODUzz+uy/yJgb+DfgccCawF3\nA38GLgA+npk/nmLf1mv7xsxcKyJWpRT1nwM8GPgH5Xfz6Mw8p9pnMXAAsB+wKbAi8DtK25NjO/3+\ntt3fRpTfyx2BjYD7UH4HfgqclplfnmpfSZI0f1iUlSRJmh/GKYXAL1CKjTCp92xEjFEKp/u37dOy\nFvCU6uuAiHhqZt7a5b6m+tk4sDgi3k0pTrZvuybwNOBpEfHKzPzILG5/cUSsCHwN2GnSthtQ+rM+\nJyJ2yMyfTL6RiNgXOIVyHNvad0NgL+C5EfESYJW2++xaGO1w+xtSMo4Dl7aKaL3KzOf1eD+HUFok\ntGt/fF+VmR+eYvdpXyvVfbwKOI5StGt/nNepvnaktFrYLTMv6XAfrX8XVQX3j03aZlNKAXa7iNiW\n8tr7MqXA2LIW8Gxg54h4UmZe2mGcAXyLUmBsH+eq1dcjgVdFxJsz8+gO+z++2n+NSfsvBjapxrl3\nRHwkM1/ZYf91gW9SirHt+y8PPBTYDHhZRJyYma+bvH/bPosiYjXgfyiF5ZYVKa/1HSPiucC3KY/T\n0yfd3yOB91b/7tvhflrP6fEs/foHeCCl6P7siDib0m7jtk63IUmS5gfbF0iSJM0fOwFPoswCfQBl\nFuwFbde/jlKQbfU5fRalELM+sANwVrXd1sAH5zCO51MKsh8EHkMpdm1S/WxJdf/HVgWo2TgWeDzw\nn5SC2eqU2ZenV9evBHxg8k5V8e5kSrHtbuBtlKLZ2pQZlldQCrabzXJcUIqcY9Xlz8/hdrrZnVJ8\n+wTluVqD8jgcDvyT8vgeXc3+nErX10pE7EV5DJcD/gS8HNi4uq8tq/u/l1IIP2ea+3oo5bVwEmVW\n8xrVff+8uv4JwCspz9+FlFmya1aZTqi2WQl49+Qbrl5D51EKskuAE4HHVfs/iFKcvJLynB8VES+e\ntP9ywBmU19DtlBmqj6AUgx9I+R35YbX5KyJivw77n8VEQfb0avxrUR7XZwMXV5u/OiLe3uVxgjJz\n+f7VfutUGQ5lopB+JPBWyuvs9cBDqvvaldIqA+CFEfGYDo/V3pTndDHw++qx2Yzy+t8G+GJ1P0+t\nHhNJkjSPOVNWkiRp/ngxcEhmnjjF9W+gFF3uAHbJzD+3XffXiLgQuIxSfHthRLwuM2+axTgOBg7O\nzGPafvZ3Sp/UhwCvAFagzOqcaeFyNUphebvMvLDt57+MiBdRimObA4+PiPUy8y9t2xxCmb04Dvxn\nZp7Udt03IuL/KKeqHzjDMbV7QtvlH83hdrrZC3hXZrYX+P4OvDci1qe0G1iRcir+6R32hy6vlaoV\nwAmUx+kflFYSV026r8Mj4i+UIuhawDur++3kP4CTMvO1bT+7qHq+fll9/1+Ux+tpbafe3wy8ISK2\noRRad46IxZn5z7bbeQeloDxe5Tm27bqbgdMj4oLqfu4PHBMRn8vMu6tttqEUPseBN096PG4C/hQR\n36G0D9isytK+ANtBlCL1OPChzHz1pOxnRsT5lMLsRsBhEXFyZv6pw+O0GqX9wRaZ+Ye2nx8dEVtX\n120OHAY8OzO/3rbN2RFxEPCN6vvdmCh6ExGrUAqy45TF5bae1Fbjh8API+KjlDYQT4uIZ2bm1zqM\nU5IkzQPOlJUkSZofxiizJDuesl4VZb5NOc361EkFWQCqQtWXqm8XUXqVzsbvJxVk253ZdvkRs7jt\nceAzkwqyAGTmOBNFqaVuv2rd8Izq2xuAj3bY/2bK7Nu5HOOu13b5yjncTjd/AY6Y4rr2/A+eYpuu\nrxVgD8osTYAPTirItjuJ0nN1DHh+1ee0033dQymeLiUzL6Nkac0sftcUvVC/W/27POUDAwAi4j6U\n2Z7j1TiO7zTIzPwj8KHqftagzDRu2bDt8mVT7H8PZSb1Y4GdJ119QPXvXcBbptj/VuCo6tv7sPSC\nbu3GKT16/9Dhuu+2bfPzSQXZlnOZaEmw+aTrXkIp+gIc2qXP8aGUD21gos2JJEmahyzKSpIkzQ/j\nwPfbZgAuJTP/kZkvz8w9Oszma3d12+Vup6R389Uu17UXnFaf5e1/ZRa3vwklzzhwQZeFkL5FWfRo\nttrv8+Y53E43X580W7TdH9surzvFNl1fK5RWFi1TPpdVEby1oNjqlF6mne7rp11mXLfGu4TShqDb\nNlB6xLY8monH++uZuUxf3DbtC5ht23a5/bl+ZTVLeBmZ+ZvM/EVVuAcgItamtGMYB87PzFtmcf+T\nnT3Fz/843TaZeRdlETZY+nGC0q4CSsuJM5lC9TxdQClg71h9mCFJkuYh2xdIkiTNH9f24Tbuarvc\naeZjL67oct0dbZc7FsAGdPsbt13+zVQ7Z+aSiLiIiVm1M9VeGJzt4zedbvnvbLu8Upftur1W2heZ\n+uWUWxXts0sfAfyiwzZXddm/Nd7rMvPOabaBpZ/TuYyz5X8ohfxW/9hLIuIUSpH38mlus+f7z8w/\nRsStlEXkus0Qv2qKn7c/Br/vsn9ru8m/W60es3+jLJa3cpfbuJzSWmRFSsuFQc34liRJc2BRVpIk\naf7423QbRMQGlEWbtqecur0O0K1AMxv/6PPt9eP212i7fMM02149zfXdtJ8WvkYP9zUbd0y/ybS6\nvVbWat1PZk53X+351phim9t7GE8v20y2Vtvl67ttmJk3RUTr1P412n5+d0Q8k9LWYz1K39j3Ufof\n/5EyK/VLwFkdZif3fP+VG4D7MfXjBIN7rFqzxNcHbp3BfuthUVaSpHnJ9gWSJEnzR9diTUS8gDJL\n9K2UBY4eQplNOT7payFaoe3yXVNuVcylqHxN2+XZ9Mwdlm6vlRWrf6eaudqu15m5g7Bi2+Vextp6\n3pcaZ2ZeDGxKWQjuCiZ+Dx4A7Efp05sRsdsc77+1zbAfp/b7nPy7Pt3X/YY+UkmS1BNnykqSJNVA\nRDwW+BTllPolwGeBMyindd9I6TUJZUGgD45giIPW3j91urYJc5k53L4A2bZ07687X91W/dtL8bC9\nMDnoGdKT3dZ2uZex3rf6d5lxVjOCjwGOiYiHArsCu1D6696X0v7iaxGxZ2Z+eZb333qshv04QRnr\nSsClmfnoEdy/JEnqM4uykiRJ9fAayrHbOPCmzOy4Un3bKd4LTfsp26tNuVXxwDncz/coBeD7AC+M\niIO7LMq1jKrX58uBj/TQOmBQWqfi3zciVsnMbkXE9lP4B9GqoZv2lgHrdNswItakLF41zjTjzMwr\nKDNmT4yI+wFvAN5W7X98RHylWuSs5/uvtFoIDPtxgrKg2UbA2iO4b0mSNAC2L5AkSaqH1kI/S4AP\nddluyyGMZRTa+8RuNNVG1WrzT5ztnWTmDcAXKQW8dYD/mOFNvAs4FrgyIrae7Tjm6OK2y4+aZttH\ntl3utMjXIA18nJl5a2a+EziZ8pxuCGze4Xa63n9EPJiyyNeM7r+PWo/VehFhYVaSpAXAoqwkSVI9\ntE7JvzMzO/ZUjYi1gBe0/Whs4KManmSip+d2XbbblbK40Vy8lzJbdgx4T0T01Fs2IvYAXl19exfw\nyzmOY7a+23b52VNtFBHLAU+vvv1TZl4+0FEt65fAdZTHefeIWNxl22e1XT6vdSEito2I10XEztPc\n16Vtl5cHyMybgZ9W979tRHRbwKvj/Q/R2W2X9+62YUQ8PyJ2j4gVu20nSZJGy6KsJElSPVxb/btS\npyJhddr8GSzd73LBzKirCtHnUc12rBY9W0pErA4cz9L9Z2dzX78CDq++vR9wfkTs2G2fiNiT8vgv\nru5/nxG2L/g2ZWbxGPDyiJhqZvHBlNfIOPCxIY3tX6q2EB+vvl0H+M9O20XEpsDLqm+vzMz2AuVJ\nwHFMtCqYyg7Vv3cDv237+Ueqf+8DvHuK+1+L8lgB3AKc3uV+BuVzwN8pz+lbpnpOI2Ir4BTgTODD\nwxueJEmaKXvKSpIk1cOZwJOry5+LiNcClwCrAk8FDqMsBLQnEzP59o2IrwE3ZubfhzzeQTiWMrNz\nDPhYVSz7GmURpH8DjqL0/TwVeMVc7igzj4mI+wNvqW7zu9VjeRrlVPK/UR77rYEDgJ0pxc07gL0z\n8//mcv9zkZlLIuKVwDcpp9z/b0QcTplB+w9K+4f9gFe1dgGOHsVYKc/ZCygLcb03ItajPH/XAKsD\nOwHvpLy2l7Ds8/o+4DPAZsCFEXEM8H1Kv9iVgIcCBwLPoTw/p07qsXsq8FLg/wEHRsQqwInA76r9\ntwXeAaxf7f/6aXr0DkRm3lL9zn+S8nq8MCLeBpxF6be8ASXj6ykLkt1KedwkSdI85UxZSZKkejgJ\n+El1+ZHAuZTC028pM+KWB55GWajqd9V2j6kunzPUkQ5IZp5PKeKNUwpmJwBXUR6Hr1IKZ8+kFEz7\ncX9vA/4d+H11n3tQZsP+Brix+vkZlKL4OPAD4AmZ+dV+3P9cZOa3KYXXuyiPyycohc4bKafsH1Rt\nejGwU2be2el2hjDO24CnAK3WCa+j9Gy9CbiSMoN3PeB24PmZed6k/U8D3kMp2D6MMkv0cspiXH+g\nfECxJ+X5+Tpl0a/2/ZdQntcfVNvsA/yobf/TKYXde4A3ZOYn+5W9i45tRzLz05T2GHdTFmj7MOUx\nup7yPL6NUoS/DnhGZv5+CGOVJEmzZFFWkiSpv8Yn/TuTfaZUnb6/PaUA9WtKf9U7KQWoI4AtM/Nn\nVZHpecBFlELW34D2WZvd7mu8x3F3y9jtNmZy+x23y8w3U2YEfpdSOLuHUmz8ILBFZn5/irHOSmZ+\nhbIw1POATwO/ojym91AKhz+jLLy2XWZum5nd+sj26/HtSWZ+CghKS4dLKKe/3wX8iTKL9iXAVpn5\np1mMY/J2s94mM6+mLFD3Ssrzeh2l8HgTpYB8JLBJZn5xiv3fCjye0oqglfNeygzq3wCfBZ6emc/q\nVHzOzBsycxtKQfbrlMfnrup2LgX+C3h4Zp4wm3wz3Kbrdpl5EqX43HpOb6a8Fv9G+UDmYCAy83s9\n3I8kSRqhsfHxOR2nSpIkSfNKRBxHmXE5Djw6My+dZhdJkiRpqJwpK0mSpIXmQW2X/zqyUUiSJElT\ncKasJEmSaiEingLsSump+4rqtPfJ2yymnH6+NvDnzNxguKOUJEmSpudMWUmSJNXFFpSFmp4KvHGK\nbQ6iFGTHgS8MaVySJEnSjDhTVpIkSbUQEfenLHK2fvWjkymLO10FrENZqOlQYDFlAaQtM/Pa4Y9U\nkiRJ6s6irCRJkmojIrYCvkmZDTvWYZNxSkH2OZl5wTDHJkmSJPXKoqwkSZJqJSJWA14N7A5sBqwM\n3AJcAXwHOCkz/za6EUqSJEndWZSVJEmSJEmSpCFyoS9JkiRJkiRJGiKLspIkSZIkSZI0RBZlJUmS\nJEmSJGmILMpKkiRJkiRJ0hBZlJUkSZIkSZKkIbIoK0mSJEmSJElDZFFWkiRJkiRJkobIoqwkSZIk\nSZIkDZFFWUmSJEmSJEkaIouykiRJkiRJkjREFmUlSZIkSZIkaYgsykqSJEmSJEnSEFmUlSRJkiRJ\nkqQhsigrSZIkSZIkSUNkUVaSJEmSJEmShsiirCRJkiRJkiQNkUVZSZIkSZIkSRoii7L/l/LtAAAA\nGklEQVSSJEmSJEmSNEQWZSVJkiRJkiRpiP4/eKN854Fq8rAAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<matplotlib.figure.Figure at 0x7fe619bcd978>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "### Show AUCs as a training X test X AUC heatmap\n",
    "chromosomes = ['chr1', 'chr2', 'chr3', 'chr4', 'chr5', 'chr6', 'chr7', 'chr8', 'chr9', 'chr10',\n",
    "               'chr11', 'chr12', 'chr13', 'chr14', 'chr15', 'chr16', 'chr17', 'chr18', 'chr19', 'chrX'\n",
    "              ]\n",
    "\n",
    "# sns.heatmap(matrix,\n",
    "#            xticklabels = chromosomes,\n",
    "#            yticklabels = chromosomes,\n",
    "#            cmap='Blues',\n",
    "# #            annot=True,\n",
    "# #            fmt='g',\n",
    "#            vmin=0,\n",
    "#            vmax=1.0\n",
    "#            )\n",
    "# plt.xlabel('Training Chromosome')        \n",
    "# plt.ylabel('Test Chromosome')\n",
    "# plt.title('ROC AUC')\n",
    "\n",
    "data = pd.DataFrame({'Training Chromosome': chromosomes, \n",
    "                         'mean prAUC':pr_matrix.sum(axis=1)/(len(chromosomes)-1)})\n",
    "\n",
    "sns.barplot(data=data, x='Training Chromosome', y='mean prAUC', palette = 'coolwarm')\n",
    "plt.ylabel('Mean prAUC')\n",
    "# plt.gca().set_xticklabels(chromosomes)\n",
    "plt.xticks(rotation=90)\n",
    "plt.ylim(0,1)\n",
    "# plt.xlabel('Training Chromosome')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "### Create bedGraph Track\n",
    "\n",
    "# average probabilities across all training/test sets\n",
    "chromosomes = sorted(os.listdir(peakDirectory))\n",
    "\n",
    "chromosome_meanProbas_dict = {} # {chromosome:[meanProbas]}\n",
    "\n",
    "for test_chromosome in chromosomes:\n",
    "    probas = []\n",
    "    for training_chromosome in chromosomes:\n",
    "        if not test_chromosome == training_chromosome:\n",
    "            probas.append(chromosome_probas_dict[training_chromosome][test_chromosome])\n",
    "    mean_probas = np.mean(probas, axis=0)\n",
    "    chromosome_meanProbas_dict[test_chromosome] = mean_probas\n",
    "\n",
    "\n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "! if [ ! -d ./bedgraph ]; then mkdir ./bedgraph; fi\n",
    "! rm ./bedgraph/*\n",
    "scalingFactor = 100\n",
    "bedgraph_file = open('./bedgraph/scores.bedgraph', 'w')\n",
    "bedgraph_file.write('track type=bedGraph name=\"Classifier Scores\" description=\"Probs * 100\"\\n')\n",
    "for chrom in chromosomes:\n",
    "    current_bedgraph_file = open('./bedgraph/'+chrom + '.bedgraph', 'w')\n",
    "    current_bedgraph_file.write('track type=bedGraph name=\"' + chrom +' Scores\" description=\"Probs * 100\"\\n')\n",
    "    print(chrom)\n",
    "    with open('./group_by_chromosome/' + chrom + '/' + chrom + '_tile_peaks.tsv') as f:\n",
    "        data = f.readlines()\n",
    "    \n",
    "    probas = chromosome_meanProbas_dict[chrom]\n",
    "    print(np.sum(probas>0.5))\n",
    "    data = data[1:-1]\n",
    "    for i in range(len(data)):\n",
    "            if probas[i] > 0.01:\n",
    "                tokens = data[i].split('\\t')\n",
    "                chrom = tokens[1]\n",
    "                start = tokens[2]\n",
    "                end = tokens[3]\n",
    "                prob = str(probas[i] * scalingFactor)\n",
    "                bedgraph_file.write('\\t'.join([chrom, start, end, prob]) + '\\n')\n",
    "                current_bedgraph_file.write('\\t'.join([chrom, start, end, prob]) + '\\n')\n",
    "\n",
    "\n",
    "    current_bedgraph_file.close()\n",
    "bedgraph_file.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "!pos2bed.pl ./idr_peak_files/C57Bl6_Thiomac_Veh_peaks.tsv > ./bedgraph/C57Bl6_Thiomac_Veh_peaks.bed\n",
    "!pos2bed.pl ./idr_peak_files/C57Bl6_Thiomac_KLA_peaks.tsv > ./bedgraph/C57Bl6_Thiomac_KLA_peaks.bed"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Show Top Motifs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "### Show Top Motifs\n",
    "\n",
    "# average coeffs across all training/test sets\n",
    "chromosomes = sorted(os.listdir(peakDirectory))\n",
    "\n",
    "chromosome_meanCoeffs_dict = {} # {chromosome:[meanCoeffs]}\n",
    "\n",
    "coeffs = []\n",
    "for training_chromosome in chromosomes:\n",
    "    coeffs.append(chromosome_coeff_dict[test_chromosome])\n",
    "mean_coeffs = np.mean(coeffs, axis=0)\n",
    "chromosome_meanCoeffs_dict[test_chromosome] = mean_coeffs\n",
    "\n",
    "coefficients_frame = pd.DataFrame({'Weight':mean_coeffs})\n",
    "coefficients_frame.index = motifs\n",
    "    \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "coeff"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "threshold = 1.4\n",
    "\n",
    "factor_motifs_dict = {}\n",
    "significant_motifs = set()\n",
    "for factor in coefficients_frame.columns.values:\n",
    "    weights = coefficients_frame[factor]\n",
    "    std = np.std(weights)\n",
    "    mean = np.mean(weights)\n",
    "    positive_hits = list(weights[weights >= mean + threshold*std].index.values)\n",
    "    negative_hits = list(weights[weights <= mean - threshold*std].index.values)\n",
    "    motif_hits = negative_hits + positive_hits\n",
    "    \n",
    "    print(factor, mean-threshold*std, mean+threshold*std)\n",
    "    for m in motif_hits:\n",
    "        significant_motifs.add(m)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "sns.distplot(coefficients_frame['Weight'].values, bins=50)\n",
    "plt.xlabel('Weight Magnitude')\n",
    "plt.ylabel('Frequency (KDE)')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "\n",
    "data = coefficients_frame[coefficients_frame.index.isin(significant_motifs)]\n",
    "\n",
    "# fig, ax = plt.subplots(figsize=(10,10))\n",
    "with sns.axes_style('whitegrid'):\n",
    "    data['Motif'] = data.index.values\n",
    "    data.columns = ['Weight', 'Motif']\n",
    "    data = data.sort('Weight')\n",
    "    sns.factorplot(data = data, x= 'Motif', y='Weight', size=4, kind='bar', palette='coolwarm')\n",
    "    plt.xticks(rotation=90)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Predictions with Top Motifs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "motif_features = sorted(significant_motifs)\n",
    "\n",
    "significant_chromosome_coeff_dict = {} # store coefficients for each chromosome {trainingChromosome:{testChromosome:[coefficients]}}\n",
    "significant_chromosome_probas_dict = {} # probabilities for each locus {trainingChromosome:{testChromosome:[probas]}}\n",
    "significant_chromosome_rocAUC_dict = {} # ROC AUC for each chromosome {trainingChromosome:{testChromosome:auc}}\n",
    "significant_chromosome_prAUC_dict = {}\n",
    "peakDirectory = './group_by_chromosome/'\n",
    "chromosomes = sorted(os.listdir(peakDirectory))\n",
    "\n",
    "for training_chromosome in chromosomes:\n",
    "    training_features = pd.read_pickle('./pickles/' + training_chromosome + '_normed_motif_frame.pickle')\n",
    "    training_features = training_features[motif_features]\n",
    "    training_labels = [False if 'tile' in x else True for x in training_features.index.values]\n",
    "\n",
    "    #  train classifier\n",
    "    lr_classifier = sklearn.linear_model.LogisticRegression(penalty='l1')\n",
    "    lr_classifier.fit(training_features, training_labels)\n",
    "\n",
    "    # initialize sub dictionaries to store results\n",
    "    significant_chromosome_probas_dict[training_chromosome] = {}\n",
    "    significant_chromosome_rocAUC_dict[training_chromosome] = {}\n",
    "    significant_chromosome_prAUC_dict[training_chromosome] = {}\n",
    "\n",
    "    coefficients = lr_classifier.coef_\n",
    "\n",
    "    significant_chromosome_coeff_dict[training_chromosome] = coefficients.flatten()\n",
    "    for chrom in chromosomes:\n",
    "        print('training on ', training_chromosome, 'testing on ', chrom)\n",
    "        if not chrom == training_chromosome:\n",
    "            test_features = pd.read_pickle('./pickles/' + chrom + '_normed_motif_frame.pickle')\n",
    "            test_features = test_features[motif_features]\n",
    "            test_labels = [False if 'tile' in x else True for x in test_features.index.values]\n",
    "\n",
    "            probas_lr = lr_classifier.predict_proba(test_features)\n",
    "\n",
    "            fpr, tpr, thresholds_roc_lr = sklearn.metrics.roc_curve(test_labels, probas_lr[:, 1])\n",
    "            precision, recall, thresholds = sklearn.metrics.precision_recall_curve(test_labels, probas_lr[:, 1])\n",
    "\n",
    "#             roc_auc = sklearn.metrics.roc_auc_score(test_labels, probas_lr[:, 1])\n",
    "#             pr_auc = sklearn.metrics.precision_score(test_labels, probas_lr[:, 1] > 0.5)\n",
    "            \n",
    "            roc_auc = auc(fpr, tpr) # calculate area under the curve\n",
    "            pr_auc = auc(recall, precision)\n",
    "            \n",
    "            \n",
    "            significant_chromosome_probas_dict[training_chromosome][chrom] = probas_lr[:,1]\n",
    "            significant_chromosome_rocAUC_dict[training_chromosome][chrom] = roc_auc\n",
    "            significant_chromosome_prAUC_dict[training_chromosome][chrom] = pr_auc\n",
    "\n",
    "            print(training_chromosome, chrom, roc_auc, pr_auc)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## comparison between different cell types"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "chromosome_coeff_dict_thiomac = pickle.load(open('/home/jenhan/analysis/atac_analysis/chromosome_coeff_dict.pickle', 'rb'))\n",
    "chromosome_coeff_dict_kupffer = pickle.load(open('/home/jenhan/analysis/atac_analysis_kupffer/chromosome_coeff_dict.pickle', 'rb'))\n",
    "chromosome_coeff_dict_microgila = pickle.load(open('/home/jenhan/analysis/atac_analysis_microglia/chromosome_coeff_dict.pickle', 'rb'))\n",
    "chromosome_coeff_dict_eyeCone = pickle.load(open('/home/jenhan/analysis/atac_analysis_eyeCone/chromosome_coeff_dict.pickle', 'rb'))\n",
    "chromosome_coeff_dict_eyeRod = pickle.load(open('/home/jenhan/analysis/atac_analysis_eyeRod/chromosome_coeff_dict.pickle', 'rb'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "chromosomes = sorted(os.listdir('./group_by_chromosome/'))\n",
    "\n",
    "cellType_coefficients_dict = {}\n",
    "\n",
    "coeffs = []\n",
    "for training_chromosome in chromosomes:\n",
    "    coeffs.append(chromosome_coeff_dict_thiomac[training_chromosome])\n",
    "mean_coeffs = np.mean(coeffs, axis=0)\n",
    "cellType_coefficients_dict['Thiomac'] = mean_coeffs\n",
    "\n",
    "coeffs = []\n",
    "for training_chromosome in chromosomes:\n",
    "    coeffs.append(chromosome_coeff_dict_kupffer[training_chromosome])\n",
    "mean_coeffs = np.mean(coeffs, axis=0)\n",
    "cellType_coefficients_dict['Kupffer'] = mean_coeffs\n",
    "\n",
    "coeffs = []\n",
    "for training_chromosome in chromosomes:\n",
    "    coeffs.append(chromosome_coeff_dict_microgila[training_chromosome])\n",
    "mean_coeffs = np.mean(coeffs, axis=0)\n",
    "cellType_coefficients_dict['Microglia'] = mean_coeffs\n",
    "\n",
    "coeffs = []\n",
    "for training_chromosome in chromosomes:\n",
    "    coeffs.append(chromosome_coeff_dict_eyeCone[training_chromosome])\n",
    "mean_coeffs = np.mean(coeffs, axis=0)\n",
    "cellType_coefficients_dict['Cone'] = mean_coeffs\n",
    "\n",
    "coeffs = []\n",
    "for training_chromosome in chromosomes:\n",
    "    coeffs.append(chromosome_coeff_dict_eyeRod[training_chromosome])\n",
    "mean_coeffs = np.mean(coeffs, axis=0)\n",
    "cellType_coefficients_dict['Rod'] = mean_coeffs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "cellType_coefficients_frame = pd.DataFrame(cellType_coefficients_dict, index=motifs)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Cone -1.11382277386 1.60059769775\n",
      "Kupffer -0.951350961224 1.84801767191\n",
      "Microglia -1.09583060536 1.68349297934\n",
      "Rod -1.15935358362 1.53473133587\n",
      "Thiomac -1.03823927763 1.97348372727\n"
     ]
    }
   ],
   "source": [
    "threshold = 2.0\n",
    "factor_motifs_positive_dict = {}\n",
    "factor_motifs_negative_dict = {}\n",
    "factor_motifs_dict = {}\n",
    "significant_motifs = set()\n",
    "for factor in cellType_coefficients_frame.columns.values:\n",
    "    weights = cellType_coefficients_frame[factor]\n",
    "    std = np.std(weights)\n",
    "    mean = np.mean(weights)\n",
    "    positive_hits = list(weights[weights >= mean + threshold*std].index.values)\n",
    "    negative_hits = list(weights[weights <= mean - threshold*std].index.values)\n",
    "    motif_hits = negative_hits + positive_hits\n",
    "    \n",
    "    print(factor, mean-threshold*std, mean+threshold*std)\n",
    "    for m in motif_hits:\n",
    "        significant_motifs.add(m)\n",
    "    factor_motifs_dict[factor] = motif_hits\n",
    "    factor_motifs_negative_dict[ factor] = negative_hits\n",
    "    factor_motifs_positive_dict[factor] = positive_hits\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAA9kAAAMWCAYAAADlCkWLAAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAAewgAAHsIBbtB1PgAAIABJREFUeJzs3XmcHFW1wPHfEMIe9h1kUZLjuAAGQQRxBUQRBVxQiQq4\ngchTXHB7AvrccFdEUZ+IEH24sAgq+lRA9LmAIIgYTkYhILIoe1hCSDLvj1tNmqF7pidTPT2T/L6f\nT3+qu+vWqVuTaZjT99a5fYODg0iSJEmSpLFbqdcdkCRJkiRpeWGSLUmSJElSTUyyJUmSJEmqiUm2\nJEmSJEk1McmWJEmSJKkmJtmSJEmSJNXEJFuSJEmSpJqYZEuSJEmSVBOTbEmSJEmSamKSLUmSJElS\nTUyyJUmSJEmqiUm2JEmSJEk1McmWJEmSJKkmJtmSJEmSJNXEJFuSJEmSpJqYZEuSJEmSVBOTbEmS\nJEmSamKSLUmSJElSTVauO+DhfdsM1h1zqJMH5/V1+xxaJl3/t5ckSbXwbylJ6pLak+wp/idbkiRJ\nkrSCqj3JXn2KM9AlSZIkSSum2pPsVVaa/EPZETEVOB44BpgCHJ+ZHx5jzCnAG4FZwOOB1YGbgIuB\nkzPz0rHElyRJkiT1Xu1J9mqTPMmOiB2A04AnU+4xHvN9xhGxHvAz4KlVvBuBO4BtgEOBQyLiqMz8\n8ljPJUmSJEnqHUeyKxGxEvAe4DhgKvA94BXUU8zrHEqCfTlwaGZeVZ1zE+ArwP7A5yLiwsycU8P5\ntIIbGBhg/vz5ve6GtEKZNm0a06dP73U3JElSj3lP9lL7Ah8F5gNvyszTIuIVYw0aEQcDe1Cmhu+V\nmXc29mXmrRHxakoSfguwCWCSrTEZGBhgxowZve6GtEKaO3euibYkSSu4LiTZk3Mkm7Jm+O+BV2fm\nvBrjHkUZDf9kc4LdkJkLgH1qPJ9WcI0R7NmzZ9Pf39/j3kgrhjlz5jBr1ixnkEiSJKeLN/kNcF5m\nLqkrYERsDuxcvfx+XXGlTvT39zNz5sxed0OSJElaoTiSXcnM27sQ9qlAH3BrZt4cEVsChwG7AusD\ntwK/BE7JzHu7cH5JkiRJ0jhyJLu7nlhtb4yIFwOnA2sNabMf8N6IeInLeEmSJEnS5FZ/kj11St0h\nJ7MNqu1GwLeBC4ATgD9TKpi/BPgksClwXkTskJm39qKjkiRJkqSxqz3JXnn12kNOZtOq7WOAH2Xm\nS4bsPzUi/gL8jpKIvxt41zj2T5IkSZJUo9rX25oydUrXH5NI8xrbH23VIDP/CJxPuXf7pePRKUmS\nJElSdziS3V3Na7lcOUy7XwMvAraKiLUsgiZJkiRJk1PtGfGUqbUPjk9m/+iw3R1Nz9cCTLIlSZIk\naRJyJLu7rmp6vi1wTZt26zY9v6t73ZEkSZIkdVPtGfHU1Uyym/wfcA+lANorgA+3afe0ajs3MxeM\nR8ckSZIkSfWrv/DZKlO6/pgsMnMh8B1KUbMjI2LtoW0iYlvKWtmDwJnj20NJkiRJUp2cLl6JiHWB\n5gy+r+n5GhGxwZBD7sjMwYjYCriWkiR/KDM/MqTdh4GDgA2Bn0XErMz8e3XOHSlJ+KrArcAXarsg\nSZIkSdK460Lhs8kz0jzEFcBWLd7vA46pHs22AW6o9q9ESbIfNTMgM2+JiH2Bc4FdgLkR8Xdglep8\ng8DtwP6Z+e9arkSSJEmS1BP1J9mrTa075HhZwiPXtR7O0HbDHpeZv4+IJwDvoEwN35qSnF8NnAd8\nNjNvG113JUmSJEkTTRemi69Sd8hxkZmPXcbjrueR08zbtbsNeH/1kCRJkiQth2pPsleaOmlHsiVJ\nkiRJGpP6R7JXm5wj2ZIkSZIkjVUXRrInZ3XxhojYFfgPYDdgE2ABkMDZwImZef8yxJwCHEypMj4T\nWL+KewNwIfCVzJxTywVIkiRJknrGe7KbRMT7gY9QCpk9CFwHTAN2plQGPzQinpuZN40i5vrA+VWM\nQeBuYC6wOtAPPBF4c0S8LTNPrvFyJEmSJEnjrAvVxSdnkh0RL2Zpgv054LjMvK/atzNlPesZwPeB\n3UcR+luUBPtB4MjMPKXpnJsCXwf2Bb4UEZdl5qU1XI4kSZIkqQceta7zWE2ZOrXrjy75NCXBPjcz\n39VIsAGqxPfl1f5dI+LlnQSMiG0oCfQg8IXmBLuKewvwKuB+ypJeR9RwHZIkSZKkHqk/yV5tla4/\n6hYRuwPbVS8/3apNZl4BXFC9PKTT0E3Pf90m7r2Ue777hrSXJEmSJE0y9Rc+W2VSFj57TrW9F/jd\nMO1+DjwPeGaHcW9per7aMO2mUka7b+4wriRJkiRpAupCkj1cLjlhbV9tBzJzyTDtrqm2a0REZGaO\nEPdqSvG0bYFXA2cObRARjwEeX708r/MuS5IkSZImmtqT7L5VJ2WSvSVlJPnGEdo179+SMs27rcxc\nFBFvAn4I7B8RJwGfpyTeq1EKqH0GmEIZJT9tmXovSZIkSZoQ6k+yV+5aYbJumlZtR1oDu3n/tLat\nmmTmLyPiGcD7gDfz6OJm1wHvBz6TmYOdxJQkSZIkTUz1J9mTc7r46tV24QjtHmx6vsYo4u9MWRO7\nD7iHMiK+GrANsDllRPt84M+jiClJkiRJmmDqr1K28qRcJ7sxQj1S55u/QRhp1BuAiPgKZQT7dsoy\nYGc3RqwjYkPgOOBI4HkR8YLMvHg0HZckSZIkTRzek13Mr7ZrjtBurabn94wUNCKeR0mwB4HDM/Os\n5v2ZeRtwVERsAewPfBl4UqedliRJkiRNLFYXL64Hnk4pZjacbZqeX9tB3NdU2/mZ+ajK4k3OoiTZ\n/RGxQ2Ze2UFsSZIkSdIEs1LtEVee2v1H/RpJ7YyIGO6Lh8ZSX3dl5rwO4m5WbUda//q2FsdIkiRJ\nkiaZLkwXX33kRhPPz4CPU4qZPRO4oE27fShTv3/SYdw7q+1WI7Rr3n9Hh7ElSZIkSRNM7SPZfStP\n7fqjbpl5BfAnSvXvY1q1iYg9gZ2ql9/oMPSvqu1qEfHiYdodUG3vrfohSZIkSZqEah/JHpwyKauL\nAxwNXAjsFREnAe/NzPnwcAGz0ymj2Gdl5kWNgyJiK8r92YPAhzLzI00xv0VZH3tL4JSIeBNwTmYu\nqY5dHzgeeH51/Kcy86FuXqQkSZIkqXvqX8JrpSm1hxwPmXlxRBwBnAQcDhwWEfOAdYBNKUnwb4BD\nhxzaR5kRMMiQmQGZeX9EvAA4l1I07QfA/Ii4kbJc2LZNx359SIIuSZIkSZpkap8uPrjyKl1/dEtm\nfg14CmUE+mZKYrwKcBHwRuDZmXlvq8seJubVlGW5/gP4JWV97ccBmwB/B04FnpmZh9d0GZIkSZKk\nHnG6+BCZ+RfgsFG0vx4Ydvg+Mx+gjJCfNLbeSZIkSZImMqeLS5IkSZJUk/pHsrs4nVuSJEmSpIms\nCyPZ9YccTxGxK+X+6d0o900vABI4GzgxM++fSHElSZIkSRNH/YXPpqzS9Ue3RMT7gd8CBwEbA9dR\n1q7eGfg4cHlEbD5R4kqSJEmSJpYuJNlTu/7ohoh4MfARSqXwzwEbZeYTMvMxwK6USuAzgO9PhLiS\nJEmSpImn/iR7pSldf3TJpymJ8LmZ+a7MvK+xIzMvBV5e7d81Il4+AeJKkiRJkiaY2pPsRUsGu/6o\nW0TsDmxXvfx0qzaZeQVwQfXykF7GlSRJkiRNTLVXKVvchSR4HDyn2t4L/G6Ydj8Hngc8s8dxJUmS\nJEkTUO0j2Q8t6f6jC7avtgOZOdwZrqm2a0RE9DCuJEmSJGkCqn0kuxvTucfBlpT7om8coV3z/i0p\nS3D1Iq4kSZIkaQJyungxrdqOtFZ18/5pbVt1P64kSZIkaQKqPcl+aHIm2atX24UjtHuw6fkaPYwr\nSZIkSZqA6h/JnpQ59sMjyauM0G61Fsf0Iq4kSZIkaQLynuxifrVdc4R2azU9v6eHcSVJkiRJE5DT\nxYvrgadTio4NZ5um59f2MK4kSZIkaQKqfQmvxUsGu/7ogiur7YyIGO6Lh8aSXHdl5rwexpUkSZIk\nTUC1J9kLFi3p+qMLflZt1wCeOUy7fShLcv2kx3ElSZIkSRNQ7Un2Q4sHu/6oW2ZeAfwJ6AOOadUm\nIvYEdqpefqOXcSVJkiRJE1Pt92Q/uLgrI83j4WjgQmCviDgJeG9mzgeIiOcBp1NGm8/KzIsaB0XE\nVpT7qAeBD2XmR+qIK0mSJEmafGpPsrs0nbvrMvPiiDgCOAk4HDgsIuYB6wCbUhLh3wCHDjm0jzIj\nYJAWMwPGEFeSJEmSNMl0Ybr4kq4/uiUzvwY8BfgWcDOl6vcqwEXAG4FnZ+a9LQ4ddg77GOJKkiRJ\nkiYRp4sPkZl/AQ4bRfvrgSl1x5UkSZIkTT71r5PdhcJkkiRJkiRNBvWPZC9aXHdISZIkSZImhfoL\nn03y6eKSJEmSJC2r2pPshZO0ungdIuIxwLuAPYGtKJXHbwD+F/h0Zt44htjPBt4A7EapSg5wC/B7\n4JTM/MWy91ySJEmSVIfak+wHFq6Y08UjYk/gHGANYAkwD1gMTAceD7w+IvYb7VrYEdEHfJWSYA8C\nC4Hrqt3bAgcBr4yIUzPTwmqSJEmS1EO1L+G1cNGSrj8mmojYAjgTWB34BbBNZm6XmQFsDfwMWBM4\nKyI2GWX4d7A0wT4R2Cgzn5CZTwDWAz5atXtdRLxr7FcjSZIkSVpWjmTX43hgGvBP4IDMvK+xIzNv\nioiXAdcAmwPHAkeOIvZRlAT7ksx8e/OOzFwAHBsRzwL2AN4CfHoM1yFJkiRJGoPak+z7V7AkOyJW\nA15BSYS/0pxgN2TmfRFxMvBflKndb8/MhzqIvSrl3u5B4DfDNP0jJcneOiKmdhJbkjo1MDDA/Pnz\ne92NCW3OnDmP2Kq1adOmMX369F53Q5Kkrqq/8NmKV138qZRR7EHKVPF2fk5JstcFZgJ/GClwZj4Y\nEXcDawOrDdN0arW9wwRbUp0GBgaYMWNGr7sxacyaNavXXZjw5s6da6ItSVqudWG6+KK6Q0502zc9\n/+sw7a5per4jHSTZlfOAWcD+EfGuzHyweWdErAQ8l5Lk/6jDmJLUkcYI9uzZs+nv7+9xbzSZzZkz\nh1mzZjkrQpK03HMJr7HbstrOz8x72zXKzHsi4l5KAbQt27Vr4X2UZbu2Bc6PiPcBV1KS6n7gQ9X2\neuA/R999SRpZf38/M2fO7HU3JEmSJjzvyR67adX2/g7a3k9JsqeN1LAhM/8ZEU8HPgAcAvxuSJN7\ngC8BH8/MWzqNK0mSJEmqn9XFgYg4FXjtKA/7W2bOoCzbBWX96pE0pnqvMcpz9QNPqo57kLIGdx+l\nKNo04GnA7pRlxCRJkiRJPVL7OtmLFy3p+qMLBpfxAUtHsFfp4DyN4mWdjHoDEBEHA78EngUcB2yQ\nmf2Z+XhKEbUjKfeFfz8ijuk0riRJkiSpfrWPZC96aPKNZGfmocChy3h4o4LLmh20Xava3tNJ4IjY\nEDiR8mXIlzPz4837M3MhcHJErAJ8Hjg+Is7IzBs66rkkSZIkqVa1j2QvWTzY9ccEc321XSsi2t5r\nHREbsXRq+bUdxn4RZbQa4JRh2p1VbVcFDugwtiRJkiSpZvWPZE/Ce7LH6Mqm50/i0YXJGpqX+rq8\nw9ibNT2/aZh2t7U5RpIkSZI0jpwuPnaXUZLcDYAX0D7J3qfa3piZf+4w9p1Nz7cBbm3Tbqum53d0\nGFuSJEmSVLPak+zFi1esdbIzc1FEnA4cDbwxIj6TmXc3t6mmih9GKZY23LTvoX7V9Pwg4A9t2h3Y\n9PziUcSXJE0SAwMDzJ8/f+SGE9ScOXMesZ3Mpk2bxvTp03vdDUnSBNWF6eIrVpJd+QjwGmBj4LyI\neG1mzgOIiO2A2cB6lPu3PzX04Ij4G7A1cFFm7tV4PzPnRMQ5wP7Af0TEv4EvZOb91XFTKQXbjqUk\n8Bdm5u+7dpWSpJ4YGBhgxowZve5GLWbNmtXrLtRi7ty5JtqSpJYcya5BZt4ZEfsC51PWq/57RFxH\nWct6m6rZTcALGgnyEFMoReimtNh3CGX96+cCH6VUEJ8HLKYk5qtTEuzfU0a7JUnLmcYI9uzZs+nv\n7+9xb1Zsc+bMYdasWZN6VoEkqbssfFaTzLw0IgJ4N/BCYFtK8nsl8EPgc5k50tJdjyqdXh2zV0Ts\nDxwM7AxsSUngb6MUUTsD+F5mrnjfcEjSCqS/v5+ZM2f2uhuSJGkYFj6rUWbeDry3eozmuG07aHMO\ncM4ydk2SJEmSNA5qT7KXLFpYd0hpXFlcaGKwsJAkSZImo/rvyX7wgbpDSuPG4kITi4WFJEmSNNk4\nki01sbjQxGBhIUmSJE1WXSh85ki2Jj+LC0mSJElaFk4XlyRJkiSpJk4XlyRJkiSpJvWPZC9cUHdI\nSZIkSZImhfqTbEeyJUmSJEkrKO/JliRJkiSpJrUn2QsuPbmv7piSJEmSJE0GK/W6A5IkSZIkLS9M\nsiVJkiRJqolJtiRJkiRJNTHJliRJkiSpJibZkiRJkiTVxCRbkiRJkqSamGRLkiRJklQTk2xJkiRJ\nkmpiki1JkiRJUk1MsiVJkiRJqolJtiRJkiRJNTHJliRJkiSpJibZkiRJkiTVxCRbkiRJkqSamGRL\nkiRJklQTk2xJkiRJkmpiki1JkiRJUk1MsiVJkiRJqolJtiRJkiRJNTHJliRJkiSpJibZkiRJkiTV\nxCRbkiRJkqSamGRLkiRJklQTk2xJkiRJkmpiki1JkiRJUk1MsiVJkiRJqknf4OBgr/sgSZIkSdJy\nwZFsSZIkSZJqsnIXYjo0vuLpAzi8bxv/7SVJmgROHpzX1+s+dFtEbAN8BngWsDZwF7BDZt7cy35J\nWv51I8mWJEmSOhIRTwReBpydmX+uMfRPgMdTBoBuBu6mGhiQpG4yyZYkSVIvHQT8J3AdUEuSHRHT\nWZpgn5SZ/1FHXEnqhPdkS5IkqZd2pv7bDTdpen5ezbElaVgm2ZIkSeqlp3YhZvNszQe7EF+S2urG\nEl4Wv1rxWPhMkqRJpJuFzyJiJWAWcDCwPbA+cAfwW+BLmXlh1e4i4JmUvx2H9mebzLyhKeZ2wFuA\nFwBbVu0HgHOAz2XmPVW7ZwEXDtO9R8SVpG5wJFuSJEm1iIg1gQuAU4E9KQn0tZTq3gcAv4yIj1XN\n5wJ/pSTMg8ANwBXAn4CFTTFfBlwFvB3YBrgRuIeSwB8HXBURj6ua31vF+Fv1erB6/qi4ktQtjmSr\nDo5kS5I0iXRrJDsi/odSyOw24OWZ+avq/dWAE4CjKH8rvjQzz4mIrSkFzwaBQzPztCHxngT8EZhK\nSdzfnpnzq327AGcBmwNXZObMpuMaI9qDwHMy8+JuXK8kteJItiRJksYsIranJNiDwFsbCTZAZi7I\nzLdRRpMB3tNh2A8Dq1BGvN/QSLCrmJcAR1Yvd4iI54/xEiSpFi7hJUmSpDq8otreB5zZps3RwHTg\n3yMFi4g1gBdSkvbZmdlqxtxPgLcBt1Pu0ZaknjPJliRJUh12rrbXZObiVg2qadudTt3egTKKPUi5\nJ7tVvIeAE0fZT0nqKqeLS5IkqQ5bUBLiEUepRxGvoa6YktR14zaSPTAwwPz580duqEln5syZIzeS\nJEnLu9Wq7aKa49UZU5K6blyS7IGBAWbMmDEep1IPdKFCvSRJmnzur7br1hyvzpiS1HXjkmQ3RrBn\nz55Nf3//eJxSkiRJ4+sfwJOAjWqM11BXTEnqunEtfNbf3+/UYkmSpOXT5cALgO0iYp3MvHtog4jY\ni1KFfBB4ywjxrgYWUtbI3gX4XqtGEXEiZWr5+Zl51rJ3X5LqYeEzSZIk1aGRBE8BDm/T5t3A64Fd\nMnMRsKRp3+rNDTPzfuDHQB8wKyKmDQ0WEbtR1so+bEgsSeoZk2xJkiSNWWZeBZxBSYqPi4iDI6IP\nICJWj4hPAHtSRrE/Xh12S/UaYL+IWDUipkTEqtV7xwIPUaaLnx0RmzXOFxFPBU6rXl4NnNe9q5Ok\nzplkS5IkqS5vAC4AVgVOB/4VEXOA24FjKAn1JzLzu/DwOtcXUBLzFwL3AQuA51f7rwZeCTwAPAe4\nISIyIv4JXAJsC8wDDmi3NrckjTeTbEmSJNUiM+/PzD2BQ4FfAouBxwJ3A2cDz83MDww57DDgJ8A9\nlAR7gDLC3Yh5NqWg2knA34HNgXWAPwPHAztm5t9bdGeQpaPkkjRu+rqw/NKjAl5++eXstNNOXHbZ\nZRY+Wz71ARzet43/I5MkaRI4eXBeX6/7IEnLK0eyJUmSJEmqiUm2JEmSJEk1McmWJEmSJKkmJtmS\nJEmSJNXEJFuSJEmSpJqs3OsOaPnxl098s9ddEPCiXbbsdRdUmffcvXrdBUlarkXERcAzgYsy87k9\n7o4kASbZkiRJmrzmAmtT1tZuKyIeB5wK7F69tU1m3tDdrklaUZlkS5IkaVLKzDeN1CYi3gKcAKxR\nvTXY1U5JWuGZZEuSJGm5ExFbAN8E9gTuA34C7NvTTklaIVj4TJIkScujE4HnAZcCM4Ef9LY7klYU\njmRLkiSpNhHxCmAWJbHdEHgQuBm4DDglM3/Z1PY5QOP1k4DbgGOBFwCbAQuAq4CvZOYZLc51Ee0L\nny0GPgEcm5mLI2K3uq5xSB+eBbwB2APYpOrzlcC3gFMz0+np0grGkWxJkiTVIiLOAM6gTMteG5gH\n/AvYBngl8POI+ELTIUuq7WDV5jLgCGAt4HrKfdR7AN+JiE+2OOUg7e+xfmtmfiAzFy/7FQ0vIj4N\nXAi8GlgHuLbqzx7AN4D/jYjVunV+SROTSbYkSZLGLCJeAryCkjgfCayfmY/PzOmUEe0PVU3fWo3+\nDvVZ4EZgemZukpn9wBbAz6r974yI3Vsc19eqP5l567Jfzcgi4s3AOyjX+1Zgg8x8YmauDxxKGUl/\nLqXomqQViEm2JEmS6tCYrn11Zp6cmYsaOzLz3sz8MPB14Bxg3SHH9lES6gMy89qm426njBLfV731\nlm51fjQiYipwPGXU+muZ+ZXMbIzKk5mnAV+qXr4xItYe/15K6hXvyZYkSVIdGiPK60XEKpm5cGiD\nzHxzm2MHgXMz85YWx9wZET8FXsrSRL7XnkW5/3qQcu91KycCCdxOm9F2Scsnk2xJkiTV4ReUadNb\nAr+LiI8A52fmgg6P/8Mw+66iJNkbR8TamXnP2Lo6Zjs3Pb+qVYNqRP7k8emOpInE6eKSJEkas8w8\nF/g8ZXR3R+BM4M6IuCgijo2IJ48Q4sZh9jXfX73R2Hpaiy2q7YLMvL+nPZE04Sx3I9kDAwPMnz+/\n191YocycObPXXZAkSRNAZr4jIv4HeBdlGa41KZW2nwkcHxH/BxyZmX9ucfhwyWrz1PNV6+pvKxHx\n38BOrfZl5lOqp42K4YtatZO0YluukuyBgQFmzJjR626scAYHXf5RkiQVmXkpcFBVHOyZwJ6UhPvJ\nwO7AryPiaZl5zZBDh0uem/c9UGd/W3gcsH2L95v/4Gl8IbBWRKzUXPRMkparJLsxgj179mz6+/t7\n3BtJkqQVV2Y+BPyyerwvIvaiTCFfC/hPYNaQQzYbJtymTc+7ujRXZj6ng2b/aHq+IWUtcEkClrMk\nu6G/v98pzJIkSeMsIqZk5uJW+zLz5xHxLcoa2q1GinemfaGwHartDRPkHujLm57vAvxoaIOI2Ar4\nYPXy85l59Xh0TFLvWfhMkiRJYxIR60bEJcC9EbHrME2nVtuhBXT6gAMjYoMWsTcE9qZM1/55Hf2t\nwUUsHb0+sk2bWcDrgdfS5dF3SRPLcjmSLUmSpPGTmXdFxIOUe6e/HxGvB36emYMAEbEy8ErgdZRk\n+ftDQgwCdwLnRMRrMnNeddymwLeB1YElwEktTv+o4jDV+dYZ8vZaTc/Xi4j7ml4/NJplwTJzcUQc\nSxl53zsiPgUcm5kPVOd/JWUUexA4JTNv6zS2pMnPJFuSJEl1eB1lreytgZ8C8yPiZsoo9eaUSuOD\nwHnAiS2Ofx9wAvD3iLieUlH8sZS/VweBD2TmlS2O62vx3u7AhW362Qf8ach7FwHPbXdhrWTm1yLi\nscC7gXcCR0TEDZT7x9et+vxL4OjRxJU0+TldXJIkSWOWmddS7rU+kpLgLgC2BbYCbgfOBl6Wmfu3\nuW/7ZmAm8FlKgr0lZVr5BcBLMvOENqcepMVodtP7nT5GLTPfCzwH+C5wB+V6AX5NmSr+/MxcsCyx\nJU1ejmRLkiSpFpl5L2UKdbsCZiMdfwdlZPjdHbZvWQk8M38FTFmWPoxWZl4MXDwe55I0OTiSLUmS\nJElSTUyyJUmSJEmqiUm2JEmSJEk1McmWJEmSJKkmJtmSJEnqpWWu7i1JE5HVxVWbS844vdddEHDJ\nGb3ugRr2+IKfiYlgwf0P9boLqixZYh6lR6qrCnhErAT8J/Ba4DGUJcA+kJlfHGtsSRotk2xJkiRN\nCBExAzgKeB4lWZ4K/Au4BPhWZp7X5tBjgOMpI+L3A9dX8bYGrhtlN47PzA+PuvOSVDHJliRJUs9F\nxJuBLwCrUJLlW4AHga2AA4EDI+IM4ODMHDotYr9qeyOwfWbeXcXcDLiiwy48mXIr5cKxXIckmWRL\nkiSppyJiH+ArlOT658BbMvPv1b4NgM8DBwMHAX8CPjUkxCbVsb9pJNgAmXkzMLOD878IOBe4A/jv\nsV6PpBWbhc8kSZLUa/9VbQeA/RsJNkBm3g4cAlwF9AFHtDi+cV/3g6M9cUSsxdIE/z2ZedtoY0hS\nM0eyJUmSVKuIOICSGD8V2BCYT7k3+ofAFzPznqa2awIbAwuAb2fmA0PjZebiiDiLMqV764hYOzPv\niYh5lOnkDYdExCHV807vrT4B2IIyCv6N0VynJLViki1JkqRaRMQqwHeBl1BGhh+kJNebUqZt7wQc\nERF7Z+bVAJl5H7B1B+GblwpYtdr+hTLF+4mUv2vvBG6o9t3SQX93Aw6vYr+5gz5I0ohMsiVJklSX\nL1AS7AWUKuHfysxFABHxDMr9zjOAH0dEf6tR62E8tdrelpn/BsjMF1Wxr6OMaJ+bmYeNImZjia+T\nMnPOKI5OdpMjAAAgAElEQVSTpLa8J1uSJEljFhHbAW+kjGAflZnfaCTYAJn5G2AfSvXuxzCKkeOI\n2AF4URX7yzX199WU0fV7gI/UEVOSwCRbkiRJ9XgV5W/LO4FvtmqQmfOAH1MKmB3YSdCI2Aj4HmUG\n5vXAJ8fa0YjoA46jJO2fz8w7xhpTkhpMsiVJklSH3artNZm5ZJh2f6i2O44UMCK2AS4GpgN3AfuO\ncop5OwdWMR8AvlRDPEl6mPdkS5IkqQ6bVtvdImK4JLthzYhYNzPvarUzInamrF29CXAbJcGu677p\nt1FGsc+olgiTpNqYZE9iAwMDzJ8/v9fdYObMmb3ugiRJ6r01KYnrPZSK4iMZBFZptSMiXg6cCqwG\n/A14YWb+rY5OVveOP6M6/+l1xJSkZibZk9TAwAAzZszodTcAGBwc7HUXJElS791bba/MzGcva5CI\neDfwierlBcArMvPOMfat2cHV9t+Z+asa40oSYJI9aTVGsGfPnk1/f3+PeyNJksSNlPust1zWABHx\nNuAEyijz14G3jHB/97JoVCn/Rc1xJQkwyZ70+vv7na4tSZImgksoCezWEbFxZv6rVaOImJqZD7V4\n/xDgs5QE+PjM/K+6OxgRG1CW7QJwFFtSV1hdXJIkSXX4H0qCvBJwzDDtzo+Iv0bErMYbEfFElq5/\n/aluJNiVHSnLhwFc3aVzSFrBmWRLkiRpzDLz78DXKEnsOyLigxGxemN/RGwZEd8GnktZPiubDv8i\nsCpwSWa+t4vdfHJzl7t4HkkrMKeLS5IkqS5HU5bcegnwIeC9EXE9MA3YjDLA8xBwVGZeChAR2wPP\nqY7fKiL+1MF5PpiZPxryXl/Llo+0adPzOzpoL0mjZpItSZKkWmTmAuDAiHgx8DpgF2BbYBEwl1It\n/EuZeU3TYetRpplDSYKbE+F21m/x3mBTnHbWqdrcn5kujyKpK0yyJUmSVKvMPBc4t8O2vwKmjPF8\n23bY7gjgiLGcS5JG4j3ZkiRJkiTVxCRbkiRJkqSamGRLkiRJklQTk2xJkiRJkmpiki1JkiRJUk2s\nLq7anHrKh3vdBQGLXZBkwvjgl/+v110QsGTRwl53QVKXRMQ8YCvg1Mw8rLe9kaTCJFuSJEmTVSdr\nY3ckIjYCvg68uHrr2Zl5cR2xJa1YnC4uSZKkFVpEvAz4K7AfNSbuklZMjmRLkiRphRQR6wInAa8C\nHgLOAQ7AJFvSGDiSLUmSpBXVBykJ9gCwO/DF3nZH0vLAkWxJkiSNWVMRsk484n7niJgCvIaS8O4I\nrAvcDSTwA+ArmTliFcOIeD1wGBDAGsANlNHpj2XmPS0O6QNOAf4jM++PiGd12H9JasskW5IkSXX4\nC3DHMPs3BzamTMV+qPFmRKwHnEsZSR4E7gOuBR4D7Fa9f3hEPDczb24XPCK+BrwBuAu4CdgUmA4c\nA7wkInbPzKH9OyEzbx3NRUrSSEyyJUmSNGaZ+aJ2+yJic+ByShJ9Vmb+rmn3dyiJ9B3AGzPz7Kbj\n9gNOpoxMnwM8rc0pngdsCLwa+G5mDkZEH/Am4MvADOAT1evmPptgS6qdSXZlYGCA+fPn97obHZsz\nZ06vuyBJkjSiiFiZMuV7Y2AOcGjTvucCz6ck3wdl5i+bj83M8yLiduA3wFMjYv/MPGfIKfqALYF3\nZOYZTccOAl+NiN0oU9FnRcTbM/P+2i9SkpqYZFMS7BkzZvS6G5IkScujLwC7AvOBAzPz3qZ9B1fb\nq4Ym2A2Z+duI+BPlXu0DKSPaQz1Eube6lTMoSfaqwB7Az0Z9BZI0CibZ8PAI9uzZs+nv7+9xbzoz\nZ84cZs2a1etuSJIktRURs4AjKCPVh2VmDmmyW7XvLyOE+gPwFGBmi32DwF8zs92UxObYj8ckW1KX\nmWQ36e/vZ+bMVv/tXrGNNJXen5kkSRoqIran3E89CHwmM89s0WzTavvqiHh1B2G3bPP+P4Y55l9N\nzzfo4BySNCYm2RpWJ1PpBwcHx6k3kiRpMoiItYEzgdWBi4D3tmm6ZrX9N/DPDkIvbvP+A8Mc07z0\n12odnEOSxsQkW8OajFPpJUlSz50OPI4ywvzKzFzSpt29wDrA+Zl5yBjOt+ow+5oTa4ueSeo6k2x1\nxKn0kiSpExHxfmA/ygjyyzPz38M0v5GSZLebBt6pzYbZt2nT8+H6Ikm1WKnXHZAkSdLyISL2BD5M\nuQ/7bZl5yQiHXEJZgmtmtdRXu7hTh4nRBzwxItqNZu/Y9PzqEfojSWNmki1JkqQxi4jHAN+hJL2n\nZeZXOzjsO9V2HeDwNnGnAldGxGUR8fw2cdYAXttmX6Og2nzg/zrokySNidPFJUmSNCZVIvwDSvXu\nK2iTMA+VmRdExE+BfYDPRMRC4JTMXFTF7aess/14yv3UrUaiB4G/AZ+OiH9l5g+rY1cC3g68tGrz\n9cx8qKnPfcD6Q2Kt2/R8nYhorka+ODPv6uS6JK3YTLIlSZI0Vi8Hdq6ebwj8PiKGa39TZu5bPZ8F\nnAs8nbLk16ci4kZKArwJJUG+D3hVZt7YJt7PgCnA2RFxB3ALsDklaR4ELgc+NOSYrYDr2sTrA344\n5L3rgW2HuyhJApNsSZIkjd1USjILpYjZSIXM1m48ycw7ImIP4DXAK4GnANsBC4A/UxLoE9sk2I1b\nHx/IzPdExB+B1wP9lIrj1wDfBT6Zma2W+RrNOqTtKqRL0iOYZEuSJGlMMvNbwLfGcPwgcFr1GM1x\nWw95/U3gmx0eez1l9FuSamXhM0mSJEmSamKSLUmSJElSTUyyJUmSJEmqiUm2JEmSJEk1McmWJEmS\nJKkmVheXljMbrjG1111QZdXVV+11FwRs/rjNet0FVQYuHeh1FzQOIuLlwHspy2itRFkD+8csrfq9\nTWbeUNO5vgm8DpiXmY+tI+ZYRcRxwHHAYGaOunp5RDSWCjs+Mz9ca+dWEBHxLODC6uWzM/PiXvan\n2yJifeCzwD7ABsB84PmZeWlPO9YlEbEa8AngQGBT4AHgkMw8u6cdazKuSfacOXMmdXxJkiS1FxG7\nAmdULxcB1wILgduBKyjrUi/sTe/GzS0svdZlNapjI+LLwOHVy+U+qezAvSz9N7i3VYOIeA7wZuDp\nwMbAYuBG4FfAlzPzyro7FRFbAG8ALqz53+g7wN6U670duJUhM5YjYk/gMGBXYLOq7b+AS4DTM/O8\nGvvTOOeawLuAKzLzhzWG/jzwJso13E35dxtxhvZ4fk7GJcmeNm0aALNmzRqP00mSJKk39gP6gCXA\nbpl5WdO+H/WmS+MrM78KfHW8zhcRz6Mki2NJ6pcr1e/dzFb7ImIl4OvAoZSf2WLgBmAtYDowAzgs\nIt6ZmV+suWt7U81yAGpJ8CJiFWCvKuZ5wAHVuvON/SsDpwMH8cjrXQXYEtgKeFlEnAUclJmL6+hX\nZRfK9Z4K1Jlk70e5lsuAPTLzwZEOGO/Pybgk2dOnT2fu3LnMnz+/q+eZM2eOibwkSVLvbFJtbx2S\nYKsLImIt4L8pidPKmGh34qMsTbC/SJmWfzdARDwJOAV4KvDZiPh9Zl5S47l3of5/o40pX2wNAj9u\nTrArn2Bpgv1R4JOZeS9ARGwIfA44GDgA+ABQ5y0Ku9QYq1njvzM/7zDBHvfPybhNF58+ffp4nUqS\nJEm90bgHecQ/fFWLz1JGIr9BmYasYUTEesDbKUnWtzPz6Ob9mfmXqqbAHGBVytTiOpPsnWuM1dCc\nzz3icxcR6wJHUq73y5l5bPP+zLwtIg4BngI8gfI7VGeSvTPdSWhXquJ2+t+Zcf+cWPhskvM+d0mS\n1GsRcSrw2qa3tmkq4HUR8C1K4bNBYNuhhc8iYhvgPynTXjcGbqPcG/uxzPxrRPwG2I1y7+jrRtGv\nxwFHAc8BtgVWB+4B/gJ8F/hqzdNjRyx8FhEzgfcDzwDWpdwX+yvghMz8yyjOszclYbgD+CBdTh4i\nYhdKwvZ0YAvK6OktwFzg28AZmfnQkGMavwOvAM4E3kgZRZ5OmZ79D0phvA83RpNbnPdZ1XGNe4lX\npvx+XAp8LTN/0uaYVoXPdgTupBQH++9W58vM6yPiD8CzqvZj1uLzcXxEHN+if0TEppSf837ANpRp\n3fOAnwCfzsxbqnZbA9c1xewDTq3OBfBs4GbgLGB92l/v4oi4gJJkbxERU4f+O45W02eg4ZAqmYdS\noOy0prZrA2+hjKRvB6wJ/BP4OeV6/9bUtvH7NEi53uaf4yPiNh0zrp+TBpPsScr73CVJ0gRyPaXQ\n1FaUP+gXAn+t9v2t3UHwcPL2C0rSNUhJ3OZTErMXR8SBlD+8G/eTdiQiXkxJpFetjv0npUDSVpQE\ndw/goIjYu5Mpp3WIiP0oyWZjyuo8ymjcy4EDqv2dxFmbkjQNAm/LzFsjoit9rs73FuDE6uVDwE2U\nis5bUr4Y2Rs4IiKel5kPDDl8kDLD4QzKdd5cPbYCHgscDTw7Ip6WmYuGnPdjlEr1g9V5b6yebwW8\nmPL78enMPKZN1x8xipqZFwKbd3DJjSSzrmVC5lE+HztWfbqV8nv+iMJsEbEH5d7ldSmFA68DVqPc\nJ/544NCIeGFm/oHyGbuCkoQ/oYr1D0oyOQjcm5kDlKngI+mrtneMNcGuNIr/PQGYSvli44aqX3c0\nGkVEP/C/lC9tllB+TndS/n3fBLw2Il7VVDTtimrb6uf4cNym+OP6OWlmkj1JeZ+7JEmaKDLzOOC4\npiW1bsrMhwtPRUTL0eeqKNP/UJLou4CXZeYF1b7NKNM7TwMWjKY/1R/Xp1GSpGuA/TLz79W+lYBj\ngI9Rku1jgP8aTfxlERHrUO73nQIMAC/JzGuqfdOAT1FG/DvxeUqCe05mfrsL3X1YRGxMmW4Lpf/v\nzsy7qn1TgJdSCok9jfKz/FCLMG+ljMo+PGpb/dt/ttq3I/AyllamJyKezdIE+/vAEZl5Z7VvPeBk\nStL+zog4LzN/XdP1rgzsUJ33ryM070hmHk8ZdW2MxJ48dHm2iNiEkmCvQxm1PiQzb6v2BeVn8CTg\nBxERmXkzMHPIiPaxrUZzhxMRW1F+joPVecesUfwvIq6jJMznZuZhQ867OqVQ2+aUWQkHZea8at/m\nwGzKaPzpEfGEzLyx8d+U4X6OQ4zb52Qok+xJzPvcJUnSJPciyjTuQeCdjQQbIDNvjoj9KaNXj2d0\n93buRRkZXwK8p5FgV3GXAJ+IiH2AZ1KKQnU9yaaMKG5AuY5DGgl21af5wOERcTElKWgrIl4IHEKZ\nMn34cG1rsjtltHQQOKaRYEOZagx8r1qneT9a3yPbR5nqv0tzMbzMXBQR7wFeTxmtfTZNSTZl+nBj\n9sLDCXZ17J0RcSiwL+UWgIOAWpJsSgXqjapzf6mmmJ14D2UE+zZKwnlfY0dmZkS8BvgTJSl9DWOo\nYF99kbAp8ELKbRobVbGPHu64mr2RMpNhIfDSzLyxsSMzb4qIgygj82tSps+/bzTBe/A5eYQR1xOT\nJEmSumTvaruQMqL9CJm5kFIdeVQy80yWLlHUbnTu0mr72NHGX0b7VNt5mfm7Nm1OHi5ANRr+dUoC\neFRm/qvG/rXT1/S85VTrzDw5M/fNzFb/VoPA71pVm6+mls+tXm45ZN/bKAn01s0JdtP++ykFyqCm\nf8OIeDpwQtXnMzPzV3XE7dArms5739Cd1brdh1Nmivx2WU4QEetUo8ALKdO3Twb+TpmavWtmPmrK\ndRc1rvfC5gS7ITP/Tbl//1BGufxXjz4nj+BItiRJknqlcYPkNZnZbkr4T5clcDVifcswTRr3wtZ1\n3+1Inkj5o//KYdr8YYQYX6QU//pBZn63ro6N4DeU6fqrARdUhaa+35jK3KFLh9l3DyWRX33ojupL\nlpuHOfbe6tjVRtGXliJiL8qU7NUpo7qHjjXmKM69GeULjEHgqnbtMvNrYzzVYsrMkKmUkez1KTMV\nVqZ8Vn48xvgdiYg+lq5jPtz1PuqLtw714nPyCCbZGpWBgYFH3Qc+c+bMNq0lSZKGtTFLi521lJn/\nioi7gbVHE7i6X/hg4EDKdPONKdNxe2XjanvrMG3aJpRVUbTXAP+mVGMeF9XP/xDK/eIbAicBX4qI\nq4ALKDMFLqi+1GjnrmH2NY7rG7qjWoLqzcDzKfd0bwysMaTZmJeIiojDKKO6U4DLgH1bjSZ30RZN\nz//drZNU62M310rYGngXZTr2eRHxzsz8XLfO32QDyhcjg9R8vb36nAxlkq2ODQwMMGPGjEe9PzjY\n9fXcJUnS8qkxejlSde97GUWSXRU++xmlGFfjD5VbKBXGG9WTN60e46WTa205ml8V+voq5VreMspR\n5DHLzO9FxG+Bd1Luf94EeDKwPWXd6XkR8e5qmn4ro/5jMSJ2AM6n/BsNUqpt30xJ2BuJeWPJp2VS\njah+Anh3dY7zgVdUU9HHU/NI/KK2rWqWmdcDR0XEA5Rk+4SI+GFmXtvlU3flenv9OWlmkq2ONUaw\nZ8+eTX9/f497I0mSlgMLq+3UEdoNHb0cyWdZmmB/HvhCi7W5h67l220LKIn2KsO0aXedX6IktrOH\nSWS7qrpv9mjg6Gqt7z0p99Q/izLK/P2IeG1mzh7ruark93uUBPt+4B3Ad4eupR0RF1bnX5ZzrFyd\nY3/K78nnKJXTezF61JzU92K2xRcpSfYU4CWUn0U3det6e/45aTDJ1qj19/c7RVySJNXhTso04Y3a\nNYiIDYH16HA0tFqi61VV+59m5jvbNF1vdF0ds9spxb3aXiuwdZv3G9fzmqrKdDt9wEWNtYAzsytF\njjPzcuBy4JMR8ThKYaonUEaFx5xkU74gmU655uOHuRe549+LZtXvyP9QEuyHgMMz85vL2Nc6/KPp\n+XC/H6MSEdsC/cC6mfmdYZo238KwSV3nbycz74iI+ylfOtV2vUygz4nVxSVJktQrf6u2Ud1D3coL\nRhlzI5ZOzb64VYNqpHSfVvu6aA7lj/snD9Pm6W3evwu4u9q2e0BJMO6tXj+qIvdYVInpo1TLozVm\nBGxWLec1Vs1fNrRcmqta3/lJyxj/G5T1vRcAL+pxgt2opP3P6uUu7dpFxPER8fWIeGOHoV8L/Iiy\n1vRWw7R7TNPz8arE/SfK52G4631rdb3v7zBmzz8nDY5kS5IkqVcuphQpWouy5vG5zTsjYipwDKMb\nrXyg6Xm7UbJ3Aw8XmomIVapK1t30S8r06ukRsWNmXtGizZtbHZiZIyau1dJMUIp21bVmNBExm7Ke\n8idpv5xaY7r/Eh45FXhZDftvWCX8X2JpsbSOq4tHxOspy2AtBl6emT8fQz9Hq/F7/KhK6pSp60cD\n+0bEVi1ub3gMZU3rPuDqDs93Hku/AHkPpcBZK81J+4Udxu5E43ey3fXuDjwlIp6WmY+orB8RawLH\nU2YrnNjJyXr5ORnKkWxJkiT1yg8oI099lIrVzZWPt6Ak3Y3lhTqSmfdQlgXqAw6LiKc0xdw4Ij5P\nSTw+1HTYHmO5iA6dBtxX9ev0iNiuqV/rR8S3KFN12y1l1qlHVekeo+so980eFxFHRMQj7huPiF0p\nyfcg8ONhlmIbztAvUX7P0oJYH4yIRmV2IuLJlGXdtqP8TPuAJ1aVyIdVFcT7ZHW+z2XmuCxZ1aRR\nPX7vai1nIqKRgJ5AGVldFfhhRDR/CRTAWZTc7VbglE5OVk3rP5vyMzoiIj7XOG8Vd1pEfIylhd9+\nmpl/GsP1DXVzde5nRMSm1Tkbheq+Tvnd6gO+GxE7N/VrC8p/G9anfB6+UGOfGur+nDyCI9mSJEnq\nicy8OyKOoNzHuwXwx4j4B2U0dDvKfcz7UBKF0Xgf5T7hdaqY11OSiK0p9+DOAi4BPkAp9vSjiLgW\n2CszbxrzhbWQmbdGxFuAb1LuX87qnIuBx1KSyhdT1moer7W7O/FRypTePSnLd302Im6mjDZvQkmE\nBoG5tB8pHckjEp5q2bDPUGYx7AzcEBHzgGmU9Y//Qfm92IEyKj0NuDYi/piZew9znkNYei/+S6q1\nsUfygszs+EueEZwPvB54CnBHRCymFBz7YnXNL6J8sbQ9MKe65iksnc59G3BA9UVSpw6hFNTbG3gb\ncGT1e7daFbeP8u/3B8rnok7nA7tRPts3RcRDlN+hd2Tmgoh4AfC/VT/+UH32H6J8TqdQ/jvw6sy8\nruZ+dZ0j2ZIkSarTIK2nd7d8PzPPAJ4J/JiSRGxY7foyMLOaVt1IwlqtxfyouJn5E2Av4BeUkfIt\nKUnF6cAumXlmZv6DMj27MS13ESMvJdapdtd6OqUa9o8o1/oYyhJUPwCenpm/YOl063b3qI903lpl\n5oLMfD7wMuBMyvrDm1O+BFlMmV78duApmfnPtoGG1+rf8H3AEZR1qxdRKpjfB3wc2Dkz51CmHH+J\n8rOcSvm3bhuTpYXSBqv+bz/C48kMXw1+tI6hFFy7nfK7dgMwr7EzM39LKVT2SeCvlM/CxpQvMD4L\nPHnotOpKu88cmTk/M19AqRr+Pcq931tSviC5ifK5ey3wjMys+/7kz1A+x7dQrvcWIJv6Nhd4InAs\npZDeNEpCfgNlKa4dMvNc6tf1CvJ9XVjjuGeLJl9++eXstNNOXHbZZaOqfr2sx60Imn82QLufUx/A\nd/50owtmTwDrrT7SKigaL28/6Xe97oKAzR9XRw0e1WHg0oFed0GVG7/9+q5OlaxbRNxGSZC+mJlH\n97o/kjQcR7IlSZLUUxGxakS0nCJd3Y/b+LZs3rh1SpKWkUm2JEmSeiIiNq/uD72Pcm9qK4c0Pb+g\n652SpDEyyZYkSVJPVEXGbqT8TfqBiHhltYY1EdEXEQdTlvEZBP43M6/qWWclqUNWF1dH5syZ0+su\nSJKk5dNrKWtIbwN8B/hKRNwEbMrSQlVXA4d2sxMRsRPwDZa9vtBNmblvjV0as4j4EKVi+bL6SmZ+\nra7+TFYR8RNKVfNl9fpqOa1JISL+G9hpDCE+mJk/qqs/k5FJtoY1bdo0AGbNqruivyRJEmTmvIjY\nATiKUgE5gOnAPcCvKesDfy0zH+hyV9aiVJNeVmvX1ZEabUWpkr2sNq2rI5NcP+VnuSwGKb9bk8nj\nGNvvzQpfcdQkW8OaPn06c+fOZf7/s3ffUZJVVcPGnyanAQQlR6Vn0yKojRlRQQRRUcFAcFAwgSKv\nGbMor75mFMUEiIADYkBRVAwfQTCgCIKK7Z6WNOQcesgw9f1xbkHRU1Wdqru6e57fWr3urTpp33Et\n6d3n3HOGhhgYGGibbK++krtaTwfL9MyoDWNnteVX9P9ip4OdtvZ3xOni8n9d1+0QNE1l5iLK0Uyf\nGWvbKkH/PPBMynnAt2TmmGcdM/P3jO/YrGkrM/dnklcALA0yc/NuxzCVMnOHbscw0/kboEbU29vb\n7RAkSZKWEBGrUc7CXpsyY3glcH9VdhxlKfqoZeao9iuKiG8AB1YfX5CZ57So1wPsXcXxFMry9zso\nZwIfV50RLmmWMcmWJEnSTPVsHkmw35+ZhzeUXQlcNIo+HgtsRJWcjyQiXggcwAjvbkfECsCPgZdV\nde8ABilLsF8E7BwRrwVem5kPjmZsSTODSbYkSZJmqnUb7k9rLMjMQ4FD2zWuZpr/DGxIWXLeVjVz\nfgzwEOX36HaJ9mcpCfa9wFuBkzJzcTXmnsCxlHfQ/xf40EhjS5o5PMJLkiRJM1XjO9T3jaP9u4Bn\nAJcCnx5F/cMpG2Ad165SRGxC2citBnwoM+dn5mKAzKxVy8Q/BPQA74mIiexcLWmacSZbkiRJExYR\nVzD6HZhfkJnnRMQLgDMpyei6wAPAe4FXAZtW3y8Ajge+lpm1aqw3AN9t6K8HuCIiAGqZOeIGZhGx\nKXBYNcbbMrNtkh4ROwNvBm4FPlbdt7Iv5Q8AQ8C3W9T5djX+asA84AsjxSxpZnAmW5IkSZ3wL8o7\n0K1+bqzq1SjJdP2+bl3gT8BHgBWAhcDywFOBrwBfbah7a9XnwoZ+/t0w1mh8m7Ib+fcz84x2FSNi\ndcoy8Rrwzsy8YYS+67sz/zEz721Wofr+j5Q/EOw4ypglzQDOZEuSJGnCMvNlrcoiYgPKjto14CeZ\n+ecm1b4F3AxskZmXV+3Worxr/WzgwIj4TGZem5mnAacNm9HeNTOvGk2sEfFyYGfgbuD9o2jyFcrm\naKdm5omjqL8NjyT+7fwHeDFl53FJs4Qz2ZIkSZo0EbEcZZftdShJZatzmzcHXlpPsAEy81bgw9XH\nZYDntRmqZ5TxLEPZlKwGHJ6ZbQ9wj4iXAPtR/gBwYLu6Vf0VKDuWA1w9QvV6+ToRMavO6JaWZibZ\nkiRJmkxHAM+ivJ+8R2YualHvuBZljcu/N+pAPPsAW1bxfLFdxYhYAziakpAfnJk3tqtfmdNwf/cI\ndRvL57SsJWlGMcmWJEnSpIiIecDbKEnqGzMz21Q/v8X3dzbcr9yBsA6p4vlWZt45Qt2vAusDp2Tm\nD0bZf2OMI5293bjZ2iqj7F/SNOc72RqzgYGBR33u7+/vUiSSJGm6iohtKO9Z14AvZeYpIzS5vdmX\nmVmrdg2HUS4JbxPTDsCTgMXAN0aouxtll/CbgLePYZjG2ekVRqi7Uot2kmYwk2yN2pw5ZRXTvHnz\nHvV9rVZrVl2SJC2lqt24T6HM6p4NfHAUzabiF4r6sVvnZubCVpUi4jGU3cdrwNsz8+YxjDHUcL/q\nCHVXa7gfaVZd0gxhkq1R6+3tZcGCBQwNDY1cWZIkLc2+BzwBuArYKzMXdzkeImJl4JWUxPlHI1Q/\nknKk2PxRzMA/SmY+EBHXV+1Heod8s+p61XT4N5LUGSbZGpPe3t5uhyBJkqaxiPgwsBvlfeTXZOZN\nXQ6pbkfKzHoN+O0Idfeu6u0bEfu2qdcDnF1fzp6Z9f2OLgZ2oSxNb6d+1NeFI9STNIO48ZkkSZI6\nIiJ2Ag6jJI7vzMy/djmkRjtX1+sz878j1L0duKO6tvqB8pyLqs+3NbT/TXXdLiKaLhmvlqQ/o/r4\nqxrSVS4AACAASURBVNE/hqTpzplsSZIkTVhEbAycRJndPSEzv93lkIar79R6yUgVM3OtkepERH15\n90sz89xhxScC/0eZOf8f4DNNungvsDzlXezR7lwuaQZwJluSJEkTEhHLAz8G1qaca31gdyNqamvK\nzHO7Y8TGY4kdz6sl8p+pyg6NiDdHxLIAEbFcRBwEfKCK5yOZ6YY30iziTLYkSZIm6jXA06v7xwLn\nNRy71cy1mfnSSY+qEhErAatTktpbpmjYTwOPB14PHAV8ISKuBTZsiOWrmfn1KYpH0hQxyZYkSdJE\nLc8jR3BtxMi7aq/ecF9jYsd3jab9Gg31Fk1grGZjN1XtFr5/RPwEeAvljxBPoCT5vwO+kZlndTAW\nSdOESbYkSZImJDOPB44fR7vfA8uOol7TVxxHO25m3jCaccaiVUxN6p0GnNbJsSVNb76TLUmSJElS\nh5hkS5IkSZLUISbZkiRJkiR1iEm2JEmSJEkdYpItSZIkSVKHuLu4Oubxj1m52yEIWHm5nm6HoMpz\nv/SObocg4OCtXt/tEFS56jsndDsE1R2zZ7cj6IiIuALYBDguM9/Y3WgkqXAmW5IkSTPVRM/YlqSO\nM8mWJEmSJKlDTLIlSZIkSeoQk2xJkiRJkjrEjc8kSZLUMRGxDDAPeB2wDbAWcCvwJ+DIzDyrSZtl\ngX2BvYGnAGsCdwAJ/Bj4ZmbeP4qx3wS8EQhgFWAhcCrwf5l557C6+wPfobzTPQfYAPgosAOwDrAI\n+Bvw5cz87Zj+ESQt1ZzJliRJUkdExKrAmcBxwE6UBPYyYHVgd+CMiPi/YW0eA5wNHFu1Wblqswrw\nHODLwMURsf4IYx8FHA30AdcDdwO9wCHAXyJirWFNFjfcbwucT0n0l6Uk52sCuwC/joiDRvlPIEkm\n2ZIkSeqYY4DnATcDO2bmBpnZB6wNfK2q84GIeGVDm5OA7Siz3a/OzNUzsy8zVwNeCVxHmZk+tc24\nL6TMnO8DrJ2ZTwIeB7ydkujPBT7bpv3RwDnABpm5UWYGsAVwYVV+eERsNpp/AEkyyZYkSdKERcQ2\nwJ6UpPYdmfn7ellm3puZ7wT+Xn31garNCymzxTVgz8z8aWOfmXka8Jrq49OGJed1PcBGwIcz8+TM\nrFVta5n5bWB+VWdeRKzSov2qwF6ZeUPD2FdSkvYa5RXLt4z6H0PSUs0kW5IkSZ3w2up6F3BKizrv\nBt4KfKb6vE91/WdmntGsQWb+iUeS8z1a9PsAZbl5MydX1xWB7ZuU14ATMvOeJmMvoMxm9wA7tuhf\nkh7Fjc8kSZLUCU+vrv/JzIeaVcjMcyjLsuueQ0ly/zVC338Bngr0NymrAf/OzKEWbRv73hL4TYv+\nW/kn5Z3tGCFGSQJMspcqg4ODDA21+u/P+PX3N/vvnSRJWspsSEl4bxpDm/Wq6z4RsU/bmsVGLb6/\nqk2bGxvu125R5+o27etLyNeIiGVb/QFBkupMspcSg4ODzJ07d1L6rtVqk9KvJEmaUVaqrg+Ooc2q\n1fUm4JpR1G+V4C6x1LtB49FfK7Woc/co2684Ql1JMsleWtRnsOfPn09fX1+Xo5EkSbNQPflccwxt\nFgFrAKdn5n4TGHvFNmWNiXWrBLld+8aydsm8JAEm2Uudvr4+l3dLkqTJcBVQPzprtK6mJNmtloGP\nVrsztNdruG+1lH194OIR2t9c37lcktpxd3FJkiR1Qv1M6S0iYo1mFSLiRRFxdEQcFRHLAX+l7Nzd\nX31uKiKWbzNuD7BVRLSajX5Kw/0lLeo8vcX3AE+m2lytTR1JetisnMkeGBiY1PqSJElawg+BjwDL\nAgcCn2tS5/3ATsA/MvPBiDgJeCNlNvtA4MjhDaoE++KIuIdyFnaz3cFXAV4PHN2krL6h2hDwxxax\n7xcRn83MB4aNvRWPJNn/r0VbSXqUWZVkz5kzB4B58+Z1ORJJkqSlS2b+MyJOBvYCDo2Iq4GTMrMW\nESsDh1IS7BrVOdmZeWZE/Bp4MfCliLgfODYzHwSIiD7gCMrRW3fTfCa6BvwX+GJE3JiZP6vaLgO8\nC3hVVefo4Ul0gx7gBxFxQGbeVLXfAvh+VX4PcMx4/20kLV1mVZLd29vLggULxnxM1cDAgIm5JEnS\nxL0ZWAfYAfge8JWIuBnYlLIBWQ34bGb+oKHNPODnwLOBbwFfqBL0tYB1qzZ3AXtnZqujtn5DmUH/\naUTcClwPbEDZhK1GWcr+yTZxH1zFe11EXEZJuh9fXR8C3pqZN7RpL0kPm1VJNpREW5IkSVMvM+8G\ndoqI1wP7AltTktVbgdOBr2bm74e1uTUitq/q7wU8FdgCuBf4ByWB/lqLBLu+v9A9mfmBiPgb8Cag\nj7Ir+H+AHwCfz8x2O4P/k/Lu9oeAnSkbod0CnAd8LjNbLTOXpCXMuiRbkiRJ3ZWZJwAnjKF+rao/\n6jZVu02Hff4u8N2x9NHQdiHwtvG0laRG7i4uSZIkSVKHmGRLkiRJktQhJtmSJEmSJHWI72TPIIOD\ng2PeOb3Os8AlSZIkafKZZM8Qg4ODzJ07t9thSJIkzSa16keSOsYke4aoz2DPnz+fvr6+Mbf3LPCl\nx8I77ut2CNK0csgbx7RZsSbRCsv0dDsETaKI2BS4vPq4X7XD+FjarwUcDrwYWBsYAnbJzPM7Gmgl\nM48Hjp+MviUt3UyyZ5i+vj76+/u7HYYkSdJw9wMXUWaGbx1H+5MoZ1TXKGdU30CL/YMiYm/g68Ca\nwNmZueN4ApakyWCSLUmSpAnLzOuAcc0ERMQKwIsoCfZpwO7V2dnD660NfAt4FS71ljRNubu4JEmS\num0doP4+wS9bJNgvBy4B9gAuBhY0tJGkacMkW5IkSd3WuLpyic1FImJ14FTgsZT3tp8BXD81oUnS\n2LhcXJIkSRPWauOziLgC2AT4IDAfOBJ4PuV96sc3tIEyM31cRBxXfX5BZp5TfX8t8PrMPLPqdxKf\nRpLGzyRbkiRJnTR8qXf93eke4BRgW+BS4DbKrPVFwArAE6t6V1E2TqsBi6o+7gK2zszbJjt4SZoo\nk2xJkiRNhV0oy723zMzLGr7vHzYL/vHhx39l5oOUpFySpj2TbEmSJE22HsoS8Z2GJdiSNOu48Zkk\nSZKmwo2ZeVa3g5CkyWaSLUmSpMlWAwa6HYQkTQWXiy9lBgY6/9+3/v7+jvcpSZJmnZu6HYAkTQWT\n7KXEnDlzAJg3b17H+67Vhm8iKkmStIS7uh2AJE0Fk+ylRG9vLwsWLGBoaKjboUiSJEnSrGWSvRTp\n7e3tdgiSJEmSNKu58ZkkSZIkSR1iki1JkiRJUoe4XFySJEnTWkSsCqzU8FUPsDzlaLDlI2LtYU2G\nMvP+qYpPkhqZZEuSJGmy9Uyw/deB17co244ljwfbDzhhgmNK0riYZEuSJKlTatXPaL8fbZ3RtG+s\nK0ldY5ItSZKkCcvMK4Flm3y/+XjbNpTvD+w/oQAlaYq48ZkkSZIkSR1iki1JkiRJUoeYZEuSJEmS\n1CG+kz1NDQ4OMjQ09PDngYGBLkYjSZIkSRoNk+xpaHBwkLlz53Y7DEmSJEnSGJlkT0P1Gez58+fT\n19cHlJnsefPmdTOsEd101/3dDkHABnNW7HYIqsxdbYVuhyDgzV/dq9shqLLSlk/udgiaRBGxKXB5\n9XG/zBzTOdURsRZwOPBiYG1gCNglM8/vaKCSNMlMsqexvr4++vv7ux2GJEnSaNwPXEQ5p/rWcbQ/\nCdi5an8LcAOwTES8AfjuGPt6QWaeM9YAImIl4M3A7sA2wBrAXcClwO+Ab2TmVWPtd9gYfcDBwA7A\nRtXXC4FfAF/IzJsn0r+k7jPJliRJ0oRl5nXAuGYHImIF4EWUBPs0YPfMrFVl61CS95GsBGxZ9THm\n5XURsTnwG2ALHkn0/wOsDjyV8mwHRcS8zPz5WPuvxvgf4IuU38EX88jMfwB9wP4RsUtm/n08/Uua\nHkyyJUmS1G3rAD2U5PaX9QQbIDNPoyTebUXEFylJ9h8y87yxDB4RywA/pSTYtwJvysyfNZQ/ATgR\neAbw/YjYcqwz2hHxEuDL1cfTgQPrfVQJ/vHAc4FfRkRk5lDzniRNdx7hJUmSpG5rnPi5b6yNI+Jp\nwDspM9gHjGP851GWh9eAjzYm2ACZeSmwd/VxJWD/cYxxGOUPCYOUmfqHk/TMvBx4CWXZ+LrA+8bR\nv6RpwplsSZIkTVirjc8i4gpgE+CDwHzgSOD5wJrA4xvaQElCj4uI46rPI75bHRHLAsdQJo8+m5n/\nGU/4DffnNquQmZdHxG1V3NGsTpsY16EsN68BR2fmEsvZM3NRRHwN+AKwL3DoWMaQNH04ky1JkqRO\nqjX5XKMk0KcALwNupCTX91Het/53Q92F1Xd/BxaNYrz3U2ahLwM+Nc6Yr2+4X6lNveWr63Vj7H/T\nhvt/tqn3+3r9aom6pBnImWxJkiRNhV2AxwJbZuZlDd/3D5sF//hoj/+KiPWAj1CS8/dm5piXmlfO\nAe6gbHK2D3BBk7GeDaxWjfWLcY4DsGqbssadxbei7GouaYYxyZYkSdJk66EsEd9pWII9UYdRktZz\nx7vjN0Bm3lbt/P0d4H8iYgg4FrgGmAPsRNm0rAYcm5lnj3GIhQ33T6RsstbMxg33G45xDEnThMvF\nJUmSNBVuzMyzOtVZNfu9HyXx/fhE+8vM71Fm239X9XcF8ABlt/EfArcBb8/Mt4yj7xuAiyl/bDgw\nIlZsUfWDDferjXUcSdODSbYkSZImWw0Y6HCf76Osyjw/M38/UuWRVMd4PROYyyPnZF9CmYWuAZsB\nL6iS+/H43+q6AfDbiHh41joiNomIkyhHeNXfQ192nONI6jKXi88yg4ODDA1N7bGK/f39UzqeJEma\nkW7qVEcRsRqPzGIf2aFufwG8mDKDvVPjrHuVEH8R2BN4UURsl5k5ls4z8ycR8b/ARynJ9OURcRll\no7UNKZvAvR74JmUJvOdkSzOUSfYsMjg4yNy5c6d83Fpt+CaikiRJS7irg33tRUlE7wJ+0qpSRKwP\nnM6SO54D/CwzP1HVezMlwV4M7JmZ5zdWzMyrImJvypFj/cBXgF2rtgcAB7YI4U2ZeWFDP4dGxNmU\nM72fDWwEXAscVfU5CJxYVb+l1XNJmt5MsmeR+gz2/Pnz6evr63I0kiRJk+Z1lMT59My8u029FYCt\nW5T9veF+XnW9ZHiCXZeZiyPiZ8C2lNnsNTPzdmA9yhFiw9Vo8l51NUPe9N30iNiCckxYjbJUXdIM\nZJI9C/X19bmEW5IkzUoRsQawXfXxt+3qZuaVjO7d5vUpie21I9SrH7HVQ0mub8/MTwKfHMUYo7F9\ndb2bzr/DLmmKmGRLkiRpJtmR8jtsDZjwhmeV2yiJ82Yj1Nuk4f7WsQwQEasCfcBDmfn3FtX2rq6n\nZeaDY+lf0vTh7uKSJEmaSerL9e4H/tuhPuvJ+tyIaLb0u777+G7Vx8HMvHGMY5wI/BU4uepreP87\nU87jrlE2P5M0Q5lkS5IkaSapv2P938zs1O6rX6Xs5t0D/CwidmgsjIgNge8DW1GS4E+NcwyALYDj\nImLNhv5fBZxc9X1iZp47jv4lTRMuF5ckSdJk6+lgX+tV147tvp2Z10TEy4EfUpaEnxERt1He0V4V\n2JTyDA8Bh2Xm/HGMcWZEfA44hLLR2msi4kpg7eqnRjlG7E0deCRJXeRMtiRJkjqlRvPjslp9P9Y6\nAGtU9Tp6jnRm/h7YEvgI8AfKcV5zgbWAfwPfAPoz838nMMaHgN0px4pdR0neHwJ+DeyVma/IzAcm\n8hySus+ZbEmSJE1Yq528M3Pz8bZtUXfSzinNzNuAz1Y/kzXGz4GfT1b/krrPmWxJkiRJkjrEJFuS\nJEmSpA4xyZYkSZIkqUN8J3uGGRgYGFeZJEmSJGnymWTPEHPmzAFg3rx5XY5EkiRJktSKSfYM0dvb\ny4IFCxgaan1axcDAQFeT8C3mf6RrY+sRax/ylW6HoMoKy3TyWFiN1+BO7+l2CKpsMGf5boegyvrd\nDkCSZjGT7Bmkt7e32yFIkiRpmoqIjYCDgV2AzYGVgFuAC4GTgO9n5mjOIpc0AW58JkmSJM1wEbE7\n8G/g/cDWwL3AQuCxwK7AfOB3EbFS14KUlhIm2ZIkSdIMFhFPBk4GVgXOB/ozc93M7AXWAo6oqu4A\nfK47UUpLD5NsSZIkaWb7BLA8cDPwksy8uF6QmYsy8z3A/wN6gP0jwg0SpEnkO9mSJEmasIjYFLi8\n+vgsyu+Zh1OWLv8lM3es6vUA+wJ7Ak8B1gYWA9cAvwe+nJmXNOn/UOBQ4D+Z+cSI+BBwALAR8MbM\nPKGh7pOBDwLPo8zkXg+cDnwmM6+KiKuBDYD/zcxDI2IfynJqgOdl5h/aPOcxwBuB24H1MvP+qXqu\nNrYA7gZOzcxbWtT5AbATZba7l7K0XNIkcCZbkiRJnVQDNgR+AWwFXEZJcomIFSkzqscBLwbmUBLz\nW4HHU5LXCyLile0GiIi3AZ+mJPL/prx/XC97OfBXSrK7XjX2/ZTE9fyI2IaSaNaAh6pmpwJ3Vd+9\nus24ywGvrOr9sCHBnvTnaiczt87M1YAD21R7oOF+xdH0K2l8TLIlSZLUaQcD51BmerfOzH2q7z9E\neS+4RplpXjsz+zJzI+BJwD+BFYDvRMScFn2vQNnca//M3Cgzt8nMHwJExJrA8ZQkdSGwbWZunpkB\n9FXffY9hqzkz827gZ5Tl1Hu0ea4XUWbG4ZGZ70l/rtEaYefwp1XXB4HBsfQraWxMsiVJktRJPcC2\nwBsy865hZbtTEtGLMvMLmfnw7GpmDgDvrD6uSTmGqpnNgXMz8/gmZfsBa1T3+2fmRQ39L6DMMq9H\nmcke7sTqumFEPLvF2HtW1yuGLSmf7OeakOpor9dXMZ6QmYs6PYakR5hkS5IkqZNqwK8y847hBZn5\nZEoSvHOLtuc33D++zRgntfi+3u/VmXlWk/FvBb7eou1vgZuq+yWWjEfECsArKM/3vWH9TvZzjVtE\nrAz8GFgduA34aKfHkPRoJtmSJEnqtIFWBdVu10035xo2893uPOdW/QclCb6wTdtftxj7IeCHlJn4\nVzWp8mIemSWfP7xwkp9rXCJibeAM4BmU97tfmZk3dHIMSUtyd/FZaGCgo///PKL+/v4pHU+SJE17\nN7UqiIgtgLcBz6Xs8L0O5fipTvS/TnW9vk3bbFN2InAQsHFEPCMz/9pQVl8q/tfM/O/whpP8XGMW\nEXOBX1Fmzu8CXtNu13RJnWOSPYvMmVP20Zg3b96UjlurtdtjQ5IkLYWGv4sNQETsCxxN2eSrRjl2\n6ipgqPoM5firtjLznhZFK1fX+9o0b/k+cmaeFxGXUd6PfjVll3IiYiVgtyrGJd6ZnoLnGpOI2IGy\nRPwxlD84vDwz/9aJviWNzCR7Funt7WXBggUMDQ11OxRJkqRHiYjNKYno8pRjvd4B/L/MfHBYvcUT\nGOZ+yvFU7WaQVxmhj5Mo7y2/Gjik+u6lwGpV/4/a8XuKnmvUqoT/GMrv+RcCr8jMa6ZibEmFSfYs\n09vb2+0QJEmSmtmTR2Z692o2s1odwTURt1F2D39cmzoxQh8nUpLsTSNi28y8ANirKvtVtXlao6l4\nrlGJiNcCx1L2XToVmNep2XFJo+fGZ5IkSZoKm1bX+9osXX7pBMf4L2Xjsie1qbNruw4yM3lk47Td\nqt25d6XJruKVqXiuEUXErpT4lgGOysxXmWBL3WGSLUmSpKlQT/hWjIg5wwsj4nHAp3jkHeZ2u3C3\ncs4j3cU2TcZYC3j7KPo5kZKsv4RyLNcqwO3AL5rUnYrnaisi1qck2MsBJ2bm2zo9hqTRM8mWJEnS\nVDi34f6LEbE8QET0RMSLgT8Cv6fs/t0DbBcRPWMc4zig/i709yLiCfWCiAjK8V2jOYblZGAx0A+8\nsfruB5n5QJO6U/FcRMRxEfFARNzdpPizwFrA5cBbxtq3pM7ynWxJkiRNhZ8BfwKeTUkE94qIayjv\nUK8JnE3ZNOyLwJbA9sCVETE/Mz88mgEy89KI+CjwGcqS8QURcTllFvkJwKWUDc2uGKGf6yLiLOCF\nPLKreLOl4lPyXJVlq59HHetSzZTvU31cHTiv/D2hra9l5rFjGFvSGDiTLUmSpE6pMSwJrMvMxZSl\n15+nvDu9IrAJJfE9CNglMxcBhwKnU47aWgW4ZTT9N4zzeeAVwJmUJd7rAfcCnwaekZkLG6q32/H7\nxIbxLs3MP3fzudrUW53yO30NWBvYZhQ/641iLEnj1DMJZxzPuEOTL7zwQrbddlsuuOAC+vv7ux3O\ntItnFHoArvnkATPuf/vZaO1DvtLtEFQ5Zr0Rj0TVFHj2v/7S7RBU2WBOu1OVNJXWX3PVMS9Xni0i\nYjXgTsrvrO/JzCO6HJKkWcaZbEmSJM0qEbFqRCzboviJDfdXTEE4kpYyJtmSJEmaFSJi24i4Chji\nkbOth9uvuj4A/GEq4pK0dHHjswYDA6PZbHLyTZc4JEmSZpiLgfuq+69ExI2Z+TuAiFgOOBh4K2Wp\n+Hcz85bm3UjS+JlkA3PmlCMN582b1+VIJEmSNF6Z+WBEvBb4DeVIq99ExM3AjZTNyFajJNjnAu/t\nWqCSZjWTbKC3t5cFCxYwNDTU7VCAMpNtwi9JkjR2mXlhRDwReDewK+Xorl7KTuN/ppyBfUJmPtS9\nKCXNZu4uPg3N1N3FD+zZzP/tpQafW+SrH9PBjXc92O0QVNl4NbeCmS5WWqV7u4tHxBWUWeXjMvON\nUzDeoZTjs2qZ2WozNEnqGGeyJUmSNJVGeyb0rBERywOfAA4BlgU+kZmHdTUoSZPGJFuSJEmaJBHx\nZOAEYGuWwj8wSEsj121JkiRJHRYRy0TEh4C/AE8Cfkj1ip2k2c0kW5IkSeq8lwKfphwptn9mtjq3\nW9Is43JxSZIkdUxEPJ6y0diOwOOAW4HzgC9l5h9btDke2Bf4bWa+OCJ2BT5AWWK9PLAA+FpmHl/V\n3wz4WDXG+sBtwO+AD2fm1SPE9xTKu9HbV/HdDvwR+Gxmnt9Q7zHAv6r+T83MPVr0Nx/Yp4ph68y8\ntipapnrufTLzinYxSZpdnMmWJElSR0TE04GLKAnzBpTzqW8DdgHOioh9WjRdTHlXeZmIeD3wCyCA\nW4CVgH7guxFxQERsDZwP7A3cVdVZB5gHnB0RK7eJb0fgT8CrgEXANZREe3fgTxHx8nrdzLwNeHP1\n8RWNZQ39vZCSYNeAdzQk2AB/AJ5rgi0tfUyyJUmSNGERsSzwPWBV4CZgu8zcJDO3AtaiLJ3+JrBm\niy56gA2BLwLzMnP9zJwLrAdcTElkP1qN8Rtg3cx8UmZuCLyt6mNz4LVtwvwu5d3o9TKzLzOfAGwD\nXEbZ9fuYiJhTr5yZpwPHVrF9LSJWaXjeFavnqQGnZOb3GwfKzFsyc3GbWCTNUibZkiRJ6oRdgLnV\n/bsy87x6QWbel5mfBH4OrNGmjy2BIxsT1sy8FfgcJdHdgDKz/YbMHGqocxQwWH3crk3/t2TmftUs\ndb3tJcBbqo9rs2SS/m7gSmAj4FMN338E2AK4gUeSfEkyyZYkSVJHvLi63guc0qLON0fRz3eafHdJ\nw/0JmflQkzr/4pFEvJVvN/syM8+iLG0H2GlY2RCwf/XxHRHx1IjYkvJedw14a2be0mZMSUsZk2xJ\nkiR1wlbVNTPz/hZ1/gY0S5Drbs/Ma5p8P9Rw/48Wbet1Wr6TDfy5TdkllCR9y+EFmXk2cCRl0+Cj\nKX8sWB44LjN/0aZPSUshdxefAQYHBxkaGhq5Ypf09/d3OwRJktR961Bmdm9oVSEz74+I2yjLspu5\ns8X3tYb7Vr8U1eu0O4v6qjZl9ZnsVrF9kLIkvv6Lz0LgXW36k7SUMsme5gYHB5k7d+7IFbuoVquN\nXEmSJM129Rnk+0aod2+bstH8UjGRXzzuaVNWn31fqVlhZt4TET8D3l/FcFbje+GSVGeSPc3VZ7Dn\nz59PX19fl6ORJElqqZ48rzBCvVVGKJ9MK9I6ya8n13c3K4yIJwPvpCTYi4F9I+J7mXlmx6OUNKOZ\nZM8QfX19LsuWJEnT2S2UpdqPa1UhIlajHOfVrWVw6wN3tChbr7reNLwgIpYDjqe8h31M1cf7gGMj\nYmtntCU1cuMzSZIkdcJAdY3qzOxmnjVVwbTw9GZfRkQPsDUl+b+kSZVPUM7TvpayXPxQytnaGwNH\nTEagkmYuk2xJkiR1whnVdVVgtxZ13j5FsbRyYIvvXwqsWd3/urEgIp4OfICSgB+UmXdm5j3AAZSZ\n+zdERKvnlbQUcrm4JEmSOuE04GpgI+DIiLgqMy8AiIiVKbO/LwSuB9btQnz3AxtExDeB92fmoiq2\npwLfqOpcDfy03iAiVqQsE18G+HFm/rxelplnRMRxwH7AURGxVWbe2tB2TaBxRr9x1/NVImL4Lua3\nZqa7yUqzgDPZkiRJmrDMvBfYn7K7+PrA+RFxRUT8G7gVeC9wMOWIr3bHbE2WB4A3AK8Hbo6IgYi4\nDLgA2LCK+w2Z2bg7+qcp52bfTol9uPdSjv5aFzhqWNlFlPe76z/1I8J6gEOGld1EWXouaRYwyZYk\nSVJHZOYZwNOAHwDXUTYTewzwG+BFmXkCZffuGkv+Hlqj/YZoI5W3qrNM9d09mXkO5b3wnwCrUf4Y\ncAPwQ+BpmXl2vVFEPIdyDvZi4D2ZeeOwfsnM2yjJdw3YPSL2aShe3BDPSD+LR3guSTNIzySccewy\nlwm68MIL2XbbbbngggsAHr6fxruL9wAc2LOZ/9tLDT63aGDkSpp0N971YLdDUGXj1fzb/nSx0iqr\ndmMmWZKWCv7XTpIkSZKkDjHJliRJkiSpQ0yyJUmSJEnqEJNsSZIkSZI6xHOy1TFfv+wn3Q5BwENX\nL+h2CKoctcFTuh2CgNhgtW6HoMqpVw11OwRV3r/I/1ZI0mRxJluSJEldFRFnR8TiiDhz2PeH5uBC\nDAAAIABJREFUVt8/1K3YJGmsTLIlSZLUbaM5A3vGiIjXRcSh3Y5DUneYZEuSJGk6mE1nd38G+Hi3\ng5DUHSbZkiRJUodExLrARt2OQ1L3mGRLkiRJnfOMbgcgqbvcXVySJEkdFRG7A/sBTwMeCwwBlwM/\nA76amXd2aJyNgPOADYC/AC/MzLs70XfV/w7AgcAzgfWAe4CFwOnAlzPzhoa6m1KesbH94ur298Ag\n8GZgEfC4zLyvzbiDwBOA32Tmrp16HklTw5lsSZIkdURErBARPwVOAV4GPIaSeC4H9AOHAQMRsVUH\nxlqdkuxuAFwC7NrhBPtI4Azg1ZQEeyHwIPAk4BAgqyS87n7gIuD6hu8uqn4WAPOr71YFWibOEdFP\nSbBrwPGdeBZJU8skW5IkSZ1yBPAK4F7grcDqmbllZq4JPJ+SbK4P/DIiVh7vIBGxPGVWfCvgCmDn\nzLx9grE39v8B4O3AYuAjwGMyc25mPg54MvBXYHXgpxGxPkBmXpeZ/cC36/1kZn/1cwBwLnB1VfTq\nNsPvWV0XAad26pkkTR2TbEmSJE1YRGwBvIUyA3twZn4nMx+sl2fmH4AXU2Z8NwYOmMBwJ1CS9huA\nF2XmdRPo61EiYg3gw5Tn+ExmfjYz76mXZ+a/gF2Am4A5wAdG029m1oDvU3ZRf1lErNCi6mursU/J\nzHvH/SCSusYkW5IkSZ2wN+V3y9uA7zarkJlXAL+kJJp7jGeQiPgCZbb3TsoS8UvH008bL6ckz4uB\nw5tVyMw7KMu/x/ocJ1bXOZRE/VEi4pnAptXH742hX0nTiEm2JEmSOuE51fU/mbm4Tb2/VNenjHWA\niDgYeC9lOfpumXnRWPsYhfpz3JCZt7WpV3+ODSPisaPpODP/QXl/HJovGa8vFb8mM88aTZ+Sph93\nFxcAg4ODDA0Njattf39/h6ORJEkz0HrV9TkNu2q3s2pErDnad6mrHcu/TFlKfWRmnjvOOEdSf44N\nRvkcUM7FvnmUdU8E/g94eUQs17ikHngN5fnmN20paUYwyRaDg4PMnTt33O1rtVoHo5EkSTPUqpQE\n8U6GHWXVQg1o9V5yMydSlmf3AAdGxNGZOTjmKEe2anW9B8hJ6P8k4NOUjdN2Bn4FEBHPBTak/Lu4\nVFyawUyy9fAM9vz58+nr6+tyNJIkaYZaVF0vzswXdLjvHmBZ4H3AO4FNgJMj4lmZ+UCHx6o/x03V\nbuEdlZkLI+KPwHaUmetfVUV7Vde/Z+ZAp8eVNHVMsvWwvr4+l35LkqTxuprynvVGk9B3jbKL+DkR\n8TfgrGqsL1KS7k6qH7O1fof7bXQi8FzKkvFlKc/3qup6wiSOK2kKuPGZJEmSOuGv1XXTiFinVaXq\njOsxy8xzquu5wKcos9vviIjdxtNfG/XnWC4intaqUkRMZLLqR8ADwJrA9tXPusBDlGO+JM1gJtmS\nJEnqhO9TZmKXAQ5pU+/0iPh3RMybwFiHAX+iJNrHRsQGE+hruJ8Bd1X3H2xT79sRcUVEvG/Y9w9v\nlhYRKzZrmJm3Ar+uPr4EeGV1/5vMvGnsIUuaTkyyJUmSNGHVedVHURLf90TExyJi5Xp5RGwUEScC\nOwK9LLmp2Kh3Uq2OCNsHuANYCzgpInom+Aj1vod4ZKZ8j4j4ekSsWS+PiLUj4nBgf2Bjltzk7bqG\n+9dUbVZlSfWN3HYFXoYbnkmzhkm2JEmSOuXdwKmUhPGTwM3VrPVVwBXA3pRl0gdl5vnD2o4pSc7M\nhcBbq3bbA5+YUOSP9nngm5TneBtwQ0T8JyKuoCTR76rK/i8zTxnW9gzKMwKcEBH3AQubjPFzYAjY\nCnhCdf+zDj6DpC4xyZYkSVJHZOa9mbkHsDvwU+BWYHPKu8cLgG8AT87Mo5o0r9F8NrvV92Tmj4Dv\nVB8/HBHPm9gTPNxvLTMPAnagLIO/lrKj+eOAK4HjgGdn5seatL2c8seE/wD3UXYr/2uTevdS/o3q\nz/ejzLyvE/FL6i53F5ckSVJHZebPKTO1o62/Q4vvP0mZEW/X9i3AW8YU4OjjOgc4ZxztfgL8ZBT1\n9gP2G3NgkqY1Z7IlSZIkSeoQk2xJkiRJkjrE5eLT2MDAQLdDkCRJkiSNgUn2NDRnzhwA5s2byPGR\njzY4OMjQ0FDTMpN5SZI0G0TEMcC2E+jiY5n5i07FI2npZJI9DfX29rJgwQKGhoYYGBiYcLI9ODjI\n3LlzOxRda/9834cmfQyNbM0t1ut2CKr86043iZ0OnrXdRt0OQZVL/3NLt0PQNBMRzwfOqj6+oNpo\nbCKeAGzT8Ln+amTLHcqHWWuC40uSSfZ01dvb27G+6jPY8+fPp6+vb4nyTiTykiRJEzCaBLipiDgI\nWC4zj2jcpTwiNgUur/rePzNPGGf/WwJvB15AOY5sBeA24B+UHcS/2+rorYhYmXKW9yuArYE1gLur\nuM4AvpmZl44nLknTl0n2UqSvr4/+/v5uhyFJktQREbECcDhwDXDEJPT/buDzwLKUZP164C7Kmdkv\nBHYC/icidsrMa4e1fRLwS2Djqu0i4L/AepTZ9icDb4+IN2fmSZ2OXVL3uLu4JEmSZqqnAstPRscR\n8TLgS5Tfl08FNs/MDTNzLjAH+AAleQ5g/rC2qwC/oCTY1wC7ZeYamfnEzFwLeD5wKbAScGxEPGEy\nnkFSd5hkS5IkaaZ6xiT2/VFKEn0x8NrMXFgvyMwHMvOLwNFAD/D8iHh8Q9vXU2a7a8ArM/NXjR1n\n5h+AfauPywNvmLSnkDTlXC4uSZKkCYuIKyiJ5Wg03eQsItamJLcvBTYE7gP+SXl3+eSGeo0bpgFs\nFhGLq/vjMvONTfpeBfggsAewKbAYSOA44FuZubihbg/wb+BG4JTMfLDFc/yW8s41lE3XLqvu76e8\nr31PZl7QrGFmnhcRQ8BqVVtJs4RJtiRJkjrhX8Ctbco3ANahzO4+0KR8fcqy6w2BhcCVwOOB7YHt\nI+Jpmfm+qu4i4CJKUr8WJan9d1W2kCWtCvyJsvnYdVXfm1GO+3oaZfn2nvXKmVkDlkjUm+hpuH/4\nnezMPBY4tl3DKpFfoq2kmc8kW5IkSROWmS9rVRYRGwAXUhLsn2Tmn5tU+zxwA7BTZi6o2q0FnAC8\nBHh3RPwkM/9UzQ73R8R3KUutr83Mdru7fpCS2D8rM8+v+l4F+ColmX51ROwzlg3Iqk3XDq4+Xg4M\njLZt5QDKu9014PQxtpU0jZlkS5IkadJExHLAjymz2APA/k2q9QBrA8/MzOvrX2bmrRHxOsrs9GrA\n2ygz0mPRQ5kdf1pmXtTQ990RcQCwA2VW+21A2yS7mn1eG9gOeD/wbOAmYK/G5eZt2q8GbEH5w8D/\nUBLsT2XmmWN8JknTmBufSZIkaTIdATwLGAL2yMxFTerUgNMaE+y6zLwD+DUlWd5pHOPXgPMbE+yG\nvh+i/AGgB3hWda51UxFxEfAQ5T3tn1KWqR8KPDEz/9YugIh4Z/XO+J2UGf03Ud4Ff05mHjqOZ5I0\njZlkS5IkaVJExDzKDHENeGNmZpvqzZaQ1/2ruq4TEWuMI5TR9L0MMLdNvQHgH5R3uh+iHN31OmBe\nRCw7wvg3Ud4hvwy4h/KO+B7AWyNioxGjlzSjmGRLkiSp4yJiG+BblAT7S5l5yghNrmpTdmPD/drj\nCGfCfWfm3pn5lMzcsKr3Hsoy9C9TZsNbysyTMrM/M7egLHt/ISXh3h/4W0S0S+4lzTC+k62HDQyM\ndb+Oor+/3T4jkiRpaRMRqwOnACsDZ1M2HhvJPW3K7m+4X2kcIXW078y8EzgiIgaBXwAvj4g3ZObx\no2hbA86ujiG7CNgc+Dbl3XBJs4BJtpgzZw4A8+bNG1f7Wq3WyXAkSdLM9z3K2c9XMcpNwYAV25Q1\nJr93jyOeSek7M38VEZdREuU9gBGT7Ia2iyLiOOAw4HkRsVZmtjsCTdIMYZItent7WbBgAUNDQ90O\nRZIkzXAR8WFgN8oM8Wsy86ZRNl2/Tdl6Dfej7W/cfVdHh/VXZb/MzNvatL+Bcp73uvUvIuI51ZjX\nZOZ5I7StW4f254xLmiFMsgWURFuSJGkiImInysxsDXhnZv51DM2fTnmHu5mnVNerMvOucYT29DZl\n9b4fBBZU91sDv6U8x/6Us7pb2biq1/hu94nAppSl8juO0Lbuxpa1JM0obnwmSZKkCYuIjSnnTPcA\nJ2Tmt8fQvAd4VUQssfFYRKxDObqrBpw+rLi+DL3l0VtV38+LiC2b9L0C8Kqq7zMz84Gq6E/ALdX9\nu6qzvpcQEbsA9d3Bz2oo+nl1fW5EPKtF25WAfaqP/3SpuDR7mGRLkiRpQiJiecoO22tTNvM6cIxd\n1ChLp0+NiM0a+l0XOJmSRC8Gvjms3XXVdZ2I2L5qs2qTvi8DfhIRWzf0PYfyDvUG1VdH1suqZPsw\nSoL+FOAXEdHb0HaZiNifMmMNcD1wTMOYX6Ccib1cNe5rIqKnof02wO8oy8xrwKdb/cNImnlcLi5J\nkqSJeg2PLMl+LHBeRLSrf21mvnTYd++kJNGXRsSVwH2UzdOWoySiH8/Mfwxr82vgw9X97yPifuAS\nYNth9Q4D3gVcHBFXA0OUzcpWqvo+JjN/2dggM78WEesDhwAvAjIiFgL3ApsBK1RtrwVenplDDW2v\njoiXAT+ivKv9A+D2iLiO8q72mlXbh4BPZOaP2v1jSZpZnMmWJEnSRC1PSRprlOXT24zwU1+6vUxD\nu4WU5PgIyvvRmwCLgDOBV2TmZ4YPmpl/AA4CLqdstHYncGGTvm8Dtqck23dWfT8AnAe8MTMPaPZQ\nmfnhKqajgAFKcrx51ce5wIeAJ2XmhU3a/qF6zg9WdR8EtqD80eDfVZ/PyExnsaVZxplsSZIkTUh1\nPvSoj69qaHcWsOywr99T/Yy2j2+y5DJyMvPyJn1/ovoZS4wXA28bS5uGtndQlo5/YTztJc1MzmRL\nkiRJktQhJtmSJEmSJHWISbYkSZIkSR1iki1JkiRJUoeYZEuSJEmS1CHuLq6O+f5pg90OQcAdD2S3\nQ1BlteX8O+Z0cPF513Q7BEmStBQxyZYkSdKsExHfAA6sPr4gM8/pZjySlh4m2ZIkSZpVIuKFwAFA\nrduxSFr6uJZRkiRJs0ZErAYcAzwE9HQ5HElLIZNsSZIkzSaHA5sAx3U5DklLKZeLS5IkqWMi4rXA\nPKAfeCxwH3AdcAFwbGae0VB3B6D++UnAzcDHgV2B9YF7gX8C38zMk0cx9s7Am4FbgY9V95I0pZzJ\nliRJUkdExMnAycBLgdWBK4Abgc2AvYDfRcQRDU0WV9daVecC4G3AasCVwCrA9sBJEfH5EcZenbJM\nvAa8MzNv6MQzSdJYmWRLkiRpwiLiFcBrKYnzQcBambllZvZSZrQ/WVV9R0Q8v0kXhwNXA72ZuW5m\n9gEbAr+pyt8bEdu1CeErwEbAzzLzxIk/kSSNj8vFJUmS1Ak7VtdLMvNbjQWZuQg4LCI2pCTcaw5r\n20NJqF+Qmdc3tLslIvYBFlJmtd8O/HH4wBHxEmA/ynLzA4eXS9JUMsmeIQYGBrrSVpIkaZTqO3k/\nJiJWyMz7h1fIzANatK0BP29MsBva3BYRvwZexSOJ/MMiYg3g6KqPgzPzxvE+gCR1gkn2NDdnzhwA\n5s2b1+VIJEmS2vp/wDsoS7b/HBGfAk7PzHtH2f4vbcr+SUmy14mI1TPzzoayr1I2SftxZv5gHHFL\nUkeZZE9zvb29LFiwgKGhoXH3MTAwYJIuSZImVWb+PCK+ArwTeApwCnBfRPwFOBP4aWb+s00XV7cp\na9zE7HHAnQARsRuwL3ATZSm5JHWdSfYM0Nvb25F+Fi5c2JF+huvv75+UfiVJ0sySme+JiO8D76Mc\nw7UqZXfw5wGfiIg/Agdl5j+aNL+7TdeNS89XBIiIxwDfpiwTf3tm3tyBR5CkCTPJXorsvvvuk9Jv\nrVablH4lSdLMk5nnA3tGxPKU5HonSsK9NbAdcG5EPDMz/zOs6Yptum0su6e6HgmsC8zPzFM6Erwk\ndYBJ9lJm/vz59PX1dTsMSZI0y2XmA8AZ1c+HIuJFlCXkqwEfBYa/y7Z+m+7Wa7ivLx3fmzKLvW9E\n7NumbQ9wdkTU4/IIW0mTyiR7KdPX1+fybkmSNCkiYtnMfKhZWWb+LiKOp5yhvU2TKk8HvtXke4An\nV9eFmVlfVn77KEJak5KI3wU8WN1L0qQyyZYkSdKERMSawG+BrSNih8w8r0XV5avr8B1de4A9IuKQ\nzLxlWN+PBXamJMi/q3+fmWuNIq7F1e1LM/PckZ9EkibO5TKSJEmakMy8HbiP8u70jyJi54ion5tN\nRCwXEfOAN1CS5R8N66IG3AacGhGbNbRbD/gBsHJV5+vjDLFn5CqS1BnOZEuSJKkT3kA5K3tT4NfA\nUERcR0lwN6DsNF4DTgO+1qT9h4DPAZdGxJWUHcUfT/l9tQZ8JDMvnuyHkKSJciZbkiRJE5aZl1He\ntT4IOAu4F9gc2AS4Bfgp8OrMfGWL97avA/qBwykJ9kaUZeVnAq/IzM+NMzTfw5Y0pZzJliRJUkdk\n5iLK5mWtNjAbqf2twPurn07E44SSpCnn//FIkiRJktQhJtmSJEmSJHWISbYkSZIkSR1iki1JkiRJ\nUoeYZEuSJKmbargDuKRZxN3F1TFPWHWFbocgYNmebkeguk3WXbXbIQhYfaPVux2CKpteu6jbIWia\nyczfA8tOtJ+IWAb4KPB6YGPKEWAfycyvTrRvSRork2xJkiRNCxExFzgYeCElWV4euBH4K3B8Zp7W\noukhwCcoM+J3A1e2GaMfOAF4IlDLzAkn+ZLUyCRbkiRJXRcRBwBHACtQkuXrgfuATYA9gD0i4mTg\ndZk5fHn5btX1amCbzLyjSf/LUma7P0IHZs8lqRXfyZYkSf+fvfuOk6sqHz/+WTCUQGihl1AkPESK\nEECxoKCCKIIGvyJoVJrS5Ivi114oflVsoKgooiB+EWwIP2JHUcBCE1HA+CQIJHRCDyW0zO+PcxeG\nzWyf2dnd+bxfr3nd2XvPvfPMbrI7zz3nPEdqq4jYHfgmpef6QmCzzFwvMzcB1gJ+UDV9K/A/DS6x\nFiUx/1MvCfY04HLgGOAB4A9NfxOSVDHJliRJUrt9utrOBd6Umf/pPpCZ9wL7A9cCXcBhDc7v7pl+\nvJfr/wDYFvgNsBVw6fBDlqTGHC4uSZKkpoqIGZTEeHtgdWAhcBPw/4CTM/OhurYrAGsCi4AfZOZj\nPa+XmU9HxM8oCfKGEbFSZj4UETdThpN32z8i9q+eH5uZx1fPFwFHZ+ZXq9ds0juVpCWZZEuSJKkp\nImIZ4EfAGynDtx+nJNdrA9OB7YDDImK3zLweIDMfATYcwOWfrHu+bLW9DrgP2ILyufZ+YH517M66\n9jMy866hvCdJGiyTbEmSJDXLVykJ9iJKlfAzM/MpgIh4OfAdYDPgFxExrVGvdR+2r7b3ZOYCgMx8\nQ3Xtmyg92hdk5oE9TzTBljSSnJMtSZKkYYuITYF3U3qwj8zM73Yn2ACZ+Sdgd8oa1hsAhwzi2i8E\n3lBd+5Rmxi1JzWaSLUmSpGbYj/LZ8n7gjEYNMvNm4BeUAmZ7D+SiEbEG8GPKCMx5wBeaEKsktYxJ\ntiRJkprhpdX235m5uI92l1fbbfq7YERsBFwCTKUsvbXHIIeYS9KIc062JEmSmmHtavvSiOgrye62\nQkSskpkPNDoYETsAF1DWwL6HkmDPbk6oktQ6JtkakLlz57Jw4cKGx6ZPnz7C0UiSpFFoBcqc6Yco\nFcX7UwOWaXQgIt4CfA9YDrgBeH1m3tCcMCWptUyy1a+5c+ey2Wab9Xq8VquNYDSSJGmUerja/iMz\ndx7qRSLig8AJ1ZcXAftk5v3DjE2SRoxJtvrV3YN91llnMW3atDZHI0mSRqlbKfOs1x/qBSLiKODz\nlF7u04DD+5nfLUmjjkm2BmzatGkODZckSb25grLM1oYRsWZm3t2oUURMyMwnG+zfHziRkmAfm5mf\nbmWwktQqVheXJElSM5xDSZCXAj7UR7tfRcS/ImJm946I2IJn17/+ogm2pLHMnmxJkiQNW2b+JyK+\nDRwCHB0RDwJf6l5yKyLWpwwFfxXwNJB1p58MLAtckZkfaWZcEbEMMKnH7ol1xyf3OLYoMx9pZgyS\nOotJtiRJkprl/ZQlt94IHAd8JCLmUZLcdSi93E8CR2bmlQARsTWwS3X+lIj4+wBe55OZ+fMe+7p6\nabsfcEYvx7qABT32fQ84cAAxSFJDJtmSJElqisxcBOwdEXsB7wJeBGwMPAXMoVQL/3pm/rvutFUp\nw8yhrLW9Nv1brcG+Wt11Gh0bKJdNkTQsJtmSJElqqsy8ALhggG0vBpYe5utt3MexM4Ezh3N9SRoM\nC591gEmTek5DkiRJkiS1gkl2B5g6dSrnnXdeu8OQJEmSpHHPJLtDTJkypd0hSJIkSdK4Z5ItSZIk\nSVKTWPhMzzF37lwWLlz4nH2zZ89uUzSSJEmSNLaYZOsZc+fOZbPNNhvy+dM2n9zEaDRUV19/T7tD\nUGXZlZZtdwgCNtp1y3aHoMo13/lzu0PQGBcRywEnAHtTlvp6DNg/My0+I2nUMMnWM7p7sM866yym\nTZv2zP7Zs2czc+bMdoUlSZLU7SvAeyhrWT8I3Eov0x8jYlfgu8D6wM2ZuclIBSmps5lkawnTpk1j\n+vTp7Q5DkiSppz0pCfbfgJ0y8/GeDSJiIvBF4NBqV23kwpMkC59JkiRp7Fir2l7YS4L9MuCfwGHA\nPOBKoGvkwpMkk2xJkiSNHd2fXZdIsCuzgI2Bs4AXAlZvlTTiHC4uSZKkYYuIjYAbqy9fROlRPgp4\nGyXxnQDcBPwE+FxmPlF37h+BVwDfAj4CfBXYA5gMbFKdB2XodxdwbEQcW+3bPzO/Xz1/ADgsM39U\nXbfJ71KS+meSLUmSpGao1W1XAH4HvAy4hVKgbGNgGnAMsB2wV49zu88/Dfgv4AbgPuBp4Jrq2DZV\nu7uAO6vn99Vd5yWZeVcz35QkDZZJtiRJkprtf4FVgG0z85/wTEGy/wNmAHtExI6ZeVmP86YBLwB2\nyMyr6/ZPr66xuPr6W5l5fM8XNcGWNBo4J1uSJEnN1AVsC7yuO8EGyMxHgaPr2u3c4LxXAB/pkWBL\n0phiki1JkqRmqgHnZ+atPQ9k5jzK+tZQ1q/u6Wngxy2MTZJaziRbkiRJzXZlH8ceqrbLNzh2a2Y+\n0oJ4JGnEmGRLkiSp2R7o41j3vOpG61cvaEEskjSiLHymAZs9u/FSk9OnTx/hSCRJ0ihX679JQ/Zi\nSxrzTLLVr0mTJgEwc+bMhsdrtaH+HZUkSZKk8cUkW/2aOnUqc+bMYeHChe0ORZIkSZJGNZNsDcjU\nqVPbHYIkSZIkjXoWPpMkSZIkqUnsyZYkSdJoMKwiLxGxPDCxx+5lq+1SETG5x7FHM/Ox4bymJDVi\nT7YkSZJGWqPluxrtG4wPUZYAq3/sWx2b0uDYB4f5epLUkD3ZkiRJapYaA+uRbtRuoOf2d91WtJWk\nATPJliRJ0rBl5jxg6QG027jBvl0G+Bq9jsLMzOOA4wZyHUlqJYeLS5IkSZLUJCbZkiRJkiQ1iUm2\nJEmSJElNYpItSZIkSVKTmGRLkiRJktQkVhdX02z5zp3aHYKAR770m3aHoMo609dtdwgClpu8crtD\nUGW9HdZpdwiSJLWcSbYkSZLGhYjYFfgusD5wc2Zu0uaQJHUgk+wOM3v27CEdkyRJGq0iYiLwReDQ\naletjeFI6nAm2R1i0qRJAMycObPNkUiSJDVPRLwMOBPYBLgZuBt4UTtjktTZTLI7xNSpU5kzZw4L\nFy7stc3s2bNNwiVJ0lgzC1gZOAs4AjgZk2xJbWSS3UGmTp3a7hAkSdI4FxH7ADOB6cDqwOPAHcDf\ngNMz8/d1bXcBur/eErgH+BTwOmAdYBFwLfDNzPxhLy/5AHBYZv6oumaz35IkDYpLeEmSJKkpIuKH\nwA+BPYCVeHb49kbAvsCFEfHVulMWV9ta1eZvwGHAisA8YCKwE3B2RHyhl5d9SXeCLUmjgUm2JEmS\nhi0i3gjsQ0mcjwBWy8zNM3MqpUf7uKrpeyPilQ0ucSJwKzA1M9fKzGnAekD32pQfqOZfP0dm3tXk\ntyJJw+JwcUmSJDXDq6rt9Zn5rfoDmfkwcHxErEdJuFfpcW4XJaHeOTPvrDvv3oh4GzCf0qt9OPDn\nFsUvSU1hki1JkqRm6Kq2q0bEMpn5RM8GmXlIL+fWgAvqE+y6c+6PiF8Db+bZRF6SRi2Hi0uSJKkZ\nfldt1wf+GhEzImK5QZx/eR/Hrq22a0bESkOKTpJGiEm2JEmShi0zLwC+QumV3gY4F7g/Iv4YEZ+K\niK36ucStfRyrn3e9xvAilaTWcri4ljB//vxBtZ8+fXqLIpEkSWNJZh4dEecA/0NZhmsFSnXwVwDH\nRsSfgSMy858NTn+0j0vXDz1ftlnxSlIrmGRrCTNmzBhU+1qt1qJIJEnSWJOZVwJvjYgJlOT6NZSE\neyvgZcClEfHizPx3j1P7Sp7rjz3WzHglqdlMstXQWWedxbRp09odhiRJGqMy80ng99XjoxGxK2UI\n+YrAJ4CZPU5Zp4/LrV333CW7JI1qJtlqaNq0aQ4DlyRJgxIRS2fm042OZeaFEXEmZQ3trRs02QH4\nVoP9AC+stvMzs69h5ZLUdhY+kyRJ0rBExCoRcQXwcETs2EfTCdV2YY/9XcDeETG5wbVXB3ajFFS7\nsBnxSlIr2ZMtSZKkYcnMByLiccrc6Z9ExEHAhZlZA4iI5wH7Au+iJMs/6XGJGnA/cH5EvCMzb67O\nWxv4AbA8sBj4Rv1JEbE8MLHHtbrnby/VIGl/NDOd0y2ppUyyJUmS1AzvoqyVvSHwa2BhRNxB6aVe\nl1JpvAbMAr7W4PyPAp8H/hMR8ygVxTehfF6tAR/PzH/0OOdDwDG9xDMFWNBj37HA8YOkT8w9AAAg\nAElEQVR6V5I0SA4XlyRJ0rBl5o2UudZHAH8AFgEbU5Lde4HzgP/KzDf1Mm/7DmA6cCIlwV6fMqz8\nIuCNmfn5Xl66NsiHJLWUPdmSJElqisx8mFK8rLcCZv2dfx/wweoxkPbHAccN5bUkqVXsyZYkSZIk\nqUlMsiVJkiRJahKTbEmSJEmSmsQkW5IkSZKkJjHJliRJUjtZ9VvSuGJ1cTXNXz77y3aHICAfWNTu\nEFR5xate1O4QBNx20RXtDkGV1aZNaXcIGmUy82Jg6Va/TkSsRlkabHdgMmVpsNdm5pWtfu3hioib\nKcugfS8zD+z0OKSxwCRbkiRJ493ZwG6UHvN7gbvoMaIzIrajrPG9E7BetfsO4C/AaZl5yYhF+1yj\npad/tMQhjXom2ZIkSRq3ImIZYFdKgjgLmJGZtR5tjgc+DnQBi4FbKJ+TNwI2Bt4eEV/KzA+NYOiS\nxijnZEuSJGk8W5OSPAP8okGCfQjwierLc4D1M3PjzNwA2AT4bXXsAxHxlpEIWNLYZpItSZKk8ax+\n5Obj9QciYmng2OrLS4GZmXln9/HMnAfsA3TvO6x1YUoaLxwuLkmSpGGLiI2AG6svXwr8G/gk8CZg\nXeAe4PfAJzLz1uqcA4BDgACWqc75WmZ+r5fX2AU4FHgxsDbwGDAf+BVwUmbeVdd2Q+CmutO7gO9F\nRPe1d6bMuX4KeAL4bs9eboDMfCgifgO8C9iml7i2ocznfjmwPrAscD9wDXBmZp7d6LyhiIiDgAMp\n37OJlPd/PvDZzHxoGNfdBDgGeBWwBnAfcBnw5cz883DjljqJSbaeMWnSpHaHIEmSxq5a3XZl4GJg\nc+A/wCOURPudwMsjYmvgK8DBwG3VY1NgW+D0iFg2M0+tv3hEfB04vLr+k5TkclVgS2Ar4NCImJGZ\nf6hOeYKS5C4DvKA67xZK8lgDHs7MucAGA3hvT1bbZXseiIhDga9TRog+DdxK6THfEHgNsGtE7JmZ\n+w3gdfoUEd+mfM8eAG6n3GiYCnwIeGNEvCwz7xvCdXeg3ABZkfK9uY2qAjvwhojYf7ixS53E4eJ6\nxtSpUznvvPPaHYYkSRr7jgUWABtk5gsyc3Xg3dWxjYEfA/8F7JaZG2TmlpSK3tdVbT5Sf7GI+DAl\nwV5MKVC2amZulplrAC8ErgBWAs6LiHUAMvOOzJwO7FF3qU9l5vTM3C4zrx7E+9m+2v6rR1ybAidT\neskvqd7vxpm5OeUGwDeqpvtExDsG8XqNvBp4O/A2YHL1PVuDZ288bAacMNiLVkPm/w9YgfIze1lm\nTsnMLYDVgM8A3wRWGWb8UscwydZzTJniGqaSJGnYNgPenJl3d+/IzNOBOZSE9HXAUZn5u7rj9/Js\nkjglItYFiIiVgY9REsnPZeYJmflY3XnXUXpcFwCTgA83841ExB6UYeI1So91vb0o63zXgCN7zOde\nlJn/Ddxc7XrrMMLoogxD/1hm/rB7WHtm1qoe/7OqNjMjYuIgr/1ays8L4H2ZeVnde3g8M48DLqCM\nTpA0ACbZkiRJarafZeaDDfZfX20fpVTy7unauufrVtu9KMnzYuDERi9WvVZ3orn3UAJuJCKmAqdT\nkugrMvPMHq97ImUI+ZTMvLbBJQD+VsW1yTDDebKKpZEfVttlKet8D8bu1XYRcG4vbb45yGtKHc0k\nW5IkSc32z172L6y2mZlP9XEcYPlq+9Jqe1dm3t/Ha15ebdeLiNUHFmbvImJb4I+UIdnzgBmN2mXm\nU5l5ex+XerjaLjeMcGrAvzJzYS/Hr6t7vvkgr71Ftc3MfKKXNldR5ptLGgALn0mSJKnZeqty3V0c\nrbdksb66d/fa1mtX23UjYvEAX399SjXzIYmI11N6h1egVEx/bf1Q8B5tl6dU+34DpXjbmpSe92a7\npY9jd9c9nzzI665J+b7f1VuDzHwiIu4fwrWljmSSrYbmz58/4LbTp09vYSSSJGkMWmIprEEer7dC\ntX0MyKGFM3AR8d/AlykjPq8C9qyfW96j7frA7yhzmmvV43ZKz3d3T/0USgGxnud+B9iu0XUzc9sG\nux9rsK9bfQ/0Mz3mEXENjb/XV2VmdyG67hEDjzdoV29RP8clVUyy1dCMGQ1HRDVUqw3m76QkSdKg\ndA+3XlBVC2+ZiPgaZb3rGqUn+8DM7Cv5/B4lwX4a+ARwRs+EPCLOoKyx3dPzga0b7O/tg9USy4fV\nqR+K/mjd8616aV+/zFd38rxMH9eHsia3pAEwyVavzjrrLKZNm9buMCRJUme7tdqu08oXiYiTeDbB\nPjYzP91P+3WAV1XtT83Mz/fSdNVGOzNzl0GG2Nf7X7vu+YK611h6ANe9lzI0f43eGkTEipTeeHtW\npAEwyVavpk2b5lBwSZLUbldU2+dFxPaZeVWjRhHxvF6KqfUrIo4BjqJUMD+oZxXxXtSve3ppL9dd\nAXjFUGLqoQvYIiKW7aVnfZu659c3ON6X2ZSK5BERS2dmowJnOw7ymlJHs7q4JEmSRrP/BzxSPf9I\nH+1OjYibI+J/BnPxiHgN8ClKL+17B5hgw3PnSPfWC/wFnl1fejjVxaEM135nL8feVm0XAn8e5HV/\nX21XAPbspc3hg7ym1NFMsiVJkjRqVctW/S/VGtgR8Y2IWKX7eERMjogTgQOADYCbBnrtiFgK+Eb1\n5bmZ+a1BhHY9ZW5zF/D+iHhmHeyI2Dgizgb+Czip2r1mRGw6iOvXqwE3AF+KiDfWxx8RRwNvrtqc\nlplPDvLasyhD8ruAr0fEM8XYImL5iDgBeDXQsLq6pCU5XFySJEmj3Rcow7MPBQ4DDo6Imyi9w+tS\nPtPWgM9m5rmDuO4bgKnV8x0i4u8DOOegzLw6M5+OiI8DpwAbAVnFtEwV6wPAXpQq40dTkti/R8Qt\nmfmCQcTY7TfA0sB5EXEfJeldF1iF8t6vBo4b7EUzc1FEHAD8nDLv+8qImE8poLYx5Xt7EPB+njv3\nW1Iv7MmWJElSs3QvYzWcNkscz8xaZh4B7AKcQ1kmawplmPY8SpXvl2TmJwf5mqvWHZtCqfbd12Mr\nYMW6uE4F3gL8hZKUblgd+jqwXWb+KTMvAz5JSYqXovc1xHvT/Xn9sep7cBDwb0pCvGz1/HjgFZn5\ncONL9C0zfw9sD/wIuIOSTK9KSex3zczvV++vhvmD1K+uFiy/ZNXBMezqq69mu+3KKKG//e1vAy18\n1gXwiw229mc/CuQDLmM5Whx6+sHtDkHAbRdd0X8jjYgV1uu1eLFG2Lqf+GZXu2OQpPHKO1GSJEmS\nJDWJSbYkSZIkSU1iki1JkiRJUpOYZEuSJEmS1CQm2XqOSZMmtTsESZIkSRqzXCdbzzF16lTOO+88\nZsyYMehzN3vDZi2ISIM161tXtTsEVWpPPdHuEASst/N27Q5BlWtO+VW7Q1Bl3U+0O4LmiYgXUtbR\nfjEwEbg3M9dpb1SSOplJtpYwZcqUdocgSZLUr4hYEfgdMJmyjOw8oOEdzoiYABwLfAhYGjg2M48f\n4uueAhxafblzZl7SS7tVgcOBPYDNKWtsPwzMoaxB/c3MvHMoMUgavRwuLkmSpLHqJTybYH8wMzfJ\nzM17Nqp6u68CPkr5/Fsb6gtGxKuBQ6pr9HqdiHglMBf4NKWXvQbcQOnk2gH4JPCviHjVUGORNDqZ\nZEuSJGmsWqvu+ayeByNiqYj4KHA5sCXwY6BrqC9W9Zx/B3i6r+tExHrA+cCqwPXATpk5OTNfkJkr\nATOABcAqwI8jYuWhxiRp9DHJliRJ0li1dN3zxxsc3wP4THXsgMzcd5ivdyIwBfheP+2OBFauXnf3\nzPxL/cHMvAA4qvpyVeDNw4xL0ijinGxJkiQNW0TcTElAB2LnzLwkInYGLqIMpV4LeBL4ACXp3LDa\nPwc4E/haZtaq13oXcEbd9bqAmyMCgMzs7khaCrgMeFtm3jy0d1ZExG7AwcB9lKHeB/fR/D5KT/aN\nmXlbL21+W/f8+cOJTdLoYpItSZKkZriOklz2Zl1gTUri/GS1r35O81qU4dybAzcC84FNgG2rx1RK\nDzHV61wDrEZJ7GvAbErRs/pr/gmYlZmLh/qmACJiJcow8RpwVGbe1Z3QN5KZXxjAZeuHm98+nPgk\njS4m2ZIkSRq2zHxDb8ciYl3gakqS+rPM/GuDZt8C7gE2zcybqvNWo8y1fglwaER8LjNvz8xZwKwe\nPdqvy8xbesR073DfV+UrwPrA+Zn5gyZd84PV9mngwiZdU9IoYJItSZKklomI5wE/pfRizwYO6KXp\nxsDmmflw947MvC8iPgb8gTL0+xXAD3s5f8gFzfoSEa8H9qfcADi079b9XmtlYAtKdfJ3AE8Bh2fm\nnGGGKWkUMcmWJElSK30V2BFYCOxdn0T38L1ejl1T93z9ZgfXlyopPo3SA39kZt49xOucxLOFzgDu\nBb4OfCsz/zXsQCWNKlYXlyRJUktExEzgMEqSemBmZh/Nr+xl/0N1z5dvVmwDdDKwDnBuZv5oGNe5\njXKzYB6l4vhqwFuBd0fEqsOOUtKoYk+2ejV79uwBtZs+fXqLI5EkSWNNRGxNmWddA76cmef2c8oD\njXZmZq2uyFhLhoQ3EhF7UoZ0LwAOH861MvNLwJeq604AXkdZDuwoYI+IeGlm3jO8iCWNFibZWsKk\nSZMAmDlz5oDa12q1/htJkqSOUVXjPpfS8/xH4CMDOG3UfKCoepdPpcR0eDMT4Mx8ErggIq6iVGR/\nPvBFep+rLmmMMcnWEqZOncqcOXNYuHBhu0ORJElj0/9RksdbgH2Hu4RWG3ydsqTYWQPogR+SzLw9\nIn4GHAjMwCRbGjdMstXQ1KlT2x2CJEkag6pq4HtS1qx+S2YuaHNIQ7EfpRf7HRHxjj7adQF/7B7O\nnplLVcPBXwKsDVyfmdf3cf5d1XZSRCyXmYuGH7qkdjPJliRJUlNExGuA4ykJ6lGZeUWbQxqqhvPD\ne1iF8j4foSzF1T3cvQv4LbAM8D1KT3VvNqi2j5hgS+OHSbYkSZKGLSI2AM6mJJnfz8xT2xzSkGXm\nav21iYjuIfB7ZOaldec+ERG/ofTmvzkiPp2ZNzU4f82qTQ24qDmRSxoNTLIlSZI0LNUQ6Z8CkylL\nVR3a3oiKiFgFWLpuV3118okRMbnHKfdl5mALsDWqeH4csDswCfh1RByemb+vi2sn4BRgZeBp4HOD\nfE1Jo5hJtiRJkobrLcAO1fPVgcvqlt1q5PbM3KPlUZWEf0qD/V3Ah6pHvY2A+cN90cy8OiL2A84A\nNgUujIgFwD2UIeIrUnqwHwcOy8zLh/uakkYPk2xJkiQN1wSenZO8fvXoy0p1z2sMb/muvs5fPIhr\nDyWGXs/JzJ9FxF+Aw4DXUpLtqcCjlOT/D8A3M/OGIbyupFGsqwVrHI+aNQ41YroA5h72X/7sR4Ev\nf+uqdoegypfPPrjdIQjoWmrp/htpRFxzyq/aHYIqL734kkZDnCVJTbBUuwOQJEmSJGm8MMmWJEmS\nJKlJTLIlSZIkSWoSk2xJkiRJkpqkFYXPJEmSJEnqSPZkS5IkSZLUJCbZkiRJkiQ1iUm2JEmSJElN\nYpItSZIkSVKTmGRLkiRJktQkJtmSJEmSJDWJSbYkSZIkSU1iki1JkiRJUpOYZEuSJEmS1CQm2ZIk\nSZIkNYlJtiRJkiRJTWKSLUmSJElSk5hkS5IkSZLUJCbZkiRJkiQ1iUm2JEmSJElNYpItSZIkSVKT\nmGRLkiRJktQkJtmSJEmSJDWJSbYkSZIkSU1iki1JkiRJUpOYZEuSJEmS1CQm2ZIkSZIkNcnz2h2A\nJEnSaBQRrwXmZuaNDY7tDSwGZmXm0yMenCRp1LInW5KkMSwi1o2Iye2OYzyJiOdHxNXAL4Ede2n2\nOuBc4O8R8fwRC06SNOrZk62OFhHbAHsD04H1gXmZ+ca649OBf2fmo20KUQ1ERFdm1todx3gSEYuB\nGzNz03bHokG7EHgKeGG7AxkPImIl4CJgg2pXbzcwFgFdwJbAhRGxTWY+NAIhSpJGOZNsdaSIWBH4\nHjCj2tVVbRf3aHoysHFEzMjMK0YovHEpIpo5nLKGv7+a7X7gyXYHoYGLiGWAnYBN2h3LOPN+SoJ9\nO3AEpTd7CZl5ZEScA3wf2Bg4Cvj0SAUpSRq9/JCqjhMRXcCvgZdQkut7gKuA3Xu0mwBMBdYAZkXE\n5pl5/wiHO5509d9EbXQx8IaI2KTR/FONnIhYFvgQ8FZKAr3sAE6b39KgOsubKDfy9u7v5mpm/iUi\n9gWuAN6MSbYkCZNsdab9gZcCDwGHAT/MzFo1XPYZmflkRGwO/J4yDPMI4H9HONbx5Lhe9i8FHEAZ\nrn8DcDlwB2Uo5kRgQ8qcyPWAucA3gUdaHWwHei+wDvCbiHhPZv6h3QF1oohYGvgt8HIGfmPqKeD4\nlgXVeaYCdw109FJmXhURdwBOtZAkASbZ6kz7UXop3p+Z5/TVMDPvj4hDKInfnphkD1lmLpFkR8RS\nwCxgArBrZv6+t/MjYi/gu5RiQ69vVZwdbCZwPvBa4HcRcTcwG1hA3zc1apl50AjE1ykOoQwBB7gE\nuBJ4GDgGuBM4FViVMvJmM8rNq9Mz85aRD3XcqgEPDvKch4GVWhCLJGkMMslWJ9oKeIIyJ7tfmXll\nRNxD+UCr5jqSkizskpmX9NUwMy+IiLdThvq/H/jSCMTXSU6gJBdQelDXAtbs55yu6hyT7OZ5K+V7\n+vHMPKF7Z0QcA9zZfbOqmvbyQUryfR1gkt08twIbRcSkzFzYX+OIWIcyrP8/LY9MkjQmuISXOtFq\nlCrig6lOvYAydFnNNRO4u78Eu1tm/pYyh36/lkbVmebXPeZVj/n9PLrbqHm2BB4DvtBXo8ysZeYX\ngNOAsyJiq5EIrkP8ijIP/ivVzYxeRcQk4P8on6d+OwKxSZLGAHuy1YkWUnrpBqT6kLU2gx8+qP5t\nSuk1Goy7AdekbbLM3KjdMQgoQ47nZmbPlQ6gTKvo6bOU+fRHU2obaPhOodTr2B94UUScBVxLudn6\nFLAKpSDmiyg3Cteg3Bg5sR3BSpJGH3uy1YmuA1aKiJ0H2H5vyhzI61sWUeeaAGwQEQO64VcVhVoP\nbxBq/HoMWLHB/gcoo3CeIzPvpowo2K3FcXWMzLwBeAdlSbstKDcyZgGXUVai+B1wDmXaypqUxPsd\nmXlzO+KVJI0+JtnqRD+jzCU9PSKm9dUwIt4AfIcyR/LcEYit09wATAI+NsD2RwMrAze3KiCpzW4C\n1ouInqM17gbWjogpDc7povSmqkky86fA9pRigE9Svsc9H08C5wHTM/O8NoUqSRqF7A1SJzoNOBTY\nHPh7RPyKZ3up14iIYynLSe0ITKN8mPo38O2RD3XcOwfYGjgmIl4GnAX8A7gLeBxYhpI8bAXsS6nw\nXgN+0pZoO0REvIkygmM65f/C3Mzcoe74a4HrM3OwQ/3Vvz9Q/r3PiogPA7/MzKcpv4OmAkcBH+hu\nHBHbUpa5czpLk2XmdcDeETGRsozjesBywKOUQnPXZ+ajbQxRkjRKddVqg6n9JI0PEbEhcAHlw2xv\n/wm6C95cC7zRoYDNFxETgAuBV9D7z6FeF/B34OWZ+VgrY+tEVZXk8yk9ePDs/4FrMnN6Xbu/AQHs\nm5k/H9kox7eqB/s6SuGtxcD6mXlnRLyTsiJCjVJh/xLKDagDKaM7fpeZr21L0CIijgDuz8yz2x2L\nJKn9HC6ujpSZ8yiJxLuBPwKLeO4wwIeBi4CDge1NsFsjM5+krMt8DGU4bKMhmd2Ph4AvAzubYDdf\nRCxD+Te/A+X7fQ3wrQbtJgDLU6rtnxMR649knONdZv6HsozXQ9Wuu6vtD4DLKT+b3SnzhN9PKcK1\nGPjcyEaqHo7GZQUlSRV7sqVKRKwMrAA8nJkP9ddezVVVcd+WUmhoTUoitwi4F5gNXJWZT7UvwvEt\nIo4CTgLuAN6SmX+p9i9myZ7sCcAvgFcDX8rMD7ch5HEtIlYCds3Mc3vsO5GyhN3y1e5rgY9m5i9H\nPsrxrfqdtB1lDezlemm2HLAT8Hbg0cxsVLROktRhTLIlSUTEpcBLgTdl5qy6/Usk2dX+aZRaBv/M\nzG1GNNgOFxFLUS0b5Q3B1oiIHYHvM/DlAruAP2fmTq2LSpI0Vlj4TNKoEhFrUdYlX4EybP/2zLyn\nvVF1hKAkbbP6bQlk5uyIuAvYuLVhqadqDe272h3HeFXV7PgNZSm1rn6aQymE9ifgyFbGJUkaO0yy\n1ZEiYmPgw8DOlOrJy/d5QlHLTP/PtEBEbAR8EJgBrNXg+HzgbOAkE+6WWRm4cZDn3EeDtZulMe5o\nytKCtwKfAq4EHqH8//gXsAewarX9APA14PiqCrwkSSbZ6jwRsTnwV2AlBtZLoRaKiD0pCfREev95\nbAh8BDggIvbMzL+NVHwd5AHKEkUDEhHPo9yguq9lEY1zEXEjZRmoPXvsG4paZg50aLP69hpKFfe3\nZuZfu3dGBMATVeHMecA1EfFjSqX3oCwzKEmSSbY60qcovXYAF1OqKD9EqdCrERQRGwA/4tm1Z2cB\nV1GKby2ijDBYD3gx8HrKMPKfR8Tmmem6wM11DfCaiJiRmecNoP2BlN6+P7c2rHFtI8rNjZ77hsIC\nK82zIbCgPsHuTWbOjYj3Az+IiFmZ+YPWhydJGu1MstWJdqZ8IJ2Zmee0OZZO9z5Kgn05sHdm3tFb\nw2pI+QWU6uPvBT4zEgF2kHOAXYHTIuKhzPx9o0ZVD/ahlOXUasAPRy7Ecef7lB7Reme2IxA9xwTK\nqgY9PU2Zp93TT4FvA++hLLUmSepwVhdXx4mIx4H7MnOddsfS6SLiH8CWQGTmDQNovxXwD+CKzNyx\n1fF1kohYmrJm/MsoyfO1wHXA24AFwK8ow8OnU0aCdAF/AXbKTP+QaNyoCvpNACbX/9uOiNsp01pW\n7flvPiL+BayXmSsjSep4S7U7AKkN7qkear8pwPyBJNgAmXktcDswtaVRdaCqaNNelKrKXcDWlPWY\na8DqwDuAXYBVquO/BvYywdY4dB3lRtLbe+y/kzJF4mX1O6v1tNei97W0JUkdxiRbnegfwPpVz53a\nayKlau9g3E9Z3ktNlpn3Z+brKIWfzgDmUOoVPA08SFkX+3TgNZn5+sy06JnGo/MoN5JOj4hzI2KV\nav9l1f6TIqJ+JNQnKdXGe53uIknqLM7JVic6kdJbdwDwnTbH0unuBTaIiKWqtX/7FBFLARtQEm21\nSGZeBFzU7jjGu4g4vYmXq2XmQU28Xif7NvBuYCvgTZQifwCnAYdQpkzcXFWCn1w9asAvRz5USdJo\nZJKtjpOZv4uI/6b0RqwEfCszH213XB3qauB1wP6UHtL+HExZeu1PLYxp3IuIvwKPZuar6/YtsZyU\nWm5/mlMVvKu6jkl2E2TmExHxKkpxvz27VzLIzL9HxMeBz1LmbEfdaXMpK1dIkmSSrc4TEV+nTJW4\nHvgi8JmI+DdlKZ2+PvDW6pMSNcU5lKW5TomINYFTM3OJXuqIWB04HPgY5Wd01ohGOf5MZ8mq1hux\n5HJSaq35uPTWqJSZ9wL7V6Nn6vefEBF/AmZSlvp6DLgUOC0zHx75SCVJo5HVxdVxImIxz36w7RrA\nKbWqXS0zncfdRFXBoN/z7LJqT1N6hG6lfHidSBkevinlxkgX8OvMfH074h0vImIhZQ3yfYBZmflk\n9f/imsyc3t7oJEmSxjaTbHWciLgY6Hf+byOZuUuTw+l4ETGJMtdxn7rd9b+Yuur2nQ4c5fD+4al6\n4l7SpMvVMtNRUZIkSRWTbEmjQrUG9j7A9sA6lArijwC3AVcA52TmnPZFOH5ExC6UIk3LNuFyjvDQ\nuBUR6wLrUkZ+9DvyKTMvaXlQkqRRzyRbkjpQRGwI7EZZ33dp4BjKOsCnDvZamXlcc6PrXBFxAGXa\nxGAtBhYCNwPXDqRav3oXEYcAHwGmDOI0R3VIkgCTbAmAiFgLWJvSe/owcHtm3tPeqNSbiHhnZn6/\n3XGMJ87JHh161IwYqnsoSxV+ITP9Iz9IEbE/A1vtoCdHdUiSAKuLq4NFxEbAB4EZlN68nsfnA2cD\nJ5lwN1f1vf8QpeDZasDtwK+AE6uqvr2dtwFlDdvdAJPsIYqIdYCuzLy9bveZQLYpJD3XQAoy9mUN\nyjJT04G3Dj+cjnNktb0OOAG4ljJKwBsWkqQBsSdbHSki9qQk0BPp+wNtDbiLslbq30YitvEuIl4D\nnM+Scxy7v9e7ZeZ1Dc47jPKBd0UAe4yGLiLuBhZk5hZ1+xYDf8/M7doXmSKie/3lbwPbUpa5+zXl\nBsjDlN9ZAewO7AdcDhxNWX5tReAFwLso68/XgLdk5s9G9l2MbRHxCGUKxXp93fSTJKk39mSr41S9\noT8ClgMeBWYBVwF3AIsoyd96wIspazivDfw8IjbPzAfbEvQ4ERGTKUnDROAJ4BLgbmATyvd7beD8\niNgiMx+vztkU+A6wEyUpfxo4eeSjH1dWoXz/expuD6qGbxJwXvV868yc26DNtcBPI+IEygiQU4Gd\nM/NmSu/rjyPi85SROu8CTLIH5wngNhNsSdJQmWSrE72PkmBfDuydmXf01rAa1nwBsAXwXuAzIxHg\nOHYQMBm4idJj/Z/uAxGxI+V7vTHw9og4A/gf4FjKz6sLuBI4JDOvGeG4x5uHgHUi4mTKTabHq/0r\nRsQrBnsxKyo31YcoN5127CXBfkZm3hARbwcuAw4Hvlx3+FjKsOftWxTneJaUG62SJA2Jw8XVcSLi\nH8CWQGTmDQNovxXwD+CKzNyx1fGNZxFxEfBKYK/M/EWD4/sBPwD+SrkJuD0luX4I+DhwioWchi8i\nzqXUIui5HvlQvrdWVG6iiJgDPC8zNxnEOTcD9/Yc6h8R1wPPz8zlmhrkOFdVeP8u5Ubg79odjyRp\n7Fmq3QFIbTAFmD+QBBsgM6+lFOaa2tKoOsMWwFOUIa6N/BR4DNgR2IGS+P0EmLgfI60AABHaSURB\nVJaZ3zDBbpoPAP+mfH+7H/T4ejAPNc/6DH4Jr8eB5zfYv+oQrtXxMvMMyhSVH0fEOyLCm0iSpEHx\nD4c60UTgtkGecz+wegti6TSrUOY6NlzDNzOfjIh5wOaUIeVHZOavRzLATlDN3X1BREylVNZ/HnAR\ncAPw7jaGplLFeuOI2Dgzb+qvcbX84CbAIz32b0+pcTC7JVGOf+8D1gW+B5wSEUn/FcZrmfnqEYhN\nkjTKmWSrE90LbBARS/WW7NWLiKWADSiJtoZnAqUKcl8ep3yQfUF38TO1RjXndy5ARAA8nJkXtzUo\nXQbsSSls9ta+RtxExJqUQoJLUwqede9/GWWJuxrw+9aGO/5ExPrAHyn1IbqAFSjLofXHkTaSJMAk\nW53pasryNvsDpw+g/cHASsCfWhiTejDBHnG7UJaIUnudBLwB2AaYHRF/ofzOupNyA2oCZR3sLSnr\nzC9LSe5OA4iIlYFLq32PVdfT4PwvZXQAwBzgelwnW5I0CCbZ6kTnUJbmOqXqCTo1M5fopY6I1SkV\nez9G+XB11ohGKY2gRj3YEbESJaFblpKw3ZWZj450bJ0kM/8YEf8NfIXSQ/3y6tFI93z4b2bmmdX5\nD1broC8LvG0gQ861hFdTfue/OzMHciNWkqTnMMlWJzqbspTUzpQluY6PiLnArZREYiJlePimlOKA\nXcCvM/OHbYlWGkHVUOPDgVcBazY4Ph+4EPhGZv5jhMPrCJn5jYj4LXAEsBulqNmEuiY1yu+rS4Dv\nNLhB8m7gz5l530jEOw6tDtxvgi1JGiqTbHWczKxFxBspwyv3ofw/mEYpttWtu4eoRlnK5agRDVJq\ng4j4InB09WVvVcM3pNykOjAiPp6Znx+R4DpMNV/+ffBMXYhVgeWBJ4AHMvOJPs6dNSJBjl+38+za\n8ZIkDZpJtjpSZi4E9o2Iz1AS7e2BdSgFbh6hVB+/AjgnM+e0LVBphETEvpSlvaD8H/g1cC1wFyWx\nW5ZSrfqFlN7V5YHPRsT1mfnzkY+4c1QFGu9tdxwd5HzgvRGxZmbe3e5gJEljT1etZh0PSSMjIhZT\nhuRnX82A5YC+hiLXMnO7ZsbW6SLiYmAn4DeUuby9VtOv6hX8iFIs7aLMfM3IRNlZImIC5Xu8LeUG\nxwqU4nS3A1cClw5khQQNTkSsAlwMLADenJkPtjkkSdIYY5KtjhIR2wEbZubP+mk3gVJ5/Phq2Kaa\noEqya/Q+FHmgapm5dBNCUiUi7qPUI1gvM/vtNY2ItYH5wMLMnNzq+DpJRHQBHwQ+TFlbvje3AR/L\nTIsyNlFE7AisDBwLbAb8DPgn8BD9VBjPzO+3Oj5J0ujncHF1jIg4CvgiZTmWPpNs4EDg7cCeEbFP\nZv621fF1iEtwGZzRajngpoEk2ACZeWdE3Ais39qwOtIZwDt47s2op4BFlGH63TeY1gfOjIhNMvP4\nkQ1xXPsLz/09deAAz6tR1ieXJHU4k2x1hIjYj2fXi90yItbKzLv6OOURyrqoKwHnR8RLrKQ8fJm5\nc7tjUK/uAFYb5DnLUKpcq0kiYnfgndWX5wOnAlfV3/yolh58Mc9WHz8mIn6emVePdLzj2HBH20iS\nOphJtsa9iJgInFx9eR0ws58Em8w8q5qjehFl+ZzTAecAazz7FXBY1St6Y3+NI2IjSqXxE1sdWIc5\nkNIj+rnM/ESjBlUxrlnArIg4GXgvJeE+aMSiHN82wRE3kqRhcE62xr2IeDelN+hOYKuBDoetzt0Q\nuIrSw/eGzPxVa6KU2qvqHf0bpWd6Rmbe2UfbyZRe1g2A6a7H3DwRMQ9YA1gtMxcNoP0kSuXxeZk5\ntdXxSZKk/tmTrU7wOkqvxJcGk2ADZOa8iDgR+AzwFkpvnzQerQ4cQumZnhsRFwCXU5LuRylLeK0D\n7ADsDdxPKcy1ZUQ0vGBmXtL6sMedNShz4/tNsKEsR+jceEmSRheTbHWCrYHFwGlDPP+bwKcpcyCl\n8eo6nh0i2wXsWz0a6aLUKzi7j+vV8G/MUDxNmes+GM4fbpGI2AGYQe/LqJ2TmXPaF6EkaTTyA5A6\nwdrA/MxcOJSTM/OBaginPUUa77p6ed5fWzXP7cAmEbFGZi7or3E1zP/5wH9aHlkHqdaCPxPYvW53\n/b/5F1JGSX0qIs4AjszMx0YwREnSKGaSrU4wgTLcdTgWUYbLSuPVqykjPtReFwNTga9FxNsys9ef\nSUQ8jzLSpgv448iEN/5FxATgN8A2lO/tk8AcSgX+7mXU1qP8nJYGDgDWBPZqR7ySpNHHJFud4D7K\nPMfhWLu6jjQuZeYf2h2DgLISwoGUGhBbRsS3gb9S5sY/BkykFJx7GfAeSqL3NPCVtkQ7Ph1IGR7+\nGPBR4IxGI6EiYlXgUOAYYI+ImJGZ541opJKkUWmpdgcgjYBbgTWqSuGDFhFbAasCtzU1KknqITOv\nAz5A6UGdBpwEXEb5PXYvcAvwF+CLwGaUue9HZObstgQ8Pr2Z8n3dPzNP7m2qUWben5mfoyyf1sWz\n65tLkjqcSbY6wcXVdv8hnn8k5QOXPX2SWi4zvwrsAfyDkrz19vgr8KrMHGpRRzW2NfBgZv5kgO3P\noBRDe1HrQpIkjSWuk61xLyJeTPkw+gjw4sz81yDO3QP4f5QPtC/JzCtaE6XUXhHx9BBPrWWmU49a\nJCKeD2xPWT5tBcrvsduAKzJzXjtjG68i4nFgTmZuNYhzrgM2zczlWheZJGms8IORxr3MvDwiLgR2\nBS6MiH0y88/9nRcRB1HmR3YBPzfB1jhntfBRKDP/g5XDR9ojlClCg7ESpSiaJEkm2eoYB1PWNF0H\nuDgiLgDOAv6amXcAREQXpYjQzsAhPFtZ9lZKcRtpPDuun+NdwCqUXtWXApdSKlsPtQdcDUTEjcAt\nmfnKdsfSwW4Eto2ILas58n2KiK0pSzxe3fLIJEljgkm2OkJm3hIRuwM/AzYC3lg9iIjFPFu1t+c6\nwXOAGd2JuDReZWZ/SfYzIuJFlP9Le2Xm21oXVUeazPCXHNTw/BqYDvwgIvbqa1h+NZz/HErdjl+O\nUHySpFHOOdnqKBGxMmVJlkMpw/t6s4DSS/eFzPQDr9RDRLyCsjbzezLzO20OZ9yIiN9RlufaIDPv\naXc8nSgiVqfcYF0ZeAL4Fb0vo7YbsAzlb8a0zHSpR0mSSbY6U0QsSxny+iJgTWAS8CBwB2W5nCsz\n88n2RSiNfhFxG3BbZlpVuUkiYnNKj+jdwNurOdkaYdVNpPMoc7P7+qDUBdwFvNG6HZKkbibZkqQh\niYhrgU0yc4V2xzJeRMQ+lCri7wBeQaklMZvSU/pIX+dm5vEtD7CDRMQ6wAeBtwDrNWgyDzgbODEz\n7x3J2CRJo5tJtiRp0CLiecA9wLKZuXy74xkvqhoR3X+Yu+i7F/U5MnPplgQlImIteiyjlpkL2huV\nJGm0svCZJGlQqukWJ1DqGjicufl6FmBUm2XmXZRh4ZIk9cskW5JERFw0gGZdlMQ6gOUpvayzWhlX\np8nMpdodgyRJGh6TbEkSlPXhawyu5/Qa+l9fWxq1qptLczPzkB77hqKWma9uTmSSpLHMJFuSBHAJ\n/c//rQGLKEsZ/RH4iVX4WyciVgK6MvPBBse2AO5yma9h2xlYpcG+wdxw6m5rkRtJEmCSLUkCMnPn\ndsegIiKWAz4PvBt4F/CTBs3eB7wzIk4H3p+Zi0YwxPHkEsqa2D33mTBLkobMJFuSNCQRMRE4OzPf\n1O5YxouI6KLMc39VtWtyH80nAO8Bnh8Rr81ME8NBanRzyRtOkqThssCKJHWgiDhpmOevA1wK7Nmc\niFTZH3g1sBg4GfhtL+0+DXwCeKJq/66RCE6SJPXPJFuSOtNREfHVoZwYEVsDlwPbNjckUZLlGnBQ\nZr4/M29s1Cgz52fmZ4EDKPOBDxjBGCVJUh8cLi5Jneu9EfG8zDxioCdExOuBc4BJlGTwE60KrkO9\nEHgwM78/kMaZ+cOIOAXYqrVhdZ6IeAnwUmAdYFX6L4RWy8yDWh6YJGnUM8mWpM70UeBzwKERsXRm\nHtrfCRHxXuBEyt+OR4B3ZuZ5rQ2z4ywHNOy97sOdwMYtiKUjRcQylBtJg6k10F1d3CRbkmSSLUmd\nKDM/HxGPU5Lmd1c92gc3alsV4zoJOJKSTNwC7JWZ/xixgDvHXcC6EbFUZi7ur3FELA9MARa0PLLO\ncTQwo+7rx4B7gafbE44kaawxyZakDpWZX4mIRcA3gAMiYmngwPoq1VUF8R8Ce1AS7MuBN2XmXe2I\nuQNcCrwNOAT45gDaHw9MpFQkV3PsR+mVvgB4X2bOa3M8kqQxpqtWc8UPSepkEbE/8B1KEn02ZRh4\nraog/nNgm+rYD4CDM/PxdsU63kXEK4E/UKqLfxs4E7g2Mx+tji8FrA68CDgC2K06dZfMvGTkIx5/\nIuJhYGlgjcx8uN3xSJLGHpNsSRIRsR8loVsa+BFlGPl5wLpVk49n5gltCq+jRMSXKEOW6/9ALwKe\nAlbs0bwL+EJmfmSEwhv3ImIhMD8zt2h3LJKkscklvCRJZOY5wL6URO6twGXAesCjwJtNsEdOZv4P\ncBhwNyWJ7gKWp1R076p7LAAONcFuupsoQ/AlSRoSe7IlSc+IiD2AnwLLUgo+vdQCZ+0REROAVwI7\nUG54LEe56XELcDXwx8y0GFeTRcQngWOBLTNzdpvDkSSNQSbZkqTniIhdgfMpSd1Rmfn1NockjZiI\nWA74c/XlazPznnbGI0kae0yyJUlLiIid+P/t3c+LVWUcx/HPTNAiK4gKWoRgEM8q6ActIhHDXUGr\nEoIasz+lVX+FOeaiklxEIkRJuCgoyTYVX0MoaBFUBiYlRU2Lc8mM0aZ7Dz537n29YBZzZjh8EWTm\nPec5z5OcSHJzkt1VdabzSDC61tqea3zptiSvZNiT4GiST5Jc/K/7VdXb400HwHYlsgGWUGvt0y18\n270ZdrL+Kcm1jjHaqKpHRhtsibTW1pJcqKp3/nVtKlV1ZJTBlkhr7c9cvcHcLDaqytGoADgnG2BJ\nPZghLla28L13TD424y+10zuc5GyGY9L+eW2af9ONJCJ7Olv5PwAAWyayAZbT6QjkebBZ4Im+G2dX\n7wEAWDwiG2AJVdXe3jOQXUl+2+QaN0hVfbPZsn0AmIV3sgGApTV5L/usvQUAGMtq7wEAADqzRB+A\n0VguDgCdtNZOjXSrjaraN9K9AIAZiGwA6Gdvtr7L+/V49wsA5oTIBoD+fkjybpKveg8CAMxGZANA\nP6eSPJHkriTPJTmX5FiSo1V1rudgAMB07C4OAB211nYmeTHJWpL7cmXp90dJDid5o6p+7jLcEpjs\nLn4pyZkZb+W9eACSiGwAmButtT1JDiZ5JsmODMF9OcnxJOtV9V7H8RbSJLJn/WVoJUNk3zTCSABs\ncyIbAOZMa21HkmczBPfuTCIuybdJ1jME9/l+Ey6OSWT/mmGp/kyq6qHZJwJguxPZADDHWmu7MsT2\nWpKdufLU9cMkh5Icq6pLncbb9iaR/VlVPdx7FgAWg8gGgG2itbYvyYEkTye5PUNw/5LkeFUd6Dnb\ndiWyARjbau8BAICtqar3q2otyaNJXs8Q2TuSPN91MADgb47wAoBtoLV2d4Yl4y8keWByeSXJd0mO\n9JoLALiayAaAOdVaW03yZJKXkjyV4ef2SpLfk5zI8E72yar6o9uQAMBVRDYAzJnW2v0ZwnotyT0Z\nwjpJPk/yapLXqur7TuMBANchsgFgDrTWbkmyP0NcPz65vJLkYob3rw9V1cedxltkB5Nc6D0EAIvD\n7uIA0FFr7bEMYb0/ya25cib2BxmWg79VVZe7DQgA/C+eZANAJ621L5K0yacrSb5M8maS9ar6utdc\nAMD0PMkGgE4mZzRvJPkxyckk56e9V1W9PNZcAMD0PMkGgP7uzOxnXYtsAJgDIhsA+jmd4Uk2ALAg\nLBcHAACAkaz2HgAAAAAWhcgGAACAkYhsAAAAGInIBgAAgJGIbAAAABiJyAYAAICRiGwAAAAYicgG\nAACAkYhsAAAAGInIBgAAgJGIbAAAABiJyAYAAICRiGwAAAAYicgGAACAkYhsAAAAGInIBgAAgJGI\nbAAAABjJX6I2lEkuCbVLAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<matplotlib.figure.Figure at 0x7fe5c2b3f7b8>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "threshold = 2.5\n",
    "coeff_std = np.std(cellType_coefficients_frame.values.flatten())\n",
    "coeff_mean = np.mean(cellType_coefficients_frame.values.flatten())\n",
    "data = cellType_coefficients_frame[cellType_coefficients_frame.index.isin(significant_motifs)]\n",
    "cg = sns.clustermap(data,\n",
    "                   vmax=(coeff_mean + threshold* coeff_std),\n",
    "                   vmin=(coeff_mean - threshold* coeff_std),\n",
    "                   center = coeff_mean,\n",
    "                   annot=False,annot_kws={\"size\": 8},\n",
    "                   figsize=(4,4)\n",
    "                   )\n",
    "plt.setp(cg.ax_heatmap.yaxis.get_majorticklabels(), rotation=0);"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.4.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
