{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Strains Analysis"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "### header ###\n",
    "__author__ = \"Jenhan Tao\"\n",
    "__license__ = \"BSD\"\n",
    "__email__ = \"jenhantao@gmail.com\"\n",
    "\n",
    "### imports ###\n",
    "import sys\n",
    "import os\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "import argparse\n",
    "import matplotlib\n",
    "import itertools\n",
    "import scipy\n",
    "import matplotlib.pyplot as plt \n",
    "import seaborn as sns\n",
    "from scipy.spatial.distance import pdist, squareform\n",
    "from scipy.cluster.hierarchy import linkage, dendrogram\n",
    "import scipy\n",
    "import pickle\n",
    "from sklearn import preprocessing\n",
    "import sklearn\n",
    "from sklearn import decomposition\n",
    "from mpl_toolkits.mplot3d import Axes3D\n",
    "from sklearn.cluster import KMeans\n",
    "from sklearn import svm, datasets\n",
    "from sklearn.utils import shuffle\n",
    "from sklearn.metrics import roc_curve, auc\n",
    "from sklearn import ensemble\n",
    "from sklearn import neighbors\n",
    "import networkx as nx\n",
    "import matplotlib_venn\n",
    "from sklearn.cross_validation import train_test_split\n",
    "from random import shuffle\n",
    "\n",
    "### notebook specific configuration ###\n",
    "%matplotlib inline\n",
    "sys.path.append(\"/home/jenhan/code/seq_merge_pipe/\")\n",
    "matplotlib.pylab.rcParams['savefig.dpi'] = 200\n",
    "sys.setrecursionlimit(3000)\n",
    "os.chdir('/home/jenhan/analysis/strains_analysis/')\n",
    "sns.set_context('notebook')\n",
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Generate Score Files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished reading merged peak file...\n",
      "Integrating scores for balb_cebpa_kla\n",
      "Integrating scores for balb_cebpa_veh\n",
      "Integrating scores for balb_pu1_kla\n",
      "Integrating scores for balb_pu1_veh\n",
      "Integrating scores for c57bl6_cebpa_kla\n",
      "Integrating scores for c57bl6_cebpa_veh\n",
      "Integrating scores for c57bl6_pu1_kla\n",
      "Integrating scores for c57bl6_pu1_veh\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\tMax distance to merge: direct overlap required (-d given)\n",
      "\tMerging peaks... \n",
      "\tComparing ./peak_files/balb_cebpa_kla_peaks.tsv (24836 total) and ./peak_files/balb_cebpa_kla_peaks.tsv (24836 total)\n",
      "\tComparing ./peak_files/balb_cebpa_kla_peaks.tsv (24836 total) and ./peak_files/balb_cebpa_veh_peaks.tsv (23349 total)\n",
      "\tComparing ./peak_files/balb_cebpa_kla_peaks.tsv (24836 total) and ./peak_files/balb_pu1_kla_peaks.tsv (60743 total)\n",
      "\tComparing ./peak_files/balb_cebpa_kla_peaks.tsv (24836 total) and ./peak_files/balb_pu1_veh_peaks.tsv (58546 total)\n",
      "\tComparing ./peak_files/balb_cebpa_kla_peaks.tsv (24836 total) and ./peak_files/c57bl6_cebpa_kla_peaks.tsv (23276 total)\n",
      "\tComparing ./peak_files/balb_cebpa_kla_peaks.tsv (24836 total) and ./peak_files/c57bl6_cebpa_veh_peaks.tsv (19599 total)\n",
      "\tComparing ./peak_files/balb_cebpa_kla_peaks.tsv (24836 total) and ./peak_files/c57bl6_pu1_kla_peaks.tsv (59749 total)\n",
      "\tComparing ./peak_files/balb_cebpa_kla_peaks.tsv (24836 total) and ./peak_files/c57bl6_pu1_veh_peaks.tsv (53313 total)\n",
      "\tComparing ./peak_files/balb_cebpa_veh_peaks.tsv (23349 total) and ./peak_files/balb_cebpa_kla_peaks.tsv (24836 total)\n",
      "\tComparing ./peak_files/balb_cebpa_veh_peaks.tsv (23349 total) and ./peak_files/balb_cebpa_veh_peaks.tsv (23349 total)\n",
      "\tComparing ./peak_files/balb_cebpa_veh_peaks.tsv (23349 total) and ./peak_files/balb_pu1_kla_peaks.tsv (60743 total)\n",
      "\tComparing ./peak_files/balb_cebpa_veh_peaks.tsv (23349 total) and ./peak_files/balb_pu1_veh_peaks.tsv (58546 total)\n",
      "\tComparing ./peak_files/balb_cebpa_veh_peaks.tsv (23349 total) and ./peak_files/c57bl6_cebpa_kla_peaks.tsv (23276 total)\n",
      "\tComparing ./peak_files/balb_cebpa_veh_peaks.tsv (23349 total) and ./peak_files/c57bl6_cebpa_veh_peaks.tsv (19599 total)\n",
      "\tComparing ./peak_files/balb_cebpa_veh_peaks.tsv (23349 total) and ./peak_files/c57bl6_pu1_kla_peaks.tsv (59749 total)\n",
      "\tComparing ./peak_files/balb_cebpa_veh_peaks.tsv (23349 total) and ./peak_files/c57bl6_pu1_veh_peaks.tsv (53313 total)\n",
      "\tComparing ./peak_files/balb_pu1_kla_peaks.tsv (60743 total) and ./peak_files/balb_cebpa_kla_peaks.tsv (24836 total)\n",
      "\tComparing ./peak_files/balb_pu1_kla_peaks.tsv (60743 total) and ./peak_files/balb_cebpa_veh_peaks.tsv (23349 total)\n",
      "\tComparing ./peak_files/balb_pu1_kla_peaks.tsv (60743 total) and ./peak_files/balb_pu1_kla_peaks.tsv (60743 total)\n",
      "\tComparing ./peak_files/balb_pu1_kla_peaks.tsv (60743 total) and ./peak_files/balb_pu1_veh_peaks.tsv (58546 total)\n",
      "\tComparing ./peak_files/balb_pu1_kla_peaks.tsv (60743 total) and ./peak_files/c57bl6_cebpa_kla_peaks.tsv (23276 total)\n",
      "\tComparing ./peak_files/balb_pu1_kla_peaks.tsv (60743 total) and ./peak_files/c57bl6_cebpa_veh_peaks.tsv (19599 total)\n",
      "\tComparing ./peak_files/balb_pu1_kla_peaks.tsv (60743 total) and ./peak_files/c57bl6_pu1_kla_peaks.tsv (59749 total)\n",
      "\tComparing ./peak_files/balb_pu1_kla_peaks.tsv (60743 total) and ./peak_files/c57bl6_pu1_veh_peaks.tsv (53313 total)\n",
      "\tComparing ./peak_files/balb_pu1_veh_peaks.tsv (58546 total) and ./peak_files/balb_cebpa_kla_peaks.tsv (24836 total)\n",
      "\tComparing ./peak_files/balb_pu1_veh_peaks.tsv (58546 total) and ./peak_files/balb_cebpa_veh_peaks.tsv (23349 total)\n",
      "\tComparing ./peak_files/balb_pu1_veh_peaks.tsv (58546 total) and ./peak_files/balb_pu1_kla_peaks.tsv (60743 total)\n",
      "\tComparing ./peak_files/balb_pu1_veh_peaks.tsv (58546 total) and ./peak_files/balb_pu1_veh_peaks.tsv (58546 total)\n",
      "\tComparing ./peak_files/balb_pu1_veh_peaks.tsv (58546 total) and ./peak_files/c57bl6_cebpa_kla_peaks.tsv (23276 total)\n",
      "\tComparing ./peak_files/balb_pu1_veh_peaks.tsv (58546 total) and ./peak_files/c57bl6_cebpa_veh_peaks.tsv (19599 total)\n",
      "\tComparing ./peak_files/balb_pu1_veh_peaks.tsv (58546 total) and ./peak_files/c57bl6_pu1_kla_peaks.tsv (59749 total)\n",
      "\tComparing ./peak_files/balb_pu1_veh_peaks.tsv (58546 total) and ./peak_files/c57bl6_pu1_veh_peaks.tsv (53313 total)\n",
      "\tComparing ./peak_files/c57bl6_cebpa_kla_peaks.tsv (23276 total) and ./peak_files/balb_cebpa_kla_peaks.tsv (24836 total)\n",
      "\tComparing ./peak_files/c57bl6_cebpa_kla_peaks.tsv (23276 total) and ./peak_files/balb_cebpa_veh_peaks.tsv (23349 total)\n",
      "\tComparing ./peak_files/c57bl6_cebpa_kla_peaks.tsv (23276 total) and ./peak_files/balb_pu1_kla_peaks.tsv (60743 total)\n",
      "\tComparing ./peak_files/c57bl6_cebpa_kla_peaks.tsv (23276 total) and ./peak_files/balb_pu1_veh_peaks.tsv (58546 total)\n",
      "\tComparing ./peak_files/c57bl6_cebpa_kla_peaks.tsv (23276 total) and ./peak_files/c57bl6_cebpa_kla_peaks.tsv (23276 total)\n",
      "\tComparing ./peak_files/c57bl6_cebpa_kla_peaks.tsv (23276 total) and ./peak_files/c57bl6_cebpa_veh_peaks.tsv (19599 total)\n",
      "\tComparing ./peak_files/c57bl6_cebpa_kla_peaks.tsv (23276 total) and ./peak_files/c57bl6_pu1_kla_peaks.tsv (59749 total)\n",
      "\tComparing ./peak_files/c57bl6_cebpa_kla_peaks.tsv (23276 total) and ./peak_files/c57bl6_pu1_veh_peaks.tsv (53313 total)\n",
      "\tComparing ./peak_files/c57bl6_cebpa_veh_peaks.tsv (19599 total) and ./peak_files/balb_cebpa_kla_peaks.tsv (24836 total)\n",
      "\tComparing ./peak_files/c57bl6_cebpa_veh_peaks.tsv (19599 total) and ./peak_files/balb_cebpa_veh_peaks.tsv (23349 total)\n",
      "\tComparing ./peak_files/c57bl6_cebpa_veh_peaks.tsv (19599 total) and ./peak_files/balb_pu1_kla_peaks.tsv (60743 total)\n",
      "\tComparing ./peak_files/c57bl6_cebpa_veh_peaks.tsv (19599 total) and ./peak_files/balb_pu1_veh_peaks.tsv (58546 total)\n",
      "\tComparing ./peak_files/c57bl6_cebpa_veh_peaks.tsv (19599 total) and ./peak_files/c57bl6_cebpa_kla_peaks.tsv (23276 total)\n",
      "\tComparing ./peak_files/c57bl6_cebpa_veh_peaks.tsv (19599 total) and ./peak_files/c57bl6_cebpa_veh_peaks.tsv (19599 total)\n",
      "\tComparing ./peak_files/c57bl6_cebpa_veh_peaks.tsv (19599 total) and ./peak_files/c57bl6_pu1_kla_peaks.tsv (59749 total)\n",
      "\tComparing ./peak_files/c57bl6_cebpa_veh_peaks.tsv (19599 total) and ./peak_files/c57bl6_pu1_veh_peaks.tsv (53313 total)\n",
      "\tComparing ./peak_files/c57bl6_pu1_kla_peaks.tsv (59749 total) and ./peak_files/balb_cebpa_kla_peaks.tsv (24836 total)\n",
      "\tComparing ./peak_files/c57bl6_pu1_kla_peaks.tsv (59749 total) and ./peak_files/balb_cebpa_veh_peaks.tsv (23349 total)\n",
      "\tComparing ./peak_files/c57bl6_pu1_kla_peaks.tsv (59749 total) and ./peak_files/balb_pu1_kla_peaks.tsv (60743 total)\n",
      "\tComparing ./peak_files/c57bl6_pu1_kla_peaks.tsv (59749 total) and ./peak_files/balb_pu1_veh_peaks.tsv (58546 total)\n",
      "\tComparing ./peak_files/c57bl6_pu1_kla_peaks.tsv (59749 total) and ./peak_files/c57bl6_cebpa_kla_peaks.tsv (23276 total)\n",
      "\tComparing ./peak_files/c57bl6_pu1_kla_peaks.tsv (59749 total) and ./peak_files/c57bl6_cebpa_veh_peaks.tsv (19599 total)\n",
      "\tComparing ./peak_files/c57bl6_pu1_kla_peaks.tsv (59749 total) and ./peak_files/c57bl6_pu1_kla_peaks.tsv (59749 total)\n",
      "\tComparing ./peak_files/c57bl6_pu1_kla_peaks.tsv (59749 total) and ./peak_files/c57bl6_pu1_veh_peaks.tsv (53313 total)\n",
      "\tComparing ./peak_files/c57bl6_pu1_veh_peaks.tsv (53313 total) and ./peak_files/balb_cebpa_kla_peaks.tsv (24836 total)\n",
      "\tComparing ./peak_files/c57bl6_pu1_veh_peaks.tsv (53313 total) and ./peak_files/balb_cebpa_veh_peaks.tsv (23349 total)\n",
      "\tComparing ./peak_files/c57bl6_pu1_veh_peaks.tsv (53313 total) and ./peak_files/balb_pu1_kla_peaks.tsv (60743 total)\n",
      "\tComparing ./peak_files/c57bl6_pu1_veh_peaks.tsv (53313 total) and ./peak_files/balb_pu1_veh_peaks.tsv (58546 total)\n",
      "\tComparing ./peak_files/c57bl6_pu1_veh_peaks.tsv (53313 total) and ./peak_files/c57bl6_cebpa_kla_peaks.tsv (23276 total)\n",
      "\tComparing ./peak_files/c57bl6_pu1_veh_peaks.tsv (53313 total) and ./peak_files/c57bl6_cebpa_veh_peaks.tsv (19599 total)\n",
      "\tComparing ./peak_files/c57bl6_pu1_veh_peaks.tsv (53313 total) and ./peak_files/c57bl6_pu1_kla_peaks.tsv (59749 total)\n",
      "\tComparing ./peak_files/c57bl6_pu1_veh_peaks.tsv (53313 total) and ./peak_files/c57bl6_pu1_veh_peaks.tsv (53313 total)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "if [ ! -d ./group ]; then mkdir ./group; fi\n",
    "mergePeaks -d given -venn venn.txt ./peak_files/* > ./group/merged_peaks.tsv\n",
    "makeSummaryFile.py ./group/merged_peaks.tsv ./group_summary.tsv ./peak_files/*\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Annotate Peak File"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "annotatePeaks.pl ./group/merged_peaks.tsv mm10 > merged_annotated_peaks.tsv &\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Retrieve Peak Sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "python extendPeaks.py ./group//merged_peaks.tsv ./group//merged_extPeaks.tsv 0\n",
      "homerTools extract ./group//merged_extPeaks.tsv /bioinformatics/homer/data/genomes/mm10 -fa > ./group/merged.fa\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n",
      "\tExtracting sequences from directory: /bioinformatics/homer/data/genomes/mm10\n",
      "\tExtracting 6640 sequences from chr1\n",
      "\tExtracting 7677 sequences from chr2\n",
      "\tExtracting 4963 sequences from chr3\n",
      "\tExtracting 5370 sequences from chr4\n",
      "\tExtracting 5534 sequences from chr5\n",
      "\tExtracting 5464 sequences from chr6\n",
      "\tExtracting 5283 sequences from chr7\n",
      "\tExtracting 4314 sequences from chr8\n",
      "\tExtracting 4878 sequences from chr9\n",
      "\tExtracting 5302 sequences from chr10\n",
      "\tExtracting 6527 sequences from chr11\n",
      "\tExtracting 3727 sequences from chr12\n",
      "\tExtracting 4540 sequences from chr13\n",
      "\tExtracting 3674 sequences from chr14\n",
      "\tExtracting 3910 sequences from chr15\n",
      "\tExtracting 3459 sequences from chr16\n",
      "\tExtracting 4051 sequences from chr17\n",
      "\tExtracting 2779 sequences from chr18\n",
      "\tExtracting 3011 sequences from chr19\n",
      "\tExtracting 2156 sequences from chrX\n",
      "\tExtracting 2 sequences from chrY\n",
      "\tExtracting 5 sequences from chr4_GL456216_random\n",
      "\tExtracting 1 sequences from chr4_JH584295_random\n",
      "\tExtracting 1 sequences from chrUn_GL456359\n",
      "\tExtracting 1 sequences from chrUn_GL456370\n",
      "\tExtracting 2 sequences from chrUn_JH584304\n",
      "\tExtracting 3 sequences from chrX_GL456233_random\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "/home/jenhan/code/motif_tools/getTargetSequencesWrapper.sh ./group/ ./group 0"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "### read in sequences as dictionary {peakID: sequence}\n",
    "with open('./group/merged.fa') as f:\n",
    "    data = f.readlines()\n",
    "\n",
    "_id_sequence_dict = {}\n",
    "for line in data:\n",
    "    if line[0] == '>':\n",
    "        sequenceName = line.strip()[1:]\n",
    "    else:\n",
    "        _id_sequence_dict[sequenceName] = line.strip().upper()\n",
    "        "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Scan for Motifs using Fimo"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# create a script to scan for motifs using FIMO\n",
    "! if [ ! -d ./fimo_out/ ]; then mkdir ./fimo_out; fi\n",
    "! if [ ! -d ./fimo_results/ ]; then mkdir ./fimo_results; fi\n",
    "\n",
    "\n",
    "pthresh = 0.01\n",
    "motif_dir = '/home/jenhan/analysis/cobinding_motif_analysis/fimo_motifs/'\n",
    "\n",
    "fimo_results_dir = './fimo_results'\n",
    "count = 0\n",
    "scriptFile = open('scanMotifs.sh','w')\n",
    "for m in os.listdir(motif_dir):\n",
    "    if 'fimo' in m:\n",
    "        fimo_out_dir = '/home/jenhan/analysis/strains_analysis/fimo_out/' + m.replace('.fimo','')\n",
    "        outPath = fimo_results_dir + '/merged_'+ m.replace('.fimo','') +'.txt'\n",
    "        scriptFile.write(\n",
    "#             '(sleep ' + str(15 * count) + \n",
    "            '(sleep ' + str(0 * count) + \n",
    "            's; fimo --text --max-stored-scores 2000000 --output-pthresh ' + \n",
    "            str(pthresh) +' --oc ' + fimo_out_dir + ' ' +\n",
    "            motif_dir + '/' + m + ' /home/jenhan/analysis/strains_analysis/group/merged.fa '+\n",
    "            '> ' + outPath + ' ) & \\n')\n",
    "        count+=1\n",
    "scriptFile.close()\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spib of width 7.\n",
      "Using motif -spib of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +duxa of width 13.\n",
      "Using motif -duxa of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +dux of width 8.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -dux of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa5 of width 8.\n",
      "Using motif +elk_etv of width 11.\n",
      "Using motif -hoxa5 of width 8.\n",
      "Using motif +ewsr1-fli1 of width 18.\n",
      "Using motif -elk_etv of width 11.\n",
      "Using motif -ewsr1-fli1 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "FATAL: Error attempting to read motif ID 2.\n",
      "\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +hes of width 12.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif -hes of width 12.\n",
      "Using motif +rora of width 10.\n",
      "Using motif +mzf1 of width 6.\n",
      "Using motif -mzf1 of width 6.\n",
      "Using motif -rora of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +rfx of width 16.\n",
      "Using motif +rar of width 16.\n",
      "Using motif +prox1 of width 12.\n",
      "Using motif +rhox11 of width 17.\n",
      "Using motif +pax1_pax9 of width 17.\n",
      "Using motif +nr2_rxr of width 15.\n",
      "Using motif -prox1 of width 12.\n",
      "Using motif -rhox11 of width 17.\n",
      "Using motif -rfx of width 16.\n",
      "Using motif -rar of width 16.\n",
      "Using motif -pax1_pax9 of width 17.\n",
      "Using motif -nr2_rxr of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e1 of width 9.\n",
      "Using motif -nr2e1 of width 9.\n",
      "Using motif +nfyb of width 15.\n",
      "Using motif +irf2 of width 18.\n",
      "Using motif -nfyb of width 15.\n",
      "Using motif -irf2 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb18 of width 13.\n",
      "Using motif -zbtb18 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +pou6f2 of width 10.\n",
      "Using motif +pou1f1 of width 14.\n",
      "Using motif -pou6f2 of width 10.\n",
      "Using motif -pou1f1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arnt_mycn of width 8.\n",
      "Using motif -arnt_mycn of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +arid3a of width 6.\n",
      "Using motif -arid3a of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +alx1_alx4_arx of width 17.\n",
      "Using motif -alx1_alx4_arx of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +rel of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +yy1 of width 12.\n",
      "Using motif +usf of width 11.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +sp1 of width 11.\n",
      "Using motif +xbp1 of width 14.\n",
      "Using motif -yy1 of width 12.\n",
      "Using motif -usf of width 11.\n",
      "Using motif +t of width 16.\n",
      "Using motif +tfap2-3 of width 13.\n",
      "Using motif +tfcp2 of width 10.\n",
      "Using motif -sp1 of width 11.\n",
      "Using motif -xbp1 of width 14.\n",
      "Using motif -tfcp2 of width 10.\n",
      "Using motif -tfap2-3 of width 13.\n",
      "Using motif -t of width 16.\n",
      "Using motif -rel of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +sox17 of width 9.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -sox17 of width 9.\n",
      "Using motif +spi1-c of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf423 of width 15.\n",
      "Using motif -spi1-c of width 14.\n",
      "Using motif +zic of width 14.\n",
      "Using motif -znf423 of width 15.\n",
      "Using motif -zic of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +stat1-3 of width 11.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -stat1-3 of width 11.\n",
      "Using motif +ebf1 of width 14.\n",
      "Using motif -ebf1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esrr of width 10.\n",
      "Using motif -esrr of width 10.\n",
      "Using motif +sox3_sox6 of width 10.\n",
      "Using motif +homeobox-5 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -homeobox-5 of width 9.\n",
      "Using motif -sox3_sox6 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp53 of width 18.\n",
      "Using motif -tp53 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +homeobox-1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif -homeobox-1 of width 8.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f2 of width 16.\n",
      "Using motif +dmrt3 of width 11.\n",
      "Using motif +bcl6 of width 14.\n",
      "Using motif -dmrt3 of width 11.\n",
      "Using motif -bcl6 of width 14.\n",
      "Using motif -e2f2 of width 16.\n",
      "Using motif +hoxa9 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif -hoxa9 of width 11.\n",
      "Using motif +cdx of width 11.\n",
      "Using motif +dmbx1 of width 17.\n",
      "Using motif +bhlh of width 10.\n",
      "Using motif +bhlhe23 of width 12.\n",
      "Using motif +ascl2_nhlh1 of width 10.\n",
      "Using motif -bhlh of width 10.\n",
      "Using motif -cdx of width 11.\n",
      "Using motif -ascl2_nhlh1 of width 10.\n",
      "Using motif -bhlhe23 of width 12.\n",
      "Using motif -dmbx1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nrf1 of width 11.\n",
      "Using motif -nrf1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -homeobox-4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl1 of width 12.\n",
      "Using motif -mybl1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1b of width 11.\n",
      "Using motif -gfi1b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f4_e2f6 of width 11.\n",
      "Using motif +mecom of width 14.\n",
      "Using motif -e2f4_e2f6 of width 11.\n",
      "Using motif +hltf of width 10.\n",
      "Using motif -mecom of width 14.\n",
      "Using motif -hltf of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +e2f7 of width 14.\n",
      "Using motif -e2f7 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +irf7-8-9 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -irf7-8-9 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox5_sox9_sry of width 9.\n",
      "Using motif -sox5_sox9_sry of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rreb1 of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -rreb1 of width 20.\n",
      "Using motif +nfi of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -nfi of width 6.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2c of width 15.\n",
      "Using motif -mef2c of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lin54 of width 9.\n",
      "Using motif -lin54 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +prdm1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -prdm1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +fox of width 8.\n",
      "Using motif -fox of width 8.\n",
      "Using motif +nr4a2 of width 8.\n",
      "Using motif -nr4a2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hoxc13 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -hoxc13 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +pknox_tgif of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pknox_tgif of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rxravdr of width 15.\n",
      "Using motif -rxravdr of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tp63 of width 18.\n",
      "Using motif -tp63 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gata of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -gata of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcf7 of width 12.\n",
      "Using motif -tcf7 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxc9 of width 13.\n",
      "Using motif -hoxc9 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +esr1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -esr1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +onecut of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif -onecut of width 14.\n",
      "Using motif +hnf4a of width 16.\n",
      "Using motif -hnf4a of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5var of width 11.\n",
      "Using motif -nkx2-5var of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +znf143 of width 16.\n",
      "Read 1 motifs.\n",
      "Using motif -znf143 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pax5 of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -pax5 of width 19.\n",
      "Using motif +hinfp of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hinfp of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mzf1var of width 10.\n",
      "Using motif -mzf1var of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +insm1 of width 12.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -insm1 of width 12.\n",
      "Using motif +thap1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -thap1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +stat6 of width 15.\n",
      "Using motif -stat6 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox8 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -sox8 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sp4 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -sp4 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tp73 of width 18.\n",
      "Using motif -tp73 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mef2a-b-d of width 12.\n",
      "Using motif -mef2a-b-d of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +elf of width 12.\n",
      "Using motif +glis of width 14.\n",
      "Using motif -elf of width 12.\n",
      "Using motif +cebp of width 10.\n",
      "Using motif -glis of width 14.\n",
      "Using motif -cebp of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cenpb of width 15.\n",
      "Using motif -cenpb of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +maf_nrl of width 12.\n",
      "Using motif -maf_nrl of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rargvar of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -rargvar of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif +srf of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -srf of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-5 of width 7.\n",
      "Using motif -nkx2-5 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr5a2 of width 15.\n",
      "Using motif -nr5a2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2f1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nr2f1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pbx1 of width 12.\n",
      "Using motif -pbx1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rar_rxr of width 17.\n",
      "Using motif -rar_rxr of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ap-1 of width 7.\n",
      "Using motif -ap-1 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zfx of width 14.\n",
      "Using motif -zfx of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +sox2 of width 8.\n",
      "Using motif -sox2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hnf1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hnf1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "FATAL: Error attempting to read motif ID 2.\n",
      "\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +crem of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +sox10 of width 6.\n",
      "Using motif -crem of width 10.\n",
      "Using motif -sox10 of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +irf1 of width 21.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxb5_hoxd3 of width 16.\n",
      "Using motif -irf1 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hoxb5_hoxd3 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +cux of width 10.\n",
      "Using motif -cux of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +six3 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -six3 of width 17.\n",
      "Using motif +ar_nr3c1_nr3c2 of width 17.\n",
      "Using motif -ar_nr3c1_nr3c2 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hsf of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hsf of width 13.\n",
      "Using motif +arid3b of width 11.\n",
      "Using motif -arid3b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +runx of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif -runx of width 11.\n",
      "Using motif +mafg of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -mafg of width 21.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +klf of width 11.\n",
      "Using motif -klf of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfat of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nfat of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pou2-3-5 of width 9.\n",
      "Using motif -pou2-3-5 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +scrt of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -scrt of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ventx of width 9.\n",
      "Using motif -ventx of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nkx2-3_nkx2-8 of width 10.\n",
      "Using motif -nkx2-3_nkx2-8 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gmeb2 of width 8.\n",
      "Using motif -gmeb2 of width 8.\n",
      "Using motif +roravar of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -roravar of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tfap1 of width 12.\n",
      "Using motif -tfap1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfya of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nfya of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mga_tbx of width 8.\n",
      "Using motif -mga_tbx of width 8.\n",
      "Using motif +zeb1 of width 9.\n",
      "Using motif -zeb1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -isl2_nkx3-1_nkx3-2 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gcm of width 10.\n",
      "Using motif -gcm of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +pparg of width 20.\n",
      "Using motif +e2f1 of width 12.\n",
      "Using motif +ctcf of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -e2f1 of width 12.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -pparg of width 20.\n",
      "Using motif -ctcf of width 19.\n",
      "Using motif +znf740 of width 10.\n",
      "Using motif -znf740 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +zbed1 of width 13.\n",
      "Using motif +phox2_prop of width 11.\n",
      "Using motif -phox2_prop of width 11.\n",
      "Using motif -zbed1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +hnf4g of width 15.\n",
      "Using motif -hnf4g of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxa11 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hoxa11 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pparg_rxra of width 15.\n",
      "Using motif -pparg_rxra of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +rest of width 21.\n",
      "Using motif -rest of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +myc of width 10.\n",
      "Using motif -myc of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nfil3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nfil3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hic of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hic of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb7 of width 12.\n",
      "Using motif -zbtb7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +myog_tcf12 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -myog_tcf12 of width 11.\n",
      "Using motif +yy2 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -yy2 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +egr of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -egr of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pax6 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +atf7_batf3_creb5 of width 12.\n",
      "Using motif +plag1 of width 14.\n",
      "Using motif -atf7_batf3_creb5 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -plag1 of width 14.\n",
      "Using motif +gli2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -gli2 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf354c of width 6.\n",
      "Using motif -znf354c of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arntl_mitf of width 10.\n",
      "Using motif -arntl_mitf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +gsc of width 10.\n",
      "Using motif -gsc of width 10.\n",
      "Using motif +figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Using motif -figla_id4_snai2_tcf3_tcf4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bhlha15 of width 8.\n",
      "Using motif -bhlha15 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +srebpf of width 10.\n",
      "Using motif -srebpf of width 10.\n",
      "Using motif +pou4 of width 16.\n",
      "Using motif -pou4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tbp of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -tbp of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +myb of width 10.\n",
      "Using motif -myb of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +hoxd8 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -hoxd8 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +bcl6b of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -bcl6b of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +e2f8 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -e2f8 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +gfi1 of width 10.\n",
      "Using motif -gfi1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +nr2e3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -nr2e3 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +grhl1 of width 12.\n",
      "Using motif -grhl1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -pax2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +nfkb of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -nfkb of width 13.\n",
      "Read 1 motifs.\n",
      "Using motif +zbtb33 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -zbtb33 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +spz1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -spz1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tcf21 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -tcf21 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mybl2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -mybl2 of width 15.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +znf263 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -znf263 of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +dux4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -dux4 of width 11.\n",
      "Using motif +lef1 of width 15.\n",
      "Using motif -lef1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +pax3_pax7 of width 10.\n",
      "Using motif -pax3_pax7 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +dbp_hlf_tef of width 11.\n",
      "Using motif -dbp_hlf_tef of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +sox1-l1-21-4 of width 16.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -sox1-l1-21-4 of width 16.\n",
      "Using motif +creb3-l1 of width 14.\n",
      "Using motif +msc_myf6_tfap4 of width 10.\n",
      "Using motif -creb3-l1 of width 14.\n",
      "Using motif -msc_myf6_tfap4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +homeobox-3 of width 11.\n",
      "Using motif -homeobox-3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +lhx3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -lhx3 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +tead of width 8.\n",
      "Using motif -tead of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +tcfl5 of width 10.\n",
      "Using motif -tcfl5 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +smad3 of width 10.\n",
      "Using motif -smad3 of width 10.\n",
      "Using motif +id2_mlxip of width 8.\n",
      "Using motif -id2_mlxip of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +mtf1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -mtf1 of width 14.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +arid5a of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -arid5a of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +meis of width 8.\n",
      "Using motif -meis of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +znf410 of width 17.\n",
      "Using motif -znf410 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +esr2 of width 15.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -esr2 of width 15.\n",
      "Using motif +tbx of width 11.\n",
      "Using motif -tbx of width 11.\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "chmod a+x ./scanMotifs.sh\n",
    "./scanMotifs.sh"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Read in Score Files"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Read in peak scores"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# read in peak data data\n",
    "summary_frame = pd.read_csv('./group_summary.tsv' , sep='\\t')\n",
    "summary_frame = summary_frame.fillna('0')\n",
    "for col in summary_frame.columns[5:]:\n",
    "    floatValues = []\n",
    "    for val in summary_frame[col].values.astype(str):\n",
    "        if ',' in val:\n",
    "            maxVal = np.max([float(x) for x in val.split(',')])\n",
    "            floatValues.append(maxVal)\n",
    "        else:\n",
    "            floatValues.append(float(val))\n",
    "    summary_frame[col] = floatValues\n",
    "summary_frame.index = summary_frame['ID'].values\n",
    "\n",
    "# remove peaks in unknown/random chromosomes\n",
    "summary_frame = summary_frame[~summary_frame['chr'].str.contains('random')]\n",
    "summary_frame = summary_frame[~summary_frame['chr'].str.contains('Un')]\n",
    "\n",
    "\n",
    "_factors = sorted(list(set([x.split('_')[1] for x in summary_frame.columns if '_' in x])))\n",
    "summary_frame.to_pickle('summary_frame.pickle')\n",
    "scaler = sklearn.preprocessing.MinMaxScaler()\n",
    "normed_summary_frame = pd.DataFrame(scaler.fit_transform(summary_frame.ix[:,5:]))\n",
    "normed_summary_frame.columns = summary_frame.columns.values[5:]\n",
    "normed_summary_frame.index = summary_frame.index.values"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Read in motif scores"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1 alx1_alx4_arx.fimo\n",
      "2 ap-1.fimo\n",
      "3 ar_nr3c1_nr3c2.fimo\n",
      "4 arid3a.fimo\n",
      "5 arid3b.fimo\n",
      "6 arid5a.fimo\n",
      "7 arnt_mycn.fimo\n",
      "8"
     ]
    }
   ],
   "source": [
    "\n",
    "motif_dir = '/home/jenhan/analysis/cobinding_motif_analysis/fimo_motifs/'\n",
    "\n",
    "peak_start_dict = dict(zip(summary_frame['ID'].values, summary_frame['start'].values))\n",
    "\n",
    "motif_score_frame = summary_frame[['ID', 'Factors', 'chr']]\n",
    "motif_score_frame.index=motif_score_frame['ID'].values\n",
    "\n",
    "motif_sequence_frame = summary_frame[['ID', 'Factors', 'chr']]\n",
    "motif_sequence_frame.index=motif_score_frame['ID'].values\n",
    "\n",
    "motif_strand_frame = summary_frame[['ID', 'Factors', 'chr']]\n",
    "motif_strand_frame.index=motif_score_frame['ID'].values\n",
    "\n",
    "motif_start_frame = summary_frame[['ID', 'Factors', 'chr']]\n",
    "motif_start_frame.index=motif_score_frame['ID'].values\n",
    "\n",
    "motif_end_frame = summary_frame[['ID', 'Factors', 'chr']]\n",
    "motif_end_frame.index=motif_score_frame['ID'].values\n",
    "\n",
    "motif_count_frame = summary_frame[['ID', 'Factors', 'chr']]\n",
    "motif_count_frame.index = motif_count_frame['ID'].values\n",
    "\n",
    "counter=0\n",
    "for m in sorted(os.listdir(motif_dir)):\n",
    "    counter+=1\n",
    "    if '.fimo' in m:\n",
    "        print(counter,m)\n",
    "        motif_results = './fimo_results//merged_' + m.replace('.fimo','') +'.txt'\n",
    "        fimo_result_frame=pd.read_csv(motif_results, \n",
    "                                      skiprows=1,\n",
    "                                      names=['motif_name', \n",
    "                                             'peak_id', \n",
    "                                             'start', \n",
    "                                             'stop', \n",
    "                                             'strand', \n",
    "                                             'score', \n",
    "                                             'pvalue', \n",
    "                                             'sequence'],\n",
    "                                      sep='\\t')\n",
    "        motif_name = m.replace('.fimo','')\n",
    "        id_score_dict = {}\n",
    "        id_strand_dict = {}\n",
    "        id_sequence_dict = {}\n",
    "        id_start_dict = {}\n",
    "        id_end_dict = {}\n",
    "        id_count_dict = {}\n",
    "        \n",
    "        ids = fimo_result_frame['peak_id'].values\n",
    "        scores = fimo_result_frame['score'].values\n",
    "        strands = fimo_result_frame['strand'].values\n",
    "        sequences = fimo_result_frame['sequence']\n",
    "        starts = fimo_result_frame['start']\n",
    "        ends = fimo_result_frame['stop']\n",
    "        \n",
    "        for i in range(len(ids)):\n",
    "            currentScore = float(scores[i])\n",
    "            currentSequence = sequences[i]\n",
    "            currentStrand = strands[i]\n",
    "            currentStart = int(starts[i])\n",
    "            currentEnd = int(ends[i])\n",
    "            if currentScore < 0.0:\n",
    "                currentScore = 0.0\n",
    "            if ids[i] in id_score_dict:\n",
    "                if currentScore > id_score_dict[ids[i]]:\n",
    "                    id_score_dict[ids[i]] = currentScore\n",
    "                    id_sequence_dict[ids[i]] = currentSequence\n",
    "                    id_strand_dict[ids[i]] = currentStrand\n",
    "                    id_start_dict[ids[i]] = currentStart\n",
    "                    id_end_dict[ids[i]] = currentEnd\n",
    "                    id_count_dict[ids[i]] += 1\n",
    "            else:\n",
    "                id_score_dict[ids[i]] = currentScore\n",
    "                id_sequence_dict[ids[i]] = currentSequence\n",
    "                id_strand_dict[ids[i]] = currentStrand\n",
    "                id_start_dict[ids[i]] = currentStart\n",
    "                id_end_dict[ids[i]] = currentEnd\n",
    "                id_count_dict[ids[i]] = 0\n",
    "                \n",
    "        motif_score_frame[motif_name] = [id_score_dict[x] if x in id_score_dict else 0 for x in motif_score_frame['ID'].values]\n",
    "        motif_sequence_frame[motif_name] = [id_sequence_dict[x] if x in id_sequence_dict else '?' for x in motif_sequence_frame['ID'].values]\n",
    "        motif_strand_frame[motif_name] = [id_strand_dict[x] if x in id_score_dict else '' for x in motif_strand_frame['ID'].values]\n",
    "        motif_start_frame[motif_name] = [id_start_dict[x] + peak_start_dict[x] if x in id_start_dict else -1 for x in motif_start_frame['ID'].values]\n",
    "        motif_end_frame[motif_name] = [id_end_dict[x] + peak_start_dict[x] if x in id_end_dict else -1 for x in motif_end_frame['ID'].values]\n",
    "        motif_count_frame[motif_name] = [id_count_dict[x] if x in id_count_dict else 0 for x in motif_count_frame['ID'].values]\n",
    "\n",
    "\n",
    "        \n",
    "motif_score_frame.to_pickle('motif_score_frame.pickle')\n",
    "motif_score_frame.to_csv('motif_scores.tsv', sep='\\t', index=False)\n",
    "\n",
    "motif_sequence_frame.to_pickle('motif_sequence_frame.pickle')\n",
    "motif_sequence_frame.to_csv('motif_sequence.tsv', sep='\\t', index=False)\n",
    "\n",
    "motif_strand_frame.to_pickle('motif_strand_frame.pickle')\n",
    "motif_strand_frame.to_csv('motif_strand.tsv', sep='\\t', index=False)\n",
    "\n",
    "motif_start_frame.to_pickle('motif_start_frame.pickle')\n",
    "motif_start_frame.to_csv('motif_start_frame.tsv', sep='\\t', index=False)\n",
    "\n",
    "motif_end_frame.to_pickle('motif_end_frame.pickle')\n",
    "motif_end_frame.to_csv('motif_end_frame.tsv', sep='\\t', index=False)\n",
    "\n",
    "motif_count_frame.to_pickle('motif_count_frame.pickle')\n",
    "motif_count_frame.to_csv('motif_count_frame.tsv', sep='\\t', index=False)\n",
    "\n",
    "scaler = preprocessing.MinMaxScaler()\n",
    "normed_motif_frame = pd.DataFrame(scaler.fit_transform(motif_score_frame.ix[:,3:]))\n",
    "normed_motif_frame.columns = motif_score_frame.columns.values[3:]\n",
    "normed_motif_frame.index = motif_score_frame.index.values\n",
    "\n",
    "scaled_motif_frame = pd.DataFrame()\n",
    "for m in motif_score_frame.columns[3:]:\n",
    "    scaled_motif_frame[m] = motif_score_frame[m]/(2*motif_sequence_frame[m].str.len().max())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div style=\"max-height:1000px;max-width:1500px;overflow:auto;\">\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>alx1_alx4_arx</th>\n",
       "      <th>ap-1</th>\n",
       "      <th>ar_nr3c1_nr3c2</th>\n",
       "      <th>arid3a</th>\n",
       "      <th>arid3b</th>\n",
       "      <th>arid5a</th>\n",
       "      <th>arnt_mycn</th>\n",
       "      <th>arntl_mitf</th>\n",
       "      <th>ascl2_nhlh1</th>\n",
       "      <th>atf7_batf3_creb5</th>\n",
       "      <th>...</th>\n",
       "      <th>zbtb7</th>\n",
       "      <th>zeb1</th>\n",
       "      <th>zfx</th>\n",
       "      <th>zic</th>\n",
       "      <th>znf143</th>\n",
       "      <th>znf263</th>\n",
       "      <th>znf354c</th>\n",
       "      <th>znf410</th>\n",
       "      <th>znf423</th>\n",
       "      <th>znf740</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>Merged-chr17-34627665-3</th>\n",
       "      <td> 0.125804</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.508086</td>\n",
       "      <td> 0.174588</td>\n",
       "      <td> 0.172317</td>\n",
       "      <td> 0.063901</td>\n",
       "      <td> 0.376679</td>\n",
       "      <td> 0.304938</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.329533</td>\n",
       "      <td> 0.263295</td>\n",
       "      <td> 0.327267</td>\n",
       "      <td> 0.052839</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.182261</td>\n",
       "      <td> 0.596505</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.714523</td>\n",
       "      <td> 0.831465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr16-57200743-3</th>\n",
       "      <td> 0.053001</td>\n",
       "      <td> 0.924300</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.508086</td>\n",
       "      <td> 0.343869</td>\n",
       "      <td> 0.250259</td>\n",
       "      <td> 0.057756</td>\n",
       "      <td> 0.224335</td>\n",
       "      <td> 0.329444</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.354112</td>\n",
       "      <td> 0.285146</td>\n",
       "      <td> 0.274590</td>\n",
       "      <td> 0.080844</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.110720</td>\n",
       "      <td> 0.375262</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.445052</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr9-70560828-3</th>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.243004</td>\n",
       "      <td> 0.027416</td>\n",
       "      <td> 0.778647</td>\n",
       "      <td> 0.044371</td>\n",
       "      <td> 0.338366</td>\n",
       "      <td> 0.315874</td>\n",
       "      <td> 0.352108</td>\n",
       "      <td> 0.314740</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.132187</td>\n",
       "      <td> 0.134118</td>\n",
       "      <td> 0.014333</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.115336</td>\n",
       "      <td> 0.678447</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.003264</td>\n",
       "      <td> 0.267618</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr5-30893945-3</th>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.038584</td>\n",
       "      <td> 0.165907</td>\n",
       "      <td> 0.145207</td>\n",
       "      <td> 0.285146</td>\n",
       "      <td> 0.302964</td>\n",
       "      <td> 0.167705</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.059167</td>\n",
       "      <td> 0.487271</td>\n",
       "      <td> 0.102512</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.182261</td>\n",
       "      <td> 0.858717</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.240017</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr16-87636040-3</th>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.643352</td>\n",
       "      <td> 0.310815</td>\n",
       "      <td> 0.500128</td>\n",
       "      <td> 0.313485</td>\n",
       "      <td> 0.297701</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.081818</td>\n",
       "      <td> 0.226519</td>\n",
       "      <td> 0.574300</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.063263</td>\n",
       "      <td> 0.312461</td>\n",
       "      <td> 0.274590</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.274571</td>\n",
       "      <td> 0.858717</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.208474</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr7-98535313-3</th>\n",
       "      <td> 0.058601</td>\n",
       "      <td> 0.243004</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.101154</td>\n",
       "      <td> 0.113067</td>\n",
       "      <td> 0.307878</td>\n",
       "      <td> 0.270629</td>\n",
       "      <td> 0.251354</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.231218</td>\n",
       "      <td> 0.164964</td>\n",
       "      <td> 0.084953</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.103829</td>\n",
       "      <td> 0.226108</td>\n",
       "      <td> 0.678447</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.255790</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr13-67484493-3</th>\n",
       "      <td> 0.170606</td>\n",
       "      <td> 0.243004</td>\n",
       "      <td> 0.024524</td>\n",
       "      <td> 0.635408</td>\n",
       "      <td> 0.382934</td>\n",
       "      <td> 0.290924</td>\n",
       "      <td> 0.475662</td>\n",
       "      <td> 0.317707</td>\n",
       "      <td> 0.211815</td>\n",
       "      <td> 0.615179</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.227122</td>\n",
       "      <td> 0.230518</td>\n",
       "      <td> 0.264055</td>\n",
       "      <td> 0.052839</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.046104</td>\n",
       "      <td> 0.621088</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr3-90604533-3</th>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.530974</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.834350</td>\n",
       "      <td> 0.387275</td>\n",
       "      <td> 0.104542</td>\n",
       "      <td> 0.113067</td>\n",
       "      <td> 0.067075</td>\n",
       "      <td> 0.589210</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.198446</td>\n",
       "      <td> 0.208667</td>\n",
       "      <td> 0.527439</td>\n",
       "      <td> 0.038837</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.332264</td>\n",
       "      <td> 0.694836</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.009819</td>\n",
       "      <td> 0.429281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr17-26676427-3</th>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.250028</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.778647</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.303583</td>\n",
       "      <td> 0.656800</td>\n",
       "      <td> 0.285334</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.292665</td>\n",
       "      <td> 0.678472</td>\n",
       "      <td> 0.351850</td>\n",
       "      <td> 0.038837</td>\n",
       "      <td> 0.103829</td>\n",
       "      <td> 0.426881</td>\n",
       "      <td> 0.858717</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.294979</td>\n",
       "      <td> 0.342535</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr1-97769642-3</th>\n",
       "      <td> 0.263009</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.452382</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.223149</td>\n",
       "      <td> 0.899713</td>\n",
       "      <td> 0.524110</td>\n",
       "      <td> 0.309839</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.075553</td>\n",
       "      <td> 0.252369</td>\n",
       "      <td> 0.334291</td>\n",
       "      <td> 0.185860</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.156875</td>\n",
       "      <td> 0.858717</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.039319</td>\n",
       "      <td> 0.393794</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr7-104446416-3</th>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.250028</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.635408</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.294313</td>\n",
       "      <td> 0.469517</td>\n",
       "      <td> 0.465138</td>\n",
       "      <td> 0.471579</td>\n",
       "      <td> 0.296322</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.313147</td>\n",
       "      <td> 0.836894</td>\n",
       "      <td> 0.358875</td>\n",
       "      <td> 0.161356</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.286110</td>\n",
       "      <td> 0.703030</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.519970</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr11-102156353-3</th>\n",
       "      <td> 0.106203</td>\n",
       "      <td> 0.243004</td>\n",
       "      <td> 0.270330</td>\n",
       "      <td> 0.046542</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.138430</td>\n",
       "      <td> 0.180669</td>\n",
       "      <td> 0.479882</td>\n",
       "      <td> 0.500985</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.358208</td>\n",
       "      <td> 0.290609</td>\n",
       "      <td> 0.425596</td>\n",
       "      <td> 0.133352</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.170722</td>\n",
       "      <td> 0.678447</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.176982</td>\n",
       "      <td> 0.598830</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr7-59027671-3</th>\n",
       "      <td> 0.243409</td>\n",
       "      <td> 0.924300</td>\n",
       "      <td> 0.093928</td>\n",
       "      <td> 0.691113</td>\n",
       "      <td> 0.482768</td>\n",
       "      <td> 0.206205</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.234163</td>\n",
       "      <td> 0.466678</td>\n",
       "      <td> 0.312673</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.181352</td>\n",
       "      <td> 0.028764</td>\n",
       "      <td> 0.042338</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.205338</td>\n",
       "      <td> 0.621088</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.050755</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr4-41319130-3</th>\n",
       "      <td> 0.030600</td>\n",
       "      <td> 0.243004</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.834350</td>\n",
       "      <td> 0.382934</td>\n",
       "      <td> 0.307867</td>\n",
       "      <td> 0.199107</td>\n",
       "      <td> 0.229249</td>\n",
       "      <td> 0.241222</td>\n",
       "      <td> 0.091925</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.317923</td>\n",
       "      <td> 0.169236</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.128411</td>\n",
       "      <td> 0.443036</td>\n",
       "      <td> 0.776778</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.065540</td>\n",
       "      <td> 0.460825</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr11-119602934-4</th>\n",
       "      <td> 0.176206</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.082361</td>\n",
       "      <td> 0.452382</td>\n",
       "      <td> 0.291782</td>\n",
       "      <td> 0.290924</td>\n",
       "      <td> 0.285146</td>\n",
       "      <td> 0.411081</td>\n",
       "      <td> 0.231420</td>\n",
       "      <td> 0.059222</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.177964</td>\n",
       "      <td> 0.268758</td>\n",
       "      <td> 0.408039</td>\n",
       "      <td> 0.203363</td>\n",
       "      <td> 0.346581</td>\n",
       "      <td> 0.323033</td>\n",
       "      <td> 0.858717</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.417452</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr12-73907835-4</th>\n",
       "      <td> 0.078202</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.770656</td>\n",
       "      <td> 0.656800</td>\n",
       "      <td> 0.309839</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.263990</td>\n",
       "      <td> 0.858744</td>\n",
       "      <td> 0.520418</td>\n",
       "      <td> 0.420396</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.320726</td>\n",
       "      <td> 0.678447</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.859065</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr11-17212255-4</th>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.160441</td>\n",
       "      <td> 0.102245</td>\n",
       "      <td> 0.083436</td>\n",
       "      <td> 0.182484</td>\n",
       "      <td> 0.186815</td>\n",
       "      <td> 0.185020</td>\n",
       "      <td> 0.505885</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.329533</td>\n",
       "      <td> 0.645694</td>\n",
       "      <td> 0.260543</td>\n",
       "      <td> 0.136852</td>\n",
       "      <td> 0.343509</td>\n",
       "      <td> 0.431498</td>\n",
       "      <td> 0.776778</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.255790</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr10-81545316-4</th>\n",
       "      <td> 0.041800</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.334312</td>\n",
       "      <td> 0.371766</td>\n",
       "      <td> 0.255926</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.325437</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.622257</td>\n",
       "      <td> 0.066841</td>\n",
       "      <td> 0.272834</td>\n",
       "      <td> 0.235339</td>\n",
       "      <td> 0.776778</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.504195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr7-12978159-4</th>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.250028</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.200631</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.205253</td>\n",
       "      <td> 0.465138</td>\n",
       "      <td> 0.437271</td>\n",
       "      <td> 0.026518</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.313147</td>\n",
       "      <td> 0.306998</td>\n",
       "      <td> 0.386968</td>\n",
       "      <td> 0.066841</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.131490</td>\n",
       "      <td> 0.670253</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.602770</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr7-127288968-4</th>\n",
       "      <td> 0.111803</td>\n",
       "      <td> 0.924300</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.102245</td>\n",
       "      <td> 0.061734</td>\n",
       "      <td> 0.233315</td>\n",
       "      <td> 0.002445</td>\n",
       "      <td> 0.283307</td>\n",
       "      <td> 0.157902</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.231218</td>\n",
       "      <td> 0.252369</td>\n",
       "      <td> 0.250008</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.128411</td>\n",
       "      <td> 0.463805</td>\n",
       "      <td> 0.678447</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.780205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr4-116687122-4</th>\n",
       "      <td> 0.240608</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.635408</td>\n",
       "      <td> 0.270080</td>\n",
       "      <td> 0.348533</td>\n",
       "      <td> 0.162233</td>\n",
       "      <td> 0.185020</td>\n",
       "      <td> 0.295135</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.202543</td>\n",
       "      <td> 0.050244</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.108848</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.281493</td>\n",
       "      <td> 0.858717</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.247903</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr15-67397489-4</th>\n",
       "      <td> 0.025000</td>\n",
       "      <td> 0.250028</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.508086</td>\n",
       "      <td> 0.361231</td>\n",
       "      <td> 0.128264</td>\n",
       "      <td> 0.322020</td>\n",
       "      <td> 0.224335</td>\n",
       "      <td> 0.123593</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.274221</td>\n",
       "      <td> 0.099000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.080720</td>\n",
       "      <td> 0.776778</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.224246</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr9-44965654-4</th>\n",
       "      <td> 0.008199</td>\n",
       "      <td> 0.243004</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.066074</td>\n",
       "      <td> 0.097765</td>\n",
       "      <td> 0.057756</td>\n",
       "      <td> 0.298050</td>\n",
       "      <td> 0.373555</td>\n",
       "      <td> 0.431225</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.358208</td>\n",
       "      <td> 0.039319</td>\n",
       "      <td> 0.411550</td>\n",
       "      <td> 0.175359</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.263031</td>\n",
       "      <td> 0.678447</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.598830</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr12-40681338-4</th>\n",
       "      <td> 0.179006</td>\n",
       "      <td> 0.243004</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.452382</td>\n",
       "      <td> 0.248377</td>\n",
       "      <td> 0.246870</td>\n",
       "      <td> 0.604722</td>\n",
       "      <td> 0.302964</td>\n",
       "      <td> 0.329444</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.203204</td>\n",
       "      <td> 0.053346</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.161491</td>\n",
       "      <td> 0.670253</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.086241</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr2-29827837-4</th>\n",
       "      <td> 0.033400</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.291782</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.205253</td>\n",
       "      <td> 0.425824</td>\n",
       "      <td> 0.020668</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.141096</td>\n",
       "      <td> 0.066633</td>\n",
       "      <td> 0.123582</td>\n",
       "      <td> 0.035337</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.154568</td>\n",
       "      <td> 0.375262</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.058641</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr1-131529086-4</th>\n",
       "      <td> 0.078202</td>\n",
       "      <td> 0.643352</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.173864</td>\n",
       "      <td> 0.317825</td>\n",
       "      <td> 0.124875</td>\n",
       "      <td> 0.322020</td>\n",
       "      <td> 0.435652</td>\n",
       "      <td> 0.030471</td>\n",
       "      <td> 0.210475</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.227122</td>\n",
       "      <td> 0.110336</td>\n",
       "      <td> 0.148165</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.048411</td>\n",
       "      <td> 0.694836</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.389851</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr5-149267158-4</th>\n",
       "      <td> 0.027800</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.041875</td>\n",
       "      <td> 0.046542</td>\n",
       "      <td> 0.066074</td>\n",
       "      <td> 0.087599</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.460224</td>\n",
       "      <td> 0.172605</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.214832</td>\n",
       "      <td> 0.214129</td>\n",
       "      <td> 0.211377</td>\n",
       "      <td> 0.035337</td>\n",
       "      <td> 0.140703</td>\n",
       "      <td> 0.475345</td>\n",
       "      <td> 0.596505</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.504195</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr14-25694037-4</th>\n",
       "      <td> 0.137004</td>\n",
       "      <td> 0.250028</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.452382</td>\n",
       "      <td> 0.053053</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.199107</td>\n",
       "      <td> 0.376679</td>\n",
       "      <td> 0.167705</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.390980</td>\n",
       "      <td> 0.208667</td>\n",
       "      <td> 0.513393</td>\n",
       "      <td> 0.070342</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.239955</td>\n",
       "      <td> 0.858717</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.045874</td>\n",
       "      <td> 0.859065</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr6-4004262-4</th>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.834350</td>\n",
       "      <td> 0.287442</td>\n",
       "      <td> 0.101154</td>\n",
       "      <td> 0.162233</td>\n",
       "      <td> 0.371766</td>\n",
       "      <td> 0.300037</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.403269</td>\n",
       "      <td> 0.678472</td>\n",
       "      <td> 0.369407</td>\n",
       "      <td> 0.245370</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.274571</td>\n",
       "      <td> 0.530952</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.425338</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr11-73128611-4</th>\n",
       "      <td> 0.153805</td>\n",
       "      <td> 0.250028</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.078372</td>\n",
       "      <td> 0.296123</td>\n",
       "      <td> 0.206205</td>\n",
       "      <td> 0.192961</td>\n",
       "      <td> 0.116219</td>\n",
       "      <td> 0.182408</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.358208</td>\n",
       "      <td> 0.132187</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.112349</td>\n",
       "      <td> 0.002425</td>\n",
       "      <td> 0.173030</td>\n",
       "      <td> 0.858717</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.342535</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr6-113442528-7</th>\n",
       "      <td> 0.010999</td>\n",
       "      <td> 0.243004</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.078372</td>\n",
       "      <td> 0.326507</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.081818</td>\n",
       "      <td> 0.138297</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.169771</td>\n",
       "      <td> 0.268758</td>\n",
       "      <td> 0.341314</td>\n",
       "      <td> 0.049339</td>\n",
       "      <td> 0.070027</td>\n",
       "      <td> 0.115336</td>\n",
       "      <td> 0.858717</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.145386</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr6-122602425-7</th>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.500128</td>\n",
       "      <td> 0.435020</td>\n",
       "      <td> 0.216371</td>\n",
       "      <td> 0.119212</td>\n",
       "      <td> 0.189934</td>\n",
       "      <td> 0.437271</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.304954</td>\n",
       "      <td> 0.290609</td>\n",
       "      <td> 0.221913</td>\n",
       "      <td> 0.133352</td>\n",
       "      <td> 0.017789</td>\n",
       "      <td> 0.276879</td>\n",
       "      <td> 0.694836</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.780205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr7-126625610-7</th>\n",
       "      <td> 0.041800</td>\n",
       "      <td> 0.250028</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.048712</td>\n",
       "      <td> 0.260425</td>\n",
       "      <td> 0.795238</td>\n",
       "      <td> 0.342279</td>\n",
       "      <td> 0.496085</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.137000</td>\n",
       "      <td> 0.252369</td>\n",
       "      <td> 0.559046</td>\n",
       "      <td> 0.052839</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.586117</td>\n",
       "      <td> 0.694836</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.476597</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr19-44009097-7</th>\n",
       "      <td> 0.061401</td>\n",
       "      <td> 0.250028</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.500128</td>\n",
       "      <td> 0.374253</td>\n",
       "      <td> 0.165540</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.396337</td>\n",
       "      <td> 0.378456</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.091939</td>\n",
       "      <td> 0.252369</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.226108</td>\n",
       "      <td> 0.375262</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr5-125183792-7</th>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.243004</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.173864</td>\n",
       "      <td> 0.213653</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.309729</td>\n",
       "      <td> 0.234163</td>\n",
       "      <td> 0.363753</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.296761</td>\n",
       "      <td> 0.203204</td>\n",
       "      <td> 0.267566</td>\n",
       "      <td> 0.203363</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.306879</td>\n",
       "      <td> 0.858717</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.519970</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr16-11001914-7</th>\n",
       "      <td> 0.044601</td>\n",
       "      <td> 0.924300</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.102245</td>\n",
       "      <td> 0.183269</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.285146</td>\n",
       "      <td> 0.460224</td>\n",
       "      <td> 0.476480</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.436042</td>\n",
       "      <td> 0.170427</td>\n",
       "      <td> 0.060370</td>\n",
       "      <td> 0.059840</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.288417</td>\n",
       "      <td> 0.776778</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.272035</td>\n",
       "      <td> 0.626430</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr16-91849137-7</th>\n",
       "      <td> 0.025000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.102245</td>\n",
       "      <td> 0.365572</td>\n",
       "      <td> 0.385807</td>\n",
       "      <td> 0.186815</td>\n",
       "      <td> 0.450395</td>\n",
       "      <td> 0.339247</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.116517</td>\n",
       "      <td> 0.055707</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.125338</td>\n",
       "      <td> 0.295340</td>\n",
       "      <td> 0.154018</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.287334</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr5-53454878-7</th>\n",
       "      <td> 0.033400</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.778647</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.301090</td>\n",
       "      <td> 0.315874</td>\n",
       "      <td> 0.288222</td>\n",
       "      <td> 0.309839</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.177964</td>\n",
       "      <td> 0.705783</td>\n",
       "      <td> 0.190307</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.109974</td>\n",
       "      <td> 0.186876</td>\n",
       "      <td> 0.703030</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.062263</td>\n",
       "      <td> 0.196645</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr8-125012803-7</th>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.087599</td>\n",
       "      <td> 0.573993</td>\n",
       "      <td> 0.396337</td>\n",
       "      <td> 0.388258</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.112421</td>\n",
       "      <td> 0.525511</td>\n",
       "      <td> 0.453693</td>\n",
       "      <td> 0.420396</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.304571</td>\n",
       "      <td> 0.703030</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.307048</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr8-72303875-7</th>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.160441</td>\n",
       "      <td> 0.038584</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.182484</td>\n",
       "      <td> 0.322020</td>\n",
       "      <td> 0.204677</td>\n",
       "      <td> 0.500985</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.132903</td>\n",
       "      <td> 0.312461</td>\n",
       "      <td> 0.492321</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.299955</td>\n",
       "      <td> 0.621088</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.016375</td>\n",
       "      <td> 0.563340</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr6-21959109-7</th>\n",
       "      <td> 0.081002</td>\n",
       "      <td> 0.221934</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.046542</td>\n",
       "      <td> 0.265739</td>\n",
       "      <td> 0.216371</td>\n",
       "      <td> 0.008590</td>\n",
       "      <td> 0.347193</td>\n",
       "      <td> 0.667625</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.137000</td>\n",
       "      <td> 0.219592</td>\n",
       "      <td> 0.214889</td>\n",
       "      <td> 0.136852</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.306879</td>\n",
       "      <td> 0.694836</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.429281</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr14-98116829-7</th>\n",
       "      <td> 0.187406</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.834350</td>\n",
       "      <td> 0.382934</td>\n",
       "      <td> 0.172317</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.067075</td>\n",
       "      <td> 0.221617</td>\n",
       "      <td> 0.018342</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.115798</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.143029</td>\n",
       "      <td> 0.678447</td>\n",
       "      <td> 0.264730</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.192702</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr11-83082970-7</th>\n",
       "      <td> 0.008199</td>\n",
       "      <td> 0.924300</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.102245</td>\n",
       "      <td> 0.187610</td>\n",
       "      <td> 0.267202</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.165362</td>\n",
       "      <td> 0.329444</td>\n",
       "      <td> 0.051046</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.124710</td>\n",
       "      <td> 0.072096</td>\n",
       "      <td> 0.007693</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.027008</td>\n",
       "      <td> 0.228415</td>\n",
       "      <td> 0.621088</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.318877</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr10-81291937-7</th>\n",
       "      <td> 0.086602</td>\n",
       "      <td> 0.243004</td>\n",
       "      <td> 0.073685</td>\n",
       "      <td> 0.635408</td>\n",
       "      <td> 0.408977</td>\n",
       "      <td> 0.362086</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.401252</td>\n",
       "      <td> 0.295135</td>\n",
       "      <td> 0.014254</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.100132</td>\n",
       "      <td> 0.645694</td>\n",
       "      <td> 0.211377</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.200723</td>\n",
       "      <td> 0.776778</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.228188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr11-116433774-7</th>\n",
       "      <td> 0.137004</td>\n",
       "      <td> 0.924300</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.778647</td>\n",
       "      <td> 0.348210</td>\n",
       "      <td> 0.229926</td>\n",
       "      <td> 0.088484</td>\n",
       "      <td> 0.180105</td>\n",
       "      <td> 0.319642</td>\n",
       "      <td> 0.235003</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.497488</td>\n",
       "      <td> 0.274221</td>\n",
       "      <td> 0.253519</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.161491</td>\n",
       "      <td> 0.858717</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.701345</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr13-3541637-7</th>\n",
       "      <td> 0.201407</td>\n",
       "      <td> 0.580139</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.508086</td>\n",
       "      <td> 0.304804</td>\n",
       "      <td> 0.253648</td>\n",
       "      <td> 0.156087</td>\n",
       "      <td> 0.278392</td>\n",
       "      <td> 0.255926</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.210736</td>\n",
       "      <td> 0.033856</td>\n",
       "      <td> 0.341314</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.113028</td>\n",
       "      <td> 0.678447</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.121728</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr5-120558954-7</th>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.070794</td>\n",
       "      <td> 0.635408</td>\n",
       "      <td> 0.048712</td>\n",
       "      <td> 0.236704</td>\n",
       "      <td> 0.106921</td>\n",
       "      <td> 0.234163</td>\n",
       "      <td> 0.603910</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.296761</td>\n",
       "      <td> 0.328849</td>\n",
       "      <td> 0.355361</td>\n",
       "      <td> 0.157856</td>\n",
       "      <td> 0.060809</td>\n",
       "      <td> 0.336879</td>\n",
       "      <td> 0.858717</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.022930</td>\n",
       "      <td> 0.665860</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr1-162570598-7</th>\n",
       "      <td> 0.005399</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.053443</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.801381</td>\n",
       "      <td> 0.529025</td>\n",
       "      <td> 0.481382</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.165675</td>\n",
       "      <td> 0.334312</td>\n",
       "      <td> 0.436132</td>\n",
       "      <td> 0.087845</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.493807</td>\n",
       "      <td> 0.539146</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.598830</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr7-19149543-7</th>\n",
       "      <td> 0.044601</td>\n",
       "      <td> 0.250028</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.452382</td>\n",
       "      <td> 0.356890</td>\n",
       "      <td> 0.189261</td>\n",
       "      <td> 0.285146</td>\n",
       "      <td> 0.361936</td>\n",
       "      <td> 0.437271</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.317244</td>\n",
       "      <td> 0.487271</td>\n",
       "      <td> 0.513393</td>\n",
       "      <td> 0.465904</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.184568</td>\n",
       "      <td> 0.858717</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.026208</td>\n",
       "      <td> 0.460825</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr1-182108730-7</th>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.102245</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.080821</td>\n",
       "      <td> 0.002445</td>\n",
       "      <td> 0.160448</td>\n",
       "      <td> 0.251025</td>\n",
       "      <td> 0.051046</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.276279</td>\n",
       "      <td> 0.246906</td>\n",
       "      <td> 0.169236</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.283802</td>\n",
       "      <td> 0.703030</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.330706</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr10-80702354-7</th>\n",
       "      <td> 0.237808</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.834350</td>\n",
       "      <td> 0.270080</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.192961</td>\n",
       "      <td> 0.219420</td>\n",
       "      <td> 0.304938</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.423750</td>\n",
       "      <td> 0.569211</td>\n",
       "      <td> 0.660889</td>\n",
       "      <td> 0.175359</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.316110</td>\n",
       "      <td> 0.858717</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.150761</td>\n",
       "      <td> 0.571230</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr5-151368615-7</th>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.243004</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.834350</td>\n",
       "      <td> 0.257058</td>\n",
       "      <td> 0.128264</td>\n",
       "      <td> 0.573993</td>\n",
       "      <td> 0.361936</td>\n",
       "      <td> 0.285334</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.272182</td>\n",
       "      <td> 0.219592</td>\n",
       "      <td> 0.650354</td>\n",
       "      <td> 0.098346</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.200723</td>\n",
       "      <td> 0.858717</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.437167</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr17-74570654-7</th>\n",
       "      <td> 0.271409</td>\n",
       "      <td> 0.924300</td>\n",
       "      <td> 0.062118</td>\n",
       "      <td> 0.500128</td>\n",
       "      <td> 0.435020</td>\n",
       "      <td> 0.341755</td>\n",
       "      <td> 0.192961</td>\n",
       "      <td> 0.298050</td>\n",
       "      <td> 0.329444</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.206639</td>\n",
       "      <td> 0.290609</td>\n",
       "      <td> 0.102512</td>\n",
       "      <td> 0.119350</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.055335</td>\n",
       "      <td> 0.383456</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.283390</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr10-26564705-7</th>\n",
       "      <td> 0.128604</td>\n",
       "      <td> 0.924300</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.778647</td>\n",
       "      <td> 0.426339</td>\n",
       "      <td> 0.395975</td>\n",
       "      <td> 0.205253</td>\n",
       "      <td> 0.420909</td>\n",
       "      <td> 0.809760</td>\n",
       "      <td> 0.296322</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.186157</td>\n",
       "      <td> 0.246906</td>\n",
       "      <td> 0.257031</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.276879</td>\n",
       "      <td> 0.858717</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.216314</td>\n",
       "      <td> 0.701345</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr14-17414520-7</th>\n",
       "      <td> 0.033400</td>\n",
       "      <td> 0.924300</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.778647</td>\n",
       "      <td> 0.139863</td>\n",
       "      <td> 0.192650</td>\n",
       "      <td> 0.297437</td>\n",
       "      <td> 0.489710</td>\n",
       "      <td> 0.476480</td>\n",
       "      <td> 0.014254</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.112421</td>\n",
       "      <td> 0.274221</td>\n",
       "      <td> 0.200842</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.078412</td>\n",
       "      <td> 0.670253</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.267618</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr7-4239319-7</th>\n",
       "      <td> 0.125804</td>\n",
       "      <td> 0.250028</td>\n",
       "      <td> 0.053443</td>\n",
       "      <td> 0.778647</td>\n",
       "      <td> 0.365572</td>\n",
       "      <td> 0.236704</td>\n",
       "      <td> 0.322020</td>\n",
       "      <td> 0.204677</td>\n",
       "      <td> 0.368653</td>\n",
       "      <td> 0.365817</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.231218</td>\n",
       "      <td> 0.110336</td>\n",
       "      <td> 0.425596</td>\n",
       "      <td> 0.245370</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.332264</td>\n",
       "      <td> 0.850525</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.032763</td>\n",
       "      <td> 0.322820</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr10-81303724-7</th>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.250028</td>\n",
       "      <td> 0.012957</td>\n",
       "      <td> 0.173864</td>\n",
       "      <td> 0.174588</td>\n",
       "      <td> 0.104542</td>\n",
       "      <td> 0.205253</td>\n",
       "      <td> 0.474967</td>\n",
       "      <td> 0.471579</td>\n",
       "      <td> 0.598829</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.390980</td>\n",
       "      <td> 0.416254</td>\n",
       "      <td> 0.313220</td>\n",
       "      <td> 0.133352</td>\n",
       "      <td> 0.128411</td>\n",
       "      <td> 0.429190</td>\n",
       "      <td> 0.858717</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.249091</td>\n",
       "      <td> 0.594885</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr10-80051910-7</th>\n",
       "      <td> 0.092202</td>\n",
       "      <td> 0.243004</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.046542</td>\n",
       "      <td> 0.148545</td>\n",
       "      <td> 0.158762</td>\n",
       "      <td> 0.309729</td>\n",
       "      <td> 0.298050</td>\n",
       "      <td> 0.510790</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.309051</td>\n",
       "      <td> 0.334312</td>\n",
       "      <td> 0.274590</td>\n",
       "      <td> 0.133352</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.500729</td>\n",
       "      <td> 0.678447</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.278591</td>\n",
       "      <td> 0.831465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr2-132229938-7</th>\n",
       "      <td> 0.260209</td>\n",
       "      <td> 0.299194</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.778647</td>\n",
       "      <td> 0.348210</td>\n",
       "      <td> 0.240092</td>\n",
       "      <td> 0.309729</td>\n",
       "      <td> 0.430738</td>\n",
       "      <td> 0.721540</td>\n",
       "      <td> 0.010166</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.210736</td>\n",
       "      <td> 0.836894</td>\n",
       "      <td> 0.134118</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.473036</td>\n",
       "      <td> 0.858717</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.165101</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>Merged-chr6-97856502-7</th>\n",
       "      <td> 0.013799</td>\n",
       "      <td> 0.250028</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.691113</td>\n",
       "      <td> 0.465405</td>\n",
       "      <td> 0.348533</td>\n",
       "      <td> 0.666181</td>\n",
       "      <td> 0.642055</td>\n",
       "      <td> 0.148099</td>\n",
       "      <td> 0.046957</td>\n",
       "      <td>...</td>\n",
       "      <td> 0.227122</td>\n",
       "      <td> 0.203204</td>\n",
       "      <td> 0.162213</td>\n",
       "      <td> 0.108848</td>\n",
       "      <td> 0.000000</td>\n",
       "      <td> 0.156875</td>\n",
       "      <td> 0.678447</td>\n",
       "      <td> 0.105671</td>\n",
       "      <td> 0.363810</td>\n",
       "      <td> 0.496312</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>93261 rows  196 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                          alx1_alx4_arx      ap-1  ar_nr3c1_nr3c2    arid3a  \\\n",
       "Merged-chr17-34627665-3        0.125804  0.000000        0.000000  0.508086   \n",
       "Merged-chr16-57200743-3        0.053001  0.924300        0.000000  0.508086   \n",
       "Merged-chr9-70560828-3         0.000000  0.243004        0.027416  0.778647   \n",
       "Merged-chr5-30893945-3         0.000000  0.000000        0.000000  0.038584   \n",
       "Merged-chr16-87636040-3        0.000000  0.643352        0.310815  0.500128   \n",
       "Merged-chr7-98535313-3         0.058601  0.243004        0.000000  0.000000   \n",
       "Merged-chr13-67484493-3        0.170606  0.243004        0.024524  0.635408   \n",
       "Merged-chr3-90604533-3         0.000000  0.530974        0.000000  0.834350   \n",
       "Merged-chr17-26676427-3        0.000000  0.250028        0.000000  0.778647   \n",
       "Merged-chr1-97769642-3         0.263009  0.000000        0.000000  0.452382   \n",
       "Merged-chr7-104446416-3        0.000000  0.250028        0.000000  0.635408   \n",
       "Merged-chr11-102156353-3       0.106203  0.243004        0.270330  0.046542   \n",
       "Merged-chr7-59027671-3         0.243409  0.924300        0.093928  0.691113   \n",
       "Merged-chr4-41319130-3         0.030600  0.243004        0.000000  0.834350   \n",
       "Merged-chr11-119602934-4       0.176206  0.000000        0.082361  0.452382   \n",
       "Merged-chr12-73907835-4        0.078202  0.000000        0.000000  0.000000   \n",
       "Merged-chr11-17212255-4        0.000000  0.000000        0.160441  0.102245   \n",
       "Merged-chr10-81545316-4        0.041800  0.000000        0.000000  0.000000   \n",
       "Merged-chr7-12978159-4         0.000000  0.250028        0.000000  0.000000   \n",
       "Merged-chr7-127288968-4        0.111803  0.924300        0.000000  0.102245   \n",
       "Merged-chr4-116687122-4        0.240608  0.000000        0.000000  0.635408   \n",
       "Merged-chr15-67397489-4        0.025000  0.250028        0.000000  0.508086   \n",
       "Merged-chr9-44965654-4         0.008199  0.243004        0.000000  0.000000   \n",
       "Merged-chr12-40681338-4        0.179006  0.243004        0.000000  0.452382   \n",
       "Merged-chr2-29827837-4         0.033400  0.000000        0.000000  0.000000   \n",
       "Merged-chr1-131529086-4        0.078202  0.643352        0.000000  0.173864   \n",
       "Merged-chr5-149267158-4        0.027800  0.000000        0.041875  0.046542   \n",
       "Merged-chr14-25694037-4        0.137004  0.250028        0.000000  0.452382   \n",
       "Merged-chr6-4004262-4          0.000000  0.000000        0.000000  0.834350   \n",
       "Merged-chr11-73128611-4        0.153805  0.250028        0.000000  0.078372   \n",
       "...                                 ...       ...             ...       ...   \n",
       "Merged-chr6-113442528-7        0.010999  0.243004        0.000000  0.078372   \n",
       "Merged-chr6-122602425-7        0.000000  0.000000        0.000000  0.500128   \n",
       "Merged-chr7-126625610-7        0.041800  0.250028        0.000000  0.000000   \n",
       "Merged-chr19-44009097-7        0.061401  0.250028        0.000000  0.500128   \n",
       "Merged-chr5-125183792-7        0.000000  0.243004        0.000000  0.173864   \n",
       "Merged-chr16-11001914-7        0.044601  0.924300        0.000000  0.102245   \n",
       "Merged-chr16-91849137-7        0.025000  0.000000        0.000000  0.102245   \n",
       "Merged-chr5-53454878-7         0.033400  0.000000        0.000000  0.778647   \n",
       "Merged-chr8-125012803-7        0.000000  0.000000        0.000000  0.000000   \n",
       "Merged-chr8-72303875-7         0.000000  0.000000        0.160441  0.038584   \n",
       "Merged-chr6-21959109-7         0.081002  0.221934        0.000000  0.046542   \n",
       "Merged-chr14-98116829-7        0.187406  0.000000        0.000000  0.834350   \n",
       "Merged-chr11-83082970-7        0.008199  0.924300        0.000000  0.102245   \n",
       "Merged-chr10-81291937-7        0.086602  0.243004        0.073685  0.635408   \n",
       "Merged-chr11-116433774-7       0.137004  0.924300        0.000000  0.778647   \n",
       "Merged-chr13-3541637-7         0.201407  0.580139        0.000000  0.508086   \n",
       "Merged-chr5-120558954-7        0.000000  0.000000        0.070794  0.635408   \n",
       "Merged-chr1-162570598-7        0.005399  0.000000        0.053443  0.000000   \n",
       "Merged-chr7-19149543-7         0.044601  0.250028        0.000000  0.452382   \n",
       "Merged-chr1-182108730-7        0.000000  0.000000        0.000000  0.102245   \n",
       "Merged-chr10-80702354-7        0.237808  0.000000        0.000000  0.834350   \n",
       "Merged-chr5-151368615-7        0.000000  0.243004        0.000000  0.834350   \n",
       "Merged-chr17-74570654-7        0.271409  0.924300        0.062118  0.500128   \n",
       "Merged-chr10-26564705-7        0.128604  0.924300        0.000000  0.778647   \n",
       "Merged-chr14-17414520-7        0.033400  0.924300        0.000000  0.778647   \n",
       "Merged-chr7-4239319-7          0.125804  0.250028        0.053443  0.778647   \n",
       "Merged-chr10-81303724-7        0.000000  0.250028        0.012957  0.173864   \n",
       "Merged-chr10-80051910-7        0.092202  0.243004        0.000000  0.046542   \n",
       "Merged-chr2-132229938-7        0.260209  0.299194        0.000000  0.778647   \n",
       "Merged-chr6-97856502-7         0.013799  0.250028        0.000000  0.691113   \n",
       "\n",
       "                            arid3b    arid5a  arnt_mycn  arntl_mitf  \\\n",
       "Merged-chr17-34627665-3   0.174588  0.172317   0.063901    0.376679   \n",
       "Merged-chr16-57200743-3   0.343869  0.250259   0.057756    0.224335   \n",
       "Merged-chr9-70560828-3    0.044371  0.338366   0.315874    0.352108   \n",
       "Merged-chr5-30893945-3    0.165907  0.145207   0.285146    0.302964   \n",
       "Merged-chr16-87636040-3   0.313485  0.297701   0.000000    0.081818   \n",
       "Merged-chr7-98535313-3    0.000000  0.101154   0.113067    0.307878   \n",
       "Merged-chr13-67484493-3   0.382934  0.290924   0.475662    0.317707   \n",
       "Merged-chr3-90604533-3    0.387275  0.104542   0.113067    0.067075   \n",
       "Merged-chr17-26676427-3   0.000000  0.000000   0.303583    0.656800   \n",
       "Merged-chr1-97769642-3    0.000000  0.223149   0.899713    0.524110   \n",
       "Merged-chr7-104446416-3   0.000000  0.294313   0.469517    0.465138   \n",
       "Merged-chr11-102156353-3  0.000000  0.138430   0.180669    0.479882   \n",
       "Merged-chr7-59027671-3    0.482768  0.206205   0.000000    0.234163   \n",
       "Merged-chr4-41319130-3    0.382934  0.307867   0.199107    0.229249   \n",
       "Merged-chr11-119602934-4  0.291782  0.290924   0.285146    0.411081   \n",
       "Merged-chr12-73907835-4   0.000000  0.000000   0.770656    0.656800   \n",
       "Merged-chr11-17212255-4   0.083436  0.182484   0.186815    0.185020   \n",
       "Merged-chr10-81545316-4   0.000000  0.000000   0.334312    0.371766   \n",
       "Merged-chr7-12978159-4    0.200631  0.000000   0.205253    0.465138   \n",
       "Merged-chr7-127288968-4   0.061734  0.233315   0.002445    0.283307   \n",
       "Merged-chr4-116687122-4   0.270080  0.348533   0.162233    0.185020   \n",
       "Merged-chr15-67397489-4   0.361231  0.128264   0.322020    0.224335   \n",
       "Merged-chr9-44965654-4    0.066074  0.097765   0.057756    0.298050   \n",
       "Merged-chr12-40681338-4   0.248377  0.246870   0.604722    0.302964   \n",
       "Merged-chr2-29827837-4    0.291782  0.000000   0.205253    0.425824   \n",
       "Merged-chr1-131529086-4   0.317825  0.124875   0.322020    0.435652   \n",
       "Merged-chr5-149267158-4   0.066074  0.087599   0.000000    0.460224   \n",
       "Merged-chr14-25694037-4   0.053053  0.000000   0.199107    0.376679   \n",
       "Merged-chr6-4004262-4     0.287442  0.101154   0.162233    0.371766   \n",
       "Merged-chr11-73128611-4   0.296123  0.206205   0.192961    0.116219   \n",
       "...                            ...       ...        ...         ...   \n",
       "Merged-chr6-113442528-7   0.326507  0.000000   0.000000    0.081818   \n",
       "Merged-chr6-122602425-7   0.435020  0.216371   0.119212    0.189934   \n",
       "Merged-chr7-126625610-7   0.048712  0.260425   0.795238    0.342279   \n",
       "Merged-chr19-44009097-7   0.374253  0.165540   0.000000    0.396337   \n",
       "Merged-chr5-125183792-7   0.213653  0.000000   0.309729    0.234163   \n",
       "Merged-chr16-11001914-7   0.183269  0.000000   0.285146    0.460224   \n",
       "Merged-chr16-91849137-7   0.365572  0.385807   0.186815    0.450395   \n",
       "Merged-chr5-53454878-7    0.000000  0.301090   0.315874    0.288222   \n",
       "Merged-chr8-125012803-7   0.000000  0.087599   0.573993    0.396337   \n",
       "Merged-chr8-72303875-7    0.000000  0.182484   0.322020    0.204677   \n",
       "Merged-chr6-21959109-7    0.265739  0.216371   0.008590    0.347193   \n",
       "Merged-chr14-98116829-7   0.382934  0.172317   0.000000    0.067075   \n",
       "Merged-chr11-83082970-7   0.187610  0.267202   0.000000    0.165362   \n",
       "Merged-chr10-81291937-7   0.408977  0.362086   0.000000    0.401252   \n",
       "Merged-chr11-116433774-7  0.348210  0.229926   0.088484    0.180105   \n",
       "Merged-chr13-3541637-7    0.304804  0.253648   0.156087    0.278392   \n",
       "Merged-chr5-120558954-7   0.048712  0.236704   0.106921    0.234163   \n",
       "Merged-chr1-162570598-7   0.000000  0.000000   0.801381    0.529025   \n",
       "Merged-chr7-19149543-7    0.356890  0.189261   0.285146    0.361936   \n",
       "Merged-chr1-182108730-7   0.000000  0.080821   0.002445    0.160448   \n",
       "Merged-chr10-80702354-7   0.270080  0.000000   0.192961    0.219420   \n",
       "Merged-chr5-151368615-7   0.257058  0.128264   0.573993    0.361936   \n",
       "Merged-chr17-74570654-7   0.435020  0.341755   0.192961    0.298050   \n",
       "Merged-chr10-26564705-7   0.426339  0.395975   0.205253    0.420909   \n",
       "Merged-chr14-17414520-7   0.139863  0.192650   0.297437    0.489710   \n",
       "Merged-chr7-4239319-7     0.365572  0.236704   0.322020    0.204677   \n",
       "Merged-chr10-81303724-7   0.174588  0.104542   0.205253    0.474967   \n",
       "Merged-chr10-80051910-7   0.148545  0.158762   0.309729    0.298050   \n",
       "Merged-chr2-132229938-7   0.348210  0.240092   0.309729    0.430738   \n",
       "Merged-chr6-97856502-7    0.465405  0.348533   0.666181    0.642055   \n",
       "\n",
       "                          ascl2_nhlh1  atf7_batf3_creb5        ...         \\\n",
       "Merged-chr17-34627665-3      0.304938          0.000000        ...          \n",
       "Merged-chr16-57200743-3      0.329444          0.000000        ...          \n",
       "Merged-chr9-70560828-3       0.314740          0.000000        ...          \n",
       "Merged-chr5-30893945-3       0.167705          0.000000        ...          \n",
       "Merged-chr16-87636040-3      0.226519          0.574300        ...          \n",
       "Merged-chr7-98535313-3       0.270629          0.251354        ...          \n",
       "Merged-chr13-67484493-3      0.211815          0.615179        ...          \n",
       "Merged-chr3-90604533-3       0.589210          0.000000        ...          \n",
       "Merged-chr17-26676427-3      0.285334          0.000000        ...          \n",
       "Merged-chr1-97769642-3       0.309839          0.000000        ...          \n",
       "Merged-chr7-104446416-3      0.471579          0.296322        ...          \n",
       "Merged-chr11-102156353-3     0.500985          0.000000        ...          \n",
       "Merged-chr7-59027671-3       0.466678          0.312673        ...          \n",
       "Merged-chr4-41319130-3       0.241222          0.091925        ...          \n",
       "Merged-chr11-119602934-4     0.231420          0.059222        ...          \n",
       "Merged-chr12-73907835-4      0.309839          0.000000        ...          \n",
       "Merged-chr11-17212255-4      0.505885          0.000000        ...          \n",
       "Merged-chr10-81545316-4      0.255926          0.000000        ...          \n",
       "Merged-chr7-12978159-4       0.437271          0.026518        ...          \n",
       "Merged-chr7-127288968-4      0.157902          0.000000        ...          \n",
       "Merged-chr4-116687122-4      0.295135          0.000000        ...          \n",
       "Merged-chr15-67397489-4      0.123593          0.000000        ...          \n",
       "Merged-chr9-44965654-4       0.373555          0.431225        ...          \n",
       "Merged-chr12-40681338-4      0.329444          0.000000        ...          \n",
       "Merged-chr2-29827837-4       0.020668          0.000000        ...          \n",
       "Merged-chr1-131529086-4      0.030471          0.210475        ...          \n",
       "Merged-chr5-149267158-4      0.172605          0.000000        ...          \n",
       "Merged-chr14-25694037-4      0.167705          0.000000        ...          \n",
       "Merged-chr6-4004262-4        0.300037          0.000000        ...          \n",
       "Merged-chr11-73128611-4      0.182408          0.000000        ...          \n",
       "...                               ...               ...        ...          \n",
       "Merged-chr6-113442528-7      0.138297          0.000000        ...          \n",
       "Merged-chr6-122602425-7      0.437271          0.000000        ...          \n",
       "Merged-chr7-126625610-7      0.496085          0.000000        ...          \n",
       "Merged-chr19-44009097-7      0.378456          0.000000        ...          \n",
       "Merged-chr5-125183792-7      0.363753          0.000000        ...          \n",
       "Merged-chr16-11001914-7      0.476480          0.000000        ...          \n",
       "Merged-chr16-91849137-7      0.339247          0.000000        ...          \n",
       "Merged-chr5-53454878-7       0.309839          0.000000        ...          \n",
       "Merged-chr8-125012803-7      0.388258          0.000000        ...          \n",
       "Merged-chr8-72303875-7       0.500985          0.000000        ...          \n",
       "Merged-chr6-21959109-7       0.667625          0.000000        ...          \n",
       "Merged-chr14-98116829-7      0.221617          0.018342        ...          \n",
       "Merged-chr11-83082970-7      0.329444          0.051046        ...          \n",
       "Merged-chr10-81291937-7      0.295135          0.014254        ...          \n",
       "Merged-chr11-116433774-7     0.319642          0.235003        ...          \n",
       "Merged-chr13-3541637-7       0.255926          0.000000        ...          \n",
       "Merged-chr5-120558954-7      0.603910          0.000000        ...          \n",
       "Merged-chr1-162570598-7      0.481382          0.000000        ...          \n",
       "Merged-chr7-19149543-7       0.437271          0.000000        ...          \n",
       "Merged-chr1-182108730-7      0.251025          0.051046        ...          \n",
       "Merged-chr10-80702354-7      0.304938          0.000000        ...          \n",
       "Merged-chr5-151368615-7      0.285334          0.000000        ...          \n",
       "Merged-chr17-74570654-7      0.329444          0.000000        ...          \n",
       "Merged-chr10-26564705-7      0.809760          0.296322        ...          \n",
       "Merged-chr14-17414520-7      0.476480          0.014254        ...          \n",
       "Merged-chr7-4239319-7        0.368653          0.365817        ...          \n",
       "Merged-chr10-81303724-7      0.471579          0.598829        ...          \n",
       "Merged-chr10-80051910-7      0.510790          0.000000        ...          \n",
       "Merged-chr2-132229938-7      0.721540          0.010166        ...          \n",
       "Merged-chr6-97856502-7       0.148099          0.046957        ...          \n",
       "\n",
       "                             zbtb7      zeb1       zfx       zic    znf143  \\\n",
       "Merged-chr17-34627665-3   0.329533  0.263295  0.327267  0.052839  0.000000   \n",
       "Merged-chr16-57200743-3   0.354112  0.285146  0.274590  0.080844  0.000000   \n",
       "Merged-chr9-70560828-3    0.000000  0.132187  0.134118  0.014333  0.000000   \n",
       "Merged-chr5-30893945-3    0.059167  0.487271  0.102512  0.000000  0.000000   \n",
       "Merged-chr16-87636040-3   0.063263  0.312461  0.274590  0.000000  0.000000   \n",
       "Merged-chr7-98535313-3    0.231218  0.164964  0.084953  0.000000  0.103829   \n",
       "Merged-chr13-67484493-3   0.227122  0.230518  0.264055  0.052839  0.000000   \n",
       "Merged-chr3-90604533-3    0.198446  0.208667  0.527439  0.038837  0.000000   \n",
       "Merged-chr17-26676427-3   0.292665  0.678472  0.351850  0.038837  0.103829   \n",
       "Merged-chr1-97769642-3    0.075553  0.252369  0.334291  0.185860  0.000000   \n",
       "Merged-chr7-104446416-3   0.313147  0.836894  0.358875  0.161356  0.000000   \n",
       "Merged-chr11-102156353-3  0.358208  0.290609  0.425596  0.133352  0.000000   \n",
       "Merged-chr7-59027671-3    0.000000  0.181352  0.028764  0.042338  0.000000   \n",
       "Merged-chr4-41319130-3    0.000000  0.317923  0.169236  0.000000  0.128411   \n",
       "Merged-chr11-119602934-4  0.177964  0.268758  0.408039  0.203363  0.346581   \n",
       "Merged-chr12-73907835-4   0.263990  0.858744  0.520418  0.420396  0.000000   \n",
       "Merged-chr11-17212255-4   0.329533  0.645694  0.260543  0.136852  0.343509   \n",
       "Merged-chr10-81545316-4   0.325437  0.000000  0.622257  0.066841  0.272834   \n",
       "Merged-chr7-12978159-4    0.313147  0.306998  0.386968  0.066841  0.000000   \n",
       "Merged-chr7-127288968-4   0.231218  0.252369  0.250008  0.000000  0.128411   \n",
       "Merged-chr4-116687122-4   0.202543  0.050244  0.000000  0.108848  0.000000   \n",
       "Merged-chr15-67397489-4   0.000000  0.274221  0.099000  0.000000  0.000000   \n",
       "Merged-chr9-44965654-4    0.358208  0.039319  0.411550  0.175359  0.000000   \n",
       "Merged-chr12-40681338-4   0.000000  0.203204  0.053346  0.000000  0.000000   \n",
       "Merged-chr2-29827837-4    0.141096  0.066633  0.123582  0.035337  0.000000   \n",
       "Merged-chr1-131529086-4   0.227122  0.110336  0.148165  0.000000  0.000000   \n",
       "Merged-chr5-149267158-4   0.214832  0.214129  0.211377  0.035337  0.140703   \n",
       "Merged-chr14-25694037-4   0.390980  0.208667  0.513393  0.070342  0.000000   \n",
       "Merged-chr6-4004262-4     0.403269  0.678472  0.369407  0.245370  0.000000   \n",
       "Merged-chr11-73128611-4   0.358208  0.132187  0.000000  0.112349  0.002425   \n",
       "...                            ...       ...       ...       ...       ...   \n",
       "Merged-chr6-113442528-7   0.169771  0.268758  0.341314  0.049339  0.070027   \n",
       "Merged-chr6-122602425-7   0.304954  0.290609  0.221913  0.133352  0.017789   \n",
       "Merged-chr7-126625610-7   0.137000  0.252369  0.559046  0.052839  0.000000   \n",
       "Merged-chr19-44009097-7   0.091939  0.252369  0.000000  0.000000  0.000000   \n",
       "Merged-chr5-125183792-7   0.296761  0.203204  0.267566  0.203363  0.000000   \n",
       "Merged-chr16-11001914-7   0.436042  0.170427  0.060370  0.059840  0.000000   \n",
       "Merged-chr16-91849137-7   0.116517  0.055707  0.000000  0.000000  0.125338   \n",
       "Merged-chr5-53454878-7    0.177964  0.705783  0.190307  0.000000  0.109974   \n",
       "Merged-chr8-125012803-7   0.112421  0.525511  0.453693  0.420396  0.000000   \n",
       "Merged-chr8-72303875-7    0.132903  0.312461  0.492321  0.000000  0.000000   \n",
       "Merged-chr6-21959109-7    0.137000  0.219592  0.214889  0.136852  0.000000   \n",
       "Merged-chr14-98116829-7   0.000000  0.115798  0.000000  0.000000  0.000000   \n",
       "Merged-chr11-83082970-7   0.124710  0.072096  0.007693  0.000000  0.027008   \n",
       "Merged-chr10-81291937-7   0.100132  0.645694  0.211377  0.000000  0.000000   \n",
       "Merged-chr11-116433774-7  0.497488  0.274221  0.253519  0.000000  0.000000   \n",
       "Merged-chr13-3541637-7    0.210736  0.033856  0.341314  0.000000  0.000000   \n",
       "Merged-chr5-120558954-7   0.296761  0.328849  0.355361  0.157856  0.060809   \n",
       "Merged-chr1-162570598-7   0.165675  0.334312  0.436132  0.087845  0.000000   \n",
       "Merged-chr7-19149543-7    0.317244  0.487271  0.513393  0.465904  0.000000   \n",
       "Merged-chr1-182108730-7   0.276279  0.246906  0.169236  0.000000  0.000000   \n",
       "Merged-chr10-80702354-7   0.423750  0.569211  0.660889  0.175359  0.000000   \n",
       "Merged-chr5-151368615-7   0.272182  0.219592  0.650354  0.098346  0.000000   \n",
       "Merged-chr17-74570654-7   0.206639  0.290609  0.102512  0.119350  0.000000   \n",
       "Merged-chr10-26564705-7   0.186157  0.246906  0.257031  0.000000  0.000000   \n",
       "Merged-chr14-17414520-7   0.112421  0.274221  0.200842  0.000000  0.000000   \n",
       "Merged-chr7-4239319-7     0.231218  0.110336  0.425596  0.245370  0.000000   \n",
       "Merged-chr10-81303724-7   0.390980  0.416254  0.313220  0.133352  0.128411   \n",
       "Merged-chr10-80051910-7   0.309051  0.334312  0.274590  0.133352  0.000000   \n",
       "Merged-chr2-132229938-7   0.210736  0.836894  0.134118  0.000000  0.000000   \n",
       "Merged-chr6-97856502-7    0.227122  0.203204  0.162213  0.108848  0.000000   \n",
       "\n",
       "                            znf263   znf354c    znf410    znf423    znf740  \n",
       "Merged-chr17-34627665-3   0.182261  0.596505  0.000000  0.714523  0.831465  \n",
       "Merged-chr16-57200743-3   0.110720  0.375262  0.000000  0.000000  0.445052  \n",
       "Merged-chr9-70560828-3    0.115336  0.678447  0.000000  0.003264  0.267618  \n",
       "Merged-chr5-30893945-3    0.182261  0.858717  0.000000  0.000000  0.240017  \n",
       "Merged-chr16-87636040-3   0.274571  0.858717  0.000000  0.000000  0.208474  \n",
       "Merged-chr7-98535313-3    0.226108  0.678447  0.000000  0.000000  0.255790  \n",
       "Merged-chr13-67484493-3   0.046104  0.621088  0.000000  0.000000  0.000000  \n",
       "Merged-chr3-90604533-3    0.332264  0.694836  0.000000  0.009819  0.429281  \n",
       "Merged-chr17-26676427-3   0.426881  0.858717  0.000000  0.294979  0.342535  \n",
       "Merged-chr1-97769642-3    0.156875  0.858717  0.000000  0.039319  0.393794  \n",
       "Merged-chr7-104446416-3   0.286110  0.703030  0.000000  0.000000  0.519970  \n",
       "Merged-chr11-102156353-3  0.170722  0.678447  0.000000  0.176982  0.598830  \n",
       "Merged-chr7-59027671-3    0.205338  0.621088  0.000000  0.000000  0.050755  \n",
       "Merged-chr4-41319130-3    0.443036  0.776778  0.000000  0.065540  0.460825  \n",
       "Merged-chr11-119602934-4  0.323033  0.858717  0.000000  0.000000  0.417452  \n",
       "Merged-chr12-73907835-4   0.320726  0.678447  0.000000  0.000000  0.859065  \n",
       "Merged-chr11-17212255-4   0.431498  0.776778  0.000000  0.000000  0.255790  \n",
       "Merged-chr10-81545316-4   0.235339  0.776778  0.000000  0.000000  0.504195  \n",
       "Merged-chr7-12978159-4    0.131490  0.670253  0.000000  0.000000  0.602770  \n",
       "Merged-chr7-127288968-4   0.463805  0.678447  0.000000  0.000000  0.780205  \n",
       "Merged-chr4-116687122-4   0.281493  0.858717  0.000000  0.000000  0.247903  \n",
       "Merged-chr15-67397489-4   0.080720  0.776778  0.000000  0.000000  0.224246  \n",
       "Merged-chr9-44965654-4    0.263031  0.678447  0.000000  0.000000  0.598830  \n",
       "Merged-chr12-40681338-4   0.161491  0.670253  0.000000  0.000000  0.086241  \n",
       "Merged-chr2-29827837-4    0.154568  0.375262  0.000000  0.000000  0.058641  \n",
       "Merged-chr1-131529086-4   0.048411  0.694836  0.000000  0.000000  0.389851  \n",
       "Merged-chr5-149267158-4   0.475345  0.596505  0.000000  0.000000  0.504195  \n",
       "Merged-chr14-25694037-4   0.239955  0.858717  0.000000  0.045874  0.859065  \n",
       "Merged-chr6-4004262-4     0.274571  0.530952  0.000000  0.000000  0.425338  \n",
       "Merged-chr11-73128611-4   0.173030  0.858717  0.000000  0.000000  0.342535  \n",
       "...                            ...       ...       ...       ...       ...  \n",
       "Merged-chr6-113442528-7   0.115336  0.858717  0.000000  0.000000  0.145386  \n",
       "Merged-chr6-122602425-7   0.276879  0.694836  0.000000  0.000000  0.780205  \n",
       "Merged-chr7-126625610-7   0.586117  0.694836  0.000000  0.000000  0.476597  \n",
       "Merged-chr19-44009097-7   0.226108  0.375262  0.000000  0.000000  0.000000  \n",
       "Merged-chr5-125183792-7   0.306879  0.858717  0.000000  0.000000  0.519970  \n",
       "Merged-chr16-11001914-7   0.288417  0.776778  0.000000  0.272035  0.626430  \n",
       "Merged-chr16-91849137-7   0.295340  0.154018  0.000000  0.000000  0.287334  \n",
       "Merged-chr5-53454878-7    0.186876  0.703030  0.000000  0.062263  0.196645  \n",
       "Merged-chr8-125012803-7   0.304571  0.703030  0.000000  0.000000  0.307048  \n",
       "Merged-chr8-72303875-7    0.299955  0.621088  0.000000  0.016375  0.563340  \n",
       "Merged-chr6-21959109-7    0.306879  0.694836  0.000000  0.000000  0.429281  \n",
       "Merged-chr14-98116829-7   0.143029  0.678447  0.264730  0.000000  0.192702  \n",
       "Merged-chr11-83082970-7   0.228415  0.621088  0.000000  0.000000  0.318877  \n",
       "Merged-chr10-81291937-7   0.200723  0.776778  0.000000  0.000000  0.228188  \n",
       "Merged-chr11-116433774-7  0.161491  0.858717  0.000000  0.000000  0.701345  \n",
       "Merged-chr13-3541637-7    0.113028  0.678447  0.000000  0.000000  0.121728  \n",
       "Merged-chr5-120558954-7   0.336879  0.858717  0.000000  0.022930  0.665860  \n",
       "Merged-chr1-162570598-7   0.493807  0.539146  0.000000  0.000000  0.598830  \n",
       "Merged-chr7-19149543-7    0.184568  0.858717  0.000000  0.026208  0.460825  \n",
       "Merged-chr1-182108730-7   0.283802  0.703030  0.000000  0.000000  0.330706  \n",
       "Merged-chr10-80702354-7   0.316110  0.858717  0.000000  0.150761  0.571230  \n",
       "Merged-chr5-151368615-7   0.200723  0.858717  0.000000  0.000000  0.437167  \n",
       "Merged-chr17-74570654-7   0.055335  0.383456  0.000000  0.000000  0.283390  \n",
       "Merged-chr10-26564705-7   0.276879  0.858717  0.000000  0.216314  0.701345  \n",
       "Merged-chr14-17414520-7   0.078412  0.670253  0.000000  0.000000  0.267618  \n",
       "Merged-chr7-4239319-7     0.332264  0.850525  0.000000  0.032763  0.322820  \n",
       "Merged-chr10-81303724-7   0.429190  0.858717  0.000000  0.249091  0.594885  \n",
       "Merged-chr10-80051910-7   0.500729  0.678447  0.000000  0.278591  0.831465  \n",
       "Merged-chr2-132229938-7   0.473036  0.858717  0.000000  0.000000  0.165101  \n",
       "Merged-chr6-97856502-7    0.156875  0.678447  0.105671  0.363810  0.496312  \n",
       "\n",
       "[93261 rows x 196 columns]"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "scaled_motif_frame"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Make Truth Table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "motif_1 = 'spi1-c'\n",
    "motif_2 = 'cebp'\n",
    "factor_1 = 'c57bl6_pu1_veh'\n",
    "factor_2 = 'c57bl6_cebpa_veh'\n",
    "\n",
    "motif_state_1s = []\n",
    "motif_state_2s = []\n",
    "peak_state_1s = []\n",
    "peak_state_2s = []\n",
    "snp_state_1s = []\n",
    "snp_state_2s = []\n",
    "counts = []\n",
    "for peak_state_1 in [True, False]:\n",
    "    for peak_state_2 in [True, False]:\n",
    "        for motif_state_1 in [True, False]:\n",
    "            for motif_state_2 in [True, False]:\n",
    "                motif_state_1s.append(motif_state_1)\n",
    "                motif_state_2s.append(motif_state_2)\n",
    "                peak_state_1s.append(peak_state_1)\n",
    "                peak_state_2s.append(peak_state_2)\n",
    "\n",
    "                peak_indices = summary_frame[(summary_frame['Factors'].str.contains(factor_1) == peak_state_1) &\n",
    "                                             (summary_frame['Factors'].str.contains(factor_2) == peak_state_2)].index.values\n",
    "\n",
    "                current_motif_frame = motif_score_frame[motif_score_frame.index.isin(peak_indices)]\n",
    "                motif_indices = current_motif_frame[((current_motif_frame[motif_1] > 0) == motif_state_1) &\n",
    "                                                    ((current_motif_frame[motif_2] > 0) == motif_state_2)].index.values                   \n",
    "\n",
    "                count = len(motif_indices)\n",
    "                counts.append(count)\n",
    "current_count_frame = pd.DataFrame({factor_1 +' binding':peak_state_1s,\n",
    "                                   factor_2 +' binding':peak_state_2s,\n",
    "                                   motif_1 + ' motif':motif_state_1s,\n",
    "                                   motif_2 + ' motif':motif_state_2s,\n",
    "\n",
    "                                   'num peaks': counts})               \n",
    "current_count_frame = current_count_frame[[factor_1 +' binding',\n",
    "                                   factor_2 +' binding',\n",
    "                                   motif_1 + ' motif',\n",
    "                                   motif_2 + ' motif',\n",
    "                                                                      'num peaks']]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "current_count_frame.to_csv('cebpa_pu1_truthTable.tsv', sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.4.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
