{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Unclustered Foreground Generation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "env: PATH=/gpfs/data01/glasslab/home/jtao/perl5/bin:/gpfs/data01/glasslab/home/jtao/software/anaconda3/bin:/home/jtao/software/bin:/usr/local/sbin:/usr/sbin:/usr/bin:/usr/local/bin:/usr/bin:/gpfs/data01/glasslab/home/jtao/software/homer/bin:/gpfs/data01/glasslab/home/jtao/software/weblogo:/home/jtao/code/seq_merge_pipe:/home/vlink/mouse_strains/marge/shifting:/bioinformatics/glassutils/scripts:/bioinformatics/software/meme/bin:/home/jtao/software/lsgkm/bin\n"
     ]
    }
   ],
   "source": [
    "### header ###\n",
    "__author__ = \"Jenhan Tao\"\n",
    "__license__ = \"BSD\"\n",
    "__email__ = \"jenhantao@gmail.com\"\n",
    "\n",
    "### imports ###\n",
    "import sys\n",
    "import os\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "import matplotlib\n",
    "import itertools\n",
    "import scipy\n",
    "import matplotlib.pyplot as plt \n",
    "import seaborn as sns\n",
    "import pickle\n",
    "from sklearn import preprocessing\n",
    "import sklearn\n",
    "from sklearn import svm, datasets\n",
    "from sklearn.utils import shuffle\n",
    "from sklearn.metrics import roc_curve, auc\n",
    "import matplotlib_venn\n",
    "from sklearn.cross_validation import train_test_split\n",
    "from random import shuffle\n",
    "import threading\n",
    "import time\n",
    "from collections import Counter\n",
    "### notebook specific configuration ###\n",
    "%matplotlib inline\n",
    "matplotlib.pylab.rcParams['savefig.dpi'] = 200\n",
    "sys.setrecursionlimit(5000)\n",
    "working_dir = '/gpfs/data01/glasslab/home/jtao/analysis/ap1_analysis_unclustered_features/'\n",
    "if not os.path.isdir(working_dir):\n",
    "    os.mkdir(working_dir)\n",
    "os.chdir(working_dir)\n",
    "sns.set_context('notebook')\n",
    "%load_ext autoreload\n",
    "%autoreload 2\n",
    "%env PATH=/gpfs/data01/glasslab/home/jtao/perl5/bin:/gpfs/data01/glasslab/home/jtao/software/anaconda3/bin:/home/jtao/software/bin:/usr/local/sbin:/usr/sbin:/usr/bin:/usr/local/bin:/usr/bin:/gpfs/data01/glasslab/home/jtao/software/homer/bin:/gpfs/data01/glasslab/home/jtao/software/weblogo:/home/jtao/code/seq_merge_pipe:/home/vlink/mouse_strains/marge/shifting:/bioinformatics/glassutils/scripts:/bioinformatics/software/meme/bin:/home/jtao/software/lsgkm/bin"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## copy motif files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "if [ ! -d ./fimo_motifs ]; then mkdir ./fimo_motifs; else rm ./fimo_motifs/*; fi\n",
    "for f in /gpfs/data01/glasslab/home/jtao/analysis/ap1_motif_merging/homer_motifs/*;\n",
    "do motif=${f##*/};\n",
    "motif=${motif%.homer}\n",
    "/gpfs/data01/glasslab/home/jtao/code/tba/homer2fimo.py $f ./fimo_motifs/${motif}.fimo\n",
    "done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "cp: cannot stat ‘/gpfs/data01/glasslab/home/jtao/analysis/ap1_analysis_features/summary_frame.tsv’: No such file or directory\n"
     ]
    }
   ],
   "source": [
    "%%bash \n",
    "cp /gpfs/data01/glasslab/home/jtao/analysis/ap1_analysis_features/summary_frame.tsv ./\n",
    "cp /gpfs/data01/glasslab/home/jtao/analysis/ap1_analysis_background_features/background_group_summary.tsv ./\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/gpfs/data01/glasslab/home/jtao/software/anaconda3/lib/python3.5/site-packages/IPython/core/interactiveshell.py:2723: DtypeWarning: Columns (5,7,8,9,11,13) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    }
   ],
   "source": [
    "# read in peak data data\n",
    "summary_frame = pd.read_csv('./group_summary.tsv' , sep='\\t')\n",
    "summary_frame = summary_frame.fillna('0')\n",
    "for col in summary_frame.columns[5:]:\n",
    "    floatValues = []\n",
    "    for val in summary_frame[col].values.astype(str):\n",
    "        if ',' in val:\n",
    "            maxVal = np.mean([float(x) for x in val.split(',')])\n",
    "            floatValues.append(maxVal)\n",
    "        else:\n",
    "            floatValues.append(float(val))\n",
    "    summary_frame[col] = floatValues\n",
    "summary_frame.index = summary_frame['ID'].values\n",
    "\n",
    "# remove peaks in unknown/random chromosomes\n",
    "summary_frame = summary_frame[~summary_frame['chr'].str.contains('random')]\n",
    "summary_frame = summary_frame[~summary_frame['chr'].str.contains('Un')]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/gpfs/data01/glasslab/home/jtao/software/anaconda3/lib/python3.5/site-packages/IPython/core/interactiveshell.py:2723: DtypeWarning: Columns (5,6,7,8,9,10,11,13,15,16) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    }
   ],
   "source": [
    "# read in peak data data\n",
    "summary_background_frame = pd.read_csv('./background_group_summary.tsv' , sep='\\t')\n",
    "summary_background_frame = summary_background_frame.fillna('0')\n",
    "for col in summary_background_frame.columns[5:]:\n",
    "    floatValues = []\n",
    "    for val in summary_background_frame[col].values.astype(str):\n",
    "        if ',' in val:\n",
    "            maxVal = np.mean([float(x) for x in val.split(',')])\n",
    "            floatValues.append(maxVal)\n",
    "        else:\n",
    "            floatValues.append(float(val))\n",
    "    summary_background_frame[col] = floatValues\n",
    "summary_background_frame.index = summary_background_frame['ID'].values"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Retrieve Sequences Under Peaks"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "if [ ! -d ./peak_sequences ] ;\n",
    "    then mkdir ./peak_sequences\n",
    "else\n",
    "    rm ./peak_sequences/*\n",
    "fi"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%%bash \n",
    "cp /gpfs/data01/glasslab/home/jtao/analysis/ap1_analysis_features/peak_sequences/C57BL6J.fa ./peak_sequences/\n",
    "cp /gpfs/data01/glasslab/home/jtao/analysis/ap1_analysis_background_features/peak_sequences/C57BL6J.fa ./peak_sequences/C57BL6J-background.fa"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### create a script to scan for motifs using FIMO\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "C57BL6J.fa\n",
      "C57BL6J-background.fa\n"
     ]
    }
   ],
   "source": [
    "# create a script to scan for motifs using FIMO\n",
    "! if [ ! -d /home/jtao/analysis/cobinding_motif_analysis/fimo_results/ ]; then mkdir /home/jtao/analysis/cobinding_motif_analysis/fimo_results/; fi\n",
    "! rm -rf fimo_out/*\n",
    "! rm -rf fimo_result/*\n",
    "\n",
    "\n",
    "pthresh = 0.01\n",
    "motif_dir = './fimo_motifs/'\n",
    "\n",
    "fimo_results_dir = './fimo_results'\n",
    "\n",
    "for fastaFile in os.listdir('./peak_sequences/'):\n",
    "    if not 'marge' in fastaFile:\n",
    "        print(fastaFile)\n",
    "        strain = fastaFile.split('.')[0]\n",
    "        count = 0\n",
    "        scriptFile = open('scanMotifs_'+ strain +'.sh','w')\n",
    "        for m in sorted(os.listdir(motif_dir)):\n",
    "            if '.fimo' in m:\n",
    "                fimo_out_dir = './fimo_out/' + strain + '_' +m.replace('.fimo','')\n",
    "                outPath = fimo_results_dir + '/' +strain + '_' + m.replace('.fimo','') +'.txt'\n",
    "                scriptFile.write(\n",
    "        #             '(sleep ' + str(15 * count) + \n",
    "                    '(sleep ' + str(0 * count) + \n",
    "                    's; fimo --text --max-stored-scores 2000000 --output-pthresh ' + \n",
    "                    str(pthresh) +' --oc ' + fimo_out_dir + ' ' +\n",
    "                    motif_dir + '/' + m + ' ./peak_sequences/' + fastaFile +\n",
    "                    '> ' + outPath + ' ) & \\n')\n",
    "                count+=1\n",
    "        scriptFile.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ARNT::HIF1A of width 8.\n",
      "Using motif -ARNT::HIF1A of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ALX3 of width 10.\n",
      "Using motif -ALX3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ATF4 of width 13.\n",
      "Using motif -ATF4 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Ar of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Ar of width 17.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +ATF7 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +Ahr::Arnt of width 6.\n",
      "Using motif -Ahr::Arnt of width 6.\n",
      "Using motif -ATF7 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Alx4 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Alx4 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Alx1 of width 17.\n",
      "Using motif -Alx1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Arid3b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Arid3b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Arid3a of width 6.\n",
      "Using motif -Arid3a of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +NFYA of width 18.\n",
      "Using motif -NFYA of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Arid5a of width 14.\n",
      "Using motif -Arid5a of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +Arntl of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -Arntl of width 10.\n",
      "Using motif +Arnt of width 6.\n",
      "Using motif -Arnt of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Arx of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Arx of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif +Atf3 of width 8.\n",
      "Using motif -Atf3 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Atf1 of width 8.\n",
      "Using motif -Atf1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Ascl2 of width 10.\n",
      "Using motif -Ascl2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +NR2F1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Atoh1 of width 10.\n",
      "Using motif -NR2F1 of width 13.\n",
      "Using motif -Atoh1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +BARHL2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -BARHL2 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +BATF::JUN of width 11.\n",
      "Using motif -BATF::JUN of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Nkx3-1 of width 9.\n",
      "Using motif -Nkx3-1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Nkx2-5(var.2) of width 11.\n",
      "Using motif -Nkx2-5(var.2) of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +Nr2e1 of width 9.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Nr2e1 of width 9.\n",
      "Read 1 motifs.\n",
      "Using motif +Nr5a2 of width 15.\n",
      "Using motif -Nr5a2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Nr2f6(var.2) of width 15.\n",
      "Using motif -Nr2f6(var.2) of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +BARX1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -BARX1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +BATF3 of width 14.\n",
      "Using motif -BATF3 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +BHLHE23 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +BCL6B of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif -BHLHE23 of width 12.\n",
      "Using motif +BHLHE22 of width 10.\n",
      "Using motif -BHLHE22 of width 10.\n",
      "Using motif -BCL6B of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +BHLHE41 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -BHLHE41 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +BSX of width 8.\n",
      "Using motif +BHLHE40 of width 10.\n",
      "Using motif -BSX of width 8.\n",
      "Using motif -BHLHE40 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +PBX1 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +Bach1::Mafk of width 15.\n",
      "Using motif -PBX1 of width 12.\n",
      "Using motif -Bach1::Mafk of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Barhl1 of width 10.\n",
      "Using motif -Barhl1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +CDX1 of width 9.\n",
      "Using motif -CDX1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Bcl6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Bcl6 of width 14.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +PAX1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Bhlha15 of width 8.\n",
      "Using motif -Bhlha15 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -PAX1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +CDX2 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -CDX2 of width 11.\n",
      "Using motif +CEBPB of width 10.\n",
      "Using motif -CEBPB of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +CEBPE of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +CEBPD of width 10.\n",
      "Using motif -CEBPE of width 10.\n",
      "Using motif -CEBPD of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +RUNX2 of width 9.\n",
      "Using motif -RUNX2 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +CEBPA of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -CEBPA of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +Stat4 of width 14.\n",
      "Using motif -Stat4 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +CENPB of width 15.\n",
      "Using motif -CENPB of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +CEBPG of width 10.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +CLOCK of width 10.\n",
      "Using motif +TEF of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -CEBPG of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -CLOCK of width 10.\n",
      "Using motif -TEF of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +CREB3 of width 14.\n",
      "Using motif -CREB3 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +CTCF of width 19.\n",
      "Using motif -CTCF of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Sox3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Sox3 of width 10.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +CREB3L1 of width 14.\n",
      "Using motif +CREB1 of width 8.\n",
      "Using motif -CREB1 of width 8.\n",
      "Using motif -CREB3L1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TFEB of width 10.\n",
      "Using motif -TFEB of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TCF7L2 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -TCF7L2 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +TCF4 of width 10.\n",
      "Using motif -TCF4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TBX1 of width 8.\n",
      "Using motif -TBX1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TBX5 of width 8.\n",
      "Using motif -TBX5 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +CUX1 of width 10.\n",
      "Using motif -CUX1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +SPDEF of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif -SPDEF of width 11.\n",
      "Using motif +Crem of width 10.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +DBP of width 12.\n",
      "Using motif +Creb5 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -Crem of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -DBP of width 12.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +Creb3l2 of width 9.\n",
      "Using motif -Creb5 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +DUX4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -DUX4 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +DMRT3 of width 11.\n",
      "Using motif +Crx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -DMRT3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Crx of width 11.\n",
      "Using motif -Creb3l2 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +Dlx2 of width 8.\n",
      "Using motif +Ddit3::Cebpa of width 12.\n",
      "Using motif +DLX6 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif -Dlx2 of width 8.\n",
      "Using motif +DUXA of width 13.\n",
      "Using motif -DLX6 of width 8.\n",
      "Using motif -Ddit3::Cebpa of width 12.\n",
      "Using motif -DUXA of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +Rarg of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Dlx1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Dlx1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Rarg of width 16.\n",
      "Read 1 motifs.\n",
      "Using motif +Dlx4 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Dlx4 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +Dlx3 of width 8.\n",
      "Using motif -Dlx3 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +E2F1 of width 12.\n",
      "Using motif -E2F1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +CUX2 of width 10.\n",
      "Using motif -CUX2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Dmbx1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Dux of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif -Dmbx1 of width 17.\n",
      "Using motif -Dux of width 8.\n",
      "Using motif +E2F3 of width 18.\n",
      "Using motif -E2F3 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +TFEC of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -TFEC of width 10.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +E2F6 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +E2F2 of width 16.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -E2F6 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +E2F4 of width 11.\n",
      "Using motif -E2F4 of width 11.\n",
      "Using motif -E2F2 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +E2F7 of width 14.\n",
      "Using motif -E2F7 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TFAP2A(var.2) of width 12.\n",
      "Using motif -TFAP2A(var.2) of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +EBF1 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +E2F8 of width 12.\n",
      "Using motif -E2F8 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -EBF1 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +Stat5a::Stat5b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Stat5a::Stat5b of width 11.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +EGR2 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +EGR1 of width 14.\n",
      "Using motif -EGR2 of width 11.\n",
      "Using motif -EGR1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +EGR3 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -EGR3 of width 15.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +EGR4 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ELF1 of width 12.\n",
      "Using motif -EGR4 of width 16.\n",
      "Using motif -ELF1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +EHF of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -EHF of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +ELF3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ELF3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +ELF4 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -ELF4 of width 12.\n",
      "Using motif +ELK1 of width 10.\n",
      "Using motif -ELK1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ELF5 of width 11.\n",
      "Using motif -ELF5 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ELK3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -ELK3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ELK4 of width 11.\n",
      "Using motif -ELK4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +EMX2 of width 10.\n",
      "Using motif -EMX2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +EMX1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -EMX1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +EN1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -EN1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +EN2 of width 10.\n",
      "Using motif -EN2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +ERF of width 10.\n",
      "Using motif +EOMES of width 13.\n",
      "Using motif -ERF of width 10.\n",
      "Using motif -EOMES of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ERG of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ERG of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +ESR1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ESR2 of width 15.\n",
      "Using motif -ESR1 of width 17.\n",
      "Using motif -ESR2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +ESX1 of width 10.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -ESX1 of width 10.\n",
      "Using motif +ETS1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif -ETS1 of width 10.\n",
      "Using motif +ETV2 of width 11.\n",
      "Using motif +ESRRB of width 11.\n",
      "Using motif -ETV2 of width 11.\n",
      "Using motif -ESRRB of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +ETV3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +ETV1 of width 10.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -ETV1 of width 10.\n",
      "Using motif -ETV3 of width 10.\n",
      "Using motif +ETV4 of width 10.\n",
      "Using motif -ETV4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +ETV6 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +EVX1 of width 10.\n",
      "Using motif -ETV6 of width 10.\n",
      "Using motif -EVX1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +EVX2 of width 10.\n",
      "Using motif -EVX2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +FEV of width 10.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -FEV of width 10.\n",
      "Using motif +FIGLA of width 10.\n",
      "Using motif -FIGLA of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Esrrg of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +FLI1 of width 10.\n",
      "Using motif -Esrrg of width 10.\n",
      "Using motif -FLI1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +ETV5 of width 10.\n",
      "Using motif +FOS of width 11.\n",
      "Using motif -ETV5 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -FOS of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOS::JUN of width 7.\n",
      "Using motif -FOS::JUN of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOSL1 of width 11.\n",
      "Using motif -FOSL1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOSL2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -FOSL2 of width 11.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +Esrra of width 11.\n",
      "Using motif +FOXA1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Esrra of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -FOXA1 of width 15.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXB1 of width 11.\n",
      "Using motif -FOXB1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +EWSR1-FLI1 of width 18.\n",
      "Using motif -EWSR1-FLI1 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXC1 of width 11.\n",
      "Using motif -FOXC1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXC2 of width 12.\n",
      "Using motif -FOXC2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXD2 of width 7.\n",
      "Using motif -FOXD2 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXG1 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXF2 of width 14.\n",
      "Using motif -FOXG1 of width 8.\n",
      "Using motif -FOXF2 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXH1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -FOXH1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXL1 of width 7.\n",
      "Using motif -FOXL1 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXI1 of width 7.\n",
      "Using motif -FOXI1 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXO6 of width 7.\n",
      "Using motif -FOXO6 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXO3 of width 8.\n",
      "Using motif -FOXO3 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXO4 of width 7.\n",
      "Using motif -FOXO4 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXD1 of width 8.\n",
      "Using motif -FOXD1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXP1 of width 15.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXP2 of width 11.\n",
      "Using motif -FOXP2 of width 11.\n",
      "Using motif -FOXP1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXP3 of width 7.\n",
      "Using motif -FOXP3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Foxa2 of width 12.\n",
      "Using motif -Foxa2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Foxj2 of width 8.\n",
      "Using motif -Foxj2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Foxd3 of width 12.\n",
      "Using motif -Foxd3 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Foxj3 of width 17.\n",
      "Using motif -Foxj3 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +Foxq1 of width 11.\n",
      "Using motif +Foxk1 of width 17.\n",
      "Using motif -Foxq1 of width 11.\n",
      "Using motif -Foxk1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Foxo1 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif -Foxo1 of width 11.\n",
      "Using motif +GATA1::TAL1 of width 18.\n",
      "Using motif -GATA1::TAL1 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +GATA3 of width 8.\n",
      "Using motif +GATA5 of width 8.\n",
      "Using motif -GATA3 of width 8.\n",
      "Using motif -GATA5 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +GBX1 of width 10.\n",
      "Using motif +GATA2 of width 14.\n",
      "Using motif -GBX1 of width 10.\n",
      "Using motif -GATA2 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +GCM1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +GLI2 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -GCM1 of width 11.\n",
      "Using motif -GLI2 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +GLIS1 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif -GLIS1 of width 16.\n",
      "Using motif +GLIS3 of width 14.\n",
      "Using motif +GCM2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -GCM2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -GLIS3 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +GRHL1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +GLIS2 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -GRHL1 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +GMEB2 of width 8.\n",
      "Using motif -GLIS2 of width 14.\n",
      "Using motif -GMEB2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +GSC of width 10.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +GSX1 of width 10.\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -GSC of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif -GSX1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +GSX2 of width 10.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +GSC2 of width 10.\n",
      "Using motif -GSX2 of width 10.\n",
      "Using motif +Gabpa of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -GSC2 of width 10.\n",
      "Using motif -Gabpa of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +Gata4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -Gata4 of width 11.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +Gfi1 of width 10.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Gata1 of width 11.\n",
      "Using motif -Gfi1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +Gfi1b of width 11.\n",
      "Using motif -Gata1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -Gfi1b of width 11.\n",
      "Using motif +Gmeb1 of width 17.\n",
      "Using motif -Gmeb1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +HES5 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -HES5 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +HESX1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -HESX1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +HEY1 of width 10.\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -HEY1 of width 10.\n",
      "Using motif +HEY2 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +HES7 of width 12.\n",
      "Using motif -HEY2 of width 10.\n",
      "Using motif -HES7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +GBX2 of width 10.\n",
      "Using motif -GBX2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +HLF of width 12.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +HIC2 of width 9.\n",
      "Using motif +HLTF of width 10.\n",
      "Using motif -HLF of width 12.\n",
      "Using motif -HIC2 of width 9.\n",
      "Using motif -HLTF of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +HINFP of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +HNF1A of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -HINFP of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +HNF1B of width 13.\n",
      "Using motif -HNF1A of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +HOXA2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -HNF1B of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -HOXA2 of width 10.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +HOXA5 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +HOXA13 of width 10.\n",
      "Using motif -HOXA5 of width 8.\n",
      "Using motif -HOXA13 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +HMBOX1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -HMBOX1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +HOXB13 of width 10.\n",
      "Using motif -HOXB13 of width 10.\n",
      "Using motif +HOXB3 of width 10.\n",
      "Using motif -HOXB3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +HOXB2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -HOXB2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +HOXC11 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -HOXC11 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +HOXC12 of width 11.\n",
      "Using motif -HOXC12 of width 11.\n",
      "Using motif +HOXA10 of width 11.\n",
      "Using motif -HOXA10 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +HNF4G of width 15.\n",
      "Using motif +HOXC10 of width 10.\n",
      "Using motif -HOXC10 of width 10.\n",
      "Using motif -HNF4G of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +HOXC13 of width 11.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +HOXD11 of width 10.\n",
      "Using motif -HOXC13 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -HOXD11 of width 10.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +HOXD13 of width 10.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +HOXD12 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -HOXD13 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -HOXD12 of width 11.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +HSF1 of width 13.\n",
      "Using motif +HSF2 of width 13.\n",
      "Using motif -HSF1 of width 13.\n",
      "Using motif -HSF2 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +HSF4 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -HSF4 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +Hand1::Tcf3 of width 10.\n",
      "Using motif +Hes2 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -Hand1::Tcf3 of width 10.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Hes2 of width 13.\n",
      "Read 1 motifs.\n",
      "Using motif +Hic1 of width 9.\n",
      "Using motif -Hic1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Hmx3 of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif +Hmx1 of width 17.\n",
      "Using motif -Hmx3 of width 17.\n",
      "Using motif -Hmx1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Hes1 of width 10.\n",
      "Using motif -Hes1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Hoxa9 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Hmx2 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Hoxa9 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +Hnf4a of width 16.\n",
      "Read 1 motifs.\n",
      "Using motif +Hoxa11 of width 12.\n",
      "Using motif -Hoxa11 of width 12.\n",
      "Using motif -Hnf4a of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Hoxb5 of width 16.\n",
      "Using motif -Hoxb5 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Hoxd8 of width 17.\n",
      "Using motif +Hoxd3 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Hoxd9 of width 10.\n",
      "Using motif -Hoxd8 of width 17.\n",
      "Using motif -Hoxd3 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Hoxd9 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +IRF1 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +INSM1 of width 12.\n",
      "Using motif -IRF1 of width 21.\n",
      "Using motif -INSM1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ID4 of width 10.\n",
      "Using motif -ID4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Hoxc9 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -Hoxc9 of width 13.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +IRF2 of width 18.\n",
      "Using motif -IRF2 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Hmx2 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +IRF7 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +IRF9 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -IRF7 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +IRF8 of width 14.\n",
      "Using motif -IRF9 of width 15.\n",
      "Using motif +ISL2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ISL2 of width 8.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ISX of width 8.\n",
      "Using motif -IRF8 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ISX of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Id2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Id2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +JDP2(var.2) of width 12.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +JDP2 of width 9.\n",
      "Read 1 motifs.\n",
      "Using motif -JDP2 of width 9.\n",
      "Using motif +JUN of width 13.\n",
      "Using motif -JDP2(var.2) of width 12.\n",
      "Using motif -JUN of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +JUND of width 11.\n",
      "Using motif -JUND of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +KLF16 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +KLF14 of width 14.\n",
      "Using motif -KLF16 of width 11.\n",
      "Using motif -KLF14 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +JUN(var.2) of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +Klf12 of width 15.\n",
      "Using motif -JUN(var.2) of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Klf12 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Klf1 of width 11.\n",
      "Using motif -Klf1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +JUND(var.2) of width 15.\n",
      "Using motif -JUND(var.2) of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Klf4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -Klf4 of width 10.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +LBX1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -LBX1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +LEF1 of width 15.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +KLF13 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +LBX2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +LHX2 of width 10.\n",
      "Using motif -LEF1 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -LBX2 of width 10.\n",
      "Using motif -LHX2 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +LHX6 of width 10.\n",
      "Using motif -KLF13 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -LHX6 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +LIN54 of width 9.\n",
      "Using motif -LIN54 of width 9.\n",
      "Read 1 motifs.\n",
      "Using motif +JUNB of width 11.\n",
      "Using motif -JUNB of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +LHX9 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -LHX9 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +KLF5 of width 10.\n",
      "Using motif -KLF5 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +LMX1A of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -LMX1A of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +LMX1B of width 8.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -LMX1B of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +Lhx3 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Lhx8 of width 8.\n",
      "Using motif -Lhx8 of width 8.\n",
      "Using motif -Lhx3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +Lhx4 of width 8.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Lhx4 of width 8.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +MAF::NFE2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MAFF of width 18.\n",
      "Using motif -MAF::NFE2 of width 15.\n",
      "Read 1 motifs.\n",
      "Using motif +MAFG of width 21.\n",
      "Using motif -MAFF of width 18.\n",
      "Using motif -MAFG of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MAFK of width 15.\n",
      "Using motif -MAFK of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MAX of width 10.\n",
      "Using motif -MAX of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MAFG::NFE2L1 of width 6.\n",
      "Using motif -MAFG::NFE2L1 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MAX::MYC of width 11.\n",
      "Using motif -MAX::MYC of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MEF2B of width 12.\n",
      "Using motif -MEF2B of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MEF2C of width 15.\n",
      "Using motif -MEF2C of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +MEIS1 of width 7.\n",
      "Using motif +MEF2A of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif -MEIS1 of width 7.\n",
      "Using motif +MEF2D of width 12.\n",
      "Using motif -MEF2A of width 12.\n",
      "Using motif -MEF2D of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +MEIS2 of width 8.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -MEIS2 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +MEIS3 of width 8.\n",
      "Using motif -MEIS3 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MGA of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -MGA of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +MEOX1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -MEOX1 of width 10.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +MIXL1 of width 10.\n",
      "Using motif +MLX of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -MIXL1 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif -MLX of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +MLXIPL of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -MLXIPL of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +MNT of width 10.\n",
      "Using motif -MNT of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MEOX2 of width 10.\n",
      "Using motif -MEOX2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MNX1 of width 10.\n",
      "Using motif -MNX1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MSC of width 10.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +MSX1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -MSX1 of width 8.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +MYBL1 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +MSX2 of width 8.\n",
      "Using motif -MYBL1 of width 12.\n",
      "Using motif -MSX2 of width 8.\n",
      "Using motif -MSC of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MTF1 of width 14.\n",
      "Using motif -MTF1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +MYBL2 of width 15.\n",
      "Using motif +MYF6 of width 10.\n",
      "Using motif -MYF6 of width 10.\n",
      "Using motif -MYBL2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MZF1 of width 6.\n",
      "Using motif -MZF1 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Mecom of width 14.\n",
      "Using motif -Mecom of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Mafb of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Mafb of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MZF1(var.2) of width 10.\n",
      "Using motif -MZF1(var.2) of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Mlxip of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Mlxip of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Mitf of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Mitf of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +Myc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Myc of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +Myog of width 11.\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -Myog of width 11.\n",
      "Using motif +Mycn of width 8.\n",
      "Using motif -Mycn of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Msx3 of width 8.\n",
      "Using motif +Myb of width 10.\n",
      "Using motif -Msx3 of width 8.\n",
      "Using motif -Myb of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Myod1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -Myod1 of width 13.\n",
      "Using motif +NEUROD2 of width 10.\n",
      "Using motif -NEUROD2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +NFATC1 of width 10.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +NEUROG2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +NFATC2 of width 7.\n",
      "Read 1 motifs.\n",
      "Using motif -NFATC1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -NFATC2 of width 7.\n",
      "Using motif +NFATC3 of width 10.\n",
      "Using motif -NEUROG2 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif -NFATC3 of width 10.\n",
      "Using motif +NFE2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -NFE2 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +NFIA of width 10.\n",
      "Using motif -NFIA of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +NFAT5 of width 10.\n",
      "Using motif -NFAT5 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +NFIC of width 6.\n",
      "Using motif -NFIC of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +NFIL3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -NFIL3 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +NFIC::TLX1 of width 14.\n",
      "Using motif -NFIC::TLX1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +NFKB1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +NFIX of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -NFIX of width 9.\n",
      "Using motif +NFKB2 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -NFKB1 of width 13.\n",
      "Using motif +NFYB of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -NFKB2 of width 13.\n",
      "Using motif -NFYB of width 15.\n",
      "Read 1 motifs.\n",
      "Using motif +NHLH1 of width 10.\n",
      "Using motif -NHLH1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +NKX2-3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -NKX2-3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +NKX2-8 of width 9.\n",
      "Using motif -NKX2-8 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +NKX3-2 of width 9.\n",
      "Using motif -NKX3-2 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +NKX6-1 of width 8.\n",
      "Using motif +NKX6-2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -NKX6-1 of width 8.\n",
      "Using motif -NKX6-2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +NR2C2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +NOTO of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +NR1H2::RXRA of width 17.\n",
      "Using motif -NOTO of width 10.\n",
      "Using motif -NR2C2 of width 15.\n",
      "Using motif -NR1H2::RXRA of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +NR4A2 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +NR3C1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +NR3C2 of width 17.\n",
      "Using motif -NR4A2 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +NRF1 of width 11.\n",
      "Using motif -NR3C1 of width 17.\n",
      "Using motif -NR3C2 of width 17.\n",
      "Using motif -NRF1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +NRL of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -NRL of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Neurog1 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +Nfe2l2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -Neurog1 of width 10.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Nkx2-5 of width 7.\n",
      "Using motif -Nkx2-5 of width 7.\n",
      "Using motif -Nfe2l2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Nobox of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +Npas2 of width 10.\n",
      "Using motif -Nobox of width 8.\n",
      "Using motif -Npas2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Nr1h3::Rxra of width 19.\n",
      "Using motif -Nr1h3::Rxra of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Nr2f6 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +OLIG1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -OLIG1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Nr2f6 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +Nr2e3 of width 7.\n",
      "Using motif -Nr2e3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +OLIG3 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +OTX1 of width 8.\n",
      "Using motif -OLIG3 of width 10.\n",
      "Using motif -OTX1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +OLIG2 of width 10.\n",
      "Using motif -OLIG2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +ONECUT1 of width 14.\n",
      "Using motif +PAX9 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ONECUT1 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +OTX2 of width 8.\n",
      "Using motif -OTX2 of width 8.\n",
      "Using motif -PAX9 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +PLAG1 of width 14.\n",
      "Using motif +PKNOX1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +PPARG of width 20.\n",
      "Using motif -PKNOX1 of width 12.\n",
      "Using motif -PLAG1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -PPARG of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +PAX3 of width 10.\n",
      "Using motif -PAX3 of width 10.\n",
      "Using motif +ONECUT3 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +ONECUT2 of width 14.\n",
      "Using motif -ONECUT3 of width 14.\n",
      "Using motif -ONECUT2 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +PDX1 of width 8.\n",
      "Using motif -PDX1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +PITX3 of width 9.\n",
      "Using motif -PITX3 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Phox2b of width 11.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +PAX5 of width 19.\n",
      "Using motif -Phox2b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +POU3F4 of width 9.\n",
      "Using motif -POU3F4 of width 9.\n",
      "Using motif -PAX5 of width 19.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Pou2f3 of width 16.\n",
      "Read 1 motifs.\n",
      "Using motif +RFX3 of width 16.\n",
      "Using motif -Pou2f3 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -RFX3 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Pou5f1::Sox2 of width 15.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +PKNOX2 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -Pou5f1::Sox2 of width 15.\n",
      "Using motif -PKNOX2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +RXRB of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -RXRB of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +POU4F1 of width 14.\n",
      "Using motif -POU4F1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +RFX5 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +PROX1 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +Pax6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -PROX1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -Pax6 of width 14.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +PRDM1 of width 15.\n",
      "Using motif +PRRX1 of width 8.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +RAX2 of width 8.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -PRRX1 of width 8.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -RFX5 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -RAX2 of width 8.\n",
      "Using motif -PRDM1 of width 15.\n",
      "Using motif +PAX4 of width 8.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -PAX4 of width 8.\n",
      "Using motif +POU4F3 of width 16.\n",
      "Using motif +RORA of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -RORA of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -POU4F3 of width 16.\n",
      "Using motif +PAX7 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +RXRA::VDR of width 15.\n",
      "Using motif -PAX7 of width 10.\n",
      "Using motif -RXRA::VDR of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +PROP1 of width 11.\n",
      "Using motif +SOX9 of width 9.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Pparg::Rxra of width 15.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -SOX9 of width 9.\n",
      "Using motif +Pitx1 of width 8.\n",
      "Using motif -PROP1 of width 11.\n",
      "Using motif -Pitx1 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif -Pparg::Rxra of width 15.\n",
      "Using motif +RARA(var.2) of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif +POU3F3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -RARA(var.2) of width 17.\n",
      "Using motif +Prrx2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -Prrx2 of width 8.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -POU3F3 of width 13.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +SP4 of width 17.\n",
      "Using motif +SOX10 of width 6.\n",
      "Using motif -SOX10 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -SP4 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +SP1 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +REST of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -SP1 of width 11.\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +POU3F2 of width 12.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -REST of width 21.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +RUNX3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -POU3F2 of width 12.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif -RUNX3 of width 10.\n",
      "Using motif +POU2F1 of width 12.\n",
      "Using motif +POU3F1 of width 12.\n",
      "Using motif -POU2F1 of width 12.\n",
      "Using motif -POU3F1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Srebf1(var.2) of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +RFX2 of width 16.\n",
      "Using motif -Srebf1(var.2) of width 10.\n",
      "Using motif -RFX2 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +POU6F1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -POU6F1 of width 10.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +SREBF2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -SREBF2 of width 10.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +RUNX1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +POU2F2 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -RUNX1 of width 11.\n",
      "Using motif +RFX4 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -POU2F2 of width 13.\n",
      "Using motif +PHOX2A of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +Pax2 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +POU4F2 of width 16.\n",
      "Using motif -PHOX2A of width 11.\n",
      "Using motif -RFX4 of width 16.\n",
      "Using motif -Pax2 of width 8.\n",
      "Using motif -POU4F2 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +POU5F1B of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +SNAI2 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -POU5F1B of width 9.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -SNAI2 of width 9.\n",
      "Read 1 motifs.\n",
      "Using motif +Rarb of width 16.\n",
      "Using motif +POU1F1 of width 14.\n",
      "Using motif -POU1F1 of width 14.\n",
      "Using motif -Rarb of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +SRY of width 9.\n",
      "Using motif -SRY of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +SP3 of width 11.\n",
      "Using motif +SPI1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -SP3 of width 11.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -SPI1 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +RARA::RXRA of width 17.\n",
      "Using motif -RARA::RXRA of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +SMAD3 of width 10.\n",
      "Using motif -SMAD3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Shox2 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +RARA of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Shox2 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +TBP of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -TBP of width 15.\n",
      "Using motif -RARA of width 18.\n",
      "Read 1 motifs.\n",
      "Using motif +Rxra of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Rxra of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Rfx1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +SOX21 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +SREBF1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -Rfx1 of width 14.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -SOX21 of width 15.\n",
      "Using motif -SREBF1 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +RHOXF1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +POU6F2 of width 10.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +Sox1 of width 15.\n",
      "Using motif +SPIB of width 7.\n",
      "Using motif -POU6F2 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif -SPIB of width 7.\n",
      "Using motif +SHOX of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Sox1 of width 15.\n",
      "Using motif -RHOXF1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -SHOX of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +REL of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -REL of width 10.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +SRF of width 16.\n",
      "Read 1 motifs.\n",
      "Using motif +SMAD2::SMAD3::SMAD4 of width 13.\n",
      "Read 1 motifs.\n",
      "Using motif +Six3 of width 17.\n",
      "Using motif -SMAD2::SMAD3::SMAD4 of width 13.\n",
      "Using motif -SRF of width 16.\n",
      "Using motif -Six3 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +RXRG of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +RORA(var.2) of width 14.\n",
      "Using motif -RORA(var.2) of width 14.\n",
      "Using motif -RXRG of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +RREB1 of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +SCRT1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -RREB1 of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -SCRT1 of width 15.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +TFAP2A of width 11.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Rarb(var.2) of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -TFAP2A of width 11.\n",
      "Using motif +STAT1 of width 11.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +RELA of width 10.\n",
      "Using motif -STAT1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Rarb(var.2) of width 17.\n",
      "Using motif -RELA of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +TFAP2C of width 12.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +RAX of width 10.\n",
      "Using motif -TFAP2C of width 12.\n",
      "Using motif +TBR1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -RAX of width 10.\n",
      "Using motif -TBR1 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +Rhox11 of width 17.\n",
      "Using motif -Rhox11 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +SP2 of width 15.\n",
      "Using motif -SP2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Sox11 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +Rarg(var.2) of width 17.\n",
      "Using motif +Sox2 of width 8.\n",
      "Using motif -Sox11 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Sox2 of width 8.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Stat6 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Rarg(var.2) of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif +SREBF2(var.2) of width 10.\n",
      "Using motif -Stat6 of width 15.\n",
      "Using motif -SREBF2(var.2) of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +SCRT2 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +SOX8 of width 16.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Twist2 of width 10.\n",
      "Using motif -SCRT2 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Twist2 of width 10.\n",
      "Using motif -SOX8 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +TAL1::TCF3 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +STAT3 of width 11.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -TAL1::TCF3 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif -STAT3 of width 11.\n",
      "Using motif +SP8 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -SP8 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TBX21 of width 10.\n",
      "Using motif -TBX21 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +SPIC of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -SPIC of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +T of width 16.\n",
      "Using motif +TCF3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +TEAD3 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -TCF3 of width 10.\n",
      "Using motif -TEAD3 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +TFAP2B(var.3) of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -T of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -TFAP2B(var.3) of width 13.\n",
      "Using motif +Sox17 of width 9.\n",
      "Using motif -Sox17 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TBX15 of width 8.\n",
      "Using motif -TBX15 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Sox6 of width 10.\n",
      "Using motif -Sox6 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +TBX2 of width 11.\n",
      "Using motif +Spz1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -TBX2 of width 11.\n",
      "Using motif -Spz1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TBX20 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -TBX20 of width 11.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Sox5 of width 7.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +USF2 of width 11.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -Sox5 of width 7.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -USF2 of width 11.\n",
      "Using motif +TP73 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +UNCX of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +STAT1::STAT2 of width 15.\n",
      "Using motif -UNCX of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -TP73 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -STAT1::STAT2 of width 15.\n",
      "Using motif +TFAP2C(var.3) of width 13.\n",
      "Using motif -TFAP2C(var.3) of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TBX19 of width 20.\n",
      "Using motif -TBX19 of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +TFAP2C(var.2) of width 11.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TEAD4 of width 10.\n",
      "Using motif -TFAP2C(var.2) of width 11.\n",
      "Using motif -TEAD4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TBX4 of width 8.\n",
      "Using motif -TBX4 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Tcfl5 of width 10.\n",
      "Using motif -Tcfl5 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +TFAP2B(var.2) of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +Tcf7 of width 12.\n",
      "Using motif -TFAP2B(var.2) of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +TFCP2 of width 10.\n",
      "Using motif -Tcf7 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif -TFCP2 of width 10.\n",
      "Using motif +TFAP2A(var.3) of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -TFAP2A(var.3) of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +VAX1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -VAX1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Tcf21 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +ZEB1 of width 9.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif -Tcf21 of width 14.\n",
      "Using motif +ZIC3 of width 15.\n",
      "Using motif +THAP1 of width 9.\n",
      "Using motif -ZEB1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -THAP1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ZIC3 of width 15.\n",
      "Read 1 motifs.\n",
      "Using motif +TEAD1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -TEAD1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +TFAP4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -TFAP4 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +SOX4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +USF1 of width 11.\n",
      "Using motif -SOX4 of width 16.\n",
      "Using motif -USF1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Znf423 of width 15.\n",
      "Using motif -Znf423 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TGIF1 of width 12.\n",
      "Using motif -TGIF1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TFAP2B of width 12.\n",
      "Using motif -TFAP2B of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TFE3 of width 10.\n",
      "Using motif -TFE3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +Tcf12 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -Tcf12 of width 11.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +ZBTB7C of width 12.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +YY1 of width 12.\n",
      "Using motif +ZNF263 of width 21.\n",
      "Using motif -ZBTB7C of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -YY1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +ZNF740 of width 10.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -ZNF740 of width 10.\n",
      "Using motif +VSX2 of width 8.\n",
      "Using motif -ZNF263 of width 21.\n",
      "Using motif -VSX2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ZBED1 of width 13.\n",
      "Read 1 motifs.\n",
      "Using motif +TP63 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -ZBED1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +ZNF354C of width 6.\n",
      "Using motif +ZIC1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ZNF354C of width 6.\n",
      "Using motif -TP63 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Zfx of width 14.\n",
      "Using motif -ZIC1 of width 14.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ZBTB7B of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Zfx of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif -ZBTB7B of width 12.\n",
      "Using motif +Vdr of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -Vdr of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ATF4 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +CREB1 of width 8.\n",
      "Using motif -ATF4 of width 13.\n",
      "Using motif +BHLHE22 of width 10.\n",
      "Using motif -CREB1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -BHLHE22 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +VENTX of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Bach1::Mafk of width 15.\n",
      "Using motif -VENTX of width 9.\n",
      "Read 1 motifs.\n",
      "Using motif +YY2 of width 11.\n",
      "Using motif -YY2 of width 11.\n",
      "Using motif -Bach1::Mafk of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ZBTB7A of width 12.\n",
      "Using motif -ZBTB7A of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TGIF2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -TGIF2 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +ZNF143 of width 16.\n",
      "Using motif -ZNF143 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ZBTB18 of width 13.\n",
      "Using motif -ZBTB18 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +VAX2 of width 8.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -VAX2 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +mix-a of width 11.\n",
      "Using motif -mix-a of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Arid3a of width 6.\n",
      "Using motif +Alx4 of width 17.\n",
      "Using motif +CEBPB of width 10.\n",
      "Using motif -Arid3a of width 6.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -CEBPB of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +VSX1 of width 8.\n",
      "Using motif -Alx4 of width 17.\n",
      "Using motif -VSX1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ZNF410 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -ZNF410 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ZBTB33 of width 15.\n",
      "Using motif -ZBTB33 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ZIC4 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +XBP1 of width 14.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -ZIC4 of width 15.\n",
      "Using motif +Barhl1 of width 10.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Ahr::Arnt of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -XBP1 of width 14.\n",
      "Using motif -Ahr::Arnt of width 6.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +Arnt of width 6.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Barhl1 of width 10.\n",
      "Using motif -Arnt of width 6.\n",
      "Read 1 motifs.\n",
      "Using motif +CEBPA of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -CEBPA of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +Atf1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Atf1 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +ALX3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -ALX3 of width 10.\n",
      "Using motif +BHLHE23 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -BHLHE23 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Ddit3::Cebpa of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +CDX1 of width 9.\n",
      "Using motif -Ddit3::Cebpa of width 12.\n",
      "Using motif -CDX1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TP53 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -TP53 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Dlx2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -Dlx2 of width 8.\n",
      "Using motif +BCL6B of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif +BHLHE40 of width 10.\n",
      "Using motif -BHLHE40 of width 10.\n",
      "Using motif -BCL6B of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +CREB3 of width 14.\n",
      "Using motif -CREB3 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +CEBPE of width 10.\n",
      "Using motif -CEBPE of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +ATF7 of width 14.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ATF7 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +Ar of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +CTCF of width 19.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Atoh1 of width 10.\n",
      "Using motif -Ar of width 17.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ARNT::HIF1A of width 8.\n",
      "Using motif -Atoh1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ARNT::HIF1A of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif -CTCF of width 19.\n",
      "Using motif +CEBPG of width 10.\n",
      "Using motif -CEBPG of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +CDX2 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +Bcl6 of width 14.\n",
      "Using motif -CDX2 of width 11.\n",
      "Using motif -Bcl6 of width 14.\n",
      "Using motif +Arx of width 17.\n",
      "Using motif -Arx of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +Creb5 of width 12.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +BARX1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Arid3b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -BARX1 of width 8.\n",
      "Using motif -Creb5 of width 12.\n",
      "Using motif +Alx1 of width 17.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +BARHL2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -Arid3b of width 11.\n",
      "Using motif +BSX of width 8.\n",
      "Using motif -BARHL2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Alx1 of width 17.\n",
      "Using motif -BSX of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +Bhlha15 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Bhlha15 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +Arntl of width 10.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -Arntl of width 10.\n",
      "Using motif +BATF::JUN of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -BATF::JUN of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Ascl2 of width 10.\n",
      "Using motif -Ascl2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Crem of width 10.\n",
      "Using motif -Crem of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Atf3 of width 8.\n",
      "Using motif -Atf3 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +CREB3L1 of width 14.\n",
      "Using motif -CREB3L1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +E2F4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -E2F4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +CUX2 of width 10.\n",
      "Using motif +Dmbx1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -CUX2 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +CEBPD of width 10.\n",
      "Using motif -Dmbx1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -CEBPD of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +DMRT3 of width 11.\n",
      "Using motif -DMRT3 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +DLX6 of width 8.\n",
      "Using motif -DLX6 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +Dlx4 of width 8.\n",
      "Using motif +E2F6 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Dlx4 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -E2F6 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +CUX1 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +DBP of width 12.\n",
      "Using motif -CUX1 of width 10.\n",
      "Using motif -DBP of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Arid5a of width 14.\n",
      "Using motif -Arid5a of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +CENPB of width 15.\n",
      "Using motif -CENPB of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +E2F3 of width 18.\n",
      "Using motif +Dux of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Dux of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +CLOCK of width 10.\n",
      "Using motif -CLOCK of width 10.\n",
      "Using motif -E2F3 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +BATF3 of width 14.\n",
      "Using motif +E2F2 of width 16.\n",
      "Read 1 motifs.\n",
      "Using motif +E2F1 of width 12.\n",
      "Using motif -BATF3 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -E2F1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -E2F2 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +EMX1 of width 10.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +Dlx1 of width 10.\n",
      "Using motif -EMX1 of width 10.\n",
      "Using motif +DUX4 of width 11.\n",
      "Using motif -Dlx1 of width 10.\n",
      "Using motif -DUX4 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ETV6 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -ETV6 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +EGR4 of width 16.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +ETV1 of width 10.\n",
      "Using motif +BHLHE41 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ETV1 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif -BHLHE41 of width 10.\n",
      "Using motif -EGR4 of width 16.\n",
      "Using motif +ELK3 of width 10.\n",
      "Using motif -ELK3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +E2F7 of width 14.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +EBF1 of width 14.\n",
      "Using motif -E2F7 of width 14.\n",
      "Using motif -EBF1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +E2F8 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Dlx3 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif -E2F8 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ELF3 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Dlx3 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +ETV2 of width 11.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +ETV5 of width 10.\n",
      "Using motif +ERG of width 10.\n",
      "Using motif -ELF3 of width 13.\n",
      "Using motif -ETV2 of width 11.\n",
      "Using motif -ETV5 of width 10.\n",
      "Using motif -ERG of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +Crx of width 11.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +EGR1 of width 14.\n",
      "Using motif -Crx of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -EGR1 of width 14.\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +ELF5 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +ELF4 of width 12.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -ELF5 of width 11.\n",
      "Using motif +Creb3l2 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -ELF4 of width 12.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Creb3l2 of width 9.\n",
      "Read 1 motifs.\n",
      "Using motif +EGR3 of width 15.\n",
      "Using motif -EGR3 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ESX1 of width 10.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ERF of width 10.\n",
      "Using motif -ESX1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ERF of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +EGR2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -EGR2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +FOS of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -FOS of width 11.\n",
      "Using motif +ETS1 of width 10.\n",
      "Using motif +ELK4 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +EN2 of width 10.\n",
      "Using motif -ETS1 of width 10.\n",
      "Using motif -ELK4 of width 11.\n",
      "Using motif -EN2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +GATA2 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -GATA2 of width 14.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +EOMES of width 13.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ELF1 of width 12.\n",
      "Using motif -EOMES of width 13.\n",
      "Using motif -ELF1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +EMX2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +EWSR1-FLI1 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -EMX2 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +EHF of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +FEV of width 10.\n",
      "Using motif -EHF of width 12.\n",
      "Using motif -EWSR1-FLI1 of width 18.\n",
      "Using motif -FEV of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +ESR1 of width 17.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXB1 of width 11.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +GATA5 of width 8.\n",
      "Using motif +FOXH1 of width 11.\n",
      "Using motif -FOXB1 of width 11.\n",
      "Using motif -GATA5 of width 8.\n",
      "Using motif -FOXH1 of width 11.\n",
      "Using motif -ESR1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +ETV3 of width 10.\n",
      "Using motif +ELK1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +ETV4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ETV3 of width 10.\n",
      "Using motif -ELK1 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif -ETV4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +FOXG1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -FOXG1 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +EVX2 of width 10.\n",
      "Using motif -EVX2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXP1 of width 15.\n",
      "Using motif -FOXP1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOSL1 of width 11.\n",
      "Using motif -FOSL1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +DUXA of width 13.\n",
      "Using motif +Esrra of width 11.\n",
      "Using motif -DUXA of width 13.\n",
      "Using motif -Esrra of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +GMEB2 of width 8.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +FLI1 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +GLI2 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif -GMEB2 of width 8.\n",
      "Using motif +FOXI1 of width 7.\n",
      "Using motif -FLI1 of width 10.\n",
      "Using motif -FOXI1 of width 7.\n",
      "Using motif -GLI2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +EVX1 of width 10.\n",
      "Using motif -EVX1 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +ESR2 of width 15.\n",
      "Using motif -ESR2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +FOXP2 of width 11.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Gabpa of width 11.\n",
      "Using motif -FOXP2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Gabpa of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +Foxd3 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +Esrrg of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Foxd3 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -Esrrg of width 10.\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +EN1 of width 8.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif -EN1 of width 8.\n",
      "Using motif +FOXF2 of width 14.\n",
      "Using motif +Foxj2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Foxj2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -FOXF2 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +ESRRB of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOSL2 of width 11.\n",
      "Using motif -ESRRB of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -FOSL2 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXA1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +FOXD1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -FOXA1 of width 15.\n",
      "Using motif -FOXD1 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +FIGLA of width 10.\n",
      "Using motif -FIGLA of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXL1 of width 7.\n",
      "Using motif -FOXL1 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Foxj3 of width 17.\n",
      "Using motif -Foxj3 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXO6 of width 7.\n",
      "Using motif -FOXO6 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +GBX2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -GBX2 of width 10.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +FOXO3 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif -FOXO3 of width 8.\n",
      "Using motif +FOS::JUN of width 7.\n",
      "Using motif -FOS::JUN of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXD2 of width 7.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +FOXC2 of width 12.\n",
      "Using motif -FOXD2 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -FOXC2 of width 12.\n",
      "Using motif +Gfi1b of width 11.\n",
      "Using motif -Gfi1b of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXC1 of width 11.\n",
      "Using motif -FOXC1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Foxa2 of width 12.\n",
      "Using motif -Foxa2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Foxo1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Foxo1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +HIC2 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -HIC2 of width 9.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +HEY1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -HEY1 of width 10.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +GRHL1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +Foxk1 of width 17.\n",
      "Using motif -GRHL1 of width 12.\n",
      "Using motif +FOXO4 of width 7.\n",
      "Using motif -FOXO4 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Foxk1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Gmeb1 of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif +GLIS1 of width 16.\n",
      "Using motif -Gmeb1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -GLIS1 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +GATA3 of width 8.\n",
      "Using motif -GATA3 of width 8.\n",
      "Using motif +HOXD11 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -HOXD11 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +GLIS3 of width 14.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Gata4 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -GLIS3 of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif -Gata4 of width 11.\n",
      "Using motif +HOXA2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -HOXA2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +HOXC10 of width 10.\n",
      "Using motif +GBX1 of width 10.\n",
      "Using motif -HOXC10 of width 10.\n",
      "Using motif -GBX1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Foxq1 of width 11.\n",
      "Using motif -Foxq1 of width 11.\n",
      "Using motif +HEY2 of width 10.\n",
      "Using motif -HEY2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +HOXB3 of width 10.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif -HOXB3 of width 10.\n",
      "Using motif +Hmx1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +GSC of width 10.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -GSC of width 10.\n",
      "Using motif +GCM2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -GCM2 of width 10.\n",
      "Using motif -Hmx1 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +HESX1 of width 10.\n",
      "Using motif -HESX1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Gfi1 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +HLF of width 12.\n",
      "Using motif -Gfi1 of width 10.\n",
      "Using motif -HLF of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +HINFP of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +HES7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -HINFP of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -HES7 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +HLTF of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -HLTF of width 10.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +GCM1 of width 11.\n",
      "Using motif +Hmx2 of width 17.\n",
      "Using motif -GCM1 of width 11.\n",
      "Using motif -Hmx2 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +HSF1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +HNF4G of width 15.\n",
      "Using motif -HSF1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +FOXP3 of width 7.\n",
      "Using motif -HNF4G of width 15.\n",
      "Read 1 motifs.\n",
      "Using motif -FOXP3 of width 7.\n",
      "Using motif +Hic1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Hic1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +GSC2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -GSC2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +HOXC11 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +Hes1 of width 10.\n",
      "Using motif -HOXC11 of width 11.\n",
      "Using motif -Hes1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +HOXD12 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -HOXD12 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Id2 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif -Id2 of width 8.\n",
      "Using motif +HNF1A of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -HNF1A of width 15.\n",
      "Read 1 motifs.\n",
      "Using motif +GSX2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -GSX2 of width 10.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +INSM1 of width 12.\n",
      "Using motif +IRF8 of width 14.\n",
      "Using motif -INSM1 of width 12.\n",
      "Using motif -IRF8 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +JDP2 of width 9.\n",
      "Using motif -JDP2 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +Gata1 of width 11.\n",
      "Using motif +HES5 of width 12.\n",
      "Using motif -Gata1 of width 11.\n",
      "Using motif -HES5 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +HSF2 of width 13.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +Hoxb5 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +HMBOX1 of width 10.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +GLIS2 of width 14.\n",
      "Using motif -HMBOX1 of width 10.\n",
      "Using motif -HSF2 of width 13.\n",
      "Using motif -Hoxb5 of width 16.\n",
      "Read 1 motifs.\n",
      "Using motif +GSX1 of width 10.\n",
      "Using motif -GLIS2 of width 14.\n",
      "Using motif -GSX1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ID4 of width 10.\n",
      "Using motif -ID4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +Hmx3 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +HOXA10 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +HOXD13 of width 10.\n",
      "Using motif -HOXA10 of width 11.\n",
      "Using motif -Hmx3 of width 17.\n",
      "Using motif -HOXD13 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +GATA1::TAL1 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -GATA1::TAL1 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Hnf4a of width 16.\n",
      "Read 1 motifs.\n",
      "Using motif +Lhx8 of width 8.\n",
      "Using motif -Lhx8 of width 8.\n",
      "Using motif -Hnf4a of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +JDP2(var.2) of width 12.\n",
      "Using motif -JDP2(var.2) of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +HOXC13 of width 11.\n",
      "Using motif -HOXC13 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +Hoxa9 of width 11.\n",
      "Using motif -Hoxa9 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +IRF7 of width 14.\n",
      "Using motif -IRF7 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Hoxc9 of width 13.\n",
      "Using motif -Hoxc9 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +Hoxd9 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Hoxd9 of width 10.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +LBX2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -LBX2 of width 10.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +HOXA5 of width 8.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -HOXA5 of width 8.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +JUN(var.2) of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +IRF2 of width 18.\n",
      "Using motif -JUN(var.2) of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +JUNB of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +IRF9 of width 15.\n",
      "Using motif -IRF2 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -JUNB of width 11.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +HOXB2 of width 10.\n",
      "Using motif -IRF9 of width 15.\n",
      "Using motif -HOXB2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +JUN of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -JUN of width 13.\n",
      "Using motif +KLF14 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +Hes2 of width 13.\n",
      "Using motif +HNF1B of width 13.\n",
      "Using motif -KLF14 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -HNF1B of width 13.\n",
      "Using motif -Hes2 of width 13.\n",
      "Read 1 motifs.\n",
      "Using motif +JUND of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -JUND of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +LBX1 of width 8.\n",
      "Using motif -LBX1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +IRF1 of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -IRF1 of width 21.\n",
      "Read 1 motifs.\n",
      "Using motif +HOXC12 of width 11.\n",
      "Using motif -HOXC12 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +Klf1 of width 11.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Hoxa11 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif -Klf1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +KLF13 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -Hoxa11 of width 12.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MZF1 of width 6.\n",
      "Read 1 motifs.\n",
      "Using motif -MZF1 of width 6.\n",
      "Using motif +LHX2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -KLF13 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -LHX2 of width 10.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +Hand1::Tcf3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Hoxd3 of width 16.\n",
      "Read 1 motifs.\n",
      "Using motif -Hand1::Tcf3 of width 10.\n",
      "Using motif +KLF16 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Hoxd3 of width 16.\n",
      "Using motif -KLF16 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MEF2D of width 12.\n",
      "Using motif -MEF2D of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +LMX1A of width 8.\n",
      "Using motif -LMX1A of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ISL2 of width 8.\n",
      "Using motif -ISL2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MEF2C of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -MEF2C of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MEIS1 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -MEIS1 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Klf4 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MYBL1 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif -Klf4 of width 10.\n",
      "Using motif +MAFG::NFE2L1 of width 6.\n",
      "Using motif -MAFG::NFE2L1 of width 6.\n",
      "Using motif -MYBL1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Lhx4 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +MAFG of width 21.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Lhx4 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Hoxd8 of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif +JUND(var.2) of width 15.\n",
      "Using motif -MAFG of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Hoxd8 of width 17.\n",
      "Using motif -JUND(var.2) of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Klf12 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +MIXL1 of width 10.\n",
      "Using motif -Klf12 of width 15.\n",
      "Read 1 motifs.\n",
      "Using motif +MAF::NFE2 of width 15.\n",
      "Using motif -MIXL1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -MAF::NFE2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MAX of width 10.\n",
      "Using motif -MAX of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MYF6 of width 10.\n",
      "Using motif -MYF6 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +MYBL2 of width 15.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +HOXB13 of width 10.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -HOXB13 of width 10.\n",
      "Using motif +MAX::MYC of width 11.\n",
      "Using motif -MYBL2 of width 15.\n",
      "Using motif -MAX::MYC of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +MEF2A of width 12.\n",
      "Using motif +LHX9 of width 8.\n",
      "Using motif -LHX9 of width 8.\n",
      "Using motif -MEF2A of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +LIN54 of width 9.\n",
      "Read 1 motifs.\n",
      "Using motif +LHX6 of width 10.\n",
      "Using motif -LIN54 of width 9.\n",
      "Using motif -LHX6 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +MLXIPL of width 10.\n",
      "Using motif +LEF1 of width 15.\n",
      "Using motif -MLXIPL of width 10.\n",
      "Using motif -LEF1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +HSF4 of width 13.\n",
      "Using motif -HSF4 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MEIS2 of width 8.\n",
      "Using motif -MEIS2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +Lhx3 of width 13.\n",
      "Using motif +MEIS3 of width 8.\n",
      "Using motif -MEIS3 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ISX of width 8.\n",
      "Using motif -Lhx3 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -ISX of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MEOX2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -MEOX2 of width 10.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +MSX2 of width 8.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -MSX2 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +MSC of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -MSC of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +NFATC2 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -NFATC2 of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MEF2B of width 12.\n",
      "Using motif -MEF2B of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MEOX1 of width 10.\n",
      "Using motif -MEOX1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MLX of width 10.\n",
      "Using motif -MLX of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +HOXA13 of width 10.\n",
      "Using motif -HOXA13 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MAFF of width 18.\n",
      "Using motif -MAFF of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Nr2e3 of width 7.\n",
      "Using motif -Nr2e3 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MGA of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -MGA of width 8.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MAFK of width 15.\n",
      "Using motif -MAFK of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +NFATC1 of width 10.\n",
      "Using motif -NFATC1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +KLF5 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -KLF5 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +NHLH1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +Myog of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +LMX1B of width 8.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -NHLH1 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif -LMX1B of width 8.\n",
      "Using motif +Mycn of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Myog of width 11.\n",
      "Using motif -Mycn of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Myod1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -Myod1 of width 13.\n",
      "Using motif +NFIL3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -NFIL3 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +Nkx2-5(var.2) of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +NFYA of width 18.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +NFIC of width 6.\n",
      "Using motif -Nkx2-5(var.2) of width 11.\n",
      "Using motif -NFIC of width 6.\n",
      "Using motif -NFYA of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +NR1H2::RXRA of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +Msx3 of width 8.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +MSX1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Msx3 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +Mecom of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif -MSX1 of width 8.\n",
      "Using motif +MTF1 of width 14.\n",
      "Using motif -NR1H2::RXRA of width 17.\n",
      "Using motif -Mecom of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -MTF1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +MNT of width 10.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -MNT of width 10.\n",
      "Using motif +Nobox of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Nobox of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +NFIC::TLX1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -NFIC::TLX1 of width 14.\n",
      "Using motif +NFIX of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -NFIX of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +NFKB1 of width 13.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +MZF1(var.2) of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -NFKB1 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -MZF1(var.2) of width 10.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +NFAT5 of width 10.\n",
      "Using motif +NKX6-2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -NKX6-2 of width 8.\n",
      "Using motif -NFAT5 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Nkx2-5 of width 7.\n",
      "Using motif -Nkx2-5 of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +NFKB2 of width 13.\n",
      "Using motif -NFKB2 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Mlxip of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Mlxip of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +OLIG2 of width 10.\n",
      "Using motif -OLIG2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +NKX6-1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -NKX6-1 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +NKX2-3 of width 10.\n",
      "Using motif -NKX2-3 of width 10.\n",
      "Using motif +NEUROG2 of width 10.\n",
      "Using motif -NEUROG2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +MNX1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -MNX1 of width 10.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +NRL of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +NFATC3 of width 10.\n",
      "Using motif -NRL of width 11.\n",
      "Using motif -NFATC3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +Nr2e1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +NR3C2 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -Nr2e1 of width 9.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Neurog1 of width 10.\n",
      "Using motif -NR3C2 of width 17.\n",
      "Using motif -Neurog1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +NR4A2 of width 8.\n",
      "Using motif -NR4A2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +NFE2 of width 11.\n",
      "Using motif -NFE2 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +NOTO of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +NR3C1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -NOTO of width 10.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +Mafb of width 12.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Myb of width 10.\n",
      "Using motif -Mafb of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -NR3C1 of width 17.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Myb of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +Myc of width 10.\n",
      "Using motif -Myc of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Mitf of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif -Mitf of width 10.\n",
      "Using motif +NR2C2 of width 15.\n",
      "Using motif -NR2C2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +NKX3-2 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -NKX3-2 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +OTX2 of width 8.\n",
      "Using motif -OTX2 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +Nr2f6 of width 14.\n",
      "Using motif -Nr2f6 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +NKX2-8 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -NKX2-8 of width 9.\n",
      "Read 1 motifs.\n",
      "Using motif +PDX1 of width 8.\n",
      "Using motif -PDX1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +Nr1h3::Rxra of width 19.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -Nr1h3::Rxra of width 19.\n",
      "Using motif +NEUROD2 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -NEUROD2 of width 10.\n",
      "Using motif +OLIG3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -OLIG3 of width 10.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +PITX3 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -PITX3 of width 9.\n",
      "Read 1 motifs.\n",
      "Using motif +NFYB of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -NFYB of width 15.\n",
      "Using motif +ONECUT1 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +PKNOX1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -ONECUT1 of width 14.\n",
      "Using motif +PBX1 of width 12.\n",
      "Using motif -PKNOX1 of width 12.\n",
      "Using motif -PBX1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +PAX1 of width 17.\n",
      "Using motif -PAX1 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +ONECUT2 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +PROP1 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +PHOX2A of width 11.\n",
      "Using motif -ONECUT2 of width 14.\n",
      "Using motif -PROP1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +Nr5a2 of width 15.\n",
      "Using motif -PHOX2A of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +POU4F3 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -Nr5a2 of width 15.\n",
      "Using motif +POU2F1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -POU4F3 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -POU2F1 of width 12.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +Nkx3-1 of width 9.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -Nkx3-1 of width 9.\n",
      "Using motif +POU3F2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -POU3F2 of width 12.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ONECUT3 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -ONECUT3 of width 14.\n",
      "Using motif +POU4F1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +RHOXF1 of width 8.\n",
      "Using motif -POU4F1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -RHOXF1 of width 8.\n",
      "Using motif +PROX1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Nfe2l2 of width 15.\n",
      "Using motif -PROX1 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +Npas2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Nfe2l2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Npas2 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +PRRX1 of width 8.\n",
      "Using motif -PRRX1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +NR2F1 of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -NR2F1 of width 13.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +OLIG1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -OLIG1 of width 10.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +POU6F1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +POU3F1 of width 12.\n",
      "Using motif -POU6F1 of width 10.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -POU3F1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +NRF1 of width 11.\n",
      "Using motif +POU3F3 of width 13.\n",
      "Using motif +PAX7 of width 10.\n",
      "Using motif -PAX7 of width 10.\n",
      "Using motif -POU3F3 of width 13.\n",
      "Using motif -NRF1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +Nr2f6(var.2) of width 15.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +RREB1 of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +PAX3 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif -PAX3 of width 10.\n",
      "Using motif +POU5F1B of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -POU5F1B of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Nr2f6(var.2) of width 15.\n",
      "Using motif -RREB1 of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +PAX9 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +PLAG1 of width 14.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +POU4F2 of width 16.\n",
      "Using motif -PAX9 of width 17.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -PLAG1 of width 14.\n",
      "Using motif +NFIA of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -POU4F2 of width 16.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif -NFIA of width 10.\n",
      "Using motif +PAX5 of width 19.\n",
      "Read 1 motifs.\n",
      "Using motif +Pou5f1::Sox2 of width 15.\n",
      "Using motif +RARA::RXRA of width 17.\n",
      "Using motif -Pou5f1::Sox2 of width 15.\n",
      "Using motif -PAX5 of width 19.\n",
      "Using motif -RARA::RXRA of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +PAX4 of width 8.\n",
      "Using motif -PAX4 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +POU3F4 of width 9.\n",
      "Using motif +POU1F1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -POU3F4 of width 9.\n",
      "Read 1 motifs.\n",
      "Using motif +RARA(var.2) of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -POU1F1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif -RARA(var.2) of width 17.\n",
      "Read 1 motifs.\n",
      "Using motif +Rarb(var.2) of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Pou2f3 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +RAX of width 10.\n",
      "Using motif -Pou2f3 of width 16.\n",
      "Using motif -Rarb(var.2) of width 17.\n",
      "Using motif -RAX of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +RFX5 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +POU6F2 of width 10.\n",
      "Using motif +Prrx2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Prrx2 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -POU6F2 of width 10.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +POU2F2 of width 13.\n",
      "Using motif -RFX5 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +RFX2 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -POU2F2 of width 13.\n",
      "Read 1 motifs.\n",
      "Using motif +Pax2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +Pax6 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -Pax2 of width 8.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -RFX2 of width 16.\n",
      "Read 1 motifs.\n",
      "Using motif +RUNX2 of width 9.\n",
      "Using motif -Pax6 of width 14.\n",
      "Using motif -RUNX2 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Rhox11 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif -Rhox11 of width 17.\n",
      "Using motif +PRDM1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -PRDM1 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +Pitx1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Pitx1 of width 8.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Pparg::Rxra of width 15.\n",
      "Read 1 motifs.\n",
      "Using motif +RFX3 of width 16.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +RARA of width 18.\n",
      "Using motif -Pparg::Rxra of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -RFX3 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Phox2b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif -Phox2b of width 11.\n",
      "Using motif +PPARG of width 20.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -RARA of width 18.\n",
      "Using motif +RXRB of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +RUNX1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -RUNX1 of width 11.\n",
      "Using motif -RXRB of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -PPARG of width 20.\n",
      "Read 1 motifs.\n",
      "Using motif +PKNOX2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -PKNOX2 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +SCRT1 of width 15.\n",
      "Using motif -SCRT1 of width 15.\n",
      "Using motif +OTX1 of width 8.\n",
      "Using motif -OTX1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +REL of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -REL of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +REST of width 21.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Rxra of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Rxra of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -REST of width 21.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Rarb of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +SOX10 of width 6.\n",
      "Using motif +RELA of width 10.\n",
      "Using motif -SOX10 of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -RELA of width 10.\n",
      "Using motif -Rarb of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +SPI1 of width 14.\n",
      "Using motif +RFX4 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +RAX2 of width 8.\n",
      "Using motif -RAX2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -SPI1 of width 14.\n",
      "Using motif -RFX4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +RORA(var.2) of width 14.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +SRY of width 9.\n",
      "Read 1 motifs.\n",
      "Using motif +SOX9 of width 9.\n",
      "Using motif -RORA(var.2) of width 14.\n",
      "Using motif -SRY of width 9.\n",
      "Using motif -SOX9 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +SMAD3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -SMAD3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +RXRG of width 14.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +SP2 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -RXRG of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Rarg(var.2) of width 17.\n",
      "Using motif -SP2 of width 15.\n",
      "Read 1 motifs.\n",
      "Using motif +RUNX3 of width 10.\n",
      "Using motif -RUNX3 of width 10.\n",
      "Using motif -Rarg(var.2) of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Shox2 of width 8.\n",
      "Using motif -Shox2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +RXRA::VDR of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -RXRA::VDR of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +RORA of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +SREBF2 of width 10.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Six3 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -RORA of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -SREBF2 of width 10.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +SP3 of width 11.\n",
      "Using motif +SREBF2(var.2) of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Six3 of width 17.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -SREBF2(var.2) of width 10.\n",
      "Using motif -SP3 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +SPDEF of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -SPDEF of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +SP8 of width 12.\n",
      "Using motif +SHOX of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -SHOX of width 8.\n",
      "Using motif -SP8 of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +SP4 of width 17.\n",
      "Using motif -SP4 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Rfx1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +SMAD2::SMAD3::SMAD4 of width 13.\n",
      "Read 1 motifs.\n",
      "Using motif -Rfx1 of width 14.\n",
      "Using motif +SOX8 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -SMAD2::SMAD3::SMAD4 of width 13.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +SOX21 of width 15.\n",
      "Using motif +SCRT2 of width 13.\n",
      "Using motif -SOX8 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +SNAI2 of width 9.\n",
      "Using motif -SCRT2 of width 13.\n",
      "Using motif -SOX21 of width 15.\n",
      "Using motif -SNAI2 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Rarg of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TBX20 of width 11.\n",
      "Using motif -Rarg of width 16.\n",
      "Using motif -TBX20 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TBP of width 15.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +SPIB of width 7.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -SPIB of width 7.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif -TBP of width 15.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +SREBF1 of width 10.\n",
      "Using motif +TBR1 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +TBX2 of width 11.\n",
      "Using motif -SREBF1 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -TBR1 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -TBX2 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +Srebf1(var.2) of width 10.\n",
      "Using motif -Srebf1(var.2) of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Spz1 of width 11.\n",
      "Using motif -Spz1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +SRF of width 16.\n",
      "Read 1 motifs.\n",
      "Using motif +Stat6 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -SRF of width 16.\n",
      "Using motif +SOX4 of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Stat6 of width 15.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Sox5 of width 7.\n",
      "Using motif +STAT1 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif -Sox5 of width 7.\n",
      "Using motif +STAT3 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -STAT1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -SOX4 of width 16.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -STAT3 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +Sox2 of width 8.\n",
      "Using motif -Sox2 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Sox1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Sox1 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Sox3 of width 10.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +SP1 of width 11.\n",
      "Using motif -Sox3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -SP1 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +T of width 16.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +TCF7L2 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +TBX4 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -TCF7L2 of width 14.\n",
      "Using motif -T of width 16.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -TBX4 of width 8.\n",
      "Using motif +TBX5 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -TBX5 of width 8.\n",
      "Using motif +TEAD3 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif -TEAD3 of width 8.\n",
      "Using motif +SPIC of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -SPIC of width 14.\n",
      "Read 1 motifs.\n",
      "Using motif +Sox11 of width 15.\n",
      "Using motif -Sox11 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +Stat4 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TBX21 of width 10.\n",
      "Using motif -Stat4 of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -TBX21 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TFAP2A of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +TBX19 of width 20.\n",
      "Using motif -TFAP2A of width 11.\n",
      "Using motif -TBX19 of width 20.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +Sox6 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -Sox6 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +Stat5a::Stat5b of width 11.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +TEAD4 of width 10.\n",
      "Using motif -Stat5a::Stat5b of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -TEAD4 of width 10.\n",
      "Using motif +Sox17 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -Sox17 of width 9.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +TBX15 of width 8.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -TBX15 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif +TFAP2C of width 12.\n",
      "Using motif -TFAP2C of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +TFAP2B of width 12.\n",
      "Using motif +TAL1::TCF3 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -TFAP2B of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif -TAL1::TCF3 of width 12.\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +TFAP2A(var.3) of width 13.\n",
      "Using motif +TBX1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -TBX1 of width 8.\n",
      "Using motif -TFAP2A(var.3) of width 13.\n",
      "Read 1 motifs.\n",
      "Using motif +Tcf12 of width 11.\n",
      "Using motif -Tcf12 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +THAP1 of width 9.\n",
      "Using motif -THAP1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TFEC of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -TFEC of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +STAT1::STAT2 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -STAT1::STAT2 of width 15.\n",
      "Using motif +XBP1 of width 14.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Twist2 of width 10.\n",
      "Using motif -XBP1 of width 14.\n",
      "Using motif -Twist2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TFAP2B(var.3) of width 13.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -TFAP2B(var.3) of width 13.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +TFAP2C(var.2) of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -TFAP2C(var.2) of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif +Vdr of width 16.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +TP73 of width 18.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +TFAP2A(var.2) of width 12.\n",
      "Using motif +TEAD1 of width 10.\n",
      "Using motif -Vdr of width 16.\n",
      "Using motif -TP73 of width 18.\n",
      "Using motif -TEAD1 of width 10.\n",
      "Using motif -TFAP2A(var.2) of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TCF4 of width 10.\n",
      "Using motif -TCF4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +Tcf7 of width 12.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +TFEB of width 10.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +ZBTB7B of width 12.\n",
      "Using motif -Tcf7 of width 12.\n",
      "Using motif -TFEB of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +ZBTB7C of width 12.\n",
      "Using motif -ZBTB7B of width 12.\n",
      "Using motif -ZBTB7C of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ZEB1 of width 9.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -ZEB1 of width 9.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +TFAP2C(var.3) of width 13.\n",
      "Using motif +VENTX of width 9.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using motif +TEF of width 12.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -VENTX of width 9.\n",
      "Read 1 motifs.\n",
      "Using motif -TFAP2C(var.3) of width 13.\n",
      "Using motif +TFCP2 of width 10.\n",
      "Using motif -TEF of width 12.\n",
      "Using motif -TFCP2 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +TP63 of width 18.\n",
      "Using motif -TP63 of width 18.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +VSX2 of width 8.\n",
      "Using motif +YY2 of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif -VSX2 of width 8.\n",
      "Using motif +TCF3 of width 10.\n",
      "Using motif -YY2 of width 11.\n",
      "Using motif -TCF3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +USF2 of width 11.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ZBTB18 of width 13.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Read 1 motifs.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -USF2 of width 11.\n",
      "Using motif +ZIC3 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +VAX1 of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ZBTB18 of width 13.\n",
      "Using motif -VAX1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -ZIC3 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +VAX2 of width 8.\n",
      "Read 1 motifs.\n",
      "Using motif -VAX2 of width 8.\n",
      "Using motif +TFAP4 of width 10.\n",
      "Using motif -TFAP4 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Read 1 motifs.\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +TP53 of width 18.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +ZNF410 of width 17.\n",
      "Using motif +TFAP2B(var.2) of width 11.\n",
      "Read 1 motifs.\n",
      "Using motif -TFAP2B(var.2) of width 11.\n",
      "Using motif -TP53 of width 18.\n",
      "Using motif +TGIF1 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -ZNF410 of width 17.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -TGIF1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using background frequencies from NR sequence database.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Read 1 motifs.\n",
      "Using motif +ZIC1 of width 14.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +ZBTB7A of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ZIC1 of width 14.\n",
      "Using motif -ZBTB7A of width 12.\n",
      "Read 1 motifs.\n",
      "Using motif +ZBED1 of width 13.\n",
      "Read 1 motifs.\n",
      "Using motif +ZIC4 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -ZBED1 of width 13.\n",
      "Using motif -ZIC4 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +ZBTB33 of width 15.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +ZNF143 of width 16.\n",
      "Read 1 motifs.\n",
      "Using motif -ZBTB33 of width 15.\n",
      "Using motif +USF1 of width 11.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -USF1 of width 11.\n",
      "Using motif -ZNF143 of width 16.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif +ZNF740 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -ZNF740 of width 10.\n",
      "Read 1 motifs.\n",
      "Using motif +TFE3 of width 10.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -TFE3 of width 10.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +Zfx of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using motif +Tcfl5 of width 10.\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Zfx of width 14.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Using motif -Tcfl5 of width 10.\n",
      "Using motif +Znf423 of width 15.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif +YY1 of width 12.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif -Znf423 of width 15.\n",
      "Read 1 motifs.\n",
      "Using motif +ZNF354C of width 6.\n",
      "Using motif -YY1 of width 12.\n",
      "Using motif -ZNF354C of width 6.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +VSX1 of width 8.\n",
      "Using motif -VSX1 of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +UNCX of width 8.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Using motif -UNCX of width 8.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Read 1 motifs.\n",
      "Using motif +Tcf21 of width 14.\n",
      "Using motif -Tcf21 of width 14.\n",
      "Using motif +TGIF2 of width 12.\n",
      "Using motif -TGIF2 of width 12.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Warning: text mode turns off computation of q-values\n",
      "Read 1 motifs.\n",
      "Reading txt file vmajor: 4, vminor: 4, vbug: 0\n",
      "Using nucleotide alphabet (ACGTURYKMSWBDHVN).\n",
      "Using background frequencies from NR sequence database.\n",
      "Using motif +ZNF263 of width 21.\n",
      "Read 1 motifs.\n",
      "Using motif +mix-a of width 11.\n",
      "Using motif -mix-a of width 11.\n",
      "Using motif -ZNF263 of width 21.\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "if [ ! -d ./fimo_results ]; then mkdir ./fimo_results; else rm -rf ./fimo_results/*; fi\n",
    "chmod a+x ./scanMotifs*\n",
    "./scanMotifs_C57BL6J.sh\n",
    "./scanMotifs_C57BL6J-background.sh"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Read in Motif Scores"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def read_fimo_file(thread_lock, motif_results_path, \n",
    "                   all_peak_ids,\n",
    "                   motif_score_dict,\n",
    "                   motif_sequence_dict,\n",
    "                   motif_strand_dict,\n",
    "                   motif_start_dict,\n",
    "                   motif_end_dict,\n",
    "                   motif_count_dict,\n",
    "                  ):\n",
    "    \n",
    "    # read in fimo result as data frame\n",
    "    fimo_result_frame=pd.read_csv(motif_results_path, \n",
    "                                  skiprows=1,\n",
    "                                  names=['motif_name', \n",
    "                                         'peak_id', \n",
    "                                         'start', \n",
    "                                         'stop', \n",
    "                                         'strand', \n",
    "                                         'score', \n",
    "                                         'pvalue', \n",
    "                                         'sequence'],\n",
    "                                  sep='\\t')\n",
    "    motif_name = fimo_result_frame['motif_name'].values[0]\n",
    "    print('reading', motif_name)\n",
    "    \n",
    "    id_values_dict = {} # {PeakID:(motifScore, motifSequence, motifStrand, motifStart, motifEnd)}\n",
    "    # drop all motif instances that has less than the maximum score\n",
    "    sorted_fimo_result_frame = fimo_result_frame.sort_values(by='score', ascending=False)\n",
    "    top_fimo_result_frame = sorted_fimo_result_frame.drop_duplicates(subset='peak_id')\n",
    "    \n",
    "    # convert data frame to a dictionary \n",
    "    unique_peak_ids = top_fimo_result_frame['peak_id'].values\n",
    "    scores = top_fimo_result_frame['score'].values\n",
    "    strands = top_fimo_result_frame['strand'].values\n",
    "    sequences = top_fimo_result_frame['sequence'].values\n",
    "    starts = top_fimo_result_frame['start'].values\n",
    "    ends = top_fimo_result_frame['stop'].values\n",
    "    \n",
    "    for i in range(len(unique_peak_ids)):\n",
    "        currentPeakID = unique_peak_ids[i]\n",
    "        currentScore = float(scores[i])\n",
    "        currentSequence = sequences[i]\n",
    "        currentStrand = strands[i]\n",
    "        currentStart = int(starts[i])\n",
    "        currentEnd = int(ends[i])\n",
    "        # bundle values\n",
    "        \n",
    "        if currentScore < 0.0:\n",
    "            currentScore = 0.0\n",
    "        \n",
    "        newValues = (currentScore, \n",
    "                     currentSequence, \n",
    "                     currentStrand, \n",
    "                     currentStart, \n",
    "                     currentEnd, \n",
    "                     )\n",
    "        id_values_dict[currentPeakID] = newValues\n",
    "    \n",
    "    # sort values according to all peak IDs\n",
    "    sorted_values = [id_values_dict[x] if x in id_values_dict else (0,'','?',-1,-1) for x in  all_peak_ids]\n",
    "    sorted_scores = [x[0] for x in sorted_values]\n",
    "    sorted_sequences = [x[1] for x in sorted_values]\n",
    "    sorted_strands = [x[2] for x in sorted_values]\n",
    "    sorted_starts = [x[3] for x in sorted_values]\n",
    "    sorted_ends = [x[4] for x in sorted_values]\n",
    "    \n",
    "    # count the number of motif instances\n",
    "    peak_ids = fimo_result_frame['peak_id'].values\n",
    "    id_count_dict = Counter(peak_ids)\n",
    "    \n",
    "    sorted_counts = [id_count_dict[x] if x in id_count_dict else 0 for x in all_peak_ids]\n",
    "    del fimo_result_frame # release result frame from memory hopefully\n",
    "    while thread_lock.locked_lock():\n",
    "        pass\n",
    "    thread_lock.acquire()\n",
    "        \n",
    "    motif_score_dict[motif_name] = sorted_scores\n",
    "    motif_sequence_dict[motif_name] = sorted_sequences\n",
    "    motif_strand_dict[motif_name] = sorted_strands\n",
    "    motif_start_dict[motif_name] = sorted_starts\n",
    "    motif_end_dict[motif_name] = sorted_ends\n",
    "    motif_count_dict[motif_name] = sorted_counts\n",
    "    # release lock\n",
    "    print('finished reading', motif_name )\n",
    "    if thread_lock.locked_lock():\n",
    "        thread_lock.release()\n",
    "    \n",
    "    return None"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "***** C57BL6J-background.fa *****\n",
      "C57BL6J-background_ALX3.txt\n",
      "C57BL6J-background_ARNT::HIF1A.txt\n",
      "C57BL6J-background_ATF4.txt\n",
      "C57BL6J-background_ATF7.txt\n",
      "C57BL6J-background_Ahr::Arnt.txt\n",
      "C57BL6J-background_Alx1.txt\n",
      "C57BL6J-background_Alx4.txt\n",
      "C57BL6J-background_Ar.txt\n",
      "C57BL6J-background_Arid3a.txt\n",
      "C57BL6J-background_Arid3b.txt\n",
      "C57BL6J-background_Arid5a.txt\n",
      "C57BL6J-background_Arnt.txt\n",
      "C57BL6J-background_Arntl.txt\n",
      "C57BL6J-background_Arx.txt\n",
      "C57BL6J-background_Ascl2.txt\n",
      "C57BL6J-background_Atf1.txt\n",
      "C57BL6J-background_Atf3.txt\n",
      "C57BL6J-background_Atoh1.txt\n",
      "C57BL6J-background_BARHL2.txt\n",
      "C57BL6J-background_BARX1.txt\n",
      "C57BL6J-background_BATF3.txt\n",
      "C57BL6J-background_BATF::JUN.txt\n",
      "C57BL6J-background_BCL6B.txt\n",
      "C57BL6J-background_BHLHE22.txt\n",
      "C57BL6J-background_BHLHE23.txt\n",
      "C57BL6J-background_BHLHE40.txt\n",
      "C57BL6J-background_BHLHE41.txt\n",
      "C57BL6J-background_BSX.txt\n",
      "C57BL6J-background_Bach1::Mafk.txt\n",
      "C57BL6J-background_Barhl1.txt\n",
      "C57BL6J-background_Bcl6.txt\n",
      "C57BL6J-background_Bhlha15.txt\n",
      "C57BL6J-background_CDX1.txt\n",
      "C57BL6J-background_CDX2.txt\n",
      "C57BL6J-background_CEBPA.txt\n",
      "C57BL6J-background_CEBPB.txt\n",
      "C57BL6J-background_CEBPD.txt\n",
      "C57BL6J-background_CEBPE.txt\n",
      "C57BL6J-background_CEBPG.txt\n",
      "C57BL6J-background_CENPB.txt\n",
      "C57BL6J-background_CLOCK.txt\n",
      "C57BL6J-background_CREB1.txt\n",
      "C57BL6J-background_CREB3.txt\n",
      "C57BL6J-background_CREB3L1.txt\n",
      "C57BL6J-background_CTCF.txt\n",
      "C57BL6J-background_CUX1.txt\n",
      "C57BL6J-background_CUX2.txt\n",
      "C57BL6J-background_Creb3l2.txt\n",
      "C57BL6J-background_Creb5.txt\n",
      "C57BL6J-background_Crem.txt\n",
      "C57BL6J-background_Crx.txt\n",
      "C57BL6J-background_DBP.txt\n",
      "C57BL6J-background_DLX6.txt\n",
      "C57BL6J-background_DMRT3.txt\n",
      "C57BL6J-background_DUX4.txt\n",
      "C57BL6J-background_DUXA.txt\n",
      "C57BL6J-background_Ddit3::Cebpa.txt\n",
      "C57BL6J-background_Dlx1.txt\n",
      "C57BL6J-background_Dlx2.txt\n",
      "C57BL6J-background_Dlx3.txt\n",
      "C57BL6J-background_Dlx4.txt\n",
      "C57BL6J-background_Dmbx1.txt\n",
      "C57BL6J-background_Dux.txt\n",
      "C57BL6J-background_E2F1.txt\n",
      "C57BL6J-background_E2F2.txt\n",
      "C57BL6J-background_E2F3.txt\n",
      "C57BL6J-background_E2F4.txt\n",
      "C57BL6J-background_E2F6.txt\n",
      "C57BL6J-background_E2F7.txt\n",
      "C57BL6J-background_E2F8.txt\n",
      "C57BL6J-background_EBF1.txt\n",
      "C57BL6J-background_EGR1.txt\n",
      "C57BL6J-background_EGR2.txt\n",
      "C57BL6J-background_EGR3.txt\n",
      "C57BL6J-background_EGR4.txt\n",
      "C57BL6J-background_EHF.txt\n",
      "C57BL6J-background_ELF1.txt\n",
      "C57BL6J-background_ELF3.txt\n",
      "C57BL6J-background_ELF4.txt\n",
      "C57BL6J-background_ELF5.txt\n",
      "C57BL6J-background_ELK1.txt\n",
      "C57BL6J-background_ELK3.txt\n",
      "C57BL6J-background_ELK4.txt\n",
      "C57BL6J-background_EMX1.txt\n",
      "C57BL6J-background_EMX2.txt\n",
      "C57BL6J-background_EN1.txt\n",
      "C57BL6J-background_EN2.txt\n",
      "C57BL6J-background_EOMES.txt\n",
      "C57BL6J-background_ERF.txt\n",
      "C57BL6J-background_ERG.txt\n",
      "C57BL6J-background_ESR1.txt\n",
      "C57BL6J-background_ESR2.txt\n",
      "C57BL6J-background_ESRRB.txt\n",
      "C57BL6J-background_ESX1.txt\n",
      "C57BL6J-background_ETS1.txt\n",
      "C57BL6J-background_ETV1.txt\n",
      "C57BL6J-background_ETV2.txt\n",
      "C57BL6J-background_ETV3.txt\n",
      "C57BL6J-background_ETV4.txt\n",
      "C57BL6J-background_ETV5.txt\n",
      "C57BL6J-background_ETV6.txt\n",
      "C57BL6J-background_EVX1.txt\n",
      "C57BL6J-background_EVX2.txt\n",
      "C57BL6J-background_EWSR1-FLI1.txt\n",
      "C57BL6J-background_Esrra.txt\n",
      "C57BL6J-background_Esrrg.txt\n",
      "C57BL6J-background_FEV.txt\n",
      "C57BL6J-background_FIGLA.txt\n",
      "C57BL6J-background_FLI1.txt\n",
      "C57BL6J-background_FOS.txt\n",
      "C57BL6J-background_FOS::JUN.txt\n",
      "C57BL6J-background_FOSL1.txt\n",
      "C57BL6J-background_FOSL2.txt\n",
      "C57BL6J-background_FOXA1.txt\n",
      "C57BL6J-background_FOXB1.txt\n",
      "C57BL6J-background_FOXC1.txt\n",
      "C57BL6J-background_FOXC2.txt\n",
      "C57BL6J-background_FOXD1.txt\n",
      "C57BL6J-background_FOXD2.txt\n",
      "C57BL6J-background_FOXF2.txt\n",
      "C57BL6J-background_FOXG1.txt\n",
      "C57BL6J-background_FOXH1.txt\n",
      "C57BL6J-background_FOXI1.txt\n",
      "C57BL6J-background_FOXL1.txt\n",
      "C57BL6J-background_FOXO3.txt\n",
      "C57BL6J-background_FOXO4.txt\n",
      "C57BL6J-background_FOXO6.txt\n",
      "C57BL6J-background_FOXP1.txt\n",
      "C57BL6J-background_FOXP2.txt\n",
      "C57BL6J-background_FOXP3.txt\n",
      "C57BL6J-background_Foxa2.txt\n",
      "C57BL6J-background_Foxd3.txt\n",
      "C57BL6J-background_Foxj2.txt\n",
      "C57BL6J-background_Foxj3.txt\n",
      "C57BL6J-background_Foxk1.txt\n",
      "C57BL6J-background_Foxo1.txt\n",
      "C57BL6J-background_Foxq1.txt\n",
      "reading Atf1\n",
      "C57BL6J-background_GATA1::TAL1.txt\n",
      "C57BL6J-background_GATA2.txt\n",
      "C57BL6J-background_GATA3.txt\n",
      "C57BL6J-background_GATA5.txt\n",
      "C57BL6J-background_GBX1.txt\n",
      "C57BL6J-background_GBX2.txt\n",
      "C57BL6J-background_GCM1.txt\n",
      "C57BL6J-background_GCM2.txt\n",
      "C57BL6J-background_GLI2.txt\n",
      "C57BL6J-background_GLIS1.txt\n",
      "C57BL6J-background_GLIS2.txt\n",
      "C57BL6J-background_GLIS3.txt\n",
      "C57BL6J-background_GMEB2.txt\n",
      "C57BL6J-background_GRHL1.txt\n",
      "C57BL6J-background_GSC.txt\n",
      "C57BL6J-background_GSC2.txt\n",
      "C57BL6J-background_GSX1.txt\n",
      "C57BL6J-background_GSX2.txt\n",
      "C57BL6J-background_Gabpa.txt\n",
      "C57BL6J-background_Gata1.txt\n",
      "C57BL6J-background_Gata4.txt\n",
      "C57BL6J-background_Gfi1.txt\n",
      "C57BL6J-background_Gfi1b.txt\n",
      "C57BL6J-background_Gmeb1.txt\n",
      "C57BL6J-background_HES5.txt\n",
      "C57BL6J-background_HES7.txt\n",
      "C57BL6J-background_HESX1.txt\n",
      "C57BL6J-background_HEY1.txt\n",
      "C57BL6J-background_HEY2.txt\n",
      "C57BL6J-background_HIC2.txt\n",
      "reading CUX1\n",
      "C57BL6J-background_HINFP.txt\n",
      "C57BL6J-background_HLF.txt\n",
      "reading BARHL2\n",
      "C57BL6J-background_HLTF.txtreading ALX3\n",
      "\n",
      "reading BARX1\n",
      "C57BL6J-background_HMBOX1.txt\n",
      "C57BL6J-background_HNF1A.txt\n",
      "reading BSX\n",
      "C57BL6J-background_HNF1B.txt\n",
      "readingC57BL6J-background_HNF4G.txt\n",
      "reading Barhl1\n",
      "C57BL6J-background_HOXA10.txt\n",
      " CUX2\n",
      "C57BL6J-background_HOXA13.txt\n",
      "C57BL6J-background_HOXA2.txt\n",
      "C57BL6J-background_HOXA5.txt\n",
      "C57BL6J-background_HOXB13.txt\n",
      "reading Arid3b\n",
      "reading Arid5a\n",
      "C57BL6J-background_HOXB2.txt\n",
      "reading E2F3\n",
      "C57BL6J-background_HOXB3.txt\n",
      "reading Alx1\n",
      "reading Crem\n",
      "C57BL6J-background_HOXC10.txt\n",
      "C57BL6J-background_HOXC11.txt\n",
      "C57BL6J-background_HOXC12.txt\n",
      "C57BL6J-background_HOXC13.txt\n",
      "C57BL6J-background_HOXD11.txt\n",
      "C57BL6J-background_HOXD12.txt\n",
      "C57BL6J-background_HOXD13.txt\n",
      "C57BL6J-background_HSF1.txt\n",
      "C57BL6J-background_HSF2.txt\n",
      "C57BL6J-background_HSF4.txt\n",
      "C57BL6J-background_Hand1::Tcf3.txt\n",
      "C57BL6J-background_Hes1.txt\n",
      "C57BL6J-background_Hes2.txt\n",
      "reading DBP\n",
      "C57BL6J-background_Hic1.txt\n",
      "C57BL6J-background_Hmx1.txt\n",
      "C57BL6J-background_Hmx2.txt\n",
      "C57BL6J-background_Hmx3.txt\n",
      "C57BL6J-background_Hnf4a.txt\n",
      "reading Arx\n",
      "reading CEBPG\n",
      "C57BL6J-background_Hoxa11.txt\n",
      "C57BL6J-background_Hoxa9.txt\n",
      "reading E2F2\n",
      "C57BL6J-background_Hoxb5.txt\n",
      "reading Alx4\n",
      "C57BL6J-background_Hoxc9.txt\n",
      "C57BL6J-background_Hoxd3.txt\n",
      "reading Arnt\n",
      "C57BL6J-background_Hoxd8.txt\n",
      "C57BL6J-background_Hoxd9.txt\n",
      "reading ATF7\n",
      "reading Creb5\n",
      "C57BL6J-background_ID4.txt\n",
      "reading Arid3a\n",
      "reading Atf3\n",
      "reading DLX6\n",
      "C57BL6J-background_INSM1.txt\n",
      "C57BL6J-background_IRF1.txt\n",
      "C57BL6J-background_IRF2.txt\n",
      "C57BL6J-background_IRF7.txt\n",
      "reading CREB3\n",
      "C57BL6J-background_IRF8.txt\n",
      "reading Dlx2\n",
      "C57BL6J-background_IRF9.txt\n",
      "reading Dlx3\n",
      "reading CEBPD\n",
      "reading CDX1\n",
      "C57BL6J-background_ISL2.txt\n",
      "C57BL6J-background_ISX.txt\n",
      "C57BL6J-background_Id2.txt\n",
      "C57BL6J-background_JDP2.txt\n",
      "reading ATF4\n",
      "C57BL6J-background_JDP2_var2.txtreading Dlx1\n",
      "reading CENPB\n",
      "\n",
      "C57BL6J-background_JUN.txt\n",
      "C57BL6J-background_JUNB.txt\n",
      "reading ESX1\n",
      "reading CEBPE\n",
      "reading Dlx4\n",
      "C57BL6J-background_JUND.txt\n",
      "reading Dmbx1\n",
      "C57BL6J-background_JUND_var2.txt\n",
      "reading CEBPB\n",
      "C57BL6J-background_JUN_var2.txt\n",
      "reading DUXA\n",
      "C57BL6J-background_KLF13.txt\n",
      "C57BL6J-background_KLF14.txt\n",
      "reading EMX1\n",
      "C57BL6J-background_KLF16.txt\n",
      "C57BL6J-background_KLF5.txt\n",
      "C57BL6J-background_Klf1.txt\n",
      "C57BL6J-background_Klf12.txt\n",
      "C57BL6J-background_Klf4.txt\n",
      "C57BL6J-background_LBX1.txt\n",
      "reading EN1\n",
      "C57BL6J-background_LBX2.txt\n",
      "reading EVX2\n",
      "C57BL6J-background_LEF1.txt\n",
      "reading EVX1\n",
      "reading EMX2\n",
      "reading BHLHE22\n",
      "C57BL6J-background_LHX2.txt\n",
      "C57BL6J-background_LHX6.txt\n",
      "C57BL6J-background_LHX9.txt\n",
      "reading Dux\n",
      "C57BL6J-background_LIN54.txt\n",
      "reading E2F1\n",
      "C57BL6J-background_LMX1A.txt\n",
      "C57BL6J-background_LMX1B.txt\n",
      "reading DUX4\n",
      "C57BL6J-background_Lhx3.txt\n",
      "C57BL6J-background_Lhx4.txt\n",
      "C57BL6J-background_Lhx8.txt\n",
      "reading GMEB2\n",
      "reading BHLHE23\n",
      "C57BL6J-background_MAF::NFE2.txt\n",
      "reading BATF::JUN\n",
      "reading E2F7\n",
      "C57BL6J-background_MAFF.txt\n",
      "C57BL6J-background_MAFG.txt\n",
      "C57BL6J-background_MAFG::NFE2L1.txt\n",
      "reading Gmeb1\n",
      "C57BL6J-background_MAFK.txt\n",
      "C57BL6J-background_MAX.txt\n",
      "C57BL6J-background_MAX::MYC.txt\n",
      "C57BL6J-background_MEF2A.txt\n",
      "C57BL6J-background_MEF2B.txt\n",
      "C57BL6J-background_MEF2C.txt\n",
      "reading BHLHE41\n",
      "C57BL6J-background_MEF2D.txt\n",
      "C57BL6J-background_MEIS1.txt\n",
      "reading BHLHE40\n",
      "C57BL6J-background_MEIS2.txt\n",
      "reading CLOCK\n",
      "C57BL6J-background_MEIS3.txt\n",
      "reading BATF3\n",
      "reading BCL6B\n",
      "C57BL6J-background_MEOX1.txt\n",
      "C57BL6J-background_MEOX2.txt\n",
      "C57BL6J-background_MGA.txt\n",
      "C57BL6J-background_MIXL1.txt\n",
      "C57BL6J-background_MLX.txt\n",
      "C57BL6J-background_MLXIPL.txt\n",
      "reading DMRT3\n",
      "C57BL6J-background_MNT.txt\n",
      "reading CREB1\n",
      "C57BL6J-background_MNX1.txt\n",
      "C57BL6J-background_MSC.txt\n",
      "C57BL6J-background_MSX1.txt\n",
      "C57BL6J-background_MSX2.txt\n",
      "C57BL6J-background_MTF1.txt\n",
      "C57BL6J-background_MYBL1.txt\n",
      "C57BL6J-background_MYBL2.txt\n",
      "C57BL6J-background_MYF6.txt\n",
      "C57BL6J-background_MZF1.txt\n",
      "C57BL6J-background_MZF1_var2.txt\n",
      "C57BL6J-background_Mafb.txt\n",
      "readingC57BL6J-background_Mecom.txt\n",
      "reading Ddit3::Cebpa\n",
      "reading GATA5\n",
      "reading CREB3L1\n",
      " CDX2\n",
      "C57BL6J-background_Mitf.txt\n",
      "C57BL6J-background_Mlxip.txt\n",
      "C57BL6J-background_Msx3.txt\n",
      "C57BL6J-background_Myb.txt\n",
      "C57BL6J-background_Myc.txt\n",
      "C57BL6J-background_Mycn.txt\n",
      "C57BL6J-background_Myod1.txt\n",
      "reading GBX1\n",
      "reading Crx\n",
      "reading Ahr::Arnt\n",
      "reading GBX2\n",
      "reading Arntl\n",
      "reading CEBPA\n",
      "reading Creb3l2\n",
      "reading ETV4\n",
      "reading GSX1\n",
      "reading Hoxa11\n",
      "reading FOXB1\n",
      "C57BL6J-background_Myog.txt\n",
      "C57BL6J-background_NEUROD2.txt\n",
      "C57BL6J-background_NEUROG2.txt\n",
      "C57BL6J-background_NFAT5.txt\n",
      "C57BL6J-background_NFATC1.txt\n",
      "C57BL6J-background_NFATC2.txt\n",
      "reading EN2\n",
      "C57BL6J-background_NFATC3.txt\n",
      "C57BL6J-background_NFE2.txt\n",
      "reading Ar\n",
      "reading GSC\n",
      "reading HOXD12\n",
      "C57BL6J-background_NFIA.txt\n",
      "reading ARNT::HIF1A\n",
      "C57BL6J-background_NFIC.txt\n",
      "reading ETV3\n",
      "C57BL6J-background_NFIC::TLX1.txt\n",
      "C57BL6J-background_NFIL3.txt\n",
      "C57BL6J-background_NFIX.txt\n",
      "C57BL6J-background_NFKB1.txt\n",
      "C57BL6J-background_NFKB2.txt\n",
      "C57BL6J-background_NFYA.txt\n",
      "C57BL6J-background_NFYB.txt\n",
      "C57BL6J-background_NHLH1.txt\n",
      "reading GSX2\n",
      "C57BL6J-background_NKX2-3.txt\n",
      "C57BL6J-background_NKX2-8.txt\n",
      "reading HOXA2\n",
      "C57BL6J-background_NKX3-2.txt\n",
      "C57BL6J-background_NKX6-1.txt\n",
      "reading HOXC11\n",
      "reading ELK1\n",
      "C57BL6J-background_NKX6-2.txt\n",
      "reading ESRRB\n",
      "C57BL6J-background_NOTO.txt\n",
      "readingreading Hmx3\n",
      "C57BL6J-background_NR1H2::RXRA.txt\n",
      " ETV1\n",
      "C57BL6J-background_NR2C2.txt\n",
      "reading HESX1\n",
      "reading ELF1\n",
      "reading FOXP3\n",
      "reading GSC2\n",
      "C57BL6J-background_NR2F1.txt\n",
      "C57BL6J-background_NR3C1.txt\n",
      "C57BL6J-background_NR3C2.txt\n",
      "C57BL6J-background_NR4A2.txt\n",
      "C57BL6J-background_NRF1.txt\n",
      "C57BL6J-background_NRL.txt\n",
      "C57BL6J-background_Neurog1.txt\n",
      "reading FOXG1\n",
      "C57BL6J-background_Nfe2l2.txt\n",
      "reading HOXC10\n",
      "C57BL6J-background_Nkx2-5.txt\n",
      "C57BL6J-background_Nkx2-5_var2.txt\n",
      "C57BL6J-background_Nkx3-1.txt\n",
      "reading Bcl6\n",
      "C57BL6J-background_Nobox.txt\n",
      "C57BL6J-background_Npas2.txtreading FOSL2\n",
      "\n",
      "C57BL6J-background_Nr1h3::Rxra.txt\n",
      "C57BL6J-background_Nr2e1.txt\n",
      "reading FOXF2\n",
      "reading EOMES\n",
      "C57BL6J-background_Nr2e3.txt\n",
      "C57BL6J-background_Nr2f6.txt\n",
      "C57BL6J-background_Nr2f6_var2.txt\n",
      "reading HOXC12\n",
      "reading FOXH1\n",
      "reading GRHL1\n",
      "C57BL6J-background_Nr5a2.txt\n",
      "reading HOXB3\n",
      "C57BL6J-background_OLIG1.txt\n",
      "C57BL6J-background_OLIG2.txt\n",
      "reading E2F8\n",
      "C57BL6J-background_OLIG3.txt\n",
      "reading HLF\n",
      "C57BL6J-background_ONECUT1.txt\n",
      "C57BL6J-background_ONECUT2.txt\n",
      "C57BL6J-background_ONECUT3.txt\n",
      "C57BL6J-background_OTX1.txt\n",
      "C57BL6J-background_OTX2.txt\n",
      "reading FOXI1\n",
      "C57BL6J-background_PAX1.txt\n",
      "reading Foxd3\n",
      "reading FOS\n",
      "reading HNF1B\n",
      "reading HNF1A\n",
      "C57BL6J-background_PAX3.txt\n",
      "reading Atoh1\n",
      "reading ELK3\n",
      "reading Hoxd8\n",
      "reading ISX\n"
     ]
    }
   ],
   "source": [
    "start = time.time()\n",
    "fimo_result_path = './fimo_results/'\n",
    "for fastaFile in os.listdir('./peak_sequences/'):\n",
    "    if not 'marge' in fastaFile:\n",
    "        print('*****',fastaFile,'*****')\n",
    "        strain = fastaFile.split('.')[0]\n",
    "        motif_score_dict = {}\n",
    "        motif_sequence_dict ={}\n",
    "        motif_strand_dict = {}\n",
    "        motif_start_dict = {}\n",
    "        motif_end_dict = {}\n",
    "        motif_count_dict = {}\n",
    "        thread_list = []\n",
    "        \n",
    "        for current_thread in thread_list:\n",
    "            current_thread.join()\n",
    "        if 'background' in fastaFile:\n",
    "            current_summary_frame = summary_background_frame\n",
    "        else:\n",
    "            current_summary_frame = summary_frame\n",
    "        all_peak_ids = current_summary_frame.index.values\n",
    "        lock = threading.Lock()\n",
    "        for m in sorted(os.listdir('./fimo_results')):\n",
    "            if (strain + '_') in m:\n",
    "                try:\n",
    "                    print(m)\n",
    "                    current_thread = threading.Thread(target = read_fimo_file,\n",
    "                                     args =(lock,\n",
    "                                            fimo_result_path + '/' + m, \n",
    "                                            all_peak_ids,\n",
    "                                            motif_score_dict,\n",
    "                                            motif_sequence_dict,\n",
    "                                            motif_strand_dict,\n",
    "                                            motif_start_dict,\n",
    "                                            motif_end_dict,\n",
    "                                            motif_count_dict,\n",
    "                                            ))\n",
    "                    thread_list.append(current_thread)\n",
    "                    current_thread.start()\n",
    "                finally:\n",
    "                    if lock.locked_lock():\n",
    "                        lock.release()\n",
    "\n",
    "        # convert dictionaries to data frames\n",
    "        motif_score_frame = pd.DataFrame(motif_score_dict , index = current_summary_frame['ID'].values)\n",
    "        motif_sequence_frame = pd.DataFrame(motif_sequence_dict , index = current_summary_frame['ID'].values)\n",
    "        motif_strand_frame = pd.DataFrame(motif_strand_dict , index = current_summary_frame['ID'].values)\n",
    "        motif_start_frame = pd.DataFrame(motif_start_dict , index = current_summary_frame['ID'].values)\n",
    "        motif_end_frame = pd.DataFrame(motif_end_dict , index = current_summary_frame['ID'].values)\n",
    "        motif_count_frame = pd.DataFrame(motif_count_dict , index = current_summary_frame['ID'].values)\n",
    "\n",
    "        for frame in [motif_score_frame, \n",
    "                      motif_sequence_frame, \n",
    "                      motif_strand_frame, \n",
    "                      motif_start_frame, \n",
    "                      motif_end_frame, \n",
    "                      motif_count_frame]:\n",
    "            motif_cols = frame.columns.values\n",
    "            frame['ID'] = current_summary_frame['ID'].values\n",
    "            frame['Factors'] = current_summary_frame['Factors'].values\n",
    "            frame['chr'] = current_summary_frame['chr'].values\n",
    "\n",
    "        motif_score_frame.to_pickle('motif_score_unclustered_frame_'+  strain + '.pickle')\n",
    "\n",
    "        motif_sequence_frame.to_pickle('motif_sequence_unclustered_frame_'+  strain + '.pickle')\n",
    "\n",
    "        motif_strand_frame.to_pickle('motif_strand_unclustered_frame_'+  strain + '.pickle')\n",
    "\n",
    "        motif_start_frame.to_pickle('motif_start_unclustered_frame_'+  strain + '.pickle')\n",
    "\n",
    "        motif_end_frame.to_pickle('motif_end_unclustered_frame_'+  strain + '.pickle')\n",
    "\n",
    "        motif_count_frame.to_pickle('motif_count_unclustered_frame_'+  strain + '.pickle')\n",
    "        \n",
    "end = time.time()\n",
    "print('total time', end - start)\n",
    "\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "motif_score_frame.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python [Root]",
   "language": "python",
   "name": "Python [Root]"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
